# Multi-omic modelling of inflammatory bowel disease with regularized canonical correlation analysis

The canonical correlation is a method that using data from the same sample but from different datasets.

Characteristics

Input Data type: numeric

More variables than samples: It is appropriate when the number of variables is higher than the number of samples.
Need to be a complete case.
Time is not considered as a specially variable

Relationship between variables

Output:

Specific factors Interpretation:

Depending on the model and options used.

Over several years of progress [@tenenhaus_component-based_2008; @esposito_vinzi_bridge_2010; @tenenhaus_regularized_2011; @tenenhaus_regularized_2014; @tenenhaus_variable_2014; @tenenhaus_kernel_2015] on the field of canonical correlations it provides a robust method and there is an implementation [@R-RGCCA].

The regularized generalized canonical correlation is a method that combines several datasets, using data from the same sample.
It is a good choice when the number of variables is higher than samples.
There have been some improvements to generalize the method when $p \gg n$.
Current practices include using a pre selected model of relations between blocks.
However this model might not be accurate.
To help find the fitting model for the data I created an R package.
The package, which is named [inteRmodel](https://llrs.github.io/inteRmodel/) helps finding the right model and how fit it is for your data.

This method applied to an existing dataset of an autologous haematopoietic stem cell transplantation [@corralizaDifferencesPeripheralTissue].
From this dataset there is data about the human transcriptome and the 16S DNA present at biopsies from colonoscopy.
For several patients we have samples at different time point and at different locations.
This allows us to see both location and time differences.

We looked to several models and searched for the model that fit better with the data.
The first model just accounted the transcriptomics and the microbiome data, then in another family of models we added the information we know about those samples.
In a further family of models we split the information we know about these samples into three different blocks grouping them according to how are they related between them.
We fitted the best model for all the three family of models and we found that the most fitting model was from the family that had the data split on several.

Additionally I introduced some changes to RGCCA package.
The modified package can be found [here](https://github.com/llrs/RGCCA/).
This version return the same results as the version on CRAN but provides also some code optimization for a lower computation time.
Some of these changes are to to be able to change the design between different dimensions.
The idea behind this change is that if the first dimensions correctly fits the data the second dimension might need a different model of relationships.
For instance, the first dimensions are dominated by two blocks of data while the second is dominated by another block of data.

## Relationships' models

Most multi-omics and integration tools assume one block for each type of data.
However, RGCCA uses linear relationships between blocks of data.
But doesn't say what constitutes a block of data.
Usually a block of data is just a source of data, such as an essay a survey or an experiment.
We decided to split the block with data about the samples to separate independent variables from the same block.
The hypothesis we made was that with independent blocks the combined linear model would fit better the data.

These news blocks can also help make more interpretable the relationships, because instead of a big metadata block we have a block for time related variables, another one for location and so on.
This allows to design a model with an expected relationships between these dimensions.

However, one limitation of the generalized canonical correlation analysis an in general on the dimension reduction methods is that the design matrix must be symmetric.
Which implies that we can't infer causal relationship using these methods.

### Designing models

What defines a block: Blocks of variables are treated independently but variables within a block are used in a linear combination.

Create models with random links between blocks.

Create models considering the knowledge of the system or disease.

### Evaluating models

To evaluate a model RGCCA provides the average variance explained (AVE), inner and outer.
Inner AVE is for how well do all the canonical dimensions correlate with the design of the experiment, so it a measure of how good the model is.
While outer AVE is a measure of how well do the variables of each block correlate with the canonical dimension, so it measures the agreement between the variables and the canonical dimensions.

To evaluate a design a bootstraping method can be used to know how well the design does apply to a variety of data.
Another option is to use an external cohort to validate the same model, or using an external model to see if you reach the same model, which is what we did.

Next it must be evaluated compared to other methods to see if it brings something new or not.
Of the multiple methods available we used [MCIA](https://bioconductor.org/packages/omicade4 "Omicade4") [@mengMultivariateApproachIntegration2014].
Which was compared by looking at the area under the curve for classifying the samples according to their location.

Besides a way to compare methods, these models do need to be evaluated by the insights they provide on the biological system they are being applied to, in our case the Crohn's disease.
In this article we didn't look in depth to the biological relevance of the microorganisms an genes found.

## Conclusions

The procedure of separating independent variables in their own block of data and later search the best model that fits the data provides a good strategy that should be consider for integration efforts.

The procedural method of searching a model and testing them is implemented on [inteRmodel](https://llrs.github.io/inteRmodel/ "inteRmodel").
But the most important thing is to consider which variables are independent of which and if they can be separated into a block for later usage on the modeling.

This was published as a [preprint](https://www.medrxiv.org/content/10.1101/2020.04.16.20031492v1) and after peer review [published on Plus One](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0246367 "Multi-omic modelling of inflammatory bowel disease with regularized canonical correlation analysis") [@revilla2021].
