# Conclusions

In this doctoral thesis, we assessed the relationships between the microbiota and the gut transcriptome as they interact in inflammatory bowel disease.

We are able to draw the following conclusions:

## Study 1: Multi-omic modelling of inflammatory bowel disease with regularized canonical correlation analysis

1.  Applying methods that use information concerning the samples show better results than omitting such information.
2.  Correlations are not sufficient for identifying relationship between genes and microorganisms.
3.  The `inteRmodel` method provides consistent evidence of the connections between blocks among different datasets of inflammatory bowel disease.

## Study 2: The relationship of genes and the microbiome with inflammatory bowel disease

1.  At host's transcriptomics is heavily influenced by location and, to a lesser degree, by the type of activity.
2.  The microbiome's composition more relates to the sample location and to the disease status both are almost equally important
3.  Those microorganisms closely tied to intestinal location might be indicative of at patient's disease type, (e.g., Crohn's disease, ulcerative colitis or non-\acr{IBD}), although disease-related microorganisms can be common to all all locations.
