# Conclusions

In this doctoral thesis, we assessed the relationships between the microbiota and the gut transcriptome in inflammatory bowel disease.

It allows us to conclude the following:

## Study 1: Multi-omic modelling of inflammatory bowel disease with regularized canonical correlation analysis

1.  Applying methods that use information about the samples show better results than omitting this information.
2.  Correlations are not enough to identify relationship between genes and microorganisms.
3.  The `inteRmodel` method provides consistent connections between blocks on different datasets of inflammatory bowel disease.

## Study 2: Genes and microbiome relationship on inflammatory bowel disease

1.  The host's transcriptomics is heavily influenced by location and to a lesser degree by the type of activity.
2.  The microbiome composition relates better to the sample location than to the disease status but both are almost equally important
3.  Microorganisms highly tied to intestinal location might indicate the disease type, (Crohn's disease, ulcerative colitis or non-IBD) of the patients, but microorganisms related to the disease are common in all the locations.
