<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>B.2 By publication | Data integration on inflammatory bowel disease</title>
  <meta name="description" content="Thesis about the relationship between microbiome and transcriptome on the gastrointestinal tract on patients with inflammatory bowel disease and some controls to discover key differences on this relationship." />
  <meta name="generator" content="bookdown 0.21 and GitBook 2.6.7" />

  <meta property="og:title" content="B.2 By publication | Data integration on inflammatory bowel disease" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="Thesis about the relationship between microbiome and transcriptome on the gastrointestinal tract on patients with inflammatory bowel disease and some controls to discover key differences on this relationship." />
  <meta name="github-repo" content="llrs/thesis" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="B.2 By publication | Data integration on inflammatory bowel disease" />
  <meta name="twitter:site" content="@Lluis_Revilla" />
  <meta name="twitter:description" content="Thesis about the relationship between microbiome and transcriptome on the gastrointestinal tract on patients with inflammatory bowel disease and some controls to discover key differences on this relationship." />
  

<meta name="author" content="Lluís Revilla Sancho" />


<meta name="date" content="2021-02-10" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="listed.html"/>

<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />












<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><strong><a href="./">Data integration on IBD</a></strong></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Preface</a></li>
<li class="chapter" data-level="1" data-path="introduction.html"><a href="introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a><ul>
<li class="chapter" data-level="1.1" data-path="integration.html"><a href="integration.html"><i class="fa fa-check"></i><b>1.1</b> Integration</a><ul>
<li class="chapter" data-level="1.1.1" data-path="integration.html"><a href="integration.html#classification-of-integration-methods"><i class="fa fa-check"></i><b>1.1.1</b> Classification of integration method's</a></li>
<li class="chapter" data-level="1.1.2" data-path="integration.html"><a href="integration.html#reviews-and-benchmarking"><i class="fa fa-check"></i><b>1.1.2</b> Reviews and benchmarking</a></li>
<li class="chapter" data-level="1.1.3" data-path="integration.html"><a href="integration.html#summary"><i class="fa fa-check"></i><b>1.1.3</b> Summary</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="IBD.html"><a href="IBD.html"><i class="fa fa-check"></i><b>1.2</b> Inflammatory bowel disease</a><ul>
<li class="chapter" data-level="1.2.1" data-path="IBD.html"><a href="IBD.html#disease-etiology"><i class="fa fa-check"></i><b>1.2.1</b> Disease etiology</a></li>
<li class="chapter" data-level="1.2.2" data-path="IBD.html"><a href="IBD.html#crohns-disease"><i class="fa fa-check"></i><b>1.2.2</b> Crohn's disease</a></li>
<li class="chapter" data-level="1.2.3" data-path="IBD.html"><a href="IBD.html#ulcerative-colitis"><i class="fa fa-check"></i><b>1.2.3</b> Ulcerative colitis</a></li>
<li class="chapter" data-level="1.2.4" data-path="IBD.html"><a href="IBD.html#clinical-care"><i class="fa fa-check"></i><b>1.2.4</b> Clinical care</a></li>
<li class="chapter" data-level="1.2.5" data-path="IBD.html"><a href="IBD.html#summary-1"><i class="fa fa-check"></i><b>1.2.5</b> Summary</a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="integration-on-ibd.html"><a href="integration-on-ibd.html"><i class="fa fa-check"></i><b>1.3</b> Integration on IBD</a><ul>
<li class="chapter" data-level="1.3.1" data-path="integration-on-ibd.html"><a href="integration-on-ibd.html#genetics"><i class="fa fa-check"></i><b>1.3.1</b> Genetics</a></li>
<li class="chapter" data-level="1.3.2" data-path="integration-on-ibd.html"><a href="integration-on-ibd.html#microbiome"><i class="fa fa-check"></i><b>1.3.2</b> Microbiome</a></li>
<li class="chapter" data-level="1.3.3" data-path="integration-on-ibd.html"><a href="integration-on-ibd.html#multiple-data"><i class="fa fa-check"></i><b>1.3.3</b> Multiple data</a></li>
<li class="chapter" data-level="1.3.4" data-path="integration-on-ibd.html"><a href="integration-on-ibd.html#summary-2"><i class="fa fa-check"></i><b>1.3.4</b> Summary</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="2" data-path="multi-omic-modelling-of-inflammatory-bowel-disease-with-regularized-canonical-correlation-analysis.html"><a href="multi-omic-modelling-of-inflammatory-bowel-disease-with-regularized-canonical-correlation-analysis.html"><i class="fa fa-check"></i><b>2</b> Multi-omic modelling of inflammatory bowel disease with regularized canonical correlation analysis</a><ul>
<li class="chapter" data-level="2.1" data-path="relationships-models.html"><a href="relationships-models.html"><i class="fa fa-check"></i><b>2.1</b> Relationships' models</a><ul>
<li class="chapter" data-level="2.1.1" data-path="relationships-models.html"><a href="relationships-models.html#designing-models"><i class="fa fa-check"></i><b>2.1.1</b> Designing models</a></li>
<li class="chapter" data-level="2.1.2" data-path="relationships-models.html"><a href="relationships-models.html#evaluating-models"><i class="fa fa-check"></i><b>2.1.2</b> Evaluating models</a></li>
</ul></li>
<li class="chapter" data-level="2.2" data-path="conclusions.html"><a href="conclusions.html"><i class="fa fa-check"></i><b>2.2</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="biological-relevance-of-results.html"><a href="biological-relevance-of-results.html"><i class="fa fa-check"></i><b>3</b> Biological relevance of results</a><ul>
<li class="chapter" data-level="3.1" data-path="comparing-different-dataset.html"><a href="comparing-different-dataset.html"><i class="fa fa-check"></i><b>3.1</b> Comparing different dataset</a></li>
<li class="chapter" data-level="3.2" data-path="shared-selected-variables.html"><a href="shared-selected-variables.html"><i class="fa fa-check"></i><b>3.2</b> Shared selected variables</a></li>
<li class="chapter" data-level="3.3" data-path="conclusions-1.html"><a href="conclusions-1.html"><i class="fa fa-check"></i><b>3.3</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="appendix"><span><b>Appendix</b></span></li>
<li class="chapter" data-level="A" data-path="online-resources.html"><a href="online-resources.html"><i class="fa fa-check"></i><b>A</b> Online resources</a></li>
<li class="chapter" data-level="B" data-path="software.html"><a href="software.html"><i class="fa fa-check"></i><b>B</b> Software</a><ul>
<li class="chapter" data-level="B.1" data-path="listed.html"><a href="listed.html"><i class="fa fa-check"></i><b>B.1</b> Listed</a></li>
<li class="chapter" data-level="B.2" data-path="by-publication.html"><a href="by-publication.html"><i class="fa fa-check"></i><b>B.2</b> By publication</a></li>
</ul></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Data integration on inflammatory bowel disease</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div class = "wip">
This tesis is ongoing and still without hard date to finish it. Watch the <a href="https://github.com/llrs/thesis">GitHub repository</a> or wait for a release announcement on Twitter on <a href="https://twitter.com/Lluis_Rev">my profile</a>.
</div>
<div id="by-publication" class="section level2">
<h2><span class="header-section-number">B.2</span> By publication</h2>
<p>All code of the analysis of the publications is available (in his messed state and complicated history) and a brief description as to why they were used:</p>
<p><a href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0246367">Multi-omic modelling of inflammatory bowel disease with regularized canonical correlation analysis</a>:</p>
<ul>
<li><a href="https://github.com/llrs/TRIM">TRIM</a>: Mangle with the sample, dataset, explore several methods...</li>
<li><a href="https://github.com/llrs/sgcca_hyperparameters">sgcca_hyperparameters</a>: Explore the effects of the hyperparameters on RGCCA on the provided dataset.</li>
<li><a href="https://github.com/llrs/inteRmodel">inteRmodel</a>: Package for easy repeating the methodology developed on TRIM.</li>
<li><a href="https://github.com/llrs/pouchitis">pouchitis</a>: Work with the pouchitis cohort used in this article.</li>
<li><a href="https://github.com/llrs/integration-helper">integration</a>: Package with functions that I wrote or used on different parts of exploring the TRIM dataset ended up here.</li>
</ul>
<p>Paper 2:</p>
<ul>
<li><a href="https://github.com/llrs/Barcelona">Barcelona</a>: Mangle with the sample, dataset ...</li>
<li><a href="https://github.com/llrs/inteRmodel">inteRmodel</a>: To repeat the same procedure as in the first analysis. Further modifications were done to ease the pain to look for a model when millions of models are possible.</li>
<li><a href="https://github.com/llrs/integration-helper">integration</a>: To not repeat myself for the same process.</li>
</ul>

<div id="refs" class="references">
<div>
<p>1. Wu C, Zhou F, Ren J, Li X, Jiang Y, Ma S. A Selective Review of Multi-Level Omics Data Integration Using Variable Selection. High-Throughput. 2019;8:4.</p>
</div>
<div>
<p>2. Cavill R, Jennen D, Kleinjans J, Bried’e JJ. Transcriptomic and metabolomic data integration. Briefings in Bioinformatics. 2016;17:891–901.</p>
</div>
<div>
<p>3. Chong J, Xia J. Computational approaches for integrative analysis of the metabolome and microbiome. Metabolites. 2017;7:62. doi:<a href="https://doi.org/10.3390/metabo7040062">10.3390/metabo7040062</a>.</p>
</div>
<div>
<p>4. Koh HWL, Fermin D, Vogel C, Choi KP, Ewing RM, Choi H. iOmicsPASS: Network-based integration of multiomics data for predictive subnetwork discovery. npj Systems Biology and Applications. 2019;5:1–10.</p>
</div>
<div>
<p>5. Tenenhaus A, Tenenhaus M. Regularized Generalized Canonical Correlation Analysis. Psychometrika. 2011;76:257–84.</p>
</div>
<div>
<p>6. Tenenhaus A, Philippe C, Guillemot V, Le Cao K-A, Grill J, Frouin V. Variable selection for generalized canonical correlation analysis. Biostatistics. 2014;15:569–83.</p>
</div>
<div>
<p>7. Culhane AC, Perri‘ere G, Higgins DG. Cross-platform comparison and visualisation of gene expression data using co-inertia analysis. BMC Bioinformatics. 2003;4:59.</p>
</div>
<div>
<p>8. Vito RD, Bellio R, Trippa L, Parmigiani G. Multi-study factor analysis. Biometrics. 2019;75:337–46.</p>
</div>
<div>
<p>9. Argelaguet R, Velten B, Arnol D, Dietrich S, Zenz T, Marioni JC, et al. Multi-Omics Factor Analysisa framework for unsupervised integration of multi-omics data sets. Molecular Systems Biology. 2018;14:e8124.</p>
</div>
<div>
<p>10. Gomez-Cabrero D, Tarazona S, Ferreir’os-Vidal I, Ramirez RN, Company C, Schmidt A, et al. STATegra, a comprehensive multi-omics dataset of B-cell differentiation in mouse. Scientific Data. 2019;6:1–15.</p>
</div>
<div>
<p>11. Franzosa EA, McIver LJ, Rahnavard G, Thompson LR, Schirmer M, Weingart G, et al. Species-level functional profiling of metagenomes and metatranscriptomes. Nature Methods. 2018;15:962. doi:<a href="https://doi.org/10.1038/s41592-018-0176-y">10.1038/s41592-018-0176-y</a>.</p>
</div>
<div>
<p>12. Truong DT, Franzosa EA, Tickle TL, Scholz M, Weingart G, Pasolli E, et al. MetaPhlAn2 for enhanced metagenomic taxonomic profiling. Nature Methods. 2015;12:902–3. doi:<a href="https://doi.org/10.1038/nmeth.3589">10.1038/nmeth.3589</a>.</p>
</div>
<div>
<p>13. Segata N, Izard J, Waldron L, Gevers D, Miropolsky L, Garrett WS, et al. Metagenomic biomarker discovery and explanation. Genome Biology. 2011;12:R60. doi:<a href="https://doi.org/10.1186/gb-2011-12-6-r60">10.1186/gb-2011-12-6-r60</a>.</p>
</div>
<div>
<p>14. Sherry A, Henson RK. Conducting and Interpreting Canonical Correlation Analysis in Personality Research: A User-Friendly Primer. 1981.</p>
</div>
<div>
<p>15. Chung R-H, Kang C-Y. A multi-omics data simulator for complex disease studies and its application to evaluate multi-omics data analysis methods for disease classification. GigaScience. 2019;8.</p>
</div>
<div>
<p>16. Martínez-Mira C, Conesa A, Tarazona S. MOSim: Multi-Omics Simulation in R. Preprint. Bioinformatics; 2018.</p>
</div>
<div>
<p>17. Cantini L, Zakeri P, Hernandez C, Naldi A, Thieffry D, Remy E, et al. Benchmarking joint multi-omics dimensionality reduction approaches for cancer study. bioRxiv. 2020;2020.01.14.905760.</p>
</div>
<div>
<p>18. Human Microbiome Project Consortium BA, Nelson KE, Pop M, Creasy HH, Giglio MG, Huttenhower C, et al. A framework for human microbiome research. Nature. 2012;486:215–21.</p>
</div>
<div>
<p>19. Häsler R, Sheibani-Tezerji R, Sinha A, Barann M, Rehman A, Esser D, et al. Uncoupling of mucosal gene regulation, mRNA splicing and adherent microbiota signatures in inflammatory bowel disease. Gut. 2016;gutjnl–2016–311651.</p>
</div>
<div>
<p>20. Tang MS, Bowcutt R, Leung JM, Wolff MJ, Gundra UM, Hudesman D, et al. Integrated Analysis of Biopsies from Inflammatory Bowel Disease Patients Identifies SAA1 as a Link Between Mucosal Microbes with TH17 and TH22 Cells. Inflammatory Bowel Diseases. 2017;23:1544–54.</p>
</div>
<div>
<p>21. De Souza HS, Fiocchi C, Iliopoulos D. The IBD interactome: An integrated view of aetiology, pathogenesis and therapy. 2017;14.</p>
</div>
<div>
<p>22. Hu S, Vila AV, Gacesa R, Collij V, Stevens C, Fu JM, et al. Whole exome sequencing analyses reveal gene–microbiota interactions in the context of ibd. Gut. 2021;70:285–96. doi:<a href="https://doi.org/10.1136/gutjnl-2019-319706">10.1136/gutjnl-2019-319706</a>.</p>
</div>
<div>
<p>23. Tenenhaus M. Component-based Structural Equation Modelling. Total Quality Management &amp; Business Excellence. 2008;19:871–86.</p>
</div>
<div>
<p>24. Tenenhaus M, Hanafi M. A Bridge Between PLS Path Modeling and Multi-Block Data Analysis. In: Esposito Vinzi V, Chin WW, Henseler J, Wang H, editors. Handbook of Partial Least Squares. Berlin, Heidelberg: Springer Berlin Heidelberg; 2010. pp. 99–123.</p>
</div>
<div>
<p>25. Tenenhaus A, Tenenhaus M. Regularized generalized canonical correlation analysis for multiblock or multigroup data analysis. European Journal of Operational Research. 2014;238:391–403.</p>
</div>
<div>
<p>26. Tenenhaus A, Philippe C, Frouin V. Kernel Generalized Canonical Correlation Analysis. Computational Statistics &amp; Data Analysis. 2015;90:114–31.</p>
</div>
<div>
<p>27. Corraliza AM, Ricart E, L’opez-García A, Carme Masamunt M, Veny M, Esteller M, et al. Differences in Peripheral and Tissue Immune Cell Populations Following Haematopoietic Stem Cell Transplantation in Crohn’s Disease Patients. Journal of Crohn’s and Colitis.</p>
</div>
<div>
<p>28. Meng C, Kuster B, Culhane AC, Gholami AM. A multivariate approach to the integration of multi-omics datasets. BMC Bioinformatics. 2014;15:162. doi:<a href="https://doi.org/10.1186/1471-2105-15-162">10.1186/1471-2105-15-162</a>.</p>
</div>
<div>
<p>29. Revilla L, Mayorgas A, Corraliza A, Masamunt M, Metwaly A, Haller D, et al. Multi-omic modelling of inflammatory bowel disease with regularized canonical correlation analysis. PLOS ONE. 2021;16:e0246367. doi:<a href="https://doi.org/10.1371/journal.pone.0246367">10.1371/journal.pone.0246367</a>.</p>
</div>
<div>
<p>30. Morgan X, Kabakchiev B, Waldron L, Tyler A, Tickle T, Milgrom R, et al. Associations between host gene expression, the mucosal microbiome, and clinical outcome in the pelvic pouch of patients with inflammatory bowel disease. Genome Biology. 2015;16:67. doi:<a href="https://doi.org/10.1186/s13059-015-0637-x">10.1186/s13059-015-0637-x</a>.</p>
</div>
</div>
</div>
<!-- </div> -->







            </section>

          </div>
        </div>
      </div>
<a href="listed.html" class="navigation navigation-prev navigation-unique" aria-label="Previous page"><i class="fa fa-angle-left"></i></a>

    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": {
"href": "https://github.com/llrs/thesis/"
},
"facebook": false,
"twitter": {
"href": "https://twitter.com/intent/tweet?text=Nice%20read%20of%20%40Lluis_Rev%20thesis"
},
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/llrs/thesis/edit/master/98_appendix.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": "https://github.com/llrs/thesis/blob/master/98_appendix.Rmd",
"text": null
},
"download": ["Data integration on inflammatory bowel disease.pdf", "Data integration on inflammatory bowel disease.epub"],
"toc": {
"collapse": false,
"toc_appendix": true,
"scroll_highlight": true
},
"toc_float": true
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
