<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 5 Discussion | Data integration on inflammatory bowel disease</title>
  <meta name="description" content="Chapter 5 Discussion | Data integration on inflammatory bowel disease" />
  <meta name="generator" content="bookdown 0.25 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 5 Discussion | Data integration on inflammatory bowel disease" />
  <meta property="og:type" content="book" />
  
  
  <meta name="github-repo" content="llrs/thesis" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 5 Discussion | Data integration on inflammatory bowel disease" />
  <meta name="twitter:site" content="@Lluis_Revilla" />
  
  

<meta name="author" content="Lluís Revilla Sancho" />


<meta name="date" content="2022-04-01" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="results.html"/>
<link rel="next" href="conclusions.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1, orientation=auto"/>
<meta name="description" content="Thesis searching the relationship between microbiome and transcriptome on the intestine on patients with IBD to discover key genes and bacteria."/>
<meta name="github-repo" content="llrs/thesis"/>
<meta name="twitter:title" content="Chapter 5 Discussion | Data integration on inflammatory bowel disease"/>
<meta name="twitter:description" content="Thesis searching the relationship between microbiome and transcriptome on the intestine on patients with IBD to discover key genes and bacteria."/>
<meta name="twitter:url" content="https://thesis.llrs.dev"/>
<meta name="twitter:image:alt" content="The cover of the thesis."/>
<meta name="twitter:card" content="summary_large_image"/>
<meta name="twitter:creator" content="@Lluis_Revilla"/>
<meta name="twitter:site" content="@Lluis_Revilla"/>
<meta property="og:title" content="Chapter 5 Discussion | Data integration on inflammatory bowel disease"/>
<meta property="og:description" content="Thesis searching the relationship between microbiome and transcriptome on the intestine on patients with IBD to discover key genes and bacteria."/>
<meta property="og:url" content="https://thesis.llrs.dev"/>
<meta property="og:image:alt" content="The cover of the thesis."/>
<meta property="og:type" content="book"/>
<meta property="og:locale" content="en_US"/>
<meta property="article:author" content="Llu&lt;U+00ED&gt;s Revilla Sancho"/>
<script src="libs/kePrint-0.0.1/kePrint.js"></script>
<link href="libs/lightable-0.0.1/lightable.css" rel="stylesheet" />



<style type="text/css">
/* Used with Pandoc 2.11+ new --citeproc when CSL is used */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><strong><a href="./">Data integration on IBD</a></strong></li>

<li class="divider"></li>
<li><a href="index.html#preface">Preface<span></span></a></li>
<li><a href="abstracts.html#abstracts">Abstracts<span></span></a>
<ul>
<li><a href="abstracts.html#english">English<span></span></a></li>
<li><a href="abstracts.html#spanish">Spanish<span></span></a></li>
<li><a href="abstracts.html#catalan">Catalan<span></span></a></li>
</ul></li>
<li class="chapter" data-level="1" data-path="introduction.html"><a href="introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction<span></span></a>
<ul>
<li class="chapter" data-level="1.1" data-path="introduction.html"><a href="introduction.html#IBD"><i class="fa fa-check"></i><b>1.1</b> Inflammatory bowel disease<span></span></a>
<ul>
<li class="chapter" data-level="1.1.1" data-path="introduction.html"><a href="introduction.html#etiology-and-pathogenesis"><i class="fa fa-check"></i><b>1.1.1</b> Etiology and pathogenesis<span></span></a></li>
<li class="chapter" data-level="1.1.2" data-path="introduction.html"><a href="introduction.html#CD"><i class="fa fa-check"></i><b>1.1.2</b> CD physiology<span></span></a></li>
<li class="chapter" data-level="1.1.3" data-path="introduction.html"><a href="introduction.html#UC"><i class="fa fa-check"></i><b>1.1.3</b> UC physiology<span></span></a></li>
<li class="chapter" data-level="1.1.4" data-path="introduction.html"><a href="introduction.html#treatment"><i class="fa fa-check"></i><b>1.1.4</b> Treatment<span></span></a></li>
<li class="chapter" data-level="1.1.5" data-path="introduction.html"><a href="introduction.html#summary-ibd"><i class="fa fa-check"></i><b>1.1.5</b> Summary<span></span></a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="introduction.html"><a href="introduction.html#integration-studies-on-ibd"><i class="fa fa-check"></i><b>1.2</b> Integration studies on IBD<span></span></a>
<ul>
<li class="chapter" data-level="1.2.1" data-path="introduction.html"><a href="introduction.html#data-origin"><i class="fa fa-check"></i><b>1.2.1</b> Type of data used for integration analysis<span></span></a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="introduction.html"><a href="introduction.html#integration"><i class="fa fa-check"></i><b>1.3</b> Integration<span></span></a>
<ul>
<li class="chapter" data-level="1.3.1" data-path="introduction.html"><a href="introduction.html#classification-of-integration-methods"><i class="fa fa-check"></i><b>1.3.1</b> Classification of integration methods<span></span></a></li>
<li class="chapter" data-level="1.3.2" data-path="introduction.html"><a href="introduction.html#interpretation"><i class="fa fa-check"></i><b>1.3.2</b> Interpretation<span></span></a></li>
<li class="chapter" data-level="1.3.3" data-path="introduction.html"><a href="introduction.html#reviews"><i class="fa fa-check"></i><b>1.3.3</b> Reviews<span></span></a></li>
<li class="chapter" data-level="1.3.4" data-path="introduction.html"><a href="introduction.html#methods-integration"><i class="fa fa-check"></i><b>1.3.4</b> Summary<span></span></a></li>
</ul></li>
<li class="chapter" data-level="1.4" data-path="introduction.html"><a href="introduction.html#summary"><i class="fa fa-check"></i><b>1.4</b> Summary<span></span></a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="hypothesis-and-objectives.html"><a href="hypothesis-and-objectives.html"><i class="fa fa-check"></i><b>2</b> Hypothesis and objectives<span></span></a>
<ul>
<li class="chapter" data-level="2.1" data-path="hypothesis-and-objectives.html"><a href="hypothesis-and-objectives.html#hypothesis"><i class="fa fa-check"></i><b>2.1</b> Hypothesis<span></span></a></li>
<li class="chapter" data-level="2.2" data-path="hypothesis-and-objectives.html"><a href="hypothesis-and-objectives.html#objectives"><i class="fa fa-check"></i><b>2.2</b> Objectives<span></span></a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="materials-and-methods.html"><a href="materials-and-methods.html"><i class="fa fa-check"></i><b>3</b> Materials and methods<span></span></a>
<ul>
<li class="chapter" data-level="3.1" data-path="materials-and-methods.html"><a href="materials-and-methods.html#datasets"><i class="fa fa-check"></i><b>3.1</b> Datasets<span></span></a>
<ul>
<li class="chapter" data-level="3.1.1" data-path="materials-and-methods.html"><a href="materials-and-methods.html#methods-puget"><i class="fa fa-check"></i><b>3.1.1</b> Puget’s dataset<span></span></a></li>
<li class="chapter" data-level="3.1.2" data-path="materials-and-methods.html"><a href="materials-and-methods.html#methods-hsct"><i class="fa fa-check"></i><b>3.1.2</b> HSCT dataset<span></span></a></li>
<li class="chapter" data-level="3.1.3" data-path="materials-and-methods.html"><a href="materials-and-methods.html#methods-hasler"><i class="fa fa-check"></i><b>3.1.3</b> Häsler’s dataset<span></span></a></li>
<li class="chapter" data-level="3.1.4" data-path="materials-and-methods.html"><a href="materials-and-methods.html#methods-morgan"><i class="fa fa-check"></i><b>3.1.4</b> Morgan’s dataset<span></span></a></li>
<li class="chapter" data-level="3.1.5" data-path="materials-and-methods.html"><a href="materials-and-methods.html#methods-howell"><i class="fa fa-check"></i><b>3.1.5</b> Howell’s dataset<span></span></a></li>
</ul></li>
<li class="chapter" data-level="3.2" data-path="materials-and-methods.html"><a href="materials-and-methods.html#processing"><i class="fa fa-check"></i><b>3.2</b> Sample processing<span></span></a>
<ul>
<li class="chapter" data-level="3.2.1" data-path="materials-and-methods.html"><a href="materials-and-methods.html#rna-sequencing"><i class="fa fa-check"></i><b>3.2.1</b> RNA sequencing<span></span></a></li>
<li class="chapter" data-level="3.2.2" data-path="materials-and-methods.html"><a href="materials-and-methods.html#microbial-dna-sequencing"><i class="fa fa-check"></i><b>3.2.2</b> Microbial DNA sequencing<span></span></a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="materials-and-methods.html"><a href="materials-and-methods.html#integration-methods"><i class="fa fa-check"></i><b>3.3</b> Integration methods<span></span></a>
<ul>
<li class="chapter" data-level="3.3.1" data-path="materials-and-methods.html"><a href="materials-and-methods.html#regularized-generalized-canonical-correlation-analysis"><i class="fa fa-check"></i><b>3.3.1</b> Regularized generalized canonical correlation analysis<span></span></a></li>
<li class="chapter" data-level="3.3.2" data-path="materials-and-methods.html"><a href="materials-and-methods.html#other"><i class="fa fa-check"></i><b>3.3.2</b> Other<span></span></a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="materials-and-methods.html"><a href="materials-and-methods.html#statistics"><i class="fa fa-check"></i><b>3.4</b> Statistics<span></span></a></li>
<li class="chapter" data-level="3.5" data-path="materials-and-methods.html"><a href="materials-and-methods.html#functional-enrichment-methods"><i class="fa fa-check"></i><b>3.5</b> Functional enrichment methods<span></span></a>
<ul>
<li class="chapter" data-level="3.5.1" data-path="materials-and-methods.html"><a href="materials-and-methods.html#ORA"><i class="fa fa-check"></i><b>3.5.1</b> Over representation analysis<span></span></a></li>
<li class="chapter" data-level="3.5.2" data-path="materials-and-methods.html"><a href="materials-and-methods.html#gene-set-enrichment-analysis"><i class="fa fa-check"></i><b>3.5.2</b> Gene Set Enrichment Analysis<span></span></a></li>
</ul></li>
<li class="chapter" data-level="3.6" data-path="materials-and-methods.html"><a href="materials-and-methods.html#variance"><i class="fa fa-check"></i><b>3.6</b> Variance and diversity methods<span></span></a>
<ul>
<li class="chapter" data-level="3.6.1" data-path="materials-and-methods.html"><a href="materials-and-methods.html#permanova"><i class="fa fa-check"></i><b>3.6.1</b> PERMANOVA<span></span></a></li>
<li class="chapter" data-level="3.6.2" data-path="materials-and-methods.html"><a href="materials-and-methods.html#globaltest"><i class="fa fa-check"></i><b>3.6.2</b> globaltest<span></span></a></li>
<li class="chapter" data-level="3.6.3" data-path="materials-and-methods.html"><a href="materials-and-methods.html#diversity-indices"><i class="fa fa-check"></i><b>3.6.3</b> Diversity indices<span></span></a></li>
</ul></li>
<li class="chapter" data-level="3.7" data-path="materials-and-methods.html"><a href="materials-and-methods.html#other-methods"><i class="fa fa-check"></i><b>3.7</b> Other methods<span></span></a>
<ul>
<li class="chapter" data-level="3.7.1" data-path="materials-and-methods.html"><a href="materials-and-methods.html#wgcna"><i class="fa fa-check"></i><b>3.7.1</b> WGCNA<span></span></a></li>
<li class="chapter" data-level="3.7.2" data-path="materials-and-methods.html"><a href="materials-and-methods.html#baseset"><i class="fa fa-check"></i><b>3.7.2</b> BaseSet<span></span></a></li>
<li class="chapter" data-level="3.7.3" data-path="materials-and-methods.html"><a href="materials-and-methods.html#experdesign"><i class="fa fa-check"></i><b>3.7.3</b> experDesign<span></span></a></li>
<li class="chapter" data-level="3.7.4" data-path="materials-and-methods.html"><a href="materials-and-methods.html#ROC"><i class="fa fa-check"></i><b>3.7.4</b> ROC- AUC<span></span></a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="results.html"><a href="results.html"><i class="fa fa-check"></i><b>4</b> Results<span></span></a>
<ul>
<li class="chapter" data-level="4.1" data-path="results.html"><a href="results.html#packagesmethods"><i class="fa fa-check"></i><b>4.1</b> Packages/methods<span></span></a>
<ul>
<li class="chapter" data-level="4.1.1" data-path="results.html"><a href="results.html#experdesign-1"><i class="fa fa-check"></i><b>4.1.1</b> experDesign<span></span></a></li>
<li class="chapter" data-level="4.1.2" data-path="results.html"><a href="results.html#baseset-1"><i class="fa fa-check"></i><b>4.1.2</b> BaseSet<span></span></a></li>
<li class="chapter" data-level="4.1.3" data-path="results.html"><a href="results.html#intermodel"><i class="fa fa-check"></i><b>4.1.3</b> inteRmodel<span></span></a></li>
</ul></li>
<li class="chapter" data-level="4.2" data-path="results.html"><a href="results.html#analysis"><i class="fa fa-check"></i><b>4.2</b> Analysis<span></span></a>
<ul>
<li class="chapter" data-level="4.2.1" data-path="results.html"><a href="results.html#results-puget"><i class="fa fa-check"></i><b>4.2.1</b> Puget’s dataset<span></span></a></li>
<li class="chapter" data-level="4.2.2" data-path="results.html"><a href="results.html#results-hsct"><i class="fa fa-check"></i><b>4.2.2</b> HSCT dataset<span></span></a></li>
<li class="chapter" data-level="4.2.3" data-path="results.html"><a href="results.html#results-hasler"><i class="fa fa-check"></i><b>4.2.3</b> Häsler’s dataset<span></span></a></li>
<li class="chapter" data-level="4.2.4" data-path="results.html"><a href="results.html#results-morgan"><i class="fa fa-check"></i><b>4.2.4</b> Morgan’s dataset<span></span></a></li>
<li class="chapter" data-level="4.2.5" data-path="results.html"><a href="results.html#results-howell"><i class="fa fa-check"></i><b>4.2.5</b> Howell’s dataset<span></span></a></li>
<li class="chapter" data-level="4.2.6" data-path="results.html"><a href="results.html#between-datasets"><i class="fa fa-check"></i><b>4.2.6</b> Between datasets<span></span></a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="discussion.html"><a href="discussion.html"><i class="fa fa-check"></i><b>5</b> Discussion<span></span></a>
<ul>
<li class="chapter" data-level="5.1" data-path="discussion.html"><a href="discussion.html#preliminary-steps"><i class="fa fa-check"></i><b>5.1</b> Preliminary steps<span></span></a>
<ul>
<li class="chapter" data-level="5.1.1" data-path="discussion.html"><a href="discussion.html#discussion-datasets"><i class="fa fa-check"></i><b>5.1.1</b> The datasets<span></span></a></li>
<li class="chapter" data-level="5.1.2" data-path="discussion.html"><a href="discussion.html#the-methods"><i class="fa fa-check"></i><b>5.1.2</b> The methods<span></span></a></li>
</ul></li>
<li class="chapter" data-level="5.2" data-path="discussion.html"><a href="discussion.html#designing-models"><i class="fa fa-check"></i><b>5.2</b> Designing models<span></span></a></li>
<li class="chapter" data-level="5.3" data-path="discussion.html"><a href="discussion.html#evaluating-models"><i class="fa fa-check"></i><b>5.3</b> Evaluating models<span></span></a></li>
<li class="chapter" data-level="5.4" data-path="discussion.html"><a href="discussion.html#implications"><i class="fa fa-check"></i><b>5.4</b> Implications<span></span></a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="conclusions.html"><a href="conclusions.html"><i class="fa fa-check"></i><b>6</b> Conclusions<span></span></a>
<ul>
<li class="chapter" data-level="6.1" data-path="conclusions.html"><a href="conclusions.html#study-1-multi-omic-modelling-of-inflammatory-bowel-disease-with-regularized-canonical-correlation-analysis"><i class="fa fa-check"></i><b>6.1</b> Study 1: Multi-omic modelling of inflammatory bowel disease with regularized canonical correlation analysis<span></span></a></li>
<li class="chapter" data-level="6.2" data-path="conclusions.html"><a href="conclusions.html#study-2-genes-and-microbiome-relationship-on-inflammatory-bowel-disease"><i class="fa fa-check"></i><b>6.2</b> Study 2: Genes and microbiome relationship on inflammatory bowel disease<span></span></a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="acknowledgments.html"><a href="acknowledgments.html"><i class="fa fa-check"></i><b>7</b> Acknowledgments<span></span></a></li>
<li><a href="references.html#references">References<span></span></a></li>
<li class="appendix"><span><b>Appendix<span></span></b></span></li>
<li class="chapter" data-level="A" data-path="online-resources.html"><a href="online-resources.html"><i class="fa fa-check"></i><b>A</b> Online resources<span></span></a></li>
<li class="chapter" data-level="B" data-path="software.html"><a href="software.html"><i class="fa fa-check"></i><b>B</b> Software<span></span></a>
<ul>
<li class="chapter" data-level="B.1" data-path="software.html"><a href="software.html#star"><i class="fa fa-check"></i><b>B.1</b> STAR<span></span></a></li>
<li class="chapter" data-level="B.2" data-path="software.html"><a href="software.html#rsem"><i class="fa fa-check"></i><b>B.2</b> RSEM<span></span></a></li>
<li class="chapter" data-level="B.3" data-path="software.html"><a href="software.html#listed"><i class="fa fa-check"></i><b>B.3</b> Listed<span></span></a></li>
<li class="chapter" data-level="B.4" data-path="software.html"><a href="software.html#by-projectpublication"><i class="fa fa-check"></i><b>B.4</b> By project/publication<span></span></a></li>
</ul></li>
<li class="chapter" data-level="C" data-path="articles.html"><a href="articles.html"><i class="fa fa-check"></i><b>C</b> Articles<span></span></a>
<ul>
<li class="chapter" data-level="C.1" data-path="articles.html"><a href="articles.html#app-multiomics"><i class="fa fa-check"></i><b>C.1</b> Multi-omic modelling of inflammatory bowel disease with regularized canonical correlation analysis<span></span></a></li>
<li class="chapter" data-level="C.2" data-path="articles.html"><a href="articles.html#app-experDesign"><i class="fa fa-check"></i><b>C.2</b> experDesign: stratifying samples into batches with minimal bias<span></span></a></li>
</ul></li>
<li class="chapter" data-level="D" data-path="other-datasets.html"><a href="other-datasets.html"><i class="fa fa-check"></i><b>D</b> Other datasets<span></span></a>
<ul>
<li class="chapter" data-level="D.1" data-path="other-datasets.html"><a href="other-datasets.html#barcelona"><i class="fa fa-check"></i><b>D.1</b> BARCELONA dataset<span></span></a></li>
<li class="chapter" data-level="D.2" data-path="other-datasets.html"><a href="other-datasets.html#hernandez"><i class="fa fa-check"></i><b>D.2</b> Hernández’ dataset<span></span></a>
<ul>
<li class="chapter" data-level="D.2.1" data-path="other-datasets.html"><a href="other-datasets.html#results-hernandez"><i class="fa fa-check"></i><b>D.2.1</b> Results<span></span></a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="E" data-path="models-output.html"><a href="models-output.html"><i class="fa fa-check"></i><b>E</b> Models output<span></span></a>
<ul>
<li class="chapter" data-level="E.1" data-path="models-output.html"><a href="models-output.html#hsct"><i class="fa fa-check"></i><b>E.1</b> HSCT<span></span></a>
<ul>
<li class="chapter" data-level="E.1.1" data-path="models-output.html"><a href="models-output.html#genes"><i class="fa fa-check"></i><b>E.1.1</b> Genes<span></span></a></li>
<li class="chapter" data-level="E.1.2" data-path="models-output.html"><a href="models-output.html#microbiome-2"><i class="fa fa-check"></i><b>E.1.2</b> Microbiome<span></span></a></li>
</ul></li>
<li class="chapter" data-level="E.2" data-path="models-output.html"><a href="models-output.html#häsler"><i class="fa fa-check"></i><b>E.2</b> Häsler<span></span></a>
<ul>
<li class="chapter" data-level="E.2.1" data-path="models-output.html"><a href="models-output.html#genes-1"><i class="fa fa-check"></i><b>E.2.1</b> Genes<span></span></a></li>
<li class="chapter" data-level="E.2.2" data-path="models-output.html"><a href="models-output.html#microbiome-3"><i class="fa fa-check"></i><b>E.2.2</b> Microbiome<span></span></a></li>
</ul></li>
<li class="chapter" data-level="E.3" data-path="models-output.html"><a href="models-output.html#morgan"><i class="fa fa-check"></i><b>E.3</b> Morgan<span></span></a>
<ul>
<li class="chapter" data-level="E.3.1" data-path="models-output.html"><a href="models-output.html#genes-2"><i class="fa fa-check"></i><b>E.3.1</b> Genes<span></span></a></li>
<li class="chapter" data-level="E.3.2" data-path="models-output.html"><a href="models-output.html#microbiome-4"><i class="fa fa-check"></i><b>E.3.2</b> Microbiome<span></span></a></li>
</ul></li>
<li class="chapter" data-level="E.4" data-path="models-output.html"><a href="models-output.html#howell"><i class="fa fa-check"></i><b>E.4</b> Howell<span></span></a>
<ul>
<li class="chapter" data-level="E.4.1" data-path="models-output.html"><a href="models-output.html#genes-3"><i class="fa fa-check"></i><b>E.4.1</b> Genes<span></span></a></li>
<li class="chapter" data-level="E.4.2" data-path="models-output.html"><a href="models-output.html#microbiome-5"><i class="fa fa-check"></i><b>E.4.2</b> Microbiome<span></span></a></li>
</ul></li>
</ul></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Data integration on inflammatory bowel disease</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div class = "wip">
This tesis is ongoing and still without hard date to finish it. Watch the <a href="https://github.com/llrs/thesis">GitHub repository</a> or wait for a release announcement on Twitter on <a href="https://twitter.com/Lluis_Revilla">my profile</a>.
</div>
<div id="discussion" class="section level1 hasAnchor" number="5">
<h1><span class="header-section-number">Chapter 5</span> Discussion<a href="discussion.html#discussion" class="anchor-section" aria-label="Anchor link to header"></a></h1>
<p>In this chapter we will summarize the main findings in relation to the broad research community and other work as well as the impact of the results on further research in or clinical practice.</p>
<div id="preliminary-steps" class="section level2 hasAnchor" number="5.1">
<h2><span class="header-section-number">5.1</span> Preliminary steps<a href="discussion.html#preliminary-steps" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>The quality of the initial RNAseq and 16S data is important for a valid integration analysis.
In sequencing data it is very important to avoid contamination and have enough amount of data for the analysis.
To avoid contamination it is very important to control the protocol used to process the samples before the sequencing.
In addition control samples can be added, a blank to control for extraneous material and a sample with known content to confirm that the sequencing worked correctly.</p>
<p>Our lab uses a well established protocol to isolate and sequence RNA (Described on methods’ section <a href="materials-and-methods.html#processing">3.2</a>) and we rarely encounter problems during this phase.
In contrast, DNA extraction and 16S sequencing had to be established and optimized, initially with the guidelines and support from collaborators [mention?]
.
After encountering some problems we added blank samples to control for bacterial contamination during sample pre-processing or sequencing.</p>
<p>We initially processed the 16S data to obtain OTUs.
OTUs were the standard some years back <span class="citation">[<a href="#ref-callahan2017" role="doc-biblioref">199</a>]</span>.
However, OTUs are not comparable between studies even those that use the same primers.
For this reason, and after a suggestion from a reviewer (on the review process of <span class="citation">[<a href="#ref-revilla2021" role="doc-biblioref">198</a>]</span>), we moved to process later 16S sequencing datasets to obtain ASV.
ASV allow to compare the taxonomic imputation between studies using the same primers.
However, comparing 16S taxa ASV or OTUs is hard and selecting the right tool to compare them is important <span class="citation">[<a href="#ref-nearing2022" role="doc-biblioref">200</a>]</span>.</p>
<p>It is also worth keeping in mind that several species have variable number of 16S rRNA genes.
Higher sequences counts of certain 16S might not mean higher abundance of that species compared to another with lower 16S counts.
However, the exact copy number status may change even within the same bacterial species, making the correction difficult <span class="citation">[<a href="#ref-louca2018" role="doc-biblioref">201</a>]</span>.
It is however possible to accurately correct for copy number variation on mock populations where the species and the genomes are known (or at least the 16S rRNA), but harder or impossible on samples whose content is unknown.
To our knowledge there isn’t a method to do this yet.</p>
<!--# In addition, there are differences on the sequencing reads depending on the  regions selected to amplify the 16S. It would be better to fully sequence the 16S. -->
<p>Furthermore, the precision of 16S sequencing for classification of taxa is not enough to understand its role on the gut.
Adherent invasive cells, are cells with same genetic content that show different behavior <span class="citation">[<a href="#ref-nadalian2021" role="doc-biblioref">202</a>]</span>.
Those bacteria is known to be present on patients with IBD.
Thus, bacteria with the same complete sequence can play a different role in .</p>
<p>While most of these steps were out of my control as they were performed by a lab teammate or by collaborators.
Other concerns are currently unavoidable or could not have been found beforehand.
They had to consider them as they affect quality and interpretation of the data.</p>
<div id="discussion-datasets" class="section level3 hasAnchor" number="5.1.1">
<h3><span class="header-section-number">5.1.1</span> The datasets<a href="discussion.html#discussion-datasets" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>There is not an established methodology to calculate the size of cohorts for interaction or multi-omic studies.
Indeed, There are no clear rules on which alpha power or which kind of relations can be tested.
This might be due to the lack of mathematical background and modelization of the relevant relations in biology.
Further research on this area might help finding which kind of relationships can be expected given certain dataset sizes.
Usually size of the datasets is determined based on practical reasons: either costs or access to patients.</p>
<p>There have been some efforts to create artificial multi-omic datasets <span class="citation">[<a href="#ref-martínez-mira2018" role="doc-biblioref">140</a>–<a href="#ref-mccarthy2012" role="doc-biblioref">145</a>, <a href="#ref-chalise2016" role="doc-biblioref">203</a>, <a href="#ref-chung2019" role="doc-biblioref">204</a>]</span>.
This could allow to benchmark different tools, impute missing data and do pilot studies.
These efforts usually focus on RNA-seq, ATAC-seq (DNase-seq), ChIP-seq, small RNA-seq, Methyl-seq or proteomics but not 16S, microbiome data or metagenomics.
These efforts also focus on relationships between samples missing relationships within cells (regulated DNA, protein recruitment, transcription factors, siRNA regulation,…) and between cells (immune response, adherence, cytokines signaling…) that might surface at the sample level.
To my knowledge there is no accurate artificial method to create 16S datasets or related microbiome datasets.
This made impossible to simulate and compare different tools on a synthetic dataset with known relationships on this thesis to evaluate performance of different methods.</p>
<p>There is not a reference dataset for integration on IBD that accurately represents the disease.
There is a large consortium effort with many samples<span class="citation">[<a href="#ref-humanmicrobiomeprojectconsortium2012" role="doc-biblioref">4</a>]</span>.
It does not have 16S and RNAseq data from the same location at the same time on as much samples as the datasets used on this thesis.</p>
<p>In order to compare different methods we are limited to comparing different datasets.
However, each dataset is collected with different goals and processed differently.
In addition, authors upload their data to different, not centralized, data repositories, such as, gene expression omnibus (<a href="https://www.ncbi.nlm.nih.gov/geo/">GEO</a>), European Genome-Phenome Archive (<a href="https://ega-archive.org/">EGA</a>), European Nucleotide Archive (<a href="https://www.ebi.ac.uk/ena/browser/home">ENA</a>), among others.
Authors might also provide the processed data as supplementary material on their articles.
Some projects whose primary purpose is providing data for the community establishing their own dedicated sites to store the data <span class="citation">[<a href="#ref-humanmicrobiomeprojectconsortium2012" role="doc-biblioref">4</a>]</span>.</p>
<p>There is not a resource were datasets of publications are collected with their characteristics.
Finding different datasets for very similarly data is thus challenging.
Furthermore when pooling together datasets batch effect correction will most likely be necessary.
There are tools to overcome this, but currently only work if they share some features or samples in common <span class="citation">[<a href="#ref-ugidos2020" role="doc-biblioref">205</a>]</span>.
These methods usually require datasets with similar RNA sequencing procedures.</p>
<p>On this thesis there were two in house datasets collected and sequenced: the HSCT and the BARCELONA dataset.
For the purpose of this thesis, we looked up for the most similar datasets published in order to be able to confirm our results outside our own cohorts.
We found several with 16S and RNAseq data from the intestine from the same location, to compare with our own datasets.</p>
<p>As explained in methods’ section <a href="materials-and-methods.html#methods-hsct">3.1.2</a>, the HSCT dataset is a very unique cohort of CD patients undergoing hematopoietic stem cell transplant.
This treatment is reserved to patients for which all the other treatment failed and this may be the only way to reach remission.
These patients are closely monitored and followed up for several years.
This treatment is not indicated for UC, so this dataset includes only CD patients.</p>
<p>The BARCELONA dataset (See appendix section <a href="other-datasets.html#barcelona">D.1</a>) includes samples from CD and UC collected propsectively in patients starting biological treatment and followed up for up to a year.
Patients were younger and had less years with the disease.
Patients in HSCTthe group were refactry to treatment and may even be naive to any biologic , which could provide with more insights on the initial relationships between the microbiome and the mucosa of IBD patients.</p>
<p>Unfortunately, the 16S microbiol sequence data of BARCELONA was not good enough to make reliable analysis and confidently extract hypothesis or relationships (See appendix section <a href="other-datasets.html#barcelona">D.1</a>).
It is not clear what happened and as a consequence it is not clear how to avoid such problems on the future.
It could be a problem of the original DNA extraction and/or of its sequencing process which did not include both positive and negative controls.
Nevertheless, sample sequencing was repeated in an independent sequencing facility to overcome the of the first dataset but problems persisted rendering this data suboptimal for further analysis.</p>
<p>What happened with the BARCELONA dataset highlights the importance of data quality checks.
In the BARCELONA cohort despite no batch effect or other quality issues with the data as checked with <code>experDesign</code>, the diversity indices suggested that there was some problem with the microbiome data.
This made us not trusting the data and not using it for data integration.
Caution and a good relationship with the sample and data generator is important to discover these kind of problems and ensure the quality of the data.</p>
<p>Other datasets used from published sourced were assumed they had already passed enough controls and had a good quality.
Nevertheless, they were screened to avoid quality issues.</p>
<p>The Puget’s dataset dataset provided a good benchmark to test the methods and understand how RGCCA works.
CNV/CGH is a completely different type of data than microbiota, but the microarray data is similar to the RNAseq data from other datasets.
We were not as much interested on the biology as on learning about the RGCCA method and possibilities of integration with it.</p>
<p>Häsler data were obtained using the same sequencing techniques from endoscopic biopsies as our dataset HSCT and Barcelona and it is of the same disease.
The 16S data was very similar to the HSCT dataset.
We could confirm that the inteRmodel approach works on more than one dataset.</p>
<p>The taxonomy of the different datasets was done differently.
After the first two datasets SILVA database was used to annotate the microbiome data on their corresponding taxa.
On HSCT following our collaborators advise we used other methods to annotate its microbiota data.</p>
<p>Morgan’s dataset is related to IBD but it is from a cohort of patients that underwent colectomy and samples are specific of the pouch or pre-pouch ileum, there are no healthy samples and there is no follow up.
It made very unlikely that a classification of the location could be achieved and the classification of the samples according to the microbiota could not be based on the disease (as all of them had undergone the same procedure with the same disease).
There seems to be gradation on the models on <a href="results.html#fig:morgan-models">4.25</a> that could partially be explained by different degrees of inflammation.</p>
<p>Howell’s dataset includes both CD and UC (and non-IBD) samples.
It is very similar to our BARCELONA dataset but on pediatric cases.
As expected time with the disease of these patients is much lower, and they have might followed less treatments and probably have not had any surgery.
However, they were not followed up at different time points.</p>
</div>
<div id="the-methods" class="section level3 hasAnchor" number="5.1.2">
<h3><span class="header-section-number">5.1.2</span> The methods<a href="discussion.html#the-methods" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>As seen on the introduction there are many different methods are available.
There are frequent releases of new tools and methods.
The most up to date list of tools might be added to a <a href="https://github.com/mikelove/awesome-multi-omics" title="Awesome multi-omics list">collaborative list</a>.
Methods are very different in how they can be used and how they can be classified.
The quality of the software and methods also differs.
Some of them were tested on several datasets but their most important validation process undergone is the mathematical validation they have and its correct implementation.</p>
<p>With the increase of tools there have been increasingly important to compare different tools.
There are reviews of different tools on the same datasets <span class="citation">[<a href="#ref-cantini2021" role="doc-biblioref">129</a>]</span>, some are more theoretical <span class="citation">[<a href="#ref-bersanelli2016" role="doc-biblioref">206</a>]</span>, others are focused on a different field <span class="citation">[<a href="#ref-cavill2016" role="doc-biblioref">99</a>]</span>.</p>
<p>Of all these methods very few have been applied on IBD datasets.
Recently there has been an article focused on integration on IBD <span class="citation">[<a href="#ref-sudhakar2022" role="doc-biblioref">148</a>]</span>.
On this publication the authors suggest that one must be mindful of the gap between the experimental conditions and the real world.
It also encourages to collect more data about the exposome (the stimuli patients have).
It ends up advising to set up guidelines for multi-omic studies tailored to the field, coordinate in a global framework to prevent redundant studies and ensure efficient funding and resources and disseminate training and education on computational approaches to analyze multi-omic datasets.</p>
<p>Current methodological approaches focus on comparing tools on (several) previously published datasets <span class="citation">[<a href="#ref-cantini2021" role="doc-biblioref">129</a>]</span>, an approach that we have taken too.
This approach was not used to compare different tools, we also did it, but to validate findings of one dataset in other datasets.
This is specially important due to the lack of golden datasets or a way to reliable simulate datasets as discussed on the previous section <a href="discussion.html#discussion-datasets">5.1.1</a>.</p>
<p>On IBD, many studies focus on finding some genes or bacteria to answer a narrow question they have in mind, like which bacteria is related to inflammation <span class="citation">[<a href="#ref-tang2017" role="doc-biblioref">80</a>]</span>.
On this thesis, the focus was on finding a good representation of the relationships that allowed to identify genes and bacteria are relevant on the disease.
We made the assumptions that microbiome presence and transcriptome are related.
This assumption is backed up by several other previous studies supporting this relationship <span class="citation">[<a href="#ref-thomas1998" role="doc-biblioref">40</a>, <a href="#ref-cornish2008" role="doc-biblioref">41</a>, <a href="#ref-holmes2012" role="doc-biblioref">207</a>–<a href="#ref-brand2021" role="doc-biblioref">209</a>]</span>.</p>
<p>Tools that relate the variance of a block with other variables, both numeric and categoric, are needed to search which variables are important.
PERMANOVA and <code>globaltest</code> identified which variables were important.
But they do not give any insight into which specific microbial species are driving the association between the microbiome and the variables <span class="citation">[<a href="#ref-susin2020" role="doc-biblioref">210</a>]</span>.
In addition, we could miss some other important variables.
It is known the role of other factors outside the omics data collected, mainly from environment factors, genetic susceptibility and immune response <span class="citation">[<a href="#ref-sartor2006" role="doc-biblioref">211</a>]</span>.
Sudhakar <em>et. al.</em> <span class="citation">[<a href="#ref-sudhakar2022" role="doc-biblioref">148</a>]</span> recommend being conscious of the gap between the data available and the biological process.
However, even if we missed them, we probably have not collected it or it is not available, there is nothing at this point that can be done.
This would be in conflict with a data protection principle: collect the minimum sufficient information needed.
We might not identify a relevant variable but to identify it we would need to collect too much information from the patients and for little or no clear benefits.
One variable we did not keep track was the microbiome load which is linked to the gut’s microbiome community variation <span class="citation">[<a href="#ref-vandeputte2017" role="doc-biblioref">212</a>]</span>.
We did not measure it because at the time of extraction we didn’t quantify the cells present and after the DNA extraction it was too late.
To our knowledge this is not usually done and several other studies lack this data.</p>
<p>We tried to find which genes and bacteria are correlated between them using <code>WGCNA</code>.
It is a tool designed to find common co-occurring patterns based on correlations.
It requires homogeneous samples, with a minimum of 12 samples per condition.
When applied to the whole dataset there might bee too much variance in order of WGCNA to find the proper signal.
As we have patients of different ages and location which is highly related to the intestinal microbiota this might be the reason why it failed to achieve a good fit on the scale free topology with around 100 of mean connectivity.
In addition, having microbiome and RNASeq on the same matrix, would probably be hard for the process to find good relationships if we applied the same normalization process to both of them without escalation.
We could have tried to make smaller groups and then compare the modules between them but it groups would be too small because samples were from multiple segments and conditions.</p>
<p>We briefly considered using <code>STATegRa</code> but it is not possible to select which interactions between the blocks exists and by that time we already knew it was important to consider several of the variables collected from the patients.
The method implemented on <code>STATegRA</code> might be useful for cases where there is a great agreement between blocks or were the variables of the samples are not so important as with our data.</p>
<p>To identify related variables other methods use correlations between the variables <span class="citation">[<a href="#ref-vila-casadesús2016" role="doc-biblioref">213</a>]</span>.
On our dataset we visually explored the correlations for all the datasets, but the significant correlations were usually driven by an outlier, or was not a good fit of the data due to missing microbiome presence (data not shown).
We tried filtering correlations from those identified by the models, being less restrictive on the significance of the correlation, removing those samples that did not have microbiome presence on at least 5 samples, and remove those samples without that microbiome presence and splitting by location and a combination of all these.
All of these variations didn’t provide a clear insight over which genes were correlated to which microbiomes, this differs to other publications that relied on them to draw their conclusions <span class="citation">[<a href="#ref-häsler2017" role="doc-biblioref">214</a>]</span>.
This might be due to them relying purely on the Spearman <span class="math inline">\(\rho\)</span> metric instead of visualizing the correlation it measures or having more common microbiota between the samples.</p>
<p>We developed <code>BaseSet</code> to find these relationships using a different approach.
<code>BaseSet</code> did not work for its intended usage on this thesis.
It failed because it is computationally expensive to calculate the likelihood of 1500 variables; there are too many combinations.
In addition, the numeric precision of said calculations suffers from the floating point problem and must be carefully considered <span class="citation">[<a href="#ref-goldberg1991" role="doc-biblioref">215</a>]</span>.
To support multiplying more than 1000 float numbers a different strategy such as using log values might be better.
We could not come up with a better strategy to find all the combinations needed, perhaps a better method exists that could be used to find which are the terms more influential to the end result.
During the peer-review process of the package for its acceptance on rOpenSci, some concerns were raised about conflating probabilities with fuzzy-sets.
For all these reasons this approach was no longer pursued.
However, the package was mentioned as <a href="https://rviews.rstudio.com/2020/12/22/november-top-40-new-cran-packages/">top 40 packages</a> added on CRAN that month and it will be useful on other circumstances or when less combinations are possible.</p>
<p>The development of the <code>experDesign</code> package helped avoid batch effects on the sequencing step.
However, as seen on the previous section, this does not prevent all problems with batch effects and still batch effects resulting on a less useful experiment might get through.
As discussed on the related publication, there are several tools already focused on this problem: <code>OSAT</code> (<span class="citation">[<a href="#ref-yan2012" role="doc-biblioref">216</a>]</span>), <code>anticlust</code> (<span class="citation">[<a href="#ref-papenberg2020" role="doc-biblioref">217</a>]</span>) and <code>Omixer</code> (<span class="citation">[<a href="#ref-sinke2021" role="doc-biblioref">218</a>]</span>).
But these tools have some shortcomings, that are covered by <code>experDesign</code>: <code>OSAT</code> cannot handle missing values and does not work well for arbitrary batches, <code>anticlust</code> only accepts numerical variables but it is based on a powerful mathematical theorem and <code>Omixer</code> has bugs that prevent comparing with the other tools with no possible workaround.<br />
In addition, <code>experDesign</code> received <a href="https://github.com/llrs/experDesign/issues/22#issuecomment-1033508578">requests</a> to have a new feature for expanding experiments.
This might help improve the quality of bigger datasets to ensure they can be extended in several sequencing runs.
This feature would be useful for multi-omics datasets or in big cohorts to minimize batch effects associated with long running collection of samples.</p>
<p>We selected <code>MCIA</code> as a baseline to compare our method because it works well on a wide range of datasets, has a good documentation as well and it is fairly widespread used.
The method was developed after <code>RGCCA</code> and recently there have been publications that show that it outperforms other methods on its versatility on different contexts <span class="citation">[<a href="#ref-cantini2021" role="doc-biblioref">129</a>]</span>.
On the dataset analyzed we found it was very competitive with <code>inteRmodel</code> but this will be discussed on section <a href="discussion.html#evaluating-models">5.3</a>.</p>
</div>
</div>
<div id="designing-models" class="section level2 hasAnchor" number="5.2">
<h2><span class="header-section-number">5.2</span> Designing models<a href="discussion.html#designing-models" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>Previous publications using <code>RGCCA</code> on IBD focused on validating inflammation markers genes <em>DUOX2</em> and <em>APOA1</em> as inflammation predictors (<span class="citation">[<a href="#ref-tang2017" role="doc-biblioref">80</a>]</span>) from previously published articles <span class="citation">[<a href="#ref-haberman2014" role="doc-biblioref">219</a>]</span>.
Some publication tried to summarize the existing relationships on IBD <span class="citation">[<a href="#ref-hasler_uncoupling_2016" role="doc-biblioref">79</a>]</span>, but none were focused on finding the existing relationships on IBD using <code>RGCCA</code> as we did.</p>
<p>There are many variables outside transcriptomics and microbiome that are relevant to the disease (and healthy) homeostasis.
These variables should be included on the models to find which genes and bacteria are really related and not confounded by other factors.
In addition, the relationships between the blocks are unknown on both the strength and interaction.
To model the relationships the connections between blocks had to be set on <code>RGCCA</code>.
Last, what belongs to a block should be carefully considered as the assumption is that the whole block is correlated with other blocks connected.</p>
<p>If one has preexisting theories about the data, a specific model can be used stating these known or hypothetical relationships.
However, if new relationships are being explored or no prior beliefs on the data are held the models should be created with random links between blocks, and evaluate which model is better.</p>
<p>The connections tested required that all models should have all blocks indirectly connected to other blocks; no isolated blocks left.
This avoids optimizing two different networks of blocks that are not connected between them.
Thus, forcing the model to represent all the information.</p>
<p>Typically blocks are defined by each omic data origin and no other information is included.
However, we knew that the transcriptome is mainly related to the location of the samples and we expected that the microbiome would be more related to the variables related to the patients demographic characteristics and dietary and personal habits.
This is specially important because in our datasets we have samples from the same patient and time from multiple locations.
On other studies there are less samples per individual and timepoint (if there are several timepoints) <span class="citation">[<a href="#ref-hasler_uncoupling_2016" role="doc-biblioref">79</a>, <a href="#ref-morgan2015" role="doc-biblioref">150</a>, <a href="#ref-howell2018" role="doc-biblioref">151</a>]</span>.</p>
<p>On models of family 2, variables of samples were separated according to what they are.
If it is about the individual it would be on the Demographic block, if it is about time on a Time block, and if about the location of the sample on a Location block (c.f. with tables on design of Puget’s <a href="results.html#tab:puget-model2-2">4.4</a>, <a href="results.html#tab:puget-model2">4.3</a>; HSCT <a href="results.html#tab:hsct-model2">4.13</a> <a href="results.html#tab:hsct-model2-2">4.14</a>, <a href="results.html#tab:hsct-model2-3">4.15</a>; Häsler: <a href="results.html#tab:hasler-model-2-1">4.21</a>, <a href="results.html#tab:hasler-model-2-2">4.22</a>, Morgan’s <a href="results.html#tab:morgan-model2">4.28</a>, <a href="results.html#tab:morgan-model2-2">4.29</a>, Howell’s <a href="results.html#tab:howell-model2-2">4.33</a> models).</p>
<p>The exploration of design on Puget’s dataset, and the datasets analysis of HSCT, Häsler and Morgan’s dataset was published after peer review <span class="citation">[<a href="#ref-revilla2021" role="doc-biblioref">198</a>]</span>.</p>
</div>
<div id="evaluating-models" class="section level2 hasAnchor" number="5.3">
<h2><span class="header-section-number">5.3</span> Evaluating models<a href="discussion.html#evaluating-models" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>To evaluate a model <code>RGCCA</code> provides the average variance explained (AVE), inner and outer (see section <a href="materials-and-methods.html#rgcca-output">3.3.1.2</a>).
As we are more interested on the model of the relationships the inner AVE makes more sense to evaluate it.</p>
<!--# TODO Add tables of mean and sd of AVEs on the bootstrapping ? -->
<p>Furthermore, bootstrapping was used to evaluate the fitness of a design on a diverse collection of datasets.
Although on <code>inteRmodel</code> there is the option to use a leave-one-out procedure, we didn’t use it to evaluate the fitness of the models.</p>
<p>Using an external cohort to validate the same model, or using a different method to see if it finds the same relationships or explains the data as accurately is also a common approach to evaluate and validate models.
Using the same approach on different data helps to ensure the replicability of the results <span class="citation">[<a href="#ref-community2021" role="doc-biblioref">220</a>]</span>.</p>
<div class="figure"><span style="display:block;" id="fig:reproducible-matrix"></span>
<img src="images/turing_way_reproducible-matrix.jpg" alt="Reproducibility matrix indicating the terminology used between using the same method and the same data. Figure from The Turing Way: A Handbook for Reproducible Data Science (Version v1.0.1)." width="100%" />
<p class="caption">
Figure 5.1: Reproducibility matrix indicating the terminology used between using the same method and the same data. Figure from The Turing Way: A Handbook for Reproducible Data Science (Version v1.0.1).
</p>
</div>
<p>We used the same approach on 4 different cohorts, from different origins and with different types of samples, but all are related to IBD, have 16S data and intestinal transcriptome.
Some of them have multiple samples from the same individuals while others do not <span class="citation">[<a href="#ref-hasler_uncoupling_2016" role="doc-biblioref">79</a>]</span>.</p>
<p>We also compared our method with a different one to see how generalisable are the results.
Of the multiple methods available we used <a href="https://bioconductor.org/packages/omicade4" title="Omicade4 package"><code>MCIA</code></a> <span class="citation">[<a href="#ref-mengMultivariateApproachIntegration2014" role="doc-biblioref">221</a>]</span>.
We compared it with <code>inteRmodel</code> by looking at the area under the curve for classifying the samples with the canonical component of the transcriptome data according to their location.</p>
<p>The procedure of separating independent variables in their own block of data and later search the best model that fits the data provides a good strategy that should be consider for integration efforts.
The procedural method of searching a model and testing them is implemented on <a href="https://llrs.github.io/inteRmodel/" title="inteRmodel"><code>inteRmodel</code></a>.
But the most important thing is to consider which variables are independent of which and if they can be separated into a block for later usage on the modeling (See <span class="citation">[<a href="#ref-pearl2011" role="doc-biblioref">222</a>]</span>).
If not done properly it can lead to undesired conclusions like the effect known as <a href="https://en.wikipedia.org/wiki/Simpson%27s_paradox">Simpson’s paradox</a> <span class="citation">[<a href="#ref-simpson1951" role="doc-biblioref">223</a>]</span>.
Unfortunately this cannot be automated and it is left for the practitioner.</p>
<p>Splitting the variables of each sample into several blocks forces <code>RGCCA</code> to adjust for a new canonical dimension.
The omics block such as gene expression and 16S data could be split, as the expression of some genes influence other genes, such as <a href="https://en.wikipedia.org/wiki/Transcription_factor">transcriptor factors</a> (First mentioned on <span class="citation">[<a href="#ref-stillman1984" role="doc-biblioref">224</a>]</span>) , <a href="https://en.wikipedia.org/wiki/MicroRNA">miRNA</a> <span class="citation">[<a href="#ref-lee1993" role="doc-biblioref">225</a>]</span>, <a href="https://en.wikipedia.org/wiki/Small_interfering_RNA">siRNA</a> <span class="citation">[<a href="#ref-hamilton1999" role="doc-biblioref">226</a>]</span>… All these interactions and regulations could distord the canonical correlations.
However, the nature of this relationships is not linear and the interaction between them is multiple and very complex.
Its complexity has prevented to accurately account for all subcellular reactions at speeds that could be useful (it has only been recently accomplished for a prokaryote cell <span class="citation">[<a href="#ref-thornburg2022" role="doc-biblioref">227</a>]</span>).
In addition, these interactions are time dependent, not linear and are highly interconected between a large number of variables.
For these reasons gene expression and 16S block were not split into several blocks.</p>
<p>Besides comparing the results of different methods, these models need to be evaluated by the insights they provide on the biological system they are being applied to, in our case IBD.
So far the models were only discussed on their technical merits.</p>
<p>It is known that mucosal transciptome is related to localization of the samples <span class="citation">[<a href="#ref-criss2021" role="doc-biblioref">228</a>]</span>.
This can be seen on the PCAs which separate colon an ileum on the first dimension: See the PCAs of the different datasets <a href="results.html#fig:hsct-pca-rnaseq">4.9</a>, <a href="results.html#fig:hasler-pca-rnaseq">4.18</a> and <a href="results.html#fig:howell-pca-rnaseq">4.30</a>.
The difference is so great that many times the colon and ileum samples are analyzed separately.
As such, it was a reasonable assumption to expect models to reflect this differences on the gene expression canonical component.</p>
<p>The variability of microbiome challenges finding clear patterns (See the PCAs of the different datasets: <a href="results.html#fig:hsct-pca-16s">4.8</a>, <a href="results.html#fig:hasler-pca-16s">4.17</a>, <a href="results.html#fig:morgan-pca-16s">4.23</a> and <a href="results.html#fig:howell-pca-16s">4.29</a>).
On healthy humans, it has been suggested that there is a common microbial composition on all humans <span class="citation">[<a href="#ref-arumugam2011" role="doc-biblioref">229</a>]</span>. Such theory suggests three groups of similar microbiota composition, named enterotypes. However, the enterotypes classification is not unanimously accepted <span class="citation">[<a href="#ref-koren2013" role="doc-biblioref">230</a>, <a href="#ref-cheng2019" role="doc-biblioref">231</a>]</span>.
Some suggest it is an artifact of the metodolody used.</p>
<p>There are many factors that influence the gut microbiome <span class="citation">[<a href="#ref-hasan2019" role="doc-biblioref">232</a>]</span>.
But the role of microbiome on IBD has recently gained some interest <span class="citation">[<a href="#ref-khanna2014" role="doc-biblioref">16</a>, <a href="#ref-bringiotti2014" role="doc-biblioref">233</a>, <a href="#ref-lloyd-price2019" role="doc-biblioref">234</a>]</span>.
It has been associated with predictors of CD treatment response and suggested that they could be useful for building an improved classifier for CD and treatment response based on sufferers’ microbiome in the future <span class="citation">[<a href="#ref-douglas2018" role="doc-biblioref">235</a>]</span>.
Thee is for instance, an association between IBD and butyrate producers (See <span class="citation">[<a href="#ref-takahashi2016" role="doc-biblioref">236</a>]</span>) which has shown to influence permeability of the epithelial barrier <span class="citation">[<a href="#ref-ferrer-picón2020" role="doc-biblioref">24</a>]</span>.</p>
<p>In addition, it is known that microbiome of patients in remission is different from non-IBD patients <span class="citation">[<a href="#ref-halfvarson2017" role="doc-biblioref">237</a>]</span>.
So even if two people seem healthy at this moment their microbiome might not be similar.</p>
<p>The relapse nature of IBD, suggests that at different time points the gene expression or the microbiome might be different.
For this reason, time is an important variable when modelling the disease.
If multiple timepoints are taken they should be taken into account to identify the state of the disease for each patient.
Even on healthy non-IBD patients, multiple samples with microbiota from the same patients could provide a common microbiome background, which can help to identify core individual microbiome and altered microbiome states later on.</p>
<p>On the cohorts analyzed on this thesis, only the HSCT and the Häsler dataset had some time related variables available.
However, Häsler dataset only had age at date of sampling, and there were not several samples for the same patients at different timepoints.
This leaves the HSCT dataset with the only one with multiple timepoints from the same patients.
Having that many samples from the same patient might explain why the classification of the disease on this cohort works so well, with relatively a very complicated cohort with several background treatments, refractory to all previous treatments , some of them having undergone surgery too and with wide systemic changes.
Despite all of this there was a huge agreement in genes identified as relevant between datasets and the pathways identified are relevant to the intestine (See section <a href="results.html#fig:pathways-hsct-howell">4.35</a>).</p>
<p>Comparing microbiome between datasets is less straight forward.
If the same primers are used the ASV could be directly compared.
However, the ASV’ length might be different.
In addition, on this thesis we used dataset with different primers and also OTUs.
For these reasons a direct comparison is not possible.
A comparison of the taxonomy of the annotated microbiome on the datasets is the next possibility.</p>
<p>However, some datasets used did not provide the annotation (Häsler’s and Morgan’s dataset).
We have two dataset with annotation of the microbiome on the taxonomy, our own HSCT dataset and Howell’s dataset.
So we are are left with comparing OTUs with ASVs.
Despite the errors on ASV annotation ( <span class="citation">[<a href="#ref-edgar2018" role="doc-biblioref">238</a>]</span>), OTUs from the HSCT dataset can be compared to ASV from Howell’s dataset.
There were very few common taxonomic levels selected on both datasets (data not shown).
This was not a surprise, as there are many factors that influence one’s microbiome profile and the diversity indexes already showed high differences between samples of the same dataset.</p>
<p>The microbiome of each dataset seem capable to classify the samples according to the disease (data not shown).
However, on further evaluation via bootstrapping this resulted not significant, as any relatively big amount of microbiome variables might classify the samples due to their specificity.</p>
<p>On Howell’s and Häsler dataset, the best model is 1.2; which separates the microbiome component by location too (See Figure <a href="results.html#fig:hasler-models">4.20</a> and table <a href="results.html#tab:hasler-aves">4.23</a> for Häsler dataset and figures <a href="results.html#fig:howell-models">4.31</a> and table <a href="results.html#tab:howell-models-ave">4.34</a>).
Those models 2.2 according to the inner AVE are not that far away from models 1.2.
Which seems to imply that the relationship of the microbiome is slightly stronger with the location than the relationship with the disease.
Both factors should be considered when looking for relationships of genes with microbiome.</p>
</div>
<div id="implications" class="section level2 hasAnchor" number="5.4">
<h2><span class="header-section-number">5.4</span> Implications<a href="discussion.html#implications" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>On this thesis several methods has been developed for helping research.
<code>experDesign</code> is on the primary steps from moving from the bench to the in-silico experiments.
<code>BaseSets</code> and <code>inteRmodel</code> are more relevant for computational analysis.
<code>BaseSets</code> might help beyond integration analysis such as single cell annotation<a href="#fn7" class="footnote-ref" id="fnref7"><sup>7</sup></a>.</p>
<p>Some studies using host transcriptomics of fecal wash infer inflammation without colonoscopy <span class="citation">[<a href="#ref-ungar2022" role="doc-biblioref">239</a>]</span>.
This would help patients to avoid an unpleasant experience, and reduce the usage of clinical facilities.
It could be possible that just sequencing the intestinal microbiome could be enough to identify the patients’ disease.
However, this requires further validation to ensure that the diagnosis is accurate enough on a diverse and big population.
There are already studies on this direction, not only for IBD or intestine but for several different human regions <span class="citation">[<a href="#ref-humanmicrobiomeprojectconsortium2012" role="doc-biblioref">4</a>]</span>.</p>
<p>With these studies we hoped that they would provid which bacteria played a role on the disease, or which genes and bacteria are related on IBD.
We obtained a list of putative genes and bacteria but not a clear pairing of which genes interact with which bacteria.
This could mean that the microbiome community is related to all the genes.
It could also mean that the methods are not powerful enough to find more tailored relationships as normalizations and generalizations present on <code>RGCCA</code> do not allow to subgroup or classify the variables already detected.
Maybe a different method that would not depend on the same principles might be able to detect finer relationships on the variables selected.</p>
<p>There is a disconnect between the computational side and the experimental side, driven by the difficulty to think an experiment to test the new information that multi-omic experiments provide.
This is referred as the gap in other publications <span class="citation">[<a href="#ref-sudhakar2022" role="doc-biblioref">148</a>]</span>.
This gap between the computational methods and the data origin and practical usages should close.
Potentially this would require more closer collaboration between clinicians, statisticians, bioinformaticians and research software engineers; in addition to creative ideas accounting for the standard procedure on hospitals and points of care.</p>
<p>However, it is also possible that further developments or creatively applying statistical methods might help closing the gap.
For instance, there are many combinations of interaction possible.
These interactions between microbiome and genes are currently hard to explore statistically.
Further research on how to reduce the space of possible combinations of bacteria or evaluate which combinations are more important might be useful on the future.
This might involve using more network integration methods.</p>
<p>Krassowski’s <em>et. al.</em> (<span class="citation">[<a href="#ref-krassowski2020" role="doc-biblioref">94</a>]</span>) advice and recommendations on software engineering and reproducibility practices to share awareness with new researchers in multi-omics for end-to-end workflow.
In addition the recent recommendation about integration on IBD (<span class="citation">[<a href="#ref-sudhakar2022" role="doc-biblioref">148</a>]</span>) suggests that there is still much to be done to increase the results on these projects and the effectivity.
We agree with them and encourage other researchers to carefully consider the proposed methodology before starting their own projects.</p>
<p>There seems to be a tendency of multi-omics project to focus on metagenomics, metatranscriptomics and metaproteomics and abandon plain 16S sequencing <span class="citation">[<a href="#ref-zhang2019" role="doc-biblioref">240</a>]</span>.
This could be explained by the numerous problems that 16S sequencing have.
Some of these are avoided or solved by using these other omics techniques.
For instance the metatranscriptomics and metaproteomics could help detect what is actively producing the microbiota and the metagenomics, what is actually there at a more detailed level than 16S.
But these methods also have their shortcomings.
Metagenomics still does not detect adherent invasive cells and metatranscriptomics and metaproteomics do not provide who is producing what (even if paired with metagenomics or 16S data).
Devonvoluting which bacteria produces something is also not clear.
The single cell revolution might provide more insights in the future, but at the moment the first studies of single-cell multi-omics are being published.</p>
<p>Perhaps as suggested in other publications network methods might be able to provide more detailed information about the relationships <span class="citation">[<a href="#ref-jiang2019" role="doc-biblioref">241</a>]</span>.
But, it is unclear how complete and valid are these networks.
There is some evidence that currently literature focus on already known and studied genes instead of more novel and with higher relevance genes <span class="citation">[<a href="#ref-haynes2018" role="doc-biblioref">242</a>]</span>.
By extension, this translates in bias in networks and pathways resources available for pathway analysis and integration methods.
This could explain why there are many genes selected by the models with few pathways or gene sets.</p>
<p>To study complex systems, new technologies like “organ-on-chip” or more specifically “gut-on-chip” are being developed <span class="citation">[<a href="#ref-collij2021" role="doc-biblioref">243</a>]</span>.
These systems expand on the already useful technique of using organoids to better mimic the epithelial cross-talk on the laboratory.
Latest developments include the addition of separate environments and distinct flow on these environments.
By introducing gut microbiota, these systems will help studying the interaction of the gut microbiota and the intestinal epithelium.
However, it is not clear how accurately accounts for other factors such as the accumulation of inflammation.</p>
<p>As shown with the PERMANOVA approach, several factors affect the disease and the relationship between gut’s microbiome and gut’s mucosa.
Analysis or comparisons without taking into account them might provide misleading or false results.
To our knowledge this is the first study using these variables as part of the integration study with canonical correlations.
We hope this provides a first approximation to accurately understand the relationships between genes and bacteria on IBD.</p>

</div>
</div>
<h3>References</h3>
<div id="refs" class="references csl-bib-body">
<div id="ref-humanmicrobiomeprojectconsortium2012" class="csl-entry">
4. Human Microbiome Project Consortium BA, Nelson KE, Pop M, Creasy HH, Giglio MG, Huttenhower C, et al. <a href="https://doi.org/10.1038/nature11209">A framework for human microbiome research.</a> Nature. 2012;486:21521.
</div>
<div id="ref-khanna2014" class="csl-entry">
16. Khanna S, Tosh PK. <a href="https://doi.org/10.1016/j.mayocp.2013.10.011">A Clinician’s Primer on the Role of the Microbiome in Human Health and Disease</a>. Mayo Clinic Proceedings. 2014;89:107–14.
</div>
<div id="ref-ferrer-picón2020" class="csl-entry">
24. Ferrer-Picón E, Dotti I, Corraliza AM, Mayorgas A, Esteller M, Perales JC, et al. <a href="https://doi.org/10.1093/ibd/izz119">Intestinal Inflammation Modulates the Epithelial Response to Butyrate in Patients With Inflammatory Bowel Disease</a>. Inflammatory Bowel Diseases. 2020;26:43–55.
</div>
<div id="ref-thomas1998" class="csl-entry">
40. Thomas GAO, Rhodes J, Green JT. <a href="https://doi.org/10.1111/j.1572-0241.1998.00144.x">Inflammatory bowel disease and smoking<span></span>a review</a>. Official journal of the American College of Gastroenterology | ACG. 1998;93:144149.
</div>
<div id="ref-cornish2008" class="csl-entry">
41. Cornish JA, Tan E, Simillis C, Clark SK, Teare J, Tekkis PP. <a href="https://doi.org/10.1111/j.1572-0241.2008.02064.x">The risk of oral contraceptives in the etiology of inflammatory bowel disease: a meta-analysis</a>. The American Journal of Gastroenterology. 2008;103:2394–400.
</div>
<div id="ref-hasler_uncoupling_2016" class="csl-entry">
79. Häsler R, Sheibani-Tezerji R, Sinha A, Barann M, Rehman A, Esser D, et al. <a href="https://doi.org/10.1136/gutjnl-2016-311651">Uncoupling of mucosal gene regulation, <span class="nocase">mRNA</span> splicing and adherent microbiota signatures in inflammatory bowel disease</a>. Gut. 2017;66:2087–97.
</div>
<div id="ref-tang2017" class="csl-entry">
80. Tang MS, Bowcutt R, Leung JM, Wolff MJ, Gundra UM, Hudesman D, et al. <a href="https://doi.org/10.1097/MIB.0000000000001208">Integrated Analysis of Biopsies from Inflammatory Bowel Disease Patients Identifies SAA1 as a Link Between Mucosal Microbes with TH17 and TH22 Cells</a>. Inflammatory Bowel Diseases. 2017;23:1544–54.
</div>
<div id="ref-krassowski2020" class="csl-entry">
94. Krassowski M, Das V, Sahu SK, Misra BB. <a href="https://doi.org/10.3389/fgene.2020.610798">State of the field in multi-omics research: From computational needs to data mining and sharing</a>. Frontiers in Genetics. 2020;11:1598.
</div>
<div id="ref-cavill2016" class="csl-entry">
99. Cavill R, Jennen D, Kleinjans J, Briedé JJ. <a href="https://doi.org/10.1093/bib/bbv090">Transcriptomic and metabolomic data integration</a>. Briefings in Bioinformatics. 2016;17:891–901.
</div>
<div id="ref-cantini2021" class="csl-entry">
129. Cantini L, Zakeri P, Hernandez C, Naldi A, Thieffry D, Remy E, et al. <a href="https://doi.org/10.1038/s41467-020-20430-7">Benchmarking joint multi-omics dimensionality reduction approaches for the study of cancer</a>. Nature Communications. 2021;12:124.
</div>
<div id="ref-martínez-mira2018" class="csl-entry">
140. Martínez-Mira C, Conesa A, Tarazona S. <a href="https://doi.org/10.1101/421834">MOSim: Multi-Omics Simulation in R</a>. 2018.
</div>
<div id="ref-mccarthy2012" class="csl-entry">
145. McCarthy DJ, Chen Y, Smyth GK. <a href="https://doi.org/10.1093/nar/gks042">Differential expression analysis of multifactor RNA-Seq experiments with respect to biological variation</a>. Nucleic Acids Research. 2012;40:4288–97.
</div>
<div id="ref-sudhakar2022" class="csl-entry">
148. Sudhakar P, Alsoud D, Wellens J, Verstockt S, Arnauts K, Verstockt B, et al. <a href="https://doi.org/10.1093/ecco-jcc/jjac027">Tailoring multi-omics to inflammatory bowel diseases: All for one and one for all</a>. Journal of Crohn’s and Colitis. 2022;jjac027.
</div>
<div id="ref-morgan2015" class="csl-entry">
150. Morgan XC, Kabakchiev B, Waldron L, Tyler AD, Tickle TL, Milgrom R, et al. <a href="https://doi.org/10.1186/s13059-015-0637-x">Associations between host gene expression, the mucosal microbiome, and clinical outcome in the pelvic pouch of patients with inflammatory bowel disease</a>. Genome Biology. 2015;16:67.
</div>
<div id="ref-howell2018" class="csl-entry">
151. Howell KJ, Kraiczy J, Nayak KM, Gasparetto M, Ross A, Lee C, et al. <a href="https://doi.org/10.1053/j.gastro.2017.10.007">DNA Methylation and Transcription Patterns in Intestinal Epithelial Cells From Pediatric Patients With Inflammatory Bowel Diseases Differentiate Disease Subtypes and Associate With Outcome</a>. Gastroenterology. 2018;154:585–98.
</div>
<div id="ref-revilla2021" class="csl-entry">
198. Revilla L, Mayorgas A, Corraliza AM, Masamunt MC, Metwaly A, Haller D, et al. <a href="https://doi.org/10.1371/journal.pone.0246367">Multi-omic modelling of inflammatory bowel disease with regularized canonical correlation analysis</a>. PLOS ONE. 2021;16:e0246367.
</div>
<div id="ref-callahan2017" class="csl-entry">
199. Callahan BJ, McMurdie PJ, Holmes SP. <a href="https://doi.org/10.1038/ismej.2017.119">Exact sequence variants should replace operational taxonomic units in marker-gene data analysis</a>. The ISME Journal. 2017;11:2639–43.
</div>
<div id="ref-nearing2022" class="csl-entry">
200. Nearing JT, Douglas GM, Hayes MG, MacDonald J, Desai DK, Allward N, et al. <a href="https://doi.org/10.1038/s41467-022-28034-z">Microbiome differential abundance methods produce different results across 38 datasets</a>. Nature Communications. 2022;13:342.
</div>
<div id="ref-louca2018" class="csl-entry">
201. Louca S, Doebeli M, Parfrey LW. <a href="https://doi.org/10.1186/s40168-018-0420-9">Correcting for 16S rRNA gene copy numbers in microbiome surveys remains an unsolved problem</a>. Microbiome. 2018;6:41.
</div>
<div id="ref-nadalian2021" class="csl-entry">
202. Nadalian B, Yadegar A, Houri H, Olfatifar M, Shahrokh S, Asadzadeh Aghdaei H, et al. <a href="https://doi.org/10.1111/jgh.15260">Prevalence of the pathobiont adherent-invasive Escherichia coli and inflammatory bowel disease: a systematic review and meta-analysis</a>. Journal of Gastroenterology and Hepatology. 2021;36:852–63.
</div>
<div id="ref-chalise2016" class="csl-entry">
203. Chalise P, Raghavan R, Fridley BL. <a href="https://doi.org/10.1016/j.cmpb.2016.02.011">InterSIM: Simulation tool for multiple integrative <span>‘</span>omic datasets<span>’</span></a>. Computer methods and programs in biomedicine. 2016;128:69–74.
</div>
<div id="ref-chung2019" class="csl-entry">
204. Chung R-H, Kang C-Y. <a href="https://doi.org/10.1093/gigascience/giz045">A multi-omics data simulator for complex disease studies and its application to evaluate multi-omics data analysis methods for disease classification</a>. GigaScience. 2019;8.
</div>
<div id="ref-ugidos2020" class="csl-entry">
205. Ugidos M, Tarazona S, Prats-Montalbán JM, Ferrer A, Conesa A. MultiBaC: A strategy to remove batch effects between different omic data types: Statistical Methods in Medical Research. 2020. <a href="https://doi.org/10.1177/0962280220907365">https://doi.org/10.1177/0962280220907365</a>.
</div>
<div id="ref-bersanelli2016" class="csl-entry">
206. Bersanelli M, Mosca E, Remondini D, Giampieri E, Sala C, Castellani G, et al. <a href="https://doi.org/10.1186/s12859-015-0857-9">Methods for the integration of multi-omics data: Mathematical aspects</a>. BMC Bioinformatics. 2016;17:S15.
</div>
<div id="ref-holmes2012" class="csl-entry">
207. Holmes E, Li JV, Marchesi JR, Nicholson JK. <a href="https://doi.org/10.1016/j.cmet.2012.10.007">Gut Microbiota Composition and Activity in Relation to Host Metabolic Phenotype and Disease Risk</a>. Cell Metabolism. 2012;16:559–64.
</div>
<div id="ref-brand2021" class="csl-entry">
209. Brand EC, Klaassen MAY, Gacesa R, Vich Vila A, Ghosh H, Zoete MR de, et al. <a href="https://doi.org/10.1053/j.gastro.2021.01.030">Healthy Cotwins Share Gut Microbiome Signatures With Their Inflammatory Bowel Disease Twins and Unrelated Patients</a>. Gastroenterology. 2021;160:1970–85.
</div>
<div id="ref-susin2020" class="csl-entry">
210. Susin A, Wang Y, Lê Cao K-A, Calle ML. <a href="https://doi.org/10.1093/nargab/lqaa029">Variable selection in microbiome compositional data analysis</a>. NAR Genomics and Bioinformatics. 2020;2:lqaa029.
</div>
<div id="ref-sartor2006" class="csl-entry">
211. Sartor RB. <a href="https://doi.org/10.1038/ncpgasthep0528">Mechanisms of disease: pathogenesis of Crohn’s disease and ulcerative colitis</a>. Nature Clinical Practice Gastroenterology &amp; Hepatology. 2006;3:390–407.
</div>
<div id="ref-vandeputte2017" class="csl-entry">
212. Vandeputte D, Kathagen G, D’hoe K, Vieira-Silva S, Valles-Colomer M, Sabino J, et al. <a href="https://doi.org/10.1038/nature24460">Quantitative microbiome profiling links gut community variation to microbial load</a>. Nature. 2017;551:507–11.
</div>
<div id="ref-vila-casadesús2016" class="csl-entry">
213. Vila-Casadesús M, Gironella M, Lozano JJ. <a href="https://doi.org/10.1371/journal.pone.0151127">MiRComb: An R Package to Analyse miRNA-mRNA Interactions. Examples across Five Digestive Cancers</a>. PloS One. 2016;11:e0151127.
</div>
<div id="ref-häsler2017" class="csl-entry">
214. Häsler R, Sheibani-Tezerji R, Sinha A, Barann M, Rehman A, Esser D, et al. <a href="https://doi.org/10.1136/gutjnl-2016-311651">Uncoupling of mucosal gene regulation, mRNA splicing and adherent microbiota signatures in inflammatory bowel disease</a>. Gut. 2017;66:2087–97.
</div>
<div id="ref-goldberg1991" class="csl-entry">
215. Goldberg D. <a href="https://doi.org/10.1145/103162.103163">What every computer scientist should know about floating-point arithmetic</a>. ACM Computing Surveys. 1991;23:548.
</div>
<div id="ref-yan2012" class="csl-entry">
216. Yan L, Ma C, Wang D, Hu Q, Qin M, Conroy JM, et al. <a href="https://doi.org/10.1186/1471-2164-13-689">OSAT: A tool for sample-to-batch allocations in genomics experiments</a>. BMC Genomics. 2012;13:689.
</div>
<div id="ref-papenberg2020" class="csl-entry">
217. Papenberg M, Klau GW. Using anticlustering to partition data sets into equivalent parts. Psychological Methods. 2020. <a href="https://doi.org/10.1037/met0000301">https://doi.org/10.1037/met0000301</a>.
</div>
<div id="ref-sinke2021" class="csl-entry">
218. Sinke L, Cats D, Heijmans BT. Omixer: Multivariate and reproducible sample randomization to proactively counter batch effects in omics studies. Bioinformatics. 2021. <a href="https://doi.org/10.1093/bioinformatics/btab159">https://doi.org/10.1093/bioinformatics/btab159</a>.
</div>
<div id="ref-haberman2014" class="csl-entry">
219. Haberman Y, Tickle TL, Dexheimer PJ, Kim M-O, Tang D, Karns R, et al. <a href="https://doi.org/10.1172/JCI75436">Pediatric Crohn disease patients exhibit specific ileal transcriptome and microbiome signature</a>. The Journal of Clinical Investigation. 2014;124:3617–33.
</div>
<div id="ref-community2021" class="csl-entry">
220. Community TTW. <a href="https://doi.org/10.5281/ZENODO.3233853">The turing way: A handbook for reproducible, ethical and collaborative research</a>. The Turing Way Community; 2021.
</div>
<div id="ref-mengMultivariateApproachIntegration2014" class="csl-entry">
221. Meng C, Kuster B, Culhane AC, Gholami AM. <a href="https://doi.org/10.1186/1471-2105-15-162">A multivariate approach to the integration of multi-omics datasets</a>. BMC Bioinformatics. 2014;15:162.
</div>
<div id="ref-pearl2011" class="csl-entry">
222. Pearl J. <a href="https://escholarship.org/uc/item/3tv1b3bg">Transportability across studies: A formal approach</a>. 2011.
</div>
<div id="ref-simpson1951" class="csl-entry">
223. Simpson EH. <a href="https://doi.org/10.1111/j.2517-6161.1951.tb00088.x">The Interpretation of Interaction in Contingency Tables</a>. Journal of the Royal Statistical Society: Series B (Methodological). 1951;13:238–41.
</div>
<div id="ref-stillman1984" class="csl-entry">
224. Stillman DJ. Interactions of a Eukaryotic RNA Polymerase III Transcription Factor with Promoters. Biology; 1984.
</div>
<div id="ref-lee1993" class="csl-entry">
225. Lee RC, Feinbaum RL, Ambros V. <a href="https://doi.org/10.1016/0092-8674(93)90529-Y">The C. elegans heterochronic gene lin-4 encodes small RNAs with antisense complementarity to lin-14</a>. Cell. 1993;75:843–54.
</div>
<div id="ref-hamilton1999" class="csl-entry">
226. Hamilton AJ, Baulcombe DC. <a href="https://doi.org/10.1126/science.286.5441.950">A species of small antisense RNA in posttranscriptional gene silencing in plants</a>. Science. 1999;286:950–2.
</div>
<div id="ref-thornburg2022" class="csl-entry">
227. Thornburg ZR, Bianchi DM, Brier TA, Gilbert BR, Earnest TM, Melo MCR, et al. <a href="https://doi.org/10.1016/j.cell.2021.12.025">Fundamental behaviors emerge from simulations of a living minimal cell</a>. Cell. 2022;185:345–360.e28.
</div>
<div id="ref-criss2021" class="csl-entry">
228. Criss ZK, Bhasin N, Di Rienzi SC, Rajan A, Deans-Fielder K, Swaminathan G, et al. <a href="https://doi.org/10.1152/physiolgenomics.00061.2021">Drivers of transcriptional variance in human intestinal epithelial organoids</a>. Physiological Genomics. 2021;53:486–508.
</div>
<div id="ref-arumugam2011" class="csl-entry">
229. Arumugam M, Raes J, Pelletier E, Le Paslier D, Yamada T, Mende DR, et al. <a href="https://doi.org/10.1038/nature09944">Enterotypes of the human gut microbiome</a>. Nature. 2011;473:174–80.
</div>
<div id="ref-koren2013" class="csl-entry">
230. Koren O, Knights D, Gonzalez A, Waldron L, Segata N, Knight R, et al. <a href="https://doi.org/10.1371/journal.pcbi.1002863">A Guide to Enterotypes across the Human Body: Meta-Analysis of Microbial Community Structures in Human Microbiome Datasets</a>. PLOS Computational Biology. 2013;9:e1002863.
</div>
<div id="ref-cheng2019" class="csl-entry">
231. Cheng M, Ning K. <a href="https://doi.org/10.1016/j.gpb.2018.02.004">Stereotypes About Enterotype: the Old and New Ideas</a>. Genomics, Proteomics &amp; Bioinformatics. 2019;17:4–12.
</div>
<div id="ref-hasan2019" class="csl-entry">
232. Hasan N, Yang H. <a href="https://doi.org/10.7717/peerj.7502">Factors affecting the composition of the gut microbiota, and its modulation</a>. PeerJ. 2019;7:e7502.
</div>
<div id="ref-bringiotti2014" class="csl-entry">
233. Bringiotti R, Ierardi E, Lovero R, Losurdo G, Leo AD, Principi M. <a href="https://doi.org/10.4291/wjgp.v5.i4.550">Intestinal microbiota: The explosive mixture at the origin of inflammatory bowel disease?</a> World Journal of Gastrointestinal Pathophysiology. 2014;5:550–9.
</div>
<div id="ref-lloyd-price2019" class="csl-entry">
234. Lloyd-Price J, Arze C, Ananthakrishnan AN, Schirmer M, Avila-Pacheco J, Poon TW, et al. <a href="https://doi.org/10.1038/s41586-019-1237-9">Multi-omics of the gut microbial ecosystem in inflammatory bowel diseases</a>. Nature. 2019;569:655.
</div>
<div id="ref-douglas2018" class="csl-entry">
235. Douglas GM, Hansen R, Jones CMA, Dunn KA, Comeau AM, Bielawski JP, et al. <a href="https://doi.org/10.1186/s40168-018-0398-3">Multi-omics differentially classify disease state and treatment outcome in pediatric crohn<span>’</span>s disease</a>. Microbiome. 2018;6:13.
</div>
<div id="ref-takahashi2016" class="csl-entry">
236. Takahashi K, Nishida A, Fujimoto T, Fujii M, Shioya M, Imaeda H, et al. <a href="https://doi.org/10.1159/000441768">Reduced abundance of butyrate-producing bacteria species in the fecal microbial community in crohn’s disease</a>. Digestion. 2016;93:59–65.
</div>
<div id="ref-halfvarson2017" class="csl-entry">
237. Halfvarson J, Brislawn CJ, Lamendella R, Vázquez-Baeza Y, Walters WA, Bramer LM, et al. <a href="https://doi.org/10.1038/nmicrobiol.2017.4">Dynamics of the human gut microbiome in inflammatory bowel disease</a>. Nature Microbiology. 2017;2:1–7.
</div>
<div id="ref-edgar2018" class="csl-entry">
238. Edgar R. <a href="https://doi.org/10.7717/peerj.5030">Taxonomy annotation and guide tree errors in 16S rRNA databases</a>. PeerJ. 2018;6:e5030.
</div>
<div id="ref-ungar2022" class="csl-entry">
239. Ungar B, Yavzori M, Fudim E, Picard O, Kopylov U, Eliakim R, et al. Host transcriptome signatures in human faecal-washes predict histological remission in patients with IBD. Gut. 2022. <a href="https://doi.org/10.1136/gutjnl-2021-325516">https://doi.org/10.1136/gutjnl-2021-325516</a>.
</div>
<div id="ref-zhang2019" class="csl-entry">
240. Zhang X, Li L, Butcher J, Stintzi A, Figeys D. <a href="https://doi.org/10.1186/s40168-019-0767-6">Advancing functional and translational microbiome research using meta-omics approaches</a>. Microbiome. 2019;7:154.
</div>
<div id="ref-jiang2019" class="csl-entry">
241. Jiang D, Armour CR, Hu C, Mei M, Tian C, Sharpton TJ, et al. <a href="https://doi.org/10.3389/fgene.2019.00995">Microbiome multi-omics network analysis: Statistical considerations, limitations, and opportunities</a>. Frontiers in Genetics. 2019;10.
</div>
<div id="ref-haynes2018" class="csl-entry">
242. Haynes WA, Tomczak A, Khatri P. <a href="https://doi.org/10.1038/s41598-018-19333-x">Gene annotation bias impedes biomedical research</a>. Scientific Reports. 2018;8:1362.
</div>
<div id="ref-collij2021" class="csl-entry">
243. Collij V, Klaassen MAY, Weersma RK, Vila AV. <a href="https://doi.org/10.1007/s00439-020-02218-3">Gut microbiota in inflammatory bowel diseases: moving from basic science to clinical applications</a>. Human Genetics. 2021;140:703–8.
</div>
</div>
<div class="footnotes">
<hr />
<ol start="7">
<li id="fn7"><p>See an exploration of this using AUCell <a href="https://bioconductor.org/packages/AUCell" class="uri">https://bioconductor.org/packages/AUCell</a> on this website: <a href="https://llrs.github.io/BaseSet_scRNAseq/AUCell.html" class="uri">https://llrs.github.io/BaseSet_scRNAseq/AUCell.html</a>.<a href="discussion.html#fnref7" class="footnote-back">↩︎</a></p></li>
</ol>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="results.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="conclusions.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": {
"href": "https://github.com/llrs/thesis/"
},
"facebook": false,
"twitter": {
"href": "https://twitter.com/intent/tweet?text=Nice%20read%20of%20%40Lluis_Revilla%20thesis"
},
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/llrs/thesis/edit/master/05_discussion.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": "https://github.com/llrs/thesis/blob/master/05_discussion.Rmd",
"text": "View"
},
"download": ["_main.pdf", "_main.epub"],
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": false,
"toc_float": true,
"toc_appendix": true,
"toc_depth": 2,
"scroll_highlight": true
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
