<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 5 Discussion | Data integration on inflammatory bowel disease</title>
  <meta name="description" content="Thesis about the relationship between microbiome and transcriptome on the gastrointestinal tract on patients with IBD and some controls to discover key differences on this relationship." />
  <meta name="generator" content="bookdown 0.24 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 5 Discussion | Data integration on inflammatory bowel disease" />
  <meta property="og:type" content="book" />
  <meta property="og:url" content="https://thesis.llrs.dev" />
  <meta property="og:image" content="https://thesis.llrs.dev/images/R_logo_science.png" />
  <meta property="og:description" content="Thesis about the relationship between microbiome and transcriptome on the gastrointestinal tract on patients with IBD and some controls to discover key differences on this relationship." />
  <meta name="github-repo" content="llrs/thesis" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 5 Discussion | Data integration on inflammatory bowel disease" />
  <meta name="twitter:site" content="@Lluis_Revilla" />
  <meta name="twitter:description" content="Thesis about the relationship between microbiome and transcriptome on the gastrointestinal tract on patients with IBD and some controls to discover key differences on this relationship." />
  <meta name="twitter:image" content="https://thesis.llrs.dev/images/R_logo_science.png" />

<meta name="author" content="Lluís Revilla Sancho" />


<meta name="date" content="2022-03-07" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="results.html"/>
<link rel="next" href="conclusions.html"/>
<script src="libs/header-attrs-2.12/header-attrs.js"></script>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>



<style type="text/css">
/* Used with Pandoc 2.11+ new --citeproc when CSL is used */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><strong><a href="./">Data integration on IBD</a></strong></li>

<li class="divider"></li>
<li><a href="index.html#preface">Preface<span></span></a></li>
<li><a href="abstract.html#abstract">Abstract<span></span></a></li>
<li class="chapter" data-level="1" data-path="introduction.html"><a href="introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction<span></span></a>
<ul>
<li class="chapter" data-level="1.1" data-path="introduction.html"><a href="introduction.html#IBD"><i class="fa fa-check"></i><b>1.1</b> Inflammatory bowel disease<span></span></a>
<ul>
<li class="chapter" data-level="1.1.1" data-path="introduction.html"><a href="introduction.html#etiology-and-pathogenesis"><i class="fa fa-check"></i><b>1.1.1</b> Etiology and pathogenesis<span></span></a></li>
<li class="chapter" data-level="1.1.2" data-path="introduction.html"><a href="introduction.html#CD"><i class="fa fa-check"></i><b>1.1.2</b> CD physiology<span></span></a></li>
<li class="chapter" data-level="1.1.3" data-path="introduction.html"><a href="introduction.html#UC"><i class="fa fa-check"></i><b>1.1.3</b> UC physiology<span></span></a></li>
<li class="chapter" data-level="1.1.4" data-path="introduction.html"><a href="introduction.html#treatment"><i class="fa fa-check"></i><b>1.1.4</b> Treatment<span></span></a></li>
<li class="chapter" data-level="1.1.5" data-path="introduction.html"><a href="introduction.html#summary-ibd"><i class="fa fa-check"></i><b>1.1.5</b> Summary<span></span></a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="introduction.html"><a href="introduction.html#integration-studies-on-ibd"><i class="fa fa-check"></i><b>1.2</b> Integration studies on IBD<span></span></a>
<ul>
<li class="chapter" data-level="1.2.1" data-path="introduction.html"><a href="introduction.html#data-origin"><i class="fa fa-check"></i><b>1.2.1</b> Type of data used for integration analysis<span></span></a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="introduction.html"><a href="introduction.html#integration"><i class="fa fa-check"></i><b>1.3</b> Integration<span></span></a>
<ul>
<li class="chapter" data-level="1.3.1" data-path="introduction.html"><a href="introduction.html#classification-of-integration-method"><i class="fa fa-check"></i><b>1.3.1</b> Classification of integration method<span></span></a></li>
<li class="chapter" data-level="1.3.2" data-path="introduction.html"><a href="introduction.html#interpretation"><i class="fa fa-check"></i><b>1.3.2</b> Interpretation<span></span></a></li>
<li class="chapter" data-level="1.3.3" data-path="introduction.html"><a href="introduction.html#reviews"><i class="fa fa-check"></i><b>1.3.3</b> Reviews<span></span></a></li>
<li class="chapter" data-level="1.3.4" data-path="introduction.html"><a href="introduction.html#methods-integration"><i class="fa fa-check"></i><b>1.3.4</b> Summary<span></span></a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="2" data-path="hypothesis-and-objectives.html"><a href="hypothesis-and-objectives.html"><i class="fa fa-check"></i><b>2</b> Hypothesis and objectives<span></span></a>
<ul>
<li class="chapter" data-level="2.1" data-path="hypothesis-and-objectives.html"><a href="hypothesis-and-objectives.html#hypothesis"><i class="fa fa-check"></i><b>2.1</b> Hypothesis<span></span></a></li>
<li class="chapter" data-level="2.2" data-path="hypothesis-and-objectives.html"><a href="hypothesis-and-objectives.html#objectives"><i class="fa fa-check"></i><b>2.2</b> Objectives<span></span></a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="materials-and-methods.html"><a href="materials-and-methods.html"><i class="fa fa-check"></i><b>3</b> Materials and methods<span></span></a>
<ul>
<li class="chapter" data-level="3.1" data-path="materials-and-methods.html"><a href="materials-and-methods.html#datasets"><i class="fa fa-check"></i><b>3.1</b> Datasets<span></span></a>
<ul>
<li class="chapter" data-level="3.1.1" data-path="materials-and-methods.html"><a href="materials-and-methods.html#methods-puget"><i class="fa fa-check"></i><b>3.1.1</b> Puget’s dataset<span></span></a></li>
<li class="chapter" data-level="3.1.2" data-path="materials-and-methods.html"><a href="materials-and-methods.html#methods-hsct"><i class="fa fa-check"></i><b>3.1.2</b> HSCT dataset<span></span></a></li>
<li class="chapter" data-level="3.1.3" data-path="materials-and-methods.html"><a href="materials-and-methods.html#methods-hasler"><i class="fa fa-check"></i><b>3.1.3</b> Häsler’s dataset<span></span></a></li>
<li class="chapter" data-level="3.1.4" data-path="materials-and-methods.html"><a href="materials-and-methods.html#methods-morgan"><i class="fa fa-check"></i><b>3.1.4</b> Morgan’s dataset<span></span></a></li>
<li class="chapter" data-level="3.1.5" data-path="materials-and-methods.html"><a href="materials-and-methods.html#methods-barcelona"><i class="fa fa-check"></i><b>3.1.5</b> BARCELONA dataset<span></span></a></li>
<li class="chapter" data-level="3.1.6" data-path="materials-and-methods.html"><a href="materials-and-methods.html#methods-howell"><i class="fa fa-check"></i><b>3.1.6</b> Howell’s dataset<span></span></a></li>
<li class="chapter" data-level="3.1.7" data-path="materials-and-methods.html"><a href="materials-and-methods.html#methods-hernandez"><i class="fa fa-check"></i><b>3.1.7</b> Hernández’ dataset<span></span></a></li>
</ul></li>
<li class="chapter" data-level="3.2" data-path="materials-and-methods.html"><a href="materials-and-methods.html#processing-samples"><i class="fa fa-check"></i><b>3.2</b> Processing samples<span></span></a>
<ul>
<li class="chapter" data-level="3.2.1" data-path="materials-and-methods.html"><a href="materials-and-methods.html#transcriptome-sequencing"><i class="fa fa-check"></i><b>3.2.1</b> Transcriptome sequencing<span></span></a></li>
<li class="chapter" data-level="3.2.2" data-path="materials-and-methods.html"><a href="materials-and-methods.html#microbial-dna-sequencing"><i class="fa fa-check"></i><b>3.2.2</b> Microbial DNA sequencing<span></span></a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="materials-and-methods.html"><a href="materials-and-methods.html#integration-methods"><i class="fa fa-check"></i><b>3.3</b> Integration methods<span></span></a>
<ul>
<li class="chapter" data-level="3.3.1" data-path="materials-and-methods.html"><a href="materials-and-methods.html#mcia"><i class="fa fa-check"></i><b>3.3.1</b> MCIA<span></span></a></li>
<li class="chapter" data-level="3.3.2" data-path="materials-and-methods.html"><a href="materials-and-methods.html#stategra"><i class="fa fa-check"></i><b>3.3.2</b> STATegRa<span></span></a></li>
<li class="chapter" data-level="3.3.3" data-path="materials-and-methods.html"><a href="materials-and-methods.html#rgcca"><i class="fa fa-check"></i><b>3.3.3</b> RGCCA<span></span></a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="materials-and-methods.html"><a href="materials-and-methods.html#regularized-generalized-canonical-correlation-analysis"><i class="fa fa-check"></i><b>3.4</b> Regularized generalized canonical correlation analysis<span></span></a>
<ul>
<li class="chapter" data-level="3.4.1" data-path="materials-and-methods.html"><a href="materials-and-methods.html#description"><i class="fa fa-check"></i><b>3.4.1</b> Description<span></span></a></li>
<li class="chapter" data-level="3.4.2" data-path="materials-and-methods.html"><a href="materials-and-methods.html#output"><i class="fa fa-check"></i><b>3.4.2</b> Output<span></span></a></li>
<li class="chapter" data-level="3.4.3" data-path="materials-and-methods.html"><a href="materials-and-methods.html#models"><i class="fa fa-check"></i><b>3.4.3</b> Models<span></span></a></li>
</ul></li>
<li class="chapter" data-level="3.5" data-path="materials-and-methods.html"><a href="materials-and-methods.html#statistics"><i class="fa fa-check"></i><b>3.5</b> Statistics<span></span></a></li>
<li class="chapter" data-level="3.6" data-path="materials-and-methods.html"><a href="materials-and-methods.html#functional-enrichment-methods"><i class="fa fa-check"></i><b>3.6</b> Functional enrichment methods<span></span></a>
<ul>
<li class="chapter" data-level="3.6.1" data-path="materials-and-methods.html"><a href="materials-and-methods.html#hypergeometric-test"><i class="fa fa-check"></i><b>3.6.1</b> Hypergeometric test<span></span></a></li>
<li class="chapter" data-level="3.6.2" data-path="materials-and-methods.html"><a href="materials-and-methods.html#proportion-test"><i class="fa fa-check"></i><b>3.6.2</b> Proportion test<span></span></a></li>
<li class="chapter" data-level="3.6.3" data-path="materials-and-methods.html"><a href="materials-and-methods.html#gene-set-enrichment-analysis"><i class="fa fa-check"></i><b>3.6.3</b> Gene Set Enrichment Analysis<span></span></a></li>
<li class="chapter" data-level="3.6.4" data-path="materials-and-methods.html"><a href="materials-and-methods.html#gsva"><i class="fa fa-check"></i><b>3.6.4</b> GSVA<span></span></a></li>
</ul></li>
<li class="chapter" data-level="3.7" data-path="materials-and-methods.html"><a href="materials-and-methods.html#variance-and-diversity-methods"><i class="fa fa-check"></i><b>3.7</b> Variance and diversity methods<span></span></a>
<ul>
<li class="chapter" data-level="3.7.1" data-path="materials-and-methods.html"><a href="materials-and-methods.html#permanova"><i class="fa fa-check"></i><b>3.7.1</b> PERMANOVA<span></span></a></li>
<li class="chapter" data-level="3.7.2" data-path="materials-and-methods.html"><a href="materials-and-methods.html#globaltest"><i class="fa fa-check"></i><b>3.7.2</b> globaltest<span></span></a></li>
<li class="chapter" data-level="3.7.3" data-path="materials-and-methods.html"><a href="materials-and-methods.html#diversity-indices"><i class="fa fa-check"></i><b>3.7.3</b> Diversity indices<span></span></a></li>
</ul></li>
<li class="chapter" data-level="3.8" data-path="materials-and-methods.html"><a href="materials-and-methods.html#other-methods"><i class="fa fa-check"></i><b>3.8</b> Other methods<span></span></a>
<ul>
<li class="chapter" data-level="3.8.1" data-path="materials-and-methods.html"><a href="materials-and-methods.html#wgcna"><i class="fa fa-check"></i><b>3.8.1</b> WGCNA<span></span></a></li>
<li class="chapter" data-level="3.8.2" data-path="materials-and-methods.html"><a href="materials-and-methods.html#baseset"><i class="fa fa-check"></i><b>3.8.2</b> BaseSet<span></span></a></li>
<li class="chapter" data-level="3.8.3" data-path="materials-and-methods.html"><a href="materials-and-methods.html#experdesign"><i class="fa fa-check"></i><b>3.8.3</b> experDesign<span></span></a></li>
<li class="chapter" data-level="3.8.4" data-path="materials-and-methods.html"><a href="materials-and-methods.html#ROC"><i class="fa fa-check"></i><b>3.8.4</b> ROC- AUC<span></span></a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="results.html"><a href="results.html"><i class="fa fa-check"></i><b>4</b> Results<span></span></a>
<ul>
<li class="chapter" data-level="4.1" data-path="results.html"><a href="results.html#packages"><i class="fa fa-check"></i><b>4.1</b> Packages<span></span></a>
<ul>
<li class="chapter" data-level="4.1.1" data-path="results.html"><a href="results.html#experdesign-1"><i class="fa fa-check"></i><b>4.1.1</b> experDesign<span></span></a></li>
<li class="chapter" data-level="4.1.2" data-path="results.html"><a href="results.html#baseset-1"><i class="fa fa-check"></i><b>4.1.2</b> BaseSet<span></span></a></li>
<li class="chapter" data-level="4.1.3" data-path="results.html"><a href="results.html#intermodels"><i class="fa fa-check"></i><b>4.1.3</b> inteRmodels<span></span></a></li>
</ul></li>
<li class="chapter" data-level="4.2" data-path="results.html"><a href="results.html#analysis"><i class="fa fa-check"></i><b>4.2</b> Analysis<span></span></a>
<ul>
<li class="chapter" data-level="4.2.1" data-path="results.html"><a href="results.html#results-puget"><i class="fa fa-check"></i><b>4.2.1</b> Puget’s dataset<span></span></a></li>
<li class="chapter" data-level="4.2.2" data-path="results.html"><a href="results.html#results-hsct"><i class="fa fa-check"></i><b>4.2.2</b> HSCT dataset<span></span></a></li>
<li class="chapter" data-level="4.2.3" data-path="results.html"><a href="results.html#results-hasler"><i class="fa fa-check"></i><b>4.2.3</b> Häsler’s dataset<span></span></a></li>
<li class="chapter" data-level="4.2.4" data-path="results.html"><a href="results.html#results-morgan"><i class="fa fa-check"></i><b>4.2.4</b> Morgan’s dataset<span></span></a></li>
<li class="chapter" data-level="4.2.5" data-path="results.html"><a href="results.html#results-barcelona"><i class="fa fa-check"></i><b>4.2.5</b> BARCELONA dataset<span></span></a></li>
<li class="chapter" data-level="4.2.6" data-path="results.html"><a href="results.html#results-howell"><i class="fa fa-check"></i><b>4.2.6</b> Howell’s dataset<span></span></a></li>
<li class="chapter" data-level="4.2.7" data-path="results.html"><a href="results.html#results-hernandez"><i class="fa fa-check"></i><b>4.2.7</b> Hernández’ dataset<span></span></a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="discussion.html"><a href="discussion.html"><i class="fa fa-check"></i><b>5</b> Discussion<span></span></a>
<ul>
<li class="chapter" data-level="5.1" data-path="discussion.html"><a href="discussion.html#preliminary-steps"><i class="fa fa-check"></i><b>5.1</b> Preliminary steps<span></span></a>
<ul>
<li class="chapter" data-level="5.1.1" data-path="discussion.html"><a href="discussion.html#the-datasets"><i class="fa fa-check"></i><b>5.1.1</b> The datasets<span></span></a></li>
<li class="chapter" data-level="5.1.2" data-path="discussion.html"><a href="discussion.html#the-methods"><i class="fa fa-check"></i><b>5.1.2</b> The methods<span></span></a></li>
</ul></li>
<li class="chapter" data-level="5.2" data-path="discussion.html"><a href="discussion.html#designing-models"><i class="fa fa-check"></i><b>5.2</b> Designing models<span></span></a></li>
<li class="chapter" data-level="5.3" data-path="discussion.html"><a href="discussion.html#evaluating-models"><i class="fa fa-check"></i><b>5.3</b> Evaluating models<span></span></a></li>
<li class="chapter" data-level="5.4" data-path="discussion.html"><a href="discussion.html#relevance-for-other-research"><i class="fa fa-check"></i><b>5.4</b> Relevance for other research<span></span></a></li>
<li class="chapter" data-level="5.5" data-path="discussion.html"><a href="discussion.html#implications-of-research"><i class="fa fa-check"></i><b>5.5</b> Implications of research<span></span></a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="conclusions.html"><a href="conclusions.html"><i class="fa fa-check"></i><b>6</b> Conclusions<span></span></a></li>
<li class="chapter" data-level="7" data-path="acknowledgments.html"><a href="acknowledgments.html"><i class="fa fa-check"></i><b>7</b> Acknowledgments<span></span></a></li>
<li><a href="references.html#references">References<span></span></a></li>
<li class="appendix"><span><b>Appendix<span></span></b></span></li>
<li class="chapter" data-level="A" data-path="online-resources.html"><a href="online-resources.html"><i class="fa fa-check"></i><b>A</b> Online resources<span></span></a></li>
<li class="chapter" data-level="B" data-path="software.html"><a href="software.html"><i class="fa fa-check"></i><b>B</b> Software<span></span></a>
<ul>
<li class="chapter" data-level="B.1" data-path="software.html"><a href="software.html#listed"><i class="fa fa-check"></i><b>B.1</b> Listed<span></span></a></li>
<li class="chapter" data-level="B.2" data-path="software.html"><a href="software.html#by-projectpublication"><i class="fa fa-check"></i><b>B.2</b> By project/publication<span></span></a></li>
</ul></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Data integration on inflammatory bowel disease</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div class = "wip">
This tesis is ongoing and still without hard date to finish it. Watch the <a href="https://github.com/llrs/thesis">GitHub repository</a> or wait for a release announcement on Twitter on <a href="https://twitter.com/Lluis_Revilla">my profile</a>.
</div>
<div id="discussion" class="section level1 hasAnchor" number="5">
<h1><span class="header-section-number">Chapter 5</span> Discussion<a href="discussion.html#discussion" class="anchor-section" aria-label="Anchor link to header"></a></h1>
<p>In this chapter we will summarize the main findings in relation to the broad research community and other work as well as the impact of the results to improve further research in the future or be used on clinical environments.</p>
<div id="preliminary-steps" class="section level2 hasAnchor" number="5.1">
<h2><span class="header-section-number">5.1</span> Preliminary steps<a href="discussion.html#preliminary-steps" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>Quality of the initial data for integration is important for a valid integration analysis.
On RNAseq and 16S analysis the sequencing data quality is important.
On sequencing data it is very important to avoid contamination and have enough amount of data for the analysis.
To avoid contamination it is very important the protocol how are processed the samples before the sequencing.
In addition control samples can be added, a blank to control for extraneous material and a sample with known content to confirm that the sequencing worked correctly.</p>
<p>On bulk RNAseq the extraction procedure and sequencing protocol has been setup many years ago and works very well.
Rarely there are problems with bulk RNAseq sequencing and its quality.
On the contrary DNA extraction and 16S sequencing was a new procedure.
Initially it was done with the guidelines of collaborators.
Later, when we did ourselves we did not include control samples but after some problems we started adding at least blank samples to identify those bacteria that might come with the solvent or be due to contamination.</p>
<p>Once the 16S data was obtained we initially processed it to obtain OTUs.
OTUs were the standard some years back.
However, OTUs are not comparable between studies even those that use the same primers.
For this reason, and after a suggestion from a reviewer, we moved to process later 16S sequencing datasets to obtain ASV.
ASV allow to compare the taxonomic imputation between studies using the same primers.
However, comparing 16S taxa ASV or OTUs is hard and selecting the right tool to compare them is important <span class="citation">[<a href="#ref-nearing2022" role="doc-biblioref">196</a>]</span>.</p>
<p>It is also worth to keep in mind that several species have variable number of 16S rRNA genes.
Higher sequences counts of certain 16S might not mean higher abundance of that species compared to another with lower 16S counts.
However, the exact copy number status may change even within the same bacterial species, making the correction difficult <span class="citation">[<a href="#ref-louca2018" role="doc-biblioref">197</a>]</span>.
It is possible to accurately correct for copy number variation on mock populations where the species and the genomes are known (or at least the 16S rRNA), but harder on samples which content is unknown.</p>
<!--# In addition, there are differences on the sequencing reads depending on the  regions selected to amplify the 16S. It would be better to fully sequence the 16S. -->
<p>Furthermore, the precision of 16S sequencing for classification of taxa is not enough to understand its role on the gut.
Adherent invasive cells, are cells with same genetic content that show different behavior <span class="citation">[<a href="#ref-nadalian2021" role="doc-biblioref">198</a>]</span>.
These microbiome is known to be present on patients with IBD.
Thus even with the complete sequence play a different role on the health of the patients.</p>
<p>Most of these steps were out of my control as they were performed by a lab teammate or by collaborators.
Other concerns are currently unavoidable or could not have been found beforehand.</p>
<div id="the-datasets" class="section level3 hasAnchor" number="5.1.1">
<h3><span class="header-section-number">5.1.1</span> The datasets<a href="discussion.html#the-datasets" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>There is not a methodology to calculate the size of cohorts for interaction or multiomic studies.
There are no clear rules about which alpha power or which kind of relations are tested.
This might be due to a lack of mathematical background and modelization of the relations on the biological field.
Further research on this area might help finding which kind of relationships can be found given certain dataset size.
In addition to the highly local and not generalizable interactions that happen on biological samples.
Usually size of the datasets is determined from practical reasons: either costs or patient recruitment on protocols.</p>
<p>There are some efforts to simulate multi-omic datasets <span class="citation">[<a href="#ref-martínez-mira2018" role="doc-biblioref">199</a>]</span>.
They usually focus on RNA-seq, ATAC-seq (DNase-seq), ChIP-seq, small RNA-seq, Methyl-seq or proteomics.
To my knowledge there is no accurate generator of 16S datasets or other microbiome datasets.
This made impossible to simulate and compare different tools on a dataset with known relationships.
Current approaches focus on comparing tools on (several) previously published datasets <span class="citation">[<a href="#ref-cantini2021" role="doc-biblioref">200</a>]</span>, an approach that we have taken too.
This approach was not used to compare different tools, that we also do it, but to validate findings of one dataset in other datasets.</p>
<p>There is not a golden dataset for integration.
There are many possible relationships between variables and different ways these relationships might be: regulated DNA, protein recruitment, transcription factors, siRNA regulation.
Not a single one can be measured on a sufficiently detailed timescale that it can matter and as a result we are left with trying to guess what are the the relationships between variables.
In addition, this is compound by the lack of explanations about how this relationships might come, despite other indications.</p>
<p>Comparing different datasets is complicated because each is collected with different goals and processed differently.
In addition, there is not a resource were datasets of publications are collected or the relationships between different datasets is not shared.
Authors upload their data to different, not centralized, data repositories, such as, gene expression omnibus (<a href="https://www.ncbi.nlm.nih.gov/geo/">GEO</a>), European Genome-Phenome Archive (<a href="https://ega-archive.org/">EGA</a>), European Nucleotide Archive (<a href="https://www.ebi.ac.uk/ena/browser/home">ENA</a>), among others.
Authors might provide the processed data as supplementary material on their articles.
Some projects whose primary purpose is providing data for the community establishing their own dedicated sites to store the data <span class="citation">[<a href="#ref-humanmicrobiomeprojectconsortium2012" role="doc-biblioref">201</a>]</span>.</p>
<p>Finding different datasets for very similarly purposes is very difficult.
And once it is found there might be batch effects.
There are tools to overcome this, but currently only work if they share some features in common <span class="citation">[<a href="#ref-ugidos2020" role="doc-biblioref">202</a>]</span>.
This usually means having undergone similar RNA sequencing procedures.</p>
<p>All this made finding annotated, appropriate and similar datasets difficult.
We looked up for the most similar datasets in order to be able to confirm our results outside our own cohorts.
On this thesis there were two datasets collected and sequenced: the HSCT and the BARCELONA dataset.</p>
<p>As explained on <a href="materials-and-methods.html#methods-hsct">methods</a>, the HSCT dataset is a very special cohort of patients undergoing hematopoietic stem cell transplant.
This treatment is reserved to patients whose all the other treatment failed and this is the only way to reach remission.
This is common on patients with long time CD.
These patients are highly characterized and there is follow up data for several years.
However, it does not include UC patients and they are not representative of the IBD, or even CD, patients.<br />
On the HSCT dataset microbiome features were OTUs.
OTUs can not be imputed afterwards to ASV.
As such, the annotation of the 16S data is not the same as in the others datasets.</p>
<p>The BARCELONA dataset includes samples from CD and UC of an ongoing study.
Similar to the HSCT dataset, patients on this study were followed up for up to a year.
The patients were younger and had less years with the disease, which could provide with more insights on the initial relationships between the microbiome and the mucosa of the IBD.</p>
<p>However, the data quality of BARCELONA was not good enough to make reliable analysis and confidently extract hypothesis or relationships.
It is not clear what happened and as a consequence it is not clear how to avoid such problems on the future.
It could be a problem of the original DNA extraction and/or of its sequencing process which did not include both positive and negative controls.</p>
<p>What happened with the BARCELONA dataset -and the organoids cohort not described here- highlights the importance of data quality checks.
On the BARCELONA cohort despite no batch effect or other quality issues with the data as checked with <code>experDesign</code>, the diversity indices made impossible using it.
The organoids cohort is a dataset that was sequenced following the <code>experDesign</code> recommendations for distribution of samples on different batches.
However, a batch effect was found.
The batch effect was created on the lab process before the samples’ sequencing.
Caution and a good relationship with the data sample and data generator is important to discover these kind of problems and ensure the quality of the data.</p>
<p>Other datasets used from published sourced were assumed they had already passed enough controls and had a good quality.
Nevertheless, they were screened to avoid quality issues.</p>
<p>The Puget’s dataset dataset provided a good benchmark to test the methods and understand how RGCCA works.
It is a completely different type of data (except for the microarray that is similar to the RNAseq).
But we were not as much interested on the biology as on learning about the methods and possibilities of integration.</p>
<p>Häsler data were obtained using the same sequencing techniques from endoscopic biopsies as our dataset HSCT and Barcelona and it is of the same disease/field.
The 16S data was very similar to the The confirmation that the inteRmodel approach works on it it helped to continue forward.</p>
<p>The taxonomy of the different datasets was done differently.
After the first two datasets it was used the SILVA database to annotate the ASV.</p>
<p>Morgan’s dataset is related to IBD but it is from a cohort of patients that underwent colectomy and samples are specific of the pouch or pre-pouch ileum, there are no healthy samples and there is no follow up.
It made very unlikely that a classification of the location could be achieved and the classification of the samples according to the microbiota could not be based on the disease (as all of them had undergone the same procedure with the same disease).
There seems to be gradation on the models on <a href="results.html#fig:morgan-models">4.16</a> that could partially be explained by different degrees of inflammation.</p>
<p>Howell’s dataset is very similar to our BARCELONA dataset but on pediatric cases.
It includes both CD and UC (and controls).
The time with the disease of these patients is much lower, and they have might followed less treatments and probably have not had any surgery</p>
<p>The Cristian’s dataset was also imputed with SILVA but using a different database version.</p>
</div>
<div id="the-methods" class="section level3 hasAnchor" number="5.1.2">
<h3><span class="header-section-number">5.1.2</span> The methods<a href="discussion.html#the-methods" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>As seen on the introduction and on the <a href="online-resources.html#online-resources">appendices</a> many methods are available.
Methods are very different in how they can be used.
Some of them were tested several datasets but their most important validation process undergone is the mathematical validation they have.</p>
<p>Many studies focus on finding some genes or bacteria to answer a narrow question they have in mind, like which bacteria is related to inflammation.
On this thesis the focus was on finding a good representation of the disease that allowed to identify genes and bacteria that were relevant to the disease.
We made the assumptions that the microbiome and the transcriptome are related.
This assumption is backed up by several other previous studies supporting this relationship [ref:needed].</p>
<p>There are many methods for integration developed, each with their shortcomings and strengths.
There are frequent new tools and methods released, and the most up to date list of tools might be added to a <a href="https://github.com/mikelove/awesome-multi-omics" title="Awesome multi-omics list">collaborative list</a> (from which the appendix table of methods has been taken).</p>
<p>With the increase of tools there have been more pressure on comparing different tools.
There have been already some articles comparing different tools, which the previous mentioned list also has.
There are reviews of different tools on the same datasets <span class="citation">[<a href="#ref-cantini2020" role="doc-biblioref">203</a>]</span>, some are more theoretical [ref:needed], others are focused on a different field <span class="citation">[<a href="#ref-cavill2016" role="doc-biblioref">98</a>]</span>.
Recently there has been a recent article focused on integration on IBD <span class="citation">[<a href="#ref-sudhakar2022" role="doc-biblioref">147</a>]</span>.</p>
<p>adonis and globaltest worked well to identify which variables were important.
But it could be that we missed some other important variable.
However, even if we missed it we probably haven’t collected it or it is not available so, no point on trying to find it.</p>
<p>WGCNA did not work well.
It is a tool designed to find common patterns based on correlations requiring homogeneous samples.
Having samples from multiple segments and conditions did not help.
We could have tried to make smaller groups and then compare the modules between them but it probably would not have ended well as the groups would be too small.
Lastly, having microbiome and RNASeq on the same matrix, would probably be hard for the process to find good relationships if we applied the same normalization process to both of them.</p>
<p>MCIA works well as baseline.</p>
<p>STATegRa is not customizable to specific interaction between the datasets.</p>
<p>RGCCA some improvements.</p>
<p><code>BaseSet</code> did not work for its intended usage on this thesis.
It failed because it is computationally expensive to calculate the likelihood of 1500 variables, there are too many combinations.
In addition, the numeric precision of said calculations suffers from the flotating point problems and must be considered carefully.
To support multiplying more than 1000 float numbers a different strategy such as using log values might be better.
We could not come up with a better strategy to find all the combinations needed, perhaps a better method exists that could be used to find which are the terms more influential to the end result.</p>
<p>Some batch effects are avoidable due to the experDesign package.
This might help to improve the quality of the datasets and to expand previously sequenced datasets.</p>
</div>
</div>
<div id="designing-models" class="section level2 hasAnchor" number="5.2">
<h2><span class="header-section-number">5.2</span> Designing models<a href="discussion.html#designing-models" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>Previous publications using RGCCA on IBD focused on validating inflammation markers genes DUOX2 and APOA1 as inflammation predictors (<span class="citation">[<a href="#ref-tang2017" role="doc-biblioref">80</a>]</span>) from previously published articles <span class="citation">[<a href="#ref-haberman2014" role="doc-biblioref">204</a>]</span>.
Some publication tried to summarize the existing relationships on IBD <span class="citation">[<a href="#ref-hasler_uncoupling_2016" role="doc-biblioref">79</a>]</span>.</p>
<p>Modeling the relationships on RGCCA the connections between blocks had to be modified.
Also what belongs to a block should be modified.
The features selected by RGCCA are highly dependent on which connections are made.</p>
<p>If one has preexisting theories about the data, a specific model can be used stating these known or hypothetical relationships.
However, if new relationships are being explored or no prior beliefs on the data are held the models should be created with random links between blocks, and evaluate which model is better.</p>
<p>The connections tested required that all models should be connected and no isolated blocks were left, all blocks should be indirectly connected to other blocks.
This avoids optimizing two different networks of blocks that are not connected between them.
Thus, forcing the model to represent all the information.</p>
<p>Typically blocks are defined</p>
</div>
<div id="evaluating-models" class="section level2 hasAnchor" number="5.3">
<h2><span class="header-section-number">5.3</span> Evaluating models<a href="discussion.html#evaluating-models" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>To evaluate a model RGCCA provides the average variance explained (AVE), inner and outer.
Inner AVE is for how well do all the canonical dimensions correlate with the design of the experiment, so it a measure of how good the model is.
While outer AVE is a measure of how well do the variables of each block correlate with the canonical dimension, so it measures the agreement between the variables and the canonical dimensions.
Depending on the goals of the research one or the other should be used.
If we are more interested on the model of the relationships the inner AVE makes more sense.</p>
<p>Furthermore, to evaluate a design bootstraping can be used to know how well the design does apply to a variety of data.
Another option is to use an external cohort to validate the same model, or using a different method to see if it finds the same relationships or explains the data as accurately.
Of the multiple methods available we used <a href="https://bioconductor.org/packages/omicade4" title="Omicade4 package">MCIA</a> <span class="citation">[<a href="#ref-mengMultivariateApproachIntegration2014" role="doc-biblioref">205</a>]</span>.
Which was compared by looking at the area under the curve for classifying the samples according to their location.</p>
<p>Besides a way to compare methods, these models do need to be evaluated by the insights they provide on the biological system they are being applied to, in our case the Crohn’s disease.
In this article we did not look in depth to the biological relevance of the microorganisms an genes found.</p>
<p>The procedure of separating independent variables in their own block of data and later search the best model that fits the data provides a good strategy that should be consider for integration efforts.</p>
<p>The procedural method of searching a model and testing them is implemented on <a href="https://llrs.github.io/inteRmodel/" title="inteRmodel">inteRmodel</a>.
But the most important thing is to consider which variables are independent of which and if they can be separated into a block for later usage on the modeling.</p>
<p>Transciptome is related to localization of the samples.</p>
<p>Microbiome is more related to the disease and other variables of the patients.</p>
<p>Time is an important variable when modelling the disease, if multiple timepoints are taken they should be taken into account to identify the state of the disease for each patient.</p>
<p><span class="citation">[<a href="#ref-jiang2019" role="doc-biblioref">206</a>]</span> A paper about multi-omics and microbiome…</p>
<p><span class="citation">[<a href="#ref-krassowski2020" role="doc-biblioref">93</a>]</span> advice and recommendations on software engineering and reproducibility practices to share a comprehensive awareness with new researchers in multi-omics for end-to-end workflow.</p>
</div>
<div id="relevance-for-other-research" class="section level2 hasAnchor" number="5.4">
<h2><span class="header-section-number">5.4</span> Relevance for other research<a href="discussion.html#relevance-for-other-research" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>It is hard to provide more information that can be later used by researchers on the wet lab.</p>
<p>Paper about using transcriptomics to infer inflammation without colonoscopy <span class="citation">[<a href="#ref-ungar2022" role="doc-biblioref">207</a>]</span></p>
<p>There is a disconnect between the computational side and the experimental side, driven by the difficulty to think an experiment to test the new information that multi-omic experiments provide.</p>
<p>Many combinations possible that are hard to explore, further research on how to reduce the space of possible combinations of bacteria or evaluate which combinations are more important might be useful on the future.
As this might help focus on the most promising bacteria.</p>
</div>
<div id="implications-of-research" class="section level2 hasAnchor" number="5.5">
<h2><span class="header-section-number">5.5</span> Implications of research<a href="discussion.html#implications-of-research" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>Translational research?
How to apply this results ?</p>
<p>Sequence just the ASV on patients to classify them on disease might be an option that would require further clinical validation.</p>
<p>Selecting which genes are related to the microorganisms is will require also further validation (for which I did not have the creativity to think of).</p>
<p>The several environmental factors highly affect the disease, so analysis or comparisons without taking into account them might provide misleading or false results.</p>
<p>Multi-omics seems to focus on metagenomics, metatranscriptomics and metaproteomics and abandon plain 16S sequencing <span class="citation">[<a href="#ref-zhang2019" role="doc-biblioref">208</a>]</span>.</p>
<!--# From @collij2021 add references/adapt-->
<p>To study complex systems, new technologies like “organ-on-chip” or more specifically “gut-on-chip” are being developed <span class="citation">[<a href="#ref-collij2021" role="doc-biblioref">209</a>]</span>.
By introducing gut microbiota, these system will be of great help in the stud of the interaction of the gut microbiota dn the intestinal epithelium.</p>

</div>
</div>
<h3>References</h3>
<div id="refs" class="references csl-bib-body">
<div id="ref-hasler_uncoupling_2016" class="csl-entry">
79. Häsler R, Sheibani-Tezerji R, Sinha A, Barann M, Rehman A, Esser D, et al. <a href="https://doi.org/10.1136/gutjnl-2016-311651">Uncoupling of mucosal gene regulation, <span class="nocase">mRNA</span> splicing and adherent microbiota signatures in inflammatory bowel disease</a>. Gut. 2017;66:2087–97.
</div>
<div id="ref-tang2017" class="csl-entry">
80. Tang MS, Bowcutt R, Leung JM, Wolff MJ, Gundra UM, Hudesman D, et al. <a href="https://doi.org/10.1097/MIB.0000000000001208">Integrated Analysis of Biopsies from Inflammatory Bowel Disease Patients Identifies SAA1 as a Link Between Mucosal Microbes with TH17 and TH22 Cells</a>. Inflammatory Bowel Diseases. 2017;23:1544–54.
</div>
<div id="ref-krassowski2020" class="csl-entry">
93. Krassowski M, Das V, Sahu SK, Misra BB. <a href="https://doi.org/10.3389/fgene.2020.610798">State of the field in multi-omics research: From computational needs to data mining and sharing</a>. Frontiers in Genetics. 2020;11:1598.
</div>
<div id="ref-cavill2016" class="csl-entry">
98. Cavill R, Jennen D, Kleinjans J, Briedé JJ. <a href="https://doi.org/10.1093/bib/bbv090">Transcriptomic and metabolomic data integration</a>. Briefings in Bioinformatics. 2016;17:891–901.
</div>
<div id="ref-sudhakar2022" class="csl-entry">
147. Sudhakar P, Alsoud D, Wellens J, Verstockt S, Arnauts K, Verstockt B, et al. <a href="https://doi.org/10.1093/ecco-jcc/jjac027">Tailoring multi-omics to inflammatory bowel diseases: All for one and one for all</a>. Journal of Crohn’s and Colitis. 2022;jjac027.
</div>
<div id="ref-nearing2022" class="csl-entry">
196. Nearing JT, Douglas GM, Hayes MG, MacDonald J, Desai DK, Allward N, et al. <a href="https://doi.org/10.1038/s41467-022-28034-z">Microbiome differential abundance methods produce different results across 38 datasets</a>. Nature Communications. 2022;13:342.
</div>
<div id="ref-louca2018" class="csl-entry">
197. Louca S, Doebeli M, Parfrey LW. <a href="https://doi.org/10.1186/s40168-018-0420-9">Correcting for 16S rRNA gene copy numbers in microbiome surveys remains an unsolved problem</a>. Microbiome. 2018;6:41.
</div>
<div id="ref-nadalian2021" class="csl-entry">
198. Nadalian B, Yadegar A, Houri H, Olfatifar M, Shahrokh S, Asadzadeh Aghdaei H, et al. <a href="https://doi.org/10.1111/jgh.15260">Prevalence of the pathobiont adherent-invasive Escherichia coli and inflammatory bowel disease: a systematic review and meta-analysis</a>. Journal of Gastroenterology and Hepatology. 2021;36:852–63.
</div>
<div id="ref-martínez-mira2018" class="csl-entry">
199. Martínez-Mira C, Conesa A, Tarazona S. <a href="https://doi.org/10.1101/421834">MOSim: Multi-Omics Simulation in R</a>. 2018.
</div>
<div id="ref-cantini2021" class="csl-entry">
200. Cantini L, Zakeri P, Hernandez C, Naldi A, Thieffry D, Remy E, et al. <a href="https://doi.org/10.1038/s41467-020-20430-7">Benchmarking joint multi-omics dimensionality reduction approaches for the study of cancer</a>. Nature Communications. 2021;12:124.
</div>
<div id="ref-humanmicrobiomeprojectconsortium2012" class="csl-entry">
201. Human Microbiome Project Consortium BA, Nelson KE, Pop M, Creasy HH, Giglio MG, Huttenhower C, et al. <a href="https://doi.org/10.1038/nature11209">A framework for human microbiome research.</a> Nature. 2012;486:21521.
</div>
<div id="ref-ugidos2020" class="csl-entry">
202. Ugidos M, Tarazona S, Prats-Montalbán JM, Ferrer A, Conesa A. MultiBaC: A strategy to remove batch effects between different omic data types: Statistical Methods in Medical Research. 2020. <a href="https://doi.org/10.1177/0962280220907365">https://doi.org/10.1177/0962280220907365</a>.
</div>
<div id="ref-cantini2020" class="csl-entry">
203. Cantini L, Zakeri P, Hernandez C, Naldi A, Thieffry D, Remy E, et al. <a href="https://doi.org/10.1101/2020.01.14.905760">Benchmarking joint multi-omics dimensionality reduction approaches for cancer study</a>. bioRxiv. 2020;2020.01.14.905760.
</div>
<div id="ref-haberman2014" class="csl-entry">
204. Haberman Y, Tickle TL, Dexheimer PJ, Kim M-O, Tang D, Karns R, et al. <a href="https://doi.org/10.1172/JCI75436">Pediatric Crohn disease patients exhibit specific ileal transcriptome and microbiome signature</a>. The Journal of Clinical Investigation. 2014;124:3617–33.
</div>
<div id="ref-mengMultivariateApproachIntegration2014" class="csl-entry">
205. Meng C, Kuster B, Culhane AC, Gholami AM. <a href="https://doi.org/10.1186/1471-2105-15-162">A multivariate approach to the integration of multi-omics datasets</a>. BMC Bioinformatics. 2014;15:162.
</div>
<div id="ref-jiang2019" class="csl-entry">
206. Jiang D, Armour CR, Hu C, Mei M, Tian C, Sharpton TJ, et al. <a href="https://www.frontiersin.org/article/10.3389/fgene.2019.00995">Microbiome multi-omics network analysis: Statistical considerations, limitations, and opportunities</a>. Frontiers in Genetics. 2019;10.
</div>
<div id="ref-ungar2022" class="csl-entry">
207. Ungar B, Yavzori M, Fudim E, Picard O, Kopylov U, Eliakim R, et al. Host transcriptome signatures in human faecal-washes predict histological remission in patients with IBD. Gut. 2022. <a href="https://doi.org/10.1136/gutjnl-2021-325516">https://doi.org/10.1136/gutjnl-2021-325516</a>.
</div>
<div id="ref-zhang2019" class="csl-entry">
208. Zhang X, Li L, Butcher J, Stintzi A, Figeys D. <a href="https://doi.org/10.1186/s40168-019-0767-6">Advancing functional and translational microbiome research using meta-omics approaches</a>. Microbiome. 2019;7:154.
</div>
<div id="ref-collij2021" class="csl-entry">
209. Collij V, Klaassen MAY, Weersma RK, Vila AV. <a href="https://doi.org/10.1007/s00439-020-02218-3">Gut microbiota in inflammatory bowel diseases: moving from basic science to clinical applications</a>. Human Genetics. 2021;140:703–8.
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="results.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="conclusions.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": {
"href": "https://github.com/llrs/thesis/"
},
"facebook": false,
"twitter": {
"href": "https://twitter.com/intent/tweet?text=Nice%20read%20of%20%40Lluis_Revilla%20thesis"
},
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/llrs/thesis/edit/master/05_discussion.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": "https://github.com/llrs/thesis/blob/master/05_discussion.Rmd",
"text": "View"
},
"download": ["_main.pdf", "_main.epub"],
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": false,
"toc_float": true,
"toc_appendix": true,
"toc_depth": 2,
"scroll_highlight": true
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
