% Options for packages loaded elsewhere
\PassOptionsToPackage{unicode}{hyperref}
\PassOptionsToPackage{hyphens}{url}
\PassOptionsToPackage{dvipsnames,svgnames,x11names}{xcolor}
%
\documentclass[
  12pt,
  a4paper,
  twoside,
  openright]{book}
\usepackage{amsmath,amssymb}
\usepackage{lmodern}
\usepackage{iftex}
\ifPDFTeX
  \usepackage[T1]{fontenc}
  \usepackage[utf8]{inputenc}
  \usepackage{textcomp} % provide euro and other symbols
\else % if luatex or xetex
  \usepackage{unicode-math}
  \defaultfontfeatures{Scale=MatchLowercase}
  \defaultfontfeatures[\rmfamily]{Ligatures=TeX,Scale=1}
\fi
% Use upquote if available, for straight quotes in verbatim environments
\IfFileExists{upquote.sty}{\usepackage{upquote}}{}
\IfFileExists{microtype.sty}{% use microtype if available
  \usepackage[]{microtype}
  \UseMicrotypeSet[protrusion]{basicmath} % disable protrusion for tt fonts
}{}
\makeatletter
\@ifundefined{KOMAClassName}{% if non-KOMA class
  \IfFileExists{parskip.sty}{%
    \usepackage{parskip}
  }{% else
    \setlength{\parindent}{0pt}
    \setlength{\parskip}{6pt plus 2pt minus 1pt}}
}{% if KOMA class
  \KOMAoptions{parskip=half}}
\makeatother
\usepackage{xcolor}
\IfFileExists{xurl.sty}{\usepackage{xurl}}{} % add URL line breaks if available
\IfFileExists{bookmark.sty}{\usepackage{bookmark}}{\usepackage{hyperref}}
\hypersetup{
  pdftitle={Data integration in inflammatory bowel disease},
  pdfauthor={Lluís Revilla Sancho},
  colorlinks=true,
  linkcolor={Maroon},
  filecolor={Maroon},
  citecolor={Blue},
  urlcolor={Blue},
  pdfcreator={LaTeX via pandoc}}
\urlstyle{same} % disable monospaced font for URLs
\usepackage[top=25.4mm,bottom=25.4mm,left=25.4mm,right=25.4mm,bindingoffset=6.4mm,asymmetric]{geometry}
\usepackage{longtable,booktabs,array}
\usepackage{calc} % for calculating minipage widths
% Correct order of tables after \paragraph or \subparagraph
\usepackage{etoolbox}
\makeatletter
\patchcmd\longtable{\par}{\if@noskipsec\mbox{}\fi\par}{}{}
\makeatother
% Allow footnotes in longtable head/foot
\IfFileExists{footnotehyper.sty}{\usepackage{footnotehyper}}{\usepackage{footnote}}
\makesavenoteenv{longtable}
\usepackage{graphicx}
\makeatletter
\def\maxwidth{\ifdim\Gin@nat@width>\linewidth\linewidth\else\Gin@nat@width\fi}
\def\maxheight{\ifdim\Gin@nat@height>\textheight\textheight\else\Gin@nat@height\fi}
\makeatother
% Scale images if necessary, so that they will not overflow the page
% margins by default, and it is still possible to overwrite the defaults
% using explicit options in \includegraphics[width, height, ...]{}
\setkeys{Gin}{width=\maxwidth,height=\maxheight,keepaspectratio}
% Set default figure placement to htbp
\makeatletter
\def\fps@figure{htbp}
\makeatother
\setlength{\emergencystretch}{3em} % prevent overfull lines
\providecommand{\tightlist}{%
  \setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}
\setcounter{secnumdepth}{5}
\newlength{\cslhangindent}
\setlength{\cslhangindent}{1.5em}
\newlength{\csllabelwidth}
\setlength{\csllabelwidth}{3em}
\newlength{\cslentryspacingunit} % times entry-spacing
\setlength{\cslentryspacingunit}{\parskip}
\newenvironment{CSLReferences}[2] % #1 hanging-ident, #2 entry spacing
 {% don't indent paragraphs
  \setlength{\parindent}{0pt}
  % turn on hanging indent if param 1 is 1
  \ifodd #1
  \let\oldpar\par
  \def\par{\hangindent=\cslhangindent\oldpar}
  \fi
  % set entry spacing
  \setlength{\parskip}{#2\cslentryspacingunit}
 }%
 {}
\usepackage{calc}
\newcommand{\CSLBlock}[1]{#1\hfill\break}
\newcommand{\CSLLeftMargin}[1]{\parbox[t]{\csllabelwidth}{#1}}
\newcommand{\CSLRightInline}[1]{\parbox[t]{\linewidth - \csllabelwidth}{#1}\break}
\newcommand{\CSLIndent}[1]{\hspace{\cslhangindent}#1}
% Include other pdf
\usepackage{pdfpages}
% \usepackage{navigator} % To mofidy the outline https://osl.ugr.es/CTAN/macros/generic/navigator/navigator-doc.pdf



% Include the indices of tables and figures
% https://tex.stackexchange.com/a/48512/178206
\usepackage{tocbibind}

% To add figure after the environment not later at the end of the section
% From https://stackoverflow.com/a/33801326/2886003
\usepackage{float}
\usepackage{colortbl}
\let\origfigure\figure
\let\endorigfigure\endfigure
\renewenvironment{figure}[1][2] {
    \expandafter\origfigure\expandafter[!htp]
} {
    \endorigfigure
}

% Page number the same even for first page of chapter
\usepackage[automark,headsepline]{scrlayer-scrpage}% sets page style scrheadings automatically
\clearpairofpagestyles
\ihead{\leftmark}
\ohead*{\pagemark}
\setkomafont{pagenumber}{}% default is \normalfont\normalcolor

% Reduce space between chapter title and top of page
% https://tex.stackexchange.com/a/63393/178206
% Remove chapter number
% https://tex.stackexchange.com/a/519236/178206
\usepackage{microtype} % For lsstyle
\usepackage{titlesec}
% \titleformat{\chapter}[display]{\normalfont\huge\bfseries}{}{20pt}{\Huge}
% \titleformat{\chapter}[display]{\normalfont\huge\bfseries}{\null\hfill \chaptertitlename\ \thechapter}{20pt}{\Huge}
\titleformat{\chapter}[display]{\fontsize{32pt}{48pt}\bfseries\sffamily\filcenter}{
  \fontsize{72pt}{72pt} \thechapter \ChapFrame
} % Content on the chapter title page
{20pt}{\lsstyle}[\thispagestyle{empty} \cleardoublepage]% https://tex.stackexchange.com/a/347162/178206
\titleformat{name=\chapter, numberless}{\normalfont\huge\bfseries\filcenter}{}{20pt}{\Huge}
\titlespacing*{\chapter}{0pt}{100pt}{40pt}

% To handle captions and make them wide even if tables are not
% https://stackoverflow.com/a/46101575/2886003
\usepackage{caption}
% Set in bold the numbering of tables and chapters
\captionsetup{labelfont=bf,width=\textwidth}
% Set table caption below tables, from example code on: https://tex.stackexchange.com/a/508243/178206
% But it doesn't work per design on kable https://github.com/yihui/knitr/issues/1189
% \captionsetup{labelfont=bf,width=\textwidth,positon=below}

% To be able to set table on the center and not overlap much on the right margin
\usepackage{adjustbox}

% Allow figures and tables in a page from https://bookdown.org/yihui/rmarkdown-cookbook/figure-placement.html#adjust-latex-placement-rules
\renewcommand{\topfraction}{.85}
\renewcommand{\bottomfraction}{.7}
\renewcommand{\textfraction}{.15}
\renewcommand{\floatpagefraction}{.66}
\setcounter{topnumber}{3}
\setcounter{bottomnumber}{3}
\setcounter{totalnumber}{4}

% Creating marker on the page margins
% https://tex.stackexchange.com/questions/113323/how-can-one-put-a-marker-to-every-page-in-a-chapter
% Check this answer https://tex.stackexchange.com/questions/262950/modify-chapter-thumb-for-appendix
\usepackage[scale=1,angle=0,opacity=1,contents={}]{background}
\usetikzlibrary{calc}
\usepackage{ifthen}
\usepackage{lipsum}
% auxiliary counter
\newcounter{chapshift}

% the list of colors to be used (add more if needed)
\newcommand\BoxColor{%
  \ifcase\thechapshift blue!30\or red!30\or olive!30\or magenta!30\or teal!30\or lime!30\or orange!30\or violet!30\or brown!30\else yellow!30\fi}

% the main command; the mandatory argument sets the color of the vertical box
\newcommand\ChapFrame{%
  \def\TitleText{\leftmark}%
  \AddEverypageHook{%
    \ifthenelse{\isodd{\value{page}}}
      {\backgroundsetup{
        contents={%
          \begin{tikzpicture}[overlay,remember picture]
          \node[fill=\BoxColor,inner sep=0pt,rectangle,text width=1cm,
            text height=3cm,align=center,anchor=north east]
          at ($ (current page.north east) + (-0cm,- 3*\thechapshift cm) $)
          % {\rotatebox{90}{\parbox{4cm}{%
          %   \centering\textcolor{black}{\scshape\thechapshift}}}};
          {};
          \end{tikzpicture}
        }%
      }
    }
    {\backgroundsetup{
      contents={%
        \begin{tikzpicture}[overlay,remember picture]
        \node[fill=\BoxColor,inner sep=0pt,rectangle,text width=1cm,
          text height=3cm,align=center,anchor=north west]
        at ($ (current page.north west) + (-0cm,-3*\thechapshift cm) $)
        % {\rotatebox{90}{\parbox{4cm}{%
        %   \centering\textcolor{black}{\scshape\thechapshift}}}};
        {};
        \end{tikzpicture}
      }
    }
  }
\BgMaterial}%
\stepcounter{chapshift}
}

% Make empty pages without headers https://tex.stackexchange.com/a/1684/178206
\usepackage{emptypage}
\usepackage{booktabs}
\usepackage{longtable}
\usepackage{array}
\usepackage{multirow}
\usepackage{wrapfig}
\usepackage{float}
\usepackage{colortbl}
\usepackage{pdflscape}
\usepackage{tabu}
\usepackage{threeparttable}
\usepackage{threeparttablex}
\usepackage[normalem]{ulem}
\usepackage{makecell}
\usepackage{xcolor}
\ifLuaTeX
  \usepackage{selnolig}  % disable illegal ligatures
\fi

\title{Data integration in inflammatory bowel disease}
\author{Lluís Revilla Sancho}
\date{2022-04-25}

\begin{document}
\maketitle

{
\hypersetup{linkcolor=}
\setcounter{tocdepth}{2}
\tableofcontents
}
\listoffigures
\listoftables
\hypertarget{preface}{%
\chapter*{Preface}\label{preface}}
\addcontentsline{toc}{chapter}{Preface}

\includegraphics[width=1.66667in,height=\textheight]{images/ub_logo.png} \includegraphics[width=1.66667in,height=\textheight]{images/idibaps_logo.png} \includegraphics[width=1.66667in,height=0.39583in]{images/logo_ciber.png}

The main topic of the thesis is \href{https://en.wikipedia.org/wiki/Data_integration}{data integration} applied in the \href{https://en.wikipedia.org/wiki/Inflammatory_bowel_disease}{inflammatory bowel disease (IBD)} research.
This disease is complex, for instance it is not known if the cause behind Crohn's disease and ulcerative colitis is the same.
There are hypothesis pointing that the \href{https://en.wikipedia.org/wiki/Microbiota}{microbiome} is a major factor in the disease, which together with an aberrant immune response is the dominant theory.
In order to find robust relationships between the microbiome and the immune system it is important to consider all the relevant variables that influence a disease.
On this thesis we seek these relationships using data from different sequencing technologies and the observed or reported phenotype of the patients.

The thesis was carried out on the \href{http://www.idibaps.org/}{Institut d'Investigacions Biomèdiques August Pi i Sunyer (IDIBAPS)} research institute and funded by \href{https://www.ciberehd.org/}{Centro de Investigación Biomédica en Red (CIBER)}.
The thesis was done on \href{https://www.ibd-bcn.org/}{the IBD unit} which is a translational team of biologist, microbiologists, veterinaries, bioinformaticians, doctors and nurses (at \href{https://www.clinicbarcelona.org/en}{Hospital Clínic}) in a multidisciplinary team.
The leading doctor of the unit was \href{https://orcid.org/0000-0002-4971-6902}{Julian Panés} whose interest on the disease made possible this thesis.

The thesis is on the \href{http://www.ub.edu/doctorat_biomedicina/eng/index.htm}{doctoral programme in biomedicine} of the \href{https://www.ub.edu/web/portal/en/}{University of Barcelona (UB)}.
My thesis directors' were \href{https://orcid.org/0000-0001-7613-3908}{Juanjo Lozano} and \href{https://orcid.org/0000-0003-4572-2907}{Azucena Salas}, my boss, who helped as bioinformatician and disease expert respectively.
They provided advice and guidance on how to analyse the data and where to focus on the different experiments/analysis.

This thesis, available on \url{https://thesis.llrs.dev} is licensed under the \href{https://creativecommons.org/licenses/by/4.0/}{Creative Commons Attribution 4.0 International License (CC-BY)}.\\
\includegraphics{images/by.png}

\hypertarget{abstracts}{%
\chapter*{Abstracts}\label{abstracts}}
\addcontentsline{toc}{chapter}{Abstracts}

\hypertarget{english}{%
\section*{English}\label{english}}
\addcontentsline{toc}{section}{English}

\textbf{Introduction}: Inflammatory bowel disease is a complex intestinal disease with several genetic and environmental factors that can influence its course.
The ethiology and pathophysiology of the disease is not fully understood.
There is some evidence that microbiome can play a role.
Finding relationships between microbiome and host's mucosa could help advance prevention, diagnosis or treatment.

\textbf{Methods}: We based our analysis on intestinal bacterial 16S rRNA and human transcriptome data from biopsies from multiple timepoints and intestine segments.
We extended regularized generalized canonical correlation analysis to find models that are coherent with previous knowledge on the disease taking into account the samples' information.
Multiple inflammatory bowel disease datasets on different treatments and conditions were analysed and the models defining those dataset were compared.
The results were compared with multiple co-inertia analysis.

\textbf{Results}: Splitting sample variables into different blocks results in models of these relationships that show differences on the genes and microorganisms selected.
The models generated using our new method inteRmodel outperformed multiple co-inertia analysis to classify the samples according to their location.
Despite being used on datasets of different sources the resulting models show similar relationships between variables.

\textbf{Discussion}:
Comparing multiple models helps find out the relationships within datasets.
Our method finds how strong are the relationships between the microbiome, transcriptome and environmental variables.
On different datasets genes selected were common.
This approach is robust and flexible to different datasets and settings.

\textbf{Conclusion}: With inteRmodel we found that the microbiome relates more closely to the sample location than to disease, but the transcriptome is highly related to the location of the sample on the intestine.
There is a common transcriptome between datasets while microorganisms depend of the dataset.
We can improve sample classification by taking into account both bacterial 16S rRNA and host transcriptome.

\hypertarget{spanish}{%
\section*{Spanish}\label{spanish}}
\addcontentsline{toc}{section}{Spanish}

\textbf{Introducción}: La enfermedad inflamatoria intestinal es una enfermedad intestinal compleja con factores genéticos y ambientales que pueden influir en su curso.
La etiología y la fisiopatología de la enfermedad no se conocen por completo.
Existen evidencias que el microbioma puede desempeñar un papel relevante.
Encontrar relaciones entre el microbioma y la mucosa del huésped podría ayudar a avanzar en la prevención, el diagnóstico o el tratamiento.

\textbf{Métodos}: Basamos nuestro análisis en el ARNr 16S bacteriano intestinal y en datos de transcriptomas humanos de biopsias de múltiples puntos temporales y segmentos intestinales.
Extendimos el análisis de correlación canónica generalizada regularizado para encontrar modelos coherentes con el conocimiento previo sobre la enfermedad teniendo en cuenta la información de las muestras.
Se analizaron múltiples conjuntos de datos de enfermedad inflamatoria intestinal en diferentes tratamientos y condiciones y se compararon los modelos que definen esos conjuntos de datos.
Los resultados se compararon con análisis de coinercia múltiple.

\textbf{Resultados}: Dividir las variables de la muestra en diferentes bloques resulta en modelos de estas relaciones que muestran diferencias en los genes y microorganismos seleccionados.
Los modelos generados con nuestro nuevo método, interRmodel, superaron el análisis de múltiples coinercias para clasificar las muestras según su ubicación.
A pesar de ser utilizados en conjuntos de datos de diferentes fuentes, los modelos resultantes muestran unas relaciones similares entre las variables.

\textbf{Discusión}: La comparación de varios modelos ayuda a descubrir las relaciones dentro de los conjuntos de datos.
Nuestro método encuentra cuán fuertes son las relaciones entre el microbioma, el transcriptoma y las variables ambientales.
En diferentes conjuntos de datos, los genes seleccionados eran comunes.
Este enfoque es robusto y flexible para diferentes conjuntos de datos y configuraciones.

\textbf{Conclusión}: Con inteRmodel descubrimos que el microbioma se relaciona más estrechamente con la ubicación de la muestra que con la enfermedad, pero el transcriptoma está muy relacionado con la ubicación de la muestra en el intestino.
Hay un transcriptoma común entre los conjuntos de datos, mientras que los microorganismos dependen del conjunto de datos.
Podemos mejorar la clasificación de las muestras teniendo en cuenta tanto el ARNr 16S bacteriano como el transcriptoma del huésped.

\hypertarget{catalan}{%
\section*{Catalan}\label{catalan}}
\addcontentsline{toc}{section}{Catalan}

\textbf{Introducció}: La malaltia inflamatòria intestinal és una malaltia intestinal complexa amb diversos factors genètics i ambientals que poden influir en el seu curs.
L'etiologia i fisiopatologia de la malaltia no es conèix del tot.
Hi ha evidències que el microbioma pot tenir un paper rellevant.
Trobar relacions entre el microbioma i la mucosa de l'hoste podria ajudar a avançar en la prevenció, el diagnòstic o el tractament.

\textbf{Mètodes}: Vam basar la nostra anàlisi en dades d'ARNr 16S bacteriana intestinal i de transcriptoma humà de biòpsies de múltiples punts de temps i segments intestinals.
Hem ampliat l'anàlisi de correlació canònica generalitzada regularitzada per trobar models coherents amb el coneixement previ sobre la malaltia tenint en compte la informació de les mostres.
Es van analitzar diversos conjunts de dades de malaltia inflamatòria intestinal sobre diferents tractaments i condicions i es van comparar els models que defineixen aquest conjunt de dades.
Els resultats es van comparar amb l'anàlisi de coinèrcia múltiple.

\textbf{Resultats}: Dividir les variables de la mostra en diferents blocs dóna com a resultat models d'aquestes relacions que mostren diferències en els gens i els microorganismes seleccionats.
Els models generats mitjançant el nostre nou mètode intermodel van superar l'anàlisi de coinèrcia múltiple per classificar les mostres segons la seva ubicació.
Tot i utilitzar-se en conjunts de dades de diferents fonts, els models resultants mostren relacions similars entre variables.

\textbf{Discussió}: La comparació de diversos models ajuda a esbrinar les relacions dins dels conjunts de dades.
El nostre mètode troba com de fortes són les relacions entre el microbioma, el transcriptoma i les variables ambientals.
En diferents conjunts de dades, els gens seleccionats eren comuns.
Aquest enfocament és robust i flexible per a diferents conjunts de dades i configuracions.

\textbf{Conclusió}: Amb inteRmodel vam trobar que el microbioma es relaciona més estretament amb la ubicació de la mostra que amb la malaltia, però el transcriptoma està molt relacionat amb la ubicació de la mostra a l'intestí.
Hi ha un transcriptoma comú entre conjunts de dades, mentre que els microorganismes depenen del conjunt de dades.
Podem millorar la classificació de les mostres tenint en compte tant l'ARNr 16S bacterià com el transcriptoma hoste.

\hypertarget{introduction}{%
\chapter{Introduction}\label{introduction}}

\href{https://en.wikipedia.org/wiki/Inflammatory_bowel_disease}{Inflammatory bowel disease} (IBD) involves \href{https://en.wikipedia.org/wiki/Crohn\%27s_disease}{Crohn's disease} (CD) and \href{https://en.wikipedia.org/wiki/Ulcerative_colitis}{ulcerative colitis} (UC).
It generally affects the terminal ileum and the colon but it can involve any segment of the gastrointestinal tract.
UC is a recurrent, chronic and continuous inflammation of the colon and rectum while the CD is not a continuous inflammation and affects the whole gastrointestinal tract causing transmural inflammation.

IBD etiology is unknown.
However, once it has initiated the most prevalent hypothesis of its chronicity suggests an aberrant immunological response to antigens of the commensal microbiome.

To diagnose IBD doctors use endoscopy and/or magnetic resonance imaging and histologies.

Treatments provided for IBD include, noninflammatory drugs, suppressors and biologics, i.e, anti TNF- anti-IL2, 23, anti-integrin \(\alpha4\beta7\).
The therapeutic options can induce remission in some patients, but they often need continuous treatment to avoid recurrence.
Nevertheless, many patients are refractory or intolerant to those therapies and need to undergo surgery or other strategies like dietetic and psychological support {[}\protect\hyperlink{ref-raine2021}{1}{]} .

\hypertarget{IBD}{%
\section{Inflammatory bowel disease}\label{IBD}}

IBD includes the CD and UC which are characterized by alternating periods of remission and clinical relapse that mainly affect the gastrointestinal tract.
CD is a progressive relapsing disease that can affect all the gastrointestinal tract but shows mostly on both terminal ileum and colon with a discontinuous inflammation.
UC is a colonic relapsing disease characterized by continuous inflammation of the colon.
Both of them have different risk factors, clinical, endoscopic an histological characteristics (see sections \ref{CD} and \ref{UC} ).

Around 3.5 million individuals have IBD in Europe and North America combined {[}\protect\hyperlink{ref-jairath2020}{2}{]}.
IBD is more commonly found in industrialized and developed regions, suggesting that environmental factors might greatly influence IBD occurrence.
In addition, the incidence of IBD is increasing in areas, such as Asia or Eastern Europe, where the number of cases was relatively low hitherto {[}\protect\hyperlink{ref-burisch2015}{3}{]}.

The dysregulation of the inflammatory response observed in IBD requires interplay between host genetic factors and the intestinal microbiome.
Several studies support the concept that IBD arise from an exacerbate immune response against commensal gut microorganisms.
Nonetheless, the disease could result from an imbalanced microbial composition leading to generalized or localized dysbiosis\footnote{A signature is usually a group of features that describe/are representative of a cell line or a process or a stage.}.

The role of the gut microbiome in IBD is an active ongoing field of research.
Several authors are currently studying the alterations reported in IBD of the intestinal microbiome.
However, it is still unclear the cause-effect relation between dysbiosis and IBD.
Partly due to the multiple variables already identified that have been linked to IBD; for instance, age, diet, usage of antibiotic, tobacco, and socioeconomic status {[}\protect\hyperlink{ref-humanmicrobiomeprojectconsortium2012}{4}, \protect\hyperlink{ref-shaw2016}{5}{]}.

The relationship between host and microbiome has been proposed to play a fundamental role in maintaining disease.
For instance, some \emph{Proteobacteria} species which have adherent and invasive properties might exploit host defenses and promote a proinflammatory environment, altering the intestinal microbiota in favor of dysbiosis {[}\protect\hyperlink{ref-mukhopadhya2012}{6}{]}.

The epithelium is often damaged and might present ulcers or other inflammation symptoms.
A segment of the gastrointestinal tract might recover if the patient receives treatment or due the natural cycles of the disease.
But once a segment is affected by the disease it can be considered as involved, as some damage remains even if the tissues is no longer inflammed.

\hypertarget{etiology-and-pathogenesis}{%
\subsection{Etiology and pathogenesis}\label{etiology-and-pathogenesis}}

Several mechanisms have been proposed to drive IBD pathogenesis {[}\protect\hyperlink{ref-zhang2014}{7}, \protect\hyperlink{ref-hugot2004}{8}{]}.
Some of them are based on a relationship between the immune system and the microbiome {[}\protect\hyperlink{ref-silva2016}{9}, \protect\hyperlink{ref-demattos2015}{10}{]}.
It is also unclear if CD and the UC share the same origin considering their different symptoms.

There is also evidence of some genetic component on the onset of the disease, specially if the disease appears very early (less than 2 years old patients) {[}\protect\hyperlink{ref-mcgovern2015}{11}, \protect\hyperlink{ref-satsangi2006}{12}{]}.
Disease can be classified based on age at onset as very early, early or adult on-set disease {[}\protect\hyperlink{ref-satsangi2006}{12}{]}.
Genome-wide association studies (GWAS) have linked IBD to over 100 genetic loci, including a \emph{NOD2} gene, but so far there is not any known mechanism how polymorfism on this genes are driving the disease {[}\protect\hyperlink{ref-horowitz2021}{13}{]}.
On early pedriatic and adult disease the genetic component is lower than on very early on set and it is thought that the environmental factors are the main cause of the disease at those ages.

On the following sections we will explore the role of several of the possible factors involved on the pahtogenesis, starting with the genetics.

\hypertarget{genetics}{%
\subsubsection{Genetics}\label{genetics}}

IBD is not an heritable disease, except for very early onset IBD, but it has some genetic influence that predisposes people to have it.

This has lead to look for genetic factors in IBD both on general population and on the early cases.
Genome wide association studies (GWAS) are one of the most common genetic studies performed, together with methylation studies.
To discover through linkage desequilibrium genetic variations linked to phenotypes and regulatory transcription changes, respectively.

With GWAS several allels on protein coding loci have been found, rising to around 300 genetic variants {[}\protect\hyperlink{ref-kumar2019}{14}{]}.
Particularly, the \emph{NOD2} gene is highly relevant for the disease on European patients, as it is a risk alleles for CD loci but show significant protective effects in UC {[}\protect\hyperlink{ref-jostins2012}{15}, \protect\hyperlink{ref-momozawa2018}{16}{]}.
The mechanism of how this gene protects from UC has not been confirmed yet {[}\protect\hyperlink{ref-horowitz2021}{13}{]}.

Many of the relevant genetic loci related to IBD are not on protein coding fragments of the genome.
Recently expression quantitative trait loci (eQTL) particularly showed {[}\protect\hyperlink{ref-mcgovern2015}{11}{]} that locis are on enhancers or promoters like e.g.
H3K27Ac or promoter e.g.
H3K4me1 marks as found by chromatin immunoprecipitation sequencing (\href{https://en.wikipedia.org/wiki/ChIP_sequencing}{ChIP-Seq}).

\hypertarget{microbiome}{%
\subsubsection{Microbiome}\label{microbiome}}

The human intestine is a large reservoir of co-existing microorganisms (bacteria, fungi, viruses, and unicellular eukaryotes) .
This microbiome community exerts different functions in the human body influencing nutrients' metabolism, the maturation of the immune system while suppressing the growth of harmful microorganisms' {[}\protect\hyperlink{ref-khanna2014}{17}{]}.

The role of the gut microbiota has been proposed to play a role in IBD pathogenesis.
IBD has been characterized by a breakdown in the balance between beneficial and harmful bacteria that are present in the human gut compared to healthy individuals {[}\protect\hyperlink{ref-swidsinski2002}{18}, \protect\hyperlink{ref-tamboli2004}{19}{]}.

Indeed, many studies show that patients with IBD have less biodiversity.
Biodiversity is measured on \(\alpha\) (alfa) and \(\beta\) (beta) diversity.
\(\alpha\)-diversity is a measure of the species present on a single sample and its abundance while \(\beta\)-diversity compares the diversity between samples.
There are some reports of taxonomic changes and increase on \emph{Enterobacteriaceae sp}, \emph{Escherichia coli} (specially the invasive strain) at the mucosal layer of IBD patients {[}\protect\hyperlink{ref-ott2004}{20}{]}.
At the same time there is often a reduction of protective species like \emph{Bifidobacterium}, \emph{Lactobacillus} and \emph{Faecalibacterium} which might be able to protect individuals from mucosal inflammation via several mechanism such as a downregulation of proinflammatory cytokines or the stimulation of IL-10 and antiinflammatory cytokines {[}\protect\hyperlink{ref-ott2004}{20}{]}.
Specially \emph{Faecalibacterium prausnitzii} is one microorganism of interest {[}\protect\hyperlink{ref-kostic2014}{21}, \protect\hyperlink{ref-sender2016}{22}{]}.

In fact, it has been recently proposed that several unique microbial species can distinguish healthy controls from UC and CD patients {[}\protect\hyperlink{ref-sankarasubramanian2020}{23}, \protect\hyperlink{ref-lopez-siles2014}{24}{]}.

\begin{figure}
\includegraphics[width=1\linewidth]{images/tesis_AM_Figure2} \caption[The microbial composition in the gut.]{The microbial composition in the gut. On the left healthy gut is represented as having a high microbiome diversity and no damage on the epithelial barrier. On the right the IBD gut were microbiome diversity is lower and some bacteria is in physical contact with the damaged epithelium}\label{fig:composition}
\end{figure}

One of the proposed mechanism of crosstalk between bacteria and host is through bacterial metabolites.
They interact with the cells and modulate the state of the intestine.
One example of such metabolite is butyrate which has been linked to microorganisms presents on healthy intestines and shown to interact with intestine cells and help regulate some genes {[}\protect\hyperlink{ref-ferrer-picuxf3n2020}{25}{]}.

As previously mentioned, adherent invasive \emph{Escherichia coli}, a proteobacteria specie, has been associated with IBD {[}\protect\hyperlink{ref-darfeuille-michaud1998}{26}{]}.
Adherent invasive strains are mainly found in ileal and colonic samples of CD patients and their presence in UC is less clear.
These adherent invasive cells enter through the epithelium of the more permeable cells and live on their cytosol.

The metabolic cocktail composed of soluble factors secreted by life probiotic bacteria, living microorganisms which, when administered in adequate amounts, confer health benefits on the host {[}\protect\hyperlink{ref-tsilingiri2012}{27}--\protect\hyperlink{ref-morelli2012}{31}{]} or any bacterial-released molecule capable of providing health benefits through a direct or indirect mechanism, has been collectively known as postbiotics since 2012 {[}\protect\hyperlink{ref-tsilingiri2012}{27}{]}.

\hypertarget{immune-response}{%
\subsubsection{Immune response}\label{immune-response}}

As explained previously the immune system plays a role in IBD pathogenesis and pathophysiology.

Loss of tolerance to commensal bacteria has been suggested as the underlying mechanism triggering the inflammation on the intestine.
The immune response involves many different cells lines and regions, which are important to know how they organize for a better understanding of the disease.

\begin{figure}
\includegraphics[width=1\linewidth]{images/tesis_AC_FigureX} \caption{The intestinal epithelial barrier.}\label{fig:barrier}
\end{figure}

From the luminal side of the intestine, the first layer is the mucosa (See sections \ref{fig:barrier} or \ref{fig:composition}).
In the colon the mucus is organized in two layers: the inner layer, a firm mucus layer; and the outer, loose mucus layer {[}\protect\hyperlink{ref-okumura2017}{32}{]}.
The intestinal epithelium is a single layer of cells organized into crypts and villi (and circular folds on the large intestine) that carries out a diverse array of functions besides digestion performed by specialized cell lineages.

Immune response in the intestinal mucous is mainly excreted by the gut associated lymphoid tissue {[}\protect\hyperlink{ref-faria2012}{33}{]}.
Genetically predisposed patients when exposed to certain environmental factors activate immune responses against microbials or self-antigens which in turn, may impair the mucosal barrier of the intestinal mucosa, the first physical barrier on the mucosal surface.

Both the adaptive and the innate immune cells are present in the intestine, right below the epithelium.
In IBD due to antigen translocation into the lamina propria, the immune response leads the adaptive cells to generate immune response to harmless components of the intestinal microbiota.
This initial response induces a local increase in the production of pro-inflammatory cytokines and mediators which damages the mucosa.
Therefore, the loss of integrity on this barrier enables the intestinal luminal bacteria to access the intestinal epithelium and to interact with the immune system underneath it more directly {[}\protect\hyperlink{ref-hisamatsu2013}{34}{]}.

The intestinal epithelium is another line of defense against bacterial invasion.
Intestinal epithelial cells play a key role in controlling the integrity of the physical barrier to the intestinal microorganisms {[}\protect\hyperlink{ref-hisamatsu2013}{34}{]} not only physically but also secreting antimicrobial peptides and defensins, both of which are altered in IBD patients {[}\protect\hyperlink{ref-michielan2015}{35}{]}.
The intestinal epithelium also plays a key role on the intake and diffusion of metabolites from the intestinal lumen to the lamina propia.

Damaging or increasing the permeability of the intestinal epithelium results on a response from the immune system.
Detecting signals of any foreign particle can also trigger the immune system.
On the intestine this starts with the identification of these signals by intestinal epithelial cells have pattern recognition receptors.
There are two main pattern recognition receptors: toll-like receptors (TLR), which are present on the surface, and nucleotide-binding oligomerization domain-like receptors (NOD-like receptors), present on the cytoplasm of the cells.
These receptors upon recognition of pathogen associated molecular patterns (PAMPs) start an amplifying signaling producing chemokines and cytokines which activates the transcription and translation of pro-inflammatory mediators to ensure an effective immune response.
Initially the innate response is triggered but the cells also increase the antigen presentation to T cells and thus activate the immune adaptive response {[}\protect\hyperlink{ref-mayorgas2021a}{36}{]}.

Other cell types, such as monocytes, macrophages and dendritic cells also present the pattern recognition receptors.
From those, macrophages and dendritic cells are antigen presenting cells too and secrete several cytokines to activate other immune cells.
Usually CD patients express higher amounts of TLR than healthy individuals, which might trigger a stronger immune response.
This response is driven by CD4\textsuperscript{+} T cells proliferation in secondary lymphoid tissues to T helpers in the presence of the antigens and cytokines nearby.

T helpers (T\textsubscript{h}) differentiate depending of the cytokines at which they are exposed.
Th type 1 are driven by exposure to IL-12 secreted primarily by dentritic cells.
T\textsubscript{h}2 are driven by cytokines secreted by macrophages.
The imbalance between T\textsubscript{h}1 and T\textsubscript{h}2-promoting cytokines determines the intensity and duration of the inflammatory response in experimental colitis {[}\protect\hyperlink{ref-neurath1996}{37}{]}.

T\textsubscript{h}17-promoting cytokines are less well characterized in human.
T\textsubscript{reg} cells differentiate after exposure to cytokines IL-10, IFN-\(\gamma\) and TGF-\(\beta\).
Overall the presence of certain cytokines and the response to self-antigens are factors leads to an inflammation and damage that is related to the onset and establishment of IBD

On these kind of diseases autologous hematopoietic stem cell transplantation has shown some benefits in IBD {[}\protect\hyperlink{ref-corraliza}{38}{]}.
The benefit of hematopoietic stem cell transplant (HSCT) in autoimmunity is thought to originate from the depletion of auto-reactive cells regardless of their specificity.
However, due to its associated risk this therapy is only given when patients are refractory to all available therapeutic options.

\hypertarget{environmental}{%
\subsubsection{Environmental}\label{environmental}}

Chronic inflammatory disorders and neoplasms have become the main cause of morbidity and mortality in the countries with high standards of personal cleanliness.
A decrease in human exposure to microbes or hygene which might affect the proper maturation of the immune system so that it provides less immune response or exacerbated response towards ``friendly'' microbes {[}\protect\hyperlink{ref-strachan1989}{39}, \protect\hyperlink{ref-scudellari2017}{40}{]}.

From other environmental factors related to IBD such as tobacco, diet, certain drugs and stress; tobacco is the most influential environmental factor.
Surprisingly, it has an opposite effect on UC and CD: in CD tobacco is a risk factor that increases the risk of relapse and/or surgical intervention.
In UC, it has been observed that smoking cessation worsens the disease {[}\protect\hyperlink{ref-thomas1998}{41}{]}.

Pharmacological treatments such as oral contraceptives, non steroidal anti-inflammatory drugs are also related to develop or relapse the disease {[}\protect\hyperlink{ref-cornish2008}{42}, \protect\hyperlink{ref-kaufmann1987}{43}{]}.

The psychological welfare of people also plays an important role in the disease progression, stress, anxiety and depression might be important in relapse and deterioration of the disease {[}\protect\hyperlink{ref-bitton2008}{44}{]}.
Other environmental factors have been linked to IBD but without enough evidence to support a causative effect in the development of the disease.

\hypertarget{CD}{%
\subsection{CD physiology}\label{CD}}

As previously introduced, CD is a chronic inflammatory disorder characterized by a discontinuous inflammation of the gastrointestinal tract.
Inflammation on the gastrointestinal tract is transmural and can affect from the mouth to the anus, but mainly it manifests on the ileum and colon {[}\protect\hyperlink{ref-demattos2015}{10}{]}.
It is frequently associated with extraintestinal manifestation and/or concomitant immune-mediated diseases.

The disease itself manifest an heterogeneous symptoms that can involve, diarrhea, weight loss, abdominal pain, fever, anorexia and malaise.
Other less frequent co-occurring manifestations are arthritis, primary sclerosing cholangitis, skin disorders venous or arterial thromboembolism and/or pulmonary involvement {[}\protect\hyperlink{ref-baumgart2012}{45}{]} .
These symptoms make it hard to correctly diagnose the disease by non-specialists, in addition there is not a non invasive easy procedure to diagnose it.
All these can lead to delays on correct diagnosis of the disease.

The detection of parasites or bacteria, such as \emph{Clostridium difficile}, have been associated with CD {[}\protect\hyperlink{ref-khanna2017}{46}{]}.
The detection of fecal calprotectin, is generally a good marker of endoscopic activity with sensitivity above 70\% and specificity above 80\% {[}\protect\hyperlink{ref-guardiola2018}{47}, \protect\hyperlink{ref-sands2015}{48}{]}.

Usually the best diagnosis method is to perform a colonoscopy, whether there is inflammation on the gastrointestinal tract on discontinuous regions then is CD.
This inflammation could also present ulceration with rectal sparing and histological lesions which also help to diagnose the patients {[}\protect\hyperlink{ref-corralizamuxe1rquez2019}{49}{]}.

Usually on CD a granulome, that is a region with big multinucleous cells, can appear on any intestinal layer.
In addition to the inflamed location(s), mosaic zones (patches of inflamed and non-inflamed areas) are more characteristic of CD {[}\protect\hyperlink{ref-bassolasmolina2018}{50}{]}.

The Montreal classification aims to classify patients according to their age of disease onset, standardized anatomical disease location an disease behavior.
This classification assumes that the location of CD remains stable over time after diagnosis but behavioral phenotypes change.
Other scores consider area affected:

\begin{itemize}
\tightlist
\item
  Montreal classification allows for early onset of disease to be categorized separately those with age of diagnosis at 16 years or younger, diagnosis at 17--40 years and \textgreater40 years, respectively {[}\protect\hyperlink{ref-satsangi2006}{12}{]}.
\item
  SES-CD: simple endoscopic score for CD {[}\protect\hyperlink{ref-daperno2004}{51}{]}. Score based on size of ulcers, ulcerate surface percentage, affected surface and presence of narrowing on the bowel.
\item
  CDAI: CD Activity Index takes into account weight, ideal body weight, sex, and events on the last week such as liquid stools, abdominal pain, general well-being and if anti-diarrhea drug usage, as well as knowing if there are fistulas, fever, and other complications {[}\protect\hyperlink{ref-best1976}{52}{]}
\end{itemize}

To some extent, there is a disassociation between clinical symptoms and the endoscopy finding.
Often patients report feeling better despite lack of muscular healing {[}\protect\hyperlink{ref-bhattacharya2016}{53}{]}.
To overcome this disassociation and be able to compare the well-being of patients several scores and thresholds are used on research and by physicians that will be described later.

In the early stages of the disease the relapsing and remitting course is more frequent.
Often relapses are accompanied by clinical symptoms, and few have prolonged clinical remission (without treatment) {[}\protect\hyperlink{ref-peyrin-biroulet2010}{54}{]}.
When there is clinical remission, there can still remain some other lesions and often subclinical inflammation persists.
Frequently the damage caused by the disease evolves to fibrostenotic stricture or penetrating lesions (fistula and abscess).

Damage of the disease might not be apparent to patients and might be only seen several years later than the first detection {[}\protect\hyperlink{ref-bhattacharya2016}{53}{]}.
Mucosal healing is a first step towards the healing of deeper layers of the inflamed bowel wall on the CD.

Patients might progress from an inflammatory phenotype to a stricturing or penetrating one {[}\protect\hyperlink{ref-satsangi2006}{12}{]}.
Stricturing is a narrowing of a part of the intestine often because of scar tissue and fibrosis in its wall.
Penetrating is when the epithelium has some holes or tubes.
If these tubes result in an abnormal connection between two body parts it is a fistula, it might also result in an abcess, a collections of pus, often developed in the abdomen, pelvis, or around the anal area.

\hypertarget{UC}{%
\subsection{UC physiology}\label{UC}}

As previously introduced, UC is a chronic inflammatory disorder characterized by a continuous inflammation of the colon.
Depending on the inflamed segments of the intestine it is classified in several phenotypes.

Around a third of the patients with UC suffer proctitis, the inflammation of the rectum.
If the segments from rectum to the sigmoid colon are affected is a distal colitis, if it affects the left colon then it becomes a left colitis.
If the inflammation continues to the descending colon it is then an extensive colitis until it affects the whole colon when it becomes a pancolitis.
The extension of UC is inversely related to the frequency.
However, the extension and severity of the disease correlates: the prognosis is worse the more extended it is {[}\protect\hyperlink{ref-etchevers2009}{55}{]}.
In addition, the damage usually consists in many neutrophil in the lumen crypt {[}\protect\hyperlink{ref-bassolasmolina2018}{50}{]}.

The goal of the clinical care is to recover.
As a first step, the symptoms of IBD have to lessen to the point that they are mostly absent, gone, or barely noticeable, this is known as clinical remission.
However, this is not enough as the mucosa might be still inflamed and thus the reconstitution of the structure and function of the intestinal mucosa is not complete.
Other lesions, might aid to the progression to other phenotypes such as fibrostenotic stricture or penetrating lesions or primary sclerosing cholangitis {[}\protect\hyperlink{ref-boonstra2012}{56}{]}.

To prevent and avoid further damage several procedures are followed:

When there is dysplasia, an abnormal development of cells within tissues or organs (which is considered a precedence before colorectal cancer growth {[}\protect\hyperlink{ref-mark-christensen2018}{57}{]}), or the damage on the colon has been too big a surgical procedure to remove part or all of the colon must be done.\\
Patients that undergo a colectomy need to have their bowel reconnected with a procedure called ileoanal anastomosis (also know as J-pouch by the shape it takes) surgery.
Often the lining of the pouch created during surgery becomes inflamed on what is known as pouchitis {[}\protect\hyperlink{ref-schieffer2016}{58}{]}.

Many scores have been proposed for several purposes, from quality of life to disease severity or patient status.
Among the scores most used are the following:

\begin{itemize}
\item
  Mayo: A score designed to be simple to calculate based on stool frequency, bleeding, mucosal appearance at endoscopy and physicians assessment {[}\protect\hyperlink{ref-schroeder1987}{59}{]}.
\item
  IBDQ: A 32 questionnaire used to assess the quality of life grouped into four categories: bowel, systemic, social and emotional {[}\protect\hyperlink{ref-irvine1999}{60}{]}.
\item
  UCEDIS: An endoscopic score based on vascular pattern, internal bleeding and erosion and ulcers {[}\protect\hyperlink{ref-travis2012}{61}{]}.
\end{itemize}

Other measured parameters include, weight, effective weight, fecal calprotectin, C reactive protein and hemoglobine.

\hypertarget{treatment}{%
\subsection{Treatment}\label{treatment}}

Current treatment attempt to induce and keep the remission of patients and reduce secondary effects of the disease instead of revert the pathogenic mechanisms.
As standard of care corticosteroids, aminosalicylates and immunosuppressor and some other drugs like antibiotics or metronidazol are util in some cases.

Acid 5-aminosalicylic (5-ASA or mesalazina, pentasa) can be given in a topic way (either liquid, enemas, or suppository) or in oral form (pills or dilutions).
In CU it helps in the clinic remission but it does not always mean that there is remission (is twice much likely than placebo to reach remission) {[}\protect\hyperlink{ref-travis2006}{62}{]}.
On CD the effects are not so stark and generally it does not produce changes on the disease {[}\protect\hyperlink{ref-akobeng2016}{63}{]}.

Antibiotics, such as metronidazol and ciprofloxacina, are effective to deal with secondary effects of IBD such as abscess and bacterian overgrowth in CD {[}\protect\hyperlink{ref-feller2010}{64}, \protect\hyperlink{ref-prantera2009}{65}{]}, but they do not seem effective on UC {[}\protect\hyperlink{ref-prantera2009}{65}{]}.

Corticosteroids can be taken orally, such as prednisolona, prednisona and Budesonide; intravenous, hidrocortisona, metilprednisolona; or via enemas and suppositories.
Budesonide is not absorved well and has a limited biodistribution but it has good therapeutic benefits with a reduced systemic toxicity in IBD {[}\protect\hyperlink{ref-peyrin-biroulet2010}{54}, \protect\hyperlink{ref-rezaie2015}{66}{]}.
These drugs work very well as antiinflammatory for mild or severe IBD but do not work well as maintenance drug {[}\protect\hyperlink{ref-lichtenstein2009}{67}, \protect\hyperlink{ref-ouellette2001}{68}{]}.

Thiopurines (Azathioprine, mercaptopurina) are immunosuppressants drugs that deactivate key process of limphocytes T that might trigger the inflammation.
As a side effect they are toxic due to their interaction with nucleic acids {[}\protect\hyperlink{ref-warner2018}{69}{]}.
on CD they are useful to induce and keep remission {[}\protect\hyperlink{ref-chande2015}{70}{]}, while on UC they are used to keep the remission {[}\protect\hyperlink{ref-gisbert2009}{71}{]}.

In the last two decades IBD treatment has moved from aminosalicylates, corticosteroids and immunomodulators to anti-TNF\(\alpha\).
Anti-TNF\(\alpha\) drugs has changed IBD treatment as it reduced the hospitalization associated with previous treatments, reducing medical costs and risk of surgery as well as induce a better mucosal healing and quality of life for patients {[}\protect\hyperlink{ref-peyrin-biroulet2011}{72}{]}.
However, 20-30\% of patients have no response to this treatments and another 30-40\% lose response in a year {[}\protect\hyperlink{ref-billioud2011}{73}{]}.

Recently a new wave of drugs has been developed targeting different molecules such as vedolizumab, targeting anti-integrin\(\alpha 4 \beta 7\), ustekinumab, targeting both IL-12 and IL-23, risankizumab an anti IL-23, tofacitinib an inhibitor of JAKs, infliximab an anti-TNF\(\alpha\).

Patients might become refractory to drug.
Thus, drugs do not have the same effect as previously and the dose might need to be increased with the risk of more secondary effects {[}\protect\hyperlink{ref-raine2021}{1}{]}.
Surgery resection might be needed on these patients.

Close to 35\% of patients with UC will need to have a surgery resection, either due to complications or because the inflammation can not be controlled.
Surgery usually removes the inflamed segment of the colon.
The most common procedure used is a colectomy (whole colon removal), with ileostomy {[}\protect\hyperlink{ref-hwang2008}{74}{]}.
CD patients usually require surgery associated to complications like stenosis, abscess, and fistulas ) between 70\% and 90\% at some point of their lives {[}\protect\hyperlink{ref-gardiner2007}{75}{]}.
Usually the surgery is limited to removing the inflamed segment but occasionally an ileostomy is required {[}\protect\hyperlink{ref-lewis2010}{76}{]}.

If the drugs fail to contain the inflammation and heal the mucosa doctors might recommend a different procedure.
In some cases HSCT is recommended which have shown to improve the life of the patients {[}\protect\hyperlink{ref-corralizab}{77}{]}.
This is a new procedure given only to the most extreme cases to reset the immunological state of the patient.

To reset or hugely modify the microbiota fecal microbiota transplantation between different people is currently being explored {[}\protect\hyperlink{ref-weingarden2017}{78}{]}.

\hypertarget{summary-ibd}{%
\subsection{Summary}\label{summary-ibd}}

IBD is a complex disease that impacts the health of many people for long time and with lasting impact on their quality of life.

Current clinical care in some cases is enough to have a sustained clinical and endoscopic remission but most often is not enough and relapse is expected.
Several factors, such as becoming refractory to drugs, intermittent course of the disease and lack of validated predictors of disease course or response to therapy make the treatment complex.

Lack of knowledge of what are the factors cause of the disease make those treatments and drugs to be addressed to block further inflammation and damage, but cannot prevent it and often they do not stop it completely.

\hypertarget{integration-studies-in-ibd}{%
\section{Integration studies in IBD}\label{integration-studies-in-ibd}}

Many studies have looked up to the origin of the disease.
As seen, one of the hypothesis behind the maintenance of the inflammation involves the microbiome and the host epithelium.
This has been studied using several data sources, mostly from sequencing data.
The technical methods used to obtain the data of the inflamed tissue differ between extracted from biopsied samples at colonoscopy or from surgical samples.

Those samples are usually used later on to diagnose or for research purpose.
To obtain research-quality data it usually imply using techniques such as immunohistochemistry, histopathology, immunohistochemistry, fluorescence in situ hybridization and polymerase chain reaction.
These techniques allow to measure or visualize where are the cells expressing certain proteins or genes, thus helping with the analysis validation.

Furthermore several studies have been carried out to discover links between microbiome and the inflammation, followed by those looking for some relationship between genetics and the disease and more recently the metabolome.
These studies, known as integration, multi-omic or interaction studies, usually use multiple sequencing assays as the bases of the analysis {[}\protect\hyperlink{ref-beck2021}{79}{]}.

However, confirming causal interactions of the variables of each essay is difficult.
To find relationships some articles use correlation {[}\protect\hyperlink{ref-hasler_uncoupling_2016}{80}{]}, there are others that use a combination of methods from correlations, partial correlations to integrative methods {[}\protect\hyperlink{ref-tang2017}{81}--\protect\hyperlink{ref-huWholeExomeSequencing2021}{83}{]} and network integrations {[}\protect\hyperlink{ref-hernuxe1ndez-rocha2021}{82}{]}.

Very rarely there is an experimental confirmation of the relationships between variables of the different assays because it is complicated to test an interaction and to set up the right conditions for the many variables that are accounted for on the integrations.
One of the few methods published that shows an interaction between genes and microorganisms in IBD is to expose the \emph{ex-vivo} sample or cell lines with microbiomes or supernatant of at their culture {[}\protect\hyperlink{ref-mayorgas2021}{84}{]}.

\hypertarget{data-origin}{%
\subsection{Type of data used for integration analysis}\label{data-origin}}

According to the data used, we can classify the studies:

\hypertarget{transcriptome}{%
\subsubsection{Transcriptome}\label{transcriptome}}

Most of the integrations refer some other source of data to the transcriptomics of the patient.
The transcriptome of patients derived samples has been extensively studied since the existence of microarrays.
There are known marker gens of inflammation and many research focus on identifying prognosis predictors and treatment response prediction based on gene expression {[}\protect\hyperlink{ref-planell2013}{85}--\protect\hyperlink{ref-massimino2021}{87}{]}.
Recently single-cell RNA-seq technology has enabled to estimate cell populations of the samples with better degree of success than bulk RNA-sequencing.
Single cell technologies are starting to be used for integration.

\hypertarget{microbiome-1}{%
\subsubsection{Microbiome}\label{microbiome-1}}

Many of the integration analysis in IBD are done between host transcriptome and the microbiome.
These studies use datasets from IBD patients usually stratified by disease activity or severity of inflammation or location of the disease.
Most of them are based on correlation analysis between the microbiome and RNA-seq {[}\protect\hyperlink{ref-hasler_uncoupling_2016}{80}{]}.
Conclusions of these integrations range from finding differences on the correlation depending on the type of disease ({[}\protect\hyperlink{ref-hasler_uncoupling_2016}{80}{]}) to finding relationships with inflammatory genes {[}\protect\hyperlink{ref-tang2017}{81}{]}.

\hypertarget{genetics-1}{%
\subsubsection{Genetics}\label{genetics-1}}

Genetics is the next most common data source used to integrate data in IBD.
Most studies on genetics and IBD are genome-wide association studies.
The genetic component is specially important in IBD that starts on children {[}\protect\hyperlink{ref-kumar2019}{14}, \protect\hyperlink{ref-huWholeExomeSequencing2021}{83}, \protect\hyperlink{ref-knights2013}{88}{]}.

When using genetic data to integrate it with transcriptomics it is usually to understand how a genetic variant is affecting a gene expression.
This has lead to the identification of expression quantitative trait loci (eQTL) {[}\protect\hyperlink{ref-repnik2016}{89}--\protect\hyperlink{ref-dai2019}{92}{]}.

\hypertarget{metabolome}{%
\subsubsection{Metabolome}\label{metabolome}}

More recently there have been an increased interest on the study of the metabolic stat of IBD patients, given that microorganisms interact with the host also via their products and metabolites.
There is evidence some of these metabolomic products come from the microbiome {[}\protect\hyperlink{ref-ferrer-picuxf3n2020}{25}, \protect\hyperlink{ref-mayorgas2021}{84}{]}.
Some studies have integrated the metabolome with the RNAseq and state of the epithelium {[}\protect\hyperlink{ref-ahmed2016}{93}, \protect\hyperlink{ref-gallagher2021}{94}{]}.

\hypertarget{integration}{%
\section{Integration}\label{integration}}

The term ``data integration'' is widely used with varying meanings.
According to the dictionary integration is defined as:

\begin{quote}
``the process of combining two or more things into one'' --- \href{https://dictionary.cambridge.org/dictionary/english/integration}{Cambridge Dictionary}
\end{quote}

Other words used are integration, and if specific to data from sequencing technologies, multi-omics or pluri-omics.
Here integration will be used as it is the more general one and not restricted to omics or sequencing technologies.

Since the beginning of the integration methods there have been many methods proposed {[}\protect\hyperlink{ref-krassowski2020}{95}{]}.
Some of the early methods were initially used for surveying the agreement of different evaluating systems {[}\protect\hyperlink{ref-yannakoudakis2015}{96}{]}, others were developed for agricultural sciences {[}\protect\hyperlink{ref-hotelling1936}{97}{]} or food industry {[}\protect\hyperlink{ref-biancolillo}{98}{]}.
Some of these methods are specific for one application or data type while others are more generally applicable.
Lately, the access to bigger datasets with more variables and often from the same samples has increased the focus of the research community on the methodologies available on several disciplines but mainly on biological sciences.
The explosion of data on biological sciences has been driven by the new sequencing technologies that allow to measure thousand of variables of many samples at the same time.
If done with multiple sequencing technologies it is usually referred as multi-omics methods, which usually only uses omic data.

It is crucial to classify, review and compare tools available, as well as, to benchmark these tools against the same dataset as a way to provide clear recommendations to anyone wishing to use them {[}\protect\hyperlink{ref-wu_selective_2019}{99}{]}.
Part of these efforts use the methods' strategies to classify them {[}\protect\hyperlink{ref-cavill2016}{100}, \protect\hyperlink{ref-chongComputationalApproachesIntegrative2017}{101}{]}.
Following this view we will review the available integration methods according to several axes: type of data used, aim of the method, relationships between variables, relationships between samples, relationship between variables and samples, input data, mathematical framework and results of the method.

\hypertarget{classification-of-integration-methods}{%
\subsection{Classification of integration methods}\label{classification-of-integration-methods}}

Integration methods have very different properties that allow them to be classified and compared {[}\protect\hyperlink{ref-huang2017}{102}{]}.
Here we classify those meant to be used with omics datasets, with references to concrete methodology and in occasionally to articles using them.

\begin{figure}
\includegraphics[width=1\linewidth]{images/figure1_huang2017} \caption[Unsupervised data integration methodology.]{Unsupervised data integration methodology. Figure 1 from Huang 2017.}\label{fig:classification}
\end{figure}

\hypertarget{data-type-numeric-or-categorical}{%
\subsubsection{Data type: numeric or categorical}\label{data-type-numeric-or-categorical}}

The most important distinction in integration methods is what kind of data are combined.
In general data can be divided between categorical and numeric variables, which are usually found in several fields.
Sometimes clinicians want to understand the relationship between a phenotype they observe and the underlying mechanism.
Usually this involves looking how metabolites, gene expression, methylation, number of variants a gene has, and other numeric variables are related to the observed phenotype (i.e.~pain).

Depending on the method's aim it handles numeric and categorical data types or just one.
Often they are used differently.
The most common way to handle different types of data is converting the categorical values to a mock or dummy variable.
For each categorical factor there is a new variable whose value is 1 if that sample had this factor and 0 otherwise.
For instance, if the categorical variable has three values (A, B, C) it would be converted to A (1, 0, 0) B (0, 1, 0) and C (0, 0, 1).
Often the number of variables created is one less than the number of factors that existed, on this example only A and B would be kept.
This transformation allows to use categorical values as numeric variables.

If the method only accepts categorical data but you want to provide numeric values usually those values are categorized.
For example if a variable is (0.123, 0.25, 0.56, 0.78) one could make to categorical values like (``\textless0.5'', ``\textless0.5'', ``\textgreater0.5'', ``\textgreater0.5'').
The number of categories to use and how the numeric value splits depends on each case.

Very rarely methods allow to use both data types as they are.
If they allow so, it is usually for classification purposes only.

\hypertarget{objective}{%
\subsubsection{Objective}\label{objective}}

The objective of any method is one of its most important defining properties.
Data integration can be classified according to the (biological) question they try to answer.
In general all of them aim to provide a better understanding of the relationships between the 'omics types.
Often a single method is not enough and several methods are used on the same dataset.
This is specially relevant when a potentially relationship between omics is discovered.
For instance, checking that in a particular case or condition a given relationship is present, might require experimental confirmation.

Most of the times one (or several) of the following results are expected from integration methods:

\begin{itemize}
\tightlist
\item
  Classification of the patients or samples
\end{itemize}

One of the purposes of integration might be to use multiple sources of data to accurately describe how do samples or patients fit on a predefined possibles states.
The objective here might be to accurately describe whether a patient has one or other related disease of a possible subset of diseases or phenotype {[}\protect\hyperlink{ref-rohart2016}{103}{]}.

\begin{itemize}
\tightlist
\item
  An overview of the role of each individual omic in a biological system
\end{itemize}

Sometimes the question is which omic method is the best describing a disease.
This knowledge could prevent performing expensive tests and replace them by more affordable or easier technique that has enough predictive power or is sensitive enough and specific for the task.
An example of this is the search of markers on blood to identify links between different cell populations {[}\protect\hyperlink{ref-tang2017}{81}, \protect\hyperlink{ref-ibrahim2017}{104}{]}.

\begin{itemize}
\tightlist
\item
  Finding a molecular signature
\end{itemize}

A signature is usually a group of features that describe/are representative of a cell type, a process or a stage.
Identifying a subset of variables from the omics that are related is often a desired goal because it reduces the amount of variables allowing to perform experiments on the bench on just those that might be important.
In other fields, such as machine learning, selecting the important variables is known as \href{https://en.wikipedia.org/wiki/Feature_selection}{feature selection}.
There are several methods that are used to do this {[}\protect\hyperlink{ref-cavill2016}{100}{]}.
An example of this is when performing eQTL analysis, where a locus is related to a change in gene expression {[}\protect\hyperlink{ref-repnik2016}{89}--\protect\hyperlink{ref-dai2019}{92}{]}.

\begin{itemize}
\tightlist
\item
  A predictive model
\end{itemize}

Predictive models usually require a very good understanding of the current and/or past relationships, as well as, a good feature selection procedure.
If a good model on previous data exists it might be used to predict future events.
Sometimes, models are only built to predict events without being able to accurately understand the underlying mechanism.
This kind of methods are used to improve treatment selection, diagnosis and prediction of prognosis {[}\protect\hyperlink{ref-wheeler2014}{105}{]}.

\begin{itemize}
\tightlist
\item
  Impute values
\end{itemize}

Some methods aim to accurately guess the values of missing data given some other information.
Missing values can happen for a variety of reasons from practical ones, like a sample not being available, to technical ones, such as laboratory mistake {[}\protect\hyperlink{ref-yin2019}{106}{]}.
However, this is often a intermediate step to other goals.

To complete these goals it is important to have enough statistical power to determine the significance of tests performed (if any) and to understand how complete are the data sources used on the integration {[}\protect\hyperlink{ref-tarazona2021}{107}{]}.
Having more statistical power helps identifying the relationships one seeks when using this methods.

\hypertarget{relationship-between-variables-and-samples}{%
\subsubsection{Relationship between variables and samples}\label{relationship-between-variables-and-samples}}

Depending on the amount of variables and samples used in studies can be classified in two types.
Traditionally for each sample few variables are measured, for instance on a biopsy with RT-PCR only a few genes are measured, however with the new omics techniques (transcriptomics, metabolomics, methylomics, genomics), thousands of variables are measured for the same sample.
This has lead to the following situation:

\begin{itemize}
\tightlist
\item
  More variables than samples
\end{itemize}

For a single sample of RNA around 50k genome identifiers (genes, long non coding RNAs, iRNA, pseudogenes,\ldots) can be measured.
Which leads to the case where there are many more variables than samples.
Thus high-throughput data analysis typically falls into the category of \(p \gg n\) problems ( big p, little n), where the number of genes or proteins, \(p\), is considerably larger than the number of samples, \(n\).
With such high number of variables the identification of the relevant variables is hard because variables will co-variate.
When many variables are tightly correlated, discovering which one is important using just numerical methods is challenging.
This is even more difficult when looking for causal relationships.

\begin{itemize}
\tightlist
\item
  More samples than variables
\end{itemize}

This was the usual case when for instance, from a cohort of patients the temperature is measured along the stage of a disease: two variables, time and temperature for each sample.
If there are more than 2 patients, then the number of samples is greater than the number of variables studied.
This is described in the literature as \(n \gg p\) (or big n, little p).
Nowadays this is less frequent on the bioscience world, and does not causes trouble analyzing it because the high number of samples allows to accurately estimate the dispersion of variables.

There are several methods available to estimate the number of samples required {[}\protect\hyperlink{ref-tarazona2020}{108}{]}.
Having just the enough samples for the desired statistical power, however, might not be enough in case some samples are not correctly processed.

In addition, variables might be separated on different blocks of variables.
These blocks might be just of the same source or from multiple methods.
Depending on the method this blocks might have special meaning: when all the data is joined in a single block it is known as superblock.

\hypertarget{relationship-between-samples}{%
\subsubsection{Relationship between samples}\label{relationship-between-samples}}

Depending on the relationship between the samples, the questions answerable and the methods that work on them differ.

A sample can have multiple or one data source, for instance we could have RNA-seq and 16S data from the same sample.
In a study if all the samples have all the data from multiple data sources it is a complete case.
If some samples have data from some data sources but not from others the study is not a with a complete case.

Sometimes because the sample is not enough, or there are some technical or organizational problems a source of data for a sample (which is known as an incomplete case) might be lost.
This results in a new source of variation that has to be dealt with, which complicates the conclusion one can draw from the studies of these kind of data.

Even when all the cases of a patient are complete the samples can come from several sites of the same individual or with different combinations of variables, which makes it relevant to understand the relationships between the different samples.

There is no easy classification of this as each experiment might be designed differently.
In general, experiments are designed to be as consistent as possible but in face of adverse events that become a variation of the design the analysis complicates.
Either some data is imputed or some samples are omitted for the analysis.
This can happen with samples taken at different timepoints as patients for instance if they miss a follow up visit.

\hypertarget{time}{%
\paragraph*{Time}\label{time}}
\addcontentsline{toc}{paragraph}{Time}

As mentioned above, time is one of the factors that sometimes cannot be controlled, despite having programmed visits every two weeks, for instance, some patients might come early or later due to multiple reasons.

Sometimes, the objective of the study is precisely to analyze the relationships at different time, or to asses how the relationships change with time.
To discover causality between two variables the cause must precede the consequence, which highlights the importance of time.
Being aware of the time differences and time scales is crucial in most cases.

During \emph{in vitro} experiments, conditions can be reproduced even if they are at different time.
However, when using patients material replicates can not usually be performed like \emph{in vitro} experiments.
This makes it harder to study time-related change on patients.

Lastly, time between the collection of a fresh samples and its processing also influences the readings of the samples of the omics technology, specially RNA-seq {[}\protect\hyperlink{ref-massoni-badosa2020}{109}, \protect\hyperlink{ref-zhu2017}{110}{]}.
Some genes are more influenced by time than others but as they are measured at the same time in all samples this might distort the data.
Keeping track of the time that it takes to process samples is also hard to due and requires a highly coordinated effort {[}\protect\hyperlink{ref-ferreira2018}{111}{]}.

\hypertarget{relationship-between-variables}{%
\subsubsection{Relationship between variables}\label{relationship-between-variables}}

Once data is collected, the next step is to understand the relationship between the variables present.
As mentioned earlier, some variables influence others which can affect the outcome in complex ways.
With many variables present in a dataset it is important to be aware of known relationships between variables.
Even in a simple dataset, like an RNA-seq dataset, it is important to be aware of the relationships between variables.

Since the discovery of the lactose \href{https://en.wikipedia.org/wiki/Operon}{operon} it is known how some genes regulate each other {[}\protect\hyperlink{ref-jacob1961}{112}{]}.
However, it is not know how other variables are related between them.
For instance, how does the increase in expression of a gene affects the growth of a microorganism?
Usually the relationships between variables are mediated by many factors or interactions.

One of the best examples of such interactions is when some variables correlate.
Their correlation can be used to reduce the number of variables being analyzed by ignoring the relationships between them and using the most representative variable (less widely correlated and with more variation).
This step is usually done by dimension reduction methods.
However, sometimes this is not desirable or feasible as the correlation does not explain the direction of the causality of the interaction between the variables (if there is any).

Network approaches relate the variables to each other {[}\protect\hyperlink{ref-koh_iomicspass_2019}{113}{]}.
These approaches are fairly new and growing in popularity partly because they can address the direction of the interaction.

In partial correlations the effect of other variables on the two being under study are taken into account {[}\protect\hyperlink{ref-yule1907}{114}{]}.
They assumes a linear relationship between the co-occurring variables and those of interest.
However, it is computationally expensive when there are thousands of variables.

\hypertarget{input-data}{%
\subsubsection{Input data}\label{input-data}}

We have classified the studies according to the data they use (as seen \protect\hyperlink{data-origin}{previously} ).
But, some methods to account for relationships of variables only work when a dataset is complete while other do not:

\begin{itemize}
\tightlist
\item
  Data from the same samples:
\end{itemize}

These methods do not handle well or at all missing data.
They need complete cases/data of the samples in order to be able to integrate the results.
These methods include Regularized Generalized Canonical Correlation Analysis (RGCCA) {[}\protect\hyperlink{ref-tenenhaus_regularized_2011}{115}, \protect\hyperlink{ref-tenenhaus_variable_2014}{116}{]}, Multiple co-inertia analysis (MCIA) {[}\protect\hyperlink{ref-culhane_cross-platform_2003}{117}{]}, Multi-Study Factor Analysis (MSFA) {[}\protect\hyperlink{ref-vito_multi-study_2019}{118}{]}, Multi-Omics Factor Analysis (MOFA) {[}\protect\hyperlink{ref-argelaguet_multi-omics_2018}{119}{]} and STATegRa {[}\protect\hyperlink{ref-gomez-cabrero2019}{120}{]}.

\begin{itemize}
\tightlist
\item
  Data from different samples:
\end{itemize}

These methods do not need data from the same sample.
They draw their conclusions generalizing from the data available.
Some of them handle missing data, while others do use the data at face value.
These method includes MetaPhlAn2 {[}\protect\hyperlink{ref-franzosaSpecieslevelFunctionalProfiling2018}{121}{]}, HUMAnN,{[}\protect\hyperlink{ref-truongMetaPhlAn2EnhancedMetagenomic2015}{122}{]} and LEfSe {[}\protect\hyperlink{ref-segataMetagenomicBiomarkerDiscovery2011}{123}{]}.

Furthermore, some methods are designed to integrate specific types of datasets, (usually because they make some assumptions that are only met on that kind of data).
For instance, HCG, 16S rRNA-seq, RNA-seq and metabolomics do not share the same data distribution, and are different between them.
Also even with the same data depending on the processing of the data they can have very different properties: OTUs (operational taxonomic unit) properties are not the same as ASV (amplicon sequence variants) when analyzing 16S rRNAseq data.

\hypertarget{mathematical-framework}{%
\subsubsection{Mathematical framework}\label{mathematical-framework}}

Methods use different mathematical frameworks to process the data.
Here we briefly describe some common mathematical frameworks, some of which have previously appeared:

\begin{itemize}
\item
  Networks

  Networks methods were mentioned because they use and find information about the interaction of variables.
  Multilayer networks, including the multiplex, Molti-C-DREAM {[}\protect\hyperlink{ref-didier2018}{124}{]}, Random Walk with Restart on Multiplex and Heterogeneous Biological Networks RWR-MH, Random walk with Restart on Multiplex RWR-M {[}\protect\hyperlink{ref-valdeolivas2019}{125}{]}.
  Network embedding MultiVERSE are some of the methods using networks {[}\protect\hyperlink{ref-pio-lopez2021}{126}{]}.

  Bayesian approaches are also quite frequent, these methods use the Bayes' theorem to see the relationships between variables.
  The Bayes' theorem explains that the conditional probability of a variable is related to the prior knowledge of conditions that might be related to the event {[}\protect\hyperlink{ref-bayes1763}{127}{]}.
  Some methods that use these approaches are Reconstructing Integrative Molecular Bayesian Network (\href{https://labs.icahn.mssm.edu/zhulab/?s=rimbanet}{RIMBANET}) {[}\protect\hyperlink{ref-zhu2012}{128}{]} and Bayesian Consensus Clustering (\href{https://github.com/ttriche/bayesCC}{BCC}) {[}\protect\hyperlink{ref-lock2013}{129}{]}.
\item
  Dimensional Reduction

  These methods focus on finding just a few variables and summarizing them using a function that has some desired property such as the correlation between thre transformed variables is maximal while the components are orthogonal.s The selection of variables is usually done with L1 or Lasso Regression regularization technique or L2 also known as Ridge Regression.
  L1-regularization adds a penalty equal to the absolute value of the magnitude of coefficients which might leads to some coefficients becoming zero and the variable eliminated from the model.
  On the other hand, L2-regularization does not result in elimination of coefficients or sparse models and can only be used when there is multicollinearity as it works well to avoid over-fitting.

  Several tools use this approach, Momix {[}\protect\hyperlink{ref-cantini2021}{130}{]}, regularized canonical correlation analysis (RGCCA) {[}\protect\hyperlink{ref-tenenhaus2017}{131}{]}, mixOmics {[}\protect\hyperlink{ref-rohart2017}{132}{]} and STATegRa {[}\protect\hyperlink{ref-gomez-cabrero2019}{120}{]}.
  Other methods use bayesian approches like the Bayesian Group Factor Analysis {[}\protect\hyperlink{ref-virtanen2012}{133}{]}.
\item
  Active module identification

  Multi-omic objective genetic algorithm (scores based in two metrics; node score and density of interactions score).
  An example of a method using this approach is Multi-Objective Genetic Algorithm to Find Active Modules in Multiplex Biological Networks (MOGAMUN) {[}\protect\hyperlink{ref-novoa-del-toro2020}{134}{]}
\end{itemize}

Usually depending on the mathematical framework used, these methods return similar outputs.

\hypertarget{output-results}{%
\subsubsection{Output results}\label{output-results}}

According to the output the integration methods can be classified in several groups:

For the network methods the following output is usually returned: Connections between the variables/nodes, a measure of how strong is the connection (or simply if there is a connection or not).

For dimensional reduction methods there are three possible outputs: Shared factor across the data, specific factors for each data or mixed factors.

\begin{itemize}
\tightlist
\item
  Shared factors:
\end{itemize}

Integration results in a vector of the samples in a lower dimensional space that is shared by all the data set used to integrate.
Such methods include iCluster, Multi-Omics Factor Analysis (MOFA) {[}\protect\hyperlink{ref-argelaguet_multi-omics_2018}{119}{]}.

\begin{itemize}
\tightlist
\item
  Specific factors:
\end{itemize}

Integration results in several vectors of the samples in a lower dimensional space of each data set used to integrate.
Such methods include Regularized Generalized Canonical Correlation Analysis (RGCCA) {[}\protect\hyperlink{ref-tenenhaus_regularized_2011}{115}, \protect\hyperlink{ref-tenenhaus_variable_2014}{116}{]}, Multiple co-inertia analysis (MCIA) {[}\protect\hyperlink{ref-culhane_cross-platform_2003}{117}{]} and Multi-Study Factor Analysis (MSFA) {[}\protect\hyperlink{ref-vito_multi-study_2019}{118}{]}.

\begin{itemize}
\tightlist
\item
  Mixed factors:
\end{itemize}

Integration results in both shared and specific factors, to each dataset and common to all them.
Such methods include Joint and Individual Variation Explained (JIVE) {[}\protect\hyperlink{ref-lock2013a}{135}{]} and integrative Non-negative Matrix Factorization (\href{https://github.com/yangzi4/iNMF}{iNMF}) {[}\protect\hyperlink{ref-yang2016}{136}{]}.

\hypertarget{interpretation}{%
\subsection{Interpretation}\label{interpretation}}

How to interpret the results of applying the different methods is highly linked to understanding the method and its output.
On a correlation between two variables, the interpretation of the analysis is clear, if one variable increase, the other one too.
The implications of this correlation can be far reaching but the principles to understand them are simple.

However, on more complex methods the interpretation becomes less clear.
The interpretation of a canonical correlation analysis is much harder {[}\protect\hyperlink{ref-sherry2005}{137}{]}.
Also on more complex methods the number of parameters required increases so the time and intellectual effort to understand the relationships between the parameters is also higher.

The interpretation also helps to discuss the results and relate it to other previously know information.

\begin{itemize}
\tightlist
\item
  Individually:
\end{itemize}

Here we study how each variable relates to another.
In the correlation analysis, the relationship between two variables under study.
Or if looking by patient: how do interpret that in these patient variable A and B is X and Y?

\begin{itemize}
\tightlist
\item
  Globally:
\end{itemize}

In a principal component analysis for instance how do we interpret that some variables have the same loading?
What happens in a more complex method like canonical correlation analysis?

There are some articles about how to interpret those methods on real datasets {[}\protect\hyperlink{ref-sherryConductingInterpretingCanonical1981}{138}{]}.
Others, to benchmark and to learn how to interpret propose analyzing a simulated dataset {[}\protect\hyperlink{ref-chung_multi-omics_2019}{139}, \protect\hyperlink{ref-martuxednez-mira2018}{140}{]}.
Which is used to compare the results of the integration with the dataset of interest and to compare different tools.
These datasets are created with some relationships that the tools are expected to find.

There exists several methods to create synthetic datasets like MOSim {[}\protect\hyperlink{ref-martuxednez-mira2018}{140}{]}, metaSPARSim {[}\protect\hyperlink{ref-patuzzi2019}{141}{]}, CAMISIM {[}\protect\hyperlink{ref-fritz2019}{142}{]}, ballgown {[}\protect\hyperlink{ref-fu2021}{143}{]}, polyester {[}\protect\hyperlink{ref-frazee2021}{144}{]} and even edgeR {[}\protect\hyperlink{ref-mccarthy2012}{145}{]}.
These methods are useful to compare different setup but they can miss some subtle not previously reported relations on real data.

\hypertarget{reviews}{%
\subsection{Reviews}\label{reviews}}

The comparison and review of methods independently from original authors have become a crucial step for selecting the right tool to apply a given dataset and research question {[}\protect\hyperlink{ref-cantini2021}{130}{]}.

Some of these reviews focus on a specific type of data integration: metabolomics {[}\protect\hyperlink{ref-cavill2016}{100}{]}, genomics {[}\protect\hyperlink{ref-mcgovern2015}{11}{]}, microbiomics\ldots{} Others focus on the disease and the challenges of each omics and the need of an integrative approach to provide better therapies {[}\protect\hyperlink{ref-tarazona2021}{107}, \protect\hyperlink{ref-de_souza_ibd_2017}{146}, \protect\hyperlink{ref-valles-colomer2016}{147}{]}.
On this regard there are several efforts to integrate data in IBD but no comprehensive review to date is known to the author.
The most comprehensive article to date is a very recent review identifying problems and providing recommendations for future work {[}\protect\hyperlink{ref-sudhakar2022}{148}{]}.

\hypertarget{methods-integration}{%
\subsection{Summary}\label{methods-integration}}

The field of integration is large and complex, with increasing interest over the last few years, specially in the psychology and omics fields.
As a methodology they are quite complex and diverse but there is a growing interest on them to help answer complex questions without using other complex tools like deep neuronal networks or other machine learning approaches (despite not being incompatible).

Methods to integrate have many characteristics, depending on the objectives and data that available.
Regardless of the method used, interpretation and reporting is usually the main challenge.

\hypertarget{summary}{%
\section{Summary}\label{summary}}

IBD is a gastrointestinal disease that includes two different diseases UC and CD.
It affects preferentially the lower intestinal tract causing lesions on the epithelial barrier.
Bacteria causes or use these lesions to further damage the patients.
But this interaction is also influenced by many other variables.

Data integration are methods to analyze datasets with data from multiple sources.
They include a variety of methods and there are methods for several purposes.
In IBD it has been used mostly sequencing data, disregarding the other variables known to be relevant.

Data integration in IBD has been underexplored despite being known that many factors interact on the pathogenesis and maintenance of the disease.

\hypertarget{hypothesis-and-objectives}{%
\chapter{Hypothesis and objectives}\label{hypothesis-and-objectives}}

\hypertarget{hypothesis}{%
\section{Hypothesis}\label{hypothesis}}

IBD is a gastrointestinal disease affecting preferentially the lower intestinal tract causing lesions on the epithelium.
Bacteria interact with host's epithelium and with many other variables.
There are data integration methods for a variety of purposes, including finding relationships between variables.
On IBD these methods have not been used much to find which bacteria is related to the host's genes.

We hypothesize that the expression state of the epithelium and the \textbf{microorganisms present might identify whether patients are suffering IBD if environmental factors are taken into account}.

\hypertarget{objectives}{%
\section{Objectives}\label{objectives}}

The main objective of this thesis is finding the relationships between microbiome and gene expression in the intestinal mucosa. For this reason we identify microorganisms and genes related to IBD.

\begin{itemize}
\item
  Identify relationships between intestinal mucosal gene expression and microbiome presence on the intestine.
\item
  Determine the influence in the microbiota and mucosa relationship of clinical variables such as location (colon and ileum), disease status, sex, age, treatment received, etc. in the microbiome and mucosa relationship.
\item
  Identify groups of possible underlying mechanism interactions between the microbiome and mucosa in IBD.
\end{itemize}

\hypertarget{materials-and-methods}{%
\chapter{Materials and methods}\label{materials-and-methods}}

This chapter contains a brief description of the main characteristics of the different datasets used on this thesis .
The complete processing protocol before obtaining the data of those dataset that were not generated at Hosptial Clínic can be found on their respective reference.

Samples of the different cohorts collected in Hospital Clínic were collected similarly and described only once.
Differences between protocols are noted on the respective dataset's section.

Here we describe all the methods used to analyze data from the multiple cohorts included in this thesis.
The code used can be found on the links provided in the \protect\hyperlink{software}{appendix}.

\hypertarget{datasets}{%
\section{Datasets}\label{datasets}}

\hypertarget{methods-puget}{%
\subsection{Puget's dataset}\label{methods-puget}}

The glioma dataset is the data provided as an example of biological data by the authors of RGCCA from a previous publication {[}\protect\hyperlink{ref-puget2012}{149}{]}.
The data came from diffuse intrinsic pontine glioma patients whose transcriptome was analyzed with Agilent 44K Whole Human Genome Array G4410B and G4112F.
The copy number variation of the samples was processed with the ADM-2 algorithm, and data from comparative genomic hybridization (CGH) analyzed using \href{https://www.softgenetics.com/mutationSurveyor.php}{Mutation Surveyor software}.
In addition, this dataset contained information on age, localization of the tumor, sex and a numerical grading of the severity of the tumor {[}\protect\hyperlink{ref-puget2012}{149}{]}.

\begin{longtable}[]{@{}lc@{}}
\caption{\label{tab:Puget} Characteristics of samples from the Puget's dataset.}\tabularnewline
\toprule
Characteristic & Puget's \\
\midrule
\endfirsthead
\toprule
Characteristic & Puget's \\
\midrule
\endhead
Samples & 53 \\
Sex (female/male) & 28/25 \\
Location (cort/dipg/midl) & 20/22/11 \\
\bottomrule
\end{longtable}

\hypertarget{methods-hsct}{%
\subsection{HSCT dataset}\label{methods-hsct}}

Samples from the HSCT dataset used in this thesis were from a cohort of patients with severe refractory CD undergoing HSCT.
Patients were treated in the Department of Gastroenterology (Hospital Clínic de Barcelona --Spain--).
The protocol was approved by the Catalan Transplantation Organization and by the Institutional Ethics Committee of the Hospital Clinic de Barcelona (Study Number HCB/2012/7244).
All patients provided written consent following extensive counselling about being included on the study and using their data on publications.

Colonic and ileal biopsies were obtained at several time points during ileo-colonoscopy, at inclusion and every 6 or 12 months after HSCT up to 4 years after the start of the treatment.
Samples were obtained when possible from both uninvolved and involved areas.
In addition, biopsies were taken from the ileum and colon regions of 19 non-IBD controls consisting of individuals with no history of IBD and who presented no significant pathological findings following endoscopic examination for colon cancer surveillance (Hospital Univesitari Mútua de Terrassa --Spain--).
The protocol was approved by the Institutional Ethics Committee of the Hospital Univesitari Mútua de Terrassa (Study Number NA1651).

At least one biopsy was collected and fresh-frozen at -80°C for microbial DNA extraction.
The remaining biopsies were placed in RNAlater RNA Stabilization Reagent (Qiagen, Hilde, Germany) and stored at -80°C until total RNA extraction.
In total 158 samples with both RNA and DNA extraction of the same segment and time were available \ref{tab:HSCT}:

\begin{longtable}[]{@{}lc@{}}
\caption{\label{tab:HSCT} Characteristics of samples from the HSCT dataset.}\tabularnewline
\toprule
Characteristic & HSCT \\
\midrule
\endfirsthead
\toprule
Characteristic & HSCT \\
\midrule
\endhead
Sex (female/male) & 22/15 \\
Age at diagnostic (\textless17/\textless40/\textgreater40 years) & 7/11/0 \\
Duration (in years): mean (min-max) & 14 (8-28) \\
Age: mean (min-max) & 44 (23-70) \\
Disease status (non-IBD/CD) & 51/107 \\
Sample location (ileum/colon/unknown) & 48/108/2 \\
Local simple endoscopic score for CD: mean (min-max) & 2.15 (0-12) \\
CDAI: mean (min-max) & 120 (0-450 \\
\bottomrule
\end{longtable}

\hypertarget{methods-hasler}{%
\subsection{Häsler's dataset}\label{methods-hasler}}

An IBD-related dataset was obtained by Prof.~Dr.~Rosentiel and Prof.~Dr.~Robert Häsler {[}\protect\hyperlink{ref-hasler_uncoupling_2016}{80}{]}.
Biopsies were obtained endoscopically during routine diagnosis RNA and DNA were extracted using standard procedures.
DNA from the 16S rRNA gene was amplified with primers 319F and 806R.
Both RNA and DNA was then sequenced on HiSeq 2000 and MiSeq respectively.
These biopsies included samples from the terminal ileum and sigma from CD, UC, infectious disease-controls and healthy non-IBD.
The dataset included information about gender, location, age, and the status (inflamed or non-inflamed) of the region from which the biopsy was taken.

\begin{longtable}[]{@{}lc@{}}
\caption{\label{tab:Hasler} Characteristics of samples from the Häsler's dataset.}\tabularnewline
\toprule
Characteristic & Häsler's \\
\midrule
\endfirsthead
\toprule
Characteristic & Häsler's \\
\midrule
\endhead
Disease status (non-IBD/IBD) & 33/26 \\
Sex (female/male) & 42/17 \\
Sample location (ileum/colon) & 30/29 \\
\bottomrule
\end{longtable}

\hypertarget{methods-morgan}{%
\subsection{Morgan's dataset}\label{methods-morgan}}

A previously published dataset from a pouchitis study was analyzed {[}\protect\hyperlink{ref-morgan2015}{150}{]}.
In this study patients having undergone proctocolectomy with ileal pouch-anal anastomosis for treatment of UC or familial adenomatous polyposis at least 1 year prior to enrollment were recruited at Mount Sinai Hospital (Toronto, Canada) excluding individuals with a diagnosis of CD.
The dataset has a total of 255 samples from 203 patients, containing data for both host transcriptome and intestinal microbiome.
On some cases several biopsyes were collected from the same patients.
This dataset included anonymous identifiers for patients, whether the sample was from the pre-pouch ileum (PPI) or from the pouch, the sex, the outcome of the procedure and an inflammation score.
The pouch ileum might be inflamed or not.

\begin{longtable}[]{@{}lc@{}}
\caption{\label{tab:Morgan} Characteristics of samples from the Morgan's dataset.}\tabularnewline
\toprule
Characteristic & Morgan's \\
\midrule
\endfirsthead
\toprule
Characteristic & Morgan's \\
\midrule
\endhead
Samples (n) & 255 \\
Sex patients (female/male) & 101/102 \\
Sample location (Pouch/PPI) & 59/196 \\
\bottomrule
\end{longtable}

\hypertarget{methods-howell}{%
\subsection{Howell's dataset}\label{methods-howell}}

This dataset included a cohort of 66 treatment-naïve children at diagnosis of their IBD, along with 30 age- and sex-matched non-inflammatory control children, recruited at the Paediatric Gastroenterology unit at Addenbrooke's Hospital (England) {[}\protect\hyperlink{ref-howell2018}{151}{]}.

Data from 77 samples that had both RNAseq and 16S data was used.
There are 10 non-IBD samples, 11 with CD and 11 with UC.
Data has the following characteristics: disease, age at diagnostic, age at time of study, sex, sample location, and disease activity:

\begin{longtable}[]{@{}lc@{}}
\caption{\label{tab:howell} Characteristics of samples from the Howell's dataset.}\tabularnewline
\toprule
Characteristic & Howell's \\
\midrule
\endfirsthead
\toprule
Characteristic & Howell's \\
\midrule
\endhead
Disease (non-IBD/CD/UC) & 11/10/11 \\
Age at diagnostic (\textless17/\textless40/\textgreater40 years) & 32/0/0 \\
Age: mean (min-max) & 12 (6-15) \\
Sex (female/male) & 10/22 \\
Segment (ileum/colon) & 31/46 \\
Clinical history (inflammation/no inflammation) & 24/53 \\
\bottomrule
\end{longtable}

\hypertarget{processing}{%
\section{Sample processing}\label{processing}}

\hypertarget{rna-sequencing}{%
\subsection{RNA sequencing}\label{rna-sequencing}}

Total RNA from mucosal samples (HSCT cohort) was isolated using the RNAeasy kit (Qiagen, Hilde, Germany).
RNA sequencing libraries were prepared for paired-end sequencing using HighSeq-4000 platform.
Samples with good enough quality as recommended by \href{https://www.bioinformatics.babraham.ac.uk/projects/fastqc/}{FastQC} were processed with \href{https://github.com/marcelm/cutadapt}{cutadapt} (version 1.7.1 {[}\protect\hyperlink{ref-martin2011}{152}{]}) for quality filtering.
Later, the libraries were mapped against the human reference genome using the \href{https://github.com/alexdobin/STAR}{STAR aligner} (2.5.2a) with Ensembl annotation (\href{https://www.gencodegenes.org/human/release_26.html}{release 26 of GENCODE}, GRCh38.p10 or superior) {[}\protect\hyperlink{ref-dobin2013}{153}{]}.

Read counts per gene were obtained with \href{https://github.com/deweylab/RSEM}{RSEM} (version 1.2.31) {[}\protect\hyperlink{ref-li2011}{154}{]} as previously described {[}\protect\hyperlink{ref-corralizaDifferencesPeripheralTissue}{155}{]}.

\hypertarget{microbial-dna-sequencing}{%
\subsection{Microbial DNA sequencing}\label{microbial-dna-sequencing}}

Biopsies from the HSCT CD cohort were resuspended in 180 \(\mu\)l TET (TrisHCl 0.02M, EDTA 0.002M, Triton 1X) buffer and 20mg/ml lysozyme (Carl Roth, Quimivita, S.A.). Samples were incubated for 1h at 37°C and vortexed with 25 \(\mu\)l Proteinase K before incubating at 56°C for 3h.
Buffer B3 (NucleoSpin Tissue Kit--Macherey-Nagel) was added followed by a heat treatment for 10 min at 70°C.
After adding 100\% ethanol, samples were centrifuged at 11000 x g for 1 min.
Two washing steps were performed before eluting DNA.
Concentrations and purity were checked using NanoDrop One (Thermo Fisher Scientific).
Samples were immediately used or placed at -20°C for long-term storage until DNA sequencing.

\hypertarget{dna-sequencing}{%
\subsubsection{DNA sequencing}\label{dna-sequencing}}

Microbial cells were disrupted by mechanical lysis using FastPrep-24.
Heat treatment and centrifugation were conducted after adding a cooling adaptor.
Supernatants were treated with RNase to eliminate RNA.
Total DNA was purified using gDNA columns as described in detail previously {[}\protect\hyperlink{ref-berry2011}{156}{]}.
Briefly, the V3-V4 regions of 16S rRNA gene were amplified (15x15 cycles) following a previously described two-step protocol {[}\protect\hyperlink{ref-klindworth2013}{157}{]} using forward and reverse primers 341F-ovh/785R-ovh {[}\protect\hyperlink{ref-edgar2013}{158}{]}.
Purification of amplicons was performed by using the AMPure XP system (Beckmann).
Next, sequencing was performed with pooled samples in paired-end modus (PE275) using an MiSeq system (Illumina, Inc.) according to the manufacturer's instructions and 25\% (v/v) PhiX standard library.

Library preparation and sequencing of the HSCT dataset were performed at the Technische Universität München.
Briefly, volumes of 600\(\mu\)L DNA stabilization solution (STRATEC biomedical) and 400\(\mu\)L Phenol:choloform:isoamyl alcohol (25:24:1, Sigma-Aldrich) were added to the aliquots.

\hypertarget{microbial-profiling}{%
\subsubsection{Microbial profiling}\label{microbial-profiling}}

For the HSCT dataset the processing of raw-reads was performed by using the \href{https://www.imngs.org/}{IMNGS} (version 1.0 Build 2007) {[}\protect\hyperlink{ref-lagkouvardos2016}{159}{]} pipeline based on the \href{https://www.drive5.com/uparse/}{UPARSE} approach {[}\protect\hyperlink{ref-edgar2013}{158}{]}.
Sequences were demultiplexed, trimmed to the first base with a quality score \textless3 and then paired.
Sequences with less than 300 and more than 600 nucleotides and paired reads with an expected error \textgreater3 were excluded from the analysis.
The 5 nucleotides from each end of the remaining reads were trimmed to avoid GC bias and non-random base composition.
Operational taxonomic units (OTUs) were clustered at 97\% sequence similarity.
Taxonomy assignment was performed at 80\% confidence level using the RDP classifier and the SILVA ribosomal RNA gene database project.
Later the data was normalized using the same method as for RNA-seq described above.
The microbiome was visually inspected for batch effects in PCA; none were found.
The resulting OTUs table was normalized using edgeR (Version 3.28 or later) {[}\protect\hyperlink{ref-mccarthy2012}{145}{]}.

For all the other datasets \href{https://bioconductor.org/packages/dada2}{dada2} {[}\protect\hyperlink{ref-callahan2016}{160}{]} (Version 1.14 or later) was used to analyze microbiome data.
It creates amplicon sequencing variants from the 16S sequencing data, without merging similar sequences at any threshold.
It is an alternative to the use of OTUs which allows to compare results between studies and provides more resolution to identify differences on the fragment of 16S amplified.

We used Silva v138.1 to annotate the 16S fragments whenever possible {[}\protect\hyperlink{ref-quast2013}{161}{]}.
If we did not have access to the direct ASV we used the annotation provided by the original authors of the dataset.

\begin{figure}
\includegraphics[width=1\linewidth]{images/workflow_integration2} \caption[Workflow of the analysis process.]{Workflow of the main analysis process of the thesis. Created with BioRender.com}\label{fig:workflow-process}
\end{figure}

\hypertarget{integration-methods}{%
\section{Integration methods}\label{integration-methods}}

The main method used on this thesis has been regularized generalized canonical correlation analysis (RGCCA) a method derived from the canonical correlation.
Canonical correlation is a method that uses data about the same unit but from different origins to find how much the different sources agree.
Regularized generalized canonical correlation analysis is implemented on the homonymous package \href{https://cran.r-project.org/package=RGCCA}{RGCCA} {[}\protect\hyperlink{ref-tenenhaus2017}{131}{]} which was used here.
The method and implementation will be explained in detail in the next section.

\begin{figure}
\includegraphics[width=1\linewidth]{images/multiomic_datasets} \caption[Multi-omic relationships.]{Multi-omic relationships on different datasets. Integration methods focus on relationships within datasets. Common relationships between datasets are used as confirmation/validation. Created with BioRender.com}\label{fig:multiomic-datasets}
\end{figure}

\hypertarget{regularized-generalized-canonical-correlation-analysis}{%
\subsection{Regularized generalized canonical correlation analysis}\label{regularized-generalized-canonical-correlation-analysis}}

To understand the regularized canonical correlation analysis we first provide a brief description of principal component analysis.

Principal component analysis defines a new orthogonal coordinate system that optimally describes variance in a single dataset.
It does so by decomposing the numerical matrix into the eigenvalues and eigenvectors with decreasing variance.
Its results include new variables (the eigenvectors) and a vector of loadings or weight indicating the importance of the original variables for these new variables.
Typically, data is normalized and standardized to avoid that a variable with higher variance and different scale dominate the results.

The canonical correlation analysis {[}\protect\hyperlink{ref-hotelling1936}{97}, \protect\hyperlink{ref-jordan1875}{162}{]} is a method to find agreement between two, or more, scorers (as it was first introduced on the literature) or sources.
It extends the PCA to a two sources of data.
It provides a similar output, new variables and weights of the existing variables indicating their importance.
The function it maximizes is:

\[
\rho = \underset{a, b}{\text{argmax}} (\text{cor}(a^T X, b^T Y))
\]

On this equation \(a\) and \(b\) are random variables that given \(X\) and \(Y\), the two data sources, maximize the correlation between \(a^T X\) and \(b^T Y\), the first canonical variables, the new variables.
\(a\) and \(b\) are also known as weights or loadings of the variables.

Over several years of progress on the field of canonical correlations {[}\protect\hyperlink{ref-tenenhaus_regularized_2011}{115}, \protect\hyperlink{ref-tenenhaus_variable_2014}{116}, \protect\hyperlink{ref-tenenhaus_component-based_2008}{163}--\protect\hyperlink{ref-gloaguen2020}{167}{]}, regularized generalized canonical correlation analysis (RGCCA) emerged with a generalization from canonical correlations extending the procedure to more than two sources of data {[}\protect\hyperlink{ref-R-RGCCA}{\textbf{R-RGCCA?}}{]} and being made more flexible generalizing from other proposed methods.

\hypertarget{description}{%
\subsubsection{Description}\label{description}}

RGCCA works with numeric matrices that can be as big as needed, as it is designed for datasets with more variables than samples ( \(p \gg n\)).
However, for each sample it needs to have a complete case with no missing values (at the time of writing this thesis there is an \href{https://github.com/rgcca-factory/RGCCA/tree/CRAN}{in-development version}, not released on CRAN\footnote{CRAN is The Comprehensive R Archive Network available at \url{https://cran.r-project.org/}} yet, that replaces any missing value by a 0) and in which time is not considered as a special variable.
It uses a dimensional reduction approach to relate the different blocks of data between them and produce specific factors for each dataset.
The objective function is:

\[
\underset{a_1,a_2, \dots,a_J}{\text{maximize}} \sum_{j, k = 1}^J c_{jk} g( \text{cov}(X_j a_j, X_k a_k)) \text{ s.t. } (1-\tau_j)var(X_j a_j)+\tau_j \Vert a_j \Vert^2 = 1, j=1, \ldots, J
\]

Being \(X_j\) the values from sample \(j\), the weights of the variables of said sample are represented by \(a\).
While \(g\) is a function that can take the form of \(x\), also known as Horst method, \(|x|\) known as centroid method, \(x^2\) known as factorial method, or any user-supplied function.
\(C\) is a symmetric matrix describing the network between blocks.

The shrinkage parameter is defined as:

\[
\widehat{\lambda}^{\star} = \dfrac{\sum_{i\neq j}\widehat{Var}(r_{ij})}{\sum_{i \neq j}r_{ij}^2}
\]

Where the \(r_{ij}\) are the correlation coefficients of the matrix between variables \(i\) and \(j\) .
Where the variance is defined as:

\[
\widehat{Var}(S_{ij}) = \dfrac{n^2}{{(n-1)}²} \widehat{Var}({w}_{ij}) = \dfrac{n}{{(n-1)}^3} \sum_{k=1}^n ( w_{kij} - \overline{w_{ij}})^2
\]

And its components are: \(w_{kij}=(x_{ki}-\overline{x}_i)(x_{kj}-\overline{x}_j)\) and \(\overline{w}_{ij}=\frac{1}{n}\sum_{k=1}^nw_{kij}\) representing \(x_{ij}\) the values of a sample \(j\) on a variable \(i\).

The authors realized that there is a special problem due to sparsity on biological data which could be handled using first another normalization to improve the stability and success of the canonical correlation methodology.
The method to perform the dimensional reduction using the sparse method consists on maximizing the same equation but with a different constraint: Specifically, RGCCA with all \(\tau_j = 1\), combined with an L1-penalty that gives rise to SGCCA:

\[
\underset{a_1,a_2, \dots,a_J}{\text{maximize}} \sum_{j, k = 1}^J c_{jk}g( \text{cov}(X_j a_j, X_k a_k)) \text{~~s.t.~~} \Vert a_j \Vert_2 = 1 \text{ and } \Vert a_j \Vert_1 \le s_j, j=1,\ldots,J
\]

The \(s_j\) controls the sparsity estimated of the data, the smaller it is, the higher the sparsity of \(X_j\) is.
As \(s_j\) is closer to 0, more features are selected as it looks to optimize covariance; while if it is closer to 1, less features are selected and the function resembles the correlation.
The values of \(s_j\) were estimated using the Schäfer method {[}\protect\hyperlink{ref-schuxe4fer2005}{168}{]} when the block included 16S data or RNAseq, otherwise 1 was used.

As previously mentioned, there are different \(g\) functions that could be used; but the centroid method was chosen to detect both positive and negative relationships.

Categorical data was encoded as binary (dummy) variables for each factor except one to keep degrees of freedom, where 0 indicates not present and 1 indicates present.
One level was omitted to avoid overfitting the data.
Each block, regardless if it had continuous numeric variables or dummy variables was standardized to zero mean and unit variance.
Later, it was divided by the square root of the number of variables of the block for an unbiased estimation.

\hypertarget{rgcca-output}{%
\subsubsection{Output}\label{rgcca-output}}

RGCCA, as other dimensional reduction techniques, provides specific weights for each variable on each dimension and a sample score on each dimension, together with quality scores.
The most important output are the canonical components of each block.
For each block there can be as many canonical components as the number of variables of a block minus 1.
These canonical components represent a block by the largest source of variation on that block that satisfies the constrains explained below.

To measure the quality of the model, the implementation provides indicators based on the Average Variance Explained (AVE).
RGCCA returns an AVE score for each block, which measures how the variables of the block correlate with the dimension component of the block.
It also provides two AVE scores of the whole model: the inner, which measures how each dimension accounts for the variance, and the outer, which measures how variables correlate with the dimension components.
The closer the inner AVE is to 1, the better the model adjusts to the data.
However, that mathematically fits better to the data does not guarantee that the model provides more insights into the biology.

It can also generate results as other related methods based on the maximization of a function of correlations: SUMCOR (sum of correlations method) {[}\protect\hyperlink{ref-horst1961}{169}{]}, SSQCOR (sum of squared correlations method) {[}\protect\hyperlink{ref-kettenring1971}{170}{]}, SABSCOR (sum of absolute values of the correlations method) {[}\protect\hyperlink{ref-vandegeer1984}{171}{]}.
Others methods are based on the maximization of a function of covariances: SUMCOV (sum of covariances method), SSQCOV (sum of squared covariances method), SABSCOV (sum of absolute values of the covariances method).
The following table summarizes the equivalent parameters needed on RGCCA to work:

\begin{longtable}[]{@{}cccc@{}}
\caption{\label{tab:RGCCA-methods} Equivalences of RGCCA to other methods}\tabularnewline
\toprule
Method & Scheme & Normalization & Shrinkage \\
\midrule
\endfirsthead
\toprule
Method & Scheme & Normalization & Shrinkage \\
\midrule
\endhead
SUMCOR & Horst & \(Var(X_j a_j) = 1\) & 0 \\
SSQCOR & Factorial & \(Var(X_j a_j) = 1\) & 0 \\
SABSCOR & Centroid & \(Var(X_j a_j) = 1\) & 0 \\
SUMCOV & Horst & \(\Vert a_j \Vert = 1\) & 1 \\
SSQCOV & Factorial & \(\Vert a_j \Vert = 1\) & 1 \\
SABSCOV & Centroid & \(\Vert a_j \Vert = 1\) & 1 \\
\bottomrule
\end{longtable}

There are other methods that can be performed with RGCCA, for example: The classical canonical correlation analysis would be equivalent to data variance equal to 1 and shrinkage of 0.
Partial least squares (PLS) regression, which maximizes covariance, would be equal to data variance of 1 and shrinkage of 1 in RGCCA.
Finally, redundancy analysis could be performed where one block's weight normalized value is 1 and the variance of the other equals to 1.

\hypertarget{models}{%
\subsubsection{Models}\label{models}}

There is no formal definition of what constitutes a block of data on multi-omics tools.
Most multi-omics and integration tools assume one block for each type of data, such as an essay a survey or an experiment.
We decided to split the block with the variables about the samples to separate independent variables.
The hypothesis we made was that more blocks with highly related variables but independent from the other blocks would fit better the data and thus help to identify causal or dependent variables.

To model what might be the relationships within datasets current practices include using a pre-selected model of relations between blocks (See figure \ref{fig:multiomic-datasets}).
However, this model might not be an accurate representation of the relations between blocks and several models might need to be fitted.
To help find the fitting model for the data we created an R package, named \href{https://llrs.github.io/inteRmodel/}{inteRmodel}, which helps finding the right model for the dataset via a bootstrapping procedure.

This method was applied to the previously described datasets to find the relationship between microorganisms and the disease.
Following this method; to provide a ground truth, a model with only the relationships between the two experimental obtained data is analyzed, on what it is called the model 0.\\
The next models analyzed consisted on relationships between the two experimental blocks and a block with all the metadata of the samples.
These models are denoted by 1.Y, where 1 denotes the family 1 and Y is used to label some of the models of this family.

Later instead of a big metadata block, following our theory we split this metadata block on several ones, having a block for time related variables, another one for location and the other about the people on the study.
This allows to design a model with an expected relationships between these blocks and makes more interpretable the relationships.
These models are from the family 2 denoted by the name 2.Y, where 2 denotes the family of the model and Y change for particular models with different relationships between the blocks change.

For each family of models we tested all possible relationships with weights between 0 and 1 by 0.1 intervals to find the best model on each datasets according to the AVE score.

The final models were further validated using a bootstrap approach to measure their accuracy and likelihood on the data available.

\hypertarget{other}{%
\subsection{Other}\label{other}}

\hypertarget{mcia}{%
\subsubsection{MCIA}\label{mcia}}

Multiple co-inertia analysis, also known as MCIA, is a method to examine covariant gene expression patterns between two blocks {[}\protect\hyperlink{ref-meng2014}{172}{]}.
It is implemented on the package \href{https://bioconductor.org/packages/omicade4}{omicade4}.
On its core MCIA maximizes the following formula:

\[
\sum_{k=1}^K w_k {cov}^2 (X_k Q_k u_k, v)
\]

where \(K\) is the total number of matrices, \(X_k\) the transformed matrices and \(Q_k\) is a square matrix with \(r_{ij}\) in diagonal elements indicating the hyperspace of features metrics, \(u_k\) are auxiliary axes, \(v\) the reference structure and w the weights of the matrices.
This can be used to obtain a dimension of \(P_k^d=u_k^d(u_k^d Q_k {u_k^d}^T)^{-1} u_k^d Q_k\) given that for each dimension the residuals are obtained following \(X_k^{d-1} = X_1^d- X_1^d P_1^{d-1}\) where \(d\) are the dimensions needed.

MCIA was used as a baseline method to compare the RGCCA integration.

\hypertarget{stategra}{%
\subsubsection{STATegRa}\label{stategra}}

We used \href{https://bioconductor.org/packages/STATegRa}{\texttt{STATegRa}} To explore how much do different blocks of a dataset have in common {[}\protect\hyperlink{ref-planell2021}{173}{]}.
It is a framework for integrating datasets with two data types using parametric and non-parametric methods.
The methods used are omics component analysis based on singular value decomposition (SVD) of the data matrix.
There are three different methods provided to this end: \emph{DISCO-SCA}, \emph{JIVE} and \emph{O2PLS}.

DISCO-SCA uses:

\[
X_k = TP_k ^ T + E_k
\]

Where \(T\) is the \(I \times R\) matrix of components scores that is shared between all blocks and \(P_k\) the \(J_k \times R\) matrix of components loadings for block \(k\).

Let \(X_1,X_2, \dots X_i\) be blocks of data and \(X=[X_1,X_2, \dots X_i]\) represent the joint data, then the JIVE decomposition is defined as:

\[X_i=J_i+A_i+\epsilon_i \text{, }i = 1,2, \dots\]

where \(J=[J_1,J_2, \dots J_i]\) is the \(p \times n\) matrix of rank \(r<rank(X)\) representing the joint structure, \(A_i\) is the \(p_i\times n\) matrix of rank \(r_i < rank(X_i)\) representing the individual structure of \(X_i\) and \(\epsilon_i\) are \(p_i \times n\) error matrices of independent entries.

Finally, the O2PLS approach uses multiple linear regression to estimate the pure constituent profiles and divides the systematic part into two, one common to both blocks and one not.
The O2PLS model can be written as a factor analysis where some factors are common between both blocks.

\[
\begin{aligned}
& \text{X model : } X = TW ^ T + T_{\text{Y-ortho}} P ^ T_{\text{Y-ortho}} + E \\
& \text{Y model : } Y = UC ^ T + U_{\text{X-ortho}} P ^ T_{\text{X-ortho}} + F \\
& \text{Inner relation : } U = T + H
\end{aligned}
\]

Each model is built similarly by adding the subtraction of the projected values of the other component keeping the relationship between them as stated on the third line.

\hypertarget{functional-enrichment-methods}{%
\section{Functional enrichment methods}\label{functional-enrichment-methods}}

\hypertarget{ORA}{%
\subsection{Over representation analysis}\label{ORA}}

Functional enrichment methods are those methods that aim to provide with more information about the variables besides their numerical value measured.
They can be very different in nature but they all use the numeric values of the variables and other information, being it from the same experiment data collection or from external data sources.

Many functional enrichment methods are based on an over representation analysis, where a group of elements is tested for their mesure in other groups.
This can be done with \href{https://bioconductor.org/packages/clusterProfiler}{\texttt{clusterProfiler}} which tests genes enrichment for functionality based on information on pathway databases {[}\protect\hyperlink{ref-wu2021}{174}{]} that it is used in several publications {[}\protect\hyperlink{ref-richter2021}{175}{]}.
\texttt{clusterProfiler} checks the enrichment of features of a given group on the (background) list provided.

\[
\begin{aligned}
H_0 : P_{subset} \leq  P_{overall} \\
H_1 : P_{subset} > P_{overall}
\end{aligned}
\]

The statistical test used is usually the fisher test, the hypergeometric test or the proportion test.
We describe the hypergeometric test and the proportion test below.

\hypertarget{fisher-test}{%
\subsubsection{Fisher test}\label{fisher-test}}

The fisher test calculates how independently are two categories.
Say if genes in category X are independent of category Y, if we use the contingency table

\begin{longtable}[]{@{}lccc@{}}
\caption{\label{tab:fisher} Fisher contingency table}\tabularnewline
\toprule
& In category X & Not in category X & Row total \\
\midrule
\endfirsthead
\toprule
& In category X & Not in category X & Row total \\
\midrule
\endhead
\textbf{In category Y} & \(a\) & \(b\) & \(a+b\) \\
\textbf{Not in category Y} & \(c\) & \(d\) & \(c+d\) \\
\textbf{Column total} & \(a+c\) & \(b+d\) & \(a+b+c+d = n\) \\
\bottomrule
\end{longtable}

Given this contingency table \ref{tab:fisher}, the probability of X and Y being independent can be calculated with:

\[
\begin{aligned}
P & = \dfrac{ \binom{a+b}{a} \binom{c+d}{c} }{\binom{n}{a+c}} \\
& = \frac{(a+b)!~(c+d)!~(a+c)!~(b+d)!}{a!~~b!~~c!~~d!~~n!}
\end{aligned}
\]

\hypertarget{hypergeometric-test}{%
\subsubsection{Hypergeometric test}\label{hypergeometric-test}}

The hypergeometric distribution describes the probability of \(k\) successes (when the object drawn has a specified feature) in \(n\) draws\footnote{without replacement}, from a finite population of size \(N\) that contains exactly \(K\) objects with that feature, wherein each draw is either a success or a failure.

In this context \(N\) is the number of genes being used and \(n\) the number of genes on a pathway.
It can be used to compare the genes found on a pathway (\(k\) genes) compared to the expected \(K\) numbers of the distribution using the following equation:

\[
P_X(k) = P(X = k) 
= \frac{\binom{K}{k} \binom{N - K}{n-k}}{\binom{N}{n}}
\]

This looks like the fisher test, because hypergeometric test assesses the extremeness of observing \(k\) or more of that overlap (\(a\) on ) and thus it is the same as a one-sided Fisher's exact test.

\hypertarget{proportion-test}{%
\subsubsection{Proportion test}\label{proportion-test}}

The overrepresentation of a given group of elements can also be tested with the proportion test, which is sometimes also used on clusterProfiler.
The proportion test uses the \(\chi^2\) distribution to test if the observed frequency (\(O_i\)) is close to the expected frequency (\(E_i\)):

\[
\chi^2 = \sum_{i =1}^n \dfrac{(O_i - E_i)^2}{E_i} \simeq \chi_{n-p}^2
\]

As this is usually done on a 2x2 contingency table it is equivalent to the Z-test of proportion.
Sometimes, the expected frequency is so low that a correction must be done to the estimation:

\[
\chi_{\text{Yates}}^2 = \sum_{i =1}^n \dfrac{(|O_i - E_i | - 0.5)^2}{E_i}
\]

This increases the \(p\)-value as it raises the Chi-square statistic.

\hypertarget{gene-set-enrichment-analysis}{%
\subsection{Gene Set Enrichment Analysis}\label{gene-set-enrichment-analysis}}

There are other methods that test if some variables show an unexpected importance according to a statistic like fold change or value, such as gene set enrichment analysis (GSEA) {[}\protect\hyperlink{ref-subramanian2005}{176}{]}.
GSEA is a computational method originally developed to determine whether a priori defined set of genes shows statistically significant and concordant differences between two biological states.
This methods check if a group of variables present in an ordered list shows a skewed distribution and it compares it against a random group of similar size.
It has been widely used since its original publication, also in IBD {[}\protect\hyperlink{ref-protiva2016}{177}{]}.

This method calculates the rank of genes \(rank(g_j)=r_j\) where each \(g\) is a gene, and then it calculates the following functions:

\[
\begin{aligned}
& P_{hit}(S, i) = \sum_{g_j \in S, j \leq i}\dfrac{|r_j|^p}{N_R} \text{, where } N_R = \sum_{g_j \in S}|r_j|^p \\
& P_{miss}(S, i) = \sum_{g_j \not \in S, j \leq i}\dfrac{1}{N - N_H}
\end{aligned}
\]

With these values the enrichment score (ES) defined as: \(ES=max(|P_{hit}(S, i)-P_{miss}(S, i)\vert)\) is calculated from the walk.
At least 1000 permutations are usually used but a high number of permutations are required for an accurate estimation of the enrichment score.
However, when more than one pathway (\(S\)) is evaluated in order to compare between their enrichment scores, they must be normalized by dividing the scores by the mean of all the ES.
When power \(p\) is 0 it is equivalent to the standard Kolmogorov--Smirnov statistic, though it is usually set to 1.

For testing GSEA we used \href{https://bioconductor.org/packages/fgsea}{\texttt{fgsea}} {[}\protect\hyperlink{ref-korotkevich2021}{178}{]} implementation for its speed and integration with other methods used in this thesis.
Gene pathways from the REACTOME database were tested on the weight of different models or on the comparisons performed {[}\protect\hyperlink{ref-fabregat2016}{179}{]}.

\hypertarget{gsva}{%
\subsubsection{GSVA}\label{gsva}}

To estimate the expression of the pathways and compare their expression levels between conditions gene set variation analysis as implemented on \href{https://bioconductor.org/packages/GSVA}{GSVA} was used {[}\protect\hyperlink{ref-huxe4nzelmann2013}{180}{]}.
It is a method that summarizes the variables' numerical value changing the space of \(\text{variable x sample}\) to \(\text{group x sample}\) .
This enables other methods to use this new space instead of the original variables, which provides a successful way to look into data {[}\protect\hyperlink{ref-escudero-hernuxe1ndez2021}{181}{]}.
GSVA was used (again from the REACTOME database) to find the relationships between the pathways and the microbiome at different taxonomic levels.

This is done via an estimation and a comparison with a discrete Poisson kernel: \(i\) indicates the gene from a total of \(k\) genes, and samples are indicated by \(j\) from \(n\) number of samples.

\[
z_{ij}=\hat{F_r}(x_{ij}) = \frac{1}{n}\sum_{k=1}^n\sum_{y=0}^{x_{ij}}\dfrac{e^{-(x_{ik}+r)}(x_{ik} + r)^y }{y!}
\]

\(r = 0.5\) is used to set the mode of the Poisson kernel at each \(x_{ik}\), that is, similar to the expression of a gene for a given sample.

Later this is converted to ranks \(z_{(i)j}\) for each sample and normalized: \(r_{ij}=|\frac{p}{2}-z_{(i)j}|\) to make the distribution of ranks symmetric around zero to later compare with a normal distribution using a Kolmogorov-Smirnov-like random walk statistic:

\[
v_{jk}(l)=\dfrac{\sum_{i=1}^l|r_{ij}|^{\tau}I(g_{(i)}\in \gamma_k)}{\sum_{i=1}^p|r_{ij}|^{\tau}I(g_{(i)} \in \gamma_k)}
- \dfrac{\sum_{i=1}^lI(g_{(i)} \not \in\gamma_k)}{p-|\gamma_k|}
\]

Here \(\tau\) describes the weight of the tail in the random walk (default is set to 1).
\(\gamma_k\) is the k-th gene set and \(I(g_{(i)}\in \gamma_k)\) is the indicator function whether the gene ranked i-th belongs to the gene set \(\gamma_k\) .
\(|\gamma_k|\) indicates the ordination of the gene set, the number of genes of the gene set and \(p\) the number of genes in the dataset.

This difference is later converted to enrichment score for each gene set for each sample, similar to GSEA.
This score can be calculated as a difference of hits and misses or the maximum deviation from zero of the random walk (which allows to detect gene sets that have genes with different opposing expression patterns).

\hypertarget{variance}{%
\section{Variance and diversity methods}\label{variance}}

\hypertarget{permanova}{%
\subsection{PERMANOVA}\label{permanova}}

The PERMANOVA method {[}\protect\hyperlink{ref-anderson2001}{182}, \protect\hyperlink{ref-warton2012}{183}{]}, provided by the \href{https://cran.r-project.org/package=vegan}{vegan} package on the \texttt{adonis} function, was used to test if microbiome data variance is due to other variables when using distances metrics.
It uses the residual sum of squares such as:

\[
SS_W = \frac{1}{n} \sum_{i=1}^{N-1}\sum_{j=i+1}^N d_{ij}^2\epsilon{ij} 
\]

When using euclidian distances (\(d\)) it is equivalent to MANOVA.
Here \(\epsilon_{ij}\) takes the value of 1 if the observation \(i\) and the observation \(j\) are in the same group, otherwise it takes the value of 0.
This can be later used to test which variance is bigger, inter-groups or intra-groups by using the following formula:

\[
F = \dfrac{SS_A/(\alpha -1)}{SS_W/(N-\alpha)}
\]

Where \(SS_A\) is the among group sum of squares, representing the intra-group variance.
\(N\) is the number of samples and \(\alpha\) the number of different groups.

This allows to test if the variables are related to the variance of the data as it can be compared with the \(F\) statistic after a high number of permutations.

\hypertarget{globaltest}{%
\subsection{globaltest}\label{globaltest}}

It is a method for testing complex hypothesis and calculate the influence of each variable on a given outcome {[}\protect\hyperlink{ref-goeman2006}{184}{]}.
We tested which variables, (sex, age, location, time since diagnosis, treatment) are important on the datasets with \href{https://bioconductor.org/packages/globaltest}{globaltest} (Version 5.40 or later).
This method provides a general statistic to test hypothesis against a high dimensional dataset.

\[
S = \sum_{i=1}^p x_i^{'} x_i g(t_i^2)
\]

The global test performs a test statistic on the transformed t-test, where if \(p\), the number of variables, is large the test is more powerful on average over all possible sparse alternatives of general functions \(g\).

It was performed with variables individually and also with interactions between the different variables.

\hypertarget{diversity-indices}{%
\subsection{Diversity indices}\label{diversity-indices}}

Microbiome diversity was measured using vegan and \href{https://bioconductor.org/packages/phyloseq}{phyloseq} methods {[}\protect\hyperlink{ref-oksanen2020}{185}{]}.
\(\alpha\)-diversity is a measure of how much a given microbiome at a taxonomic level is present on a sample.
Several measures exists, on the thesis we used the effective Simpson or effective Shannon diversity index to compare diversity between samples and conditions.

The effective Simpson (also known as inverse Simpson) and the effective Shannon are:

\[
\begin{aligned}
& D_{\text{effective Simpson}} = \dfrac{1}{\sum_{1=1}^Sp_i^2} \\
& D_{\text{effective Shannon}} = \dfrac{1}{-\sum_{i =1}^S p_i \log_e{p_i}}
\end{aligned}
\]

Where \(p_i\) is the proportion of species \(i\) and \(S\) is the number of species.

\(\beta\)-diversity is a diversity index that compares how similar are two samples.
It was calculated using the phyloseq package for exploratory analysis.

\hypertarget{other-methods}{%
\section{Other methods}\label{other-methods}}

\hypertarget{statistics}{%
\subsection{Statistics}\label{statistics}}

Differential expression analysis was performed with the limma-trend method {[}\protect\hyperlink{ref-ritchie2015}{186}, \protect\hyperlink{ref-law2014}{187}{]} and edgeR {[}\protect\hyperlink{ref-mccarthy2012}{145}{]} (Bioconductor version 3.10 or superior) packages.
Data was normalized using the trimmed mean of M-values and log-2 transformed into counts per millions following the workflow previously described using voom {[}\protect\hyperlink{ref-law2018}{188}{]}.
This approach assumes tests for each gene that:

\[
H_0 : \mu_{X} = \mu_{Y}
H_1 : \mu_{X} \neq \mu_{Y}
\]

Limma assumes that \(E(y_g) = X \alpha_g\), and \(var(y_g) = W_g \sigma_g^2\) where X is a design matrix and \(\alpha_g\) is an unknown coefficient vector and \(\sigma_g^2\) is the gene-variance and \(W_g\) is a known non-negative definite weight matrix.
The design matrix of limma can be combined with contrast estimators defined such that \(\beta_g = C^T \alpha_g\) where \(C^T\) is a constant vector typically defining how a given variable is related to a sample.

These statistics are later estimated via bayesian method whose prior is defined as:
\[
\dfrac{1}{\sigma_g^2} \sim \dfrac{1}{d_0 S_0^2} \chi_{d_0}^2
\]

This results on a moderated t-test:

\[
\widetilde t_{gj} = \dfrac{\hat \beta_{gj}}{\tilde S_g \sqrt{v_{gj}}} \sim t_{d_g + d_0}
\]

Were \(\hat \beta_g\) can be interpreted as the difference of the effect of two factors.

As gene comparisons are done along many genes there is an increase probability to have a type I error.
Multiple testing correction method have been designed to correct this kind of family-wise comparisons.
To correct for multiple testing, the false discovery rate was estimated using the method of Benjamini and Hochberg {[}\protect\hyperlink{ref-yoavbenjamini}{189}{]}.
A gene was considered differentially expressed when it was significant at 5\% FDR.
Special attention was given to those genes that showed a fold-change higher than \textbar1.5\textbar.

\hypertarget{wgcna}{%
\subsection{WGCNA}\label{wgcna}}

To look for relationships between the microbiome and the RNAseq we used weighted gene co-expression network analysis.
We used weighted gene co-expression network analysis as implemented on \href{https://cran.r-project.org/package=WGCNA}{WGCNA} {[}\protect\hyperlink{ref-langfelder2008}{190}{]} as well as correlations.
The Spearman rank correlation coefficient is:

\[
R_s(X,Y) = \dfrac{\sum_{i=1}^n (R_i - \bar{R}) (S_i - \bar{S} )}{\sqrt{\sum_{i=1}^n (R_i - \bar{R})^2}\sqrt{\sum_{i=1}^n (S_i - \bar{S})^2}}
\]

Being \((X_1 , Y_1 ),\dots, (X_n , Y_n)\), assign a rank where \((R_1 , S_1 ), \dots , (R_n , S_n )\) for \(n\) being all the variables where \(\bar{R}=\dfrac{1}{n}\sum_{i=1}^n R_i\) and likewise \(\bar{S}=\dfrac{1}{n}\sum_{i=1}^n S_i\).
The distribution of the Spearman correlation coefficient is symmetric around 0 and can be approximated to a normal distribution as \(\sqrt{n-1}R_{s(X,Y)} \sim N(0,1)\) which can be used to calculate the p-value of a given estimation.

\hypertarget{baseset}{%
\subsection{BaseSet}\label{baseset}}

\href{https://cran.r-project.org/package=BaseSet}{BaseSet} was developed and used to find which variables are really involved on the interaction and how likely they were to be together.
It is a package that uses fuzzy set logic to calculate the probability to belong to a group, in this case, those genes and bacteria selected by the model that interact with the other.

Under the standard fuzzy set logic a set \(S\) is a group of elements for which each element \(e\) has an \(\alpha\) membership to that set {[}\protect\hyperlink{ref-filzmoser2004}{191}, \protect\hyperlink{ref-dubois1993}{192}{]}.
\(\alpha\) is usually bounded between 0 and 1: \(\alpha \in [0, 1]\) .
A given element \(e\) can belong to more than one set.
Assimilating the membership function to probability we can calculate the probability of a given element \(e\) to belong to a set \(S\) and not any other set: \(P(e \in S|e \not \in S^c)\).
Which applied to the data and the case at hand, it is the probability that a given variable is associated with a given outcome and not with any other outcome.

The membership function was derived from the bootstraps used for each model on the thousand iterations of the integrative method applied to give an estimation of how probable is a given gene and bacteria to be selected as relevant for the model.
The bootstraps of the models where used to calculate the probability of a variable to be selected by RGCCA.
This probability was used to calculate (via \texttt{set\_size}) the genes and bacteria that are specific of the model that allows to separate the transcriptome by its location and the microbiome by the disease status.

\hypertarget{experdesign}{%
\subsection{experDesign}\label{experdesign}}

\href{https://cran.r-project.org/package=experDesign}{experDesign} was developed {[}\protect\hyperlink{ref-revillasancho2021}{193}{]} to prevent and quantify if a given experiment has batch effect due to the batches used to measure the values or other known variable.
It might help to detect a bad design of the experiment.

On pseudo code the core of the program can be described as:

\hypertarget{pseudocode-experDesign}{%
\label{pseudocode-experDesign}}%
\begin{verbatim}
for each index:

    for each batch

        pick size(batch) samples

        if samples are in another batch

            pick other samples

    for each batch

        calculate some summary statistics

        compare with the summary statistics of all the samples

keep the index with less differences between the index and all the samples
\end{verbatim}

Summary statistics taken into account are the median, the variance, the range, the number of missing values, and the entropy of the categorical variables.
It can take into account spatial distribution and, given the number of samples that fit on a batch, provide which technical replicates\footnote{Technical replicates are samples that are measured multiple times to ensure the accuracy of the measurement.
  In contrast biological replicates are sample with the same conditions but different individuals or process before the measurement.} are best to use.

\hypertarget{ROC}{%
\subsection{ROC- AUC}\label{ROC}}

To estimate if the selected features (genes or microorganism) by the integration methods have some biological meaningful contribution we measured if they can classify features, such as, which gastrointestinal segment is each sample from, or which type of disease does each patient have.

To compare between different models the area under the curve (AUC) of the receiver operating characteristic (ROC) was calculated with the \href{https://cran.r-project.org/package=pROC}{pROC} package {[}\protect\hyperlink{ref-robin2011}{194}{]}.
It is based on the following formulas, where \(\text{FP}\) is false positive, \(N\) is a negative, \(P\) is positive, \(\text{TP}\) is true positive and \(\text{FN}\) is false negative:

\[
\begin{aligned}
FPR = \frac{FP}{N} = \dfrac{FP}{FP+TN} \\
TPR = \frac{TP}{P} = \dfrac{FP}{FP+FN} 
\end{aligned}
\]

The ROC curve is that where the true positive rate (TPR) or sensitivity, recall or hit rate is represented against the false positive rate (FPR) on the abscissa.
The area under this curve is a measure of how good such classifications performs overall, being 0.5 as good as a random selection.
The closer it is to 1 the better as it classifies incorrectly less samples and accurately classify more.

\hypertarget{results}{%
\chapter{Results}\label{results}}

\hypertarget{packagesmethods}{%
\section{Packages/methods}\label{packagesmethods}}

\hypertarget{experdesign-1}{%
\subsection{experDesign}\label{experdesign-1}}

\texttt{experDesign} package built in R was released for the first time on CRAN on 2020-09-08 after nearly a year after the initial release made on \href{https://github.com/llrs/experDesign}{github}.
After peer review it was published on a journal on 2021-11 {[}\protect\hyperlink{ref-revillasancho2021}{193}{]}.

The package uses functional programming to create and modify objects and the features used.
The package bases its performance on the large body of work made by the R core team.
It adds the information to the introduced \texttt{data.frame} or returns an vector with the appropriate information.

\texttt{experDesign} functions are divided into several categories:

\begin{itemize}
\item
  Helper functions to aid on deciding how many batches are or how many samples per batch.
  There are some also that report how good a given distribution of the samples felt for a given dataset.
\item
  Functions generating indexes.
\item
  Functions distributing the samples on indexes
\end{itemize}

\begin{figure}
\includegraphics[width=1\linewidth]{images/experDesign-flowchart} \caption[experDesign functions and workflow]{`experDesign` functions and workflow. Workflow for users of the package showing which functions can be used depending on the experiment design they have.}\label{fig:experdesign-flowchart}
\end{figure}

Regarding time related variables \texttt{experDesign} will use them as factors, while issuing a warning to the user.

Since its development it has been used on a couple of RNA sequencing experiments that required a batch design, one of organoids bulk RNA-seq (data not related to this thesis) and another one of biopsies bulk RNA-seq from the BARCELONA cohort (See appendix section \ref{barcelona}).
It was also used to check if there is any observable batch effect on the datasets analyzed.

On the designed datasets \texttt{experDesign} avoided batch effects from the sequencing process.
However, on the organoids dataset, a change on the matrigel used to produce them introduced a batch effect that made it impossible to compare samples before and after that change (there were not any shared sample before and after the change of matrigel).
On the BARCELONA cohort there were other problems described on the appendices (section \ref{barcelona}).

Since its release on CRAN it has had a median of \textasciitilde400 downloads each month from RStudio repository mirror, showing the interest the community have on solutions like this.

\hypertarget{baseset-1}{%
\subsection{BaseSet}\label{baseset-1}}

\texttt{BaseSet} package, built in R, was released for the first time on CRAN on 2020-11-11, nearly two years after the initial work started on \href{https://github.com/llrs/BaseSet}{github}.

The package uses both functional programming and object oriented program to create and modify the TidySet S4 object defined\footnote{S4 is one of the object programming paradigms on R.
  For a more complete overview and differences see \href{https://adv-r.hadley.nz/oo.html}{Advanced R} {[}\protect\hyperlink{ref-wickham2019}{195}{]}.}.
Mixing it with S3 generic functions it provides a powerful interface compatible with the tidyverse principles, a \href{https://www.tidyverse.org/}{group of packages} following the same design.
The package provides a new class to handle fuzzy sets and the associate information.

\texttt{BaseSet} methods are divided into several categories:

\begin{itemize}
\item
  \href{https://docs.ropensci.org/BaseSet/reference/index.html\#general}{General functions} to create sets of the TidySet class or convert from it to a list or about the package.
\item
  \href{https://docs.ropensci.org/BaseSet/reference/index.html\#set-operations}{Set operations} like adjacency cartesian product, cardinality, complement, incidence, independence, intersection, union, subtract, power set or size.
\item
  \href{https://docs.ropensci.org/BaseSet/reference/index.html\#set-operations}{Functions to work with TidySets} to add relationships, sets, elements or some complimentary data about them.
  Remove the same or simply move around data or calculate the number of elements, relations and sets.
\item
  \href{https://docs.ropensci.org/BaseSet/reference/index.html\#reading-files}{Functions to read} files from formats where sets are usually stored in the bioinformatician field: GAF, GMT and OBO formats.
\item
  Last, some \href{https://docs.ropensci.org/BaseSet/reference/index.html\#utiles}{utility functions} to use set name conventions and other auxiliary functions.
\end{itemize}

The package had a long development process with initial iterations based on GSEABase package which was later abandoned (\href{https://github.com/llrs/GSEAdv}{GSEAdv}) to also include some uncertainty on the relationship of a gene with a given gene set.

The package formed part of an exploration of the Bioconductor community (project to develop, support, and disseminate free open source software that facilitates rigorous and reproducible analysis of data from current and emerging biological assay) for more modern and faster handling of sets than \href{https://bioconductor.org/packages/GSEABase}{GSEABase}.
There were three different packages created as part of this process, \texttt{BaseSet}, \texttt{BiocSet} released \href{https://bioconductor.org/packages/BiocSet/}{on Bioconductor} and \texttt{unisets}, available \href{https://github.com/kevinrue/unisets}{on github}.
The three different approaches were presented at a birds of feather on BioC2019, the annual conference of Bioconductor on 2019.

The package passed the review on the rOpenSci organization (\href{https://github.com/ropensci/software-review/issues/359}{See review}) and is now part of the packages hosted there too.

Since its release on CRAN it has had a median close to \textasciitilde400 downloads each month from RStudio package manager.

\hypertarget{intermodel}{%
\subsection{inteRmodel}\label{intermodel}}

The package was build once the method used to find accurate models of the relationships of the data available of a dataset using RGCCA was established.
Using the package \href{https://github.com/llrs/intermodel}{on github} simplifies the process and makes easier to redo the model optimization used on this thesis.

The package has functions that can be grouped in three categories:

\begin{itemize}
\item
  Look for models and evaluate them:
  To search for a model given some conditions, such as that all the blocks are connected, and check the models via bootstrapping or leave one out procedures.
\item
  Reporting: To make better reports by improving handling of names or simplifying the objects or how to calculate scores.
\item
  Building: To easier build correct models on RGCCA, simplifying the process to create a symmetric matrix.
\end{itemize}

Currently it is only \href{https:intermodel.llrs.dev}{available on github}, so the number of downloads and usage is unknown but since its release a user has contacted to keep it up to date with development versions of RGCCA.
Currently, it is compatible with the next release of \href{https://github.com/rgcca-factory/RGCCA/tree/CRAN}{RGCCA being prepared}\footnote{We also contributed with some comments and feedback to the package to make it easier to read the source and check the inputs and improve the documentation so that it is coherent with the code and previous results of the functions.}.

The functions \texttt{analyze} helps to analyze code of a single integration, providing the results on a tidy format for further processing.
To create the connections between blocks the function \texttt{weight\_design} is available.
It creates all the possible matrices given a number of blocks and a number of weights.
Optionally it can create just a subset of those based on a numeric vector.
However, it does not provide a way to have the design named.\\
If the user wants to create their own design matrices, they can use \texttt{symm} and modify the design of the model with \texttt{subSymm}.
\texttt{symm}, takes an initial matrix to pick up the row and column names.
It is recommended that the user checks the design matrix is fully connected, which the package facilitates with the function \texttt{correct}.
This is also recommended even if the design matrices are created with \texttt{weight\_design}.

To search models \texttt{search\_models} starts with a initial connectivity of the blocks and creates all the combinations of connections given.

For the bootstrap procedure there is the function \texttt{boot\_index} to create the bootstrapped index of samples to be used by \texttt{boot\_index\_sgcca}.
\texttt{boot\_index} randomly selects as much samples as specified by the arguments to create as much indices as the required by the second argument.
If the bootstrapped samples used is not important one can use \texttt{boot\_samples\_sgcca}.
If the users want to perform a leave-one-out procedure they can use \texttt{looIndex}.

\begin{figure}
\includegraphics[width=1\linewidth]{images/inteRmodel-workflows} \caption[inteRmodel functions and workflow.]{`inteRmodel` functions and workflow. Functions provided by the inteRmodel package to search and validate models of relationships using RGCCA. Created with BioRender.com}\label{fig:intermodel-workflows}
\end{figure}

For more information, you can access the \href{https://llrs.github.io/inteRmodel/}{manual online} or once it is installed.

The package cannot choose which variables use from the block with information to split into several blocks.
However, it provides the \texttt{model\_RGCCA} function to make it easy to prepare such variables for RGCCA input.
The inteRmodel procedure will only be useful if the user should decide which variables are independent from others and split them into different blocks.
To asses this the user can use the methods we used, as described on the above section \ref{variance}.
It is important to keep in mind the possible causal relationships on user's data {[}\protect\hyperlink{ref-greenland2002}{196}{]}.

\hypertarget{analysis}{%
\section{Analysis}\label{analysis}}

On the following sections the main results of analyzing each dataset are presented.

\hypertarget{results-puget}{%
\subsection{Puget's dataset}\label{results-puget}}

On this dataset the different parameters and capabilities of RGCCA were tested.

The three different methods, centroid, factorial or horst were tested and compared.
The main result of this comparison was that the differences of the selection of the variables mattered more than the number of variables selected with each method.
The models were tested with different weights on all three schemes: horst, centroid and factorial.
The horst and the centroid scheme were similar while the factorial resulted in the most different AVE values (see \href{https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0246367\#pone.0246367.s001}{S1 Data} of {[}\protect\hyperlink{ref-revilla2021}{197}{]}).
The centroid scheme was selected because it takes into account all the relationship regardless of the sign of the canonical correlation between the blocks.
It is similarity to horst scheme.

The effect of the sparsity value was measured by its effect on the inner AVE scores and the combination of the different values for each block as can be seen on Figure \ref{fig:puget-tau2AVE}.

\begin{figure}
\includegraphics[width=1\linewidth]{images/pugets_tau_to_AVE} \caption[Effect of tau on the inner AVE on Puget's dataset.]{Effect of tau on the inner AVE on Puget's dataset. The suggested tau value is the column between the regular grid, on the ordinate axis the y's tau values and on the abscissa the gene expression (GE) on the left and the comparative genomic hybridization (CGH) on the right. The highest inner AVE is with high tau values for y and middle to upper values for GE and CGH.}\label{fig:puget-tau2AVE}
\end{figure}

The first model of the family of models 1 can be seen on Table \ref{tab:puget-model1}:

\begin{table}[H]

\caption[Model 1 for the Puget's dataset.]{\label{tab:puget-model1}Model 1 for Puget's dataset. Relationships between the different blocks in the Puget dataset for model 1. 0 indicates no relationship and 1 indicates a strong relationship.}
\centering
\begin{tabular}[t]{|>{}c|c|c|>{}c|}
\hline
\em{\textbf{Model 1}} & \em{\textbf{GE}} & \em{\textbf{CGH}} & \em{\textbf{Localization}}\\
\hline
\textbf{\cellcolor{gray!6}{GE}} & \cellcolor{gray!6}{0} & \cellcolor{gray!6}{0} & \cellcolor{gray!6}{1}\\
\hline
\textbf{CGH} & 0 & 0 & 1\\
\hline
\textbf{\cellcolor{gray!6}{Localization}} & \cellcolor{gray!6}{1} & \cellcolor{gray!6}{1} & \cellcolor{gray!6}{0}\\
\hline
\end{tabular}
\end{table}

When looking for the model that adjust better following this structure we arrived to model 1.2, described below (Table \ref{tab:puget-model1-2}) :

\begin{table}[H]

\caption[Model 1.2 for the Puget's dataset.]{\label{tab:puget-model1-2}Model 1.2 for Puget's dataset. Relationships between the different blocks in the Puget dataset for model 1. 0 indicates no relationship and 1 indicates a strong relationship.}
\centering
\begin{tabular}[t]{|>{}c|c|c|>{}c|}
\hline
\em{\textbf{Model.1.2}} & \em{\textbf{GE}} & \em{\textbf{CGH}} & \em{\textbf{Localization}}\\
\hline
\textbf{\cellcolor{gray!6}{GE}} & \cellcolor{gray!6}{0} & \cellcolor{gray!6}{0.0} & \cellcolor{gray!6}{1.0}\\
\hline
\textbf{CGH} & 0 & 0.0 & 0.1\\
\hline
\textbf{\cellcolor{gray!6}{Localization}} & \cellcolor{gray!6}{1} & \cellcolor{gray!6}{0.1} & \cellcolor{gray!6}{0.0}\\
\hline
\end{tabular}
\end{table}

On model 2 we split the invariable variables from those related to the location (Table \ref{tab:puget-model2}):

\begin{table}[H]

\caption[Model 2 for the Puget's dataset.]{\label{tab:puget-model2}Model 2 for Puget's dataset. Relationships between the different blocks in the Puget dataset for model 1. 0 indicates no relationship and 1 indicates a strong relationship.}
\centering
\begin{tabular}[t]{|>{}c|c|c|c|>{}c|}
\hline
\em{\textbf{Model.2}} & \em{\textbf{GE}} & \em{\textbf{CGH}} & \em{\textbf{Invariable}} & \em{\textbf{Localization}}\\
\hline
\textbf{\cellcolor{gray!6}{GE}} & \cellcolor{gray!6}{0} & \cellcolor{gray!6}{1} & \cellcolor{gray!6}{1} & \cellcolor{gray!6}{1}\\
\hline
\textbf{CGH} & 1 & 0 & 1 & 1\\
\hline
\textbf{\cellcolor{gray!6}{Invariable}} & \cellcolor{gray!6}{1} & \cellcolor{gray!6}{1} & \cellcolor{gray!6}{0} & \cellcolor{gray!6}{0}\\
\hline
\textbf{Localization} & 1 & 1 & 0 & 0\\
\hline
\end{tabular}
\end{table}

Following this split, the model that has higher inner AVE for these blocks is the following (Table \ref{tab:puget-model2-2}):

\begin{table}[H]

\caption[Model 2.2 the for Puget's dataset.]{\label{tab:puget-model2-2}Model 2.2 for Puget's dataset. Relationships between the different blocks in the Puget dataset for model 1. 0 indicates no relationship and 1 indicates a strong relationship.}
\centering
\begin{tabular}[t]{|>{}c|c|c|c|>{}c|}
\hline
\em{\textbf{Model.2.2}} & \em{\textbf{GE}} & \em{\textbf{CGH}} & \em{\textbf{Invariable}} & \em{\textbf{Localization}}\\
\hline
\textbf{\cellcolor{gray!6}{GE}} & \cellcolor{gray!6}{1} & \cellcolor{gray!6}{1/3} & \cellcolor{gray!6}{0} & \cellcolor{gray!6}{1}\\
\hline
\textbf{CGH} & 1/3 & 0 & 1/3 & 0\\
\hline
\textbf{\cellcolor{gray!6}{Invariable}} & \cellcolor{gray!6}{0} & \cellcolor{gray!6}{1/3} & \cellcolor{gray!6}{0} & \cellcolor{gray!6}{0}\\
\hline
\textbf{Localization} & 1 & 0 & 0 & 0\\
\hline
\end{tabular}
\end{table}

If we added a superblock with all the data of the different blocks from model 1 we started with the standard relationship between blocks (Table \ref{tab:puget-model-superblock}):

\begin{table}[H]

\caption[Model with superblock for Puget's dataset.]{\label{tab:puget-model-superblock}Relationships between the different blocks in the Puget's dataset for model superblock. 0 indicates no relationship and 1 indicates a strong relationship.}
\centering
\begin{tabular}[t]{|>{}c|c|c|c|>{}c|}
\hline
\em{\textbf{Model.superblock}} & \em{\textbf{GE}} & \em{\textbf{CGH}} & \em{\textbf{Superblock}} & \em{\textbf{Localization}}\\
\hline
\textbf{\cellcolor{gray!6}{GE}} & \cellcolor{gray!6}{0} & \cellcolor{gray!6}{0} & \cellcolor{gray!6}{1} & \cellcolor{gray!6}{0}\\
\hline
\textbf{CGH} & 0 & 0 & 1 & 0\\
\hline
\textbf{\cellcolor{gray!6}{Superblock}} & \cellcolor{gray!6}{1} & \cellcolor{gray!6}{1} & \cellcolor{gray!6}{0} & \cellcolor{gray!6}{1}\\
\hline
\textbf{Localization} & 0 & 0 & 1 & 0\\
\hline
\end{tabular}
\end{table}

But when the best model with the superblock that had highest inner AVE is quite different (Table \ref{tab:puget-model-superblock-2}):

\begin{table}[H]

\caption[Model with superblock.2 for the Puget's dataset.]{\label{tab:puget-model-superblock-2}Relationships between the different blocks in the Puget's dataset for model superblock.2. 0 indicates no relationship and 1 indicates a strong relationship.}
\centering
\begin{tabular}[t]{|>{}c|c|c|c|>{}c|}
\hline
\em{\textbf{Model.superblock}} & \em{\textbf{GE}} & \em{\textbf{CGH}} & \em{\textbf{Superblock}} & \em{\textbf{Localization}}\\
\hline
\textbf{\cellcolor{gray!6}{GE}} & \cellcolor{gray!6}{1} & \cellcolor{gray!6}{1/3} & \cellcolor{gray!6}{0} & \cellcolor{gray!6}{1}\\
\hline
\textbf{CGH} & 1/3 & 0 & 1 & 0\\
\hline
\textbf{\cellcolor{gray!6}{Superblock}} & \cellcolor{gray!6}{0} & \cellcolor{gray!6}{1} & \cellcolor{gray!6}{0} & \cellcolor{gray!6}{0}\\
\hline
\textbf{Localization} & 1 & 0 & 1 & 0\\
\hline
\end{tabular}
\end{table}

Exploratory analysis with the superblock model was done.
The first two components of the superblock did not help to explain the biology or classify the tumors (See \ref{fig:puget-superblock}):

\begin{figure}
\includegraphics[width=1\linewidth]{images/pugets_superblock} \caption[Superblock components on Puget's dataset]{First components of the superblock which has all the data of the samples on the Puget's dataset.}\label{fig:puget-superblock}
\end{figure}

The same data was used to look for a good model from the data itself including a model with a superblock but looking at the first component of the CGH and transcriptome block.
This allowed to visually inspect if each model's components helped to classify the samples (Figure \ref{fig:puget-models}):

\begin{figure}
\includegraphics[width=1\linewidth]{images/pugets_models} \caption[Different RGCCA models in the Puget's dataset.]{Different RGCCA models in the Puget's dataset. The different models with the same data showing the sample position on the first components of the CGH  and the transcriptome block. Model 1 and 1.2 with transcriptomics, CGH data and all the data about the samples together. Model 2 and 2.2 with transcriptomics, CGH data and all the data bout the samples on different blocks. Model superblock and superblock.2 have all the data in different blokcs and one block with all the data.}\label{fig:puget-models}
\end{figure}

The first components of the CGH and the transcriptomics blocks of the superblock.2 model show better classification than that of the superblock.
However, the other models show a better classification of the samples with much simpler models.

To find these models the three blocks with the best tau and the centroid scheme were analyzed by changing the weights between 0 and 1 by 0.1 intervals.
According to the inner AVE, the best model was the one in which the weights (1) between the host transcriptome and location, (2) the host transcriptome and the CGH, and (3) the CGH block were linked to variables related to the location with weights of 1, 0.1 and 0.1, respectively.

When we added a superblock to the data, there was a slight increase of 0.01 on the inner AVE of the model (See Table \ref{tab:puget-models-ave}).
The model with the superblock that explained most of the variance was that in which the weights of the interaction within (1) the host transcriptome, (2) between the superblock and the CGH, (3) between the host transcriptome and the localization, and (4) between CGH and the host transcriptome were 1, 1, 1 and 1/3, respectively (See table \ref{tab:puget-model-superblock-2}.
To see if the superblock could classify the sample by location, we plotted the first two components of the superblock.

We can clearly see that they do not classify the samples according to the location of the tumor, which is known to affect the tumor phenotype {[}\protect\hyperlink{ref-puget2012}{149}{]}.

Adding one block containing the age of the patient and the severity of the tumor to the model, decreased the inner AVE.
The best model with these blocks, according to the inner AVE, was that in which the interactions (1) within the host transcriptome, (2) between the host transcriptome and the localization, (3) between the host transcriptome and(4) the CGH and between the CGH and the other variables were 1, 1, 1/3 and 1/3, respectively.
The first components of each model can be seen in the figure:

We can observe on the figure \ref{fig:puget-models}, the strong dependency between gene expression and location since the first model while the weaker relationship with the CGH assay {[}\protect\hyperlink{ref-puget2012}{149}{]}.
On the other hand, the major difference is the dispersion on the CGH component on each model.

The effect of the superblock and weights on different models to the inner AVE.
There are significant differences between having the superblock and not having it.

\begin{figure}
\includegraphics[width=1\linewidth]{images/pugets-weights} \caption[Effect of superblock and weights on the inner AVE on Puget's dataset.]{Effect of superblock and weights on the inner AVE on Puget's dataset. Designs with the superblock showed higher inner AVe scores than without it. Interaction yes/no indicates RNA and RNA interaction.}\label{fig:puget-weights}
\end{figure}

The different models resulted on the following AVE values as reported on \ref{tab:puget-models-ave}:

\begin{table}[H]

\caption[AVE values of RGCCA models in the Puget's dataset.]{\label{tab:puget-models-ave}AVE values of RGCCA models in the Puget's dataset. Values for both inner and outer AVE of the first canonical component of models 1, 1.2, 2 2.2 and superblock and superblock.2 are shown.}
\centering
\begin{tabular}[t]{|>{}c|c|>{}c|}
\hline
\em{\textbf{Model}} & \em{\textbf{inner AVE}} & \em{\textbf{outer AVE}}\\
\hline
\textbf{\cellcolor{gray!6}{1}} & \cellcolor{gray!6}{0.6333592} & \cellcolor{gray!6}{0.0692097}\\
\hline
\textbf{1.2} & 0.8512360 & 0.0692319\\
\hline
\textbf{\cellcolor{gray!6}{2}} & \cellcolor{gray!6}{0.2791546} & \cellcolor{gray!6}{0.0738695}\\
\hline
\textbf{2.2} & 0.6902329 & 0.0692707\\
\hline
\textbf{\cellcolor{gray!6}{superblock}} & \cellcolor{gray!6}{0.7055847} & \cellcolor{gray!6}{0.0734578}\\
\hline
\textbf{superblock.2} & 0.8047477 & 0.0695821\\
\hline
\end{tabular}
\end{table}

\hypertarget{results-hsct}{%
\subsection{HSCT dataset}\label{results-hsct}}

The permanova analysis was performed on this dataset to estimate which were the variables that are more relevant.
From the many variables the location, sex, patient id and others were found to be related to the variability of the microbiome or the transcriptome on this dataset.

With the permanova analysis we found that more of the 50\% of the variance of normalized RNA-seq data and microbiome data respectively is explained by the variables of location, disease, sex, and the interaction between disease and sex.
On the transcriptome the most important factor is location which is more than 15\% of the variance, while on the microbiome data the most important factor is the patient id followed by location of the sample.

\begin{table}[H]

\caption[Permanova analysis of transcriptome.]{\label{tab:hsct-permanova-rnaseq}Permanova analysis of transcriptome. The variables and their interactions (shown with :) and the $R^2$ values and the associated p-value. The higher the $R^2$ the more variance is explained by that factor.}
\centering
\begin{tabular}[t]{|>{}c|c|>{}c|}
\hline
\em{\textbf{Factor}} & \em{\textbf{R}} & \em{\textbf{p-value}}\\
\hline
\textbf{\cellcolor{gray!6}{Location}} & \cellcolor{gray!6}{0.18057} & \cellcolor{gray!6}{0.00020}\\
\hline
\textbf{IBD} & 0.03120 & 0.00020\\
\hline
\textbf{\cellcolor{gray!6}{Sex}} & \cellcolor{gray!6}{0.01306} & \cellcolor{gray!6}{0.00120}\\
\hline
\textbf{IBD:Sex} & 0.01279 & 0.11798\\
\hline
\textbf{\cellcolor{gray!6}{Location:IBD}} & \cellcolor{gray!6}{0.02427} & \cellcolor{gray!6}{0.11458}\\
\hline
\textbf{Location:Sex} & 0.00816 & 0.03519\\
\hline
\textbf{\cellcolor{gray!6}{Location:IBD:Sex}} & \cellcolor{gray!6}{0.00486} & \cellcolor{gray!6}{0.52190}\\
\hline
\textbf{Residuals} & 0.72508 & \\
\hline
\end{tabular}
\end{table}

\begin{table}[H]

\caption[Permanova analysis of microbiome]{\label{tab:hsct-permanova-microbiome}Permanova analysis of microbiome. The variables and their interactions (shown with :) and the $R^2$ values and the associated p-value. The higher the $R^2$ the more variance is explained by that factor.}
\centering
\begin{tabular}[t]{|>{}c|c|>{}c|}
\hline
\em{\textbf{Factor}} & \em{\textbf{R}} & \em{\textbf{p-value}}\\
\hline
\textbf{\cellcolor{gray!6}{Location}} & \cellcolor{gray!6}{0.06061} & \cellcolor{gray!6}{0.0002000}\\
\hline
\textbf{IBD} & 0.04967 & 0.0002000\\
\hline
\textbf{\cellcolor{gray!6}{Sex}} & \cellcolor{gray!6}{0.01712} & \cellcolor{gray!6}{0.0003999}\\
\hline
\textbf{IBD:Sex} & 0.01091 & 0.6604679\\
\hline
\textbf{\cellcolor{gray!6}{Location:IBD}} & \cellcolor{gray!6}{0.02089} & \cellcolor{gray!6}{0.8476305}\\
\hline
\textbf{Location:Sex} & 0.01139 & 0.0075918\\
\hline
\textbf{\cellcolor{gray!6}{Location:IBD:Sex}} & \cellcolor{gray!6}{0.00289} & \cellcolor{gray!6}{0.9994001}\\
\hline
\textbf{Residuals} & 0.82652 & \\
\hline
\end{tabular}
\end{table}

With globaltest the results were similar: sex, ibd, location, age and time since diagnosis were able to explain the SESCD score (p-value \(5.7 \cdot 10^{-21}\) ).
The resulting p-value was well below the 0,05 threshold defined for RNA-seq data on the models including the segment of the sample, sex and treatment.

On the microbiome data the results were similar but the p-value was considerably higher but still below the threshold.

\begin{figure}
\includegraphics[width=1\linewidth]{images/hsct-ASV-diversity} \caption[Microbiome diversity in the HSCT dataset.]{Microbiome diversity in the HSCT dataset. On the upper section the Shannon effective and on the lower row the Simpson effective diversity splitted by colon and ileum and controls and CD.}\label{fig:hsct-diversity}
\end{figure}

Diversity indices of the samples were explored and compared for several subsets.
Splitting by location of the sample and disease provided the highest differences and the diversity index along time did not change much.

The PCA didn't show any pattern on the microbiome according to the location, as can be seen on the first dimensions of the PCA on figure \ref{fig:hsct-pca-16s}:

\begin{figure}
\includegraphics[width=1\linewidth]{images/HSCT_PCA_16S} \caption[PCA of 16S data of the HSCT dataset.]{PCA of the 16S data of the HSCT dataset. Samples colored by location of the segment. There are no clear patterns according to the location.}\label{fig:hsct-pca-16s}
\end{figure}

The PCA on the transcriptome shows a clear pattern splitting by location of the sample on the first dimension (see figure \ref{fig:hsct-pca-rnaseq}).

\begin{figure}
\includegraphics[width=1\linewidth]{images/HSCT_PCA_RNAseq} \caption[PCA of RNAseq data of the HSCT dataset.]{PCA of the RNAseq data of the HSCT dataset. Samples colored by location of the segment. The samples separate according to the location.}\label{fig:hsct-pca-rnaseq}
\end{figure}

Weighted gene co-expression network analysis did not provide relevant links between bacteria and transcriptome as it failed to find an acceptable scale free degree.
As can be seen on the Figure \ref{fig:hsct-wgcna-power}, the scale free topology does not reach the recommended threshold of 0.9 and the mean connectivity is also very low even for the first power.

\begin{figure}
\includegraphics[width=1\linewidth]{images/hsct-wgcna-power} \caption[Power evaluation of WGCNA of the HSCT dataset.]{Power evaluation of WGCNA of the HSCT dataset. On the ordinate the power on the abscissa on the left the scale free topology model fit; on the right the mean connectivity. There is a low fit even on large power and the mean connectivity is below 100 from the very first value.}\label{fig:hsct-wgcna-power}
\end{figure}

STATegRa was used between stool 16S data and intestinal 16S data under the assumption that there is a shared common factor without influence of other categorical variables.
However, it did not find a good agreement between these two data sources and 16S data source was not longer used on the analysis.
In addition, the model is fixed, so it did not allow to find new or other relationships that are not one to one.

With RGCCA we could select different models and use all the data available without much assumptions.
The models with the highest inner AVE of the family 1 and the family 2 models were similar to those on the Häsler dataset.

\begin{figure}
\includegraphics[width=1\linewidth]{images/hsct-models} \caption[Models in the HSCT dataset.]{Models in the HSCT dataset. On the abscissa the transcriptome, on the ordinate the Microbiome. Each square represents a different model of the HSCT dataset. On panel A colored by disease status, on panel B colored by sample location. Model 0 has only transcriptome and microbiome data, models 1 to 1.2 with data about the samples and models 2.1 to 2.3 with data about the samples split in 3 blocks.}\label{fig:hsct-models}
\end{figure}

The weights of these models can be observed here:

\begin{table}[H]

\caption[Model 0 of the HSCT dataset.]{\label{tab:hsct-model0}Relationships between the different blocks in the HSCT dataset for model 0. 0 indicates no relationship and 1 indicates a strong relationship.}
\centering
\begin{tabular}[t]{|>{}c|c|>{}c|}
\hline
\em{\textbf{Model 0}} & \em{\textbf{Transcriptome}} & \em{\textbf{Microbiome}}\\
\hline
\textbf{\cellcolor{gray!6}{Transcriptome}} & \cellcolor{gray!6}{0} & \cellcolor{gray!6}{1}\\
\hline
\textbf{Microbiome} & 1 & 0\\
\hline
\end{tabular}
\end{table}

If we include the information about the samples all together in a block called metadata we can start from this model on \ref{tab:hsct-model1-1}:

\begin{table}[H]

\caption[Model 1.1 the of HSCT dataset.]{\label{tab:hsct-model1-1}elationships between the different blocks in the HSCT dataset for model 1.1. 0 indicates no relationship and 1 indicates a strong relationship.}
\centering
\begin{tabular}[t]{|>{}c|c|c|>{}c|}
\hline
\em{\textbf{Model 1.1}} & \em{\textbf{Transcriptome}} & \em{\textbf{Microbiome}} & \em{\textbf{metadata}}\\
\hline
\textbf{\cellcolor{gray!6}{Transcriptome}} & \cellcolor{gray!6}{0} & \cellcolor{gray!6}{0} & \cellcolor{gray!6}{1}\\
\hline
\textbf{Microbiome} & 0 & 0 & 1\\
\hline
\textbf{\cellcolor{gray!6}{metadata}} & \cellcolor{gray!6}{1} & \cellcolor{gray!6}{1} & \cellcolor{gray!6}{0}\\
\hline
\end{tabular}
\end{table}

When looking for the model that adjust better following this blocks we arrived to model 1.2 thanks to \texttt{inteRmodel}, described below on table \ref{tab:hsct-model1-2}:

\begin{table}[H]

\caption[Model 1.2 of the HSCT dataset.]{\label{tab:hsct-model1-2}Relationships between the different blocks in the HSCT dataset for model 1.2. 0 indicates no relationship and 1 indicates a strong relationship.}
\centering
\begin{tabular}[t]{|>{}c|c|c|>{}c|}
\hline
\em{\textbf{Model 1.2}} & \em{\textbf{Transcriptome}} & \em{\textbf{Microbiome}} & \em{\textbf{metadata}}\\
\hline
\textbf{\cellcolor{gray!6}{Transcriptome}} & \cellcolor{gray!6}{0} & \cellcolor{gray!6}{0.0} & \cellcolor{gray!6}{1.0}\\
\hline
\textbf{Microbiome} & 0 & 0.0 & 0.1\\
\hline
\textbf{\cellcolor{gray!6}{metadata}} & \cellcolor{gray!6}{1} & \cellcolor{gray!6}{0.1} & \cellcolor{gray!6}{0.0}\\
\hline
\end{tabular}
\end{table}

On model two we split the invariable variables from those related to the location:

\begin{table}[H]

\caption[Model 2 of the HSCT dataset.]{\label{tab:hsct-model2}Relationships between the different blocks in the HSCT dataset for model 2. 0 indicates no relationship and 1 indicates a strong relationship.}
\centering
\begin{tabular}[t]{|>{}c|c|c|c|c|>{}c|}
\hline
\em{\textbf{Model 2}} & \em{\textbf{Transcriptome}} & \em{\textbf{Microbiome}} & \em{\textbf{Demographic}} & \em{\textbf{Location}} & \em{\textbf{Time}}\\
\hline
\textbf{\cellcolor{gray!6}{Transcriptome}} & \cellcolor{gray!6}{0} & \cellcolor{gray!6}{1} & \cellcolor{gray!6}{1} & \cellcolor{gray!6}{1} & \cellcolor{gray!6}{0}\\
\hline
\textbf{Microbiome} & 1 & 0 & 1 & 1 & 0\\
\hline
\textbf{\cellcolor{gray!6}{Demographic}} & \cellcolor{gray!6}{1} & \cellcolor{gray!6}{1} & \cellcolor{gray!6}{0} & \cellcolor{gray!6}{0} & \cellcolor{gray!6}{1}\\
\hline
\textbf{Location} & 1 & 1 & 0 & 0 & 0\\
\hline
\textbf{\cellcolor{gray!6}{Time}} & \cellcolor{gray!6}{0} & \cellcolor{gray!6}{0} & \cellcolor{gray!6}{1} & \cellcolor{gray!6}{0} & \cellcolor{gray!6}{0}\\
\hline
\end{tabular}
\end{table}

Following this split, we used \texttt{inteRmodel} (See section above \ref{intermodel}) to find the model that has higher inner AVE for these blocks is the one on table \ref{tab:hsct-model2-2}:

\begin{table}[H]

\caption[Model 2.2 of the HSCT dataset.]{\label{tab:hsct-model2-2}Relationships between the different blocks in the HSCT dataset for model 2.2. 0 indicates no relationship and 1 indicates a strong relationship.}
\centering
\begin{tabular}[t]{|>{}c|c|c|c|c|>{}c|}
\hline
\em{\textbf{Model 2.2}} & \em{\textbf{Transcriptome}} & \em{\textbf{Microbiome}} & \em{\textbf{Demographic}} & \em{\textbf{Location}} & \em{\textbf{Time}}\\
\hline
\textbf{\cellcolor{gray!6}{Transcriptome}} & \cellcolor{gray!6}{0} & \cellcolor{gray!6}{0.0} & \cellcolor{gray!6}{0.0} & \cellcolor{gray!6}{1.0} & \cellcolor{gray!6}{0.0}\\
\hline
\textbf{Microbiome} & 0 & 0.0 & 0.2 & 0.1 & 0.0\\
\hline
\textbf{\cellcolor{gray!6}{Demographic}} & \cellcolor{gray!6}{0} & \cellcolor{gray!6}{0.2} & \cellcolor{gray!6}{0.0} & \cellcolor{gray!6}{0.0} & \cellcolor{gray!6}{0.6}\\
\hline
\textbf{Location} & 1 & 0.1 & 0.0 & 0.0 & 0.0\\
\hline
\textbf{\cellcolor{gray!6}{Time}} & \cellcolor{gray!6}{0} & \cellcolor{gray!6}{0.0} & \cellcolor{gray!6}{0.6} & \cellcolor{gray!6}{0.0} & \cellcolor{gray!6}{0.0}\\
\hline
\end{tabular}
\end{table}

We also tested specifically a model from the family 2.3, which can be seen on table \ref{tab:hsct-model2-3}:

\begin{table}[H]

\caption[Model 2.3 of the HSCT dataset.]{\label{tab:hsct-model2-3}Relationships between the different blocks in the HSCT dataset for model 2.3. 0 indicates no relationship and 1 indicates a strong relationship.}
\centering
\begin{tabular}[t]{|>{}c|c|c|c|c|>{}c|}
\hline
\em{\textbf{Model 2.3}} & \em{\textbf{Transcriptome}} & \em{\textbf{Microbiome}} & \em{\textbf{Demographic}} & \em{\textbf{Location}} & \em{\textbf{Time}}\\
\hline
\textbf{\cellcolor{gray!6}{Transcriptome}} & \cellcolor{gray!6}{0.0} & \cellcolor{gray!6}{0.1} & \cellcolor{gray!6}{0.0} & \cellcolor{gray!6}{1.0} & \cellcolor{gray!6}{0}\\
\hline
\textbf{Microbiome} & 0.1 & 0.0 & 0.1 & 0.1 & 0\\
\hline
\textbf{\cellcolor{gray!6}{Demographic}} & \cellcolor{gray!6}{0.0} & \cellcolor{gray!6}{0.1} & \cellcolor{gray!6}{0.0} & \cellcolor{gray!6}{0.0} & \cellcolor{gray!6}{1}\\
\hline
\textbf{Location} & 1.0 & 0.1 & 0.0 & 0.0 & 0\\
\hline
\textbf{\cellcolor{gray!6}{Time}} & \cellcolor{gray!6}{0.0} & \cellcolor{gray!6}{0.0} & \cellcolor{gray!6}{1.0} & \cellcolor{gray!6}{0.0} & \cellcolor{gray!6}{0}\\
\hline
\end{tabular}
\end{table}

The best model of the family 2 confirmed a relationship between the host transcriptome and the location-related variables, while the microbiome was associated with the demographic and location-related variables (see figure \ref{fig:hsct-models} and \href{https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0246367\#pone.0246367.s002}{S2} data of {[}\protect\hyperlink{ref-revilla2021}{197}{]}).
Overall, we see that the relationships in the model affected the distribution of samples on the components of both the host transcriptome and the microbiome.

\begin{table}[H]

\caption[The models in the HSCT dataset and their AVE values.]{\label{tab:hsct-models-ave}The models in the HSCT and their AVE values. For each model the inner AVE and the outer AVE is presented.}
\centering
\begin{tabular}[t]{|>{}c|c|>{}c|}
\hline
\em{\textbf{Model}} & \em{\textbf{inner AVE}} & \em{\textbf{outer AVE}}\\
\hline
\textbf{\cellcolor{gray!6}{0.0}} & \cellcolor{gray!6}{0.3999234} & \cellcolor{gray!6}{0.1001689}\\
\hline
\textbf{1.0} & 0.6230190 & 0.0842333\\
\hline
\textbf{\cellcolor{gray!6}{1.1}} & \cellcolor{gray!6}{0.5678189} & \cellcolor{gray!6}{0.0848714}\\
\hline
\textbf{1.2} & 0.7043881 & 0.0775766\\
\hline
\textbf{\cellcolor{gray!6}{2.0}} & \cellcolor{gray!6}{0.2517363} & \cellcolor{gray!6}{0.0982050}\\
\hline
\textbf{2.1} & 0.6940253 & 0.0940266\\
\hline
\textbf{\cellcolor{gray!6}{2.2}} & \cellcolor{gray!6}{0.8187640} & \cellcolor{gray!6}{0.0941628}\\
\hline
\textbf{2.3} & 0.7761846 & 0.0943938\\
\hline
\end{tabular}
\end{table}

The different models selected different variables, some of which are shared between models.
The most similar models are those that have split the metadata into 3 blocks, followed by those that have the metadata in a single block.

In order to analyze the accuracy of the models, one thousand bootstraps were used to integrate the data from the HSCT CD dataset.
Each bootstrap had its own dispersion on the variables according to the samples selected, the distribution of the bootstraps used are represented here:

\begin{figure}
\includegraphics[width=1\linewidth]{images/hsct-bootstrap-characteristics} \caption[Characteristics of the samples of the HSCT bootstraped samples.]{Dispersion of the bootstraped samples on age and percentage of colon and controls samples.}\label{fig:hsct-bootstrap-quality}
\end{figure}

Evaluating the same model on each bootstrap lead to a dispersion on the inner AVE of the model.
The lower the dispersion, the more robust the model was to different conditions than in the initial testing.

\begin{figure}
\includegraphics[width=1\linewidth]{images/hsct-bootstrap} \caption[Bootstrap results on HSCT dataset.]{Bootstrap of the models 0, 1.2 and 2.2. The point with the black circle is the AVE of the original data. The dispersion is shown by the ellipses. Model 0 and 1.2 have lower inner and outer AVE score, model 2.2 has lower outer score but higher inner value than the bootstrapped.}\label{fig:hsct-bootstrap}
\end{figure}

With the bootstrapped models we used BaseSet to estimate the probability that each variable to be relevant for the association with a disease.
However, due to big amount of small probabilities when using the BaseSet package to calculate which variables are more relevant it could not provide a good estimation on time.

MCIA was applied as a baseline of the integration, the first two components were represented similarly to those of the blocks when using RGCCA \ref{fig:hsct-mcia}.

\begin{figure}
\includegraphics[width=1\linewidth]{images/hsct-mcia} \caption[MCIA dimensions in the HSCT dataset.]{MCIA dimensions in the HSCT dataset. MCIA first two synthetic variables. In red circles the colon and in blue triangles the ileum.}\label{fig:hsct-mcia}
\end{figure}

The AUC of classifying the transcriptome in colon or ileum segments was compared between the models (see \ref{fig:hsct-auc-plot} and with MCIA.

\begin{figure}
\includegraphics[width=1\linewidth]{images/hsct-auc-models} \caption[AUC of the RGCCA models in the HSCT dataset.]{AUC of the RGCCA models in the HSCT dataset. The classification of the localization of the sample according to the first component of the gene expression of the models generated with RGCCA on the HSCT dataset.}\label{fig:hsct-auc-plot}
\end{figure}

These models have the following AUC to classify the location of the sample according to the first component of the transcriptome block.

\begin{table}[H]

\caption[AUC values of RGCCA models in the HSCT dataset.]{\label{tab:hsct-auc}AUC values of RGCCA models of the HSCT dataset classyfing the location of the sample according to the first component of the transcriptome block. From model 0 to model 2.3, the best classification is achieved with model 2.1. Note that this is removing two samples for which the location is unkown.}
\centering
\begin{tabular}[t]{|>{}c|>{}c|}
\hline
\em{\textbf{Model}} & \em{\textbf{AUC}}\\
\hline
\textbf{\cellcolor{gray!6}{0}} & \cellcolor{gray!6}{0.4537037}\\
\hline
\textbf{1} & 0.4309414\\
\hline
\textbf{\cellcolor{gray!6}{1.1}} & \cellcolor{gray!6}{0.4639275}\\
\hline
\textbf{1.2} & 0.5958719\\
\hline
\textbf{\cellcolor{gray!6}{2}} & \cellcolor{gray!6}{0.9450231}\\
\hline
\textbf{2.1} & 0.9988426\\
\hline
\textbf{\cellcolor{gray!6}{2.2}} & \cellcolor{gray!6}{0.9980710}\\
\hline
\textbf{2.3} & 0.9969136\\
\hline
\end{tabular}
\end{table}

On MCIA the AUC for the classification of ileum or colon samples is of 0.9851 once those two samples with unknown location are excluded.
This is on par with the models of family 2 as can be seen on the table \ref{tab:hsct-auc}.

The different models selected different variables as can be seen below:

\begin{figure}
\includegraphics[width=1\linewidth]{images/hsct-features} \caption[Upset plot of variables selected in the HSCT dataset.]{Upset plot of variables selected in the HSCT dataset. The variables selected on each model from 0 to 2.3 showing the intersection between them regarding genes, panel A, and OTUs, panel B.}\label{fig:hsct-features}
\end{figure}

Differences and similarities between the selected features of each model can be observed on Figure \ref{fig:hsct-features}.
Genes are very similar between model 0 to 1.2 and between 2 to 2.3, while OTUs are very unique on model 0 and others shared between most models.

\hypertarget{results-hasler}{%
\subsection{Häsler's dataset}\label{results-hasler}}

If we look at the dataset 16S, the PCA doe not show a pattern as can be seen on figure \ref{fig:hasler-pca-16s}:

\begin{figure}
\includegraphics[width=1\linewidth]{images/hasler_PCA_16S} \caption[PCA of 16S of the Häsler's dataset.]{PCA of 16S of Häsler's dataset colored by the location of the samples.}\label{fig:hasler-pca-16s}
\end{figure}

The PCA on the transcriptome shows a distinction between colon and ileum according to the first dimension of the pca that can be seen on \ref{fig:hasler-pca-rnaseq}:

\begin{figure}
\includegraphics[width=1\linewidth]{images/hasler_PCA_RNAseq} \caption[PCA of RNAseq of the Häsler's dataset.]{PCA of RNAseq of the Häsler's dataset. The samples are colored by location sigma and ileum. There are two clear groups according to the location.}\label{fig:hasler-pca-rnaseq}
\end{figure}

In this dataset, the parameter tau behaved slightly differently than with the previous dataset but the value from the Schäfer's method for tau was close to the best value.

\begin{figure}
\includegraphics[width=1\linewidth]{images/hasler-centroid_tau} \caption[Tau effect on inner AVE in the Häsler's dataset.]{Changes on tau on the centroid scheme in the Häsler dataset affect the inner AVE score on the model 1. The panel A shows on the ordinate the RNAseq tau value, the panel B on the right, shows the tau of the microorganism; both of them show the y's tau on the abscissa.}\label{fig:hasler-tau}
\end{figure}

Models for Häsler dataset are:

\begin{table}[H]

\caption[Model 0 of Häsler dataset.]{\label{tab:hasler-model-0}Model 0 of Häsler dataset. 0 indicates no relationship and 1 indicates a strong relationship.}
\centering
\begin{tabular}[t]{|>{}c|c|>{}c|}
\hline
\em{\textbf{Model 0}} & \em{\textbf{RNAseq}} & \em{\textbf{micro}}\\
\hline
\textbf{\cellcolor{gray!6}{RNAseq}} & \cellcolor{gray!6}{0} & \cellcolor{gray!6}{1}\\
\hline
\textbf{micro} & 1 & 0\\
\hline
\end{tabular}
\end{table}

The first model for family 1 is on table \ref{tab:hasler-model-1-1}:

\begin{table}[H]

\caption[Model 1.1 of the Häsler dataset.]{\label{tab:hasler-model-1-1}Model 1.1 of the Häsler dataset. 0 indicates no relationship and 1 indicates a strong relationship.}
\centering
\begin{tabular}[t]{|>{}c|c|c|>{}c|}
\hline
\em{\textbf{Model 1.1}} & \em{\textbf{RNAseq}} & \em{\textbf{micro}} & \em{\textbf{meta}}\\
\hline
\textbf{\cellcolor{gray!6}{RNAseq}} & \cellcolor{gray!6}{0} & \cellcolor{gray!6}{0} & \cellcolor{gray!6}{1}\\
\hline
\textbf{micro} & 0 & 0 & 1\\
\hline
\textbf{\cellcolor{gray!6}{meta}} & \cellcolor{gray!6}{1} & \cellcolor{gray!6}{1} & \cellcolor{gray!6}{0}\\
\hline
\end{tabular}
\end{table}

The after optimization of the model of family 1, the best one is on table \ref{tab:hasler-model-1-2}:

\begin{table}[H]

\caption[Model 1.2 of the Häsler dataset.]{\label{tab:hasler-model-1-2}Model 1.2 of the Häsler dataset. 0 indicates no relationship and 1 indicates a strong relationship.}
\centering
\begin{tabular}[t]{|>{}c|c|c|>{}c|}
\hline
\em{\textbf{Model 1.2}} & \em{\textbf{RNAseq}} & \em{\textbf{micro}} & \em{\textbf{meta}}\\
\hline
\textbf{\cellcolor{gray!6}{RNAseq}} & \cellcolor{gray!6}{0} & \cellcolor{gray!6}{0.0} & \cellcolor{gray!6}{1.0}\\
\hline
\textbf{micro} & 0 & 0.0 & 0.1\\
\hline
\textbf{\cellcolor{gray!6}{meta}} & \cellcolor{gray!6}{1} & \cellcolor{gray!6}{0.1} & \cellcolor{gray!6}{0.0}\\
\hline
\end{tabular}
\end{table}

The first model for family 2 is on table \ref{tab:hasler-model-2-1}:

\begin{table}[H]

\caption[Model 2.1 of the Häsler dataset.]{\label{tab:hasler-model-2-1}Model 2.1 of the Häsler dataset. 0 indicates no relationship and 1 indicates a strong relationship.}
\centering
\begin{tabular}[t]{|>{}c|c|c|c|c|>{}c|}
\hline
\em{\textbf{Model 2.1}} & \em{\textbf{RNAseq}} & \em{\textbf{micro}} & \em{\textbf{Location}} & \em{\textbf{Demographic}} & \em{\textbf{Time}}\\
\hline
\textbf{\cellcolor{gray!6}{RNAseq}} & \cellcolor{gray!6}{0} & \cellcolor{gray!6}{0.0} & \cellcolor{gray!6}{1.0} & \cellcolor{gray!6}{0} & \cellcolor{gray!6}{0}\\
\hline
\textbf{micro} & 0 & 0.0 & 0.5 & 1 & 0\\
\hline
\textbf{\cellcolor{gray!6}{Location}} & \cellcolor{gray!6}{1} & \cellcolor{gray!6}{0.5} & \cellcolor{gray!6}{0.0} & \cellcolor{gray!6}{0} & \cellcolor{gray!6}{0}\\
\hline
\textbf{Demographic} & 0 & 1.0 & 0.0 & 0 & 1\\
\hline
\textbf{\cellcolor{gray!6}{Time}} & \cellcolor{gray!6}{0} & \cellcolor{gray!6}{0.0} & \cellcolor{gray!6}{0.0} & \cellcolor{gray!6}{1} & \cellcolor{gray!6}{0}\\
\hline
\end{tabular}
\end{table}

After optimization of models of family 1 the best model according to the inner AVE score is on table \ref{tab:hasler-model-2-2}:

\begin{table}[H]

\caption[Model 2.2 of the Häsler dataset.]{\label{tab:hasler-model-2-2}Model 2.2 of the Häsler dataset. 0 indicates no relationship and 1 indicates a strong relationship.}
\centering
\begin{tabular}[t]{|>{}c|c|c|c|c|>{}c|}
\hline
\em{\textbf{Model 2.2}} & \em{\textbf{RNAseq}} & \em{\textbf{micro}} & \em{\textbf{Location}} & \em{\textbf{Demographic}} & \em{\textbf{Time}}\\
\hline
\textbf{\cellcolor{gray!6}{RNAseq}} & \cellcolor{gray!6}{0.0} & \cellcolor{gray!6}{0.1} & \cellcolor{gray!6}{1} & \cellcolor{gray!6}{0.0} & \cellcolor{gray!6}{0.0}\\
\hline
\textbf{micro} & 0.1 & 0.0 & 0 & 0.1 & 1.0\\
\hline
\textbf{\cellcolor{gray!6}{Location}} & \cellcolor{gray!6}{1.0} & \cellcolor{gray!6}{0.0} & \cellcolor{gray!6}{0} & \cellcolor{gray!6}{0.0} & \cellcolor{gray!6}{0.0}\\
\hline
\textbf{Demographic} & 0.0 & 0.1 & 0 & 0.0 & 0.1\\
\hline
\textbf{\cellcolor{gray!6}{Time}} & \cellcolor{gray!6}{0.0} & \cellcolor{gray!6}{1.0} & \cellcolor{gray!6}{0} & \cellcolor{gray!6}{0.1} & \cellcolor{gray!6}{0.0}\\
\hline
\end{tabular}
\end{table}

On table \ref{tab:hasler-aves}, we can see here the AVE scores of each of the previous models:

\begin{table}[H]

\caption[AVE values of RGCCA models in Häsler's dataset.]{\label{tab:hasler-aves}AVE values of RGCCA models in Häsler's dataset. The inner and the outer AVE scores of multiple models tested on the Häsler dataset are shown. The model with the hightest inner AVE is model 1.2.}
\centering
\begin{tabular}[t]{|>{}c|c|>{}c|}
\hline
\em{\textbf{Model}} & \em{\textbf{inner AVE}} & \em{\textbf{outer AVE}}\\
\hline
\textbf{\cellcolor{gray!6}{0}} & \cellcolor{gray!6}{0.8217371} & \cellcolor{gray!6}{0.0961236}\\
\hline
\textbf{1.1} & 0.7461423 & 0.1024148\\
\hline
\textbf{\cellcolor{gray!6}{1.2}} & \cellcolor{gray!6}{0.8349410} & \cellcolor{gray!6}{0.1025486}\\
\hline
\textbf{2.1} & 0.4980681 & 0.1008395\\
\hline
\textbf{\cellcolor{gray!6}{2.2}} & \cellcolor{gray!6}{0.7513065} & \cellcolor{gray!6}{0.1009915}\\
\hline
\end{tabular}
\end{table}

In contrast to the HSCT's dataset (table \ref{tab:hsct-models-ave}), the model with the highest inner AVE was model 1.2 but model 2.2 was close to it (see table \ref{tab:hasler-aves}).
Model 2.2 has a relationship of 0.1 between microbiome and the host transcriptome and of 1 between the location and the host transcriptome.
The microbiome block is also related by a factor of 0.1 with the demographic block and of 1 with the time block.
Lastly, the time and the demographic block are related by a factor of a 0.1.
In either case the family 1 and family 2 models can correctly separate by sample location (colon or ileum) but not by disease type or inflammation status as can be seen on figure \ref{fig:hasler-models}.

\begin{figure}
\includegraphics[width=1\linewidth]{images/hasler-models} \caption[Models from inteRmodel in the Häsler's datset.]{Models from inteRmodel in the Häsler's dataset. Model 0 with just the transcriptome and microbiome data. Models 1.1 to 1.2 with transcriptome, microbiome and sample data in a single block. Models 2.1 and 2.2 with transcriptome, microbiome and sample data in multiple blocks. On the A panel colored by disease on the B panel colored by location of the sample.}\label{fig:hasler-models}
\end{figure}

There is no observable cluster of IBD samples and the other samples, showing that on this dataset the differences of the microbiome between the different type of samples are less stark.
The classification of samples was very accurate in all the models, specially on model 2.2, see figure \ref{fig:hasler-auc-plot}:

\begin{figure}
\includegraphics[width=1\linewidth]{images/hasler-auc} \caption[AUC of RGCCA models in the Häsler's dataset]{AUC of models with RGCCA in the Häsler's dataset. The classification of the localization of the sample according to the first component of the gene expression of the models generated with RGCCA on the Häsler's dataset.}\label{fig:hasler-auc-plot}
\end{figure}

This accuracy resulted on high AUC values for all the models, as can be seen on table \ref{tab:hasler-auc}:

\begin{table}[H]

\caption[AUC of the RGCCA models in Häsler's dataset.]{\label{tab:hasler-auc}AUC of the RGCCA models in Häsler's dataset. The AUC was calculated with the first dimension of the gene expression block ability to predict location of the sample.}
\centering
\begin{tabular}[t]{|>{}c|>{}c|}
\hline
\em{\textbf{Model}} & \em{\textbf{AUC}}\\
\hline
\textbf{\cellcolor{gray!6}{0}} & \cellcolor{gray!6}{0.8011494}\\
\hline
\textbf{1.1} & 0.9781609\\
\hline
\textbf{\cellcolor{gray!6}{1.2}} & \cellcolor{gray!6}{0.9977011}\\
\hline
\textbf{2.1} & 1.0000000\\
\hline
\textbf{\cellcolor{gray!6}{2.2}} & \cellcolor{gray!6}{1.0000000}\\
\hline
\end{tabular}
\end{table}

MCIA was applied as a baseline of the integration and compared to the different models to know which one separates best colon and ileum samples.
The result on the first two dimension is shown on figure \ref{fig:hasler-mcia}:

\begin{figure}
\includegraphics[width=1\linewidth]{images/hasler-mcia} \caption[MCIA dimensions in the Häsler's dataset.]{MCIA dimensions in the Häsler's dataset. MCIA first two dimensions of the dataset colored by state, the shape is according to the location of the samples. Shows two vertical groups on the first syntethic dimension according to the location of the samples.}\label{fig:hasler-mcia}
\end{figure}

MCIA's AUC results was as high as the model 2.2 to classify samples according to their location.
It was even better to classify the samples according to the type of sample they are: 0.6248 vs 1 the best AUC from RGCCA that corresponds to model 1.2.

\hypertarget{results-morgan}{%
\subsection{Morgan's dataset}\label{results-morgan}}

To explore this dataset that is different from the others with IBD.

The PCA didn't show any pattern on the microbiome according to the location, see figure \ref{fig:morgan-pca-16s}.

\begin{figure}
\includegraphics[width=1\linewidth]{images/morgan_pca_16S} \caption[PCA of 16S of the Morgan's dataset]{PCA of 16S of the Morgan's dataset. The sample are colored by location. }\label{fig:morgan-pca-16s}
\end{figure}

The PCA on the transcriptome did not show a clear distinction between the prepouch ileum and the pouch on figure \ref{fig:morgan-pca-rnaseq} but there is a pattern:

\begin{figure}
\includegraphics[width=1\linewidth]{images/morgan_pca_RNAseq} \caption[PCA of RNAseq of the Morgan's dataset]{PCA of RNAseq of the Morgan's dataset. The sample are colored by location. }\label{fig:morgan-pca-rnaseq}
\end{figure}

We tested if results of inteRmodel were consistent on this dataset with the other datasets.
The first model we tried is model 0 as in table \ref{tab:morgan-model0}:

\begin{table}[H]

\caption[Model 0 of the Morgan's dataset.]{\label{tab:morgan-model0}Relationships between the different blocks in the Morgan's dataset for model 0. 0 indicates no relationship and 1 indicates a strong relationship.}
\centering
\begin{tabular}[t]{|>{}c|c|>{}c|}
\hline
\em{\textbf{Model 0}} & \em{\textbf{Transcriptome}} & \em{\textbf{Microbiome}}\\
\hline
\textbf{\cellcolor{gray!6}{Transcriptome}} & \cellcolor{gray!6}{0} & \cellcolor{gray!6}{1}\\
\hline
\textbf{Microbiome} & 1 & 0\\
\hline
\end{tabular}
\end{table}

We then added the data about the samples as provided \ref{methods-morgan}, on a simple model as in table \ref{tab:morgan-model1}:

\begin{table}[H]

\caption[Model 1 of the Morgan's dataset.]{\label{tab:morgan-model1}Relationships between the different blocks in the Morgan's dataset for model 1. 0 indicates no relationship and 1 indicates a strong relationship.}
\centering
\begin{tabular}[t]{|>{}c|c|c|>{}c|}
\hline
\em{\textbf{Model 1}} & \em{\textbf{Transcriptome}} & \em{\textbf{Microbiome}} & \em{\textbf{metadata}}\\
\hline
\textbf{\cellcolor{gray!6}{Transcriptome}} & \cellcolor{gray!6}{0} & \cellcolor{gray!6}{0} & \cellcolor{gray!6}{1}\\
\hline
\textbf{Microbiome} & 0 & 0 & 1\\
\hline
\textbf{\cellcolor{gray!6}{metadata}} & \cellcolor{gray!6}{1} & \cellcolor{gray!6}{1} & \cellcolor{gray!6}{0}\\
\hline
\end{tabular}
\end{table}

When looking for the model that adjust better following this structure we arrived to model 1.2, described below:

\begin{table}[H]

\caption[Model 1.2 of the Morgan's dataset.]{\label{tab:morgan-model1-2}Relationships between the different blocks in the Morgan's dataset for model 1.2. 0 indicates no relationship and 1 indicates a strong relationship.}
\centering
\begin{tabular}[t]{|>{}c|c|c|>{}c|}
\hline
\em{\textbf{Model 1.2}} & \em{\textbf{Transcriptome}} & \em{\textbf{Microbiome}} & \em{\textbf{metadata}}\\
\hline
\textbf{\cellcolor{gray!6}{Transcriptome}} & \cellcolor{gray!6}{0.0} & \cellcolor{gray!6}{0.1} & \cellcolor{gray!6}{0}\\
\hline
\textbf{Microbiome} & 0.1 & 0.0 & 1\\
\hline
\textbf{\cellcolor{gray!6}{metadata}} & \cellcolor{gray!6}{0.0} & \cellcolor{gray!6}{1.0} & \cellcolor{gray!6}{0}\\
\hline
\end{tabular}
\end{table}

On model two we split the invariable variables from those related to the location (see \ref{tab:morgan-model2}):

\begin{table}[H]

\caption[Model 2 of the Morgan's dataset.]{\label{tab:morgan-model2}Relationships between the different blocks in the Morgan's dataset for model 2. 0 indicates no relationship and 1 indicates a strong relationship.}
\centering
\begin{tabular}[t]{|>{}c|c|c|c|>{}c|}
\hline
\em{\textbf{Model 2}} & \em{\textbf{Transcriptome}} & \em{\textbf{Microbiome}} & \em{\textbf{Demographic}} & \em{\textbf{Location}}\\
\hline
\textbf{\cellcolor{gray!6}{Transcriptome}} & \cellcolor{gray!6}{0} & \cellcolor{gray!6}{1} & \cellcolor{gray!6}{1} & \cellcolor{gray!6}{1}\\
\hline
\textbf{Microbiome} & 1 & 0 & 1 & 1\\
\hline
\textbf{\cellcolor{gray!6}{Demographic}} & \cellcolor{gray!6}{1} & \cellcolor{gray!6}{1} & \cellcolor{gray!6}{0} & \cellcolor{gray!6}{0}\\
\hline
\textbf{Location} & 1 & 1 & 0 & 0\\
\hline
\end{tabular}
\end{table}

The model that has higher inner AVE for these blocks is the following:

\begin{table}[H]

\caption[Model 2.2 of the Morgan's dataset.]{\label{tab:morgan-model2-2}Relationships between the different blocks in the Morgan's dataset for model 2.2. 0 indicates no relationship and 1 indicates a strong relationship.}
\centering
\begin{tabular}[t]{|>{}c|c|c|c|>{}c|}
\hline
\em{\textbf{Model 2.2}} & \em{\textbf{Transcriptome}} & \em{\textbf{Microbiome}} & \em{\textbf{Demographic}} & \em{\textbf{Location}}\\
\hline
\textbf{\cellcolor{gray!6}{Transcriptome}} & \cellcolor{gray!6}{0.0} & \cellcolor{gray!6}{0.1} & \cellcolor{gray!6}{1} & \cellcolor{gray!6}{0.1}\\
\hline
\textbf{Microbiome} & 0.1 & 0.0 & 0 & 0.0\\
\hline
\textbf{\cellcolor{gray!6}{Demographic}} & \cellcolor{gray!6}{1.0} & \cellcolor{gray!6}{0.0} & \cellcolor{gray!6}{0} & \cellcolor{gray!6}{0.0}\\
\hline
\textbf{Location} & 0.1 & 0.0 & 0 & 0.0\\
\hline
\end{tabular}
\end{table}

Each model is different from previous models.
After model 2.2 we looked on the model similar to model 2.3 in the HSCT dataset showed but it is the same as in model 2.2.
However, it is kept on the further analysis.

The different models were not able to separate the samples neither by location or sex.

\begin{figure}
\includegraphics[width=1\linewidth]{images/morgan_models} \caption[Models from inteRmodel in the Morgan's dataset. ]{Models from inteRmodel in the Morgan's dataset. First component of the transcriptome and microbiome of models on the Morgan's dataset. Model 0 without sample data. Model 1 to 1.2 with all the sample data in a single block and models 2.1 to 2.3 with sample data in several blocks. Panel A shows samples colored by sex and panel B by segment of the sample. There is no clear classification neither by location nor sex on any of the models.}\label{fig:morgan-models}
\end{figure}

Nevertheless, we compared the classification with the MCIA algorithm and still resulted that model 2.2 provide a better classification than MCIA.

\begin{table}[H]

\caption[AVE values of RGCCA for the Morgan's dataset.]{\label{tab:morgan-models-ave}AVE values of RGCCA for the Morgan's dataset. The inner and the outer AVE scores of multiple models tested on the Morgan dataset are shown. The model with the hightest inner AVE is model 1.2.}
\centering
\begin{tabular}[t]{|>{}c|c|>{}c|}
\hline
\em{\textbf{Model}} & \em{\textbf{inner AVE}} & \em{\textbf{outer AVE}}\\
\hline
\textbf{\cellcolor{gray!6}{0.0}} & \cellcolor{gray!6}{0.4735601} & \cellcolor{gray!6}{0.1098639}\\
\hline
\textbf{1.0} & 0.6333592 & 0.1152280\\
\hline
\textbf{\cellcolor{gray!6}{1.1}} & \cellcolor{gray!6}{0.2448234} & \cellcolor{gray!6}{0.1104746}\\
\hline
\textbf{1.2} & 0.7868443 & 0.0422660\\
\hline
\textbf{\cellcolor{gray!6}{2.0}} & \cellcolor{gray!6}{0.4404123} & \cellcolor{gray!6}{0.1088730}\\
\hline
\textbf{2.1} & 0.6052598 & 0.1074900\\
\hline
\textbf{\cellcolor{gray!6}{2.2}} & \cellcolor{gray!6}{0.6895661} & \cellcolor{gray!6}{0.1081315}\\
\hline
\textbf{2.3} & 0.6895661 & 0.1081315\\
\hline
\end{tabular}
\end{table}

When exploring the bootstraps of the data we found that model 1.2 is highly variable:

\begin{figure}
\includegraphics[width=1\linewidth]{images/morgan_bootstrap} \caption[AVE scores of bootstrapped models on Morgans' dataset.]{Inner and outer AVE scores of the bootstrapped models 0 1.2 and 2.2. Model 0 does not have sample data. Model 1.2 has microbiome, transcriptonme and sample data in a single block and model 2.2 has microbiome, transcriptome and the sample data split in several blocks.}\label{fig:morgan-bootstrap}
\end{figure}

In addition the model 2.2 usually has a lower inner AVE compared to model 1.2.

\begin{figure}
\includegraphics[width=1\linewidth]{images/morgan-auc} \caption[AUC of RGCCA models in the Morgan's dataset]{AUC of RGCCA models in the Morgan's dataset. The classification of the localization of the sample according to the first component of the gene expression of the models generated with RGCCA on the Morgan's dataset.}\label{fig:morgan-auc-plot}
\end{figure}

The area under the curve for these models is:

\begin{table}[H]

\caption[AUC for the Morgan's dataset]{\label{tab:morgan-auc}AUC for the Morgan's dataset classificating the localization of the sample according to the first component of the gene expression of the models generated with RGCCA.}
\centering
\begin{tabular}[t]{|>{}c|>{}c|}
\hline
\em{\textbf{Model}} & \em{\textbf{AUC}}\\
\hline
\textbf{\cellcolor{gray!6}{0}} & \cellcolor{gray!6}{0.4969734}\\
\hline
\textbf{1} & 0.7934971\\
\hline
\textbf{\cellcolor{gray!6}{1.1}} & \cellcolor{gray!6}{0.8161536}\\
\hline
\textbf{1.2} & 0.5606192\\
\hline
\textbf{\cellcolor{gray!6}{2}} & \cellcolor{gray!6}{0.8546351}\\
\hline
\textbf{2.1} & 0.8473712\\
\hline
\textbf{\cellcolor{gray!6}{2.2}} & \cellcolor{gray!6}{0.8352646}\\
\hline
\textbf{2.3} & 0.8352646\\
\hline
\end{tabular}
\end{table}

With MCIA:

\begin{figure}
\includegraphics[width=1\linewidth]{images/morgan-mcia} \caption[MCIA dimensions in the Morgan's dataset.]{MCIA dimensions in the Morgan's dataset. MCIA first two dimensions of the dataset. The color and shape match the location of the sample. There is some separation between samples by location.}\label{fig:morgan-mcia-plot}
\end{figure}

If we quantify this separation by the first dimension of MCIA, the AUC is 0.818, which is slightly worst than the models of family 2.

\hypertarget{results-howell}{%
\subsection{Howell's dataset}\label{results-howell}}

The 16S of this dataset doesn't show any clear pattern regarding the location of the samples according to the firsts dimensions of the PCA on \ref{fig:howell-pca-16s}:

\begin{figure}
\includegraphics[width=1\linewidth]{images/howell_PCA_16S} \caption[PCA of 16S data of the Howell's dataset]{PCA of 16S data of the Howell's dataset. Samples are colored by location. There are no pattern on any of the first do dimensions.}\label{fig:howell-pca-16s}
\end{figure}

The PCA on the transcriptome shows a distinction between colon and ileum \ref{fig:howell-pca-rnaseq} there are almost two distinct groups according to location:

\begin{figure}
\includegraphics[width=1\linewidth]{images/howell_PCA_RNAseq} \caption[PCA of RNASeq data of the Howell's dataset]{PCA of RNAseq data of the Howell's dataset. Samples are colored by location. There are two groups of samples according to their location.}\label{fig:howell-pca-rnaseq}
\end{figure}

This dataset was processed to confirm the results on the previous datasets.
As always first we started with model 0, connecting both the RNAseq and the 16S blocks:

Later we look for the best model of family 1 (without looking at any previous model of family 1).
This resulted on the following model \ref{tab:howell-model1-2}:

\begin{table}[H]

\caption[Model 1.2 of the Howell's dataset.]{\label{tab:howell-model1-2}Relationships between the different blocks in the Howell's dataset for model 1.2. 0 indicates no relationship and 1 indicates a strong relationship.}
\centering
\begin{tabular}[t]{|>{}c|c|c|>{}c|}
\hline
\em{\textbf{Model 1.2}} & \em{\textbf{RNAseq}} & \em{\textbf{16S}} & \em{\textbf{metadata}}\\
\hline
\textbf{\cellcolor{gray!6}{RNAseq}} & \cellcolor{gray!6}{0.0} & \cellcolor{gray!6}{0.1} & \cellcolor{gray!6}{1}\\
\hline
\textbf{16S} & 0.1 & 0.0 & 0\\
\hline
\textbf{\cellcolor{gray!6}{metadata}} & \cellcolor{gray!6}{1.0} & \cellcolor{gray!6}{0.0} & \cellcolor{gray!6}{0}\\
\hline
\end{tabular}
\end{table}

Model 1.2 \ref{tab:howell-model1-2} was the best according to the AVE score but perform worse when attempting to recreate known biological differences via classifying samples as we can see below \ref{fig:howell-models}:

\begin{figure}
\includegraphics[width=1\linewidth]{images/howell-models} \caption[Models from inteRmodel in the Howell's datset.]{Models from inteRmodel in the Howell's datset. The three main models, model 0, 1.2 and 2.2 on the Howell's dataset colored by section colon, ileum and shape according to the disease: square, ulcerative colitis; triangle, normal; circle, Crohn's disease. Model 0 has just trancriptomic and microbiome data, model 1.2 has transcriptomic, microbiome and sample data and model 2.2 has transcriptomic, microbiome and sample data split in different blocks.}\label{fig:howell-models}
\end{figure}

Model 2.2 was selected for further analysis as it describes more accurately the biology of the dataset it.
Model 2.2 can be seen on \ref{tab:howell-model2-2}.

\begin{table}[H]

\caption[Model 2.2 of the Howell's dataset.]{\label{tab:howell-model2-2}Relationships between the different blocks in the Howell's dataset for model 2.2. 0 indicates no relationship and 1 indicates a strong relationship.}
\centering
\begin{tabular}[t]{|>{}c|c|c|c|>{}c|}
\hline
\em{\textbf{Model 2.2}} & \em{\textbf{RNAseq}} & \em{\textbf{16S}} & \em{\textbf{demographics}} & \em{\textbf{location}}\\
\hline
\textbf{\cellcolor{gray!6}{RNAseq}} & \cellcolor{gray!6}{0} & \cellcolor{gray!6}{0} & \cellcolor{gray!6}{0.0} & \cellcolor{gray!6}{1.0}\\
\hline
\textbf{16S} & 0 & 0 & 1.0 & 0.0\\
\hline
\textbf{\cellcolor{gray!6}{demographics}} & \cellcolor{gray!6}{0} & \cellcolor{gray!6}{1} & \cellcolor{gray!6}{0.0} & \cellcolor{gray!6}{0.1}\\
\hline
\textbf{location} & 1 & 0 & 0.1 & 0.0\\
\hline
\end{tabular}
\end{table}

Model 1.2 has a 0.1 relationship between the ASV and the transcriptome and 1 between transcriptome and metadata.
While model 2.2 has a relationship of 1 between location and transcriptome and demographics and ASV but only of 0.1 between demographics and location.

\begin{table}[H]

\caption[AVE values of RGCCA for the Howell's dataset.]{\label{tab:howell-models-ave}AVE values of RGCCA for the Howell's dataset. The inner and the outer AVE scores of multiple models tested in the Howell dataset are shown. The model with the hightest inner AVE is model 1.2.}
\centering
\begin{tabular}[t]{|>{}c|c|>{}c|}
\hline
\em{\textbf{Model}} & \em{\textbf{inner AVE}} & \em{\textbf{outer AVE}}\\
\hline
\textbf{\cellcolor{gray!6}{0.0}} & \cellcolor{gray!6}{0.7180980} & \cellcolor{gray!6}{0.1112390}\\
\hline
\textbf{1.2} & 0.8972258 & 0.1660267\\
\hline
\textbf{\cellcolor{gray!6}{2.2}} & \cellcolor{gray!6}{0.8433274} & \cellcolor{gray!6}{0.1659844}\\
\hline
\end{tabular}
\end{table}

\begin{figure}
\includegraphics[width=1\linewidth]{images/howells_models_bootstrap} \caption[Bootstrap of models in Howell's datset.]{Bootstrap of models in Howell's datset. Bootstrap of the different models on the inner and outer AVE: Model 0 has just trancriptomic and microbiome data, model 1.2 has transcriptomic, microbiome and sample data and model 2.2 has transcriptomic, microbiome and sample data split in different blocks. The bigger points are the models on the original dataset.}\label{fig:howell-bootstrap}
\end{figure}

The bootstrapping showed that model 1.2 has indeed higher inner AVE values than model 2.2 and is more stable than model 1.2.
While model 0 shows a high variation according to which samples are selected.

\begin{figure}
\includegraphics[width=1\linewidth]{images/howell-mcia} \caption[MCIA dimensions in the Howell's dataset.]{MCIA dimensions on the Howell's dataset. The first dimensions separates by location.}\label{fig:howell-mcia}
\end{figure}

If we look at the classification of the models on figure \ref{fig:howell-auc-plot}, we can see that models 1.2, 2 and 2.2 classify perfectly the samples by the transcriptome into the location of the sample.

\begin{figure}
\includegraphics[width=1\linewidth]{images/howell-rgcca-auc} \caption[AUC of the RGCCA models in the Howell's dataset.]{AUC of the RGCCA models in the Howell's dataset. The classification of the localization of the sample according to the first component of the gene expression of the models generated with RGCCA on the Howell's dataset.}\label{fig:howell-auc-plot}
\end{figure}

The AUC of each model can be seen on \ref{tab:howell-auc}:

\begin{table}[H]

\caption[AUC of the RGCCA models in the Howell's dataset]{\label{tab:howell-auc}AUC of the RGCCA models in the Howell's dataset. The classification of the location of the sample according to the first componenent of the models shown. Model 0, 2 and 2.2 have a perfect classification of the samples to their respective location.}
\centering
\begin{tabular}[t]{|>{}c|>{}c|}
\hline
\em{\textbf{model}} & \em{\textbf{AUC}}\\
\hline
\textbf{\cellcolor{gray!6}{0}} & \cellcolor{gray!6}{0.6255259}\\
\hline
\textbf{1} & 0.5974755\\
\hline
\textbf{\cellcolor{gray!6}{1.2}} & \cellcolor{gray!6}{1.0000000}\\
\hline
\textbf{2} & 1.0000000\\
\hline
\textbf{\cellcolor{gray!6}{2.2}} & \cellcolor{gray!6}{1.0000000}\\
\hline
\end{tabular}
\end{table}

In this dataset we also focused on the most important ASV according to the model 2.2 that were present in more than 2 samples that in total were present in the whole dataset.
These ASV were summarized to a single value and then used to calculate the AUC, which was 0.85.
The dot product of the ASV and genes were also calculated and used to find out which ASV are related to which genes.

\hypertarget{between-datasets}{%
\subsection{Between datasets}\label{between-datasets}}

The HSCT genes were compared to the Howell's genes from model 2.2.
There are 3580 selected on model 2.2 in the HSCT dataset and 2189 genes on the Howell's dataset.
From them the 1228 genes in common were analyzed for which GO terms and pathways they are enriched.
The results is represented on \ref{fig:pathways-hsct-howell}.

\begin{figure}
\includegraphics[width=1\linewidth]{images/pathways_hsct_howell} \caption[Pathways from common genes on HSCT and Howell's dataset.]{Significance of pathways on common genes in HSCT and Howell's dataset ordered by p-value, size according to the number of genes on the pathway found on the dataset and color blue for pathways and red for gene ontologies of biological process.}\label{fig:pathways-hsct-howell}
\end{figure}

\hypertarget{discussion}{%
\chapter{Discussion}\label{discussion}}

In this chapter we will summarize the main findings in relation to the broad research community and other work as well as the impact of the results on further research in or clinical practice.

\hypertarget{preliminary-steps}{%
\section{Preliminary steps}\label{preliminary-steps}}

The quality of the initial RNAseq and 16S data is crucial for a valid integration analysis.
In sequencing data, it is very important to avoid contamination and have enough amount of data for the analysis.
To avoid contamination it is very important to control the protocol used to process the samples before the sequencing.
In addition control samples can be added, a blank to correct for extraneous material and a sample with known content to confirm that the sequencing worked correctly.

Our lab uses a well established protocol to isolate and sequence RNA (Described on methods' section \ref{processing}) and we rarely encounter problems during this phase.
In contrast, DNA extraction and 16S sequencing had to be established and optimized, initially with the guidelines and support from collaborators especially Dr.~Ilias Lagkouvardos.
Indeed, as we encountered some problems we added blank samples to control for bacterial contamination during sample pre-processing or sequencing.

We initially processed the 16S data to obtain OTUs.
OTUs were the standard some years back {[}\protect\hyperlink{ref-callahan2017}{198}{]}.
However, OTUs are not comparable between studies even those that use the same primers.
For this reason, and after a suggestion from a reviewer (on the review process of {[}\protect\hyperlink{ref-revilla2021}{197}{]}), we moved to process later 16S sequencing datasets to obtain ASV.
ASV allow the comparison of the taxonomic imputation between studies using the same primers.
However, comparing 16S taxa ASV or OTUs is hard and selecting the right tool to compare them is important {[}\protect\hyperlink{ref-nearing2022}{199}{]}.

It is also worth keeping in mind that several species have variable number of 16S rRNA genes.
Higher sequence counts of certain 16S might not mean higher abundance of that species compared to another with lower 16S counts.
However, the exact copy number status may change even within the same bacterial species, making the correction difficult {[}\protect\hyperlink{ref-louca2018}{200}{]}.
It is however possible to accurately correct for copy number variation on mock populations where the species and the genomes are known (or at least the 16S rRNA), but harder or impossible on samples whose composition is unknown.
To our knowledge there is no available method yet to do this.

Furthermore, the precision of 16S sequencing for classification of taxa is not enough to understand its role in the gut.
Adherent invasive cells, are bacteria share genetic content with but show different behavior {[}\protect\hyperlink{ref-nadalian2021}{201}{]}.
These bacteria are known to be more abundant on patients with IBD {[}\protect\hyperlink{ref-nadalian2021}{201}{]}.
Thus, bacteria with the same complete sequence can play a different role \emph{in vivo}.

Most of these steps were, however, out of my control as they were performed by a lab teammate or by collaborators.
Other concerns, like primers used, 16S regions amplified are currently unavoidable.

\hypertarget{discussion-datasets}{%
\subsection{The datasets}\label{discussion-datasets}}

There is not an established methodology to calculate the size of cohorts for interaction or multi-omic studies.
Indeed, there are no clear rules on which alpha power or which kind of relations can be tested.
This might be due to the lack of mathematical background and modelization of the relevant relations in biology.
Further research in this area might help finding which kind of relationships can be expected given certain dataset sizes.
Usually size of the datasets is determined based on practical reasons: either costs or access to patients.

There have been some efforts to create artificial multi-omic datasets {[}\protect\hyperlink{ref-martuxednez-mira2018}{140}--\protect\hyperlink{ref-mccarthy2012}{145}, \protect\hyperlink{ref-chalise2016}{202}, \protect\hyperlink{ref-chung2019}{203}{]}.
This could allow benchmarking different tools, impute missing data and do pilot studies.
These efforts usually focus on RNA-seq, ATAC-seq (DNase-seq), ChIP-seq, small RNA-seq, Methyl-seq or proteomics but not 16S, microbiome or metagenomics data.
In addition, they have also focused on finding relationships between samples missing relationships within cells (regulated DNA, protein recruitment, transcription factors, siRNA regulation, etc. ) and between cells (immune response, adherence, cytokines signaling, etc.) that might surface at the sample level.
To my knowledge there is no accurate artificial method to create 16S datasets or related microbiome datasets.
This made impossible to simulate and compare different tools on a synthetic dataset with known relationships on this thesis to evaluate performance of different methods.

There is also no reference dataset for integration in IBD that accurately represents the disease.
There is a large consortium effort with many samples {[}\protect\hyperlink{ref-humanmicrobiomeprojectconsortium2012}{4}{]}, but this does not contain 16S and RNAseq data from the same location at the same time for as many samples as the datasets used on this thesis.

In order to validate results of methods we are limited to compare their results on different datasets.
However, each dataset is collected with different goals and processed differently.
In addition, authors upload their data to different, not centralized, data repositories, such as, gene expression omnibus (\href{https://www.ncbi.nlm.nih.gov/geo/}{GEO}), European Genome-Phenome Archive (\href{https://ega-archive.org/}{EGA}), European Nucleotide Archive (\href{https://www.ebi.ac.uk/ena/browser/home}{ENA}), among others.
Authors might also provide the processed data as supplementary material on their articles.
Some projects whose primary purpose is providing data for the community establish their own dedicated sites to store the data {[}\protect\hyperlink{ref-humanmicrobiomeprojectconsortium2012}{4}{]}.

There is not a resource were datasets of publications are collected with their characteristics, age, sex, sample location, extraction method, sequencing protocol, etc..
Finding different datasets with comparable data is thus challenging.
Furthermore when pooling together datasets batch effect correction will most likely be necessary.
There are tools to overcome this, but currently only work if the datasets share some features or samples in common {[}\protect\hyperlink{ref-ugidos2020}{204}{]}.
These methods usually require datasets with similar RNA sequencing procedures.

In this thesis there were two in house datasets collected and sequenced: the HSCT and the BARCELONA dataset.
In addition, we looked up for the most similar datasets published to confirm our results outside our own cohorts.
We found several with intestinal 16S and RNAseq data from the same location, to compare with our own datasets.

As explained in the methods' section \ref{methods-hsct}, the HSCT dataset is a very unique cohort of CD patients undergoing hematopoietic stem cell transplant.
This treatment is reserved to patients for which all the other treatment failed and this may be the only way to reach remission.
These patients are closely monitored and followed up for several years.
This treatment is not indicated for UC, so this dataset includes only CD patients.

The BARCELONA dataset (See appendix section \ref{barcelona}) includes samples from CD and UC collected prospectively in patients starting biological treatment and followed up for up to a year.
Patients had shorter disease duration but were not refractory to biologic treatments and may even be naive to any biologic.
Thus, analysis of this cohort could have provided us with a better understanding on the initial relationships between the microbiome and the mucosa of IBD patients.

Unfortunately, the 16S microbial sequence data from the BARCELONA cohort was not of enough qualtiy to make reliable analysis and confidently extract hypothesis or relationships (See appendix section \ref{barcelona}).
It is not clear what happened and therefore we cannot hypothesize how to avoid such problems in the future.
It could have been a problem with the DNA sample and/or of the sequencing process which did not include both positive and negative controls.
Nevertheless, sample sequencing was repeated in an independent sequencing facility to overcome the limitations of the first dataset but problems persisted rendering this data suboptimal for further analysis.

What happened with the BARCELONA dataset highlights the importance of data quality checks.
In the BARCELONA cohort despite no batch effect as checked with \texttt{experDesign} on the second sequencing, the low diversity indices suggested that there was some problem with the microbiome data.
Caution and communication between all team members (i.e, clinicians, technicians and bioinformaticians) is important to discover this kind of problems and ensure the quality of the data.

Other datasets used from published sourced were assumed they had already passed enough controls and were of good quality.
Nevertheless, they were screened to avoid quality issues by visually inspecting several principal components colored by several factors, checking gene expression and the microbiome profile and looking for known gene markers and microbiome taxonomy that might indicate quality problems.
RNAseq data was usually compared using previously described methods on \ref{statistics}.

The Puget's dataset provided a good benchmark to test the methods and performance of RGCCA works.
CNV/CGH is not comparable to microbial data, but the microarray data is similar to the RNAseq data from other datasets.
We were not as much interested on the biology as on learning about the RGCCA method and its applicability to integrate different omics.

The Häsler IBD dataset was obtained using the same sequencing techniques from endoscopic biopsies as our dataset HSCT and BARCELONA.
The 16S data was very similar to the HSCT dataset.
We could confirm that the inteRmodel approach works on more than one dataset.

The taxonomy analysis of the different datasets was done differently.
On HSCT following our collaborators advise we used IMNGS to annotate its microbiota data {[}\protect\hyperlink{ref-lagkouvardos2016}{159}{]}.
Afterwards we used SILVA database to annotate the microbiome data on their corresponding taxa.

The Morgan pouchitis dataset was also related to IBD but it included patients that underwent colectomy and samples are specific of the pouch or pre-pouch ileum.
No healthy samples are included and there is no follow up.
Thus, it was unlikely that a classification of the location could be achieved and that classification of the samples according to the microbiota could not be based on the disease (as all of them had undergone the same procedure and had the same disease type).
Nonetheless, there seems to be a partial separation by location on the models on \ref{fig:morgan-models} that could partially be explained by different degrees of inflammation.

The Howell dataset includes both pediatric CD and UC (and non-IBD) samples.
It is very similar to our BARCELONA dataset.
As expected, time of the disease duration of these patients is lower, however, they were not followed up at different time points.

Overall, while challenging we were able to identify a few published datasets that were used to validate and compare the different models.
Running such tests is essential to establish the best approach for data integration in the future an validate its results.

\hypertarget{the-methods}{%
\subsection{The methods}\label{the-methods}}

As seen on the introduction there are many different methods available and new tools and methods are frequently being released.
The most up to date list of tools can be added to a collaborative list that was created with the purpose of providing access to the growing list of methodology of the scientific community.
Methods differ in their quality, usage and the quality of the software.
Some of them have been tested on several datasets but the most important validation process a method has to undergo is the mathematical validation.

With the increase of available analytical tools side by side comparison has become increasingly important.
There are reviews that apply different tools to the same datasets {[}\protect\hyperlink{ref-cantini2021}{130}{]}, some are more theoretical {[}\protect\hyperlink{ref-bersanelli2016}{205}{]}, others are focused on a different area like metabolomics {[}\protect\hyperlink{ref-cavill2016}{100}{]}.

Few of these methods have been applied in IBD datasets.
However, recently there has been a review focused on integration in IBD {[}\protect\hyperlink{ref-sudhakar2022}{148}{]}.
In this publication the authors suggest that one must be mindful of the gap between the experimental conditions and the real world.
It also encourages to collect more data about the exposome (the environment patients are exposed to).
It ends up advising to set up guidelines for multi-omic studies tailored to the field, coordinate a global framework to prevent redundant studies and to ensure efficient funding and resources and disseminate training and education on computational approaches to analyze multi-omic datasets.

Current methodological approaches focus on comparing tools on (several) previously published datasets {[}\protect\hyperlink{ref-cantini2021}{130}{]}, an approach that we have taken too.
This approach was not used to compare different tools, but to validate findings of one dataset in other datasets.
This is especially important due to the lack of golden datasets or a way to reliable simulate datasets as discussed on the previous section \ref{discussion-datasets}.

In IBD, many studies focus on finding some genes or bacteria to answer a narrow question they have in mind, like which bacteria are related to inflammation {[}\protect\hyperlink{ref-tang2017}{81}{]} or disease activity.
In this thesis, the focus was on finding a good representation of the relationships that identify groups of genes and bacteria that were relevant to the disease in an orchestrated manner.
We made the assumptions that the microbiome composition and the host's transcriptome were related.
This assumption is backed up by several other previous studies supporting this relationship {[}\protect\hyperlink{ref-thomas1998}{41}, \protect\hyperlink{ref-cornish2008}{42}, \protect\hyperlink{ref-holmes2012}{206}--\protect\hyperlink{ref-brand2021}{208}{]}.

Tools that relate the variance of a block with other variables, both numeric and categoric, are needed to search which variables are important.
PERMANOVA and \texttt{globaltest} served that purpose, but they do not give any insight into which specific microbial species are driving the association between the microbiome and the variables {[}\protect\hyperlink{ref-susin2020}{209}{]}.
In addition, we could be missing some other important variables.
It is known that other factors beyond the omics data collected, mainly environmental factors, genetic susceptibility and the immune response may play a role {[}\protect\hyperlink{ref-sartor2006}{210}{]}.
Sudhakar \emph{et. al.} {[}\protect\hyperlink{ref-sudhakar2022}{148}{]} recommend being conscious of the gap between the data available and the biological process.
One variable we did not keep track was the microbial load which is linked to the gut's microbiome community variation {[}\protect\hyperlink{ref-vandeputte2017}{211}{]}.
This needs to be quantified at the time of DNA isolation and most studies, including our own lack this data.

We tried to find which genes and bacteria are correlated between them using \texttt{WGCNA}, a tool designed to find common co-occurring patterns based on correlations.
It requires homogeneous samples, with a minimum of 12 samples per condition.
However, when applied to the whole dataset there might be too much variance in order of WGCNA to find the proper signal.
As we have patients of different ages and samples from different intestinal segments, both variables are highly related to the intestinal microbiota, this might be the reason why this method failed to achieve a good fit on the scale free topology with around 100 of mean connectivity.
In addition, having microbiome and RNASeq in the same matrix, would probably be hard for the process to find good relationships if we applied the same normalization process to both of them without escalation.
We could have tried to make smaller groups and then compare the modules between them but groups may have been too small since samples were from multiple segments and conditions.

We briefly considered using \texttt{STATegRa}.
However, it is not possible to model specific interactions between blocks.
The method implemented on \texttt{STATegRA} might be useful for cases where there is a great agreement between blocks or were environmental factor do not play a huge role on any of the blocks of data.

To identify related variables other methods use correlations between the variables {[}\protect\hyperlink{ref-vila-casadesuxfas2016}{212}{]}.
On our dataset we visually explored the correlations for all the datasets, but the significant correlations were usually driven by an outlier, or there was not a good fit of the data due to missing data (data not shown).
We tried filtering those correlations identified by the models, being less restrictive by removing those samples that did not have microbiome presence on at least 5 samples, and removing those samples without that microbiome presence, separating based on intestinal segment and a combination of all.
None of these variations provided a clear insight over which genes were correlated to which bacterial signatures.
This is in contrast to other publications that relied purely on the Spearman \(\rho\) metric {[}\protect\hyperlink{ref-hasler_uncoupling_2016}{80}{]}.

We developed and applied \texttt{BaseSet} to find these relationships using a different approach.
However, it failed because it is computationally expensive to calculate the likelihood of 1500 variables; there are too many combinations.
In addition, the numeric precision of said calculations suffers from the floating point problem and must be carefully considered {[}\protect\hyperlink{ref-goldberg1991}{213}{]}.
To support multiplying more than 1000 float numbers a different strategy such as using log values might be better.
We could not come up with a better strategy to find all the combinations needed, perhaps a better method exists that could be used to find which are the terms more influential to the end result.
During the peer-review process of the package for its acceptance on rOpenSci, some concerns were raised about conflating probabilities with fuzzy-sets.
For all these reasons this approach was no longer pursued.
However, the package was mentioned as \href{https://rviews.rstudio.com/2020/12/22/november-top-40-new-cran-packages/}{top 40 packages} added on CRAN that month and it will be useful under other circumstances or when less combinations are possible.

The development of the \texttt{experDesign} package helped us to avoid batch effects on the sequencing step.
However, as seen in the previous section, batch effects were not completely prevented.
As discussed in the related publication {[}\protect\hyperlink{ref-revillasancho2021}{193}{]}, there are several tools already focused on this problem: \texttt{OSAT} ({[}\protect\hyperlink{ref-yan2012}{214}{]}), \texttt{anticlust} ({[}\protect\hyperlink{ref-papenberg2020}{215}{]}) and \texttt{Omixer} ({[}\protect\hyperlink{ref-sinke2021}{216}{]}).
But these tools have some shortcomings, that are covered by \texttt{experDesign}: \texttt{OSAT} cannot handle missing values and does not work well for arbitrary batches, \texttt{anticlust} only accepts numerical variables but it is based on a powerful mathematical theorem and \texttt{Omixer} has bugs that prevent comparing with the other tools with no possible workaround.\\
In addition, \texttt{experDesign} received \href{https://github.com/llrs/experDesign/issues/22\#issuecomment-1033508578}{requests} to have a new feature for expanding experiments.
This might help improve the quality of bigger datasets to ensure they can be extended in several sequencing runs.
This feature would be useful for multi-omics datasets or in big cohorts to minimize batch effects associated with long running collection of samples.

We selected \texttt{MCIA} as a baseline to compare our method because it works well on a wide range of datasets, has a good documentation as well and it is widespread used.
The method was developed after \texttt{RGCCA} and recently there have been publications that show that it outperforms other methods on its versatility on different contexts {[}\protect\hyperlink{ref-cantini2021}{130}{]}.
On the dataset analyzed we found that it performed similarly to \texttt{inteRmodel} but this will be discussed on section \ref{evaluating-models}.

\hypertarget{designing-models}{%
\section{Designing models}\label{designing-models}}

Previous publications using \texttt{RGCCA} in IBD have focused on validating genes \emph{DUOX2} and \emph{APOA1} as inflammation predictors ({[}\protect\hyperlink{ref-tang2017}{81}{]}) from previously published articles {[}\protect\hyperlink{ref-haberman2014}{217}{]}.
Some publication tried to summarize the existing relationships in IBD {[}\protect\hyperlink{ref-hasler_uncoupling_2016}{80}{]}, but none were focused on finding the relationships in IBD using \texttt{RGCCA} as we did.

There are many variables outside transcriptomics and the microbiome that may be relevant in disease homeostasis.
These variables should be included on the models to find which genes and bacteria are truly related and not confounded by other factors.
In addition, the relationships between the blocks are unknown on both the strength and interaction.
To model the relationships the connections between blocks had to be selected on \texttt{RGCCA}.
Last, the variables that belong to a block should be carefully considered as the assumption is that the whole block is correlated with other blocks connected.

If one has preexisting theories about the data, a specific model can be used stating these known or hypothetical relationships.
However, if new relationships are being explored or no prior assumptions on the data are held the models should be created with random links between blocks, and evaluate which model is better.

The connections tested required that all models should have all blocks indirectly connected to other blocks with no blocks left unconnected.
This avoids optimizing two different networks of blocks that are not connected between them, thus, forcing the model to represent all the information.

Typically blocks are defined by each omic data origin and no other information is included.
However, we knew that the transcriptome is mainly related to the location of the samples, as we have seen it on the PCAs, and we expected that the microbiome would be more related to the patients demographic characteristics and influenced by dietary and other environmental factors.
This is especially important because in our datasets we have samples from the same patient and time from multiple locations.
On other studies there are less samples per individual and timepoint (if there are several timepoints) {[}\protect\hyperlink{ref-hasler_uncoupling_2016}{80}, \protect\hyperlink{ref-morgan2015}{150}, \protect\hyperlink{ref-howell2018}{151}{]}.

On models of family 2, variables were grouped according to type into the Demographic block, Time block, and Location block (see tables on design of Puget's \ref{tab:puget-model2-2}, \ref{tab:puget-model2}; HSCT \ref{tab:hsct-model2} \ref{tab:hsct-model2-2}, \ref{tab:hsct-model2-3}; Häsler: \ref{tab:hasler-model-2-1}, \ref{tab:hasler-model-2-2}, Morgan's \ref{tab:morgan-model2}, \ref{tab:morgan-model2-2}, Howell's \ref{tab:howell-model2-2} models).

The exploration of design on Puget's dataset, and the datasets analysis of HSCT, Häsler and Morgan's datasets was published after peer review {[}\protect\hyperlink{ref-revilla2021}{197}{]}.

\hypertarget{evaluating-models}{%
\section{Evaluating models}\label{evaluating-models}}

To evaluate a model \texttt{RGCCA} provides the inner and outer average variance explained (AVE); (see section \ref{rgcca-output}).
As the inner AVE measures how good does the data fit in the model, we used it to evaluate and compare the models.

Furthermore, bootstrapping was used to evaluate the fitness of a design on a diverse collection of datasets.
Although on \texttt{inteRmodel} there is the option to use a leave-one-out procedure, we did not use it to evaluate the fitness of the models.

Using an external cohort to validate the same model, or using a different method to see if it finds the same relationships or explains the data as accurately is also a common approach to evaluate and validate models.
Using the same approach on different data helps to ensure the replicability of the results {[}\protect\hyperlink{ref-community2021}{218}{]}.

\begin{figure}
\includegraphics[width=1\linewidth]{images/turing_way_reproducible-matrix} \caption[Reproducibility matrix]{Reproducibility matrix indicating the terminology used between using the same method and the same data. Figure from The Turing Way: A Handbook for Reproducible Data Science (Version v1.0.1).}\label{fig:reproducible-matrix}
\end{figure}

We used the same approach on four different cohorts, with different origins and types of samples, but all related to the IBD popultation including 16S data and intestinal transcriptome.
Some of them have multiple samples from the same individuals while others do not {[}\protect\hyperlink{ref-hasler_uncoupling_2016}{80}{]}.

We also compared our method with a different one to see how generalizable are the results.
Of the multiple methods available we used \href{https://bioconductor.org/packages/omicade4}{\texttt{MCIA}} {[}\protect\hyperlink{ref-mengMultivariateApproachIntegration2014}{219}{]}.
We compared it with \texttt{inteRmodel} by looking at the area under the curve for classifying the samples with the canonical component of the transcriptome data according to their location.

The procedure of separating independent variables in their own block of data and later search the best model that fits the data provides a good strategy that should be considered for integration efforts.
The procedural method of searching a model and testing it is implemented on \href{https://llrs.github.io/inteRmodel/}{\texttt{inteRmodel}}.
But the most important thing is to consider which variables are independent of which and if they can be separated into a block for later usage on the modeling (See {[}\protect\hyperlink{ref-pearl2011}{220}{]}).
If not done properly it can lead to undesired conclusions like the effect known as \href{https://en.wikipedia.org/wiki/Simpson\%27s_paradox}{Simpson's paradox} {[}\protect\hyperlink{ref-simpson1951}{221}{]}.
Unfortunately this cannot be automated and it is left for the practitioner.

Splitting the variables of each sample into several blocks forces \texttt{RGCCA} to adjust for a new canonical dimension.
The omics block such as gene expression and 16S data could be split, as the expression of some genes influence other genes, such as \href{https://en.wikipedia.org/wiki/Transcription_factor}{transcriptor factors} (First mentioned on {[}\protect\hyperlink{ref-stillman1984}{222}{]}) , \href{https://en.wikipedia.org/wiki/MicroRNA}{miRNA} {[}\protect\hyperlink{ref-lee1993}{223}{]}, and \href{https://en.wikipedia.org/wiki/Small_interfering_RNA}{siRNA} {[}\protect\hyperlink{ref-hamilton1999}{224}{]}.
All these interactions and regulations could distort the canonical correlations.
However, the nature of this relationships is not linear and the interaction between them is multiple and very complex.
Its complexity has prevented to accurately account for all subcellular reactions at speeds that could be useful (it has only been recently accomplished for a prokaryote cell {[}\protect\hyperlink{ref-thornburg2022}{225}{]}).
In addition, these interactions are time dependent, not linear and are highly interconnected between many variables.
For these reasons gene expression and 16S blocks were not split into several blocks.

Besides comparing the results of different methods, these models need to be evaluated by the insights they provide on the biological system they are being applied to, in our case IBD.
So far the models were only discussed on their technical merits.

It is known that the mucosal transciptome is related to sample localization (of)i.e.
small or large intestine) and disease activity {[}\protect\hyperlink{ref-criss2021}{226}{]}.
This can be seen on the PCAs which separate colon an ileum on the first dimension: See the PCAs of the different datasets \ref{fig:hsct-pca-rnaseq}, \ref{fig:hasler-pca-rnaseq} and \ref{fig:howell-pca-rnaseq}.
The difference is so great that many times the colon and ileum samples are analyzed separately.
As such, it was a reasonable assumption to expect models to reflect these differences on the gene expression canonical component.

The variability of microbiome difficults finding clear patterns (See the PCAs of the different datasets: \ref{fig:hsct-pca-16s}, \ref{fig:hasler-pca-16s}, \ref{fig:morgan-pca-16s} and \ref{fig:howell-pca-16s}).
In healthy humans, it has been suggested that there is a shared microbial patterns among groups of individuals {[}\protect\hyperlink{ref-arumugam2011}{227}{]}. Such theory proposes three groups of similar microbiota composition, named enterotypes. However, the enterotypes classification is not unanimously accepted {[}\protect\hyperlink{ref-koren2013}{228}, \protect\hyperlink{ref-cheng2019}{229}{]}.
Some suggest it is an artifact of the methodology used.

There are many factors that influence the gut microbiome {[}\protect\hyperlink{ref-hasan2019}{230}{]}.
But the role of microbiome in IBD has gained a lot of attention in the last 10 years {[}\protect\hyperlink{ref-khanna2014}{17}, \protect\hyperlink{ref-bringiotti2014}{231}, \protect\hyperlink{ref-lloyd-price2019}{232}{]}.
The microbiome has been associated for instance with treatment response in CD and suggested it could be useful for building an improved classifier for CD {[}\protect\hyperlink{ref-douglas2018}{233}{]}.

In addition, it is known that microbiome of patients in remission is different from non-IBD patients {[}\protect\hyperlink{ref-halfvarson2017}{234}{]}.
So even if two people are apparently healthy at a given time their microbiome might not be similar.

The relapsing nature of IBD, suggests that at different time points the gene expression or the microbiome might be different.
For this reason, time is an important variable when modelling the disease.
If samples at multiple timepoints are available the time difference should be taken into account to identify the state of the disease for each patient.
Even in healthy non-IBD patients, multiple samples with microbiota from the same patients could provide common microbiome signature, which can help identify altered microbiome states later on.

Of the cohorts analyzed in this thesis, the HSCT and the Häsler dataset are the only ones including time related variables.
However, the Häsler dataset only had age at date of sampling, and there were not several samples for the same patients at different timepoints.
This leaves the HSCT dataset as the only one with multiple timepoints from the same patients.
Having that many samples from the same patient might explain why the classification of the disease on this cohort works so well, despite being a cohort of complicated patients with several background treatments, refractory to all previous treatments, some of them having undergone surgery too and undergoing severe immunosupression during follow up.
Despite all of this, there was large agreement in the genes and pathways identified as relevant between datasets (See figure \ref{fig:pathways-hsct-howell}).

Comparing the microbiome between datasets is less straight forward.
If the same primers are used the ASV could be directly compared.
However, the ASV' length might be different.
In addition, in this thesis we used dataset obtained with different primers and also OTUs.
For these reasons a direct comparison is not possible.
A comparison of the taxonomy of the annotated microbiome on the datasets is the next possibility.

However, some datasets used did not provide the annotation (Häsler's and Morgan's datasets).
So we are are left with comparing OTUs (HSCT dataset) with ASVs (Howell's).
Despite the errors on ASV annotation ({[}\protect\hyperlink{ref-edgar2018}{235}{]}), OTUs from the HSCT dataset can be compared to ASV from the Howell's dataset.
There were very few common taxonomic levels selected on both datasets (data not shown).
This was not a surprise, as there are many factors that influence one's microbiome profile and the diversity indexes already showed high differences between samples of the same dataset.

The microbiome of each dataset seemed capable to classify the samples according to the disease (data not shown).
However, on further evaluation via bootstrapping this classification was not significant, as different relatively big amount of ASVs were able to classify the samples.
This might be due to the uniqueness of microbial composition to each sample.

In Howell's and Häsler dataset, the best model based on AVE is 1.2; which separates the microbiome component by location too (See Figure \ref{fig:hasler-models} and table \ref{tab:hasler-aves} for Häsler dataset and figures \ref{fig:howell-models} and table \ref{tab:howell-models-ave} for Howell dataset).
The 2.2 models according to the inner AVE were not that far away from models 1.2.
This indicates that the relationship of the microbiome is stronger than with the location, but both factors should be considered when looking for relationships of genes with microbiome.

\hypertarget{implications}{%
\section{Implications}\label{implications}}

On this thesis several methods has been developed to help multi-omic data integration.
\texttt{experDesign} was implemented during the initial steps when moving from bench to in-silico analysis.
\texttt{BaseSets} and \texttt{inteRmodel} are useful for computational analysis.
\texttt{BaseSets} might help beyond integration analysis such as single cell annotation\footnote{See an exploration of this using AUCell \url{https://bioconductor.org/packages/AUCell} on this website: \url{https://llrs.github.io/BaseSet_scRNAseq/AUCell.html}.}.

Some studies using host transcriptomics of fecal wash infer inflammation without colonoscopy {[}\protect\hyperlink{ref-ungar2022}{236}{]}.
This would help patients to avoid an unpleasant experience, and reduce the usage of clinical facilities.
It could be possible that just sequencing the intestinal microbiome could be enough to identify the patients' disease.
However, this requires further validation to ensure that the diagnosis is accurate enough on a diverse and big population.
There are already studies on this direction, not only for IBD or intestine but for several different human regions {[}\protect\hyperlink{ref-humanmicrobiomeprojectconsortium2012}{4}{]}.

With our studies we hoped that we would provide which bacteria played a role on the disease, or which genes and bacteria are related in IBD.
We obtained a list of putative genes and bacteria but not a clear pairing of which genes interact with which bacteria.
This could mean that the microbiome community is related to all the genes.
It could also mean that the methods are not powerful enough to find more tailored relationships as normalizations and generalizations present on \texttt{RGCCA} do not allow to subgroup or classify the variables already detected.
Maybe a different method that would not depend on the same principles might be able to detect finer relationships on the variables selected.

There is a disconnect between the computational side and the experimental side, driven by the difficulty to design and perform an experiment to test the new information that multi-omic experiments provide.
This is referred as the gap in other publications {[}\protect\hyperlink{ref-sudhakar2022}{148}{]}.
Closing this gap between the computational methods and the data origin and practical usages would potentially require closer collaboration between clinicians, statisticians, bioinformaticians and research software engineers; in addition to creative ideas accounting for the standard procedure on hospitals and points of care.

However, it is also possible that further developments or creatively applying statistical methods might help closing the gap.
For instance, there are many combinations of possible interactions.
These interactions between microbiome and genes are currently hard to explore statistically.
Further research on how to reduce the space of possible combinations of bacteria or evaluate which combinations are more important might be useful on the future.
This might involve using more network integration methods.

Krassowski's \emph{et. al.} ({[}\protect\hyperlink{ref-krassowski2020}{95}{]}) advise on software engineering and reproducibility practices to share awareness with new researchers in multi-omics for end-to-end workflow.
In addition the recent recommendation about integration in IBD ({[}\protect\hyperlink{ref-sudhakar2022}{148}{]}) suggests that there is still much to be done to increase the results on these projects and the effectivity.
We agree with them and encourage other researchers to carefully consider the proposed methodology before starting their own projects.

There seems to be a tendency of multi-omics projects to focus on metagenomics, metatranscriptomics and metaproteomics and abandon plain 16S sequencing {[}\protect\hyperlink{ref-zhang2019}{237}{]}.
This could be explained by the numerous problems that 16S sequencing has.
Some of these are avoided or solved by using these other omics techniques.
For instance the metatranscriptomics and metaproteomics of bacteria could help detect what is actively being produced by the microbiota and the metagenomics, what genes are actually there not just restricted to 16S rRNA sequences.
But these methods also have their shortcomings.
Metagenomics still does not detect adherent invasive cells and metatranscriptomics and metaproteomics can not explain which bacteria is expressing which genes (even if paired with metagenomics or 16S data).
The single cell revolution might provide more insights in the future, but at the moment the first studies of single-cell multi-omics are being published.

Perhaps as suggested in other publications network methods might be able to provide more detailed information about the relationships {[}\protect\hyperlink{ref-jiang2019}{238}{]}.
But, it is unclear how complete and valid these networks are.
Current literature focus on already known and studied genes instead of on more novel and with higher relevant genes {[}\protect\hyperlink{ref-haynes2018}{239}{]}.
By extension, this translates in bias in networks and pathways resources available for pathway analysis and integration methods.
This could explain why there are many genes selected by the models with few pathways or gene sets.

To study complex systems, new technologies like ``organ-on-chip'' or more specifically ``gut-on-chip'' are being developed {[}\protect\hyperlink{ref-collij2021}{240}{]}.
These systems expand on the already useful technique of using organoids to better mimic the epithelial cross-talk in the laboratory.
Latest developments include the addition of separate environments and distinct flow on these environments.
By introducing gut microbiota, these systems will help studying the interaction of the gut microbiota and the intestinal epithelium.
However, it is not clear how accurately account for other factors such as the presence of inflammation.

In summary, as shown with the PERMANOVA approach, several factors affect the disease and the relationship between gut's microbiome and gut's mucosa.
Analysis or comparisons without taking them into account might provide misleading or false results.
To our knowledge this is the first study using these variables as part of the integration study with canonical correlations.
We hope this provides a first approximation to accurately understand the relationships between genes and bacteria in IBD.

\hypertarget{conclusions}{%
\chapter{Conclusions}\label{conclusions}}

In this doctoral thesis, we assessed the relationships between the microbiota and the gut transcriptome in inflammatory bowel disease.

It allows us to conclude the following:

\hypertarget{study-1-multi-omic-modelling-of-inflammatory-bowel-disease-with-regularized-canonical-correlation-analysis}{%
\section{Study 1: Multi-omic modelling of inflammatory bowel disease with regularized canonical correlation analysis}\label{study-1-multi-omic-modelling-of-inflammatory-bowel-disease-with-regularized-canonical-correlation-analysis}}

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\tightlist
\item
  Applying methods that use information about the samples show better results than omitting this information.
\item
  Correlations are not enough to identify relationship between genes and microorganisms.
\item
  The \texttt{inteRmodel} method provides consistent connections between blocks on different datasets of inflammatory bowel disease.
\end{enumerate}

\hypertarget{study-2-genes-and-microbiome-relationship-on-inflammatory-bowel-disease}{%
\section{Study 2: Genes and microbiome relationship on inflammatory bowel disease}\label{study-2-genes-and-microbiome-relationship-on-inflammatory-bowel-disease}}

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\tightlist
\item
  The host's transcriptomics is heavily influenced by location and to a lesser degree by the type of activity.
\item
  The microbiome composition relates better to the sample location than to the disease status but both are almost equally important
\item
  Microorganisms highly tied to intestinal location might indicate the disease type, (Crohn's disease, ulcerative colitis or non-IBD) of the patients, but microorganisms related to the disease are common in all the locations.
\end{enumerate}

\hypertarget{acknowledgments}{%
\chapter{Acknowledgments}\label{acknowledgments}}

A la directora, tutora, jefa i tants papers que fas, moltes gràcies Azu per haver estat a sobre de la tesis, he aprés molt de tot sobre com treballar amb intensitat i eficiencia, tocant de peus a terra. També per preguntar i tenir presents sobre com es pot fer servir això i com pot ajudar cada idea que tenia.

Gràcies Juanjo per donar-me la oportunitat d'entrar a IDIBAPS i per tota la ajuda al llarg d'aquests anys per ser un bioinformàtic independent. Sempre has estat disponible quan no sabia com seguir i aportaves alguna idea o paquet per probar.

A tot el laboratori per el bon ambient que hi ha hagut fent les alegries més grans al llarg de aquests anys i les adversitats més lleugeres.
Cadascú ho ha fet a la seva manera: a l'Ana per ser el complement perfecte de l'equip bioinformàtic i sempre estar disponible per aconsellar-me.
L'Alba pel seu optimisme i perseverancia, también por tu buen criterio diseñando, espero que algo se me pegue.
A l'Isa pel seu detallisme i exemple de fer bé les coses, les orquidies siempre están preciosas!
La Marisol per la seva senzillesa i estar sempre pendent dels petits detalls de la paperasa.
La Elisa per la seva alegria i il·lusió que mai defalleixen i s'encomanen.
La Victòria per la seva paciència i consells sobre com descriure els projectes.
L'Angela per acceptar amb paciència les meves sortides friquis.
La Míriam perqué sempre ens toca moure trastos, a veure si algun dia montem el wallapop científic! i l'Iris que sempre té alguna cosa a aportar.\\
Moltes gràcies a totes, només lamento no haver aprés més de vosaltres.

Hi ha persones que estan sempre com la Pepa, gràcies per no tancar-me mai el email, i altres que marxen a altres projectes.
Gràcies per compartir una part d'aquest projecte amb mí:
l'Aida per tota l'empenta que ha posat en aquests anys, moltes gràcies per introduir-me en el món de la microbiota! també per la teva acollida al lab. La Montse per organitzar els primers viatges fòra i ensenyar-me a organitzar-me.
La Núria per introduir-me en el grup i animar-me a fer el doctorat amb l'Azu.
També en Daniel pel curset de processament de RNAseq.

A tots de l'equip de l'hospital, per intentar entendre aquestes caixes negres de les tesis i ajudar a que no perdi de vista l'objectiu de la recerca; en especial al Julià, per fer sempre les millors preguntes i a la Maica per resoldre molts dubtes a l'inici.

A tot l'equip del Pau que em van acompanyar en els meus primers passos en la investigació, la Bea, la Júlia, la Délia, la Mar, en Luis, la Elisa\ldots{}
Alguns que ja no estan per la planta 3 com en Josep o la Carolina.

A tota la familia, els que estan més aprop i els que estan més lluny.
Als pares per seguir amb interès tot el procés i procurar entendre alguna cosa.
A la Maria per convidar-me sovint a casa seva a descansar.
A la Maria del Mar per la teva alegria i plantejar reptes amb el python.
A la Marta pels consells, el cotxe i les excursions.
A l'Ana per seguir amb interés la investigació i estar pendent dels avanços sobre la microbiota.
A l'Albert per convidar-me a dinar sovint i conèixer racons i històries de Barcelona o la familia.

Als amics de Xaloc, en Mario per posar la casa tantes vegades al llarg de la tesis, un día tienes que venir a la mía!
L'Albert Mateo per les refrescants sortides en bici i excursions, a ver si hacemos más y con todos!
A, l'Alejandro que sempre ha estat un suport moral com a doctorand.
En Bruno per acollir-me a casa seva (he de repetir la visita a Munich!).
A l'Edu per estar sempre disposat a acostar-me en cotxe o trobar un lloc fàcil d'arribar en transport públic.
Als amics de la carrera, en Frederic per compartir molts dinars, penes i alegries.
En Joan per marcar el camí, sempre has estat per davant! i l'Aleix, espero que la bioinformàtica vagi a més!

Els amics d'Obenc, el Cristian per tantes aventures que em viscut aquests anys.
En Daniel per ensenyar-me que el temps no es barrera per fer coses (Encara no m'has presentat els teus gossos!).
A en Lluís que tampoc no pares mai i m'has transmés la teva gran ambició.
A en Ricard per totes les discussions sobre el futur profesional, de la programació, els jocs, películes i tantes coses, segur que trobes una manera de passar al seguent nivell!
A en Joaquín per ensenyar-me el món empresarial de software i tants detalls tècnics.
A tots els que han vingut a fútbol, en especial al Josep Mª per ser l'incansable impulsor i treure gent de sota les pedres i a en Josep per ser el més constant, ja chutaré més sovint!

I also received many help from numerous and anonymous people from several corners of the world.
They might have shared some interesting tip, advice or tale on twitter, or answered my questions on \href{https://support.bioconductor.org}{Bioconductor's support forum}, \href{https://biostars.org}{Biostars}, \href{https://bioinformatics.stackexchange.com/}{Bioinformatics SE} or Stackexchange network.
Also those that made and provided the software for free that made all this possible on your own time, R core, CRAN, Bioconductor and rOpenSci team, package developers and maintainers of the \textasciitilde500 packages I use.

Dr.~Häsler for kindly providing data of the samples from their cohort and Dr.~Cristian Hernández and Dr.~Mark S. Silverber for their collaboration and selfless sharing the data of their cohort.

Last but not least, those without all this could not be possible.
Thanks to all the patients that allowed clinicians to collect samples and data for research.
I hope that the knowledge we gain with on this thesis and other studies will bring us closer to heal you.

\hypertarget{references}{%
\chapter*{References}\label{references}}
\addcontentsline{toc}{chapter}{References}

\ChapFrame

\hypertarget{refs}{}
\begin{CSLReferences}{0}{0}
\leavevmode\vadjust pre{\hypertarget{ref-raine2021}{}}%
1. Raine T, Verstockt B, Kopylov U, Karmiris K, Goldberg R, Atreya R, et al. \href{https://doi.org/10.1093/ecco-jcc/jjab112}{ECCO topical review: Refractory inflammatory bowel disease}. Journal of Crohn's and Colitis. 2021;15:1605--20.

\leavevmode\vadjust pre{\hypertarget{ref-jairath2020}{}}%
2. Jairath V, Feagan BG. \href{https://doi.org/10.1016/S2468-1253(19)30358-9}{Global burden of inflammatory bowel disease}. The Lancet Gastroenterology \& Hepatology. 2020;5:2--3.

\leavevmode\vadjust pre{\hypertarget{ref-burisch2015}{}}%
3. Burisch J, Munkholm P. \href{https://doi.org/10.3109/00365521.2015.1014407}{The epidemiology of inflammatory bowel disease}. Scandinavian Journal of Gastroenterology. 2015;50:942--51.

\leavevmode\vadjust pre{\hypertarget{ref-humanmicrobiomeprojectconsortium2012}{}}%
4. Human Microbiome Project Consortium BA, Nelson KE, Pop M, Creasy HH, Giglio MG, Huttenhower C, et al. \href{https://doi.org/10.1038/nature11209}{A framework for human microbiome research.} Nature. 2012;486:21521.

\leavevmode\vadjust pre{\hypertarget{ref-shaw2016}{}}%
5. Shaw KA, Bertha M, Hofmekler T, Chopra P, Vatanen T, Srivatsa A, et al. \href{https://doi.org/10.1186/s13073-016-0331-y}{Dysbiosis, inflammation, and response to treatment: A longitudinal study of pediatric subjects with newly diagnosed inflammatory bowel disease}. Genome Medicine. 2016;8:75.

\leavevmode\vadjust pre{\hypertarget{ref-mukhopadhya2012}{}}%
6. Mukhopadhya I, Hansen R, El-Omar EM, Hold GL. \href{https://doi.org/10.1038/nrgastro.2012.14}{IBD{\textemdash}what role do Proteobacteria play?} Nature Reviews Gastroenterology \& Hepatology. 2012;9:219--30.

\leavevmode\vadjust pre{\hypertarget{ref-zhang2014}{}}%
7. Zhang Y-Z, Li Y-Y. \href{https://doi.org/10.3748/wjg.v20.i1.91}{Inflammatory bowel disease: pathogenesis}. World Journal of Gastroenterology. 2014;20:91--9.

\leavevmode\vadjust pre{\hypertarget{ref-hugot2004}{}}%
8. Hugot J-P. \href{https://doi.org/10.1097/00054725-200402001-00003}{Genetic origin of IBD}. Inflammatory Bowel Diseases. 2004;10:S11--5.

\leavevmode\vadjust pre{\hypertarget{ref-silva2016}{}}%
9. Silva FAR, Rodrigues BL, Ayrizono M de LS, Leal RF. \href{https://doi.org/10.1155/2016/2097274}{The Immunological Basis of Inflammatory Bowel Disease}. Gastroenterology Research and Practice. 2016;2016:2097274.

\leavevmode\vadjust pre{\hypertarget{ref-demattos2015}{}}%
10. de Mattos BRR, Garcia MPG, Nogueira JB, Paiatto LN, Albuquerque CG, Souza CL, et al. \href{https://doi.org/10.1155/2015/493012}{Inflammatory Bowel Disease: An Overview of Immune Mechanisms and Biological Treatments}. Mediators of Inflammation. 2015;2015:493012.

\leavevmode\vadjust pre{\hypertarget{ref-mcgovern2015}{}}%
11. McGovern DPB, Kugathasan S, Cho JH. \href{https://doi.org/10.1053/j.gastro.2015.08.001}{Genetics of Inflammatory Bowel Diseases}. Gastroenterology. 2015;149:1163--1176.e2.

\leavevmode\vadjust pre{\hypertarget{ref-satsangi2006}{}}%
12. Satsangi J, Silverberg MS, Vermeire S, Colombel J-F. \href{https://doi.org/10.1136/gut.2005.082909}{The Montreal classification of inflammatory bowel disease: controversies, consensus, and implications}. Gut. 2006;55:749--53.

\leavevmode\vadjust pre{\hypertarget{ref-horowitz2021}{}}%
13. Horowitz JE, Warner N, Staples J, Crowley E, Gosalia N, Murchie R, et al. \href{https://doi.org/10.1038/s41598-021-84938-8}{Mutation spectrum of NOD2 reveals recessive inheritance as a main driver of early onset crohn{'}s disease}. Scientific Reports. 2021;11:5595.

\leavevmode\vadjust pre{\hypertarget{ref-kumar2019}{}}%
14. Kumar M, Garand M, Al Khodor S. \href{https://doi.org/10.1186/s12967-019-02174-1}{Integrating omics for a better understanding of inflammatory bowel disease: A step towards personalized medicine}. Journal of Translational Medicine. 2019;17:419.

\leavevmode\vadjust pre{\hypertarget{ref-jostins2012}{}}%
15. Jostins L, Ripke S, Weersma RK, Duerr RH, McGovern DP, Hui KY, et al. \href{https://doi.org/10.1038/nature11582}{Host{\textendash}microbe interactions have shaped the genetic architecture of inflammatory bowel disease}. Nature. 2012;491:119--24.

\leavevmode\vadjust pre{\hypertarget{ref-momozawa2018}{}}%
16. Momozawa Y, Dmitrieva J, Théâtre E, Deffontaine V, Rahmouni S, Charloteaux B, et al. \href{https://doi.org/10.1038/s41467-018-04365-8}{IBD risk loci are enriched in multigenic regulatory modules encompassing putative causative genes}. Nature Communications. 2018;9.

\leavevmode\vadjust pre{\hypertarget{ref-khanna2014}{}}%
17. Khanna S, Tosh PK. \href{https://doi.org/10.1016/j.mayocp.2013.10.011}{A Clinician's Primer on the Role of the Microbiome in Human Health and Disease}. Mayo Clinic Proceedings. 2014;89:107--14.

\leavevmode\vadjust pre{\hypertarget{ref-swidsinski2002}{}}%
18. Swidsinski A, Ladhoff A, Pernthaler A, Swidsinski S, LoeningBaucke V, Ortner M, et al. \href{https://doi.org/10.1053/gast.2002.30294}{Mucosal flora in inflammatory bowel disease}. Gastroenterology. 2002;122:44--54.

\leavevmode\vadjust pre{\hypertarget{ref-tamboli2004}{}}%
19. Tamboli CP, Neut C, Desreumaux P, Colombel JF. \href{https://doi.org/10.1136/gut.53.1.1}{Dysbiosis in inflammatory bowel disease}. Gut. 2004;53:1--4.

\leavevmode\vadjust pre{\hypertarget{ref-ott2004}{}}%
20. Ott SJ. \href{https://doi.org/10.1136/gut.2003.025403}{Reduction in diversity of the colonic mucosa associated bacterial microflora in patients with active inflammatory bowel disease}. Gut. 2004;53:685--93.

\leavevmode\vadjust pre{\hypertarget{ref-kostic2014}{}}%
21. Kostic AD, Xavier RJ, Gevers D. \href{https://doi.org/10.1053/j.gastro.2014.02.009}{The microbiome in inflammatory bowel disease: Current status and the future ahead}. Gastroenterology. 2014;146:1489--99.

\leavevmode\vadjust pre{\hypertarget{ref-sender2016}{}}%
22. Sender R, Fuchs S, Milo R. \href{https://doi.org/10.1371/journal.pbio.1002533}{Revised Estimates for the Number of Human and Bacteria Cells in the Body}. PLOS Biology. 2016;14:e1002533.

\leavevmode\vadjust pre{\hypertarget{ref-sankarasubramanian2020}{}}%
23. Sankarasubramanian J, Ahmad R, Avuthu N, Singh AB, Guda C. \href{https://doi.org/10.3389/fmed.2020.606298}{Gut microbiota and metabolic specificity in ulcerative colitis and crohn's disease}. Frontiers in Medicine. 2020;7.

\leavevmode\vadjust pre{\hypertarget{ref-lopez-siles2014}{}}%
24. Lopez-Siles M, Martinez-Medina M, Busquets D, Sabat-Mir M, Duncan SH, Flint HJ, et al. \href{https://doi.org/10.1016/j.ijmm.2014.02.009}{Mucosa-associated Faecalibacterium prausnitzii and Escherichia coli co-abundance can distinguish Irritable Bowel Syndrome and Inflammatory Bowel Disease phenotypes}. International journal of medical microbiology: IJMM. 2014;304:464--75.

\leavevmode\vadjust pre{\hypertarget{ref-ferrer-picuxf3n2020}{}}%
25. Ferrer-Picón E, Dotti I, Corraliza AM, Mayorgas A, Esteller M, Perales JC, et al. \href{https://doi.org/10.1093/ibd/izz119}{Intestinal Inflammation Modulates the Epithelial Response to Butyrate in Patients With Inflammatory Bowel Disease}. Inflammatory Bowel Diseases. 2020;26:43--55.

\leavevmode\vadjust pre{\hypertarget{ref-darfeuille-michaud1998}{}}%
26. Darfeuille-Michaud A, Neut C, Barnich N, Lederman E, Di Martino P, Desreumaux P, et al. \href{https://doi.org/10.1016/S0016-5085(98)70019-8}{Presence of adherent Escherichia coli strains in ileal mucosa of patients with Crohn's disease}. Gastroenterology. 1998;115:1405--13.

\leavevmode\vadjust pre{\hypertarget{ref-tsilingiri2012}{}}%
27. Tsilingiri K, Barbosa T, Penna G, Caprioli F, Sonzogni A, Viale G, et al. \href{https://doi.org/10.1136/gutjnl-2011-300971}{Probiotic and postbiotic activity in health and disease: comparison on a novel polarised ex-vivo organ culture model}. Gut. 2012;61:1007--15.

\leavevmode\vadjust pre{\hypertarget{ref-vanderpool2008}{}}%
28. Vanderpool C, Yan F, Polk BD. \href{https://doi.org/10.1002/ibd.20525}{Mechanisms of probiotic action: Implications for therapeutic applications in inflammatory bowel diseases}. Inflammatory Bowel Diseases. 2008;14:1585--96.

\leavevmode\vadjust pre{\hypertarget{ref-isaacs2008}{}}%
29. Isaacs K, Herfarth H. \href{https://doi.org/10.1002/ibd.20465}{Role of probiotic therapy in IBD}. Inflammatory Bowel Diseases. 2008;14:1597--605.

\leavevmode\vadjust pre{\hypertarget{ref-plaza-diaz2019}{}}%
30. Plaza-Diaz J, Ruiz-Ojeda FJ, Gil-Campos M, Gil A. \href{https://doi.org/10.1093/advances/nmy063}{Mechanisms of action of probiotics}. Advances in Nutrition. 2019;10:S49--66.

\leavevmode\vadjust pre{\hypertarget{ref-morelli2012}{}}%
31. Morelli L, Capurso L. \href{https://doi.org/10.1097/MCG.0b013e318269fdd5}{FAO/WHO guidelines on probiotics: 10 years later}. Journal of Clinical Gastroenterology. 2012;46:S1.

\leavevmode\vadjust pre{\hypertarget{ref-okumura2017}{}}%
32. Okumura R, Takeda K. \href{https://doi.org/10.1038/emm.2017.20}{Roles of intestinal epithelial cells in the maintenance of gut homeostasis}. Experimental \& Molecular Medicine. 2017;49:e338--8.

\leavevmode\vadjust pre{\hypertarget{ref-faria2012}{}}%
33. Faria AMC, Mucida D, McCafferty D-M, Tsuji NM, Verhasselt V. \href{https://doi.org/10.1155/2012/738475}{Tolerance and inflammation at the gut mucosa}. Clinical \& Developmental Immunology. 2012;2012:738475.

\leavevmode\vadjust pre{\hypertarget{ref-hisamatsu2013}{}}%
34. Hisamatsu T, Kanai T, Mikami Y, Yoneno K, Matsuoka K, Hibi T. \href{https://doi.org/10.1016/j.pharmthera.2012.10.008}{Immune aspects of the pathogenesis of inflammatory bowel disease}. Pharmacology \& Therapeutics. 2013;137:283--97.

\leavevmode\vadjust pre{\hypertarget{ref-michielan2015}{}}%
35. Michielan A, D'Incà R. \href{https://doi.org/10.1155/2015/628157}{Intestinal Permeability in Inflammatory Bowel Disease: Pathogenesis, Clinical Evaluation, and Therapy of Leaky Gut}. Mediators of Inflammation. 2015;2015:628157.

\leavevmode\vadjust pre{\hypertarget{ref-mayorgas2021a}{}}%
36. Mayorgas A. \href{http://www.tdx.cat/handle/10803/672414}{Human Primary Organoid-Derived Epithelial Monolayers as a Novel Strategy for the Study of Adherent Invasive Escherichia coli pathogenicity and the effects of Postbiotics on Intestinal Epithelial Function}. PhD thesis. 2021.

\leavevmode\vadjust pre{\hypertarget{ref-neurath1996}{}}%
37. Neurath MF, Fuss I, Kelsall BL, Presky DH, Waegell W, Strober W. \href{https://doi.org/10.1084/jem.183.6.2605}{Experimental granulomatous colitis in mice is abrogated by induction of TGF-beta-mediated oral tolerance.} Journal of Experimental Medicine. 1996;183:2605--16.

\leavevmode\vadjust pre{\hypertarget{ref-corraliza}{}}%
38. Corraliza AM, Ricart E, López-García A, Carme Masamunt M, Veny M, Esteller M, et al. Differences in Peripheral and Tissue Immune Cell Populations Following Haematopoietic Stem Cell Transplantation in Crohn{'}s Disease Patients. Journal of Crohn's and Colitis. \url{https://doi.org/10.1093/ecco-jcc/jjy203}.

\leavevmode\vadjust pre{\hypertarget{ref-strachan1989}{}}%
39. Strachan DP. \href{https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1838109/}{Hay fever, hygiene, and household size.} BMJ : British Medical Journal. 1989;299:1259--60.

\leavevmode\vadjust pre{\hypertarget{ref-scudellari2017}{}}%
40. Scudellari M. \href{https://doi.org/10.1073/pnas.1700688114}{News feature: Cleaning up the hygiene hypothesis}. Proceedings of the National Academy of Sciences of the United States of America. 2017;114:1433--6.

\leavevmode\vadjust pre{\hypertarget{ref-thomas1998}{}}%
41. Thomas GAO, Rhodes J, Green JT. \href{https://doi.org/10.1111/j.1572-0241.1998.00144.x}{Inflammatory bowel disease and smoking{\textemdash}a review}. Official journal of the American College of Gastroenterology \textbar{} ACG. 1998;93:144149.

\leavevmode\vadjust pre{\hypertarget{ref-cornish2008}{}}%
42. Cornish JA, Tan E, Simillis C, Clark SK, Teare J, Tekkis PP. \href{https://doi.org/10.1111/j.1572-0241.2008.02064.x}{The risk of oral contraceptives in the etiology of inflammatory bowel disease: a meta-analysis}. The American Journal of Gastroenterology. 2008;103:2394--400.

\leavevmode\vadjust pre{\hypertarget{ref-kaufmann1987}{}}%
43. Kaufmann HJ, Taubin HL. \href{https://doi.org/10.7326/0003-4819-107-4-513}{Nonsteroidal anti-inflammatory drugs activate quiescent inflammatory bowel disease}. Annals of Internal Medicine. 1987;107:513--6.

\leavevmode\vadjust pre{\hypertarget{ref-bitton2008}{}}%
44. Bitton A, Dobkin PL, Edwardes MD, Sewitch MJ, Meddings JB, Rawal S, et al. \href{https://doi.org/10.1136/gut.2007.134817}{Predicting relapse in Crohn's disease: a biopsychosocial model}. Gut. 2008;57:1386--92.

\leavevmode\vadjust pre{\hypertarget{ref-baumgart2012}{}}%
45. Baumgart DC, Sandborn WJ. \href{https://doi.org/10.1016/S0140-6736(12)60026-9}{Crohn's disease}. The Lancet. 2012;380:1590--605.

\leavevmode\vadjust pre{\hypertarget{ref-khanna2017}{}}%
46. Khanna S, Shin A, Kelly CP. \href{https://doi.org/10.1016/j.cgh.2016.10.024}{Management of clostridium difficile infection in inflammatory bowel disease: Expert review from the clinical practice updates committee of the AGA institute}. Clinical Gastroenterology and Hepatology. 2017;15:166--74.

\leavevmode\vadjust pre{\hypertarget{ref-guardiola2018}{}}%
47. Guardiola J, Lobatón T, Cerrillo E, Ferreiro-Iglesias R, Gisbert JP, Domènech E, et al. \href{https://doi.org/10.1016/j.gastre.2018.05.018}{Recommendations of the Spanish Working Group on Crohn's Disease and Ulcerative Colitis (GETECCU) on the utility of the determination of faecal calprotectin in inflammatory bowel disease}. Gastroenterología y Hepatología (English Edition). 2018;41:514--29.

\leavevmode\vadjust pre{\hypertarget{ref-sands2015}{}}%
48. Sands BE. \href{https://doi.org/10.1053/j.gastro.2015.07.003}{Biomarkers of Inflammation in Inflammatory Bowel Disease}. Gastroenterology. 2015;149:1275--1285.e2.

\leavevmode\vadjust pre{\hypertarget{ref-corralizamuxe1rquez2019}{}}%
49. Corraliza Márquez A. \href{http://diposit.ub.edu/dspace/handle/2445/143300}{Immune mechanisms involved in inducing remission in Crohn{'}s disease patients undergoing hematopoietic stem cell transplant}. PhD thesis. 2019.

\leavevmode\vadjust pre{\hypertarget{ref-bassolasmolina2018}{}}%
50. Bassolas Molina H. \href{http://diposit.ub.edu/dspace/handle/2445/131017}{Resposta T específica contra antígens de la microbiota comensal en la malaltia de Crohn i inhibició de ROR?t com a estratègia terapèutica}. PhD thesis. 2018.

\leavevmode\vadjust pre{\hypertarget{ref-daperno2004}{}}%
51. Daperno M, D'Haens G, Van Assche G, Baert F, Bulois P, Maunoury V, et al. \href{https://doi.org/10.1016/S0016-5107(04)01878-4}{Development and validation of a new, simplified endoscopic activity score for crohn's disease: The SES-CD}. Gastrointestinal Endoscopy. 2004;60:505--12.

\leavevmode\vadjust pre{\hypertarget{ref-best1976}{}}%
52. Best WR, Becktel JM, Singleton JW, Kern F. \href{https://doi.org/10.1016/S0016-5085(76)80163-1}{Development of a Crohn's Disease Activity Index: National Cooperative Crohn's Disease Study}. Gastroenterology. 1976;70:439--44.

\leavevmode\vadjust pre{\hypertarget{ref-bhattacharya2016}{}}%
53. Bhattacharya A, Rao BB, Koutroubakis IE, Click B, Vargas EJ, Regueiro M, et al. \href{https://doi.org/10.1097/MIB.0000000000000935}{Silent crohn's disease predicts increased bowel damage during multiyear follow-up: The consequences of under-reporting active inflammation}. Inflammatory Bowel Diseases. 2016;22:2665--71.

\leavevmode\vadjust pre{\hypertarget{ref-peyrin-biroulet2010}{}}%
54. Peyrin-Biroulet L, Loftus EVJ, Colombel J-F, Sandborn WJ. \href{https://doi.org/10.1038/ajg.2009.579}{The natural history of adult crohn's disease in population-based cohorts}. Official journal of the American College of Gastroenterology \textbar{} ACG. 2010;105:289297.

\leavevmode\vadjust pre{\hypertarget{ref-etchevers2009}{}}%
55. Etchevers MJ, Aceituno M, García-Bosch O, Ordás I, Sans M, Ricart E, et al. \href{https://doi.org/10.1002/ibd.20897}{Risk factors and characteristics of extent progression in ulcerative colitis}. Inflammatory Bowel Diseases. 2009;15:1320--5.

\leavevmode\vadjust pre{\hypertarget{ref-boonstra2012}{}}%
56. Boonstra K, van Erpecum KJ, van Nieuwkerk KMJ, Drenth JPH, Poen AC, Witteman BJM, et al. \href{https://doi.org/10.1002/ibd.22938}{Primary sclerosing cholangitis is associated with a distinct phenotype of inflammatory bowel disease}. Inflammatory Bowel Diseases. 2012;18:2270--6.

\leavevmode\vadjust pre{\hypertarget{ref-mark-christensen2018}{}}%
57. Mark-Christensen A, Laurberg S, Haboubi N. \href{https://doi.org/10.1093/ibd/izy075}{Dysplasia in Inflammatory Bowel Disease: Historical Review, Critical Histopathological Analysis, and Clinical Implications}. Inflammatory Bowel Diseases. 2018;24:1895--903.

\leavevmode\vadjust pre{\hypertarget{ref-schieffer2016}{}}%
58. Schieffer KM, Williams ED, Yochum GS, Koltun WA. \href{https://doi.org/10.1111/apt.13780}{Review article: the pathogenesis of pouchitis}. Alimentary Pharmacology \& Therapeutics. 2016;44:817--35.

\leavevmode\vadjust pre{\hypertarget{ref-schroeder1987}{}}%
59. Schroeder KW, Tremaine WJ, Ilstrup DM. \href{https://doi.org/10.1056/nejm198712243172603}{Coated Oral 5-Aminosalicylic Acid Therapy for Mildly to Moderately Active Ulcerative Colitis}. New England Journal of Medicine. 1987;317:1625--9.

\leavevmode\vadjust pre{\hypertarget{ref-irvine1999}{}}%
60. Irvine EJ. \href{https://doi.org/10.1097/00005176-199904001-00003}{Development and Subsequent Refinement of the Inflammatory Bowel Disease Questionnaire: A Quality-of-Life Instrument for Adult Patients with Inflammatory Bowel Disease}. Journal of Pediatric Gastroenterology \& Nutrition. 1999;28 Supplement:S23--7.

\leavevmode\vadjust pre{\hypertarget{ref-travis2012}{}}%
61. Travis SPL, Schnell D, Krzeski P, Abreu MT, Altman DG, Colombel J-F, et al. \href{https://doi.org/10.1136/gutjnl-2011-300486}{Developing an instrument to assess the endoscopic severity of ulcerative colitis: The ulcerative colitis endoscopic index of severity (UCEIS)}. Gut. 2012;61:535--42.

\leavevmode\vadjust pre{\hypertarget{ref-travis2006}{}}%
62. Travis SPL, Stange EF, Lémann M, Öresland T, Chowers Y, Forbes A, et al. \href{https://doi.org/10.1136/gut.2005.081950b}{European evidence based consensus on the diagnosis and management of Crohn{'}s disease: current management}. Gut. 2006;55 suppl 1:i16--35.

\leavevmode\vadjust pre{\hypertarget{ref-akobeng2016}{}}%
63. Akobeng AK, Zhang D, Gordon M, MacDonald JK. Oral 5{-}aminosalicylic acid for maintenance of medically{-}induced remission in Crohn's disease. Cochrane Database of Systematic Reviews. 2016. \url{https://doi.org/10.1002/14651858.CD003715.pub3}.

\leavevmode\vadjust pre{\hypertarget{ref-feller2010}{}}%
64. Feller M, Huwiler K, Schoepfer A, Shang A, Furrer H, Egger M. \href{https://doi.org/10.1086/649923}{Long-term antibiotic treatment for crohn's disease: Systematic review and meta-analysis of placebo-controlled trials}. Clinical Infectious Diseases. 2010;50:473--80.

\leavevmode\vadjust pre{\hypertarget{ref-prantera2009}{}}%
65. Prantera C, Scribano ML. \href{https://doi.org/10.1097/MOG.0b013e32832b20bf}{Antibiotics and probiotics in inflammatory bowel disease: Why, when, and how}. Current Opinion in Gastroenterology. 2009;25:329333.

\leavevmode\vadjust pre{\hypertarget{ref-rezaie2015}{}}%
66. Rezaie A, Kuenzig ME, Benchimol EI, Griffiths AM, Otley AR, Steinhart AH, et al. Budesonide for induction of remission in Crohn's disease. Cochrane Database of Systematic Reviews. 2015. \url{https://doi.org/10.1002/14651858.CD000296.pub4}.

\leavevmode\vadjust pre{\hypertarget{ref-lichtenstein2009}{}}%
67. Lichtenstein GR, Hanauer SB, Sandborn WJ, Gastroenterology TPPC of the AC of. \href{https://journals.lww.com/ajg/Abstract/2009/02000/Management_of_Crohn_s_Disease_in_Adults.32.aspx}{Management of crohn's disease in adults}. Official journal of the American College of Gastroenterology \textbar{} ACG. 2009;104:465483.

\leavevmode\vadjust pre{\hypertarget{ref-ouellette2001}{}}%
68. Ouellette AJ, Bevins CL. \href{https://doi.org/10.1097/00054725-200102000-00007}{Paneth cell defensins and innate immunity of the small bowel}. Inflammatory Bowel Diseases. 2001;7:43--50.

\leavevmode\vadjust pre{\hypertarget{ref-warner2018}{}}%
69. Warner B, Johnston E, Arenas-Hernandez M, Marinaki A, Irving P, Sanderson J. \href{https://doi.org/10.1136/flgastro-2016-100738}{A practical guide to thiopurine prescribing and monitoring in IBD}. Frontline Gastroenterology. 2018;9:10--5.

\leavevmode\vadjust pre{\hypertarget{ref-chande2015}{}}%
70. Chande N, Patton PH, Tsoulis DJ, Thomas BS, MacDonald JK. Azathioprine or 6{-}mercaptopurine for maintenance of remission in Crohn's disease. Cochrane Database of Systematic Reviews. 2015. \url{https://doi.org/10.1002/14651858.CD000067.pub3}.

\leavevmode\vadjust pre{\hypertarget{ref-gisbert2009}{}}%
71. Gisbert JP, Linares PM, Mcnicholl AG, Maté J, Gomollón F. \href{https://doi.org/10.1111/j.1365-2036.2009.04023.x}{Meta-analysis: the efficacy of azathioprine and mercaptopurine in ulcerative colitis}. Alimentary Pharmacology \& Therapeutics. 2009;30:126--37.

\leavevmode\vadjust pre{\hypertarget{ref-peyrin-biroulet2011}{}}%
72. Peyrin-Biroulet L, Lémann M. \href{https://doi.org/10.1111/j.1365-2036.2011.04599.x}{Review article: remission rates achievable by current therapies for inflammatory bowel disease}. Alimentary Pharmacology \& Therapeutics. 2011;33:870--9.

\leavevmode\vadjust pre{\hypertarget{ref-billioud2011}{}}%
73. Billioud V, Sandborn WJ, Peyrin-Biroulet L. \href{https://doi.org/10.1038/ajg.2011.60}{Loss of response and need for adalimumab dose intensification in crohn's disease: A systematic review}. Official journal of the American College of Gastroenterology \textbar{} ACG. 2011;106:674684.

\leavevmode\vadjust pre{\hypertarget{ref-hwang2008}{}}%
74. Hwang JM, Varma MG. \href{https://doi.org/10.3748/wjg.14.2678}{Surgery for inflammatory bowel disease}. World Journal of Gastroenterology : WJG. 2008;14:2678--90.

\leavevmode\vadjust pre{\hypertarget{ref-gardiner2007}{}}%
75. Gardiner KR, Dasari BVM. \href{https://doi.org/10.1016/j.suc.2007.03.011}{Operative Management of Small Bowel Crohn's Disease}. Surgical Clinics of North America. 2007;87:587--610.

\leavevmode\vadjust pre{\hypertarget{ref-lewis2010}{}}%
76. Lewis RT, Maron DJ. \href{https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2976865/}{Efficacy and complications of surgery for crohn's disease}. Gastroenterology \& Hepatology. 2010;6:587--96.

\leavevmode\vadjust pre{\hypertarget{ref-corralizab}{}}%
77. Corraliza AM, Ricart E, López-García A, Carme Masamunt M, Veny M, Esteller M, et al. Differences in Peripheral and Tissue Immune Cell Populations Following Haematopoietic Stem Cell Transplantation in Crohn{'}s Disease Patients. Journal of Crohn's and Colitis. \url{https://doi.org/10.1093/ecco-jcc/jjy203}.

\leavevmode\vadjust pre{\hypertarget{ref-weingarden2017}{}}%
78. Weingarden AR, Vaughn BP. \href{https://doi.org/10.1080/19490976.2017.1290757}{Intestinal microbiota, fecal microbiota transplantation, and inflammatory bowel disease}. Gut Microbes. 2017;8:238--52.

\leavevmode\vadjust pre{\hypertarget{ref-beck2021}{}}%
79. Beck LC, Granger CL, Masi AC, Stewart CJ. \href{https://doi.org/10.1080/14789450.2021.1922278}{Use of omic technologies in early life gastrointestinal health and disease: From bench to bedside}. Expert Review of Proteomics. 2021;18:247--59.

\leavevmode\vadjust pre{\hypertarget{ref-hasler_uncoupling_2016}{}}%
80. Häsler R, Sheibani-Tezerji R, Sinha A, Barann M, Rehman A, Esser D, et al. \href{https://doi.org/10.1136/gutjnl-2016-311651}{Uncoupling of mucosal gene regulation, {mRNA} splicing and adherent microbiota signatures in inflammatory bowel disease}. Gut. 2017;66:2087--97.

\leavevmode\vadjust pre{\hypertarget{ref-tang2017}{}}%
81. Tang MS, Bowcutt R, Leung JM, Wolff MJ, Gundra UM, Hudesman D, et al. \href{https://doi.org/10.1097/MIB.0000000000001208}{Integrated Analysis of Biopsies from Inflammatory Bowel Disease Patients Identifies SAA1 as a Link Between Mucosal Microbes with TH17 and TH22 Cells}. Inflammatory Bowel Diseases. 2017;23:1544--54.

\leavevmode\vadjust pre{\hypertarget{ref-hernuxe1ndez-rocha2021}{}}%
82. Hernández-Rocha C, Borowski K, Turpin W, Filice M, Nayeri S, Raygoza Garay JA, et al. Integrative analysis of colonic biopsies from inflammatory bowel disease patients identifies an interaction between microbial bile-acid inducible gene abundance and human angiopoietin-like 4 gene expression. Journal of Crohn's and Colitis. 2021. \url{https://doi.org/10.1093/ecco-jcc/jjab096}.

\leavevmode\vadjust pre{\hypertarget{ref-huWholeExomeSequencing2021}{}}%
83. Hu S, Vila AV, Gacesa R, Collij V, Stevens C, Fu JM, et al. \href{https://doi.org/10.1136/gutjnl-2019-319706}{Whole exome sequencing analyses reveal gene--microbiota interactions in the context of IBD}. Gut. 2021;70:285--96.

\leavevmode\vadjust pre{\hypertarget{ref-mayorgas2021}{}}%
84. Mayorgas A, Dotti I, Salas A. \href{https://doi.org/10.1002/mnfr.202000188}{Microbial Metabolites, Postbiotics, and Intestinal Epithelial Function}. Molecular Nutrition \& Food Research. 2021;65:2000188.

\leavevmode\vadjust pre{\hypertarget{ref-planell2013}{}}%
85. Planell N, Lozano JJ, Mora-Buch R, Masamunt MC, Jimeno M, Ordás I, et al. \href{https://doi.org/10.1136/gutjnl-2012-303333}{Transcriptional analysis of the intestinal mucosa of patients with ulcerative colitis in remission reveals lasting epithelial cell alterations}. Gut. 2013;62:967--76.

\leavevmode\vadjust pre{\hypertarget{ref-leal2015}{}}%
86. Leal RF, Planell N, Kajekar R, Lozano JJ, Ordás I, Dotti I, et al. \href{https://doi.org/10.1136/gutjnl-2013-306518}{Identification of inflammatory mediators in patients with Crohn's disease unresponsive to anti-TNF? therapy}. Gut. 2015;64:233--42.

\leavevmode\vadjust pre{\hypertarget{ref-massimino2021}{}}%
87. Massimino L, Lamparelli LA, Houshyar Y, D'Alessio S, Peyrin-Biroulet L, Vetrano S, et al. \href{https://doi.org/10.1038/s43588-021-00114-y}{The Inflammatory Bowel Disease Transcriptome and Metatranscriptome Meta-Analysis (IBD TaMMA) framework}. Nature Computational Science. 2021;1:511--5.

\leavevmode\vadjust pre{\hypertarget{ref-knights2013}{}}%
88. Knights D, Lassen KG, Xavier RJ. \href{https://doi.org/10.1136/gutjnl-2012-303954}{Advances in inflammatory bowel disease pathogenesis: Linking host genetics and the microbiome}. Gut. 2013;62.

\leavevmode\vadjust pre{\hypertarget{ref-repnik2016}{}}%
89. Repnik K, Potočnik U. \href{https://doi.org/10.1007/s13353-015-0334-1}{eQTL analysis links inflammatory bowel disease associated 1q21 locus to ECM1 gene}. Journal of Applied Genetics. 2016;57:363--72.

\leavevmode\vadjust pre{\hypertarget{ref-hu2021}{}}%
90. Hu S, Uniken Venema WT, Westra H-J, Vich Vila A, Barbieri R, Voskuil MD, et al. \href{https://doi.org/10.1038/s41467-021-21458-z}{Inflammation status modulates the effect of host genetic variation on intestinal gene expression in inflammatory bowel disease}. Nature Communications. 2021;12:1122.

\leavevmode\vadjust pre{\hypertarget{ref-jung2020}{}}%
91. Jung S, Liu W, Baek J, Moon JW, Ye BD, Lee H-S, et al. \href{https://www.frontiersin.org/article/10.3389/fgene.2020.00486}{Expression quantitative trait loci (eQTL) mapping in korean patients with crohn{'}s disease and identification of potential causal genes through integration with disease associations}. Frontiers in Genetics. 2020;11.

\leavevmode\vadjust pre{\hypertarget{ref-dai2019}{}}%
92. Dai Y, Pei G, Zhao Z, Jia P. \href{https://www.frontiersin.org/article/10.3389/fgene.2019.00318}{A convergent study of genetic variants associated with crohn{'}s disease: Evidence from GWAS, gene expression, methylation, eQTL and TWAS}. Frontiers in Genetics. 2019;10.

\leavevmode\vadjust pre{\hypertarget{ref-ahmed2016}{}}%
93. Ahmed I, Roy BC, Khan SA, Septer S, Umar S. \href{https://doi.org/10.3390/microorganisms4020020}{Microbiome, metabolome and inflammatory bowel disease}. Microorganisms. 2016;4:20.

\leavevmode\vadjust pre{\hypertarget{ref-gallagher2021}{}}%
94. Gallagher K, Catesson A, Griffin JL, Holmes E, Williams HRT. \href{https://doi.org/10.1093/ecco-jcc/jjaa227}{Metabolomic Analysis in Inflammatory Bowel Disease: A Systematic Review}. Journal of Crohn's \& Colitis. 2021;15:813--26.

\leavevmode\vadjust pre{\hypertarget{ref-krassowski2020}{}}%
95. Krassowski M, Das V, Sahu SK, Misra BB. \href{https://doi.org/10.3389/fgene.2020.610798}{State of the field in multi-omics research: From computational needs to data mining and sharing}. Frontiers in Genetics. 2020;11:1598.

\leavevmode\vadjust pre{\hypertarget{ref-yannakoudakis2015}{}}%
96. Yannakoudakis H, Cummins R. \href{https://doi.org/10.3115/v1/W15-0625}{Evaluating the performance of automated text scoring systems}. Denver, Colorado: Association for Computational Linguistics; 2015. p. 213223.

\leavevmode\vadjust pre{\hypertarget{ref-hotelling1936}{}}%
97. HOTELLING H. \href{https://doi.org/10.1093/biomet/28.3-4.321}{RELATIONS BETWEEN TWO SETS OF VARIATES}. Biometrika. 1936;28:321--77.

\leavevmode\vadjust pre{\hypertarget{ref-biancolillo}{}}%
98. Biancolillo A. \href{http://models.life.ku.dk/sites/default/files/Thesis_Alessandra_Biancolillo.pdf}{Method development in the area of multi-block analysis focused on food analysis}. PhD thesis.

\leavevmode\vadjust pre{\hypertarget{ref-wu_selective_2019}{}}%
99. Wu C, Zhou F, Ren J, Li X, Jiang Y, Ma S. \href{https://doi.org/10.3390/ht8010004}{A {Selective Review} of {Multi}-{Level Omics Data Integration Using Variable Selection}}. High-Throughput. 2019;8:4.

\leavevmode\vadjust pre{\hypertarget{ref-cavill2016}{}}%
100. Cavill R, Jennen D, Kleinjans J, Briedé JJ. \href{https://doi.org/10.1093/bib/bbv090}{Transcriptomic and metabolomic data integration}. Briefings in Bioinformatics. 2016;17:891--901.

\leavevmode\vadjust pre{\hypertarget{ref-chongComputationalApproachesIntegrative2017}{}}%
101. Chong J, Xia J. \href{https://doi.org/10.3390/metabo7040062}{Computational approaches for integrative analysis of the metabolome and microbiome}. Metabolites. 2017;7:62.

\leavevmode\vadjust pre{\hypertarget{ref-huang2017}{}}%
102. Huang S, Chaudhary K, Garmire LX. \href{https://doi.org/10.3389/fgene.2017.00084}{More is better: Recent progress in multi-omics data integration methods}. Frontiers in Genetics. 2017;8.

\leavevmode\vadjust pre{\hypertarget{ref-rohart2016}{}}%
103. Rohart F, Mason EA, Matigian N, Mosbergen R, Korn O, Chen T, et al. \href{https://doi.org/10.7717/peerj.1845}{A molecular classification of human mesenchymal stromal cells}. PeerJ. 2016;4:e1845.

\leavevmode\vadjust pre{\hypertarget{ref-ibrahim2017}{}}%
104. Ibrahim EC, Guillemot V, Comte M, Tenenhaus A, Zendjidjian XY, Cancel A, et al. \href{https://doi.org/10.1038/s41537-017-0027-3}{Modeling a linkage between blood transcriptional expression and activity in brain regions to infer the phenotype of schizophrenia patients}. npj Schizophrenia. 2017;3:25.

\leavevmode\vadjust pre{\hypertarget{ref-wheeler2014}{}}%
105. Wheeler HE, Aquino-Michaels K, Gamazon ER, Trubetskoy VV, Dolan ME, Huang RS, et al. \href{https://doi.org/10.1002/gepi.21808}{Poly-Omic Prediction of Complex Traits: OmicKriging}. Genetic Epidemiology. 2014;38:402--15.

\leavevmode\vadjust pre{\hypertarget{ref-yin2019}{}}%
106. Yin L, Chau CKL, Sham P-C, So H-C. \href{https://doi.org/10.1016/j.ajhg.2019.10.012}{Integrating Clinical Data and Imputed Transcriptome from GWAS to Uncover Complex Disease Subtypes: Applications in Psychiatry and Cardiology}. The American Journal of Human Genetics. 2019;105:1193--212.

\leavevmode\vadjust pre{\hypertarget{ref-tarazona2021}{}}%
107. Tarazona S, Arzalluz-Luque A, Conesa A. \href{https://doi.org/10.1038/s43588-021-00086-z}{Undisclosed, unmet and neglected challenges in multi-omics studies}. Nature Computational Science. 2021;1--8.

\leavevmode\vadjust pre{\hypertarget{ref-tarazona2020}{}}%
108. Tarazona S, Balzano-Nogueira L, Gómez-Cabrero D, Schmidt A, Imhof A, Hankemeier T, et al. \href{https://doi.org/10.1038/s41467-020-16937-8}{Harmonization of quality metrics and power calculation in multi-omic studies}. Nature Communications. 2020;11:3092.

\leavevmode\vadjust pre{\hypertarget{ref-massoni-badosa2020}{}}%
109. Massoni-Badosa R, Iacono G, Moutinho C, Kulis M, Palau N, Marchese D, et al. \href{https://doi.org/10.1186/s13059-020-02032-0}{Sampling time-dependent artifacts in single-cell genomics studies}. Genome Biology. 2020;21:112.

\leavevmode\vadjust pre{\hypertarget{ref-zhu2017}{}}%
110. Zhu Y, Wang L, Yin Y, Yang E. \href{https://doi.org/10.1038/s41598-017-05882-0}{Systematic analysis of gene expression patterns associated with postmortem interval in human tissues}. Scientific Reports. 2017;7:5435.

\leavevmode\vadjust pre{\hypertarget{ref-ferreira2018}{}}%
111. Ferreira PG, Muñoz-Aguirre M, Reverter F, Sá Godinho CP, Sousa A, Amadoz A, et al. \href{https://doi.org/10.1038/s41467-017-02772-x}{The effects of death and post-mortem cold ischemia on human tissue transcriptomes}. Nature Communications. 2018;9:490.

\leavevmode\vadjust pre{\hypertarget{ref-jacob1961}{}}%
112. Jacob F, Monod J. \href{https://doi.org/10.1016/S0022-2836(61)80072-7}{Genetic regulatory mechanisms in the synthesis of proteins}. Journal of Molecular Biology. 1961;3:318--56.

\leavevmode\vadjust pre{\hypertarget{ref-koh_iomicspass_2019}{}}%
113. Koh HWL, Fermin D, Vogel C, Choi KP, Ewing RM, Choi H. \href{https://doi.org/10.1038/s41540-019-0099-y}{{iOmicsPASS}: Network-based integration of multiomics data for predictive subnetwork discovery}. npj Systems Biology and Applications. 2019;5:1--0.

\leavevmode\vadjust pre{\hypertarget{ref-yule1907}{}}%
114. Yule GU. \href{https://www.jstor.org/stable/92723}{On the theory of correlation for any number of variables, treated by a new system of notation}. Proceedings of the Royal Society of London Series A, Containing Papers of a Mathematical and Physical Character. 1907;79:182--93.

\leavevmode\vadjust pre{\hypertarget{ref-tenenhaus_regularized_2011}{}}%
115. Tenenhaus A, Tenenhaus M. \href{https://doi.org/10.1007/s11336-011-9206-8}{Regularized {Generalized Canonical Correlation Analysis}}. Psychometrika. 2011;76:257--84.

\leavevmode\vadjust pre{\hypertarget{ref-tenenhaus_variable_2014}{}}%
116. Tenenhaus A, Philippe C, Guillemot V, Le Cao K-A, Grill J, Frouin V. \href{https://doi.org/10.1093/biostatistics/kxu001}{Variable selection for generalized canonical correlation analysis}. Biostatistics. 2014;15:569--83.

\leavevmode\vadjust pre{\hypertarget{ref-culhane_cross-platform_2003}{}}%
117. Culhane AC, Perri`ere G, Higgins DG. \href{https://doi.org/10.1186/1471-2105-4-59}{Cross-platform comparison and visualisation of gene expression data using co-inertia analysis}. BMC Bioinformatics. 2003;4:59.

\leavevmode\vadjust pre{\hypertarget{ref-vito_multi-study_2019}{}}%
118. Vito RD, Bellio R, Trippa L, Parmigiani G. \href{https://doi.org/10.1111/biom.12974}{Multi-study factor analysis}. Biometrics. 2019;75:337--46.

\leavevmode\vadjust pre{\hypertarget{ref-argelaguet_multi-omics_2018}{}}%
119. Argelaguet R, Velten B, Arnol D, Dietrich S, Zenz T, Marioni JC, et al. \href{https://doi.org/10.15252/msb.20178124}{Multi-{Omics Factor Analysis}\textemdash{}a framework for unsupervised integration of multi-omics data sets}. Molecular Systems Biology. 2018;14:e8124.

\leavevmode\vadjust pre{\hypertarget{ref-gomez-cabrero2019}{}}%
120. Gomez-Cabrero D, Tarazona S, Ferreirós-Vidal I, Ramirez RN, Company C, Schmidt A, et al. \href{https://doi.org/10.1038/s41597-019-0202-7}{STATegra, a comprehensive multi-omics dataset of B-cell differentiation in mouse}. Scientific Data. 2019;6:1--5.

\leavevmode\vadjust pre{\hypertarget{ref-franzosaSpecieslevelFunctionalProfiling2018}{}}%
121. Franzosa EA, McIver LJ, Rahnavard G, Thompson LR, Schirmer M, Weingart G, et al. \href{https://doi.org/10.1038/s41592-018-0176-y}{Species-level functional profiling of metagenomes and metatranscriptomes}. Nature Methods. 2018;15:962.

\leavevmode\vadjust pre{\hypertarget{ref-truongMetaPhlAn2EnhancedMetagenomic2015}{}}%
122. Truong DT, Franzosa EA, Tickle TL, Scholz M, Weingart G, Pasolli E, et al. \href{https://doi.org/10.1038/nmeth.3589}{MetaPhlAn2 for enhanced metagenomic taxonomic profiling}. Nature Methods. 2015;12:902--3.

\leavevmode\vadjust pre{\hypertarget{ref-segataMetagenomicBiomarkerDiscovery2011}{}}%
123. Segata N, Izard J, Waldron L, Gevers D, Miropolsky L, Garrett WS, et al. \href{https://doi.org/10.1186/gb-2011-12-6-r60}{Metagenomic biomarker discovery and explanation}. Genome Biology. 2011;12:R60.

\leavevmode\vadjust pre{\hypertarget{ref-didier2018}{}}%
124. Didier G, Valdeolivas A, Baudot A. \href{https://doi.org/10.12688/f1000research.15486.2}{Identifying communities from multiplex biological networks by randomized optimization of modularity}. F1000Research. 2018;7.

\leavevmode\vadjust pre{\hypertarget{ref-valdeolivas2019}{}}%
125. Valdeolivas A, Tichit L, Navarro C, Perrin S, Odelin G, Levy N, et al. \href{https://doi.org/10.1093/bioinformatics/bty637}{Random walk with restart on multiplex and heterogeneous biological networks}. Bioinformatics (Oxford, England). 2019;35:497--505.

\leavevmode\vadjust pre{\hypertarget{ref-pio-lopez2021}{}}%
126. Pio-Lopez L, Valdeolivas A, Tichit L, Remy É, Baudot A. \href{http://arxiv.org/abs/2008.10085}{MultiVERSE: A multiplex and multiplex-heterogeneous network embedding approach}. arXiv:200810085 {[}cs, q-bio{]}. 2021.

\leavevmode\vadjust pre{\hypertarget{ref-bayes1763}{}}%
127. Bayes T, Price null. \href{https://doi.org/10.1098/rstl.1763.0053}{An essay towards solving a problem in the doctrine of chances. By the late rev. Mr. Bayes, f. R. s.} Philosophical Transactions of the Royal Society of London. 1763;53:370--418.

\leavevmode\vadjust pre{\hypertarget{ref-zhu2012}{}}%
128. Zhu J, Sova P, Xu Q, Dombek KM, Xu EY, Vu H, et al. \href{https://doi.org/10.1371/journal.pbio.1001301}{Stitching together Multiple Data Dimensions Reveals Interacting Metabolomic and Transcriptomic Networks That Modulate Cell Regulation}. PLOS Biology. 2012;10:e1001301.

\leavevmode\vadjust pre{\hypertarget{ref-lock2013}{}}%
129. Lock EF, Dunson DB. \href{https://doi.org/10.1093/bioinformatics/btt425}{Bayesian consensus clustering}. Bioinformatics. 2013;29:2610--6.

\leavevmode\vadjust pre{\hypertarget{ref-cantini2021}{}}%
130. Cantini L, Zakeri P, Hernandez C, Naldi A, Thieffry D, Remy E, et al. \href{https://doi.org/10.1038/s41467-020-20430-7}{Benchmarking joint multi-omics dimensionality reduction approaches for the study of cancer}. Nature Communications. 2021;12:124.

\leavevmode\vadjust pre{\hypertarget{ref-tenenhaus2017}{}}%
131. Tenenhaus M, Tenenhaus A, Groenen PJF. \href{https://doi.org/10.1007/s11336-017-9573-x}{Regularized Generalized Canonical Correlation Analysis: A Framework for Sequential Multiblock Component Methods}. Psychometrika. 2017;82:737--77.

\leavevmode\vadjust pre{\hypertarget{ref-rohart2017}{}}%
132. Rohart F, Gautier B, Singh A, Cao K-AL. \href{https://doi.org/10.1371/journal.pcbi.1005752}{mixOmics: An R package for {`}omics feature selection and multiple data integration}. PLOS Computational Biology. 2017;13:e1005752.

\leavevmode\vadjust pre{\hypertarget{ref-virtanen2012}{}}%
133. Virtanen S, Klami A, Khan S, Kaski S. \href{https://proceedings.mlr.press/v22/virtanen12.html}{Bayesian Group Factor Analysis}. PMLR; 2012. p. 1269--77.

\leavevmode\vadjust pre{\hypertarget{ref-novoa-del-toro2020}{}}%
134. Novoa-del-Toro E-M, Mezura-Montes E, Vignes M, Magdinier F, Tichit L, Baudot A. \href{https://doi.org/10.1101/2020.05.25.114215}{A Multi-Objective Genetic Algorithm to Find Active Modules in Multiplex Biological Networks}. bioRxiv. 2020;2020.05.25.114215.

\leavevmode\vadjust pre{\hypertarget{ref-lock2013a}{}}%
135. Lock EF, Hoadley KA, Marron JS, Nobel AB. \href{https://doi.org/10.1214/12-AOAS597}{JOINT AND INDIVIDUAL VARIATION EXPLAINED (JIVE) FOR INTEGRATED ANALYSIS OF MULTIPLE DATA TYPES}. The annals of applied statistics. 2013;7:523--42.

\leavevmode\vadjust pre{\hypertarget{ref-yang2016}{}}%
136. Yang Z, Michailidis G. \href{https://doi.org/10.1093/bioinformatics/btv544}{A non-negative matrix factorization method for detecting modules in heterogeneous omics multi-modal data}. Bioinformatics. 2016;32:1--8.

\leavevmode\vadjust pre{\hypertarget{ref-sherry2005}{}}%
137. Sherry A, Henson RK. \href{https://doi.org/10.1207/s15327752jpa8401_09}{Conducting and Interpreting Canonical Correlation Analysis in Personality Research: A User-Friendly Primer}. Journal of Personality Assessment. 2005;84:37--48.

\leavevmode\vadjust pre{\hypertarget{ref-sherryConductingInterpretingCanonical1981}{}}%
138. Sherry A, Henson RK. Conducting and {Interpreting Canonical Correlation Analysis} in {Personality Research}: {A User}-{Friendly Primer}. 1981.

\leavevmode\vadjust pre{\hypertarget{ref-chung_multi-omics_2019}{}}%
139. Chung R-H, Kang C-Y. \href{https://doi.org/10.1093/gigascience/giz045}{A multi-omics data simulator for complex disease studies and its application to evaluate multi-omics data analysis methods for disease classification}. GigaScience. 2019;8.

\leavevmode\vadjust pre{\hypertarget{ref-martuxednez-mira2018}{}}%
140. Martínez-Mira C, Conesa A, Tarazona S. \href{https://doi.org/10.1101/421834}{MOSim: Multi-Omics Simulation in R}. 2018.

\leavevmode\vadjust pre{\hypertarget{ref-patuzzi2019}{}}%
141. Patuzzi I, Baruzzo G, Losasso C, Ricci A, Di Camillo B. \href{https://doi.org/10.1186/s12859-019-2882-6}{metaSPARSim: A 16S rRNA gene sequencing count data simulator}. BMC Bioinformatics. 2019;20:416.

\leavevmode\vadjust pre{\hypertarget{ref-fritz2019}{}}%
142. Fritz A, Hofmann P, Majda S, Dahms E, Dröge J, Fiedler J, et al. \href{https://doi.org/10.1186/s40168-019-0633-6}{CAMISIM: Simulating metagenomes and microbial communities}. Microbiome. 2019;7:17.

\leavevmode\vadjust pre{\hypertarget{ref-fu2021}{}}%
143. Fu J, Frazee AC, Collado-Torres L, Jaffe AE, Leek JT. \href{https://doi.org/10.18129/B9.bioc.ballgown}{Ballgown: Flexible, isoform-level differential expression analysis}. Bioconductor version: Release (3.13); 2021.

\leavevmode\vadjust pre{\hypertarget{ref-frazee2021}{}}%
144. Frazee AC, Jaffe AE, Kirchner R, Leek JT. \href{https://doi.org/10.18129/B9.bioc.polyester}{Polyester: Simulate RNA-seq reads}. Bioconductor version: Release (3.13); 2021.

\leavevmode\vadjust pre{\hypertarget{ref-mccarthy2012}{}}%
145. McCarthy DJ, Chen Y, Smyth GK. \href{https://doi.org/10.1093/nar/gks042}{Differential expression analysis of multifactor RNA-Seq experiments with respect to biological variation}. Nucleic Acids Research. 2012;40:4288--97.

\leavevmode\vadjust pre{\hypertarget{ref-de_souza_ibd_2017}{}}%
146. De Souza HSP, Fiocchi C, Iliopoulos D. \href{https://doi.org/10.1038/nrgastro.2017.110}{The {IBD} interactome: {An} integrated view of aetiology, pathogenesis and therapy}. 2017;14.

\leavevmode\vadjust pre{\hypertarget{ref-valles-colomer2016}{}}%
147. Valles-Colomer M, Darzi Y, Vieira-Silva S, Falony G, Raes J, Joossens M. \href{https://doi.org/10.1093/ecco-jcc/jjw024}{Meta-omics in inflammatory bowel disease research: Applications, challenges, and guidelines}. Journal of Crohn's and Colitis. 2016;10:735--46.

\leavevmode\vadjust pre{\hypertarget{ref-sudhakar2022}{}}%
148. Sudhakar P, Alsoud D, Wellens J, Verstockt S, Arnauts K, Verstockt B, et al. \href{https://doi.org/10.1093/ecco-jcc/jjac027}{Tailoring multi-omics to inflammatory bowel diseases: All for one and one for all}. Journal of Crohn's and Colitis. 2022;jjac027.

\leavevmode\vadjust pre{\hypertarget{ref-puget2012}{}}%
149. Puget S, Philippe C, Bax DA, Job B, Varlet P, Junier M-P, et al. \href{https://doi.org/10.1371/journal.pone.0030313}{Mesenchymal Transition and PDGFRA Amplification/Mutation Are Key Distinct Oncogenic Events in Pediatric Diffuse Intrinsic Pontine Gliomas}. PLOS ONE. 2012;7:e30313.

\leavevmode\vadjust pre{\hypertarget{ref-morgan2015}{}}%
150. Morgan XC, Kabakchiev B, Waldron L, Tyler AD, Tickle TL, Milgrom R, et al. \href{https://doi.org/10.1186/s13059-015-0637-x}{Associations between host gene expression, the mucosal microbiome, and clinical outcome in the pelvic pouch of patients with inflammatory bowel disease}. Genome Biology. 2015;16:67.

\leavevmode\vadjust pre{\hypertarget{ref-howell2018}{}}%
151. Howell KJ, Kraiczy J, Nayak KM, Gasparetto M, Ross A, Lee C, et al. \href{https://doi.org/10.1053/j.gastro.2017.10.007}{DNA Methylation and Transcription Patterns in Intestinal Epithelial Cells From Pediatric Patients With Inflammatory Bowel~Diseases Differentiate Disease Subtypes and Associate With Outcome}. Gastroenterology. 2018;154:585--98.

\leavevmode\vadjust pre{\hypertarget{ref-martin2011}{}}%
152. Martin M. \href{https://doi.org/10.14806/ej.17.1.200}{Cutadapt removes adapter sequences from high-throughput sequencing reads}. EMBnetjournal. 2011;17:10--2.

\leavevmode\vadjust pre{\hypertarget{ref-dobin2013}{}}%
153. Dobin A, Davis CA, Schlesinger F, Drenkow J, Zaleski C, Jha S, et al. \href{https://doi.org/10.1093/bioinformatics/bts635}{STAR: ultrafast universal RNA-seq aligner}. Bioinformatics (Oxford, England). 2013;29:15--21.

\leavevmode\vadjust pre{\hypertarget{ref-li2011}{}}%
154. Li B, Dewey CN. \href{https://doi.org/10.1186/1471-2105-12-323}{RSEM: Accurate transcript quantification from RNA-seq data with or without a reference genome}. BMC Bioinformatics. 2011;12:323.

\leavevmode\vadjust pre{\hypertarget{ref-corralizaDifferencesPeripheralTissue}{}}%
155. Corraliza AM, Ricart E, L'opez-García A, Carme Masamunt M, Veny M, Esteller M, et al. Differences in {Peripheral} and {Tissue Immune Cell Populations Following Haematopoietic Stem Cell Transplantation} in {Crohn}'s {Disease Patients}. Journal of Crohn's and Colitis. \url{https://doi.org/10.1093/ecco-jcc/jjy203}.

\leavevmode\vadjust pre{\hypertarget{ref-berry2011}{}}%
156. Berry D, Ben Mahfoudh K, Wagner M, Loy A. \href{https://doi.org/10.1128/AEM.05220-11}{Barcoded primers used in multiplex amplicon pyrosequencing bias amplification}. Applied and Environmental Microbiology. 2011;77:7846--9.

\leavevmode\vadjust pre{\hypertarget{ref-klindworth2013}{}}%
157. Klindworth A, Pruesse E, Schweer T, Peplies J, Quast C, Horn M, et al. \href{https://doi.org/10.1093/nar/gks808}{Evaluation of general 16S ribosomal RNA gene PCR primers for classical and next-generation sequencing-based diversity studies}. Nucleic Acids Research. 2013;41:e1--1.

\leavevmode\vadjust pre{\hypertarget{ref-edgar2013}{}}%
158. Edgar RC. \href{https://doi.org/10.1038/nmeth.2604}{UPARSE: highly accurate OTU sequences from microbial amplicon reads}. Nature Methods. 2013;10:996--8.

\leavevmode\vadjust pre{\hypertarget{ref-lagkouvardos2016}{}}%
159. Lagkouvardos I, Joseph D, Kapfhammer M, Giritli S, Horn M, Haller D, et al. \href{https://doi.org/10.1038/srep33721}{IMNGS: A comprehensive open resource of processed 16S rRNA microbial profiles for ecology and diversity studies}. Scientific Reports. 2016;6.

\leavevmode\vadjust pre{\hypertarget{ref-callahan2016}{}}%
160. Callahan BJ, McMurdie PJ, Rosen MJ, Han AW, Johnson AJA, Holmes SP. \href{https://doi.org/10.1038/nmeth.3869}{DADA2: High-resolution sample inference from Illumina amplicon data}. Nature Methods. 2016;13:581--3.

\leavevmode\vadjust pre{\hypertarget{ref-quast2013}{}}%
161. Quast C, Pruesse E, Yilmaz P, Gerken J, Schweer T, Yarza P, et al. \href{https://doi.org/10.1093/nar/gks1219}{The SILVA ribosomal RNA gene database project: Improved data processing and web-based tools}. Nucleic Acids Research. 2013;41 Database issue:D590--6.

\leavevmode\vadjust pre{\hypertarget{ref-jordan1875}{}}%
162. Jordan C. \href{https://doi.org/10.24033/bsmf.90}{Essai sur la géométrie à n dimensions}. Bulletin de la Société Mathématique de France. 1875;3:103--74.

\leavevmode\vadjust pre{\hypertarget{ref-tenenhaus_component-based_2008}{}}%
163. Tenenhaus M. \href{https://doi.org/10.1080/14783360802159543}{Component-based {Structural Equation Modelling}}. Total Quality Management \& Business Excellence. 2008;19:871--86.

\leavevmode\vadjust pre{\hypertarget{ref-esposito_vinzi_bridge_2010}{}}%
164. Tenenhaus M, Hanafi M. \href{https://doi.org/10.1007/978-3-540-32827-8_5}{A {Bridge Between PLS Path Modeling} and {Multi}-{Block Data Analysis}}. In: Esposito Vinzi V, Chin WW, Henseler J, Wang H, editors. Handbook of {Partial Least Squares}. Berlin, Heidelberg: {Springer Berlin Heidelberg}; 2010. p. 99--123.

\leavevmode\vadjust pre{\hypertarget{ref-tenenhaus_regularized_2014}{}}%
165. Tenenhaus A, Tenenhaus M. \href{https://doi.org/10.1016/j.ejor.2014.01.008}{Regularized generalized canonical correlation analysis for multiblock or multigroup data analysis}. European Journal of Operational Research. 2014;238:391--403.

\leavevmode\vadjust pre{\hypertarget{ref-tenenhaus_kernel_2015}{}}%
166. Tenenhaus A, Philippe C, Frouin V. \href{https://doi.org/10.1016/j.csda.2015.04.004}{Kernel {Generalized Canonical Correlation Analysis}}. Computational Statistics \& Data Analysis. 2015;90:114--31.

\leavevmode\vadjust pre{\hypertarget{ref-gloaguen2020}{}}%
167. Gloaguen A, Philippe C, Frouin V, Gennari G, Dehaene-Lambertz G, Le Brusquet L, et al. Multiway generalized canonical correlation analysis. Biostatistics. 2020. \url{https://doi.org/10.1093/biostatistics/kxaa010}.

\leavevmode\vadjust pre{\hypertarget{ref-schuxe4fer2005}{}}%
168. Schäfer J, Strimmer K. \href{https://doi.org/10.2202/1544-6115.1175}{A Shrinkage Approach to Large-Scale Covariance Matrix Estimation and Implications for Functional Genomics}. Statistical Applications in Genetics and Molecular Biology. 2005;4.

\leavevmode\vadjust pre{\hypertarget{ref-horst1961}{}}%
169. Horst P. \href{https://doi.org/10.1007/BF02289710}{Relations among m sets of measures}. Psychometrika. 1961;26:129--49.

\leavevmode\vadjust pre{\hypertarget{ref-kettenring1971}{}}%
170. KETTENRING JR. \href{https://doi.org/10.1093/biomet/58.3.433}{Canonical analysis of several sets of variables}. Biometrika. 1971;58:433--51.

\leavevmode\vadjust pre{\hypertarget{ref-vandegeer1984}{}}%
171. Van de Geer JP. \href{https://doi.org/10.1007/BF02294207}{Linear relations among k sets of variables}. Psychometrika. 1984;49:79--94.

\leavevmode\vadjust pre{\hypertarget{ref-meng2014}{}}%
172. Meng C, Kuster B, Culhane AC, Gholami AM. \href{https://doi.org/10.1186/1471-2105-15-162}{A multivariate approach to the integration of multi-omics datasets}. BMC Bioinformatics. 2014;15:162.

\leavevmode\vadjust pre{\hypertarget{ref-planell2021}{}}%
173. Planell N, Lagani V, Sebastian-Leon P, van der Kloet F, Ewing E, Karathanasis N, et al. \href{https://doi.org/10.3389/fgene.2021.620453}{STATegra: Multi-omics data integration {\textendash} a conceptual scheme with a bioinformatics pipeline}. Frontiers in Genetics. 2021;12.

\leavevmode\vadjust pre{\hypertarget{ref-wu2021}{}}%
174. Wu T, Hu E, Xu S, Chen M, Guo P, Dai Z, et al. \href{https://doi.org/10.1016/j.xinn.2021.100141}{clusterProfiler 4.0: A universal enrichment tool for interpreting omics data}. The Innovation. 2021;2:100141.

\leavevmode\vadjust pre{\hypertarget{ref-richter2021}{}}%
175. Richter FC, Friedrich M, Pohin M, Alsaleh G, Guschina I, Wideman SK, et al. \href{https://doi.org/10.1101/2021.10.25.465200}{Cell-extrinsic autophagy in mature adipocytes regulates anti-inflammatory response to intestinal tissue injury through lipid mobilization}. 2021;2021.10.25.465200.

\leavevmode\vadjust pre{\hypertarget{ref-subramanian2005}{}}%
176. Subramanian A, Tamayo P, Mootha VK, Mukherjee S, Ebert BL, Gillette MA, et al. \href{https://doi.org/10.1073/pnas.0506580102}{Gene set enrichment analysis: A knowledge-based approach for interpreting genome-wide expression profiles}. Proceedings of the National Academy of Sciences. 2005;102:15545--50.

\leavevmode\vadjust pre{\hypertarget{ref-protiva2016}{}}%
177. Protiva P, Pendyala S, Nelson C, Augenlicht LH, Lipkin M, Holt PR. \href{https://doi.org/10.3945/ajcn.114.105304}{Calcium and 1,25-dihydroxyvitamin D3 modulate genes of immune and inflammatory pathways in the human colon: A human crossover trial}. The American Journal of Clinical Nutrition. 2016;103:1224--31.

\leavevmode\vadjust pre{\hypertarget{ref-korotkevich2021}{}}%
178. Korotkevich G, Sukhov V, Budin N, Shpak B, Artyomov MN, Sergushichev A. Fast gene set enrichment analysis. 2021. https://doi.org/\url{https://doi.org/10.1101/060012}.

\leavevmode\vadjust pre{\hypertarget{ref-fabregat2016}{}}%
179. Fabregat A, Sidiropoulos K, Garapati P, Gillespie M, Hausmann K, Haw R, et al. \href{https://doi.org/10.1093/nar/gkv1351}{The reactome pathway knowledgebase}. Nucleic Acids Research. 2016;44.

\leavevmode\vadjust pre{\hypertarget{ref-huxe4nzelmann2013}{}}%
180. Hänzelmann S, Castelo R, Guinney J. \href{https://doi.org/10.1186/1471-2105-14-7}{GSVA: Gene set variation analysis for microarray and RNA-seq data}. BMC Bioinformatics. 2013;14:7.

\leavevmode\vadjust pre{\hypertarget{ref-escudero-hernuxe1ndez2021}{}}%
181. Escudero-Hernández C, Beelen Granlund A van, Bruland T, Sandvik AK, Koch S, Østvik AE, et al. \href{https://doi.org/10.1016/j.jcmgh.2021.04.011}{Transcriptomic Profiling of Collagenous Colitis Identifies Hallmarks of Nondestructive Inflammatory Bowel Disease}. Cellular and Molecular Gastroenterology and Hepatology. 2021;12:665--87.

\leavevmode\vadjust pre{\hypertarget{ref-anderson2001}{}}%
182. Anderson MJ. \href{https://doi.org/10.1111/j.1442-9993.2001.01070.pp.x}{A new method for non-parametric multivariate analysis of variance}. Austral Ecology. 2001;26:32--46.

\leavevmode\vadjust pre{\hypertarget{ref-warton2012}{}}%
183. Warton DI, Wright ST, Wang Y. \href{https://doi.org/10.1111/j.2041-210X.2011.00127.x}{Distance-based multivariate analyses confound location and dispersion effects}. Methods in Ecology and Evolution. 2012;3:89--101.

\leavevmode\vadjust pre{\hypertarget{ref-goeman2006}{}}%
184. Goeman JJ, van de Geer SA, van Houwelingen HC. \href{https://www.jstor.org/stable/3879286}{Testing against a high dimensional alternative}. Journal of the Royal Statistical Society Series B (Statistical Methodology). 2006;68:477--93.

\leavevmode\vadjust pre{\hypertarget{ref-oksanen2020}{}}%
185. Oksanen J, Blanchet FG, Friendly M, Kindt R, Legendre P, McGlinn D, et al. \href{https://CRAN.R-project.org/package=vegan}{Vegan: Community ecology package}. 2020.

\leavevmode\vadjust pre{\hypertarget{ref-ritchie2015}{}}%
186. Ritchie MEM, Phipson B, Wu D, Hu Y, Law CWC, Shi W, et al. \href{https://doi.org/10.1093/nar/gkv007}{Limma powers differential expression analyses for RNA-sequencing and microarray studies}. Nucleic acids research. 2015;43:e47.

\leavevmode\vadjust pre{\hypertarget{ref-law2014}{}}%
187. Law CW, Chen Y, Shi W, Smyth GK. \href{https://doi.org/10.1186/gb-2014-15-2-r29}{Voom: Precision weights unlock linear model analysis tools for RNA-seq read counts}. Genome Biology. 2014;15:R29.

\leavevmode\vadjust pre{\hypertarget{ref-law2018}{}}%
188. Law CW, Alhamdoosh M, Su S, Dong X, Tian L, Smyth GK, et al. \href{https://doi.org/10.12688/f1000research.9005.3}{RNA-seq analysis is easy as 1-2-3 with limma, Glimma and edgeR}. F1000Research. 2018;5.

\leavevmode\vadjust pre{\hypertarget{ref-yoavbenjamini}{}}%
189. Yoav Benjamini, Yosef Hochberg. \href{http://www.jstor.org/stable/2346101}{Controlling the False Discovery Rate: A Practical and Powerful Approach to Multiple Testing}. Journal of the Royal Statistical Society Series B (Methodological). 57.

\leavevmode\vadjust pre{\hypertarget{ref-langfelder2008}{}}%
190. Langfelder P, Horvath S. \href{https://doi.org/10.1186/1471-2105-9-559}{WGCNA: An r package for weighted correlation network analysis}. BMC Bioinformatics. 2008;9:559.

\leavevmode\vadjust pre{\hypertarget{ref-filzmoser2004}{}}%
191. Filzmoser P, Viertl R. \href{https://doi.org/10.1007/s001840300269}{Testing hypotheses with fuzzy data: The fuzzy p -value}. Metrika. 2004;59:21--9.

\leavevmode\vadjust pre{\hypertarget{ref-dubois1993}{}}%
192. Dubois D, Prade H. \href{https://doi.org/10.1109/FUZZY.1993.327367}{{[}Proceedings 1993{]} second IEEE international conference on fuzzy systems}. 1993. p. 1059--1068 vol.2.

\leavevmode\vadjust pre{\hypertarget{ref-revillasancho2021}{}}%
193. Revilla Sancho L, Lozano J-J, Salas A. \href{https://doi.org/10.21105/joss.03358}{experDesign: stratifying samples into batches with minimal bias}. Journal of Open Source Software. 2021;6:3358.

\leavevmode\vadjust pre{\hypertarget{ref-robin2011}{}}%
194. Robin X, Turck N, Hainard A, Tiberti N, Lisacek F, Sanchez J-C, et al. \href{https://doi.org/10.1186/1471-2105-12-77}{pROC: An open-source package for r and s+ to analyze and compare ROC curves}. BMC Bioinformatics. 2011;12:77.

\leavevmode\vadjust pre{\hypertarget{ref-wickham2019}{}}%
195. Wickham H. \href{https://adv-r.hadley.nz/index.html}{Advanced r}. Second Edition. Boca Raton: Chapman \& Hall; 2019.

\leavevmode\vadjust pre{\hypertarget{ref-greenland2002}{}}%
196. Greenland S, Brumback B. \href{https://doi.org/10.1093/ije/31.5.1030}{An overview of relations among causal modelling methods}. International Journal of Epidemiology. 2002;31:1030--7.

\leavevmode\vadjust pre{\hypertarget{ref-revilla2021}{}}%
197. Revilla L, Mayorgas A, Corraliza AM, Masamunt MC, Metwaly A, Haller D, et al. \href{https://doi.org/10.1371/journal.pone.0246367}{Multi-omic modelling of inflammatory bowel disease with regularized canonical correlation analysis}. PLOS ONE. 2021;16:e0246367.

\leavevmode\vadjust pre{\hypertarget{ref-callahan2017}{}}%
198. Callahan BJ, McMurdie PJ, Holmes SP. \href{https://doi.org/10.1038/ismej.2017.119}{Exact sequence variants should replace operational taxonomic units in marker-gene data analysis}. The ISME Journal. 2017;11:2639--43.

\leavevmode\vadjust pre{\hypertarget{ref-nearing2022}{}}%
199. Nearing JT, Douglas GM, Hayes MG, MacDonald J, Desai DK, Allward N, et al. \href{https://doi.org/10.1038/s41467-022-28034-z}{Microbiome differential abundance methods produce different results across 38 datasets}. Nature Communications. 2022;13:342.

\leavevmode\vadjust pre{\hypertarget{ref-louca2018}{}}%
200. Louca S, Doebeli M, Parfrey LW. \href{https://doi.org/10.1186/s40168-018-0420-9}{Correcting for 16S rRNA gene copy numbers in microbiome surveys remains an unsolved problem}. Microbiome. 2018;6:41.

\leavevmode\vadjust pre{\hypertarget{ref-nadalian2021}{}}%
201. Nadalian B, Yadegar A, Houri H, Olfatifar M, Shahrokh S, Asadzadeh Aghdaei H, et al. \href{https://doi.org/10.1111/jgh.15260}{Prevalence of the pathobiont adherent-invasive Escherichia coli and inflammatory bowel disease: a systematic review and meta-analysis}. Journal of Gastroenterology and Hepatology. 2021;36:852--63.

\leavevmode\vadjust pre{\hypertarget{ref-chalise2016}{}}%
202. Chalise P, Raghavan R, Fridley BL. \href{https://doi.org/10.1016/j.cmpb.2016.02.011}{InterSIM: Simulation tool for multiple integrative {`}omic datasets{'}}. Computer methods and programs in biomedicine. 2016;128:69--74.

\leavevmode\vadjust pre{\hypertarget{ref-chung2019}{}}%
203. Chung R-H, Kang C-Y. \href{https://doi.org/10.1093/gigascience/giz045}{A multi-omics data simulator for complex disease studies and its application to evaluate multi-omics data analysis methods for disease classification}. GigaScience. 2019;8.

\leavevmode\vadjust pre{\hypertarget{ref-ugidos2020}{}}%
204. Ugidos M, Tarazona S, Prats-Montalbán JM, Ferrer A, Conesa A. MultiBaC: A strategy to remove batch effects between different omic data types: Statistical Methods in Medical Research. 2020. \url{https://doi.org/10.1177/0962280220907365}.

\leavevmode\vadjust pre{\hypertarget{ref-bersanelli2016}{}}%
205. Bersanelli M, Mosca E, Remondini D, Giampieri E, Sala C, Castellani G, et al. \href{https://doi.org/10.1186/s12859-015-0857-9}{Methods for the integration of multi-omics data: Mathematical aspects}. BMC Bioinformatics. 2016;17:S15.

\leavevmode\vadjust pre{\hypertarget{ref-holmes2012}{}}%
206. Holmes E, Li JV, Marchesi JR, Nicholson JK. \href{https://doi.org/10.1016/j.cmet.2012.10.007}{Gut Microbiota Composition and Activity in Relation to Host Metabolic Phenotype and Disease Risk}. Cell Metabolism. 2012;16:559--64.

\leavevmode\vadjust pre{\hypertarget{ref-stappenbeck2002}{}}%
207. Stappenbeck TS, Hooper LV, Gordon JI. \href{https://doi.org/10.1073/pnas.202604299}{Developmental regulation of intestinal angiogenesis by indigenous microbes via Paneth cells}. Proceedings of the National Academy of Sciences of the United States of America. 2002;99:15451--5.

\leavevmode\vadjust pre{\hypertarget{ref-brand2021}{}}%
208. Brand EC, Klaassen MAY, Gacesa R, Vich Vila A, Ghosh H, Zoete MR de, et al. \href{https://doi.org/10.1053/j.gastro.2021.01.030}{Healthy Cotwins Share Gut Microbiome Signatures With Their Inflammatory Bowel Disease Twins and Unrelated Patients}. Gastroenterology. 2021;160:1970--85.

\leavevmode\vadjust pre{\hypertarget{ref-susin2020}{}}%
209. Susin A, Wang Y, Lê~Cao K-A, Calle ML. \href{https://doi.org/10.1093/nargab/lqaa029}{Variable selection in microbiome compositional data analysis}. NAR Genomics and Bioinformatics. 2020;2:lqaa029.

\leavevmode\vadjust pre{\hypertarget{ref-sartor2006}{}}%
210. Sartor RB. \href{https://doi.org/10.1038/ncpgasthep0528}{Mechanisms of disease: pathogenesis of Crohn's disease and ulcerative colitis}. Nature Clinical Practice Gastroenterology \& Hepatology. 2006;3:390--407.

\leavevmode\vadjust pre{\hypertarget{ref-vandeputte2017}{}}%
211. Vandeputte D, Kathagen G, D'hoe K, Vieira-Silva S, Valles-Colomer M, Sabino J, et al. \href{https://doi.org/10.1038/nature24460}{Quantitative microbiome profiling links gut community variation to microbial load}. Nature. 2017;551:507--11.

\leavevmode\vadjust pre{\hypertarget{ref-vila-casadesuxfas2016}{}}%
212. Vila-Casadesús M, Gironella M, Lozano JJ. \href{https://doi.org/10.1371/journal.pone.0151127}{MiRComb: An R Package to Analyse miRNA-mRNA Interactions. Examples across Five Digestive Cancers}. PloS One. 2016;11:e0151127.

\leavevmode\vadjust pre{\hypertarget{ref-goldberg1991}{}}%
213. Goldberg D. \href{https://doi.org/10.1145/103162.103163}{What every computer scientist should know about floating-point arithmetic}. ACM Computing Surveys. 1991;23:548.

\leavevmode\vadjust pre{\hypertarget{ref-yan2012}{}}%
214. Yan L, Ma C, Wang D, Hu Q, Qin M, Conroy JM, et al. \href{https://doi.org/10.1186/1471-2164-13-689}{OSAT: A tool for sample-to-batch allocations in genomics experiments}. BMC Genomics. 2012;13:689.

\leavevmode\vadjust pre{\hypertarget{ref-papenberg2020}{}}%
215. Papenberg M, Klau GW. Using anticlustering to partition data sets into equivalent parts. Psychological Methods. 2020. \url{https://doi.org/10.1037/met0000301}.

\leavevmode\vadjust pre{\hypertarget{ref-sinke2021}{}}%
216. Sinke L, Cats D, Heijmans BT. Omixer: Multivariate and reproducible sample randomization to proactively counter batch effects in omics studies. Bioinformatics. 2021. \url{https://doi.org/10.1093/bioinformatics/btab159}.

\leavevmode\vadjust pre{\hypertarget{ref-haberman2014}{}}%
217. Haberman Y, Tickle TL, Dexheimer PJ, Kim M-O, Tang D, Karns R, et al. \href{https://doi.org/10.1172/JCI75436}{Pediatric Crohn disease patients exhibit specific ileal transcriptome and microbiome signature}. The Journal of Clinical Investigation. 2014;124:3617--33.

\leavevmode\vadjust pre{\hypertarget{ref-community2021}{}}%
218. Community TTW. \href{https://doi.org/10.5281/ZENODO.3233853}{The turing way: A handbook for reproducible, ethical and collaborative research}. The Turing Way Community; 2021.

\leavevmode\vadjust pre{\hypertarget{ref-mengMultivariateApproachIntegration2014}{}}%
219. Meng C, Kuster B, Culhane AC, Gholami AM. \href{https://doi.org/10.1186/1471-2105-15-162}{A multivariate approach to the integration of multi-omics datasets}. BMC Bioinformatics. 2014;15:162.

\leavevmode\vadjust pre{\hypertarget{ref-pearl2011}{}}%
220. Pearl J. \href{https://escholarship.org/uc/item/3tv1b3bg}{Transportability across studies: A formal approach}. 2011.

\leavevmode\vadjust pre{\hypertarget{ref-simpson1951}{}}%
221. Simpson EH. \href{https://doi.org/10.1111/j.2517-6161.1951.tb00088.x}{The Interpretation of Interaction in Contingency Tables}. Journal of the Royal Statistical Society: Series B (Methodological). 1951;13:238--41.

\leavevmode\vadjust pre{\hypertarget{ref-stillman1984}{}}%
222. Stillman DJ. Interactions of a Eukaryotic RNA Polymerase III Transcription Factor with Promoters. Biology; 1984.

\leavevmode\vadjust pre{\hypertarget{ref-lee1993}{}}%
223. Lee RC, Feinbaum RL, Ambros V. \href{https://doi.org/10.1016/0092-8674(93)90529-Y}{The C. elegans heterochronic gene lin-4 encodes small RNAs with antisense complementarity to lin-14}. Cell. 1993;75:843--54.

\leavevmode\vadjust pre{\hypertarget{ref-hamilton1999}{}}%
224. Hamilton AJ, Baulcombe DC. \href{https://doi.org/10.1126/science.286.5441.950}{A species of small antisense RNA in posttranscriptional gene silencing in plants}. Science. 1999;286:950--2.

\leavevmode\vadjust pre{\hypertarget{ref-thornburg2022}{}}%
225. Thornburg ZR, Bianchi DM, Brier TA, Gilbert BR, Earnest TM, Melo MCR, et al. \href{https://doi.org/10.1016/j.cell.2021.12.025}{Fundamental behaviors emerge from simulations of a living minimal cell}. Cell. 2022;185:345--360.e28.

\leavevmode\vadjust pre{\hypertarget{ref-criss2021}{}}%
226. Criss ZK, Bhasin N, Di Rienzi SC, Rajan A, Deans-Fielder K, Swaminathan G, et al. \href{https://doi.org/10.1152/physiolgenomics.00061.2021}{Drivers of transcriptional variance in human intestinal epithelial organoids}. Physiological Genomics. 2021;53:486--508.

\leavevmode\vadjust pre{\hypertarget{ref-arumugam2011}{}}%
227. Arumugam M, Raes J, Pelletier E, Le Paslier D, Yamada T, Mende DR, et al. \href{https://doi.org/10.1038/nature09944}{Enterotypes of the human gut microbiome}. Nature. 2011;473:174--80.

\leavevmode\vadjust pre{\hypertarget{ref-koren2013}{}}%
228. Koren O, Knights D, Gonzalez A, Waldron L, Segata N, Knight R, et al. \href{https://doi.org/10.1371/journal.pcbi.1002863}{A Guide to Enterotypes across the Human Body: Meta-Analysis of Microbial Community Structures in Human Microbiome Datasets}. PLOS Computational Biology. 2013;9:e1002863.

\leavevmode\vadjust pre{\hypertarget{ref-cheng2019}{}}%
229. Cheng M, Ning K. \href{https://doi.org/10.1016/j.gpb.2018.02.004}{Stereotypes About Enterotype: the Old and New Ideas}. Genomics, Proteomics \& Bioinformatics. 2019;17:4--12.

\leavevmode\vadjust pre{\hypertarget{ref-hasan2019}{}}%
230. Hasan N, Yang H. \href{https://doi.org/10.7717/peerj.7502}{Factors affecting the composition of the gut microbiota, and its modulation}. PeerJ. 2019;7:e7502.

\leavevmode\vadjust pre{\hypertarget{ref-bringiotti2014}{}}%
231. Bringiotti R, Ierardi E, Lovero R, Losurdo G, Leo AD, Principi M. \href{https://doi.org/10.4291/wjgp.v5.i4.550}{Intestinal microbiota: The explosive mixture at the origin of inflammatory bowel disease?} World Journal of Gastrointestinal Pathophysiology. 2014;5:550--9.

\leavevmode\vadjust pre{\hypertarget{ref-lloyd-price2019}{}}%
232. Lloyd-Price J, Arze C, Ananthakrishnan AN, Schirmer M, Avila-Pacheco J, Poon TW, et al. \href{https://doi.org/10.1038/s41586-019-1237-9}{Multi-omics of the gut microbial ecosystem in inflammatory bowel diseases}. Nature. 2019;569:655.

\leavevmode\vadjust pre{\hypertarget{ref-douglas2018}{}}%
233. Douglas GM, Hansen R, Jones CMA, Dunn KA, Comeau AM, Bielawski JP, et al. \href{https://doi.org/10.1186/s40168-018-0398-3}{Multi-omics differentially classify disease state and treatment outcome in pediatric crohn{'}s disease}. Microbiome. 2018;6:13.

\leavevmode\vadjust pre{\hypertarget{ref-halfvarson2017}{}}%
234. Halfvarson J, Brislawn CJ, Lamendella R, Vázquez-Baeza Y, Walters WA, Bramer LM, et al. \href{https://doi.org/10.1038/nmicrobiol.2017.4}{Dynamics of the human gut microbiome in inflammatory bowel disease}. Nature Microbiology. 2017;2:1--7.

\leavevmode\vadjust pre{\hypertarget{ref-edgar2018}{}}%
235. Edgar R. \href{https://doi.org/10.7717/peerj.5030}{Taxonomy annotation and guide tree errors in 16S rRNA databases}. PeerJ. 2018;6:e5030.

\leavevmode\vadjust pre{\hypertarget{ref-ungar2022}{}}%
236. Ungar B, Yavzori M, Fudim E, Picard O, Kopylov U, Eliakim R, et al. Host transcriptome signatures in human faecal-washes predict histological remission in patients with IBD. Gut. 2022. \url{https://doi.org/10.1136/gutjnl-2021-325516}.

\leavevmode\vadjust pre{\hypertarget{ref-zhang2019}{}}%
237. Zhang X, Li L, Butcher J, Stintzi A, Figeys D. \href{https://doi.org/10.1186/s40168-019-0767-6}{Advancing functional and translational microbiome research using meta-omics approaches}. Microbiome. 2019;7:154.

\leavevmode\vadjust pre{\hypertarget{ref-jiang2019}{}}%
238. Jiang D, Armour CR, Hu C, Mei M, Tian C, Sharpton TJ, et al. \href{https://doi.org/10.3389/fgene.2019.00995}{Microbiome multi-omics network analysis: Statistical considerations, limitations, and opportunities}. Frontiers in Genetics. 2019;10.

\leavevmode\vadjust pre{\hypertarget{ref-haynes2018}{}}%
239. Haynes WA, Tomczak A, Khatri P. \href{https://doi.org/10.1038/s41598-018-19333-x}{Gene annotation bias impedes biomedical research}. Scientific Reports. 2018;8:1362.

\leavevmode\vadjust pre{\hypertarget{ref-collij2021}{}}%
240. Collij V, Klaassen MAY, Weersma RK, Vila AV. \href{https://doi.org/10.1007/s00439-020-02218-3}{Gut microbiota in inflammatory bowel diseases: moving from basic science to clinical applications}. Human Genetics. 2021;140:703--8.

\leavevmode\vadjust pre{\hypertarget{ref-parkhomenko2009}{}}%
241. Parkhomenko E, Tritchler D, Beyene J. \href{https://doi.org/10.2202/1544-6115.1406}{Sparse Canonical Correlation Analysis with Application to Genomic Data Integration}. Statistical Applications in Genetics and Molecular Biology. 2009;8.

\leavevmode\vadjust pre{\hypertarget{ref-waaijenborg2008}{}}%
242. Waaijenborg S, Hamer PCV de W, Zwinderman AH. \href{https://doi.org/10.2202/1544-6115.1329}{Quantifying the Association between Gene Expressions and DNA-Markers by Penalized Canonical Correlation Analysis}. Statistical Applications in Genetics and Molecular Biology. 2008;7.

\leavevmode\vadjust pre{\hypertarget{ref-witten2009}{}}%
243. Witten DM, Tibshirani RJ. \href{https://doi.org/10.2202/1544-6115.1470}{Extensions of sparse canonical correlation analysis with applications to genomic data}. Statistical Applications in Genetics and Molecular Biology. 2009;8:127.

\leavevmode\vadjust pre{\hypertarget{ref-luxeacao2009}{}}%
244. Lê Cao K-A, Martin PG, Robert-Granié C, Besse P. \href{https://doi.org/10.1186/1471-2105-10-34}{Sparse canonical methods for biological data integration: Application to a cross-platform study}. BMC Bioinformatics. 2009;10:34.

\leavevmode\vadjust pre{\hypertarget{ref-hwang2009}{}}%
245. Hwang H. \href{https://doi.org/10.1007/s11336-009-9119-y}{Regularized Generalized Structured Component Analysis}. Psychometrika. 2009;74:517--30.

\leavevmode\vadjust pre{\hypertarget{ref-soneson2010}{}}%
246. Soneson C, Lilljebjörn H, Fioretos T, Fontes M. \href{https://doi.org/10.1186/1471-2105-11-191}{Integrative analysis of gene expression and copy number alterations using canonical correlation analysis}. BMC Bioinformatics. 2010;11:191.

\leavevmode\vadjust pre{\hypertarget{ref-zhang2011}{}}%
247. Zhang S, Li Q, Liu J, Zhou XJ. \href{https://doi.org/10.1093/bioinformatics/btr206}{A novel computational framework for simultaneous integration of multiple types of genomic data to identify microRNA-gene regulatory modules}. Bioinformatics. 2011;27:i401--9.

\leavevmode\vadjust pre{\hypertarget{ref-lee2011}{}}%
248. Lee W, Lee D, Lee Y, Pawitan Y. \href{https://doi.org/10.2202/1544-6115.1638}{Sparse Canonical Covariance Analysis for High-throughput Data}. Statistical Applications in Genetics and Molecular Biology. 2011;10.

\leavevmode\vadjust pre{\hypertarget{ref-abdi2012}{}}%
249. Abdi H, Williams LJ, Valentin D, Bennani-Dosse M. \href{https://doi.org/10.1002/wics.198}{STATIS and DISTATIS: optimum multitable principal component analysis and three way metric multidimensional scaling}. WIREs Computational Statistics. 2012;4:124--67.

\leavevmode\vadjust pre{\hypertarget{ref-zhang2012}{}}%
250. Zhang S, Liu C-C, Li W, Shen H, Laird PW, Zhou XJ. \href{https://doi.org/10.1093/nar/gks725}{Discovery of multi-dimensional modules by integrative analysis of cancer genomic data}. Nucleic Acids Research. 2012;40:9379--91.

\leavevmode\vadjust pre{\hypertarget{ref-li2012}{}}%
251. Li W, Zhang S, Liu C-C, Zhou XJ. \href{https://doi.org/10.1093/bioinformatics/bts476}{Identifying multi-layer gene regulatory modules from multi-dimensional genomic data}. Bioinformatics. 2012;28:2458--66.

\leavevmode\vadjust pre{\hypertarget{ref-abdi2013}{}}%
252. Abdi H, Williams LJ, Valentin D. \href{https://doi.org/10.1002/wics.1246}{Multiple factor analysis: principal component analysis for multitable and multiblock data sets}. WIREs Computational Statistics. 2013;5:149--79.

\leavevmode\vadjust pre{\hypertarget{ref-schlauch2017}{}}%
253. Schlauch D, Paulson JN, Young A, Glass K, Quackenbush J. \href{https://doi.org/10.1093/bioinformatics/btx139}{Estimating gene regulatory networks with pandaR}. Bioinformatics. 2017;33:2232--4.

\leavevmode\vadjust pre{\hypertarget{ref-ray2014}{}}%
254. Ray P, Zheng L, Lucas J, Carin L. \href{https://doi.org/10.1093/bioinformatics/btu064}{Bayesian joint analysis of heterogeneous genomics data}. Bioinformatics. 2014;30:1370--6.

\leavevmode\vadjust pre{\hypertarget{ref-bunte2016}{}}%
255. Bunte K, Leppäaho E, Saarinen I, Kaski S. \href{https://doi.org/10.1093/bioinformatics/btw207}{Sparse group factor analysis for biclustering of multiple data sources}. Bioinformatics. 2016;32:2457--63.

\leavevmode\vadjust pre{\hypertarget{ref-chen2013}{}}%
256. Chen M, Gao C, Ren Z, Zhou HH. \href{http://arxiv.org/abs/1311.6186}{Sparse CCA via precision adjusted iterative thresholding}. arXiv:13116186 {[}math, stat{]}. 2013.

\leavevmode\vadjust pre{\hypertarget{ref-leppuxe4aho2017}{}}%
257. Leppäaho E, Ammad-ud-din M, Kaski S. \href{http://jmlr.org/papers/v18/16-509.html}{GFA: Exploratory analysis of multiple data sources with group factor analysis}. Journal of Machine Learning Research. 2017;18:1--5.

\leavevmode\vadjust pre{\hypertarget{ref-klami2014}{}}%
258. Klami A, Bouchard G, Tripathi A. \href{http://arxiv.org/abs/1312.5921}{Group-sparse embeddings in collective matrix factorization}. arXiv:13125921 {[}cs, stat{]}. 2014.

\leavevmode\vadjust pre{\hypertarget{ref-meng2018}{}}%
259. Meng C, Basunia A, Peters B, Gholami AM, Kuster B, Culhane AC. \href{https://www.biorxiv.org/content/10.1101/046904v2}{MOGSA: integrative single sample gene-set analysis of multiple omics data}. 2018.

\leavevmode\vadjust pre{\hypertarget{ref-zhao2015}{}}%
260. Zhao S, Gao C, Mukherjee S, Engelhardt BE. \href{http://arxiv.org/abs/1411.2698}{Bayesian group latent factor analysis with structured sparsity}. arXiv:14112698 {[}q-bio, stat{]}. 2015.

\leavevmode\vadjust pre{\hypertarget{ref-voillet2016}{}}%
261. Voillet V, Besse P, Liaubet L, San Cristobal M, González I. \href{https://doi.org/10.1186/s12859-016-1273-5}{Handling missing rows in multi-omics data integration: Multiple imputation in multiple factor analysis framework}. BMC Bioinformatics. 2016;17:402.

\leavevmode\vadjust pre{\hypertarget{ref-beaton2016}{}}%
262. Beaton D, Dunlop J, Abdi H, Alzheimer's Disease Neuroimaging Initiative. \href{https://doi.org/10.1037/met0000053}{Partial least squares correspondence analysis: A framework to simultaneously analyze behavioral and genetic data}. Psychological Methods. 2016;21:621--51.

\leavevmode\vadjust pre{\hypertarget{ref-singh2019}{}}%
263. Singh A, Shannon CP, Gautier B, Rohart F, Vacher M, Tebbutt SJ, et al. \href{https://doi.org/10.1093/bioinformatics/bty1054}{DIABLO: An integrative approach for identifying key molecular drivers from multi-omics assays}. Bioinformatics. 2019;35:3055--62.

\leavevmode\vadjust pre{\hypertarget{ref-yoon2019}{}}%
264. Yoon G, Carroll RJ, Gaynanova I. \href{http://arxiv.org/abs/1807.05274}{Sparse semiparametric canonical correlation analysis for data of mixed types}. arXiv:180705274 {[}stat{]}. 2019.

\leavevmode\vadjust pre{\hypertarget{ref-gaynanova2017}{}}%
265. Gaynanova I, Li G. \href{http://arxiv.org/abs/1707.06573}{Structural learning and integrative decomposition of multi-view data}. arXiv:170706573 {[}stat{]}. 2017.

\leavevmode\vadjust pre{\hypertarget{ref-madrigal2017}{}}%
266. Madrigal P. \href{https://doi.org/10.1093/bioinformatics/btw724}{fCCAC: Functional canonical correlation analysis to evaluate covariance between nucleic acid sequencing datasets}. Bioinformatics. 2017;33:746--8.

\leavevmode\vadjust pre{\hypertarget{ref-yoshida2017}{}}%
267. Yoshida K, Yoshimoto J, Doya K. \href{https://doi.org/10.1186/s12859-017-1543-x}{Sparse kernel canonical correlation analysis for discovery of nonlinear interactions in high-dimensional data}. BMC Bioinformatics. 2017;18:108.

\leavevmode\vadjust pre{\hypertarget{ref-kawaguchi2017}{}}%
268. Kawaguchi A, Yamashita F. \href{https://doi.org/10.1093/biostatistics/kxx011}{Supervised multiblock sparse multivariable analysis with application to multimodal brain imaging genetics}. Biostatistics. 2017;18:651--65.

\leavevmode\vadjust pre{\hypertarget{ref-feng2018}{}}%
269. Feng Q, Jiang M, Hannig J, Marron JS. \href{http://arxiv.org/abs/1704.02060}{Angle-based joint and individual variation explained}. arXiv:170402060 {[}stat{]}. 2018.

\leavevmode\vadjust pre{\hypertarget{ref-argelaguet2020}{}}%
270. Argelaguet R, Arnol D, Bredikhin D, Deloro Y, Velten B, Marioni JC, et al. \href{https://doi.org/10.1186/s13059-020-02015-1}{MOFA+: A statistical framework for comprehensive integration of multi-modal single-cell data}. Genome Biology. 2020;21:111.

\leavevmode\vadjust pre{\hypertarget{ref-brown2018}{}}%
271. Brown BC, Bray NL, Pachter L. Expression reflects population structure. 2018. \url{https://doi.org/10.1101/364448}.

\leavevmode\vadjust pre{\hypertarget{ref-zhang2020}{}}%
272. Zhang Y, Gaynanova I. \href{http://arxiv.org/abs/1811.08511}{Joint association and classification analysis of multi-view data}. arXiv:181108511 {[}cs, stat{]}. 2020.

\leavevmode\vadjust pre{\hypertarget{ref-tang2021}{}}%
273. Tang TM, Allen GI. \href{http://arxiv.org/abs/1810.00832}{Integrated principal components analysis}. arXiv:181000832 {[}stat{]}. 2021.

\leavevmode\vadjust pre{\hypertarget{ref-min2019}{}}%
274. Min EJ, Safo SE, Long Q. \href{https://doi.org/10.1093/bioinformatics/bty726}{Penalized co-inertia analysis with applications to -omics data}. Bioinformatics (Oxford, England). 2019;35:1018--25.

\leavevmode\vadjust pre{\hypertarget{ref-safo2018}{}}%
275. Safo SE, Li S, Long Q. \href{https://doi.org/10.1111/biom.12715}{Integrative analysis of transcriptomic and metabolomic data via sparse canonical correlation analysis with incorporation of biological information}. Biometrics. 2018;74:300--12.

\leavevmode\vadjust pre{\hypertarget{ref-min2017}{}}%
276. Min W, Liu J, Zhang S. \href{http://arxiv.org/abs/1710.04792}{Sparse weighted canonical correlation analysis}. arXiv:171004792 {[}cs, stat{]}. 2017.

\leavevmode\vadjust pre{\hypertarget{ref-bouhaddani2018}{}}%
277. Bouhaddani S el, Uh H-W, Jongbloed G, Hayward C, Klarić L, Kiełbasa SM, et al. \href{https://doi.org/10.1186/s12859-018-2371-3}{Integrating omics datasets with the OmicsPLS package}. BMC Bioinformatics. 2018;19:371.

\leavevmode\vadjust pre{\hypertarget{ref-pimentel2018}{}}%
278. Pimentel H, Zhiyue H, Huang H. \href{https://doi.org/10.1007/s40484-017-0127-0}{Biclustering by sparse canonical correlation analysis}. 2018;6:11.

\leavevmode\vadjust pre{\hypertarget{ref-kim2019}{}}%
279. Kim Y, Bismeijer T, Zwart W, Wessels LFA, Vis DJ. \href{https://doi.org/10.1038/s41467-019-13027-2}{Genomic data integration by WON-PARAFAC identifies interpretable factors for predicting drug-sensitivity in vivo}. Nature Communications. 2019;10:5034.

\leavevmode\vadjust pre{\hypertarget{ref-lock2020}{}}%
280. Lock EF, Park JY, Hoadley KA. \href{http://arxiv.org/abs/2002.02601}{Bidimensional linked matrix factorization for pan-omics pan-cancer analysis}. arXiv:200202601 {[}cs, q-bio, stat{]}. 2020.

\leavevmode\vadjust pre{\hypertarget{ref-ronen2019}{}}%
281. Ronen J, Hayat S, Akalin A. \href{https://doi.org/10.26508/lsa.201900517}{Evaluation of colorectal cancer subtypes and cell lines using deep learning}. Life Science Alliance. 2019;2.

\leavevmode\vadjust pre{\hypertarget{ref-shi2019}{}}%
282. Shi WJ, Zhuang Y, Russell PH, Hobbs BD, Parker MM, Castaldi PJ, et al. \href{https://doi.org/10.1093/bioinformatics/btz226}{Unsupervised discovery of phenotype-specific multi-omics networks}. Bioinformatics. 2019;35:4336--43.

\leavevmode\vadjust pre{\hypertarget{ref-csala2020}{}}%
283. Csala A, Zwinderman AH, Hof MH. \href{https://doi.org/10.1186/s12859-019-3286-3}{Multiset sparse partial least squares path modeling for high dimensional omics data analysis}. BMC Bioinformatics. 2020;21:9.

\leavevmode\vadjust pre{\hypertarget{ref-fan2020}{}}%
284. Fan Z, Zhou Y, Ressom HW. \href{https://doi.org/10.3390/metabo10040144}{MOTA: Network-Based Multi-Omic Data Integration for Biomarker Discovery}. Metabolites. 2020;10:144.

\leavevmode\vadjust pre{\hypertarget{ref-shu2020}{}}%
285. Shu H, Wang X, Zhu H. \href{https://doi.org/10.1080/01621459.2018.1543599}{D-CCA: A decomposition-based canonical correlation analysis for high-dimensional datasets}. Journal of the American Statistical Association. 2020;115:292--306.

\leavevmode\vadjust pre{\hypertarget{ref-hawinkel2020}{}}%
286. Hawinkel S, Bijnens L, Cao K-AL, Thas O. \href{https://doi.org/10.1093/nargab/lqaa050}{Model-based joint visualization of multiple compositional omics datasets}. NAR Genomics and Bioinformatics. 2020;2.

\leavevmode\vadjust pre{\hypertarget{ref-gundersen2020}{}}%
287. Gundersen G, Dumitrascu B, Ash JT, Engelhardt BE. \href{https://proceedings.mlr.press/v115/gundersen20a.html}{Uncertainty in Artificial Intelligence}. PMLR; 2020. p. 945--55.

\leavevmode\vadjust pre{\hypertarget{ref-velten2020}{}}%
288. Velten B, Braunger JM, Arnol D, Argelaguet R, Stegle O. \href{https://doi.org/10.1101/2020.11.03.366674}{Identifying temporal and spatial patterns of variation from multi-modal data using MEFISTO}. bioRxiv. 2020;2020.11.03.366674.

\end{CSLReferences}

\afterpage{\setcounter{chapshift}{0}}

\hypertarget{appendix-appendix}{%
\appendix \addcontentsline{toc}{chapter}{\appendixname}}


\hypertarget{online-resources}{%
\chapter{Online resources}\label{online-resources}}

Some links that we found useful on the thesis and could be useful if you are interested on the multi-omics field.

\begin{itemize}
\tightlist
\item
  \href{https://github.com/mikelove/awesome-multi-omics}{Awesome multi-omics https://github.com/mikelove/awesome-multi-omics} : An online repository of references to multi-omics methods. Reproduced here with their references \footnote{Consulted on 2021/11/10 from \url{https://github.com/mikelove/awesome-multi-omics}}:
\end{itemize}

\begin{longtable}[]{@{}cc@{}}
\caption{\label{tab:appendix-methods} Integration methods available and their references.}\tabularnewline
\toprule
Method & Publication \\
\midrule
\endfirsthead
\toprule
Method & Publication \\
\midrule
\endhead
SCCA & {[}\protect\hyperlink{ref-parkhomenko2009}{241}{]} \\
PCCA & {[}\protect\hyperlink{ref-waaijenborg2008}{242}{]} \\
PMA & {[}\protect\hyperlink{ref-witten2009}{243}{]} \\
sPLS & {[}\protect\hyperlink{ref-luxeacao2009}{244}{]} \\
gesca & {[}\protect\hyperlink{ref-hwang2009}{245}{]} \\
Regularized dual CCA & {[}\protect\hyperlink{ref-soneson2010}{246}{]} \\
RGCCA & {[}\protect\hyperlink{ref-tenenhaus_regularized_2011}{115}{]} \\
SNMNMF & {[}\protect\hyperlink{ref-zhang2011}{247}{]} \\
scca & {[}\protect\hyperlink{ref-lee2011}{248}{]} \\
STATIS & {[}\protect\hyperlink{ref-abdi2012}{249}{]} \\
joint NMF & {[}\protect\hyperlink{ref-zhang2012}{250}{]} \\
sMBPLS & {[}\protect\hyperlink{ref-li2012}{251}{]} \\
Bayesian group factor analysis & {[}\protect\hyperlink{ref-virtanen2012}{133}{]} \\
RIMBANET & {[}\protect\hyperlink{ref-zhu2012}{128}{]} \\
FactoMineR & {[}\protect\hyperlink{ref-abdi2013}{252}{]} \\
JIVE & {[}\protect\hyperlink{ref-lock2013a}{135}{]} \\
pandaR & {[}\protect\hyperlink{ref-schlauch2017}{253}{]} \\
omicade4 & {[}\protect\hyperlink{ref-meng2014}{172}{]} \\
STATegRa & {[}\protect\hyperlink{ref-planell2021}{173}{]} \\
Joint factor model & {[}\protect\hyperlink{ref-ray2014}{254}{]} \\
GFAsparse & {[}\protect\hyperlink{ref-bunte2016}{255}{]} \\
Sparse CCA & {[}\protect\hyperlink{ref-chen2013}{256}{]} \\
CCAGFA & {[}\protect\hyperlink{ref-leppuxe4aho2017}{257}{]} \\
CMF & {[}\protect\hyperlink{ref-klami2014}{258}{]} \\
MOGSA & {[}\protect\hyperlink{ref-meng2018}{259}{]} \\
iNMF & {[}\protect\hyperlink{ref-yang2016}{136}{]} \\
BASS & {[}\protect\hyperlink{ref-zhao2015}{260}{]} \\
imputeMFA & {[}\protect\hyperlink{ref-voillet2016}{261}{]} \\
PLSCA & {[}\protect\hyperlink{ref-beaton2016}{262}{]} \\
mixOmics & {[}\protect\hyperlink{ref-singh2019}{263}{]} \\
mixedCCA & {[}\protect\hyperlink{ref-yoon2019}{264}{]} \\
SLIDE & {[}\protect\hyperlink{ref-gaynanova2017}{265}{]} \\
fCCAC & {[}\protect\hyperlink{ref-madrigal2017}{266}{]} \\
TSKCCA & {[}\protect\hyperlink{ref-yoshida2017}{267}{]} \\
SMSMA & {[}\protect\hyperlink{ref-kawaguchi2017}{268}{]} \\
AJIVE & {[}\protect\hyperlink{ref-feng2018}{269}{]} \\
MOFA & {[}\protect\hyperlink{ref-argelaguet2020}{270}{]} \\
PCA+CCA & {[}\protect\hyperlink{ref-brown2018}{271}{]} \\
JACA & {[}\protect\hyperlink{ref-zhang2020}{272}{]} \\
iPCA & {[}\protect\hyperlink{ref-tang2021}{273}{]} \\
pCIA & {[}\protect\hyperlink{ref-min2019}{274}{]} \\
sSCCA & {[}\protect\hyperlink{ref-safo2018}{275}{]} \\
SWCCA & {[}\protect\hyperlink{ref-min2017}{276}{]} \\
OmicsPLS & {[}\protect\hyperlink{ref-bouhaddani2018}{277}{]} \\
SCCA-BC & {[}\protect\hyperlink{ref-pimentel2018}{278}{]} \\
WON-PARAFAC & {[}\protect\hyperlink{ref-kim2019}{279}{]} \\
BIDIFAC & {[}\protect\hyperlink{ref-lock2020}{280}{]} \\
maui & {[}\protect\hyperlink{ref-ronen2019}{281}{]} \\
SmCCNet & {[}\protect\hyperlink{ref-shi2019}{282}{]} \\
msPLS & {[}\protect\hyperlink{ref-csala2020}{283}{]} \\
MOTA & {[}\protect\hyperlink{ref-fan2020}{284}{]} \\
D-CCA & {[}\protect\hyperlink{ref-shu2020}{285}{]} \\
COMBI & {[}\protect\hyperlink{ref-hawinkel2020}{286}{]} \\
DPCCA & {[}\protect\hyperlink{ref-gundersen2020}{287}{]} \\
MEFISTO & {[}\protect\hyperlink{ref-velten2020}{288}{]} \\
MultiPower & {[}\protect\hyperlink{ref-tarazona2020}{108}{]} \\
\bottomrule
\end{longtable}

\begin{itemize}
\item
  \href{https://github.com/rstudio/bookdown/}{Bookdown}: The book about how to write this type of books.
\item
  \href{https://bioconducto.org}{Bioconductor}: The project about bioinformatics on R mostly related to sequencing technologies.
\item
  \href{https://cran.r-project.org}{CRAN}: The main archive of R extensions/packages for R.
\item
  \href{https://github.com}{GitHub}: Company which allows to freely host remote \href{https://git-scm.com/}{git} repositories of many projects, including some used or developed on this thesis.
\end{itemize}

\hypertarget{software}{%
\chapter{Software}\label{software}}

Along the years of this thesis several pieces of software have been generated as well as packages.
Here they are listed for easier retrieval.
They are listed on two ways, one with a brief explanation and another one ordered by what software piece is used on each analysis.

\hypertarget{star}{%
\section{STAR}\label{star}}

The parameters and options used with STAR are:

\begin{verbatim}
STAR \
    --outSAMtype BAM SortedByCoordinate \
    --outFilterIntronMotifs RemoveNoncanonical \
    --outSAMattributes All \
    --outReadsUnmapped Fastx \
    --outSAMstrandField intronMotif \
    --outFilterScoreMinOverLread 0.5 \
    --outFilterMatchNminOverLread 0.5 \
    --outFilterType BySJout \
    --alignSJoverhangMin 8 \
    --alignSJDBoverhangMin 1 \
    --outFilterMismatchNmax 999 \
    --outFilterMismatchNoverLmax 0.04 \
    --genomeDir "$genome/STAR" \
    --limitBAMsortRAM 10000000000 \
    --runMode alignReads \
    --genomeLoad NoSharedMemory \
    --quantMode TranscriptomeSAM \
    --outFileNamePrefix $output \
    --runThreadN "$threads" \
    --readFilesCommand zcat \
    --readFilesIn "$file1" "$file2"
\end{verbatim}

The \texttt{\$genome} is the path to the location on the computer where the genome is, \texttt{\$output} is the prefix of the output file, \texttt{\$threads} is the number of threads used and \texttt{\$file1} and \texttt{\$file2} are the paired fastq files.

\hypertarget{rsem}{%
\section{RSEM}\label{rsem}}

Code used for RSEM where \texttt{\$threads} is the number of threads used, \texttt{\$rseminp} is the input file in BAM format, \texttt{\$genome} is the path to the location on the computer where the genome is, and \texttt{\$rsem} is the output file.

\begin{verbatim}
rsem-calculate-expression \
    --quiet \
    --paired-end \
    -p "$threads"  \
    --estimate-rspd \
    --append-names \
    --no-bam-output \
    --bam "$rseminp" "$genome/RSEM/RSEM" "$rsem"
    
    
\end{verbatim}

\hypertarget{listed}{%
\section{Listed}\label{listed}}

An improved/tested version of \href{https://github.com/llrs/RGCCA}{RGCCA}, some modifications on the internal functions to ease the maintenance as well as adding tests and sometimes improving the documentation.
Also modified so that it is possible to provide a vector of models so that the model of the first dimension is not the same as the model on the second dimension (not sure if mathematically speaking makes sense but from a biological one we think might be interesting to have it).

Designed to be used with RGCCA we wrote the package \href{https:github.com/llrs/inteRmodel}{inteRmodel} to ease the bootstrapping and model selection.

A package to design batches to avoid batch effect \href{https://cran.r-project.org/package=experDesign}{experDesign} and its website on \href{https://github.com/llrs/experDesign}{GitHub}.

Explore the effects of the hyperparameters on RGCCA on the provided dataset of \href{https://github.com/llrs/gliomaData}{gliomaData} (Originally provided \href{http://biodev.cea.fr/sgcca/}{here}) there is this repository \href{https://github.com/llrs/sgcca_hyperparameters}{sgcca\_hyperparameters}.

We used a pouchitis cohort published in this \href{https://dx.doi.org/10.1186/s13059-015-0637-x}{article}{[}\protect\hyperlink{ref-morgan2015}{150}{]} that was used to compare how performs our method in other's dataset.
The code used can be found in \href{https://github.com/llrs/pouchitis}{this repository}.

Some functions used to explore the TRIM dataset ended up in the \href{https://github.com/llrs/integration-helper}{integration} package.This include functions for correlation, network analysis, enrichment, normalization of metadata\ldots{}

We developed a package to analyze sets and fuzzy sets \href{https://github.com/rOpenSci/BaseSet}{BaseSet} (based on what we learned from a previous iteration of the \href{https://github.com/llrs/GSEAdv}{package}).
This package was meant to be used with the probabilities that arise from bootstrapping the models.
However, due to the long times of calculation that it would require it was not used.

To analyze the BARCELONA cohort (also named antiTNF) a \href{https://github.com/llrs/Barcelona}{different repository} was created to analyze the data using the previously developed packages.

\hypertarget{by-projectpublication}{%
\section{By project/publication}\label{by-projectpublication}}

All code of the analysis of the publications is available (in his messed state and complicated history) and a brief description as to why they were used:

\href{https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0246367}{Multi-omic modelling of inflammatory bowel disease with regularized canonical correlation analysis}:

\begin{itemize}
\tightlist
\item
  \href{https://github.com/llrs/TRIM}{TRIM}: Mangle with the sample, dataset, explore several methods\ldots{}
\item
  \href{https://github.com/llrs/sgcca_hyperparameters}{Puget's}: Explore the effects of the hyperparameters on RGCCA on the provided dataset.
\item
  \href{https://github.com/llrs/inteRmodel}{inteRmodel}: Package for easy repeating the methodology developed on TRIM.
\item
  \href{https://github.com/llrs/pouchitis}{Morgan's}: Work with the pouchitis cohort used in this article.
\item
  \href{https://github.com/llrs/Uncoupling}{Häsler's}: Work with the UC/CD dataset used in this article.
\item
  \href{https://github.com/llrs/integration-helper}{integration}: Package with functions that we wrote or used on different parts of exploring the TRIM dataset ended up here.
\end{itemize}

BaseSet:

\begin{itemize}
\tightlist
\item
  \href{https://cran.r-project.org/package=BaseSet}{BaseSet}: Fuzzy logic implementation, available on \href{https://github.com/ropensci/BaseSet}{rOpenSci too} and its \href{https://docs.ropensci.org/BaseSet}{documentation website} .
\end{itemize}

experDesign:

\begin{itemize}
\tightlist
\item
  \href{https://cran.r-project.org/package=experDesign}{experDesign}: Help design experiments in batches. It has a \href{https://experDesign.llrs.dev/}{documentation website too}.
\end{itemize}

BARCELONA:

\begin{itemize}
\tightlist
\item
  \href{https://github.com/llrs/Barcelona}{BARCELONA}: Code to analyze the BARCELONA's dataset
\end{itemize}

Validation:

\begin{itemize}
\tightlist
\item
  \href{https://github.com/llrs/howell_2018}{Howell's}: Work with Howell's 2018 dataset.
\item
  \href{https://github.com/llrs/cristian_mark}{Cristian's}: Work with Cristian's 2020 dataset.
\end{itemize}

\hypertarget{articles}{%
\chapter{Articles}\label{articles}}

Articles published on peer-review journals about this thesis:

\hypertarget{app-multiomics}{%
\section{Multi-omic modelling of inflammatory bowel disease with regularized canonical correlation analysis}\label{app-multiomics}}

\sectionmark{inteRmodel: paper}

Article peer-reviewed published on 2021, freely \href{https://doi.org/10.1371/journal.pone.0246367}{available online}.

The same article can be find below.

\includepdf[pages=-, pagecommand={}, templatesize={\textwidth}{\textheight  - 25pt}, trim=0 0 0 20pt,]{paper_multiomics.pdf}

\hypertarget{app-experDesign}{%
\section{experDesign: stratifying samples into batches with minimal bias}\label{app-experDesign}}

\sectionmark{experDesign: paper}

Article peer-reviewed published on 2021, freely \href{https://joss.theoj.org/papers/10.21105/joss.03358}{available online}.

The same article can be find below.

\includepdf[pages=-, pagecommand={}, templatesize={\textwidth}{\textheight - \voffset}, trim=0 0 0 20pt,]{paper_experDesign.pdf}

\hypertarget{other-datasets}{%
\chapter{Other datasets}\label{other-datasets}}

\hypertarget{barcelona}{%
\section{BARCELONA dataset}\label{barcelona}}

All patients with an established diagnosis of IBD, including Crohn's disease, ulcerative colitis, unclassified IBD, indeterminate colitis, or pouchitis, starting treatment with a biologic agent were monitored following the schedule of clinical visits, laboratory tests, imaging procedures and biologic sampling at the beginning of their treatment with anti-TNF therapy and after 14 weeks and 46 weeks a biopsy from an ileocolonoscopy.
The protocol was approved by the Institutional Ethics Committee of the Hospital Clinic de Barcelona (Study Number HCB/2012/7845 and HCB/2012/7956).

Patients that were referred to the Hospital Clínic de Barcelona IBD unit, who had already started treatment with a biologic agent in another center, were also included adapting to the corresponding time-schedule of their treatment.
In all patients, starting anti-TNF treatment will be decided before the protocol entry decision according to medical clinical practice.

Anonymized identification of the patients, disease, sex age at diagnostic, age at the moment of the sample taking, time since the start of the treatment and sample segment was collected.

\begin{longtable}[]{@{}ll@{}}
\caption{\label{tab:BARCELONA} Samples included from the BARCELONA dataset characteristics.}\tabularnewline
\toprule
Characteristic & BARCELONA \\
\midrule
\endfirsthead
\toprule
Characteristic & BARCELONA \\
\midrule
\endhead
Individuals & 62 \\
Status (non-IBD/CD/UC) & 8/33/21 \\
Sex (female/male) & 29/33 \\
Age at diagnostic (\textless17/\textless40/\textgreater40 years) & 2/44/8 \\
Years of disease: mean (min-max) & 7.6 (0-32) \\
Age: mean (min-max) & 41 (18-68) \\
Time (0/14/46 weeks) & 41/40/32 \\
Sample segment (ileum/colon) & 39/87 \\
\bottomrule
\end{longtable}

The process of DNA extraction and sequencing was different for this dataset.
We used different 16S-V3V4 primers pair 341f/806r on a MiSeq Nano sequencing by the RTSF Genomics Core at Michigan State University, United State of America.
The sequence of the primers used was:

341f: 5'-CCTACGGGAGGCAGCAG-3'\\
806r: 5'-GGACTACHVHHHTWTCTAAT-3'

The result of MiSeq Nano were processed with bcl2fastq (v1.8.4).

This dataset was processed as usual but as part of the quality controls of the dataset the diversity measures of the samples was analyzed on \ref{fig:barcelona-diversity}:

\begin{figure}
\includegraphics[width=1\linewidth]{images/barcelona-diversity} \caption[Diversity indices of Barcelona according to the location and disease status.]{Diversity indices of Barcelona according to the location and disease status. There is a lot of diversity between different groups but importantly the control samples overlap with the patients with inflammatory bowel disease.}\label{fig:barcelona-diversity}
\end{figure}

Control samples diversity should be lower and not on the same range as with samples of patients with IBD.
The dataset's 16S was sequenced several times by different platforms.
Despite the pilots and the negative controls on the sequencing process, each time there were different problems: contamination, low quality and then this suspicious diversity.
It does not seem to be a problem of the sequencing facility, so this data was abandoned as unreliable.

\hypertarget{hernandez}{%
\section{Hernández' dataset}\label{hernandez}}

This dataset was obtained from collaborators at Mount Sinai, Toronto, Canada {[}\protect\hyperlink{ref-hernuxe1ndez-rocha2021}{82}{]}.

Patients with UC, CD were recruited when attending regularly scheduled visits or surveillance.
In addition, asymptomatic healthy controls (HC) were recruited during routine, age-related colorectal cancer screening by colonoscopy.
290 samples were collected together with information about the disease, age at diagnosis, age at the moment of the sampling, sex, sample location and smoking status.

\begin{longtable}[]{@{}ll@{}}
\caption{\label{tab:hernandez} Characteristics of samples included from Hernández's dataset.}\tabularnewline
\toprule
Characteristic & Hernández' \\
\midrule
\endfirsthead
\toprule
Characteristic & Hernández' \\
\midrule
\endhead
Disease (non-IBD/CD/UC) & 46/54/66 \\
Age at diagnostic (\textless17/\textless40/\textgreater40 years) & 29/73/18 \\
Age: mean (min-max) & 40 (17-71) \\
Sex (female/male) & 81/85 \\
Smoking (never/ex/current) & 115/34/16 \\
Sample Location (ileum/colon) & 97/193 \\
\bottomrule
\end{longtable}

\hypertarget{results-hernandez}{%
\subsection{Results}\label{results-hernandez}}

It substitutes the BARCELONA dataset to confirm the results on the previous datasets.
However, at the time of writing the process is not complete.

\hypertarget{models-output}{%
\chapter{Models output}\label{models-output}}

`

The following appendices show the top variables for models of each dataset and their weight according to a model.

\hypertarget{hsct}{%
\section{HSCT}\label{hsct}}

\hypertarget{genes}{%
\subsection{Genes}\label{genes}}

\begin{longtable}[t]{cc}
\caption[HSCT genes weight in model 0]{\label{tab:hsct-genes}HSCT genes weight in model 0. Genes (with the ENSEMBL ID) and their weight on the first dimension of model 0 in HSCT dataset.}\\
\toprule
Gens & Weight\\
\midrule
ENSG00000266472.5 & 0.0456035\\
ENSG00000130332.14 & 0.0440667\\
ENSG00000137413.15 & -0.0433810\\
ENSG00000143256.4 & 0.0433159\\
ENSG00000165502.6 & 0.0424198\\
\addlinespace
ENSG00000129968.15 & 0.0423272\\
ENSG00000049239.12 & -0.0419903\\
ENSG00000114480.12 & -0.0419641\\
ENSG00000129559.12 & 0.0418544\\
ENSG00000160813.6 & 0.0416377\\
\addlinespace
ENSG00000167565.12 & 0.0415899\\
ENSG00000278615.4 & 0.0412584\\
ENSG00000182117.5 & 0.0409828\\
ENSG00000125743.10 & 0.0409573\\
ENSG00000135108.14 & -0.0407248\\
\addlinespace
ENSG00000198937.8 & 0.0402243\\
ENSG00000108384.14 & 0.0400644\\
ENSG00000254505.9 & 0.0398580\\
ENSG00000184990.12 & 0.0397149\\
ENSG00000123144.10 & 0.0396117\\
\addlinespace
ENSG00000172428.10 & 0.0395770\\
ENSG00000163257.10 & -0.0394262\\
ENSG00000127445.13 & 0.0390458\\
ENSG00000232112.3 & 0.0390367\\
ENSG00000131381.12 & -0.0390211\\
\addlinespace
ENSG00000273730.1 & 0.0390062\\
ENSG00000274917.1 & 0.0390062\\
ENSG00000275215.1 & 0.0390062\\
ENSG00000275757.1 & 0.0390062\\
ENSG00000276700.1 & 0.0390062\\
\bottomrule
\end{longtable}

\begin{longtable}[t]{cc}
\caption[HSCT genes weight in model 1.2]{\label{tab:hsct-genes-1-2}HSCT genes weight in model 1.2. Genes (with the ENSEMBL ID) and their weight on the first dimension of model 1.2 in HSCT dataset.}\\
\toprule
Gens & Weight\\
\midrule
ENSG00000178084.1 & -0.0649773\\
ENSG00000197444.9 & -0.0610161\\
ENSG00000181577.15 & 0.0573638\\
ENSG00000186038.9 & -0.0557866\\
ENSG00000136689.18 & 0.0537755\\
\addlinespace
ENSG00000127588.4 & -0.0535801\\
ENSG00000187714.6 & -0.0528026\\
ENSG00000197119.12 & 0.0524631\\
ENSG00000100196.10 & 0.0518669\\
ENSG00000137709.9 & -0.0515734\\
\addlinespace
ENSG00000161647.18 & 0.0511345\\
ENSG00000163874.10 & 0.0498909\\
ENSG00000103647.12 & -0.0487382\\
ENSG00000105732.12 & -0.0484703\\
ENSG00000143819.12 & -0.0477608\\
\addlinespace
ENSG00000153561.12 & -0.0475078\\
ENSG00000013392.7 & 0.0472729\\
ENSG00000186462.8 & -0.0467112\\
ENSG00000258504.2 & 0.0465228\\
ENSG00000112902.11 & -0.0454357\\
\addlinespace
ENSG00000084710.13 & -0.0450441\\
ENSG00000110619.17 & 0.0448852\\
ENSG00000154122.12 & -0.0447928\\
ENSG00000169429.10 & 0.0447690\\
ENSG00000013375.15 & 0.0445391\\
\addlinespace
ENSG00000140274.13 & 0.0439783\\
ENSG00000170381.12 & -0.0439493\\
ENSG00000122565.18 & 0.0435753\\
ENSG00000133962.7 & 0.0435285\\
ENSG00000198346.10 & -0.0434817\\
\bottomrule
\end{longtable}

\begin{longtable}[t]{cc}
\caption[HSCT genes weight in model 2.2]{\label{tab:hsct-genes-2-2}HSCT genes weight in model 2.2. Genes (with the ENSEMBL ID) and their weight on the first dimension of model 2.2 in HSCT dataset.}\\
\toprule
Gens & Weight\\
\midrule
ENSG00000256162.2 & 0.0390586\\
ENSG00000156222.11 & 0.0389785\\
ENSG00000163581.13 & 0.0389398\\
ENSG00000204978.2 & 0.0388415\\
ENSG00000278505.4 & 0.0386795\\
\addlinespace
ENSG00000273777.4 & 0.0386249\\
ENSG00000114771.13 & 0.0385101\\
ENSG00000115255.10 & 0.0384254\\
ENSG00000165443.11 & 0.0383690\\
ENSG00000021488.12 & 0.0383220\\
\addlinespace
ENSG00000007306.14 & -0.0381505\\
ENSG00000138823.13 & 0.0379713\\
ENSG00000136305.11 & 0.0379486\\
ENSG00000131142.13 & 0.0378967\\
ENSG00000216588.8 & 0.0378897\\
\addlinespace
ENSG00000120057.4 & 0.0378796\\
ENSG00000167588.12 & 0.0378474\\
ENSG00000115474.6 & 0.0377957\\
ENSG00000261012.2 & 0.0376443\\
ENSG00000110245.11 & 0.0375760\\
\addlinespace
ENSG00000065154.11 & 0.0375487\\
ENSG00000213918.10 & 0.0374909\\
ENSG00000186862.17 & 0.0374837\\
ENSG00000023839.10 & 0.0374759\\
ENSG00000229005.2 & 0.0374313\\
\addlinespace
ENSG00000149476.15 & 0.0374248\\
ENSG00000146426.17 & 0.0373228\\
ENSG00000021826.14 & 0.0372650\\
ENSG00000204740.10 & 0.0371954\\
ENSG00000114113.6 & 0.0370250\\
\bottomrule
\end{longtable}

\hypertarget{microbiome-2}{%
\subsection{Microbiome}\label{microbiome-2}}

\begin{longtable}[t]{cc}
\caption[HSCT microbiome weight in model 0]{\label{tab:hsct-microbiome}HSCT microbiome weight in model 0. OTUs and their weight on the first dimension of model 0 in HSCT dataset.}\\
\toprule
Bacterias ID & Weight\\
\midrule
OTU\_79 & 0.2881383\\
OTU\_2907 & 0.2785132\\
OTU\_102 & -0.2784619\\
OTU\_827 & 0.2194316\\
OTU\_3161 & 0.2022577\\
\addlinespace
OTU\_9 & 0.1924814\\
OTU\_1895 & -0.1874757\\
OTU\_634 & 0.1803819\\
OTU\_60 & -0.1749227\\
OTU\_16 & -0.1703163\\
\addlinespace
OTU\_83 & -0.1664484\\
OTU\_242 & 0.1654999\\
OTU\_13 & -0.1619381\\
OTU\_346 & 0.1491495\\
OTU\_3682 & 0.1472728\\
\addlinespace
OTU\_2861 & 0.1467828\\
OTU\_2198 & -0.1348251\\
OTU\_1766 & 0.1302680\\
OTU\_43 & 0.1301313\\
OTU\_3791 & 0.1294442\\
\addlinespace
OTU\_357 & 0.1290163\\
OTU\_2087 & 0.1247316\\
OTU\_38 & 0.1225272\\
OTU\_2619 & 0.1216535\\
OTU\_1613 & 0.1198396\\
\addlinespace
OTU\_3462 & 0.1193395\\
OTU\_142 & 0.1167569\\
OTU\_216 & -0.1161351\\
OTU\_2 & 0.1134775\\
OTU\_327 & 0.1094547\\
\bottomrule
\end{longtable}

\begin{longtable}[t]{cc}
\caption[HSCT microbiome weight in model 1.2]{\label{tab:hsct-microbiome-1-2}HSCT microbiome weight in model 1.2. OTUs and their weight on the first dimension of model 1.2 in HSCT dataset.}\\
\toprule
Bacterias ID & Weight\\
\midrule
OTU\_53 & -0.2926358\\
OTU\_83 & -0.2712193\\
OTU\_102 & -0.2663068\\
OTU\_708 & -0.2407498\\
OTU\_1325 & -0.2151213\\
\addlinespace
OTU\_915 & -0.2130421\\
OTU\_124 & -0.2046890\\
OTU\_2218 & -0.1970560\\
OTU\_189 & -0.1943113\\
OTU\_112 & -0.1843460\\
\addlinespace
OTU\_20 & 0.1800015\\
OTU\_161 & -0.1789666\\
OTU\_104 & -0.1754370\\
OTU\_86 & -0.1541973\\
OTU\_26 & -0.1517154\\
\addlinespace
OTU\_226 & -0.1450273\\
OTU\_116 & -0.1427346\\
OTU\_9 & 0.1395169\\
OTU\_228 & -0.1287262\\
OTU\_151 & -0.1259956\\
\addlinespace
OTU\_195 & -0.1251407\\
OTU\_4762 & -0.1191339\\
OTU\_629 & -0.1151510\\
OTU\_15 & -0.1117814\\
OTU\_8 & -0.1074008\\
\addlinespace
OTU\_203 & -0.1031445\\
OTU\_1084 & -0.0900318\\
OTU\_60 & -0.0888786\\
OTU\_4376 & 0.0876770\\
OTU\_3823 & -0.0875398\\
\bottomrule
\end{longtable}

\begin{longtable}[t]{cc}
\caption[HSCT microbiome weight in model 2.2]{\label{tab:hsct-microbiome-2-2}HSCT microbiome weight in model 2.2. OTUs and their weight on the first dimension of model 2.2 in HSCT dataset.}\\
\toprule
Bacterias ID & Weight\\
\midrule
OTU\_102 & 0.3652173\\
OTU\_83 & 0.2877509\\
OTU\_20 & -0.2767424\\
OTU\_53 & 0.2204974\\
OTU\_242 & -0.2017950\\
\addlinespace
OTU\_216 & 0.1882468\\
OTU\_41 & 0.1867378\\
OTU\_116 & 0.1809677\\
OTU\_452 & 0.1785050\\
OTU\_9 & -0.1746704\\
\addlinespace
OTU\_327 & -0.1635900\\
OTU\_5203 & 0.1433555\\
OTU\_241 & -0.1404794\\
OTU\_169 & 0.1403548\\
OTU\_1766 & -0.1360090\\
\addlinespace
OTU\_157 & -0.1343136\\
OTU\_3462 & -0.1284234\\
OTU\_13 & 0.1269981\\
OTU\_16 & 0.1269795\\
OTU\_122 & 0.1242579\\
\addlinespace
OTU\_1895 & 0.1230724\\
OTU\_300 & 0.1211617\\
OTU\_1084 & 0.1166934\\
OTU\_1118 & 0.1165312\\
OTU\_4376 & -0.1140843\\
\addlinespace
OTU\_629 & 0.1124190\\
OTU\_26 & 0.1087521\\
OTU\_500 & 0.1082343\\
OTU\_33 & 0.1046028\\
OTU\_4024 & 0.1046011\\
\bottomrule
\end{longtable}

\hypertarget{huxe4sler}{%
\section{Häsler}\label{huxe4sler}}

\hypertarget{genes-1}{%
\subsection{Genes}\label{genes-1}}

\begin{longtable}[t]{cc}
\caption[Häsler genes weight in model 0]{\label{tab:hasler-genes}Häsler genes weight in model 0. Genes and their weight on the first dimension of model 0 in Häsler dataset.}\\
\toprule
Gens & Weight\\
\midrule
ENSG00000166432.10 & -0.0256950\\
ENSG00000089775.7 & -0.0255069\\
ENSG00000256594.3 & -0.0249707\\
ENSG00000137364.4 & 0.0249337\\
ENSG00000188643.6 & 0.0247278\\
\addlinespace
ENSG00000257027.1 & -0.0245974\\
ENSG00000005471.11 & -0.0245969\\
ENSG00000128881.12 & -0.0244917\\
ENSG00000163492.9 & -0.0244566\\
ENSG00000180626.9 & -0.0244226\\
\addlinespace
ENSG00000172349.12 & -0.0242140\\
ENSG00000164048.9 & -0.0242108\\
ENSG00000173258.8 & -0.0241094\\
ENSG00000254285.2 & 0.0240426\\
ENSG00000188848.11 & -0.0239381\\
\addlinespace
ENSG00000135426.10 & -0.0238463\\
ENSG00000235532.1 & -0.0237835\\
ENSG00000100100.8 & -0.0237585\\
ENSG00000112365.4 & -0.0237069\\
ENSG00000044115.16 & 0.0236926\\
\addlinespace
ENSG00000108829.9 & 0.0236828\\
ENSG00000134744.9 & -0.0235820\\
ENSG00000112182.10 & -0.0235139\\
ENSG00000155657.19 & -0.0234856\\
ENSG00000116497.13 & -0.0234690\\
\addlinespace
ENSG00000113384.9 & 0.0234360\\
ENSG00000170421.7 & 0.0234250\\
ENSG00000108799.8 & -0.0234161\\
ENSG00000117308.10 & 0.0233947\\
ENSG00000171681.8 & -0.0232961\\
\bottomrule
\end{longtable}

\begin{longtable}[t]{cc}
\caption[Häsler genes weight in model 1.2]{\label{tab:hasler-genes-1-2}Häsler genes weight in model 1.2. Genes and their weight on the first dimension of model 1.2 in Häsler dataset.}\\
\toprule
Gens & Weight\\
\midrule
ENSG00000168060.10 & -0.0231742\\
ENSG00000162989.3 & -0.0230893\\
ENSG00000108576.5 & -0.0230390\\
ENSG00000149582.11 & -0.0230267\\
ENSG00000172689.1 & -0.0229890\\
\addlinespace
ENSG00000106404.9 & -0.0229805\\
ENSG00000120057.4 & -0.0229580\\
ENSG00000170482.12 & -0.0229343\\
ENSG00000176928.4 & -0.0229184\\
ENSG00000110693.11 & -0.0228633\\
\addlinespace
ENSG00000110080.14 & 0.0228500\\
ENSG00000148942.10 & -0.0228023\\
ENSG00000143627.13 & -0.0227952\\
ENSG00000149476.10 & -0.0227507\\
ENSG00000249948.2 & -0.0227502\\
\addlinespace
ENSG00000163032.7 & 0.0227023\\
ENSG00000156920.6 & -0.0227009\\
ENSG00000158865.8 & -0.0226897\\
ENSG00000256162.2 & -0.0226878\\
ENSG00000105398.3 & -0.0226290\\
\addlinespace
ENSG00000213918.6 & -0.0226275\\
ENSG00000173406.11 & -0.0225849\\
ENSG00000135929.4 & -0.0225431\\
ENSG00000146426.13 & -0.0225222\\
ENSG00000261012.2 & -0.0225182\\
\addlinespace
ENSG00000229005.2 & -0.0225148\\
ENSG00000163803.8 & -0.0225005\\
ENSG00000178573.6 & -0.0224647\\
ENSG00000109193.6 & -0.0224641\\
ENSG00000134042.8 & -0.0224304\\
\bottomrule
\end{longtable}

\begin{longtable}[t]{cc}
\caption[Häsler genes weight in model 2.2]{\label{tab:hasler-genes-2-2}Häsler genes weight in model 2.2. Genes and their weight on the first dimension of model 2.2 in Häsler dataset.}\\
\toprule
Gens & Weight\\
\midrule
ENSG00000137634.5 & 0.0238500\\
ENSG00000130700.6 & -0.0237268\\
ENSG00000204740.5 & -0.0235952\\
ENSG00000174885.8 & -0.0235645\\
ENSG00000007306.10 & 0.0235292\\
\addlinespace
ENSG00000154252.11 & 0.0233875\\
ENSG00000167230.6 & -0.0233772\\
ENSG00000110245.7 & -0.0233415\\
ENSG00000172318.4 & 0.0233192\\
ENSG00000176395.8 & -0.0233119\\
\addlinespace
ENSG00000204978.2 & -0.0232979\\
ENSG00000165443.7 & -0.0232848\\
ENSG00000104267.5 & 0.0232729\\
ENSG00000243766.3 & 0.0232634\\
ENSG00000019102.7 & 0.0232361\\
\addlinespace
ENSG00000154646.4 & -0.0231627\\
ENSG00000167588.8 & -0.0231575\\
ENSG00000115474.6 & -0.0231492\\
ENSG00000163581.9 & -0.0231436\\
ENSG00000138792.5 & -0.0231323\\
\addlinespace
ENSG00000138823.8 & -0.0230876\\
ENSG00000021488.8 & -0.0230651\\
ENSG00000156222.7 & -0.0230557\\
ENSG00000104450.8 & 0.0230393\\
ENSG00000110244.5 & -0.0230369\\
\addlinespace
ENSG00000159184.7 & 0.0229899\\
ENSG00000256162.2 & -0.0229870\\
ENSG00000106031.6 & 0.0229779\\
ENSG00000109193.6 & -0.0229747\\
ENSG00000114771.9 & -0.0229468\\
\bottomrule
\end{longtable}

\hypertarget{microbiome-3}{%
\subsection{Microbiome}\label{microbiome-3}}

\begin{longtable}[t]{cc}
\caption[Häsler microbiome weight in model 0]{\label{tab:hasler-microbiome}Häsler microbiome weight in model 0. ASVs and their weight on the first dimension of model 0 in Häsler dataset.}\\
\toprule
Bacterias ID & Weight\\
\midrule
Otu00100 & 0.0672144\\
Otu00856 & 0.0649676\\
Otu00322 & 0.0644017\\
Otu00396 & 0.0633326\\
Otu00038 & 0.0607676\\
\addlinespace
Otu00607 & 0.0583497\\
Otu00839 & 0.0575094\\
Otu00021 & 0.0570872\\
Otu01072 & 0.0562603\\
Otu00013 & 0.0562207\\
\addlinespace
Otu00394 & 0.0560274\\
Otu01275 & 0.0559946\\
Otu01327 & 0.0559946\\
Otu01356 & 0.0559946\\
Otu01371 & 0.0559946\\
\addlinespace
Otu01413 & 0.0559946\\
Otu01534 & 0.0559946\\
Otu01551 & 0.0559946\\
Otu01612 & 0.0559946\\
Otu01887 & 0.0559946\\
\addlinespace
Otu01911 & 0.0559946\\
Otu01966 & 0.0559946\\
Otu02014 & 0.0559946\\
Otu02027 & 0.0559946\\
Otu02143 & 0.0559946\\
\addlinespace
Otu02155 & 0.0559946\\
Otu02255 & 0.0559946\\
Otu02345 & 0.0559946\\
Otu02363 & 0.0559946\\
Otu02463 & 0.0559946\\
\bottomrule
\end{longtable}

\begin{longtable}[t]{cc}
\caption[Häsler microbiome weight in model 1.2]{\label{tab:hasler-microbiome-1-2}Häsler microbiome weight in model 1.2. ASVs and their weight on the first dimension of model 1.2 in Häsler dataset.}\\
\toprule
Bacterias ID & Weight\\
\midrule
Otu00961 & 0.0564135\\
Otu00353 & 0.0541289\\
Otu01113 & 0.0483349\\
Otu00228 & 0.0475690\\
Otu00914 & 0.0471097\\
\addlinespace
Otu00764 & 0.0461756\\
Otu00275 & 0.0459816\\
Otu00048 & 0.0451933\\
Otu00003 & 0.0429347\\
Otu00509 & -0.0424117\\
\addlinespace
Otu00915 & 0.0418800\\
Otu00450 & 0.0415114\\
Otu00773 & 0.0415114\\
Otu00984 & 0.0415114\\
Otu01049 & 0.0415114\\
\addlinespace
Otu01076 & 0.0415114\\
Otu01127 & 0.0415114\\
Otu01187 & 0.0415114\\
Otu01197 & 0.0415114\\
Otu01239 & 0.0415114\\
\addlinespace
Otu01278 & 0.0415114\\
Otu01322 & 0.0415114\\
Otu01369 & 0.0415114\\
Otu01511 & 0.0415114\\
Otu01519 & 0.0415114\\
\addlinespace
Otu01535 & 0.0415114\\
Otu01549 & 0.0415114\\
Otu01550 & 0.0415114\\
Otu01601 & 0.0415114\\
Otu01679 & 0.0415114\\
\bottomrule
\end{longtable}

\begin{longtable}[t]{cc}
\caption[Häsler microbiome weight in model 2.2]{\label{tab:hasler-microbiome-2-2}Häsler microbiome weight in model 2.2. ASVs and their weight on the first dimension of model 2.2 in Häsler dataset.}\\
\toprule
Bacterias ID & Weight\\
\midrule
Otu00158 & 0.0686844\\
Otu00097 & 0.0674805\\
Otu00228 & 0.0673602\\
Otu00193 & 0.0673237\\
Otu00351 & 0.0669005\\
\addlinespace
Otu00261 & 0.0658888\\
Otu00230 & 0.0655882\\
Otu01058 & 0.0631144\\
Otu00529 & 0.0613007\\
Otu00399 & 0.0593297\\
\addlinespace
Otu00392 & -0.0585349\\
Otu00069 & -0.0562498\\
Otu00653 & 0.0557877\\
Otu00675 & 0.0529940\\
Otu00003 & 0.0527516\\
\addlinespace
Otu00187 & -0.0523768\\
Otu00111 & -0.0504157\\
Otu00050 & 0.0503847\\
Otu00504 & -0.0498398\\
Otu00806 & 0.0495751\\
\addlinespace
Otu00220 & 0.0495472\\
Otu00969 & 0.0493767\\
Otu00052 & -0.0487349\\
Otu00350 & 0.0485622\\
Otu00353 & 0.0483550\\
\addlinespace
Otu00120 & 0.0477017\\
Otu00411 & 0.0469352\\
Otu00548 & 0.0469334\\
Otu00580 & 0.0469319\\
Otu00651 & 0.0469220\\
\bottomrule
\end{longtable}

\hypertarget{morgan}{%
\section{Morgan}\label{morgan}}

\hypertarget{genes-2}{%
\subsection{Genes}\label{genes-2}}

\begin{longtable}[t]{cc}
\caption[Morgan genes weight in model 0]{\label{tab:morgan-genes}Morgan genes weight in model 0. Genes and their weight on the first dimension of model 0 in Morgan dataset.}\\
\toprule
Gens & Weight\\
\midrule
51095 & 0.1737967\\
51306 & 0.1591623\\
55023 & 0.1508991\\
23350 & 0.1327847\\
23517 & 0.1327092\\
\addlinespace
6093 & 0.1295198\\
55035 & 0.1280594\\
23358 & 0.1229318\\
1385 & 0.1187076\\
23451 & 0.1177209\\
\addlinespace
6873 & 0.1173943\\
84146 & 0.1172078\\
5922 & 0.1162091\\
9852 & 0.1152178\\
7175 & 0.1149911\\
\addlinespace
55578 & 0.1138315\\
100130428 & 0.1126843\\
26098 & 0.1123347\\
23047 & 0.1118195\\
9044 & 0.1107288\\
\addlinespace
167153 & 0.1107056\\
10260 & 0.1097313\\
253260 & 0.1085950\\
64895 & 0.1074803\\
286410 & 0.1073855\\
\addlinespace
27332 & 0.1072184\\
905 & 0.1024215\\
8880 & 0.1003297\\
4659 & 0.0991866\\
7155 & 0.0989912\\
\bottomrule
\end{longtable}

\begin{longtable}[t]{cc}
\caption[Morgan genes weight in model 1.2]{\label{tab:morgan-genes-1-2}Morgan genes weight in model 1.2. Genes and their weight on the first dimension of model 1.2 in Morgan dataset.}\\
\toprule
Gens & Weight\\
\midrule
9457 & 0.1921094\\
6622 & 0.1796822\\
28513 & 0.1759460\\
11326 & 0.1598010\\
6598 & -0.1566792\\
\addlinespace
387775 & 0.1521147\\
3202 & -0.1512539\\
3201 & -0.1488354\\
4660 & 0.1292649\\
6752 & -0.1282156\\
\addlinespace
9475 & 0.1237949\\
55734 & -0.1196241\\
71 & -0.1183231\\
5464 & 0.1182357\\
383 & 0.1175214\\
\addlinespace
55510 & 0.1164200\\
1381 & -0.1157208\\
5803 & 0.1153165\\
6603 & -0.1129680\\
171019 & 0.1113954\\
\addlinespace
406995 & 0.1103490\\
284232 & -0.1073072\\
220002 & -0.1072004\\
60314 & -0.1057022\\
91355 & 0.1055908\\
\addlinespace
6689 & -0.1017615\\
91647 & -0.1008438\\
6317 & 0.0962729\\
4360 & 0.0960040\\
63935 & -0.0958012\\
\bottomrule
\end{longtable}

\begin{longtable}[t]{cc}
\caption[Morgan genes weight in model 2.2]{\label{tab:morgan-genes-2-2}Morgan genes weight in model 2.2. Genes and their weight on the first dimension of model 2.2 in Morgan dataset.}\\
\toprule
Gens & Weight\\
\midrule
6532 & 0.1664379\\
4129 & 0.1473841\\
9154 & 0.1470430\\
9429 & 0.1409609\\
83661 & 0.1349351\\
\addlinespace
92840 & 0.1327726\\
10004 & 0.1320366\\
8029 & 0.1318087\\
4094 & 0.1303898\\
159963 & 0.1282590\\
\addlinespace
7512 & 0.1238329\\
9963 & 0.1212076\\
151295 & 0.1204538\\
115584 & 0.1187887\\
283375 & 0.1180023\\
\addlinespace
151056 & 0.1179486\\
8556 & 0.1167189\\
1800 & 0.1162887\\
1600 & 0.1159077\\
2980 & 0.1110967\\
\addlinespace
51301 & 0.1106059\\
140803 & 0.1061265\\
3795 & 0.1055214\\
22949 & 0.1043400\\
4225 & 0.1016994\\
\addlinespace
4311 & 0.1015507\\
5243 & 0.1011880\\
79154 & 0.1003405\\
26230 & 0.0988540\\
84647 & 0.0983498\\
\bottomrule
\end{longtable}

\hypertarget{microbiome-4}{%
\subsection{Microbiome}\label{microbiome-4}}

\begin{longtable}[t]{cc}
\caption[Morgan microbiome weight in model 0]{\label{tab:morgan-microbiome}Morgan microbiome weight in model 0. OTUs and their weight on the first dimension of model 0 in Morgan dataset.}\\
\toprule
Bacterias ID & Weight\\
\midrule
2522 & -0.0800791\\
566 & -0.0685763\\
4902 & -0.0674629\\
2698 & -0.0674629\\
6119 & -0.0674629\\
\addlinespace
601 & -0.0674629\\
5202 & 0.0604631\\
1115 & -0.0594071\\
2749 & -0.0586973\\
1471 & -0.0573682\\
\addlinespace
3509 & -0.0568626\\
1212 & -0.0564636\\
5186 & -0.0550803\\
3450 & -0.0540439\\
5435 & -0.0533142\\
\addlinespace
6326 & -0.0530629\\
1053 & -0.0529661\\
2018 & -0.0521585\\
399 & -0.0520791\\
6272 & -0.0519789\\
\addlinespace
5603 & -0.0517056\\
6910 & 0.0516021\\
295 & -0.0513494\\
1229 & -0.0506591\\
303 & -0.0505856\\
\addlinespace
214 & -0.0502970\\
1945 & -0.0502189\\
325 & -0.0499548\\
1462 & -0.0497684\\
6285 & -0.0494503\\
\bottomrule
\end{longtable}

\begin{longtable}[t]{cc}
\caption[Morgan microbiome weight in model 1.2]{\label{tab:morgan-microbiome-1-2}Morgan microbiome weight in model 1.2. OTUs and their weight on the first dimension of model 1.2 in Morgan dataset.}\\
\toprule
Bacterias ID & Weight\\
\midrule
2715 & -0.0590339\\
4430 & -0.0560565\\
5254 & -0.0557235\\
6478 & -0.0548823\\
5635 & -0.0544966\\
\addlinespace
240 & -0.0525685\\
111 & -0.0517381\\
6 & -0.0516542\\
40 & -0.0513908\\
7 & -0.0513428\\
\addlinespace
3520 & -0.0507224\\
4656 & -0.0506837\\
26 & -0.0499941\\
51 & -0.0494708\\
20 & -0.0493355\\
\addlinespace
4433 & -0.0483064\\
4620 & -0.0474256\\
6126 & -0.0469061\\
1516 & -0.0467743\\
53 & -0.0465712\\
\addlinespace
45 & -0.0464921\\
6169 & -0.0452640\\
3364 & -0.0450982\\
5245 & -0.0448922\\
4538 & -0.0448875\\
\addlinespace
5194 & -0.0446537\\
4667 & -0.0446010\\
4998 & -0.0445878\\
95 & -0.0443084\\
4628 & -0.0439571\\
\bottomrule
\end{longtable}

\begin{longtable}[t]{cc}
\caption[Morgan microbiome weight in model 2.2]{\label{tab:morgan-microbiome-2-2}Morgan microbiome weight in model 2.2. OTUs and their weight on the first dimension of model 2.2 in Morgan dataset.}\\
\toprule
Bacterias ID & Weight\\
\midrule
6302 & 0.0838393\\
717 & 0.0720657\\
1838 & 0.0687877\\
2019 & 0.0677190\\
2158 & 0.0677190\\
\addlinespace
113 & 0.0674896\\
6477 & 0.0660561\\
1253 & 0.0656977\\
2741 & 0.0656977\\
2553 & 0.0648255\\
\addlinespace
306 & 0.0640717\\
2197 & 0.0637912\\
519 & 0.0615093\\
5772 & 0.0597852\\
2212 & 0.0597480\\
\addlinespace
5546 & 0.0595234\\
5202 & 0.0587761\\
3348 & 0.0579831\\
4668 & 0.0579831\\
5163 & 0.0575430\\
\addlinespace
4423 & 0.0568857\\
4578 & 0.0555136\\
6184 & 0.0546266\\
5207 & 0.0537541\\
4350 & 0.0536939\\
\addlinespace
2213 & 0.0529738\\
1710 & 0.0508599\\
1299 & 0.0506356\\
4509 & 0.0499265\\
2853 & 0.0497109\\
\bottomrule
\end{longtable}

\hypertarget{howell}{%
\section{Howell}\label{howell}}

\hypertarget{genes-3}{%
\subsection{Genes}\label{genes-3}}

\begin{longtable}[t]{cc}
\caption[Howell genes weight in model 0]{\label{tab:howell-genes}Howell genes weight in model 0. Genes and their weight on the first dimension of model 0 in Howell dataset.}\\
\toprule
Gens & Weight\\
\midrule
ENSG00000165629 & -0.0546586\\
ENSG00000125863 & -0.0542015\\
ENSG00000100347 & -0.0531023\\
ENSG00000184752 & -0.0529031\\
ENSG00000004779 & -0.0524148\\
\addlinespace
ENSG00000109919 & -0.0519989\\
ENSG00000143612 & -0.0517753\\
ENSG00000156411 & -0.0516415\\
ENSG00000147475 & -0.0513040\\
ENSG00000148943 & -0.0509856\\
\addlinespace
ENSG00000119541 & -0.0508802\\
ENSG00000116459 & -0.0507788\\
ENSG00000144840 & -0.0503280\\
ENSG00000108588 & -0.0502693\\
ENSG00000137210 & -0.0501921\\
\addlinespace
ENSG00000142864 & -0.0499362\\
ENSG00000140740 & -0.0498572\\
ENSG00000143155 & -0.0492047\\
ENSG00000154174 & -0.0491224\\
ENSG00000184983 & -0.0489342\\
\addlinespace
ENSG00000198015 & -0.0488989\\
ENSG00000166170 & -0.0488463\\
ENSG00000204370 & -0.0485859\\
ENSG00000164258 & -0.0484551\\
ENSG00000143727 & -0.0480934\\
\addlinespace
ENSG00000053770 & -0.0480545\\
ENSG00000143742 & -0.0479293\\
ENSG00000164022 & -0.0478868\\
ENSG00000138777 & -0.0478589\\
ENSG00000014824 & -0.0478026\\
\bottomrule
\end{longtable}

\begin{longtable}[t]{cc}
\caption[Howell genes weight in model 1.2]{\label{tab:howell-genes-1-2}Howell genes weight in model 1.2. Genes and their weight on the first dimension of model 1.2 in Howell dataset.}\\
\toprule
Gens & Weight\\
\midrule
ENSG00000164920 & 0.0523297\\
ENSG00000183742 & 0.0512658\\
ENSG00000130700 & 0.0511573\\
ENSG00000150527 & 0.0505312\\
ENSG00000143921 & 0.0502000\\
\addlinespace
ENSG00000108576 & 0.0494316\\
ENSG00000204740 & 0.0489483\\
ENSG00000110245 & 0.0486795\\
ENSG00000156222 & 0.0484512\\
ENSG00000133636 & 0.0483357\\
\addlinespace
ENSG00000167588 & 0.0482908\\
ENSG00000021488 & 0.0479126\\
ENSG00000012504 & 0.0478273\\
ENSG00000166268 & 0.0477829\\
ENSG00000138792 & 0.0477696\\
\addlinespace
ENSG00000015413 & 0.0477391\\
ENSG00000163900 & 0.0477181\\
ENSG00000146426 & 0.0475924\\
ENSG00000138030 & 0.0473961\\
ENSG00000134042 & 0.0473942\\
\addlinespace
ENSG00000158865 & 0.0473545\\
ENSG00000166959 & 0.0471633\\
ENSG00000114113 & 0.0470936\\
ENSG00000163581 & 0.0470072\\
ENSG00000135111 & 0.0469149\\
\addlinespace
ENSG00000138109 & 0.0469076\\
ENSG00000172689 & 0.0468983\\
ENSG00000169116 & -0.0468340\\
ENSG00000138823 & 0.0467165\\
ENSG00000144410 & 0.0466834\\
\bottomrule
\end{longtable}

\begin{longtable}[t]{cc}
\caption[Howell genes weight in model 2.2]{\label{tab:howell-genes-2-2}Howell genes weight in model 2.2. Genes and their weight on the first dimension of model 2.2 in Howell dataset.}\\
\toprule
Gens & Weight\\
\midrule
ENSG00000130700 & 0.0552195\\
ENSG00000164920 & 0.0538502\\
ENSG00000143921 & 0.0516188\\
ENSG00000204740 & 0.0515900\\
ENSG00000132517 & 0.0510265\\
\addlinespace
ENSG00000012504 & 0.0508416\\
ENSG00000106031 & -0.0507384\\
ENSG00000150527 & 0.0505005\\
ENSG00000183742 & 0.0494936\\
ENSG00000015413 & 0.0493307\\
\addlinespace
ENSG00000138792 & 0.0491054\\
ENSG00000174885 & 0.0491027\\
ENSG00000166268 & 0.0490889\\
ENSG00000133636 & 0.0490181\\
ENSG00000273777 & 0.0490106\\
\addlinespace
ENSG00000131142 & 0.0486012\\
ENSG00000136872 & 0.0484939\\
ENSG00000167588 & 0.0484363\\
ENSG00000110245 & 0.0484222\\
ENSG00000235122 & 0.0483907\\
\addlinespace
ENSG00000179603 & -0.0483898\\
ENSG00000021488 & 0.0483263\\
ENSG00000156222 & 0.0482785\\
ENSG00000109193 & 0.0482777\\
ENSG00000164946 & 0.0482170\\
\addlinespace
ENSG00000166959 & 0.0481486\\
ENSG00000138823 & 0.0480886\\
ENSG00000021826 & 0.0480123\\
ENSG00000108576 & 0.0477467\\
ENSG00000146426 & 0.0473801\\
\bottomrule
\end{longtable}

\hypertarget{microbiome-5}{%
\subsection{Microbiome}\label{microbiome-5}}

\begin{longtable}[t]{cc}
\caption[Howell microbiome weight in model 0]{\label{tab:howell-microbiome}Howell microbiome weight in model 0. OTUs and their weight on the first dimension of model 0 in Howell dataset.}\\
\toprule
Bacterias ID & Weight\\
\midrule
GGCGGCGCGCCTAACACATG... & 0.0803729\\
GGCGGCGTGCTTAACACATG... & 0.0800010\\
GGCGGCGCGCCTAACACATG... & 0.0740485\\
GGCGGCGTGCTTAACACATG... & 0.0723710\\
GGCGGCGTGCTTAACACATG... & 0.0720383\\
\addlinespace
GGCGGCGTGCTTAACACATG... & 0.0720173\\
AGCTACAGGCTTAACACATG... & 0.0683574\\
GGCGGCGTGCTTAACACATG... & 0.0680664\\
AGCGACAGGCTTAACACATG... & 0.0654374\\
GGCGGCGTGCTTAACACATG... & 0.0649646\\
\addlinespace
GGCGGCGTGCTTAACACATG... & 0.0647652\\
GGCGGCGCGCCTAACACATG... & 0.0644855\\
GGCGGCGTGCTTAACACATG... & 0.0639237\\
GGCGGCGCGCCTAACACATG... & 0.0637146\\
GGCGGCGTGCTTAACACATG... & 0.0636283\\
\addlinespace
GGCGGCGCGCCTAACACATG... & 0.0635899\\
GGCGGCGTGCTTAACACATG... & 0.0635262\\
GGCGGCGTGCTTAACACATG... & 0.0634932\\
GGCGGCGTGCTTAACACATG... & 0.0634499\\
GGCGGCGTGCTTAACACATG... & 0.0632328\\
\addlinespace
GGCGGCGCGCCTAACACATG... & 0.0632074\\
GGCGGCGTGCTTAACACATG... & 0.0631248\\
GGCGGCGTGCTTAACACATG... & 0.0630956\\
GGCGGCGTGCTTAACACATG... & 0.0627362\\
GGCGGCGCGCCTAACACATG... & 0.0626136\\
\addlinespace
GGCGGCGCGCCTAACACATG... & 0.0625425\\
GGCGGCGCGCCTAACACATG... & 0.0625093\\
GGCGGCGCGCCTAACACATG... & 0.0624945\\
GGCGGCGTGCTTAACACATG... & 0.0624716\\
GGCGGCGCGCCTAACACATG... & 0.0622848\\
\bottomrule
\end{longtable}

\begin{longtable}[t]{cc}
\caption[Howell microbiome weight in model 1.2]{\label{tab:howell-microbiome-1-2}Howell microbiome weight in model 1.2. OTUs and their weight on the first dimension of model 1.2 in Howell dataset.}\\
\toprule
Bacterias ID & Weight\\
\midrule
GGCGGCGTGCCTAACACATG... & 0.0757448\\
GGCGGCGTGCTTAACACATG... & 0.0694968\\
GGCGGCGTGCTTAACACATG... & 0.0641809\\
GGCGGCGTGCTTAACACATG... & -0.0617806\\
GGCGGCGCGCCTAACACATG... & -0.0600289\\
\addlinespace
GGCGGCGTGCTTAACACATG... & -0.0597203\\
GGCGGCGTGCTTAACACATG... & 0.0586173\\
GGCGGCGCGCCTAACACATG... & 0.0586173\\
GGCGGCGTGCTTAACACATG... & 0.0586173\\
GGCGGCGTGCCTAACACATG... & 0.0586173\\
\addlinespace
GGCGGCGCGCTTAACACATG... & 0.0586173\\
GGCGGCGCGCCTAACACATG... & 0.0586173\\
GGCGGCGTGCCTAACACATG... & 0.0586173\\
GGCGGCGCGCCTAACACATG... & 0.0586173\\
GGCGGCGCGCCTAACACATG... & 0.0586173\\
\addlinespace
GGCGGCGCGCCTAACACATG... & 0.0586173\\
GGCGGCGTGCTTAACACATG... & 0.0586173\\
GGCGGCATGCCTTACACATG... & -0.0580615\\
GGCGGCGTGCCTAATACATG... & 0.0576848\\
GGCGGCGTGCCTAATACATG... & 0.0556642\\
\addlinespace
GGCGGCGTGCTTAACACATG... & -0.0542767\\
GGCGGCGTGCTTAACACATG... & 0.0540257\\
GGCGGCGTGCTTAACACATG... & -0.0524921\\
GGCGGCGTGCTTAACACATG... & 0.0523110\\
GGCGGCGTGCCTAACACATG... & 0.0523110\\
\addlinespace
GGCGGCGTGCTTAACACATG... & 0.0523110\\
GGCGGCGTGCTTAACACATG... & 0.0523110\\
AGCTACAGGCTTAACACATG... & 0.0523110\\
GGCGGCGTGCTTAACACATG... & 0.0523110\\
GGCGGCGTGCTTAACACATG... & -0.0516570\\
\bottomrule
\end{longtable}

\begin{longtable}[t]{cc}
\caption[Howell microbiome weight in model 2.2]{\label{tab:howell-microbiome-2-2}Howell microbiome weight in model 2.2. OTUs and their weight on the first dimension of model 2.2 in Howell dataset.}\\
\toprule
Bacterias ID & Weight\\
\midrule
GGCGGCAGGCTTAACACATG... & -0.0720569\\
GGCGGCGTGCCTAACACATG... & -0.0704926\\
AGCTACAGGCTTAACACATG... & -0.0672298\\
GGCGGCGTGCCTAATACATG... & -0.0662263\\
GGCGGCGTGCCTAACACATG... & -0.0659790\\
\addlinespace
GGCGGCGTGCCTAATACATG... & -0.0653346\\
GGCGGCAGGCTTAACACATG... & -0.0653203\\
AGCTACAGGCTTAACACATG... & -0.0653147\\
GGCGGCGCGCCTAACACATG... & -0.0653129\\
GGCGGCGTGCCTAATACATG... & -0.0653075\\
\addlinespace
GGCGGCGCGCCTAACACATG... & -0.0653016\\
GGCGGCGCGCCTAACACATG... & -0.0652995\\
GGCGGCATGCTTTACACATG... & -0.0652981\\
GGCGGCGCGCCTAACACATG... & -0.0652913\\
AGCTACAGGCTTAACACATG... & -0.0652795\\
\addlinespace
AGCTACAGGCTTAACACATG... & -0.0652778\\
AGCGACAGGCTTAACACATG... & -0.0652764\\
GGCGGCGCGCCTAACACATG... & -0.0652585\\
GGCGGCGTGCCTAACACATG... & -0.0652556\\
GGCGGCGCGCCTAACACATG... & -0.0652494\\
\addlinespace
GGCGGAACGCTTTACACATG... & -0.0651797\\
GGCGGAACGCTTTACACATG... & -0.0651401\\
GGCGGCGTGCTTAACACATG... & -0.0651092\\
GGCGGCAGGCCTAACACATG... & -0.0650990\\
AGCTACAGGCTTAACACATG... & -0.0650823\\
\addlinespace
GGCGGCGTGCCTAACACATG... & -0.0650731\\
GGCGGCGCGCCTAACACATG... & -0.0650576\\
GGCGGCGCGCCTAACACATG... & -0.0649906\\
AGCGACAGGCTTAACACATG... & -0.0627209\\
GGCGGCAGGCCTAACACATG... & -0.0624542\\
\bottomrule
\end{longtable}

\end{document}
