---
title: "Data integration on inflammatory bowel disease"
author: "Lluís Revilla Sancho"
date: "`r Sys.Date()`"
documentclass: book
knit: "bookdown::render_book"
site: bookdown::bookdown_site
github-repo: "llrs/thesis"
url: 'https\://thesis.llrs.dev'
csl: bmc_bioinformatics.csl
book_filename: data_integration_IBD
split_by: chapter
twitter-handle: Lluis_Revilla
# Pandoc options
link-citations: true
colorlinks: yes
# https://bookdown.org/yihui/rmarkdown-cookbook/latex-variables.html
# Only works here (set when printing and sending)
links-as-notes: true
# fontfamily: boisik
papersize: a4
fontsize: 12pt
# Not needed https://tex.stackexchange.com/a/188337/178206
# fontenc: utf8 
geometry:
 - top=25.4mm
 - bottom=31.8mm
classoption: 
  - twoside
  - openright
lot: yes
lof: yes
bibliography:
- references.bib
- packages.bib
---

<!--# TODO add the first two pages with the formal requirements of the faculty  -->

```{r setup, echo=FALSE, include=FALSE}
Sys.setlocale('LC_ALL','C')
knitr::opts_chunk$set(
	echo = FALSE,
	out.width = "100%"
)
library("knitr")
```

<!--# TODO Fix metathis to provide good media coverage  -->

```{r metathis, cache=TRUE, include=FALSE, eval=knitr::is_html_output(excludes = "epub")}
library("metathis")
meta() %>%
  meta_description(
    "Thesis searching the relationship between microbiome and transcriptome on the intestine on patients with IBD to discover key genes and bacteria."
  ) %>% 
  meta_name("github-repo" = "llrs/thesis") %>% 
  meta_viewport() %>% 
  meta_social(
    title = "Data integration on inflammatory bowel disease",
    url = "https://thesis.llrs.dev",
    # image = "https://r4ds.had.co.nz/cover.png", # TODO
    image_alt = "The cover of the thesis.",
    og_type = "book",
    og_author = c("Lluís Revilla Sancho"),
    twitter_card_type = "summary_large_image",
    twitter_creator = "@Lluis_Revilla"
  )
```

```{r tables-formatting}
library("kableExtra")
options(knitr.kable.NA = "")
format_design <- function(x, data){
  dims <- dim(data)
  nrow <- nrow(data)
  ncol <- ncol(data) + as.numeric(!all(rownames(data) == seq_len(nrow(data))))
  k <- kable_styling(x, latex_options = c("striped", "HOLD_position", "repeat_header"), 
                            full_width = FALSE, fixed_thead = TRUE)
  k <- column_spec(k, column = 1, border_left = TRUE, bold = TRUE, include_thead = FALSE)
  k <- column_spec(k, column = ncol, border_right = TRUE, bold = FALSE, include_thead = FALSE)
  row_spec(k, row = 0, bold = TRUE, italic = TRUE)
}
format_kable <- function(x) {
  k <- format_design(x)
  column_spec(format_design(x), column = 1, border_right = TRUE, bold = TRUE)
}
```

# Preface {.unnumbered}

![UB logo](images/ub_logo.png "University where I defend the thesis"){width="160"} ![IDIBAPS logo](images/idibaps_logo.png "Research center"){width="160"} ![CIBER logo](images/logo_ciber.png "Hiring institution"){width="160" height="38"}

The main topic of the thesis is [data integration](https://en.wikipedia.org/wiki/Data_integration "Wikipedia page of integration") applied in the [inflammatory bowel disease (IBD)](https://en.wikipedia.org/wiki/Inflammatory_bowel_disease) research.
This disease is complex, for instance it is not known if the cause behind Crohn's disease and ulcerative colitis is the same.
There are hypothesis pointing that the [microbiome](https://en.wikipedia.org/wiki/Microbiota) is a major factor in the disease, which together with an aberrant immune response is the dominant theory.
In order to find robust relationships between the microbiome and the immune system it is important to consider all the relevant variables that influence a disease.
On this thesis I seek these relationships using data from different sequencing technologies and the observed or reported phenotype of the patients.

The thesis was carried out on the [Institut d'Investigacions Biomèdiques August Pi i Sunyer (IDIBAPS)](http://www.idibaps.org/ "IDIBAPS website") research institute and funded by [Centro de Investigación Biomédica en Red (CIBER)](https://www.ciberehd.org/ "CIBER website").
The thesis was done on [the IBD unit](https://www.ibd-bcn.org/ "Group's website") which is a translational team of biologist, microbiologists, veterinaries, bioinformaticians, doctors and nurses (at [Hospital Clínic](https://www.clinicbarcelona.org/en "Hospital Clínic")) in a multidisciplinary team.
The leading doctor of the unit was [Julian Panés](https://orcid.org/0000-0002-4971-6902 "Julià website") whose interest on the disease made possible this thesis.

The thesis is on the [doctoral programme in biomedicine](http://www.ub.edu/doctorat_biomedicina/eng/index.htm "Doctoral program's website") of the [University of Barcelona (UB)](https://www.ub.edu/web/portal/en/ "UB website").
My thesis directors' were [Juanjo Lozano](https://orcid.org/0000-0001-7613-3908 "Juanjo's ORCID") and [Azucena Salas](https://orcid.org/0000-0003-4572-2907 "Azu's ORCID"), my boss, who helped as bioinformatician and disease expert respectively.
They provided advice and guidance on how to analyse the data and where to focus on the different experiments/analysis.

This thesis, available on <https://thesis.llrs.dev> is licensed under the [Creative Commons Attribution 4.0 International License (CC-BY)](https://creativecommons.org/licenses/by/4.0/ "License summary link").\
!["CC-BY image"](images/by.png "CC-BY image of creative commons")

# Abstracts {.unnumbered}

<!--# Max length of 600 words-->

## English {.unnumbered}

**Introduction**: Inflammatory bowel disease is a complex disease with several factors that affect each other.
The origin and process that maintains the disease is not fully understood, and current treatments focus on early detection of the disease and prevent the accumulation of damage on the intestine.
There are some clues that microbiome is related to maintaining or starting this disease.
Finding relationships between the microbiome and the gastrointestinal mucosa will help advance treatments, diagnosis or prevent it.

**Methods**: We based our analysis on intestinal bacterial 16S data and human transcriptome from biopsies.
We used canonical correlation analysis to find models that are coherent with previous knowledge on the disease.
Multiple datasets of patients on different treatments and conditions were analysed.

**Results**: Our new method inteRmodel outperformed other existing methods to modelling the relationships and to classify the samples according to their location.
Despite being used on datasets of different sources the resulting models are very similar.

**Discussion**: Our method finds how strong are the relationships of the microbiome and the transcriptome taking into account other variables.
This approach is robust and flexible to different datasets.
Comparing multiple approaches to models helps elucidate the relationships of the data on the dataset.

**Conclusion**: With inteRmodel we found that the microbiome is slightly more related to the location than to the disease, but the transcriptome is highly related to the location of the sample on the intestine.

## Spanish {.unnumbered}

**Introducción**: Inflammatory bowel disease is a complex disease with several factors that affect each other.
The origin and process that maintains the disease is not fully understood, and current treatments focus on early detection of the disease and prevent the accumulation of damage on the intestine.
Finding those relationships will help to advance treatments, diagnosis or prevent it.

**Métodos**: To identify these relationships we based our analysis on intestinal bacterial 16S data and human transcriptome from biopsies.
We used canonical correlation analysis to find models that are coherent with previous knowledge on the biologiy of the disease and provide insights on the relationships between bacteria and genes.

**Resultados**: We compared our own methods with previously published methods, mainly MCIA, to assess the advantages and disadvantages of the different proposed relationships.
Furthermore, the classification of samples was used to find which methods provide are more accurate.

**Discussión**: Biological relevance, difficult.

**Conclusión**: Chips or further tests.

## Catalan {.unnumbered}

**Introducció**: Inflammatory bowel disease is a complex disease with several factors that affect each other.
The origin and process that maintains the disease is not fully understood, and current treatments focus on early detection of the disease and prevent the accumulation of damage on the intestine.
Finding those relationships will help to advance treatments, diagnosis or prevent it.

**Mètodos**: To identify these relationships we based our analysis on intestinal bacterial 16S data and human transcriptome from biopsies.
We used canonical correlation analysis to find models that are coherent with previous knowledge on the biologiy of the disease and provide insights on the relationships between bacteria and genes.

**Resultats**: We compared our own methods with previously published methods, mainly MCIA, to assess the advantages and disadvantages of the different proposed relationships.
Furthermore, the classification of samples was used to find which methods provide are more accurate.

**Discussió**: Biological relevance, difficult.

**Conclusió**: Chips or further tests.
