<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>1.1 Integration | Data integration on inflammatory bowel disease</title>
  <meta name="description" content="Thesis about the relationship between microbiome and transcriptome on the gastrointestinal tract on patients with inflammatory bowel disease and some controls to discover key differences on this relationship." />
  <meta name="generator" content="bookdown 0.21 and GitBook 2.6.7" />

  <meta property="og:title" content="1.1 Integration | Data integration on inflammatory bowel disease" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="Thesis about the relationship between microbiome and transcriptome on the gastrointestinal tract on patients with inflammatory bowel disease and some controls to discover key differences on this relationship." />
  <meta name="github-repo" content="llrs/thesis" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="1.1 Integration | Data integration on inflammatory bowel disease" />
  <meta name="twitter:site" content="@Lluis_Revilla" />
  <meta name="twitter:description" content="Thesis about the relationship between microbiome and transcriptome on the gastrointestinal tract on patients with inflammatory bowel disease and some controls to discover key differences on this relationship." />
  

<meta name="author" content="LluÃ­s Revilla Sancho" />


<meta name="date" content="2021-02-24" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="introduction.html"/>
<link rel="next" href="IBD.html"/>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />












<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><strong><a href="./">Data integration on IBD</a></strong></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Preface</a></li>
<li class="chapter" data-level="1" data-path="introduction.html"><a href="introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a><ul>
<li class="chapter" data-level="1.1" data-path="integration.html"><a href="integration.html"><i class="fa fa-check"></i><b>1.1</b> Integration</a><ul>
<li class="chapter" data-level="1.1.1" data-path="integration.html"><a href="integration.html#classification-of-integration-methods"><i class="fa fa-check"></i><b>1.1.1</b> Classification of integration method's</a></li>
<li class="chapter" data-level="1.1.2" data-path="integration.html"><a href="integration.html#reviews-and-benchmarking"><i class="fa fa-check"></i><b>1.1.2</b> Reviews and benchmarking</a></li>
<li class="chapter" data-level="1.1.3" data-path="integration.html"><a href="integration.html#summary"><i class="fa fa-check"></i><b>1.1.3</b> Summary</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="IBD.html"><a href="IBD.html"><i class="fa fa-check"></i><b>1.2</b> Inflammatory bowel disease</a><ul>
<li class="chapter" data-level="1.2.1" data-path="IBD.html"><a href="IBD.html#disease-etiology"><i class="fa fa-check"></i><b>1.2.1</b> Disease etiology</a></li>
<li class="chapter" data-level="1.2.2" data-path="IBD.html"><a href="IBD.html#crohns-disease"><i class="fa fa-check"></i><b>1.2.2</b> Crohn's disease</a></li>
<li class="chapter" data-level="1.2.3" data-path="IBD.html"><a href="IBD.html#ulcerative-colitis"><i class="fa fa-check"></i><b>1.2.3</b> Ulcerative colitis</a></li>
<li class="chapter" data-level="1.2.4" data-path="IBD.html"><a href="IBD.html#clinical-care"><i class="fa fa-check"></i><b>1.2.4</b> Clinical care</a></li>
<li class="chapter" data-level="1.2.5" data-path="IBD.html"><a href="IBD.html#summary-1"><i class="fa fa-check"></i><b>1.2.5</b> Summary</a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="integration-on-ibd.html"><a href="integration-on-ibd.html"><i class="fa fa-check"></i><b>1.3</b> Integration on IBD</a><ul>
<li class="chapter" data-level="1.3.1" data-path="integration-on-ibd.html"><a href="integration-on-ibd.html#genetics"><i class="fa fa-check"></i><b>1.3.1</b> Genetics</a></li>
<li class="chapter" data-level="1.3.2" data-path="integration-on-ibd.html"><a href="integration-on-ibd.html#microbiome"><i class="fa fa-check"></i><b>1.3.2</b> Microbiome</a></li>
<li class="chapter" data-level="1.3.3" data-path="integration-on-ibd.html"><a href="integration-on-ibd.html#multiple-data"><i class="fa fa-check"></i><b>1.3.3</b> Multiple data</a></li>
<li class="chapter" data-level="1.3.4" data-path="integration-on-ibd.html"><a href="integration-on-ibd.html#summary-2"><i class="fa fa-check"></i><b>1.3.4</b> Summary</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="2" data-path="multi-omic-modelling-of-inflammatory-bowel-disease-with-regularized-canonical-correlation-analysis.html"><a href="multi-omic-modelling-of-inflammatory-bowel-disease-with-regularized-canonical-correlation-analysis.html"><i class="fa fa-check"></i><b>2</b> Multi-omic modelling of inflammatory bowel disease with regularized canonical correlation analysis</a><ul>
<li class="chapter" data-level="2.1" data-path="relationships-models.html"><a href="relationships-models.html"><i class="fa fa-check"></i><b>2.1</b> Relationships' models</a><ul>
<li class="chapter" data-level="2.1.1" data-path="relationships-models.html"><a href="relationships-models.html#designing-models"><i class="fa fa-check"></i><b>2.1.1</b> Designing models</a></li>
<li class="chapter" data-level="2.1.2" data-path="relationships-models.html"><a href="relationships-models.html#evaluating-models"><i class="fa fa-check"></i><b>2.1.2</b> Evaluating models</a></li>
</ul></li>
<li class="chapter" data-level="2.2" data-path="conclusions.html"><a href="conclusions.html"><i class="fa fa-check"></i><b>2.2</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="biological-relevance-of-results.html"><a href="biological-relevance-of-results.html"><i class="fa fa-check"></i><b>3</b> Biological relevance of results</a><ul>
<li class="chapter" data-level="3.1" data-path="comparing-different-dataset.html"><a href="comparing-different-dataset.html"><i class="fa fa-check"></i><b>3.1</b> Comparing different dataset</a></li>
<li class="chapter" data-level="3.2" data-path="shared-selected-variables.html"><a href="shared-selected-variables.html"><i class="fa fa-check"></i><b>3.2</b> Shared selected variables</a></li>
<li class="chapter" data-level="3.3" data-path="conclusions-1.html"><a href="conclusions-1.html"><i class="fa fa-check"></i><b>3.3</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="appendix"><span><b>Appendix</b></span></li>
<li class="chapter" data-level="A" data-path="online-resources.html"><a href="online-resources.html"><i class="fa fa-check"></i><b>A</b> Online resources</a></li>
<li class="chapter" data-level="B" data-path="software.html"><a href="software.html"><i class="fa fa-check"></i><b>B</b> Software</a><ul>
<li class="chapter" data-level="B.1" data-path="listed.html"><a href="listed.html"><i class="fa fa-check"></i><b>B.1</b> Listed</a></li>
<li class="chapter" data-level="B.2" data-path="by-publication.html"><a href="by-publication.html"><i class="fa fa-check"></i><b>B.2</b> By publication</a></li>
</ul></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Data integration on inflammatory bowel disease</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div class = "wip">
This tesis is ongoing and still without hard date to finish it. Watch the <a href="https://github.com/llrs/thesis">GitHub repository</a> or wait for a release announcement on Twitter on <a href="https://twitter.com/Lluis_Rev">my profile</a>.
</div>
<div id="integration" class="section level2">
<h2><span class="header-section-number">1.1</span> Integration</h2>
<p>Integration is defined as:</p>
<blockquote>
<p>&quot;the process of combining two or more things into one&quot; --- <a href="https://dictionary.cambridge.org/dictionary/english/integration">Cambridge Dictionary</a></p>
</blockquote>
<p>Other words that are used integrati(-on, -ve), multi-omics, pluri-omics. However, here integration will be used as it is the more general one and not restricted to omics. Since the beginning of the integration methods there have been many methods proposed. Some of these methods are specific for one application or data while others are more general. On the recent years with the increase of more datasets with more variables than previously has seen an increase on the number of methodologies available on several disciplines but mainly on the biological science. This in turn has increased the importance of classification, review and comparison of the tools available, as well as, benchmarking these tools against the same dataset <span class="citation">[<a href="#ref-wu_selective_2019">1</a>]</span>. Other times only the strategies used are used to classify the methods <span class="citation">[<a href="#ref-cavill_transcriptomic_2016">2</a>, <a href="#ref-chongComputationalApproachesIntegrative2017">3</a>]</span>.</p>
<div id="classification-of-integration-methods" class="section level3">
<h3><span class="header-section-number">1.1.1</span> Classification of integration method's</h3>
<p>We can characterize and classify integration methods in several ways. Here I outline some classifications in the bioscience field, with relationships (and references) to concrete methodology and usages.</p>
<div id="data-type-numeric-vs-categorical" class="section level4">
<h4><span class="header-section-number">1.1.1.1</span> Data type: numeric vs categorical</h4>
<p>The most important distinction in integration methods is what kind of data are combined. In general data can be divided between categorical and numeric variables, which are usually found in several fields. Sometimes doctors want to understand the relationship between a phenotype they observe and the underlying mechanism. Usually this involves looking how the metabolites, the gene expression, the methylation, the number of variants a gene has, and other numeric variables are related to the observed (categorical) phenotype (like pain).</p>
<p>Depending on what is a method aimed for it handles both data types or just one, often they are used differently. The most common way to handle different type of data is converting the categorical values to a mock or dummy variable. For each categorical factor there is a new numerical value with 1 if that sample had this value or 0 otherwise. Sometimes Often the number of variables created is one less than the number of factors that existed. For instance, if the categorical variable has three values (A, B, C) it would be converted to A (1, 0, 0) B (0, 1, 0) and C (0, 0, 1) and usually one of these three variables is omitted. This transformation allows to use the categorical values in methods though for numeric variables.</p>
</div>
<div id="aim" class="section level4">
<h4><span class="header-section-number">1.1.1.2</span> Aim</h4>
<p>The results and methods of data integration depend on the objective of the method and on the (biological) question. Most of the times one (or all) of the following results are expected:</p>
<ul>
<li>An overview of the role of each 'omics in a biological system</li>
</ul>
<p>Sometimes several omics are used and the question is which omic method is the best one to describe the disease. Or several methods are used to understand better how two omics interact.</p>
<ul>
<li>A better understanding of the relationships between the 'omics types</li>
</ul>
<p>When the relationship between omics is known there are methods to find those relationships and improve upon previous knowledge. For instance, checking that in a particular case or condition there is a given relationship, and that this relationships follows our model or not.</p>
<ul>
<li>A molecular signature<a href="#fn1" class="footnoteRef" id="fnref1"><sup>1</sup></a> leading to more insight into molecular mechanisms</li>
</ul>
<p>The usage of multifactor analysis might lead to the need to redefine the features that are essential for identifying a phase or a cell line.</p>
<ul>
<li>A predictive model</li>
</ul>
<p>If a good model is known it can be used as a prediction if enough information is gathered, which might improve the treatment.</p>
</div>
<div id="relationship-between-variables-and-samples" class="section level4">
<h4><span class="header-section-number">1.1.1.3</span> Relationship between variables and samples</h4>
<p>Depending on the amount of variables and in which samples they have been measures we can classify in two types of integration. Traditionally for each sample few variable has been measured, for instance for a tree only the height and width are measured, however with the new omics techniques (transcriptomics, metabolomics, methylomics, genomics), thousands of variables are measured for the same sample.</p>
<ul>
<li>More variables than samples:</li>
</ul>
<p>For a single sample of RNA around 50k genome identifiers (genes, long non coding RNAs, iRNA, pseudogenes,...) can be measured. High-throughput data analysis typically falls into the category of <span class="math inline">\(p \gg n\)</span> problems, where the number of genes or proteins, <span class="math inline">\(p\)</span>, is considerably larger than the number of samples, <span class="math inline">\(n\)</span>. Which leads to the case where there are (many) more variables than samples, generally &quot;old&quot; statistics don't consider this case, as it has its own complications like co-variance between the variables. When two variables are tightly correlated, discerning which is the lead and which is following is near to impossible.</p>
<ul>
<li>More samples than variables:</li>
</ul>
<p>An example would be when from a cohort of patients the temperature is measured along the stage of a disease: two variables for each sample. If in the cohort there are more than 2 patients, then the number of samples is greater than the number of variables studied. This is described in the literature as <span class="math inline">\(n \gg p\)</span>.</p>
</div>
<div id="relationship-between-samples" class="section level4">
<h4><span class="header-section-number">1.1.1.4</span> Relationship between samples</h4>
<p>Depending on the relationship between the samples, the questions answerable and the methods available differ. If each sample has all the expected data we wanted to measure it is a complete case.</p>
<p>Sometimes because the sample is not enough, or there are some technical or organizational problems we might lose a source of data for a sample (which is known as an incomplete case). This results in a new source of variation that has to be dealt with, which complicates the conclusion one can draw from the studies of these kind of data.</p>
<p>Even when all the cases of a sample are complete the samples can come from several sites of the same individual or with different combinations of variables, which makes is relevant to understand.</p>
<div id="time" class="section level5 unnumbered">
<h5>Time</h5>
<p>Time is one of the factors that sometimes cannot be controlled, despite having programmed visits every two weeks some patients might come early or later due to calendar reasons (holidays), family reasons, or disease state. Sometimes is precisely the object of the study, to see the relationships at different time, or see how the relationships change with time. Simultaneously is very important to consider it because two variables can seem correlated if we don't take time into consideration. Also, to discover causality between two variables the cause must be before the consequence, which highlights the importance of time. Being aware of the time differences and time scales is crucial in most cases.</p>
</div>
</div>
<div id="relationship-between-variables" class="section level4">
<h4><span class="header-section-number">1.1.1.5</span> Relationship between variables</h4>
<p>Since the lactose <a href="https://en.wikipedia.org/wiki/Operon">operon</a> we know how some genes regulate each other. For other variables we don't know how they are related. For instance, how does the increase in expression of a gene affects the growth of a microorganism? Usually the relationships between variables are mediated by many factors or interactions.</p>
<p>Network approaches relate the variables between them (such as <span class="citation">[<a href="#ref-koh_iomicspass_2019">4</a>]</span>). And are fairly new and despite being used in rare occasions they are growing in popularity.</p>
<p>In partial correlations some or all of the other variables and considered on how much do they affect and deduced.</p>
</div>
<div id="input-data" class="section level4">
<h4><span class="header-section-number">1.1.1.6</span> Input data</h4>
<p>Methods can be classified by the kind of input data required. Some of them need data from the same patients on each data set used to integrate while other do not.</p>
<ul>
<li>Data from the same samples:</li>
</ul>
<p>These methods do not handle well or at all missing data. They need complete cases/data of the samples in order to be able to integrate the results. These methods include Regularized Generalized Canonical Correlation Analysis (RGCCA) <span class="citation">[<a href="#ref-tenenhaus_regularized_2011">5</a>, <a href="#ref-tenenhaus_variable_2014">6</a>]</span>, Multiple co-inertia analysis (MCIA) <span class="citation">[<a href="#ref-culhane_cross-platform_2003">7</a>]</span>, Multi-Study Factor Analysis (MSFA) <span class="citation">[<a href="#ref-vito_multi-study_2019">8</a>]</span>, Multi-Omics Factor Analysis (MOFA) <span class="citation">[<a href="#ref-argelaguet_multi-omics_2018">9</a>]</span>, STATegRa <span class="citation">[<a href="#ref-gomez-cabreroSTATegraComprehensiveMultiomics2019">10</a>]</span>.</p>
<ul>
<li>Data from different samples:</li>
</ul>
<p>These methods do not need data from the same sample. They draw their conclusions generalizing from the the data available. Some of them handle missing data, while others do use the data at face value. These method includes MetaPhlAn2, HUMAnN, LEfSe <span class="citation">[<a href="#ref-franzosaSpecieslevelFunctionalProfiling2018">11</a>â<a href="#ref-segataMetagenomicBiomarkerDiscovery2011">13</a>]</span>.</p>
<ul>
<li>Data types:</li>
</ul>
<p>HCG, 16S, RNA-seq, metabolomics do not share the same data distribution, and are different between them. Also even with the same data depending on the processing of the data they can have very different properties: OTUs (operational taxonomic unit) do not behave equally as ASV (amplicon sequence variants)</p>
</div>
<div id="mathematical-framework" class="section level4">
<h4><span class="header-section-number">1.1.1.7</span> Mathematical framework</h4>
<p>Depending on the input and the objective methods use different mathematical framework to process the data. The most important ones are listed below:</p>
<ul>
<li>Network</li>
</ul>
<p>Multilayer, including the multiplex, Molti-C-DREAM, RWR-MH, RWR-M. Network embedding MultiVERSE</p>
<ul>
<li>Dimensional Reduction Momix, RGCCA,</li>
<li>Active module identification Multiomic objective genetic algorithm (scores based in two metrics, node score and density of interactions score). MOGAMUN</li>
</ul>
</div>
<div id="output-results" class="section level4">
<h4><span class="header-section-number">1.1.1.8</span> Output results</h4>
<p>According to the output the integration methods can be classified in several groups: For those from dimensional reduction methods there are three: Shared factor across the data, specific factors for each data or mixed factors.</p>
<ul>
<li>Shared factors:</li>
</ul>
<p>The integration results in a vector of the samples in a lower dimensional space that is shared by all the data used to integrate. Such methods include iCluster, Multi-Omics Factor Analysis (MOFA) <span class="citation">[<a href="#ref-argelaguet_multi-omics_2018">9</a>]</span>.</p>
<ul>
<li>Specific factors:</li>
</ul>
<p>The integration results in several vectors of the samples in a lower dimensional space of each data used to integrate. Such methods include Regularized Generalized Canonical Correlation Analysis (RGCCA) <span class="citation">[<a href="#ref-tenenhaus_regularized_2011">5</a>, <a href="#ref-tenenhaus_variable_2014">6</a>]</span>, Multiple co-inertia analysis (MCIA) <span class="citation">[<a href="#ref-culhane_cross-platform_2003">7</a>]</span>, Multi-Study Factor Analysis (MSFA) <span class="citation">[<a href="#ref-vito_multi-study_2019">8</a>]</span>.</p>
<ul>
<li>Mixed factors:</li>
</ul>
<p>The integration results in both previous factors, specific of each data and common to all the data. Such methods include Joint and Individual Variation Explained (JIVE), integrative Non-negative Matrix Factorization (intNMF).</p>
</div>
<div id="interpretation" class="section level4">
<h4><span class="header-section-number">1.1.1.9</span> Interpretation</h4>
<p>Understanding how to interpret the results of the methods is highly tight to understanding the method. If one does a correlation between two variables, the interpretation of the analysis is clear, if one variable increase, the other one too. However as more complicated methodologies are developed the interpretation becomes less clear, for instance how can one interpret the result of a canonical correlation analysis?</p>
<ul>
<li>Individually:</li>
</ul>
<p>How each variable relates to another, like in the correlation analysis, the relationship between two variables under study. Or by patient: how do interpret that in these patient variable A and B is X and Y?</p>
<ul>
<li>Globally:</li>
</ul>
<p>In a PCA for instance how do we interpret that some variables have the same loading? What happens in a more difficult method like canonical correlation analysis?</p>
<p>There have been some articles about how to interpret those methods on real datasets <span class="citation">[<a href="#ref-sherryConductingInterpretingCanonical1981">14</a>]</span>. Others, to benchmark and to learn how to interpret propose analyzing a simulated dataset <span class="citation">[<a href="#ref-chung_multi-omics_2019">15</a>, <a href="#ref-martinez-mira_mosim_2018">16</a>]</span>.</p>
<p>To help interpretation frequently synthetic datasets are used to compare the results of the integration with the dataset of interest and to compare different tools. This datasets are created with some relationships that the tools are expected to find.</p>
<p>There exists several methods to create synthetic datasets like MOSim, metaSPARSim, CAMISIM, ballgown, polyester and even edgeR can be used.</p>
</div>
<div id="conclusion" class="section level4">
<h4><span class="header-section-number">1.1.1.10</span> Conclusion</h4>
<p>The field of integration is large and complex, with high interest in the recent days, specially in the psychology and omics field, which lacks of a large study.</p>
</div>
</div>
<div id="reviews-and-benchmarking" class="section level3">
<h3><span class="header-section-number">1.1.2</span> Reviews and benchmarking</h3>
<p>The comparison and review of methods independently from original authors have become a crucial step for selecting the right tool for a research <span class="citation">[<a href="#ref-cantini_benchmarking_2020">17</a>]</span>.</p>
<p>Some of these reviews are focused on a field: metabolomics (ref), genomics, microbiomics... or on a specific characteristic: ?.</p>
</div>
<div id="summary" class="section level3">
<h3><span class="header-section-number">1.1.3</span> Summary</h3>
<p>Methods to integrate have many characteristics that allow to classify them which explains the diverse results one can have using one of them.</p>
</div>
</div>
<h3>References</h3>
<div id="refs" class="references">
<div id="ref-wu_selective_2019">
<p>1. Wu C, Zhou F, Ren J, Li X, Jiang Y, Ma S. A Selective Review of Multi-Level Omics Data Integration Using Variable Selection. High-Throughput. 2019;8:4.</p>
</div>
<div id="ref-cavill_transcriptomic_2016">
<p>2. Cavill R, Jennen D, Kleinjans J, Briedâe JJ. Transcriptomic and metabolomic data integration. Briefings in Bioinformatics. 2016;17:891â901.</p>
</div>
<div id="ref-chongComputationalApproachesIntegrative2017">
<p>3. Chong J, Xia J. Computational approaches for integrative analysis of the metabolome and microbiome. Metabolites. 2017;7:62. doi:<a href="https://doi.org/10.3390/metabo7040062">10.3390/metabo7040062</a>.</p>
</div>
<div id="ref-koh_iomicspass_2019">
<p>4. Koh HWL, Fermin D, Vogel C, Choi KP, Ewing RM, Choi H. iOmicsPASS: Network-based integration of multiomics data for predictive subnetwork discovery. npj Systems Biology and Applications. 2019;5:1â10.</p>
</div>
<div id="ref-tenenhaus_regularized_2011">
<p>5. Tenenhaus A, Tenenhaus M. Regularized Generalized Canonical Correlation Analysis. Psychometrika. 2011;76:257â84.</p>
</div>
<div id="ref-tenenhaus_variable_2014">
<p>6. Tenenhaus A, Philippe C, Guillemot V, Le Cao K-A, Grill J, Frouin V. Variable selection for generalized canonical correlation analysis. Biostatistics. 2014;15:569â83.</p>
</div>
<div id="ref-culhane_cross-platform_2003">
<p>7. Culhane AC, Perriâere G, Higgins DG. Cross-platform comparison and visualisation of gene expression data using co-inertia analysis. BMC Bioinformatics. 2003;4:59.</p>
</div>
<div id="ref-vito_multi-study_2019">
<p>8. Vito RD, Bellio R, Trippa L, Parmigiani G. Multi-study factor analysis. Biometrics. 2019;75:337â46.</p>
</div>
<div id="ref-argelaguet_multi-omics_2018">
<p>9. Argelaguet R, Velten B, Arnol D, Dietrich S, Zenz T, Marioni JC, et al. Multi-Omics Factor Analysisa framework for unsupervised integration of multi-omics data sets. Molecular Systems Biology. 2018;14:e8124.</p>
</div>
<div id="ref-gomez-cabreroSTATegraComprehensiveMultiomics2019">
<p>10. Gomez-Cabrero D, Tarazona S, Ferreirâos-Vidal I, Ramirez RN, Company C, Schmidt A, et al. STATegra, a comprehensive multi-omics dataset of B-cell differentiation in mouse. Scientific Data. 2019;6:1â15.</p>
</div>
<div id="ref-franzosaSpecieslevelFunctionalProfiling2018">
<p>11. Franzosa EA, McIver LJ, Rahnavard G, Thompson LR, Schirmer M, Weingart G, et al. Species-level functional profiling of metagenomes and metatranscriptomes. Nature Methods. 2018;15:962. doi:<a href="https://doi.org/10.1038/s41592-018-0176-y">10.1038/s41592-018-0176-y</a>.</p>
</div>
<div id="ref-segataMetagenomicBiomarkerDiscovery2011">
<p>13. Segata N, Izard J, Waldron L, Gevers D, Miropolsky L, Garrett WS, et al. Metagenomic biomarker discovery and explanation. Genome Biology. 2011;12:R60. doi:<a href="https://doi.org/10.1186/gb-2011-12-6-r60">10.1186/gb-2011-12-6-r60</a>.</p>
</div>
<div id="ref-sherryConductingInterpretingCanonical1981">
<p>14. Sherry A, Henson RK. Conducting and Interpreting Canonical Correlation Analysis in Personality Research: A User-Friendly Primer. 1981.</p>
</div>
<div id="ref-chung_multi-omics_2019">
<p>15. Chung R-H, Kang C-Y. A multi-omics data simulator for complex disease studies and its application to evaluate multi-omics data analysis methods for disease classification. GigaScience. 2019;8.</p>
</div>
<div id="ref-martinez-mira_mosim_2018">
<p>16. MartÃ­nez-Mira C, Conesa A, Tarazona S. MOSim: Multi-Omics Simulation in R. Preprint. Bioinformatics; 2018.</p>
</div>
<div id="ref-cantini_benchmarking_2020">
<p>17. Cantini L, Zakeri P, Hernandez C, Naldi A, Thieffry D, Remy E, et al. Benchmarking joint multi-omics dimensionality reduction approaches for cancer study. bioRxiv. 2020;2020.01.14.905760.</p>
</div>
</div>
<div class="footnotes">
<hr />
<ol start="1">
<li id="fn1"><p>A signature is usually a group of features that describe/are representative of a cell line, a process or a stage.<a href="integration.html#fnref1">â©</a></p></li>
</ol>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="introduction.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="IBD.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": {
"href": "https://github.com/llrs/thesis/"
},
"facebook": false,
"twitter": {
"href": "https://twitter.com/intent/tweet?text=Nice%20read%20of%20%40Lluis_Rev%20thesis"
},
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/llrs/thesis/edit/master/01_introduction.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": "https://github.com/llrs/thesis/blob/master/01_introduction.Rmd",
"text": null
},
"download": ["Data integration on inflammatory bowel disease.pdf", "Data integration on inflammatory bowel disease.epub"],
"toc": {
"collapse": false,
"toc_appendix": true,
"scroll_highlight": true
},
"toc_float": true
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
