<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>1.3 Integration on IBD | Data integration on inflammatory bowel disease</title>
  <meta name="description" content="Thesis about the relationship between microbiome and transcriptome on the gastrointestinal tract on patients with inflammatory bowel disease and some controls to discover key differences on this relationship." />
  <meta name="generator" content="bookdown 0.21 and GitBook 2.6.7" />

  <meta property="og:title" content="1.3 Integration on IBD | Data integration on inflammatory bowel disease" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="Thesis about the relationship between microbiome and transcriptome on the gastrointestinal tract on patients with inflammatory bowel disease and some controls to discover key differences on this relationship." />
  <meta name="github-repo" content="llrs/thesis" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="1.3 Integration on IBD | Data integration on inflammatory bowel disease" />
  <meta name="twitter:site" content="@Lluis_Revilla" />
  <meta name="twitter:description" content="Thesis about the relationship between microbiome and transcriptome on the gastrointestinal tract on patients with inflammatory bowel disease and some controls to discover key differences on this relationship." />
  

<meta name="author" content="LluÃ­s Revilla Sancho" />


<meta name="date" content="2021-02-24" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="IBD.html"/>
<link rel="next" href="multi-omic-modelling-of-inflammatory-bowel-disease-with-regularized-canonical-correlation-analysis.html"/>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />












<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><strong><a href="./">Data integration on IBD</a></strong></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Preface</a></li>
<li class="chapter" data-level="1" data-path="introduction.html"><a href="introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a><ul>
<li class="chapter" data-level="1.1" data-path="integration.html"><a href="integration.html"><i class="fa fa-check"></i><b>1.1</b> Integration</a><ul>
<li class="chapter" data-level="1.1.1" data-path="integration.html"><a href="integration.html#classification-of-integration-methods"><i class="fa fa-check"></i><b>1.1.1</b> Classification of integration method's</a></li>
<li class="chapter" data-level="1.1.2" data-path="integration.html"><a href="integration.html#reviews-and-benchmarking"><i class="fa fa-check"></i><b>1.1.2</b> Reviews and benchmarking</a></li>
<li class="chapter" data-level="1.1.3" data-path="integration.html"><a href="integration.html#summary"><i class="fa fa-check"></i><b>1.1.3</b> Summary</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="IBD.html"><a href="IBD.html"><i class="fa fa-check"></i><b>1.2</b> Inflammatory bowel disease</a><ul>
<li class="chapter" data-level="1.2.1" data-path="IBD.html"><a href="IBD.html#disease-etiology"><i class="fa fa-check"></i><b>1.2.1</b> Disease etiology</a></li>
<li class="chapter" data-level="1.2.2" data-path="IBD.html"><a href="IBD.html#crohns-disease"><i class="fa fa-check"></i><b>1.2.2</b> Crohn's disease</a></li>
<li class="chapter" data-level="1.2.3" data-path="IBD.html"><a href="IBD.html#ulcerative-colitis"><i class="fa fa-check"></i><b>1.2.3</b> Ulcerative colitis</a></li>
<li class="chapter" data-level="1.2.4" data-path="IBD.html"><a href="IBD.html#clinical-care"><i class="fa fa-check"></i><b>1.2.4</b> Clinical care</a></li>
<li class="chapter" data-level="1.2.5" data-path="IBD.html"><a href="IBD.html#summary-1"><i class="fa fa-check"></i><b>1.2.5</b> Summary</a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="integration-on-ibd.html"><a href="integration-on-ibd.html"><i class="fa fa-check"></i><b>1.3</b> Integration on IBD</a><ul>
<li class="chapter" data-level="1.3.1" data-path="integration-on-ibd.html"><a href="integration-on-ibd.html#genetics"><i class="fa fa-check"></i><b>1.3.1</b> Genetics</a></li>
<li class="chapter" data-level="1.3.2" data-path="integration-on-ibd.html"><a href="integration-on-ibd.html#microbiome"><i class="fa fa-check"></i><b>1.3.2</b> Microbiome</a></li>
<li class="chapter" data-level="1.3.3" data-path="integration-on-ibd.html"><a href="integration-on-ibd.html#multiple-data"><i class="fa fa-check"></i><b>1.3.3</b> Multiple data</a></li>
<li class="chapter" data-level="1.3.4" data-path="integration-on-ibd.html"><a href="integration-on-ibd.html#summary-2"><i class="fa fa-check"></i><b>1.3.4</b> Summary</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="2" data-path="multi-omic-modelling-of-inflammatory-bowel-disease-with-regularized-canonical-correlation-analysis.html"><a href="multi-omic-modelling-of-inflammatory-bowel-disease-with-regularized-canonical-correlation-analysis.html"><i class="fa fa-check"></i><b>2</b> Multi-omic modelling of inflammatory bowel disease with regularized canonical correlation analysis</a><ul>
<li class="chapter" data-level="2.1" data-path="relationships-models.html"><a href="relationships-models.html"><i class="fa fa-check"></i><b>2.1</b> Relationships' models</a><ul>
<li class="chapter" data-level="2.1.1" data-path="relationships-models.html"><a href="relationships-models.html#designing-models"><i class="fa fa-check"></i><b>2.1.1</b> Designing models</a></li>
<li class="chapter" data-level="2.1.2" data-path="relationships-models.html"><a href="relationships-models.html#evaluating-models"><i class="fa fa-check"></i><b>2.1.2</b> Evaluating models</a></li>
</ul></li>
<li class="chapter" data-level="2.2" data-path="conclusions.html"><a href="conclusions.html"><i class="fa fa-check"></i><b>2.2</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="biological-relevance-of-results.html"><a href="biological-relevance-of-results.html"><i class="fa fa-check"></i><b>3</b> Biological relevance of results</a><ul>
<li class="chapter" data-level="3.1" data-path="comparing-different-dataset.html"><a href="comparing-different-dataset.html"><i class="fa fa-check"></i><b>3.1</b> Comparing different dataset</a></li>
<li class="chapter" data-level="3.2" data-path="shared-selected-variables.html"><a href="shared-selected-variables.html"><i class="fa fa-check"></i><b>3.2</b> Shared selected variables</a></li>
<li class="chapter" data-level="3.3" data-path="conclusions-1.html"><a href="conclusions-1.html"><i class="fa fa-check"></i><b>3.3</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="appendix"><span><b>Appendix</b></span></li>
<li class="chapter" data-level="A" data-path="online-resources.html"><a href="online-resources.html"><i class="fa fa-check"></i><b>A</b> Online resources</a></li>
<li class="chapter" data-level="B" data-path="software.html"><a href="software.html"><i class="fa fa-check"></i><b>B</b> Software</a><ul>
<li class="chapter" data-level="B.1" data-path="listed.html"><a href="listed.html"><i class="fa fa-check"></i><b>B.1</b> Listed</a></li>
<li class="chapter" data-level="B.2" data-path="by-publication.html"><a href="by-publication.html"><i class="fa fa-check"></i><b>B.2</b> By publication</a></li>
</ul></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Data integration on inflammatory bowel disease</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div class = "wip">
This tesis is ongoing and still without hard date to finish it. Watch the <a href="https://github.com/llrs/thesis">GitHub repository</a> or wait for a release announcement on Twitter on <a href="https://twitter.com/Lluis_Rev">my profile</a>.
</div>
<div id="integration-on-ibd" class="section level2">
<h2><span class="header-section-number">1.3</span> Integration on IBD</h2>
<p>One of the hypothesis behind the maintenance of the inflammation involves the microbiome. Several studies have been carried out to discover links between microbiome and the inflammation. Some of these studies used metabolites, DNA-seq sequencing of the microbiome content, or targeted 16S sequencing.</p>
<p>Also the technical method used can differ between extracted from stools or from biopsied samples at colonoscopy or from surgical samples.</p>
<p>Some articles use correlation like <span class="citation">[<a href="#ref-hasler_uncoupling_2016">24</a>]</span>. There are others that use a combination of methods</p>
<p>Very rarely there is an experimental confirmation. This is due to how complicated it is to test an interaction and the simulate the conditions.</p>
<p>One of the few methods published were the interaction is measured is to expose the ex-vivo sample or cell line with supernatant of a microbiome culture.</p>
<p>Previous methods used include the use of RGCCA, <span class="citation">[<a href="#ref-tang_integrated_2017">25</a>]</span></p>
<p><span class="citation">[<a href="#ref-de_souza_ibd_2017">26</a>]</span></p>
<p><a href="https://bioconductor.org/packages/Maaslin2" title="Maaslin2 on Bioconductor">Maaslin2</a> <span class="citation">[<a href="#ref-huWholeExomeSequencing2021">27</a>]</span></p>
<div id="genetics" class="section level3">
<h3><span class="header-section-number">1.3.1</span> Genetics</h3>
<p>Genome wide association studies (GWAS) is one of the most common genetic studies, together with methylation studies.</p>
<p>With GWAS it has been seen that <em>NOD2</em> is one highly relevant for the disease <span class="citation">[<a href="#ref-momozawa2018">28</a>]</span>. However, there hasn't been a proposed mechanism for how would this work.</p>
</div>
<div id="microbiome" class="section level3">
<h3><span class="header-section-number">1.3.2</span> Microbiome</h3>
<p>Microbiome, is often studies, both with 16S DNA studies or with whole metagenome study on the fecal or intestinal samples.</p>
<p><em>Faecalibacterium prausnitzii</em> is the microorganism that has been more associated with IBD. But several other microorganisms have been also proposed. The mechanism for them has been associated to chemical compounds such as methane, which is later associated with mucosal healing.</p>
<!--Tesis Elena Ferrer-->
</div>
<div id="multiple-data" class="section level3">
<h3><span class="header-section-number">1.3.3</span> Multiple data</h3>
<p>Some studies work with many dataset from IBD patients.</p>
<p>Most of them are based on correlation analysis and microbiome.</p>
</div>
<div id="summary-2" class="section level3">
<h3><span class="header-section-number">1.3.4</span> Summary</h3>
<p>Multiple methods and multiple studies have been done</p>
<p>Few focus on discovering the relationships on the disease <span class="citation">[<a href="#ref-huWholeExomeSequencing2021">27</a>]</span></p>
<p>The integration of data might help to improve the medicine and reveal links in difficult diseases like IBD. So far it has been applied in IBD with partial success.</p>

</div>
</div>
<!-- </div> -->
<h3>References</h3>
<div id="refs" class="references">
<div id="ref-hasler_uncoupling_2016">
<p>24. HÃ¤sler R, Sheibani-Tezerji R, Sinha A, Barann M, Rehman A, Esser D, et al. Uncoupling of mucosal gene regulation, mRNA splicing and adherent microbiota signatures in inflammatory bowel disease. Gut. 2016;gutjnlâ2016â311651.</p>
</div>
<div id="ref-tang_integrated_2017">
<p>25. Tang MS, Bowcutt R, Leung JM, Wolff MJ, Gundra UM, Hudesman D, et al. Integrated Analysis of Biopsies from Inflammatory Bowel Disease Patients Identifies SAA1 as a Link Between Mucosal Microbes with TH17 and TH22 Cells. Inflammatory Bowel Diseases. 2017;23:1544â54.</p>
</div>
<div id="ref-de_souza_ibd_2017">
<p>26. De Souza HS, Fiocchi C, Iliopoulos D. The IBD interactome: An integrated view of aetiology, pathogenesis and therapy. 2017;14.</p>
</div>
<div id="ref-huWholeExomeSequencing2021">
<p>27. Hu S, Vila AV, Gacesa R, Collij V, Stevens C, Fu JM, et al. Whole exome sequencing analyses reveal geneâmicrobiota interactions in the context of ibd. Gut. 2021;70:285â96. doi:<a href="https://doi.org/10.1136/gutjnl-2019-319706">10.1136/gutjnl-2019-319706</a>.</p>
</div>
<div id="ref-momozawa2018">
<p>28. Momozawa YDmitrieva J, ThÃ©Ã¢tre E, Deffontaine V, Rahmouni S, et al. IBD risk loci are enriched in multigenic regulatory modules encompassing putative causative genes. Nature Communications. 2018;9. doi:<a href="https://doi.org/10.1038/s41467-018-04365-8">10.1038/s41467-018-04365-8</a>.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="IBD.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="multi-omic-modelling-of-inflammatory-bowel-disease-with-regularized-canonical-correlation-analysis.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": {
"href": "https://github.com/llrs/thesis/"
},
"facebook": false,
"twitter": {
"href": "https://twitter.com/intent/tweet?text=Nice%20read%20of%20%40Lluis_Rev%20thesis"
},
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/llrs/thesis/edit/master/01_introduction.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": "https://github.com/llrs/thesis/blob/master/01_introduction.Rmd",
"text": null
},
"download": ["Data integration on inflammatory bowel disease.pdf", "Data integration on inflammatory bowel disease.epub"],
"toc": {
"collapse": false,
"toc_appendix": true,
"scroll_highlight": true
},
"toc_float": true
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
