<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>1.1 Integration | My thesis</title>
  <meta name="description" content="This is the home of the thesis.">
  <meta name="generator" content="bookdown  and GitBook 2.6.7">

  <meta property="og:title" content="1.1 Integration | My thesis" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="This is the home of the thesis." />
  <meta name="github-repo" content="llrs/thesis" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="1.1 Integration | My thesis" />
  <meta name="twitter:site" content="@Lluis_Rev" />
  <meta name="twitter:description" content="This is the home of the thesis." />
  

<meta name="author" content="Lluís Revilla Sancho">


<meta name="date" content="2019-01-24">

  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  
<link rel="prev" href="introduction.html">
<link rel="next" href="IBD.html">
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />










</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">PhD thesis</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Preface</a></li>
<li class="chapter" data-level="1" data-path="introduction.html"><a href="introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a><ul>
<li class="chapter" data-level="1.1" data-path="integration.html"><a href="integration.html"><i class="fa fa-check"></i><b>1.1</b> Integration</a><ul>
<li class="chapter" data-level="1.1.1" data-path="integration.html"><a href="integration.html#data-type-numeric-vs-categorical"><i class="fa fa-check"></i><b>1.1.1</b> Data type: numeric vs categorical</a></li>
<li class="chapter" data-level="1.1.2" data-path="integration.html"><a href="integration.html#aim"><i class="fa fa-check"></i><b>1.1.2</b> Aim</a></li>
<li class="chapter" data-level="1.1.3" data-path="integration.html"><a href="integration.html#relationship-between-variables-and-samples"><i class="fa fa-check"></i><b>1.1.3</b> Relationship between variables and samples</a></li>
<li class="chapter" data-level="1.1.4" data-path="integration.html"><a href="integration.html#relationship-between-samples"><i class="fa fa-check"></i><b>1.1.4</b> Relationship between samples</a></li>
<li class="chapter" data-level="1.1.5" data-path="integration.html"><a href="integration.html#relationship-between-the-variables"><i class="fa fa-check"></i><b>1.1.5</b> Relationship between the variables</a></li>
<li class="chapter" data-level="1.1.6" data-path="integration.html"><a href="integration.html#interpretation"><i class="fa fa-check"></i><b>1.1.6</b> Interpretation</a></li>
<li class="chapter" data-level="1.1.7" data-path="integration.html"><a href="integration.html#conclusion"><i class="fa fa-check"></i><b>1.1.7</b> Conclusion</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="IBD.html"><a href="IBD.html"><i class="fa fa-check"></i><b>1.2</b> Inflammatory bowel disease</a></li>
<li class="chapter" data-level="1.3" data-path="conclusion-1.html"><a href="conclusion-1.html"><i class="fa fa-check"></i><b>1.3</b> Conclusion</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="rgcca.html"><a href="rgcca.html"><i class="fa fa-check"></i><b>2</b> RGCCA</a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://bookdown.org/yihui/bookdown/" target="_blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">My thesis</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="integration" class="section level2">
<h2><span class="header-section-number">1.1</span> Integration</h2>
<p>Integration is usually defined as:</p>
<blockquote>
<p>“the process of combining two or more things into one” — <a href="https://dictionary.cambridge.org/dictionary/english/integration">Cambridge Dictionary</a></p>
</blockquote>
<p>We can classify and explore integration in several ways, here I outline some classifications in the bioscience field, with relationships (and references) to concrete methodology and usages.</p>
<div id="data-type-numeric-vs-categorical" class="section level3">
<h3><span class="header-section-number">1.1.1</span> Data type: numeric vs categorical</h3>
<p>The most important distinction in integration methods is what kind of data are combined. In general data can be divided between categorical and numeric values, which are usually found in several fields. Sometimes doctors want to understand the relationship between some phenotype they observe as a defined state from the underlying mechanistic point of view. Usually this involves looking how the metabolites, the gene expression, the methylation, the number of variants a gene has, and other numeric variables are related to the observed (categorical) phenotype (like pain).</p>
<p>Depending on what is a method aimed for it handles both data types or just one, often they are used differently. The most common way to handle different type of data is converting the categorical values to a mock variable, where each number represents a category, then the methods use these values to correlate and relate the variables. If the categorical variable has three values (A, B, C) it would be converted to A (1, 0, 0) B (0, 1, 0) and C (0, 0, 1).</p>
</div>
<div id="aim" class="section level3">
<h3><span class="header-section-number">1.1.2</span> Aim</h3>
<p>The results and methods of data integration depend on the question they seek to answer and on the (biological) question. Most of the times one (or all) of the following results are expected:</p>
<ul>
<li>An overview of the role of each ‘omics in a biological system</li>
</ul>
<p>Sometimes several omics are used and the question is which omic method is the best one to describe the disease. Or several methods are used to understand better how two omics interact.</p>
<ul>
<li>A better understanding of the relationship (correlation) between the ’omics types</li>
</ul>
<p>When the relationship between omics is known there are methods to find those relationships and improve upon previous knowledge. For instance, checking that in a particular case or condition there is a given relationship, and that this relationships follows our model or not.</p>
<ul>
<li>A molecular signature<a href="#fn1" class="footnoteRef" id="fnref1"><sup>1</sup></a> leading to more insight into molecular mechanisms</li>
</ul>
<p>The usage of multifactor analysis might lead to the need to redefine the features that are essential for identifying a phase or a cell line.</p>
<ul>
<li>A predictive analytical model towards personalised medicine</li>
</ul>
<p>If a good model is known it can be used as a prediction if enough information is gathered, which might improve the treatment.</p>
</div>
<div id="relationship-between-variables-and-samples" class="section level3">
<h3><span class="header-section-number">1.1.3</span> Relationship between variables and samples</h3>
<p>Depending on the amount of variables and in which samples they have been measures we can classify in two types of integrations. Traditionally for each sample few variable has been measured, for instance for a tree only the height and width are measured, however with the new omics techniques (transcriptomics, metabolomics, methylomics, genomics), thousands of variables are measured for the same sample.</p>
<ul>
<li>More variables than samples:</li>
</ul>
<p>For a single sample of RNA around 50k genome identifiers (genes, long non coding RNAs, iRNA, pseudogenes,…) can be measured. High-throughput data analysis typically falls into the category of <span class="math inline">\(p \gg n\)</span> problems, where the number of genes or proteins, <span class="math inline">\(p\)</span>, is considerably larger than the number of samples, <span class="math inline">\(n\)</span>. Which leads to the case where there are (many) more variables than samples, generally “old” statistics don’t consider this case, as it has its own complications like covariance between the variables. When two variables are tightly correlated, discerning which is the lead and which is following is near to impossible.</p>
<ul>
<li>More samples than variables:</li>
</ul>
<p>This is the typical example when from a cohort of patients the temperature is measured along the stage of a disease, two variables for each sample. This is described in the literature as <span class="math inline">\(n \gg p\)</span>.</p>
</div>
<div id="relationship-between-samples" class="section level3">
<h3><span class="header-section-number">1.1.4</span> Relationship between samples</h3>
<p>Depending on the relationship between the samples the questions answerable and the methods available differ. If each sample has all the cases we wanted to measure it is a complete case.</p>
<p>Sometimes because the sample is not enough, or there are some technical or organizational problems we might lose a case for a samples. This results in a new source of variation that has to be dealt with, which complicates the conclusion one can draw from the studies of these kind of data.</p>
<p>Even when all the cases of a sample are complete the samples can be from several sites of the same individual or from several sites or with different combinations of variables</p>
<div id="time" class="section level4 unnumbered">
<h4>Time</h4>
<p>Time is one of the factors that sometimes cannot be controled, despite having programmed visits every two weeks some patients might come early due to calendar reasons (holidays), family reasons, or disease state. Sometimes is precisely the object of the study, to see the relationships at different time, or see how the relationships change with time. Simultaneously is very important to consider it because two variables can seem correlated if we don’t take time into consideration. Also, to discover causality between two variables the cause must be before the consequence, which highlights the importance of time. Being aware of the time differences and time scales is crucial in most cases.</p>
</div>
</div>
<div id="relationship-between-the-variables" class="section level3">
<h3><span class="header-section-number">1.1.5</span> Relationship between the variables</h3>
<p>Since the lactose operon we know how some genes regulated between them, but for other variables we don’t know how they are related. For instance how does the increase in expression of a gene affects the growth of a microorganism?</p>
<p>It is important to note that some integrations may lead to interactions while others don’t, that is, the relationship of two different type of variables might be due to a physical relationship or through a more obscure relationship with no known direct contact.</p>
</div>
<div id="interpretation" class="section level3">
<h3><span class="header-section-number">1.1.6</span> Interpretation</h3>
<p>Understanding how to interpret the results of the methods is highly tight to understanding the method. If one does a correlation between two variables, the interpretation of the analysis is clear, if one variable increase, the other too. However as more complicated methodologies are developed the interpretation is not so clear, for instance how can one interpret the result of a canonical correlation analysis?</p>
<ul>
<li>Individually:</li>
</ul>
<p>How each variable relates to another, like in the correlation analysis, the relationship between two variables under study. Or by patient: how do interpret that in these patient variable A and B is X and Y?</p>
<ul>
<li>Globally:</li>
</ul>
<p>In a PCA for instance how do we interpret that some variables have the same loading? What happens in a more difficult method like canonical correlation analysis?</p>
</div>
<div id="conclusion" class="section level3">
<h3><span class="header-section-number">1.1.7</span> Conclusion</h3>
<p>The field of integration is large and complex, with high interest in the recent days, specially in the psychology and omics field, which lacks of a large study.</p>
</div>
</div>
<div class="footnotes">
<hr />
<ol start="1">
<li id="fn1"><p>A signature is usually a group of features that describe/are representative of a cell line or a process or a stage.<a href="integration.html#fnref1">↩</a></p></li>
</ol>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="introduction.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="IBD.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": true,
"facebook": false,
"twitter": true,
"google": false,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/llrs/thesis/edit/master/01_introduction.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"download": ["Thesis.pdf"],
"toc": {
"collapse": "section",
"toc_appendix": true,
"scroll_highlight": true
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:" && /^https?:/.test(src))
      src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
