<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 1 Introduction | Data integration on inflammatory bowel disease</title>
  <meta name="description" content="Thesis about the relationship between microbiome and transcriptome on the gastrointestinal tract on patients with IBD and some controls to discover key differences on this relationship." />
  <meta name="generator" content="bookdown 0.24 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 1 Introduction | Data integration on inflammatory bowel disease" />
  <meta property="og:type" content="book" />
  <meta property="og:url" content="https://thesis.llrs.dev" />
  <meta property="og:image" content="https://thesis.llrs.dev/images/R_logo_science.png" />
  <meta property="og:description" content="Thesis about the relationship between microbiome and transcriptome on the gastrointestinal tract on patients with IBD and some controls to discover key differences on this relationship." />
  <meta name="github-repo" content="llrs/thesis" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 1 Introduction | Data integration on inflammatory bowel disease" />
  <meta name="twitter:site" content="@Lluis_Revilla" />
  <meta name="twitter:description" content="Thesis about the relationship between microbiome and transcriptome on the gastrointestinal tract on patients with IBD and some controls to discover key differences on this relationship." />
  <meta name="twitter:image" content="https://thesis.llrs.dev/images/R_logo_science.png" />

<meta name="author" content="Lluís Revilla Sancho" />


<meta name="date" content="2022-01-31" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="abstract.html"/>
<link rel="next" href="hypothesis-and-objectives.html"/>
<script src="libs/header-attrs-2.11/header-attrs.js"></script>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>



<style type="text/css">
/* Used with Pandoc 2.11+ new --citeproc when CSL is used */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><strong><a href="./">Data integration on IBD</a></strong></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Preface</a></li>
<li class="chapter" data-level="" data-path="abstract.html"><a href="abstract.html"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="1" data-path="introduction.html"><a href="introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a>
<ul>
<li class="chapter" data-level="1.1" data-path="introduction.html"><a href="introduction.html#IBD"><i class="fa fa-check"></i><b>1.1</b> Inflammatory bowel disease</a>
<ul>
<li class="chapter" data-level="1.1.1" data-path="introduction.html"><a href="introduction.html#etiology-and-pathogenesis"><i class="fa fa-check"></i><b>1.1.1</b> Etiology and pathogenesis</a></li>
<li class="chapter" data-level="1.1.2" data-path="introduction.html"><a href="introduction.html#CD"><i class="fa fa-check"></i><b>1.1.2</b> Physiology</a></li>
<li class="chapter" data-level="1.1.3" data-path="introduction.html"><a href="introduction.html#treament"><i class="fa fa-check"></i><b>1.1.3</b> Treament</a></li>
<li class="chapter" data-level="1.1.4" data-path="introduction.html"><a href="introduction.html#summary-ibd"><i class="fa fa-check"></i><b>1.1.4</b> Summary</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="introduction.html"><a href="introduction.html#integration-studies-on-ibd"><i class="fa fa-check"></i><b>1.2</b> Integration studies on IBD</a>
<ul>
<li class="chapter" data-level="1.2.1" data-path="introduction.html"><a href="introduction.html#data-origin"><i class="fa fa-check"></i><b>1.2.1</b> Data origin</a></li>
<li class="chapter" data-level="1.2.2" data-path="introduction.html"><a href="introduction.html#summary-studies"><i class="fa fa-check"></i><b>1.2.2</b> Summary</a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="introduction.html"><a href="introduction.html#integration"><i class="fa fa-check"></i><b>1.3</b> Integration</a>
<ul>
<li class="chapter" data-level="1.3.1" data-path="introduction.html"><a href="introduction.html#classification-of-integration-methods"><i class="fa fa-check"></i><b>1.3.1</b> Classification of integration method’s</a></li>
<li class="chapter" data-level="1.3.2" data-path="introduction.html"><a href="introduction.html#interpretation"><i class="fa fa-check"></i><b>1.3.2</b> Interpretation</a></li>
<li class="chapter" data-level="1.3.3" data-path="introduction.html"><a href="introduction.html#reviews"><i class="fa fa-check"></i><b>1.3.3</b> Reviews</a></li>
<li class="chapter" data-level="1.3.4" data-path="introduction.html"><a href="introduction.html#methods-integration"><i class="fa fa-check"></i><b>1.3.4</b> Summary</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="2" data-path="hypothesis-and-objectives.html"><a href="hypothesis-and-objectives.html"><i class="fa fa-check"></i><b>2</b> Hypothesis and objectives</a>
<ul>
<li class="chapter" data-level="2.1" data-path="hypothesis-and-objectives.html"><a href="hypothesis-and-objectives.html#hypothesis"><i class="fa fa-check"></i><b>2.1</b> Hypothesis</a></li>
<li class="chapter" data-level="2.2" data-path="hypothesis-and-objectives.html"><a href="hypothesis-and-objectives.html#objectives"><i class="fa fa-check"></i><b>2.2</b> Objectives</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="materials-and-methods.html"><a href="materials-and-methods.html"><i class="fa fa-check"></i><b>3</b> Materials and methods</a>
<ul>
<li class="chapter" data-level="3.1" data-path="materials-and-methods.html"><a href="materials-and-methods.html#datasets"><i class="fa fa-check"></i><b>3.1</b> Datasets</a>
<ul>
<li class="chapter" data-level="3.1.1" data-path="materials-and-methods.html"><a href="materials-and-methods.html#methods-puget"><i class="fa fa-check"></i><b>3.1.1</b> Puget’s dataset</a></li>
<li class="chapter" data-level="3.1.2" data-path="materials-and-methods.html"><a href="materials-and-methods.html#methods-hsct"><i class="fa fa-check"></i><b>3.1.2</b> HSCT dataset</a></li>
<li class="chapter" data-level="3.1.3" data-path="materials-and-methods.html"><a href="materials-and-methods.html#methods-hasler"><i class="fa fa-check"></i><b>3.1.3</b> Häsler’s dataset</a></li>
<li class="chapter" data-level="3.1.4" data-path="materials-and-methods.html"><a href="materials-and-methods.html#methods-morgan"><i class="fa fa-check"></i><b>3.1.4</b> Morgan’s dataset</a></li>
<li class="chapter" data-level="3.1.5" data-path="materials-and-methods.html"><a href="materials-and-methods.html#methods-barcelona"><i class="fa fa-check"></i><b>3.1.5</b> BARCELONA dataset</a></li>
<li class="chapter" data-level="3.1.6" data-path="materials-and-methods.html"><a href="materials-and-methods.html#methods-howell"><i class="fa fa-check"></i><b>3.1.6</b> Howell’s dataset</a></li>
<li class="chapter" data-level="3.1.7" data-path="materials-and-methods.html"><a href="materials-and-methods.html#methods-hernandez"><i class="fa fa-check"></i><b>3.1.7</b> Hernández’ dataset</a></li>
</ul></li>
<li class="chapter" data-level="3.2" data-path="materials-and-methods.html"><a href="materials-and-methods.html#processing-samples"><i class="fa fa-check"></i><b>3.2</b> Processing samples</a>
<ul>
<li class="chapter" data-level="3.2.1" data-path="materials-and-methods.html"><a href="materials-and-methods.html#transcriptome-sequencing"><i class="fa fa-check"></i><b>3.2.1</b> Transcriptome sequencing</a></li>
<li class="chapter" data-level="3.2.2" data-path="materials-and-methods.html"><a href="materials-and-methods.html#microbial-dna-sequencing"><i class="fa fa-check"></i><b>3.2.2</b> Microbial DNA sequencing</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="materials-and-methods.html"><a href="materials-and-methods.html#statistics"><i class="fa fa-check"></i><b>3.3</b> Statistics</a>
<ul>
<li class="chapter" data-level="3.3.1" data-path="materials-and-methods.html"><a href="materials-and-methods.html#functional-enrichment"><i class="fa fa-check"></i><b>3.3.1</b> Functional enrichment</a></li>
<li class="chapter" data-level="3.3.2" data-path="materials-and-methods.html"><a href="materials-and-methods.html#other-methods-used"><i class="fa fa-check"></i><b>3.3.2</b> Other methods used</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="materials-and-methods.html"><a href="materials-and-methods.html#regularized-generalized-canonical-correlation-analysis"><i class="fa fa-check"></i><b>3.4</b> Regularized generalized canonical correlation analysis</a>
<ul>
<li class="chapter" data-level="3.4.1" data-path="materials-and-methods.html"><a href="materials-and-methods.html#description"><i class="fa fa-check"></i><b>3.4.1</b> Description</a></li>
<li class="chapter" data-level="3.4.2" data-path="materials-and-methods.html"><a href="materials-and-methods.html#results"><i class="fa fa-check"></i><b>3.4.2</b> Results</a></li>
<li class="chapter" data-level="3.4.3" data-path="materials-and-methods.html"><a href="materials-and-methods.html#models"><i class="fa fa-check"></i><b>3.4.3</b> Models</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="results-1.html"><a href="results-1.html"><i class="fa fa-check"></i><b>4</b> Results</a>
<ul>
<li class="chapter" data-level="4.1" data-path="results-1.html"><a href="results-1.html#packages"><i class="fa fa-check"></i><b>4.1</b> Packages</a>
<ul>
<li class="chapter" data-level="4.1.1" data-path="results-1.html"><a href="results-1.html#experdesign-1"><i class="fa fa-check"></i><b>4.1.1</b> experDesign</a></li>
<li class="chapter" data-level="4.1.2" data-path="results-1.html"><a href="results-1.html#baseset-1"><i class="fa fa-check"></i><b>4.1.2</b> BaseSet</a></li>
<li class="chapter" data-level="4.1.3" data-path="results-1.html"><a href="results-1.html#intermodels"><i class="fa fa-check"></i><b>4.1.3</b> inteRmodels</a></li>
</ul></li>
<li class="chapter" data-level="4.2" data-path="results-1.html"><a href="results-1.html#analysis"><i class="fa fa-check"></i><b>4.2</b> Analysis</a>
<ul>
<li class="chapter" data-level="4.2.1" data-path="results-1.html"><a href="results-1.html#results-puget"><i class="fa fa-check"></i><b>4.2.1</b> Puget’s dataset</a></li>
<li class="chapter" data-level="4.2.2" data-path="results-1.html"><a href="results-1.html#results-hsct"><i class="fa fa-check"></i><b>4.2.2</b> HSCT dataset</a></li>
<li class="chapter" data-level="4.2.3" data-path="results-1.html"><a href="results-1.html#results-hasler"><i class="fa fa-check"></i><b>4.2.3</b> Häsler’s dataset</a></li>
<li class="chapter" data-level="4.2.4" data-path="results-1.html"><a href="results-1.html#results-morgan"><i class="fa fa-check"></i><b>4.2.4</b> Morgan’s dataset</a></li>
<li class="chapter" data-level="4.2.5" data-path="results-1.html"><a href="results-1.html#results-barcelona"><i class="fa fa-check"></i><b>4.2.5</b> BARCELONA dataset</a></li>
<li class="chapter" data-level="4.2.6" data-path="results-1.html"><a href="results-1.html#results-howell"><i class="fa fa-check"></i><b>4.2.6</b> Howell’s dataset</a></li>
<li class="chapter" data-level="4.2.7" data-path="results-1.html"><a href="results-1.html#results-hernandez"><i class="fa fa-check"></i><b>4.2.7</b> Hernández’ dataset</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="discussion.html"><a href="discussion.html"><i class="fa fa-check"></i><b>5</b> Discussion</a>
<ul>
<li class="chapter" data-level="5.0.1" data-path="discussion.html"><a href="discussion.html#other"><i class="fa fa-check"></i><b>5.0.1</b> Other</a></li>
<li class="chapter" data-level="5.1" data-path="discussion.html"><a href="discussion.html#relevance-for-other-research"><i class="fa fa-check"></i><b>5.1</b> Relevance for other research</a></li>
<li class="chapter" data-level="5.2" data-path="discussion.html"><a href="discussion.html#implications-of-research"><i class="fa fa-check"></i><b>5.2</b> Implications of research</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="conclusions.html"><a href="conclusions.html"><i class="fa fa-check"></i><b>6</b> Conclusions</a></li>
<li class="chapter" data-level="7" data-path="acknowledgments.html"><a href="acknowledgments.html"><i class="fa fa-check"></i><b>7</b> Acknowledgments</a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="appendix.html"><a href="appendix.html"><i class="fa fa-check"></i>Appendix</a>
<ul>
<li class="chapter" data-level="7.1" data-path="appendix.html"><a href="appendix.html#online-resources"><i class="fa fa-check"></i><b>7.1</b> Online resources</a></li>
<li class="chapter" data-level="7.2" data-path="appendix.html"><a href="appendix.html#software"><i class="fa fa-check"></i><b>7.2</b> Software</a>
<ul>
<li class="chapter" data-level="7.2.1" data-path="appendix.html"><a href="appendix.html#listed"><i class="fa fa-check"></i><b>7.2.1</b> Listed</a></li>
<li class="chapter" data-level="7.2.2" data-path="appendix.html"><a href="appendix.html#by-project"><i class="fa fa-check"></i><b>7.2.2</b> By project</a></li>
</ul></li>
</ul></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Data integration on inflammatory bowel disease</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div class = "wip">
This tesis is ongoing and still without hard date to finish it. Watch the <a href="https://github.com/llrs/thesis">GitHub repository</a> or wait for a release announcement on Twitter on <a href="https://twitter.com/Lluis_Revilla">my profile</a>.
</div>
<div id="introduction" class="section level1" number="1">
<h1><span class="header-section-number">Chapter 1</span> Introduction</h1>
<p><a href="https://en.wikipedia.org/wiki/Inflammatory_bowel_disease">Inflammatory bowel disease</a> (IBD) involves <a href="https://en.wikipedia.org/wiki/Crohn%27s_disease">Crohn’s disease</a> (CD) and <a href="https://en.wikipedia.org/wiki/Ulcerative_colitis">ulcerative colitis</a> (UC).
It generally affects the terminal ileum and the colon but it can involve any segment of the gastrointestinal tract.
UC is a recurrent, chronic and continuous inflammation of the colon and rectum while the CD is not a continuous inflammation and affects the whole gastrointestinal tract causing transmural inflammation.</p>
<p>IBD etiology is unknown.
However, once it has initiated the most prevalent hypothesis of its chronicity suggests an aberrant immunological response to antigens of the commensal microbiome.</p>
<p>To diagnose the IBD doctors use endoscopy and/or magnetic resonance imaging and histologies.</p>
<p>Treatments provided for the IBD include, noninflammatory drugs, suppressors and biologics, i.e, anti TNF- anti-IL2, 23, anti-<span class="math inline">\(\alpha4\beta7\)</span>.
The therapeutic options can induce remission in some patients, but they often need continuous treatment to avoid recurrence.
Nevertheless, many patients <span class="math display">\[ref:needed\]</span> are refractory or intolerant to those therapies and need to undergo surgery.</p>
<div id="IBD" class="section level2" number="1.1">
<h2><span class="header-section-number">1.1</span> Inflammatory bowel disease</h2>
<p>IBD includes the CD and UC which are characterized by alternating periods of remission and clinical relapse that mainly affect the gastrointestinal tract.
CD is a progressive relapsing disease that can affect all the gastrointestinal tract but shows mostly on both terminal ileum and colon with a discontinuous inflammation.
UC is a colonic relapsing disease characterized by continuous inflammation of the colon.
Both of them have different risk factors, clinical, endoscopic an histological characteristics (see sections <a href="introduction.html#CD">1.1.2</a> and <a href="introduction.html#UC">1.1.2.2</a> ).</p>
<p>Around 3.5 million individuals have IBD in Europe and North America combined <span class="citation">[<a href="#ref-jairath2020" role="doc-biblioref">1</a>]</span>.
IBD is more commonly found in industrialized and developed regions, suggesting that environmental factors might greatly influence IBD occurrence.
In addition, the incidence of IBD is increasing in areas, such as Asia or Eastern Europe, where the number of cases was relatively low hitherto <span class="citation">[<a href="#ref-burisch2015" role="doc-biblioref">2</a>]</span>.</p>
<p>The dysregulation of the inflammatory response observed in IBD requires interplay between host genetic factors and the intestinal microbiome.
Several studies support the concept that IBD arise from an exacerbate immune response against commensal gut microorganisms.
Nonetheless, the disease could result from an imbalanced microbial composition leading to generalized or localized dysbiosis<a href="#fn1" class="footnote-ref" id="fnref1"><sup>1</sup></a>.</p>
<p>The role of the gut microbiome in IBD is an active ongoing field of research.
Several authors are currently studying the alterations reported in IBD of the intestinal microbiome.
However, it is still unclear the cause-effect relation between dysbiosis and IBD.
Partly due to the multiple variables already identified that have been linked to IBD; for instance, age, diet, usage of antibiotic, tobacco, and socioeconomic status <span class="citation">[<a href="#ref-human_microbiome_project_consortium_framework_2012" role="doc-biblioref">3</a>]</span>.</p>
<p>The relationship between host and microbiome has been proposed to play a fundamental role in maintaining disease.
For instance, some <em>Proteobacteria</em> species which have adherent and invasive properties might exploit host defenses and promote a proinflammatory environment, altering the intestinal microbiota in favor of dysbiosis <span class="citation">[<a href="#ref-mukhopadhya2012" role="doc-biblioref">4</a>]</span>.</p>
<p>The epithelium is often damaged and might present ulcers or other inflammation symptoms.
A segment of the gastrointestinal tract might recover if the patient receives treatment or due the natural cycles of the disease.
But once a segment is affected by the disease it can be considered as involved, as some damage remains even if the tissues is no longer inflammed.</p>
<div id="etiology-and-pathogenesis" class="section level3" number="1.1.1">
<h3><span class="header-section-number">1.1.1</span> Etiology and pathogenesis</h3>
<p>Several mechanisms have been proposed to drive IBD pathogenesis <span class="citation">[<a href="#ref-silva2016a" role="doc-biblioref">8</a>]</span>.
It is also unclear if CD and the UC share the same origin considering their different symptoms.</p>
<p>There is also evidence of some genetic component on the onset of the disease, specially if the disease appears very early (less than 2 years old patients) <span class="citation">[<a href="#ref-satsangi2006c" role="doc-biblioref">10</a>]</span>.
Genome-wide association studies (GWAS) have linked IBD to over 100 genetic loci, including a <em>NOD2</em> gene, but so far there is not any known mechanism how polymorfism on this genes are driving the disease <span class="citation">[<a href="#ref-horowitz2021" role="doc-biblioref">11</a>]</span>.
On early pedriatic and adult disease the genetic component is lower than on very early on set and it is thought that the environmental factors are the main cause of the disease at those ages.</p>
<p>On the following sections I’ll explore the role of several of the possible factors involved on the pahtogenesis, starting with the genetics.</p>
<div id="genetics" class="section level4" number="1.1.1.1">
<h4><span class="header-section-number">1.1.1.1</span> Genetics</h4>
<p>IBD is not an heritable disease, except for very early onset IBD, but it has some genetic influence that predisposes people to have it.</p>
<p>This has lead to look for genetic factors on IBD both on general population and on the early cases.
Genome wide association studies (GWAS) are one of the most common genetic studies performed, together with methylation studies.
To discover through linkage desequilibrium genetic variations linked to phenotypes and regulatory transcription changes, respectively.</p>
<p>With GWAS several allels on protein coding loci have been found, rising to around 300 genetic variants <span class="citation">[<a href="#ref-jostins2012" role="doc-biblioref">14</a>]</span>.
The mechanism of how this gene protects from UC has not been confirmed yet <span class="citation">[<a href="#ref-horowitz2021" role="doc-biblioref">11</a>]</span>.</p>
<p>Many of the relevant genetic loci related to IBD are not on protein coding fragments of the genome.
Recently expression quantitative trait loci (eQTL) particularly showed <span class="citation">[<a href="#ref-mcgovern2015" role="doc-biblioref">15</a>]</span> that locis are on enhancers or promoters like e.g.
H3K27Ac or promoter e.g.
H3K4me1 marks as found by chromatin immunoprecipitation sequencing (<a href="https://en.wikipedia.org/wiki/ChIP_sequencing" title="Wikipedia: ChIP-seq">ChIP-Seq</a>).</p>
</div>
<div id="microbiome" class="section level4" number="1.1.1.2">
<h4><span class="header-section-number">1.1.1.2</span> Microbiome</h4>
<!--# Aida 2.3 p24-->
<p>The human intestine is a large reservoir of co-existing microorganisms (bacteria, fungi, viruses, and unicellular eukaryotes) .
This microbiome community exerts different functions in the human body influencing nutrients’ metabolism, the maturation of the immune system while suppressing the growth of harmful microorganisms’ <span class="citation">[<a href="#ref-khanna2014" role="doc-biblioref">16</a>]</span>.</p>
<p>The role of the gut microbiota has been proposed to play a role in the IBD pathogenesis.
IBD has been characterized by a breakdown in the balance between beneficial and harmful bacteria that are present in the human gut compared to healthy individuals <span class="citation">[<a href="#ref-swidsinski2002" role="doc-biblioref">17</a>, <a href="#ref-tamboli2004" role="doc-biblioref">18</a>]</span>.</p>
<!--# End-->
<!--# Elena tesis p18-->
<p>Indeed, many studies show that patients with IBD have less biodiversity.
Biodiversity is measured on <span class="math inline">\(\alpha\)</span> (alfa) and <span class="math inline">\(\beta\)</span> (beta) diversity.
<span class="math inline">\(\alpha\)</span>-diversity is a measure of the species present on a single sample and its abundance while <span class="math inline">\(\beta\)</span>-diversity compares the diversity between samples.
There are some reports of taxonomic changes and increase on <em>Enterobacteriaceae sp</em>, <em>Escherichia coli</em> (specially the invasive strain) at the mucosal layer of IBD patients <span class="citation">[<a href="#ref-kostic2014" role="doc-biblioref">21</a>]</span>.</p>
<!--# Aida 2.3 p24-->
<p>In fact, it has been recently proposed that several unique microbial species can distinguish healthy controls from UC and CD patients <span class="citation">[<a href="#ref-sankarasubramanian2020a" role="doc-biblioref">22</a>, <a href="#ref-lopez-siles2014" role="doc-biblioref">23</a>]</span>.</p>
<div class="figure"><span style="display:block;" id="fig:composition"></span>
<img src="images/tesis_AM_Figure2.png" title="Microbial composition and epithelial barrier status in healthy versus IBD gut." alt="" />
<p class="caption">Figure 1.1:  The microbial composition in a healthy and IBD gut.</p>
</div>
<!--# Aida p25-->
<p>One of the proposed mechanism of crosstalk between bacteria and host is through bacterial metabolites.
They interact with the cells and modulate the state of the intestine.
One example of such metabolite is butyrate which has been linked to microorganisms presents on healthy intestines and shown to interact with intestine cells and help regulate some genes <span class="citation">[<a href="#ref-ferrer-picón2020" role="doc-biblioref">24</a>]</span>.</p>
<p>As previously mentioned, adherent invasive <em>Escherichia coli</em>, a proteobacteria specie, has been associated with IBD <span class="citation">[<a href="#ref-darfeuille-michaud1998" role="doc-biblioref">25</a>]</span>.
Adherent invasive strains are mainly found in ileal and colonic samples of CD patients and their presence in UC is less clear.
These adherent invasive cells enter through the epithelium of the more permeable cells and live on their cytosol.</p>
<!--# End-->
<p>In addition, there has been some observations that IBD patients response to treatments that modify their gut microbiota, such as: fecal microbiota transplantation, probiotics and postbiotics <span class="math display">\[ref:needed\]</span>.</p>
</div>
<div id="immune-response" class="section level4" number="1.1.1.3">
<h4><span class="header-section-number">1.1.1.3</span> Immune response</h4>
<p>As explained previously the immune system plays a role in IBD pathogenesis and pathophysiology.</p>
<p>Loss of tolerance to commensal bacteria has been suggested as the underlying mechanism triggering the inflammation on the intestine.
The immune response involves many different cells lines and regions, which are important to know how they organize for a better understanding of the disease.</p>
<!--# Image from Ana tesis p29: -->
<div class="figure"><span style="display:block;" id="fig:barrier"></span>
<img src="images/tesis_AC_FigureX.png" title="The intestinal epithelial barrier" alt="" />
<p class="caption">Figure 1.2:  The intestinal epithelial barrier.</p>
</div>
<p>From the luminal side of the intestine, the first layer is the mucosa (See sections <a href="introduction.html#fig:barrier">1.2</a> or <a href="introduction.html#fig:composition">1.1</a>).
In the colon the mucus is organized in two layers: the inner layer, a firm mucus layer; and the outer, loose mucus layer <span class="citation">[<a href="#ref-okumura2017" role="doc-biblioref">26</a>]</span>.
The intestinal epithelium is a single layer of cells organized into crypts and villi (and circular folds on the large intestine) that carries out a diverse array of functions besides digestion performed by specialized cell lineages.</p>
<!--# From tesis Aida p20 Immune Response-->
<p>Immune response in the intestinal mucous is mainly excreted by the gut associated lymphoid tissue <span class="citation">[<a href="#ref-faria2012" role="doc-biblioref">27</a>]</span>.
Genetically predisposed patients when exposed to certain environmental factors activate immune responses against microbials or self-antigens which in turn, may impair the mucosal barrier of the intestinal mucosa, the first physical barrier on the mucosal surface.</p>
<p>Both the adaptive and the innate immune cells are present in the intestine, right below the epithelium.
On IBD due to antigen translocation into the lamina propria, the immune response leads the adaptive cells to generate immune response to harmless components of the intestinal microbiota.
This initial response induces a local increase in the production of pro-inflammatory cytokines and mediators which damages the mucosa.
Therefore, the loss of integrity on this barrier enables the intestinal luminal bacteria to access the intestinal epithelium and to interact with the immune system underneath it more directly <span class="citation">[<a href="#ref-hisamatsu2013" role="doc-biblioref">28</a>]</span>.</p>
<p>The intestinal epithelium is another line of defense against bacterial invasion.
Intestinal epithelial cells play a key role in controlling the integrity of the physical barrier to the intestinal microorganisms <span class="citation">[<a href="#ref-hisamatsu2013" role="doc-biblioref">28</a>]</span> not only physically but also secreting antimicrobial peptides and defensins, both of which are altered in IBD patients <span class="citation">[<a href="#ref-michielan2015" role="doc-biblioref">29</a>]</span>.
The intestinal epithelium also plays a key role on the intake and diffusion of metabolites from the intestinal lumen to the lamina propia.</p>
<p>Damaging or increasing the permeability of the intestinal epithelium results on a response from the immune system.
Detecting signals of any foreign particle can also trigger the immune system.
On the intestine this starts with the identification of these signals by intestinal epithelal cells have pattern recognition receptors.
There are two main pattern recognition receptors: toll-like receptors (TLR), which are present on the surface, and nucleotide-binding oligomerization domain-like receptors (NOD-like receptors), present on the cytoplasm of the cells.
These receptors upon recognition of pathogen associated molecular patterns (PAMPs) start an amplifying signaling producing chemokines and cytokines which activates the transcription and translation of pro-inflammatory mediators to ensure an effective immune response.
Initially the innate response is triggered but the cells also increase the antigen presentation to T cells and thus activate the immune adaptive response <span class="citation">[<a href="#ref-mayorgas2021a" role="doc-biblioref">30</a>]</span>.</p>
<p>Other cell types, such as monocytes, macrophages and dendritic cells also present the pattern recognition receptors.
From those, macrophages and dendritic cells are antigen presenting cells too and secrete several cytokines to activate other immune cells.
Usually CD patients express higher amounts of TLR than healthy individuals, which might trigger a stronger immune response.
This response is driven by CD4<sup>+</sup> T cells proliferation in secondary lymphoid tissues to T helpers in the presence of the antigens and cytokines nearby.</p>
<p>T helpers (T<sub>h</sub>) differentiate depending of the cytokines at which they are exposed.
Th type 1 are driven by exposure to IL-12 secreted primarily by dentritic cells.
T<sub>h</sub>2 are dtriven by cytokines secreted by macrophages.
The imbalance between T<sub>h</sub>1 and T<sub>h</sub>2-promoting cytokines determines the intensity and duration of the inflammatory response in experimental colitis <span class="citation">[<a href="#ref-neurath1996" role="doc-biblioref">31</a>]</span>.</p>
<p>T<sub>h</sub>17-promoting cytokines are less well characterized in human.
T<sub>reg</sub> cells differentiate after exposure to cytokines IL-10, IFN-<span class="math inline">\(\gamma\)</span> and TGF-<span class="math inline">\(\beta\)</span>.
Overall the presence of certain cytokines and the response to self-antigens are factors leads to an inflammation and damage that is related to the onset and establishment of IBD</p>
<!--# End tesis Aida/ more or less -->
<p>On these kind of diseases autologous hematopoietic stem cell transplantation has shown some benefits on IBD <span class="citation">[<a href="#ref-corraliza" role="doc-biblioref">32</a>]</span>.
The benefit of hematopoietic stem cell transplant (HSCT) in autoimmunity is thought to originate from the depletion of auto-reactive cells regardless of their specificity.
However, due to its associated risk this therapy is only given when patients are refractory to all available therapeutic options.</p>
</div>
<div id="environmental" class="section level4" number="1.1.1.4">
<h4><span class="header-section-number">1.1.1.4</span> Environmental</h4>
<!--# p26 Aida sec 2.4 -->
<p>Chronic inflammatory disorders and neoplasms have become the main cause of morbidity and mortality in the countries with high standards of personal cleanliness.
A decrease in human exposure to microbes or hygene which might affect the proper maturation of the immune system so that it provides less immune response or exacerbated response towards “friendly” microbes <span class="citation">[<a href="#ref-strachan1989" role="doc-biblioref">33</a>, <a href="#ref-scudellari2017" role="doc-biblioref">34</a>]</span>.</p>
<p>From other environmental factors related to IBD such as tobacco, diet, certain drugs and stress; tobacco is the most influential environmental factor.
Surprisingly, it has an opposite effect on UC and CD: in CD tobacco is a risk factor that increases the risk of relapse and/or surgical intervention.
In UC, it has been observed that smoking cessation worsens the disease <span class="citation">[<a href="#ref-thomas1998" role="doc-biblioref">35</a>]</span>.</p>
<p>Pharmacological treatments such as oral contraceptives, non steroidal anti-inflammatory drugs are also related to develop or relapse the disease <span class="citation">[<a href="#ref-cornish2008" role="doc-biblioref">36</a>, <a href="#ref-kaufmann1987" role="doc-biblioref">37</a>]</span>.</p>
<p>The psychological welfare of people also plays an important role in the disease progression, stress, anxiety and depression might be important in relapse and deterioration of the disease <span class="citation">[<a href="#ref-bitton2008" role="doc-biblioref">38</a>]</span>.
Other environmental factors have been linked to IBD but without enough evidence to support a causative effect in the development of the disease.</p>
<!--# End-->
<!--# Check with Elena Basolas Tesis p18-->
<p>Perhaps the environment factors influence the samples disease etiology towards CD or UC as there are some differences between them.
On CD a granulome, that is a region with big multinucleous cells, can appear on any intestinal layer, while on UC the damage is different: it consists in many neutrophil in the crypt lumen <span class="citation">[<a href="#ref-bassolasmolina2018a" role="doc-biblioref">39</a>]</span>.</p>
<p>To distinguish between the diseases, in addition to the inflamed location(s), mosaic zones (patches of inflamed and non-inflamed areas) are more characteristic of CD <span class="citation">[<a href="#ref-bassolasmolina2018" role="doc-biblioref">40</a>]</span>.</p>
<!--# End -->
</div>
</div>
<div id="CD" class="section level3" number="1.1.2">
<h3><span class="header-section-number">1.1.2</span> Physiology</h3>
<div id="cd" class="section level4" number="1.1.2.1">
<h4><span class="header-section-number">1.1.2.1</span> CD</h4>
<p>As previously introduced, CD is a chronic inflammatory disorder characterized by a discontinuous inflammation of the gastrointestinal tract.
Inflammation on the gastrointestinal tract is transmural and can affect from the mouth to the anus, but mainly it manifests on the ileum and colon <span class="citation">[<a href="#ref-demattos2015a" role="doc-biblioref">41</a>]</span>.
It is frequently associated with extraintestinal manifestation and/or concomitant immuno-mediated diseases.</p>
<p>The disease itself manifest an heterogeneous symptoms that can involve, diarrhea, weight loss, abdominal pain, fever, anorexia and malaise.
Other less frequent co-occurring manifestations are arthritis, primary sclerosing cholangitis, skin disorders venous or arterial thromboembolism and/or pulmonary involvement <span class="citation">[<a href="#ref-baumgart2012" role="doc-biblioref">42</a>]</span> .
These symptoms make it hard to correctly diagnose the disease by non-specialists, in addition there is not a non invasive easy procedure to diagnose it.
All these can lead to delays on correct diagnosis of the disease.</p>
<!--# Tesis Ana p21-->
<p>The detection of parasites or bacteria, such as <em>Clostridium difficile</em>, have been associated with CD <span class="citation">[<a href="#ref-guardiola2018" role="doc-biblioref">45</a>]</span>.</p>
<!--# End-->
<p>Usually the best diagnosis method is to perform a colonoscopy, whether there is inflammation on the gastroinestinal tract on discontinuous regions then is CD.
This inflammation could also present ulceration with rectal sparing and histological lesions which also help to diagnose the patients <span class="citation">[<a href="#ref-corralizamárquez2019" role="doc-biblioref">46</a>]</span>.</p>
<p>The Montreal classification aims to classify patients according to their age of disease onset, standardized anatomical disease location an disease behaviour .
This classification assumes that the location of CD remains stable over time after diagnosis but behavioral phenotypes change.
Other scores consider area affected:</p>
<ul>
<li>Montreal classification allows for early onset of disease to be categorized separately those with age of diagnosis at 16 years or younger, diagnosis at 17–40 years and &gt;40 years, respectively <span class="citation">[<a href="#ref-satsangi2006" role="doc-biblioref">47</a>]</span>.</li>
<li>SES-CD: simple endoscopic score for CD <span class="citation">[<a href="#ref-daperno2004" role="doc-biblioref">48</a>]</span>. Score based on size of ulcers, ulcerate surface percentage, affected surface and presence of narrowing on the bowel.</li>
<li>CDAI: CD Activity Index takes into account weight, ideal body weight, sex, and events on the last week such as liquid stools, abdominal pain, general well-being and if anti-diarrhea drug usage, as well as knowing if there are fistulas, fever, and other complications <span class="citation">[<a href="#ref-best1976" role="doc-biblioref">49</a>]</span></li>
</ul>
<!--# Add Figure 2 from ACM thesis?-->
<p>To some extent, there is a disassociation between clinical symptomes and the endoscopy finding.
Often patients report feeling better despite lack of muscular healing <span class="citation">[<a href="#ref-bhattacharya2016a" role="doc-biblioref">50</a>]</span>.
To overcome this disassociation and be able to compare the well-being of patients several scores and thresholds are used on research and by physicians that will be described later.</p>
<!--# Disease course -->
<p>In the early stages of the disease the relapsing and remitting course is more frequent.
Often relapses are accompanied by clinical symptoms, and few have prolonged clinical remission (without treatment) <span class="citation">[<a href="#ref-peyrin-biroulet2010" role="doc-biblioref">51</a>]</span>.
When there is clinical remission, there can still remain some other lesions and often sublinical inflammation persists.
Frequently the damage caused by the disease evolves to fibrostenotic stricture or penetrating lesions (fistula and abscess).</p>
<p>Damage of the disease might not be apparent to patients and might be only seen several years later than the first detection <span class="citation">[<a href="#ref-bhattacharya2016" role="doc-biblioref">52</a>]</span>.
Mucosal healing is a first step towards the healing of deeper layers of the inflamed bowel wall on the CD.</p>
<p>Patients might progress from an inflammatory phenotype to a stricturing or penetrating one <span class="citation">[<a href="#ref-satsangi2006a" role="doc-biblioref">53</a>]</span>.
Stricturing is a narrowing of a part of the intestine often because of scar tissue and fibrosis in its wall.
Penetrating is when the epithelium has some holes or tubes.
If these tubes result in an abnormal connection between two body parts it is a fistula, it might also result in an abcess, a collections of pus, often developed in the abdomen, pelvis, or around the anal area.</p>
</div>
<div id="UC" class="section level4" number="1.1.2.2">
<h4><span class="header-section-number">1.1.2.2</span> UC</h4>
<p>As previously introduced, UC is a chronic inflammatory disorder characterized by a continuous inflammation of the colon.
Depending on the inflamed segments of the intestine it is classified in several phenotypes.</p>
<p>Around a third of the patients with UC suffer proctitis, the inflammation of the rectum.
If the segments from rectum to the sigmoid colon are affected is a distal colitis, if it affects the left colon then it becomes a left colitis.
If the inflammation continues to the descending colon it is then an extensive colitis until it affects the whole colon when it becomes a pancolitis.
The extension of UC is inversely related to the frequency.
However, the extension and severity of the disease correlates: the prognosis is worse the more extended it is <span class="citation">[<a href="#ref-etchevers2009" role="doc-biblioref">54</a>]</span>.</p>
<p>The goal of the clinical care is to recover and reach a healthy life.
As a first step, the symptoms of IBD have to lessen to the point that they are mostly absent, gone, or barely noticeable, this is known as clinical remission.
However, this is not enough as the mucosa might be still inflamed and thus the reconstitution of the structure and function of the intestinal mucosa is not complete.
Other lesions, might aid to the progression to other phenotypes such as fibrostenotic stricture or penetrating lesions increasing the structural bowel damage or primary sclerosing cholangitis <span class="citation">[<a href="#ref-boonstra2012" role="doc-biblioref">55</a>]</span>.</p>
<p>To prevent and avoid further damage several procedures are followed:</p>
<p>When there is dysplasia, an abnormal development of cells within tissues or organs (which is considered a precedence before colorectal cancer growth <span class="citation">[<a href="#ref-mark-christensen2018" role="doc-biblioref">56</a>]</span>), or the damage on the colon has been too big a surgical procedure to remove part or all of the colon must be done.<br />
Patients that undergo a colectomy need to have their bowel reconnected with a procedure called ileoanal anastomosis (also know as J-pouch by the shape it takes) surgery.
Often the lining of the pouch created during surgery becomes inflamed on what is known as pouchitis <span class="citation">[<a href="#ref-schieffer2016" role="doc-biblioref">57</a>]</span>.</p>
<p>Many scores have been proposed for several purposes, from quality of life to disease severity or patient status.
Among the scores most used are the following:</p>
<ul>
<li><p>Mayo: A score designed to be simple to calculate based on stool frequency, bleeding, mucosal appearance at endoscopy and physicians assessment <span class="citation">[<a href="#ref-schroeder1987" role="doc-biblioref">58</a>]</span>.</p></li>
<li><p>IBDQ: A 32 questionnaire used to assess the quality of life grouped into four categories: bowel, systemic, social and emotional <span class="citation">[<a href="#ref-irvine1999" role="doc-biblioref">59</a>]</span>.</p></li>
<li><p>UCEDIS: An endoscopic score based on vascular pattern, internal bleeding and erosions and ulcers <span class="citation">[<a href="#ref-travis2012" role="doc-biblioref">60</a>]</span>.</p></li>
</ul>
<p>Other measured parameters include, weight, effective weight, fecal calprotectin, C reactive protein (CRP) and hemoglobine.</p>
</div>
</div>
<div id="treament" class="section level3" number="1.1.3">
<h3><span class="header-section-number">1.1.3</span> Treament</h3>
<p>Patients might require many drugs to treat UC.
Although this varies between patients according to their physicians’ prescription and recommendations, here is a short list of drugs or treatments targeting a particular pathway: anti-TNF<span class="math inline">\(\alpha\)</span>, vedolizumab, ustekinumab, risankizumab, tofacitinib, infliximab, adalimumab a 5-asa treatement, corticoesteroids, Azathiprine/Mercaptopurine 6MCP, Methotrexate MTX, Tacrolimus FK, Cyclosporin A CyA. Patients are taking one or more of these drugs as standard of care.</p>
<p>To reduce secondary effects of drugs or to find the right dose of the drugs often a reduction of dose is recommended.
However, dose reduction often results in diminished response to the drug.
Thus, the patients do not recover the initial response they had later on if the dose is raised to the previous levels.
This builds up also on patients being refractory to the drugs and treatments given <span class="citation">[<a href="#ref-raine2021" role="doc-biblioref">61</a>]</span>.</p>
<p>One of the most relevant advances in the treatment of IBD has been the introduction of anti-TNF antibodies, but this therapy is associated to significant medical costs.
Reports indicated that in recent years the cost of this therapy is the main driver of IBD health care, which represents a shift from previous drivers that included mainly hospitalizations and surgeries.
Furthermore, response rates to anti-TNF therapy are in the range of 60%, and remission rates are 30%.
Therefore, predictors of response would result in high benefits for patients and significant savings for public health care systems.
Despite the high efficacy of anti-TNF drugs, loss of response occurs in 30-50% of patients, and development of anti-drug antibodies is the main reason for loss of response.</p>
<p>If the drugs fail to contain the inflammation and heal the mucosa doctors might recommend to perform a surgery to remove the damaged section.
In some cases HSCT is recommended which have shown to improve the life of the patients <span class="citation">[<a href="#ref-corralizab" role="doc-biblioref">62</a>]</span>.
This is a new procedure given only to the most extreme cases for which several publications have checked that it resets the immunological state of the patient.</p>
<p>But are proposed to reset or change the intestinal microbiome of the patients to help them on the disease.
This has been doing experimentally on mice and mouse for some time and some experiments on patients resulted have been performed already.</p>
</div>
<div id="summary-ibd" class="section level3" number="1.1.4">
<h3><span class="header-section-number">1.1.4</span> Summary</h3>
<p>IBD is a complex disease that impacts the health of many people for long time and with lasting impact on their quality of life.</p>
<p>Current clinical care in some cases is enough to have a sustained clinical and endoscopic remission but most often is not enough and relapse is expected.
Several factors, such as becoming refractory to drugs, intermittent course of the disease and lack of validated predictors of disease course or response to therapy make the treatment complex.</p>
<p>Lack of knowledge of what are the factors cause of the disease make those treatments and drugs to be addressed to block further inflammation and damage, but cannot prevent it and often they do not stop it completely.</p>
</div>
</div>
<div id="integration-studies-on-ibd" class="section level2" number="1.2">
<h2><span class="header-section-number">1.2</span> Integration studies on IBD</h2>
<p>Many studies have looked up to the origin of the disease.
As seen, one of the hypothesis behind the maintenance of the inflammation involves the microbiome and the host epithelium.
This has been studied using several data sources, mostly from sequencing data.
The technical methods used to obtain the data of the inflamed tissue differ between extracted from biopsied samples at colonoscopy or from surgical samples.</p>
<p>Those samples are usually used later on to diagnose or for research purpose.
To obtain research-quality data it usually imply using techniques such as immunohistochemistry, histopathology, immunohistochemistry, fluorescence in situ hybridization and polymerase chain reaction.
These techniques allow to measure or visualize where are the cells expressing certain proteins or genes, thus helping with the analysis validation.</p>
<p>Furthermore several studies have been carried out to discover links between microbiome and the inflammation, followed by those looking for some relationship between genetics and the disease and more recently the metabolome.
These studies, known as integration, multi-omic or interaction studies, usually use multiple sequencing assays as the bases of the analysis <span class="citation">[<a href="#ref-beck2021" role="doc-biblioref">63</a>]</span>.</p>
<p>However, confirming causal interactions of the variables of each essay is difficult.
To find relationships some articles use correlation, like <span class="citation">[<a href="#ref-tang_integrated_2017" role="doc-biblioref">67</a>]</span> and network integrations.</p>
<p>Very rarely there is an experimental confirmation of the relationships between variables of the different essays because it is complicated to test an interaction and to set up the right conditions for the many variables that are accounted for on the integrations.
One of the few methods published that shows an interaction between genes and microorganisms on the IBD is to expose the <em>ex-vivo</em> sample or cell lines with microbiomes or supernatant of at their culture <span class="citation">[<a href="#ref-mayorgas2021" role="doc-biblioref">68</a>]</span>.</p>
<div id="data-origin" class="section level3" number="1.2.1">
<h3><span class="header-section-number">1.2.1</span> Data origin</h3>
<p>According to the data used, we can classify the studies:</p>
<div id="transcriptome" class="section level4" number="1.2.1.1">
<h4><span class="header-section-number">1.2.1.1</span> Transcriptome</h4>
<p>Most of the integrations refer some other source of data to the transcriptomics of the patient.
The transcriptome of the patients has been extensively studied since the existence of microarrays.
There are known marker gens of inflammation and many research focus on identifying prognosis predictors, treatment response prediction based on gene expression <span class="citation">[<a href="#ref-planell2013" role="doc-biblioref">69</a>, <a href="#ref-leal2015" role="doc-biblioref">70</a>]</span>.
Recently single-cell technology has enabled to estimate cell populations of the samples with better degree of success than bulk RNA-sequencing, which is helpful to understand the immune response and the population change of the cells.
Single cell technologies are starting to be used for integration.</p>
<p>Notably this is probably the most common data available from IBD patients, probably due to existing commercial solutions to gather this data and the experience with this kind of protocols.</p>
</div>
<div id="microbiome-1" class="section level4" number="1.2.1.2">
<h4><span class="header-section-number">1.2.1.2</span> Microbiome</h4>
<p>Many of the integration analysis on inflammatory bowel are done between host transcriptome and the microbiome.
These studies use datasets from IBD patients usually stratified by some inflammatory index or location of the disease.
Most of them are based on correlation analysis between the microbiome and RNA-seq <span class="citation">[<a href="#ref-hasler_uncoupling_2016" role="doc-biblioref">64</a>]</span>.
Conclusions of these integrations range from finding differences on the correlation depending on the disease (<span class="citation">[<a href="#ref-hasler_uncoupling_2016" role="doc-biblioref">64</a>]</span>) to finding relationships with inflammatory genes <span class="citation">[<a href="#ref-tang_integrated_2017" role="doc-biblioref">67</a>]</span>.</p>
</div>
<div id="genetics-1" class="section level4" number="1.2.1.3">
<h4><span class="header-section-number">1.2.1.3</span> Genetics</h4>
<p>Genetics is the next most common data source used to integrate data on the IBD.
Most studies on genetics and IBD are genome-wide association studies <span class="citation">[<a href="#ref-kumar2019" role="doc-biblioref">12</a>, <a href="#ref-huWholeExomeSequencing2021" role="doc-biblioref">66</a>, <a href="#ref-knights2013" role="doc-biblioref">71</a>]</span>.
No clear mechanism has been proved despite clear and significant findings, probably because IBD relapses and remits with time.</p>
<p>When using genetic data to integrate it with transcriptomics it is usually to understand how a genetic variant is affecting a gene expression.
This has lead to expression quantitative trait loci (eQTL).</p>
</div>
<div id="metabolome" class="section level4" number="1.2.1.4">
<h4><span class="header-section-number">1.2.1.4</span> Metabolome</h4>
<p>More recently there have been an increased interest on metabolome, given that microorganisms interact with the host also via their products and metabolites.
Some studies have integrated the metabolome with the RNAseq and state of the epithelium <span class="citation">[<a href="#ref-ahmed2016" role="doc-biblioref">72</a>, <a href="#ref-gallagher2021" role="doc-biblioref">73</a>]</span>.</p>
<p>There is evidence some of these metabolomic products, such as small chemicals regulate the epithelium cells and their interaction with the microbiome and outer agents <span class="citation">[<a href="#ref-mayorgas2021" role="doc-biblioref">68</a>, <a href="#ref-ferrer-picón2020b" role="doc-biblioref">74</a>]</span>.</p>
</div>
</div>
<div id="summary-studies" class="section level3" number="1.2.2">
<h3><span class="header-section-number">1.2.2</span> Summary</h3>
<p>Multiple methods and multiple studies have been used to learn more about the interaction between different data and understand the IBD.
The task is challenging and with many different aspects one could use.</p>
<p>There are many studies focused on understanding different aspects of the disease: the relation between location and severity, relapse, remission, treatments, origin.
The integration of data might help to improve the medicine and reveal links in difficult diseases like IBD.</p>
</div>
</div>
<div id="integration" class="section level2" number="1.3">
<h2><span class="header-section-number">1.3</span> Integration</h2>
<p>Data integration is informally widely used with varying definitions, according to the dictionary integration is defined as:</p>
<blockquote>
<p>“the process of combining two or more things into one” — <a href="https://dictionary.cambridge.org/dictionary/english/integration">Cambridge Dictionary</a></p>
</blockquote>
<p>Other words used are integrati(-on, -ve), and if specific to data from sequencing technologies multi-omics, pluri-omics.
Here integration will be used as it is the more general one and not restricted to omics or sequencing technologies.</p>
<p>Since the beginning of the integration methods there have been many methods proposed <span class="citation">[<a href="#ref-krassowski2020" role="doc-biblioref">75</a>]</span>.
Some of the early methods were initially used for surveying the agreement of different evaluating systems, others were developed for agricultural sciences <span class="citation">[<a href="#ref-hotelling_relations_1936" role="doc-biblioref">76</a>]</span> or food industry <span class="citation">[<a href="#ref-biancolillo" role="doc-biblioref">77</a>]</span>.
Some of these methods are specific for one application or data while others are more generally applicable.
Lately, the increase of bigger datasets with more variables and often from the same samples has increased the focus of the research community on the methodologies available on several disciplines but mainly on the biological science.
The explosion of data on the biological science has been driven by the new sequencing technologies that allow to measure thousand of variables of many samples at the same time, if done with multiple sequencing tecnologies it is usually referred as multi-omics methods, which usually only integrate omic data.</p>
<p>It has been observed the importance of classification, review and comparison of the tools available, as well as, benchmarking these tools against the same dataset as a way to provide clear recommendations to anyone wishing to use them <span class="citation">[<a href="#ref-cavill_transcriptomic_2016" role="doc-biblioref">80</a>]</span>.
Following this view I’ll review the integration methods according to several axis: type of data used, aim of the method, relationships between variables, relationships between samples, relationship between variables and samples, input data, mathematical framework and results of the method.</p>
<div id="classification-of-integration-methods" class="section level3" number="1.3.1">
<h3><span class="header-section-number">1.3.1</span> Classification of integration method’s</h3>
<p>Integration methods have very different properties that allow them to be classified and compared.
I classified them in the bioscience field, which means that they are meant to be used with omics datasets, with references to concrete methodology and in occasionaly to articles using them.</p>
<div id="data-type-numeric-or-categorical" class="section level4" number="1.3.1.1">
<h4><span class="header-section-number">1.3.1.1</span> Data type: numeric or categorical</h4>
<p>The most important distinction in integration methods is what kind of data are combined.
In general data can be divided between categorical and numeric variables, which are usually found in several fields.
Sometimes doctors want to understand the relationship between a phenotype they observe and the underlying mechanism.
Usually this involves looking how the metabolites, the gene expression, the methylation, the number of variants a gene has, and other numeric variables are related to the observed phenotype (like pain).</p>
<p>Depending on what does a method aim for it handles both data types or just one, but often they are used differently.
The most common way to handle different type of data is converting the categorical values to a mock or dummy variable.
For each categorical factor there is a new variable whose value is 1 if that sample had this factor and 0 otherwise.
For instance, if the categorical variable has three values (A, B, C) it would be converted to A (1, 0, 0) B (0, 1, 0) and C (0, 0, 1).
Often the number of variables created is one less than the number of factors that existed, on this example only A and B would be kept.
This transformation allows to use the categorical values in methods though for numeric variables.</p>
<p>If the method only accepts categorical data but you want to provide numeric values usually those values are categorized.
For example if a variable is (0.123, 0.25, 0.56, 0.78) one could make to categorical values like (“&lt;0.5”, “&lt;0.5”, “&gt;0.5”, “&gt;0.5”).
The number of categories to use and how is the numeric value split depends on case by case.</p>
<p>Very rarely methods allow to use both of them as they are.
If they allow so, it is usually for classification purposes.</p>
</div>
<div id="objective" class="section level4" number="1.3.1.2">
<h4><span class="header-section-number">1.3.1.2</span> Objective</h4>
<p>The objective of the method is one of its most important defining properties.
Data integration can be classified according to their objective and the (biological) question they try to answer.
Most of the times one (or several) of the following results are expected from integration methods:</p>
<ul>
<li><p>Classification of the patients or samples</p>
<p>As seen one of the purposes of the integration might be to use multiple sources of data to accurately describe how do samples or patients fit on a predefined possibles states.
This might be to accurately describe if a patient has one or other related disease of a possible subset of diseases given their condition.<br />
</p></li>
</ul>
<!-- -->
<ul>
<li>An overview of the role of each individual block in a biological system</li>
</ul>
<p>Sometimes the question is which omic method is the best describing the disease.
This could prevent peforming expensive tests that can be estimated by other cheaper or easier technique that has enough predictive power or is enough sensitive and specific for the task.<br />
An example of this is the search of markers on blood to identify if there are cancer cells on tissues.</p>
<ul>
<li>A better understanding of the relationships between the ’omics types</li>
</ul>
<p>Several methods are used to understand better how the different data sets interact.
When a potentially relationship between omics is disvoered then other focused methods are used to understand the mechanistic relationship between them.<br />
For instance, checking that in a particular case or condition is present on a given relationship, and that this relationships follows our model or not.</p>
<ul>
<li>Finding a molecular signature.</li>
</ul>
<p>A signature is usually a group of features that describe/are representative of a cell line, a process or a stage.
Identifying a subset of the variables from the omics that are related is often a desired goal because it reduces the amount of variables allowing to perform experiments on the bench on just those that might be important.
In other fields, such as machine learning, selecting the important variables is known as <a href="https://en.wikipedia.org/wiki/Feature_selection" title="Wikipedia page of feature selection">feature selection</a>.
There are several methods that are used to do this.<br />
An example of this is when performing eQTL, where a locus is related to the expression change of genes, so finding the right positions that induce a specific expression level is a very common task.</p>
<ul>
<li>A predictive model</li>
</ul>
<p>Predictive models usually require a very good understanding of the current and/or past relationships, as well as, a good feature selection procedure.
If a good model on previous data exists it might be used to predict future events.
Sometimes, models are only build to predict events without being able to accurately describe the mechanism, why they work.<br />
This kind of methods are used improve treatments, diagnosis and prognosis prediction.</p>
<ul>
<li>Impute values</li>
</ul>
<p>Some methods aim to accurately guess which values have one block given some other information.
Missing values can happen for a variety of reasons from practical ones, like a sample not being available, to technical ones, such as laboratory method failing.
However, this is often a intermediate step to other goals.</p>
<p>To complete these goals it is important to have enough statistical power to determine the significance of tests performed (if any) and to understand how complete are the data sources used on the integration <span class="citation">[<a href="#ref-tarazona2021" role="doc-biblioref">81</a>]</span>.
Having more statistical power helps identifying the relationships one seeks when using this methods.</p>
<p>Formally the objective of methods might be defined as a function to be optimized or a procedure to be followed until certain conditions are meet between the variables received as input.</p>
</div>
<div id="relationship-between-variables-and-samples" class="section level4" number="1.3.1.3">
<h4><span class="header-section-number">1.3.1.3</span> Relationship between variables and samples</h4>
<p>Depending on the amount of variables and in which samples have been measured studies can be classiffied in two types of integration.
Traditionally for each sample few variables are measured, for instance on a biopsy with RT-PCR only a few genes are measured, however with the new omics techniques (transcriptomics, metabolomics, methylomics, genomics), thousands of variables are measured for the same sample.
This has lead to the following situation:</p>
<ul>
<li>More variables than samples</li>
</ul>
<p>For a single sample of RNA around 50k genome identifiers (genes, long non coding RNAs, iRNA, pseudogenes,…) can be measured.
Which leads to the case where there are many more variables than samples.
Thus high-throughput data analysis typically falls into the category of <span class="math inline">\(p \gg n\)</span> problems ( big p, little n), where the number of genes or proteins, <span class="math inline">\(p\)</span>, is considerably larger than the number of samples, <span class="math inline">\(n\)</span>.
With such high number of variables the identification of the relevant variables is hard because variables will co-variate.
When many variables are tightly correlated, discovering which one is important using just numerical methods is very difficult.
It is even more difficult when looking for causal relationships.</p>
<ul>
<li>More samples than variables</li>
</ul>
<p>This was the usual case when for instance, when from a cohort of patients the temperature is measured along the stage of a disease: two variables for each sample.
If there are more than 2 patients, then the number of samples is greater than the number of variables studied.
This is described in the literature as <span class="math inline">\(n \gg p\)</span> (or big n, little p).
Nowadays this is less frequent on the bioscience world, and does not causes trouble analyzing it because the high number of samples allow to accurately estimate variables’ dispersion.</p>
<p>To estimate the number of samples required there are several methods available <span class="citation">[<a href="#ref-tarazona2020" role="doc-biblioref">82</a>]</span>.
Having just the enough amount of samples for the desired statistical power is not enough if the samples of different blocks do not match between them.</p>
</div>
<div id="relationship-between-samples" class="section level4" number="1.3.1.4">
<h4><span class="header-section-number">1.3.1.4</span> Relationship between samples</h4>
<p>Depending on the relationship between the samples of the different blocks of data, the questions answerable and the methods that work on them differ.</p>
<p>A sample can have multiple or one data source.
In a study if all the samples have all the data from multiple data sources it is a complete case.
If some samples have data from some data sources but not from others the study is not a with a complete case.</p>
<p>Sometimes because the sample is not enough, or there are some technical or organizational problems a source of data for a sample (which is known as an incomplete case) might be lost.
This results in a new source of variation that has to be dealt with, which complicates the conclusion one can draw from the studies of these kind of data.</p>
<p>Even when all the cases of a patient are complete the samples can come from several sites of the same individual or with different combinations of variables, which makes is relevant to understand the relationships between the different samples.</p>
<p>There is no easy classification of this as each experiment might be designed differently.
In general, experiments are designed to be as consistent as possible but in face of adverse events that become a variation of the design the analysis complicates.
Either some data is imputed or some samples are omitted for the analysis, or if the differences are minor these samples are used as being from a different group.
This can happen with samples taken at different timepoints as patients might not come to the visit with the doctor on time.</p>
<div id="time" class="section level5 unnumbered">
<h5>Time</h5>
<p>As mentioned above, time is one of the factors that sometimes cannot be controlled, despite having programmed visits every two weeks some patients might come early or later due to multiple reasons (holidays, other conflicting duties, …).</p>
<p>Sometimes, precisely the objective of the study is to analyze the relationships at different time, or see how the relationships change with time.
To discover causality between two variables the cause must be before the consequence, which highlights the importance of time.
Being aware of the time differences and time scales is crucial in most cases.</p>
<p>On cell lines or other lab experiments conditions can be repeated even if they are at different time points.
However, with patients there are not biological replicates like with cell-lines.
This makes it harder to study time-related change on patients.</p>
<p>Lastly, time between the collection of a fresh samples and its processing also influences the readings of the samples of the omics technology, specially RNA-seq <span class="citation">[<a href="#ref-massoni-badosa2020" role="doc-biblioref">83</a>, <a href="#ref-zhu2017" role="doc-biblioref">84</a>]</span>.
Some features and genes are more affected by time than others but as they are measured at the same time it might distort the data.
Processing time is also very hard to keep track on practical settings and requires a highly coordinated effort <span class="citation">[<a href="#ref-ferreira2018" role="doc-biblioref">85</a>]</span>.</p>
</div>
</div>
<div id="relationship-between-variables" class="section level4" number="1.3.1.5">
<h4><span class="header-section-number">1.3.1.5</span> Relationship between variables</h4>
<p>Once the data is collected, the next step is understand the relationship between the variables present.
As seen, some variables influence other variables which affects the outcome in a complex ways.
With many variables present in a dataset it is important to be aware of known relationships between variables.
Even in a simple dataset, like a RNA-seq dataset, it is important to be aware of the relationships between variables.</p>
<p>Since the discovery of the lactose <a href="https://en.wikipedia.org/wiki/Operon">operon</a> it is known how some genes regulate each other.
It is no know how other variables are related between them.
For instance, how does the increase in expression of a gene affects the growth of a microorganism?
Usually the relationships between variables are mediated by many factors or interactions.</p>
<p>One of the best example of such interactions is when some variables correlate.
Their correlation can be used to reduce the number of variables being analyzed by ignoring the relationships between them and using the most representative variable (less correlated and with more variation).
This step is usually done by the dimension reduction methods… However, sometimes this is not desired or feasible as the correlation does not explain the direction of the causality of the interaction between the variables (if there is any).</p>
<p>Network approaches relate the variables between them (such as <span class="citation">[<a href="#ref-koh_iomicspass_2019" role="doc-biblioref">86</a>]</span>).
These approaches are fairly new and growing in popularity partly because they can address the direction of the interaction.</p>
<p>In partial correlations some or all of the other variables are considered on how much do they affect the others and this effect is reduced.
This assumes a linear relationship and is computationally expensive when there are thousands of variables.</p>
</div>
<div id="input-data" class="section level4" number="1.3.1.6">
<h4><span class="header-section-number">1.3.1.6</span> Input data</h4>
<p>We have classified the studies according to the data they use (as <a href="introduction.html#data-origin">1.2.1</a> ).
But, some methods to account for relationships of variables only work when data is from the same patients on each data set used to integrate while other do not:</p>
<ul>
<li>Data from the same samples:</li>
</ul>
<p>These methods do not handle well or at all missing data.
They need complete cases/data of the samples in order to be able to integrate the results.
These methods include Regularized Generalized Canonical Correlation Analysis (RGCCA) <span class="citation">[<a href="#ref-tenenhaus_regularized_2011" role="doc-biblioref">87</a>, <a href="#ref-tenenhaus_variable_2014" role="doc-biblioref">88</a>]</span>, Multiple co-inertia analysis (MCIA) <span class="citation">[<a href="#ref-culhane_cross-platform_2003" role="doc-biblioref">89</a>]</span>, Multi-Study Factor Analysis (MSFA) <span class="citation">[<a href="#ref-vito_multi-study_2019" role="doc-biblioref">90</a>]</span>, Multi-Omics Factor Analysis (MOFA) <span class="citation">[<a href="#ref-argelaguet_multi-omics_2018" role="doc-biblioref">91</a>]</span>, STATegRa <span class="citation">[<a href="#ref-gomez-cabreroSTATegraComprehensiveMultiomics2019" role="doc-biblioref">92</a>]</span>.</p>
<ul>
<li>Data from different samples:</li>
</ul>
<p>These methods do not need data from the same sample.
They draw their conclusions generalizing from the data available.
Some of them handle missing data, while others do use the data at face value.
These method includes MetaPhlAn2 <span class="citation">[<a href="#ref-franzosaSpecieslevelFunctionalProfiling2018" role="doc-biblioref">93</a>]</span>, HUMAnN,<span class="citation">[<a href="#ref-truongMetaPhlAn2EnhancedMetagenomic2015" role="doc-biblioref">94</a>]</span>, LEfSe <span class="citation">[<a href="#ref-segataMetagenomicBiomarkerDiscovery2011" role="doc-biblioref">95</a>]</span>.</p>
<p>Furthermore, some methods are designed to integrate specific types of datasets, (usually because they make some assumptions that are only met on that kind of data).
For instance, HCG, 16S, RNA-seq and metabolomics do not share the same data distribution, and are different between them.
Also even with the same data depending on the processing of the data they can have very different properties: OTUs (operational taxonomic unit) properties are not the same as ASV (amplicon sequence variants).</p>
</div>
<div id="mathematical-framework" class="section level4" number="1.3.1.7">
<h4><span class="header-section-number">1.3.1.7</span> Mathematical framework</h4>
<p>Depending on the input and the objective methods use different mathematical framework to process the data.
Some of them have previously appeared:</p>
<ul>
<li><p>Networks</p>
<p>Networks methods were mentioned because they use and find information about interaction of variables.
Multilayer networks, including the multiplex, Molti-C-DREAM<span class="citation">[<a href="#ref-didier2018" role="doc-biblioref">96</a>]</span>, RWR-MH, RWR-M <span class="citation">[<a href="#ref-valdeolivas2019" role="doc-biblioref">97</a>]</span>.
Network embedding MultiVERSE are some of the methods using networks <span class="citation">[<a href="#ref-pio-lopez2021" role="doc-biblioref">98</a>]</span>.</p>
<p>Bayesian approaches are also quite frequent, these methods use the Bayes theorem to see the relationships between variables.
Some methods that use these approaches are <a href="http://research.mssm.edu/integrative-network-biology/RIMBANET/RIMBANET_overview.html">RIMBANET</a> <span class="citation">[<a href="#ref-zhu2012" role="doc-biblioref">99</a>]</span>, <a href="https://github.com/ttriche/bayesCC">BCC</a> <span class="citation">[<a href="#ref-lock2013" role="doc-biblioref">100</a>]</span> and others like <span class="citation">[<a href="#ref-virtanen2012" role="doc-biblioref">101</a>]</span>.</p></li>
<li><p>Dimensional Reduction</p>
<p>These methods focus on finding just a few variables and summarizing them using some function that has some properties.
The selection of variables is usually done with L1 or Lasso Regression regularization technique or L2 also known as Ridge Regression.
L1-regularization adds a penalty equal to the absolute value of the magnitude of coefficients which might leads to some coefficients becoming zero and the variable eliminated from the model.
On the other hand, L2-regularization does not result in elimination of coefficients or sparse models and can only be used when there is multicollinearity as it works well to avoid over-fitting.</p>
<p>Several methods use this approach: Momix <span class="citation">[<a href="#ref-cantini_benchmarking_2020" role="doc-biblioref">102</a>]</span>, RGCCA <span class="citation">[<a href="#ref-tenenhaus2017" role="doc-biblioref">103</a>]</span>, mixOmics <span class="citation">[<a href="#ref-rohart2017" role="doc-biblioref">104</a>]</span>, STATegRa <span class="citation">[<a href="#ref-gomez-cabrero2019" role="doc-biblioref">105</a>]</span>, …</p></li>
<li><p>Active module identification</p>
<p>Multiomic objective genetic algorithm (scores based in two metrics; node score and density of interactions score).
An example of a method using this approach is MOGAMUN <span class="citation">[<a href="#ref-novoa-del-toro2020" role="doc-biblioref">106</a>]</span></p></li>
</ul>
<p>Usually depending on the mathematical framework used they return these methods return similar output.</p>
</div>
<div id="output-results" class="section level4" number="1.3.1.8">
<h4><span class="header-section-number">1.3.1.8</span> Output results</h4>
<p>According to the output the integration methods can be classified in several groups:</p>
<p>For the network methods the following output is usually returned: Connections between the variables/nodes, a measure of how strong is the connection (or simply if there is a connection or if there is not one).</p>
<p>For those from dimensional reduction methods there are three: Shared factor across the data, specific factors for each data or mixed factors.</p>
<ul>
<li>Shared factors:</li>
</ul>
<p>The integration results in a vector of the samples in a lower dimensional space that is shared by all the data used to integrate.
Such methods include iCluster, Multi-Omics Factor Analysis (MOFA) <span class="citation">[<a href="#ref-argelaguet_multi-omics_2018" role="doc-biblioref">91</a>]</span>.</p>
<ul>
<li>Specific factors:</li>
</ul>
<p>The integration results in several vectors of the samples in a lower dimensional space of each data used to integrate.
Such methods include Regularized Generalized Canonical Correlation Analysis (RGCCA) <span class="citation">[<a href="#ref-tenenhaus_regularized_2011" role="doc-biblioref">87</a>, <a href="#ref-tenenhaus_variable_2014" role="doc-biblioref">88</a>]</span>, Multiple co-inertia analysis (MCIA) <span class="citation">[<a href="#ref-culhane_cross-platform_2003" role="doc-biblioref">89</a>]</span>, Multi-Study Factor Analysis (MSFA) <span class="citation">[<a href="#ref-vito_multi-study_2019" role="doc-biblioref">90</a>]</span>.</p>
<ul>
<li>Mixed factors:</li>
</ul>
<p>The integration results in both previous factors, specific of each data and common to all the data.
Such methods include Joint and Individual Variation Explained (JIVE), integrative Non-negative Matrix Factorization (<a href="https://github.com/yangzi4/iNMF">iNMF</a>) <span class="citation">[<a href="#ref-yang2016" role="doc-biblioref">107</a>]</span>.</p>
<p>These methods are used to understand and discuss the results with previous knowledge or independent sources of data.</p>
</div>
</div>
<div id="interpretation" class="section level3" number="1.3.2">
<h3><span class="header-section-number">1.3.2</span> Interpretation</h3>
<p>How to interpret the results of the methods is highly linked to understanding the method and its output.
On a correlation between two variables, the interpretation of the analysis is clear, if one variable increase, the other one too.
The implications of these observation can be far reaching but the principles to understand them are simple.</p>
<p>However, on more complex methods the interpretation becomes less clear.
The interpretation of a canonical correlation analysis is much harder <span class="citation">[<a href="#ref-sherry2005" role="doc-biblioref">108</a>]</span>.
Also on more complex methods the number of parameters required increases so the time and intellectual effort to understand the relationships between the parameters is also higher.</p>
<p>The interpretation also helps to discuss the results and relate it to other previously know information.</p>
<ul>
<li>Individually:</li>
</ul>
<p>How each variable relates to another, like in the correlation analysis, the relationship between two variables under study.
Or by patient: how do interpret that in these patient variable A and B is X and Y?</p>
<ul>
<li>Globally:</li>
</ul>
<p>In a principal component analysis for instance how is interpreted that some variables have the same loading?
What happens in a more difficult method like canonical correlation analysis?</p>
<p>There have been some articles about how to interpret those methods on real datasets <span class="citation">[<a href="#ref-chung_multi-omics_2019" role="doc-biblioref">111</a>]</span>.
Which is used to compare the results of the integration with the dataset of interest and to compare different tools.
These datasets are created with some relationships that the tools are expected to find.</p>
<p>There exists several methods to create synthetic datasets like MOSim <span class="citation">[<a href="#ref-haberman2019" role="doc-biblioref">112</a>]</span>, metaSPARSim <span class="citation">[<a href="#ref-patuzzi2019" role="doc-biblioref">113</a>]</span>, CAMISIM <span class="citation">[<a href="#ref-fritz2019" role="doc-biblioref">114</a>]</span>, ballgown <span class="citation">[<a href="#ref-fu2021" role="doc-biblioref">115</a>]</span>, polyester <span class="citation">[<a href="#ref-frazee2021" role="doc-biblioref">116</a>]</span> and even edgeR <span class="citation">[<a href="#ref-mccarthy2012" role="doc-biblioref">117</a>]</span> can be used.
These methods are useful to compare different setup and methods but they can miss some subtle not previously reported relations on real data.</p>
</div>
<div id="reviews" class="section level3" number="1.3.3">
<h3><span class="header-section-number">1.3.3</span> Reviews</h3>
<p>The comparison and review of methods independently from original authors have become a crucial step for selecting the right tool for a research <span class="citation">[<a href="#ref-cantini_benchmarking_2020" role="doc-biblioref">102</a>]</span>.</p>
<p>Some of these reviews are focused on a specific type of data integration: metabolomics <span class="citation">[<a href="#ref-de_souza_ibd_2017" role="doc-biblioref">121</a>]</span>.
On this regard there are several efforts to integrate data on IBD but no comprehensive review to date is known to the author.</p>
</div>
<div id="methods-integration" class="section level3" number="1.3.4">
<h3><span class="header-section-number">1.3.4</span> Summary</h3>
<p>The field of integration is large and complex, with high interest over the last few years, specially in the psychology and omics field.
As a methodology they are quite complex and diverse but there is a growing interest on them to help answer complex questions without using other complex tools like deep neuronal networks or other machine learning approches (despite not being incompatible).</p>
<p>Methods to integrate have many characteristics, depending on the objectives and data that available.
Regardless of the method used, interpretation and reporting is usually a main problem when using any of these methods as there are less people familiar with it.</p>
<p>See table below for a list of existing methods<a href="#fn2" class="footnote-ref" id="fnref2"><sup>2</sup></a>:</p>
<table>
<caption>Integration methods available and their references.</caption>
<thead>
<tr class="header">
<th align="center">Method</th>
<th align="center">Publication</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="center">SCCA</td>
<td align="center"><span class="citation">[<a href="#ref-parkhomenko2009" role="doc-biblioref">122</a>]</span></td>
</tr>
<tr class="even">
<td align="center">PCCA</td>
<td align="center"><span class="citation">[<a href="#ref-waaijenborg2008" role="doc-biblioref">123</a>]</span></td>
</tr>
<tr class="odd">
<td align="center">PMA</td>
<td align="center"><span class="citation">[<a href="#ref-witten2009" role="doc-biblioref">124</a>]</span></td>
</tr>
<tr class="even">
<td align="center">sPLS</td>
<td align="center"><span class="citation">[<a href="#ref-lêcao2009" role="doc-biblioref">125</a>]</span></td>
</tr>
<tr class="odd">
<td align="center">gesca</td>
<td align="center"><span class="citation">[<a href="#ref-hwang2009" role="doc-biblioref">126</a>]</span></td>
</tr>
<tr class="even">
<td align="center">Regularized dual CCA</td>
<td align="center"><span class="citation">[<a href="#ref-soneson2010" role="doc-biblioref">127</a>]</span></td>
</tr>
<tr class="odd">
<td align="center">RGCCA</td>
<td align="center"><span class="citation">[<a href="#ref-tenenhaus_regularized_2011" role="doc-biblioref">87</a>]</span></td>
</tr>
<tr class="even">
<td align="center">SNMNMF</td>
<td align="center"><span class="citation">[<a href="#ref-zhang2011" role="doc-biblioref">128</a>]</span></td>
</tr>
<tr class="odd">
<td align="center">scca</td>
<td align="center"><span class="citation">[<a href="#ref-lee2011" role="doc-biblioref">129</a>]</span></td>
</tr>
<tr class="even">
<td align="center">STATIS</td>
<td align="center"><span class="citation">[<a href="#ref-abdi2012" role="doc-biblioref">130</a>]</span></td>
</tr>
<tr class="odd">
<td align="center">joint NMF</td>
<td align="center"><span class="citation">[<a href="#ref-zhang2012" role="doc-biblioref">131</a>]</span></td>
</tr>
<tr class="even">
<td align="center">sMBPLS</td>
<td align="center"><span class="citation">[<a href="#ref-li2012" role="doc-biblioref">132</a>]</span></td>
</tr>
<tr class="odd">
<td align="center">Bayesian group factor analysis</td>
<td align="center"><span class="citation">[<a href="#ref-virtanen2012a" role="doc-biblioref">133</a>]</span></td>
</tr>
<tr class="even">
<td align="center">RIMBANET</td>
<td align="center"><span class="citation">[<a href="#ref-zhu2012a" role="doc-biblioref">134</a>]</span></td>
</tr>
<tr class="odd">
<td align="center">FactoMineR</td>
<td align="center"><span class="citation">[<a href="#ref-abdi2013" role="doc-biblioref">135</a>]</span></td>
</tr>
<tr class="even">
<td align="center">JIVE</td>
<td align="center"><span class="citation">[<a href="#ref-lock2013a" role="doc-biblioref">136</a>]</span></td>
</tr>
<tr class="odd">
<td align="center">pandaR</td>
<td align="center"><span class="citation">[<a href="#ref-schlauch2017" role="doc-biblioref">137</a>]</span></td>
</tr>
<tr class="even">
<td align="center">omicade4</td>
<td align="center"><span class="citation">[<a href="#ref-meng2014a" role="doc-biblioref">138</a>]</span></td>
</tr>
<tr class="odd">
<td align="center">STATegRa</td>
<td align="center"><span class="citation">[<a href="#ref-planell2021a" role="doc-biblioref">139</a>]</span></td>
</tr>
<tr class="even">
<td align="center">Joint factor model</td>
<td align="center"><span class="citation">[<a href="#ref-ray2014" role="doc-biblioref">140</a>]</span></td>
</tr>
<tr class="odd">
<td align="center">GFAsparse</td>
<td align="center"><span class="citation">[<a href="#ref-bunte2016" role="doc-biblioref">141</a>]</span></td>
</tr>
<tr class="even">
<td align="center">Sparse CCA</td>
<td align="center"><span class="citation">[<a href="#ref-chen2013" role="doc-biblioref">142</a>]</span></td>
</tr>
<tr class="odd">
<td align="center">CCAGFA</td>
<td align="center"><span class="citation">[<a href="#ref-leppäaho2017" role="doc-biblioref">143</a>]</span></td>
</tr>
<tr class="even">
<td align="center">CMF</td>
<td align="center"><span class="citation">[<a href="#ref-klami2014" role="doc-biblioref">144</a>]</span></td>
</tr>
<tr class="odd">
<td align="center">MOGSA</td>
<td align="center"><span class="citation">[<a href="#ref-meng2018" role="doc-biblioref">145</a>]</span></td>
</tr>
<tr class="even">
<td align="center">iNMF</td>
<td align="center"><span class="citation">[<a href="#ref-yang2016b" role="doc-biblioref">146</a>]</span></td>
</tr>
<tr class="odd">
<td align="center">BASS</td>
<td align="center"><span class="citation">[<a href="#ref-zhao2015" role="doc-biblioref">147</a>]</span></td>
</tr>
<tr class="even">
<td align="center">imputeMFA</td>
<td align="center"><span class="citation">[<a href="#ref-voillet2016" role="doc-biblioref">148</a>]</span></td>
</tr>
<tr class="odd">
<td align="center">PLSCA</td>
<td align="center"><span class="citation">[<a href="#ref-beaton2016" role="doc-biblioref">149</a>]</span></td>
</tr>
<tr class="even">
<td align="center">mixOmics</td>
<td align="center"><span class="citation">[<a href="#ref-singh2019" role="doc-biblioref">150</a>]</span></td>
</tr>
<tr class="odd">
<td align="center">mixedCCA</td>
<td align="center"><span class="citation">[<a href="#ref-yoon2019" role="doc-biblioref">151</a>]</span></td>
</tr>
<tr class="even">
<td align="center">SLIDE</td>
<td align="center"><span class="citation">[<a href="#ref-gaynanova2017a" role="doc-biblioref">152</a>]</span></td>
</tr>
<tr class="odd">
<td align="center">fCCAC</td>
<td align="center"><span class="citation">[<a href="#ref-madrigal2017" role="doc-biblioref">153</a>]</span></td>
</tr>
<tr class="even">
<td align="center">TSKCCA</td>
<td align="center"><span class="citation">[<a href="#ref-yoshida2017" role="doc-biblioref">154</a>]</span></td>
</tr>
<tr class="odd">
<td align="center">SMSMA</td>
<td align="center"><span class="citation">[<a href="#ref-kawaguchi2017" role="doc-biblioref">155</a>]</span></td>
</tr>
<tr class="even">
<td align="center">AJIVE</td>
<td align="center"><span class="citation">[<a href="#ref-feng2018" role="doc-biblioref">156</a>]</span></td>
</tr>
<tr class="odd">
<td align="center">MOFA</td>
<td align="center"><span class="citation">[<a href="#ref-argelaguet2020" role="doc-biblioref">157</a>]</span></td>
</tr>
<tr class="even">
<td align="center">PCA+CCA</td>
<td align="center"><span class="citation">[<a href="#ref-brown2018" role="doc-biblioref">158</a>]</span></td>
</tr>
<tr class="odd">
<td align="center">JACA</td>
<td align="center"><span class="citation">[<a href="#ref-zhang2020" role="doc-biblioref">159</a>]</span></td>
</tr>
<tr class="even">
<td align="center">iPCA</td>
<td align="center"><span class="citation">[<a href="#ref-tang2021" role="doc-biblioref">160</a>]</span></td>
</tr>
<tr class="odd">
<td align="center">pCIA</td>
<td align="center"><span class="citation">[<a href="#ref-min2019" role="doc-biblioref">161</a>]</span></td>
</tr>
<tr class="even">
<td align="center">sSCCA</td>
<td align="center"><span class="citation">[<a href="#ref-safo2018" role="doc-biblioref">162</a>]</span></td>
</tr>
<tr class="odd">
<td align="center">SWCCA</td>
<td align="center"><span class="citation">[<a href="#ref-min2017" role="doc-biblioref">163</a>]</span></td>
</tr>
<tr class="even">
<td align="center">OmicsPLS</td>
<td align="center"><span class="citation">[<a href="#ref-bouhaddani2018" role="doc-biblioref">164</a>]</span></td>
</tr>
<tr class="odd">
<td align="center">SCCA-BC</td>
<td align="center"><span class="citation">[<a href="#ref-pimentel2018" role="doc-biblioref">165</a>]</span></td>
</tr>
<tr class="even">
<td align="center">WON-PARAFAC</td>
<td align="center"><span class="citation">[<a href="#ref-kim2019" role="doc-biblioref">166</a>]</span></td>
</tr>
<tr class="odd">
<td align="center">BIDIFAC</td>
<td align="center"><span class="citation">[<a href="#ref-lock2020" role="doc-biblioref">167</a>]</span></td>
</tr>
<tr class="even">
<td align="center">maui</td>
<td align="center"><span class="citation">[<a href="#ref-ronen2019" role="doc-biblioref">168</a>]</span></td>
</tr>
<tr class="odd">
<td align="center">SmCCNet</td>
<td align="center"><span class="citation">[<a href="#ref-shi2019" role="doc-biblioref">169</a>]</span></td>
</tr>
<tr class="even">
<td align="center">msPLS</td>
<td align="center"><span class="citation">[<a href="#ref-csala2020" role="doc-biblioref">170</a>]</span></td>
</tr>
<tr class="odd">
<td align="center">MOTA</td>
<td align="center"><span class="citation">[<a href="#ref-fan2020" role="doc-biblioref">171</a>]</span></td>
</tr>
<tr class="even">
<td align="center">D-CCA</td>
<td align="center"><span class="citation">[<a href="#ref-shu2020" role="doc-biblioref">172</a>]</span></td>
</tr>
<tr class="odd">
<td align="center">COMBI</td>
<td align="center"><span class="citation">[<a href="#ref-hawinkel2020" role="doc-biblioref">173</a>]</span></td>
</tr>
<tr class="even">
<td align="center">DPCCA</td>
<td align="center"><span class="citation">[<a href="#ref-gundersen2020" role="doc-biblioref">174</a>]</span></td>
</tr>
<tr class="odd">
<td align="center">MEFISTO</td>
<td align="center"><span class="citation">[<a href="#ref-velten2020" role="doc-biblioref">175</a>]</span></td>
</tr>
<tr class="even">
<td align="center">MultiPower</td>
<td align="center"><span class="citation">[<a href="#ref-tarazona2020" role="doc-biblioref">82</a>]</span></td>
</tr>
<tr class="odd">
<td align="center">mixedCCA</td>
<td align="center"><span class="citation">[<a href="#ref-yoon2019a" role="doc-biblioref">176</a>]</span></td>
</tr>
</tbody>
</table>
<!--# Check https://github.com/mikelove/awesome-multi-omics -->

</div>
</div>
</div>
<h3>References</h3>
<div id="refs" class="references csl-bib-body">
<div id="ref-jairath2020" class="csl-entry">
1. Jairath V, Feagan BG. <a href="https://doi.org/10.1016/S2468-1253(19)30358-9">Global burden of inflammatory bowel disease</a>. The Lancet Gastroenterology &amp; Hepatology. 2020;5:2–3.
</div>
<div id="ref-burisch2015" class="csl-entry">
2. Burisch J, Munkholm P. <a href="https://doi.org/10.3109/00365521.2015.1014407">The epidemiology of inflammatory bowel disease</a>. Scandinavian Journal of Gastroenterology. 2015;50:942–51.
</div>
<div id="ref-human_microbiome_project_consortium_framework_2012" class="csl-entry">
3. Human Microbiome Project Consortium BA, Nelson KE, Pop M, Creasy HH, Giglio MG, Huttenhower C, et al. <a href="https://doi.org/10.1038/nature11209">A framework for human microbiome research.</a> Nature. 2012;486:215–21.
</div>
<div id="ref-mukhopadhya2012" class="csl-entry">
4. Mukhopadhya I, Hansen R, El-Omar EM, Hold GL. <a href="https://doi.org/10.1038/nrgastro.2012.14">IBD<span></span>what role do Proteobacteria play?</a> Nature Reviews Gastroenterology &amp; Hepatology. 2012;9:219–30.
</div>
<div id="ref-silva2016a" class="csl-entry">
8. Silva FAR, Rodrigues BL, Ayrizono M de LS, Leal RF. <a href="https://doi.org/10.1155/2016/2097274">The Immunological Basis of Inflammatory Bowel Disease</a>. Gastroenterology Research and Practice. 2016;2016:2097274.
</div>
<div id="ref-satsangi2006c" class="csl-entry">
10. Satsangi J, Silverberg MS, Vermeire S, Colombel J-F. <a href="https://doi.org/10.1136/gut.2005.082909">The montreal classification of inflammatory bowel disease: Controversies, consensus, and implications</a>. Gut. 2006;55:749–53.
</div>
<div id="ref-horowitz2021" class="csl-entry">
11. Horowitz JE, Warner N, Staples J, Crowley E, Gosalia N, Murchie R, et al. <a href="https://doi.org/10.1038/s41598-021-84938-8">Mutation spectrum of NOD2 reveals recessive inheritance as a main driver of early onset crohn<span>’</span>s disease</a>. Scientific Reports. 2021;11:5595.
</div>
<div id="ref-kumar2019" class="csl-entry">
12. Kumar M, Garand M, Al Khodor S. <a href="https://doi.org/10.1186/s12967-019-02174-1">Integrating omics for a better understanding of inflammatory bowel disease: A step towards personalized medicine</a>. Journal of Translational Medicine. 2019;17:419.
</div>
<div id="ref-jostins2012" class="csl-entry">
14. Jostins L, Ripke S, Weersma RK, Duerr RH, McGovern DP, Hui KY, et al. <a href="https://doi.org/10.1038/nature11582">Host<span></span>microbe interactions have shaped the genetic architecture of inflammatory bowel disease</a>. Nature. 2012;491:119–24.
</div>
<div id="ref-mcgovern2015" class="csl-entry">
15. McGovern DPB, Kugathasan S, Cho JH. <a href="https://doi.org/10.1053/j.gastro.2015.08.001">Genetics of Inflammatory Bowel Diseases</a>. Gastroenterology. 2015;149:1163–1176.e2.
</div>
<div id="ref-khanna2014" class="csl-entry">
16. Khanna S, Tosh PK. <a href="https://doi.org/10.1016/j.mayocp.2013.10.011">A Clinician’s Primer on the Role of the Microbiome in Human Health and Disease</a>. Mayo Clinic Proceedings. 2014;89:107–14.
</div>
<div id="ref-swidsinski2002" class="csl-entry">
17. Swidsinski A, Ladhoff A, Pernthaler A, Swidsinski S, LoeningBaucke V, Ortner M, et al. <a href="https://doi.org/10.1053/gast.2002.30294">Mucosal flora in inflammatory bowel disease</a>. Gastroenterology. 2002;122:44–54.
</div>
<div id="ref-tamboli2004" class="csl-entry">
18. Tamboli CP, Neut C, Desreumaux P, Colombel JF. <a href="https://doi.org/10.1136/gut.53.1.1">Dysbiosis in inflammatory bowel disease</a>. Gut. 2004;53:1–4.
</div>
<div id="ref-kostic2014" class="csl-entry">
21. Kostic AD, Xavier RJ, Gevers D. <a href="https://doi.org/10.1053/j.gastro.2014.02.009">The microbiome in inflammatory bowel disease: Current status and the future ahead</a>. Gastroenterology. 2014;146:1489–99.
</div>
<div id="ref-sankarasubramanian2020a" class="csl-entry">
22. Sankarasubramanian J, Ahmad R, Avuthu N, Singh AB, Guda C. <a href="https://doi.org/10.3389/fmed.2020.606298">Gut microbiota and metabolic specificity in ulcerative colitis and crohn’s disease</a>. Frontiers in Medicine. 2020;7.
</div>
<div id="ref-lopez-siles2014" class="csl-entry">
23. Lopez-Siles M, Martinez-Medina M, Busquets D, Sabat-Mir M, Duncan SH, Flint HJ, et al. <a href="https://doi.org/10.1016/j.ijmm.2014.02.009">Mucosa-associated Faecalibacterium prausnitzii and Escherichia coli co-abundance can distinguish Irritable Bowel Syndrome and Inflammatory Bowel Disease phenotypes</a>. International journal of medical microbiology: IJMM. 2014;304:464–75.
</div>
<div id="ref-ferrer-picón2020" class="csl-entry">
24. Ferrer-Picón E, Dotti I, Corraliza AM, Mayorgas A, Esteller M, Perales JC, et al. <a href="https://doi.org/10.1093/ibd/izz119">Intestinal Inflammation Modulates the Epithelial Response to Butyrate in Patients With Inflammatory Bowel Disease</a>. Inflammatory Bowel Diseases. 2020;26:43–55.
</div>
<div id="ref-darfeuille-michaud1998" class="csl-entry">
25. Darfeuille-Michaud A, Neut C, Barnich N, Lederman E, Di Martino P, Desreumaux P, et al. <a href="https://doi.org/10.1016/S0016-5085(98)70019-8">Presence of adherent Escherichia coli strains in ileal mucosa of patients with Crohn’s disease</a>. Gastroenterology. 1998;115:1405–13.
</div>
<div id="ref-okumura2017" class="csl-entry">
26. Okumura R, Takeda K. <a href="https://doi.org/10.1038/emm.2017.20">Roles of intestinal epithelial cells in the maintenance of gut homeostasis</a>. Experimental &amp; Molecular Medicine. 2017;49:e338–8.
</div>
<div id="ref-faria2012" class="csl-entry">
27. Faria AMC, Mucida D, McCafferty D-M, Tsuji NM, Verhasselt V. <a href="https://doi.org/10.1155/2012/738475">Tolerance and inflammation at the gut mucosa</a>. Clinical &amp; Developmental Immunology. 2012;2012:738475.
</div>
<div id="ref-hisamatsu2013" class="csl-entry">
28. Hisamatsu T, Kanai T, Mikami Y, Yoneno K, Matsuoka K, Hibi T. <a href="https://doi.org/10.1016/j.pharmthera.2012.10.008">Immune aspects of the pathogenesis of inflammatory bowel disease</a>. Pharmacology &amp; Therapeutics. 2013;137:283–97.
</div>
<div id="ref-michielan2015" class="csl-entry">
29. Michielan A, D’Incà R. <a href="https://doi.org/10.1155/2015/628157">Intestinal Permeability in Inflammatory Bowel Disease: Pathogenesis, Clinical Evaluation, and Therapy of Leaky Gut</a>. Mediators of Inflammation. 2015;2015:628157.
</div>
<div id="ref-mayorgas2021a" class="csl-entry">
30. Mayorgas A. <a href="http://www.tdx.cat/handle/10803/672414">Human Primary Organoid-Derived Epithelial Monolayers as a Novel Strategy for the Study of Adherent Invasive Escherichia coli pathogenicity and the effects of Postbiotics on Intestinal Epithelial Function</a>. PhD thesis. 2021.
</div>
<div id="ref-neurath1996" class="csl-entry">
31. Neurath MF, Fuss I, Kelsall BL, Presky DH, Waegell W, Strober W. <a href="https://doi.org/10.1084/jem.183.6.2605">Experimental granulomatous colitis in mice is abrogated by induction of TGF-beta-mediated oral tolerance.</a> Journal of Experimental Medicine. 1996;183:2605–16.
</div>
<div id="ref-corraliza" class="csl-entry">
32. Corraliza AM, Ricart E, López-García A, Carme Masamunt M, Veny M, Esteller M, et al. Differences in Peripheral and Tissue Immune Cell Populations Following Haematopoietic Stem Cell Transplantation in Crohn<span>’</span>s Disease Patients. Journal of Crohn’s and Colitis. <a href="https://doi.org/10.1093/ecco-jcc/jjy203">https://doi.org/10.1093/ecco-jcc/jjy203</a>.
</div>
<div id="ref-strachan1989" class="csl-entry">
33. Strachan DP. <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1838109/">Hay fever, hygiene, and household size.</a> BMJ : British Medical Journal. 1989;299:1259–60.
</div>
<div id="ref-scudellari2017" class="csl-entry">
34. Scudellari M. <a href="https://doi.org/10.1073/pnas.1700688114">News feature: Cleaning up the hygiene hypothesis</a>. Proceedings of the National Academy of Sciences of the United States of America. 2017;114:1433–6.
</div>
<div id="ref-thomas1998" class="csl-entry">
35. Thomas GAO, Rhodes J, Green JT. <a href="https://doi.org/10.1111/j.1572-0241.1998.00144.x">Inflammatory bowel disease and smoking<span></span>a review</a>. Official journal of the American College of Gastroenterology | ACG. 1998;93:144149.
</div>
<div id="ref-cornish2008" class="csl-entry">
36. Cornish JA, Tan E, Simillis C, Clark SK, Teare J, Tekkis PP. <a href="https://doi.org/10.1111/j.1572-0241.2008.02064.x">The risk of oral contraceptives in the etiology of inflammatory bowel disease: a meta-analysis</a>. The American Journal of Gastroenterology. 2008;103:2394–400.
</div>
<div id="ref-kaufmann1987" class="csl-entry">
37. Kaufmann HJ, Taubin HL. <a href="https://doi.org/10.7326/0003-4819-107-4-513">Nonsteroidal anti-inflammatory drugs activate quiescent inflammatory bowel disease</a>. Annals of Internal Medicine. 1987;107:513–6.
</div>
<div id="ref-bitton2008" class="csl-entry">
38. Bitton A, Dobkin PL, Edwardes MD, Sewitch MJ, Meddings JB, Rawal S, et al. <a href="https://doi.org/10.1136/gut.2007.134817">Predicting relapse in Crohn’s disease: a biopsychosocial model</a>. Gut. 2008;57:1386–92.
</div>
<div id="ref-bassolasmolina2018a" class="csl-entry">
39. Bassolas Molina H. <a href="http://diposit.ub.edu/dspace/handle/2445/131017">Resposta T específica contra antígens de la microbiota comensal en la malaltia de Crohn i inhibició de ROR?t com a estratègia terapèutica</a>. PhD thesis. 2018.
</div>
<div id="ref-bassolasmolina2018" class="csl-entry">
40. Bassolas Molina H. <a href="http://diposit.ub.edu/dspace/handle/2445/131017">Resposta T específica contra antígens de la microbiota comensal en la malaltia de Crohn i inhibició de ROR?t com a estratègia terapèutica</a>. PhD thesis. 2018.
</div>
<div id="ref-demattos2015a" class="csl-entry">
41. de Mattos BRR, Garcia MPG, Nogueira JB, Paiatto LN, Albuquerque CG, Souza CL, et al. <a href="https://doi.org/10.1155/2015/493012">Inflammatory Bowel Disease: An Overview of Immune Mechanisms and Biological Treatments</a>. Mediators of Inflammation. 2015;2015:493012.
</div>
<div id="ref-baumgart2012" class="csl-entry">
42. Baumgart DC, Sandborn WJ. <a href="https://doi.org/10.1016/S0140-6736(12)60026-9">Crohn’s disease</a>. The Lancet. 2012;380:1590–605.
</div>
<div id="ref-guardiola2018" class="csl-entry">
45. Guardiola J, Lobatón T, Cerrillo E, Ferreiro-Iglesias R, Gisbert JP, Domènech E, et al. <a href="https://doi.org/10.1016/j.gastre.2018.05.018">Recommendations of the Spanish Working Group on Crohn’s Disease and Ulcerative Colitis (GETECCU) on the utility of the determination of faecal calprotectin in inflammatory bowel disease</a>. Gastroenterología y Hepatología (English Edition). 2018;41:514–29.
</div>
<div id="ref-corralizamárquez2019" class="csl-entry">
46. Corraliza Márquez A. <a href="http://diposit.ub.edu/dspace/handle/2445/143300">Immune mechanisms involved in inducing remission in Crohn<span>’</span>s disease patients undergoing hematopoietic stem cell transplant</a>. PhD thesis. 2019.
</div>
<div id="ref-satsangi2006" class="csl-entry">
47. Satsangi J, Silverberg MS, Vermeire S, Colombel J-F. <a href="https://doi.org/10.1136/gut.2005.082909">The Montreal classification of inflammatory bowel disease: controversies, consensus, and implications</a>. Gut. 2006;55:749–53.
</div>
<div id="ref-daperno2004" class="csl-entry">
48. Daperno M, D’Haens G, Van Assche G, Baert F, Bulois P, Maunoury V, et al. <a href="https://doi.org/10.1016/S0016-5107(04)01878-4">Development and validation of a new, simplified endoscopic activity score for crohn’s disease: The SES-CD</a>. Gastrointestinal Endoscopy. 2004;60:505–12.
</div>
<div id="ref-best1976" class="csl-entry">
49. Best WR, Becktel JM, Singleton JW, Kern F. <a href="https://doi.org/10.1016/S0016-5085(76)80163-1">Development of a Crohn’s Disease Activity Index: National Cooperative Crohn’s Disease Study</a>. Gastroenterology. 1976;70:439–44.
</div>
<div id="ref-bhattacharya2016a" class="csl-entry">
50. Bhattacharya A, Rao BB, Koutroubakis IE, Click B, Vargas EJ, Regueiro M, et al. <a href="https://doi.org/10.1097/MIB.0000000000000935">Silent crohn’s disease predicts increased bowel damage during multiyear follow-up: The consequences of under-reporting active inflammation</a>. Inflammatory Bowel Diseases. 2016;22:2665–71.
</div>
<div id="ref-peyrin-biroulet2010" class="csl-entry">
51. Peyrin-Biroulet L, Loftus EVJ, Colombel J-F, Sandborn WJ. <a href="https://doi.org/10.1038/ajg.2009.579">The natural history of adult crohn’s disease in population-based cohorts</a>. Official journal of the American College of Gastroenterology | ACG. 2010;105:289297.
</div>
<div id="ref-bhattacharya2016" class="csl-entry">
52. Bhattacharya A, Rao BB, Koutroubakis IE, Click B, Vargas EJ, Regueiro M, et al. <a href="https://doi.org/10.1097/MIB.0000000000000935">Silent crohn’s disease predicts increased bowel damage during multiyear follow-up: The consequences of under-reporting active inflammation</a>. Inflammatory Bowel Diseases. 2016;22:2665–71.
</div>
<div id="ref-satsangi2006a" class="csl-entry">
53. Satsangi J, Silverberg MS, Vermeire S, Colombel J-F. <a href="https://doi.org/10.1136/gut.2005.082909">The Montreal classification of inflammatory bowel disease: controversies, consensus, and implications</a>. Gut. 2006;55:749–53.
</div>
<div id="ref-etchevers2009" class="csl-entry">
54. Etchevers MJ, Aceituno M, García-Bosch O, Ordás I, Sans M, Ricart E, et al. <a href="https://doi.org/10.1002/ibd.20897">Risk factors and characteristics of extent progression in ulcerative colitis</a>. Inflammatory Bowel Diseases. 2009;15:1320–5.
</div>
<div id="ref-boonstra2012" class="csl-entry">
55. Boonstra K, van Erpecum KJ, van Nieuwkerk KMJ, Drenth JPH, Poen AC, Witteman BJM, et al. <a href="https://doi.org/10.1002/ibd.22938">Primary sclerosing cholangitis is associated with a distinct phenotype of inflammatory bowel disease</a>. Inflammatory Bowel Diseases. 2012;18:2270–6.
</div>
<div id="ref-mark-christensen2018" class="csl-entry">
56. Mark-Christensen A, Laurberg S, Haboubi N. <a href="https://doi.org/10.1093/ibd/izy075">Dysplasia in Inflammatory Bowel Disease: Historical Review, Critical Histopathological Analysis, and Clinical Implications</a>. Inflammatory Bowel Diseases. 2018;24:1895–903.
</div>
<div id="ref-schieffer2016" class="csl-entry">
57. Schieffer KM, Williams ED, Yochum GS, Koltun WA. <a href="https://doi.org/10.1111/apt.13780">Review article: the pathogenesis of pouchitis</a>. Alimentary Pharmacology &amp; Therapeutics. 2016;44:817–35.
</div>
<div id="ref-schroeder1987" class="csl-entry">
58. Schroeder KW, Tremaine WJ, Ilstrup DM. <a href="https://doi.org/10.1056/nejm198712243172603">Coated Oral 5-Aminosalicylic Acid Therapy for Mildly to Moderately Active Ulcerative Colitis</a>. New England Journal of Medicine. 1987;317:1625–9.
</div>
<div id="ref-irvine1999" class="csl-entry">
59. Irvine EJ. <a href="https://doi.org/10.1097/00005176-199904001-00003">Development and Subsequent Refinement of the Inflammatory Bowel Disease Questionnaire: A Quality-of-Life Instrument for Adult Patients with Inflammatory Bowel Disease</a>. Journal of Pediatric Gastroenterology &amp; Nutrition. 1999;28 Supplement:S23–7.
</div>
<div id="ref-travis2012" class="csl-entry">
60. Travis SPL, Schnell D, Krzeski P, Abreu MT, Altman DG, Colombel J-F, et al. <a href="https://doi.org/10.1136/gutjnl-2011-300486">Developing an instrument to assess the endoscopic severity of ulcerative colitis: The ulcerative colitis endoscopic index of severity (UCEIS)</a>. Gut. 2012;61:535–42.
</div>
<div id="ref-raine2021" class="csl-entry">
61. Raine T, Verstockt B, Kopylov U, Karmiris K, Goldberg R, Atreya R, et al. <a href="https://doi.org/10.1093/ecco-jcc/jjab112">ECCO topical review: Refractory inflammatory bowel disease</a>. Journal of Crohn’s and Colitis. 2021;15:1605–20.
</div>
<div id="ref-corralizab" class="csl-entry">
62. Corraliza AM, Ricart E, López-García A, Carme Masamunt M, Veny M, Esteller M, et al. Differences in Peripheral and Tissue Immune Cell Populations Following Haematopoietic Stem Cell Transplantation in Crohn<span>’</span>s Disease Patients. Journal of Crohn’s and Colitis. <a href="https://doi.org/10.1093/ecco-jcc/jjy203">https://doi.org/10.1093/ecco-jcc/jjy203</a>.
</div>
<div id="ref-beck2021" class="csl-entry">
63. Beck LC, Granger CL, Masi AC, Stewart CJ. <a href="https://doi.org/10.1080/14789450.2021.1922278">Use of omic technologies in early life gastrointestinal health and disease: From bench to bedside</a>. Expert Review of Proteomics. 2021;18:247–59.
</div>
<div id="ref-hasler_uncoupling_2016" class="csl-entry">
64. Häsler R, Sheibani-Tezerji R, Sinha A, Barann M, Rehman A, Esser D, et al. <a href="https://doi.org/10.1136/gutjnl-2016-311651">Uncoupling of mucosal gene regulation, <span class="nocase">mRNA</span> splicing and adherent microbiota signatures in inflammatory bowel disease</a>. Gut. 2017;66:2087–97.
</div>
<div id="ref-huWholeExomeSequencing2021" class="csl-entry">
66. Hu S, Vila AV, Gacesa R, Collij V, Stevens C, Fu JM, et al. <a href="https://doi.org/10.1136/gutjnl-2019-319706">Whole exome sequencing analyses reveal gene–microbiota interactions in the context of IBD</a>. Gut. 2021;70:285–96.
</div>
<div id="ref-tang_integrated_2017" class="csl-entry">
67. Tang MS, Bowcutt R, Leung JM, Wolff MJ, Gundra UM, Hudesman D, et al. <a href="https://doi.org/10.1097/MIB.0000000000001208">Integrated <span>Analysis</span> of <span>Biopsies</span> from <span>Inflammatory Bowel Disease Patients Identifies SAA1</span> as a <span>Link Between Mucosal Microbes</span> with <span>TH17</span> and <span>TH22 Cells</span></a>. Inflammatory Bowel Diseases. 2017;23:1544–54.
</div>
<div id="ref-mayorgas2021" class="csl-entry">
68. Mayorgas A, Dotti I, Salas A. <a href="https://doi.org/10.1002/mnfr.202000188">Microbial Metabolites, Postbiotics, and Intestinal Epithelial Function</a>. Molecular Nutrition &amp; Food Research. 2021;65:2000188.
</div>
<div id="ref-planell2013" class="csl-entry">
69. Planell N, Lozano JJ, Mora-Buch R, Masamunt MC, Jimeno M, Ordás I, et al. <a href="https://doi.org/10.1136/gutjnl-2012-303333">Transcriptional analysis of the intestinal mucosa of patients with ulcerative colitis in remission reveals lasting epithelial cell alterations</a>. Gut. 2013;62:967–76.
</div>
<div id="ref-leal2015" class="csl-entry">
70. Leal RF, Planell N, Kajekar R, Lozano JJ, Ordás I, Dotti I, et al. <a href="https://doi.org/10.1136/gutjnl-2013-306518">Identification of inflammatory mediators in patients with Crohn’s disease unresponsive to anti-TNF? therapy</a>. Gut. 2015;64:233–42.
</div>
<div id="ref-knights2013" class="csl-entry">
71. Knights D, Lassen KG, Xavier RJ. <a href="https://doi.org/10.1136/gutjnl-2012-303954">Advances in inflammatory bowel disease pathogenesis: Linking host genetics and the microbiome</a>. Gut. 2013;62.
</div>
<div id="ref-ahmed2016" class="csl-entry">
72. Ahmed I, Roy BC, Khan SA, Septer S, Umar S. <a href="https://doi.org/10.3390/microorganisms4020020">Microbiome, metabolome and inflammatory bowel disease</a>. Microorganisms. 2016;4:20.
</div>
<div id="ref-gallagher2021" class="csl-entry">
73. Gallagher K, Catesson A, Griffin JL, Holmes E, Williams HRT. <a href="https://doi.org/10.1093/ecco-jcc/jjaa227">Metabolomic Analysis in Inflammatory Bowel Disease: A Systematic Review</a>. Journal of Crohn’s &amp; Colitis. 2021;15:813–26.
</div>
<div id="ref-ferrer-picón2020b" class="csl-entry">
74. Ferrer-Picón E, Dotti I, Corraliza AM, Mayorgas A, Esteller M, Perales JC, et al. <a href="https://doi.org/10.1093/ibd/izz119">Intestinal Inflammation Modulates the Epithelial Response to Butyrate in Patients With Inflammatory Bowel Disease</a>. Inflammatory Bowel Diseases. 2020;26:43–55.
</div>
<div id="ref-krassowski2020" class="csl-entry">
75. Krassowski M, Das V, Sahu SK, Misra BB. <a href="https://doi.org/10.3389/fgene.2020.610798">State of the field in multi-omics research: From computational needs to data mining and sharing</a>. Frontiers in Genetics. 2020;11:1598.
</div>
<div id="ref-hotelling_relations_1936" class="csl-entry">
76. HOTELLING H. <a href="https://doi.org/10.1093/biomet/28.3-4.321"><span>RELATIONS BETWEEN TWO SETS OF VARIATES</span></a>. Biometrika. 1936;28:321–77.
</div>
<div id="ref-biancolillo" class="csl-entry">
77. Biancolillo A. <a href="http://models.life.ku.dk/sites/default/files/Thesis_Alessandra_Biancolillo.pdf">Method development in the area of multi-block analysis focused on food analysis</a>. PhD thesis.
</div>
<div id="ref-cavill_transcriptomic_2016" class="csl-entry">
80. Cavill R, Jennen D, Kleinjans J, Bried’e JJ. <a href="https://doi.org/10.1093/bib/bbv090">Transcriptomic and metabolomic data integration</a>. Briefings in Bioinformatics. 2016;17:891–901.
</div>
<div id="ref-tarazona2021" class="csl-entry">
81. Tarazona S, Arzalluz-Luque A, Conesa A. <a href="https://doi.org/10.1038/s43588-021-00086-z">Undisclosed, unmet and neglected challenges in multi-omics studies</a>. Nature Computational Science. 2021;1–8.
</div>
<div id="ref-tarazona2020" class="csl-entry">
82. Tarazona S, Balzano-Nogueira L, Gómez-Cabrero D, Schmidt A, Imhof A, Hankemeier T, et al. <a href="https://doi.org/10.1038/s41467-020-16937-8">Harmonization of quality metrics and power calculation in multi-omic studies</a>. Nature Communications. 2020;11:3092.
</div>
<div id="ref-massoni-badosa2020" class="csl-entry">
83. Massoni-Badosa R, Iacono G, Moutinho C, Kulis M, Palau N, Marchese D, et al. <a href="https://doi.org/10.1186/s13059-020-02032-0">Sampling time-dependent artifacts in single-cell genomics studies</a>. Genome Biology. 2020;21:112.
</div>
<div id="ref-zhu2017" class="csl-entry">
84. Zhu Y, Wang L, Yin Y, Yang E. <a href="https://doi.org/10.1038/s41598-017-05882-0">Systematic analysis of gene expression patterns associated with postmortem interval in human tissues</a>. Scientific Reports. 2017;7:5435.
</div>
<div id="ref-ferreira2018" class="csl-entry">
85. Ferreira PG, Muñoz-Aguirre M, Reverter F, Sá Godinho CP, Sousa A, Amadoz A, et al. <a href="https://doi.org/10.1038/s41467-017-02772-x">The effects of death and post-mortem cold ischemia on human tissue transcriptomes</a>. Nature Communications. 2018;9:490.
</div>
<div id="ref-koh_iomicspass_2019" class="csl-entry">
86. Koh HWL, Fermin D, Vogel C, Choi KP, Ewing RM, Choi H. <a href="https://doi.org/10.1038/s41540-019-0099-y"><span class="nocase">iOmicsPASS</span>: Network-based integration of multiomics data for predictive subnetwork discovery</a>. npj Systems Biology and Applications. 2019;5:1–0.
</div>
<div id="ref-tenenhaus_regularized_2011" class="csl-entry">
87. Tenenhaus A, Tenenhaus M. <a href="https://doi.org/10.1007/s11336-011-9206-8">Regularized <span>Generalized Canonical Correlation Analysis</span></a>. Psychometrika. 2011;76:257–84.
</div>
<div id="ref-tenenhaus_variable_2014" class="csl-entry">
88. Tenenhaus A, Philippe C, Guillemot V, Le Cao K-A, Grill J, Frouin V. <a href="https://doi.org/10.1093/biostatistics/kxu001">Variable selection for generalized canonical correlation analysis</a>. Biostatistics. 2014;15:569–83.
</div>
<div id="ref-culhane_cross-platform_2003" class="csl-entry">
89. Culhane AC, Perri‘ere G, Higgins DG. <a href="https://doi.org/10.1186/1471-2105-4-59">Cross-platform comparison and visualisation of gene expression data using co-inertia analysis</a>. BMC Bioinformatics. 2003;4:59.
</div>
<div id="ref-vito_multi-study_2019" class="csl-entry">
90. Vito RD, Bellio R, Trippa L, Parmigiani G. <a href="https://doi.org/10.1111/biom.12974">Multi-study factor analysis</a>. Biometrics. 2019;75:337–46.
</div>
<div id="ref-argelaguet_multi-omics_2018" class="csl-entry">
91. Argelaguet R, Velten B, Arnol D, Dietrich S, Zenz T, Marioni JC, et al. <a href="https://doi.org/10.15252/msb.20178124">Multi-<span>Omics Factor Analysis</span>a framework for unsupervised integration of multi-omics data sets</a>. Molecular Systems Biology. 2018;14:e8124.
</div>
<div id="ref-gomez-cabreroSTATegraComprehensiveMultiomics2019" class="csl-entry">
92. Gomez-Cabrero D, Tarazona S, Ferreir’os-Vidal I, Ramirez RN, Company C, Schmidt A, et al. <a href="https://doi.org/10.1038/s41597-019-0202-7"><span>STATegra</span>, a comprehensive multi-omics dataset of <span>B</span>-cell differentiation in mouse</a>. Scientific Data. 2019;6:1–5.
</div>
<div id="ref-franzosaSpecieslevelFunctionalProfiling2018" class="csl-entry">
93. Franzosa EA, McIver LJ, Rahnavard G, Thompson LR, Schirmer M, Weingart G, et al. <a href="https://doi.org/10.1038/s41592-018-0176-y">Species-level functional profiling of metagenomes and metatranscriptomes</a>. Nature Methods. 2018;15:962.
</div>
<div id="ref-truongMetaPhlAn2EnhancedMetagenomic2015" class="csl-entry">
94. Truong DT, Franzosa EA, Tickle TL, Scholz M, Weingart G, Pasolli E, et al. <a href="https://doi.org/10.1038/nmeth.3589">MetaPhlAn2 for enhanced metagenomic taxonomic profiling</a>. Nature Methods. 2015;12:902–3.
</div>
<div id="ref-segataMetagenomicBiomarkerDiscovery2011" class="csl-entry">
95. Segata N, Izard J, Waldron L, Gevers D, Miropolsky L, Garrett WS, et al. <a href="https://doi.org/10.1186/gb-2011-12-6-r60">Metagenomic biomarker discovery and explanation</a>. Genome Biology. 2011;12:R60.
</div>
<div id="ref-didier2018" class="csl-entry">
96. Didier G, Valdeolivas A, Baudot A. <a href="https://doi.org/10.12688/f1000research.15486.2">Identifying communities from multiplex biological networks by randomized optimization of modularity</a>. F1000Research. 2018;7.
</div>
<div id="ref-valdeolivas2019" class="csl-entry">
97. Valdeolivas A, Tichit L, Navarro C, Perrin S, Odelin G, Levy N, et al. <a href="https://doi.org/10.1093/bioinformatics/bty637">Random walk with restart on multiplex and heterogeneous biological networks</a>. Bioinformatics (Oxford, England). 2019;35:497–505.
</div>
<div id="ref-pio-lopez2021" class="csl-entry">
98. Pio-Lopez L, Valdeolivas A, Tichit L, Remy É, Baudot A. <a href="http://arxiv.org/abs/2008.10085">MultiVERSE: A multiplex and multiplex-heterogeneous network embedding approach</a>. arXiv:200810085 [cs, q-bio]. 2021.
</div>
<div id="ref-zhu2012" class="csl-entry">
99. Zhu J, Sova P, Xu Q, Dombek KM, Xu EY, Vu H, et al. <a href="https://doi.org/10.1371/journal.pbio.1001301">Stitching together Multiple Data Dimensions Reveals Interacting Metabolomic and Transcriptomic Networks That Modulate Cell Regulation</a>. PLOS Biology. 2012;10:e1001301.
</div>
<div id="ref-lock2013" class="csl-entry">
100. Lock EF, Dunson DB. <a href="https://doi.org/10.1093/bioinformatics/btt425">Bayesian consensus clustering</a>. Bioinformatics. 2013;29:2610–6.
</div>
<div id="ref-virtanen2012" class="csl-entry">
101. Virtanen S, Klami A, Khan S, Kaski S. <a href="https://proceedings.mlr.press/v22/virtanen12.html">Artificial Intelligence and Statistics</a>. PMLR; 2012. p. 1269–77.
</div>
<div id="ref-cantini_benchmarking_2020" class="csl-entry">
102. Cantini L, Zakeri P, Hernandez C, Naldi A, Thieffry D, Remy E, et al. <a href="https://doi.org/10.1101/2020.01.14.905760">Benchmarking joint multi-omics dimensionality reduction approaches for cancer study</a>. bioRxiv. 2020;2020.01.14.905760.
</div>
<div id="ref-tenenhaus2017" class="csl-entry">
103. Tenenhaus M, Tenenhaus A, Groenen PJF. <a href="https://doi.org/10.1007/s11336-017-9573-x">Regularized Generalized Canonical Correlation Analysis: A Framework for Sequential Multiblock Component Methods</a>. Psychometrika. 2017;82:737–77.
</div>
<div id="ref-rohart2017" class="csl-entry">
104. Rohart F, Gautier B, Singh A, Cao K-AL. <a href="https://doi.org/10.1371/journal.pcbi.1005752">mixOmics: An R package for <span>‘</span>omics feature selection and multiple data integration</a>. PLOS Computational Biology. 2017;13:e1005752.
</div>
<div id="ref-gomez-cabrero2019" class="csl-entry">
105. Gomez-Cabrero D, Tarazona S, Ferreirós-Vidal I, Ramirez RN, Company C, Schmidt A, et al. <a href="https://doi.org/10.1038/s41597-019-0202-7">STATegra, a comprehensive multi-omics dataset of B-cell differentiation in mouse</a>. Scientific Data. 2019;6:1–5.
</div>
<div id="ref-novoa-del-toro2020" class="csl-entry">
106. Novoa-del-Toro E-M, Mezura-Montes E, Vignes M, Magdinier F, Tichit L, Baudot A. <a href="https://doi.org/10.1101/2020.05.25.114215">A Multi-Objective Genetic Algorithm to Find Active Modules in Multiplex Biological Networks</a>. bioRxiv. 2020;2020.05.25.114215.
</div>
<div id="ref-yang2016" class="csl-entry">
107. Yang Z, Michailidis G. <a href="https://doi.org/10.1093/bioinformatics/btv544">A non-negative matrix factorization method for detecting modules in heterogeneous omics multi-modal data</a>. Bioinformatics. 2016;32:1–8.
</div>
<div id="ref-sherry2005" class="csl-entry">
108. Sherry A, Henson RK. <a href="https://doi.org/10.1207/s15327752jpa8401_09">Conducting and Interpreting Canonical Correlation Analysis in Personality Research: A User-Friendly Primer</a>. Journal of Personality Assessment. 2005;84:37–48.
</div>
<div id="ref-chung_multi-omics_2019" class="csl-entry">
111. Chung R-H, Kang C-Y. <a href="https://doi.org/10.1093/gigascience/giz045">A multi-omics data simulator for complex disease studies and its application to evaluate multi-omics data analysis methods for disease classification</a>. GigaScience. 2019;8.
</div>
<div id="ref-haberman2019" class="csl-entry">
112. Haberman Y, Karns R, Dexheimer PJ, Schirmer M, Somekh J, Jurickova I, et al. <a href="https://doi.org/10.1038/s41467-018-07841-3">Ulcerative colitis mucosal transcriptomes reveal mitochondriopathy and personalized mechanisms underlying disease severity and treatment response</a>. Nature Communications. 2019;10.
</div>
<div id="ref-patuzzi2019" class="csl-entry">
113. Patuzzi I, Baruzzo G, Losasso C, Ricci A, Di Camillo B. <a href="https://doi.org/10.1186/s12859-019-2882-6">metaSPARSim: A 16S rRNA gene sequencing count data simulator</a>. BMC Bioinformatics. 2019;20:416.
</div>
<div id="ref-fritz2019" class="csl-entry">
114. Fritz A, Hofmann P, Majda S, Dahms E, Dröge J, Fiedler J, et al. <a href="https://doi.org/10.1186/s40168-019-0633-6">CAMISIM: Simulating metagenomes and microbial communities</a>. Microbiome. 2019;7:17.
</div>
<div id="ref-fu2021" class="csl-entry">
115. Fu J, Frazee AC, Collado-Torres L, Jaffe AE, Leek JT. <a href="https://doi.org/10.18129/B9.bioc.ballgown">Ballgown: Flexible, isoform-level differential expression analysis</a>. Bioconductor version: Release (3.13); 2021.
</div>
<div id="ref-frazee2021" class="csl-entry">
116. Frazee AC, Jaffe AE, Kirchner R, Leek JT. <a href="https://doi.org/10.18129/B9.bioc.polyester">Polyester: Simulate RNA-seq reads</a>. Bioconductor version: Release (3.13); 2021.
</div>
<div id="ref-mccarthy2012" class="csl-entry">
117. McCarthy DJ, Chen Y, Smyth GK. <a href="https://doi.org/10.1093/nar/gks042">Differential expression analysis of multifactor RNA-Seq experiments with respect to biological variation</a>. Nucleic Acids Research. 2012;40:4288–97.
</div>
<div id="ref-de_souza_ibd_2017" class="csl-entry">
121. De Souza HSP, Fiocchi C, Iliopoulos D. <a href="https://doi.org/10.1038/nrgastro.2017.110">The <span>IBD</span> interactome: <span>An</span> integrated view of aetiology, pathogenesis and therapy</a>. 2017;14.
</div>
<div id="ref-parkhomenko2009" class="csl-entry">
122. Parkhomenko E, Tritchler D, Beyene J. <a href="https://doi.org/10.2202/1544-6115.1406">Sparse Canonical Correlation Analysis with Application to Genomic Data Integration</a>. Statistical Applications in Genetics and Molecular Biology. 2009;8.
</div>
<div id="ref-waaijenborg2008" class="csl-entry">
123. Waaijenborg S, Hamer PCV de W, Zwinderman AH. <a href="https://doi.org/10.2202/1544-6115.1329">Quantifying the Association between Gene Expressions and DNA-Markers by Penalized Canonical Correlation Analysis</a>. Statistical Applications in Genetics and Molecular Biology. 2008;7.
</div>
<div id="ref-witten2009" class="csl-entry">
124. Witten DM, Tibshirani RJ. <a href="https://doi.org/10.2202/1544-6115.1470">Extensions of sparse canonical correlation analysis with applications to genomic data</a>. Statistical Applications in Genetics and Molecular Biology. 2009;8:127.
</div>
<div id="ref-lêcao2009" class="csl-entry">
125. Lê Cao K-A, Martin PG, Robert-Granié C, Besse P. <a href="https://doi.org/10.1186/1471-2105-10-34">Sparse canonical methods for biological data integration: Application to a cross-platform study</a>. BMC Bioinformatics. 2009;10:34.
</div>
<div id="ref-hwang2009" class="csl-entry">
126. Hwang H. <a href="https://doi.org/10.1007/s11336-009-9119-y">Regularized Generalized Structured Component Analysis</a>. Psychometrika. 2009;74:517–30.
</div>
<div id="ref-soneson2010" class="csl-entry">
127. Soneson C, Lilljebjörn H, Fioretos T, Fontes M. <a href="https://doi.org/10.1186/1471-2105-11-191">Integrative analysis of gene expression and copy number alterations using canonical correlation analysis</a>. BMC Bioinformatics. 2010;11:191.
</div>
<div id="ref-zhang2011" class="csl-entry">
128. Zhang S, Li Q, Liu J, Zhou XJ. <a href="https://doi.org/10.1093/bioinformatics/btr206">A novel computational framework for simultaneous integration of multiple types of genomic data to identify microRNA-gene regulatory modules</a>. Bioinformatics. 2011;27:i401–9.
</div>
<div id="ref-lee2011" class="csl-entry">
129. Lee W, Lee D, Lee Y, Pawitan Y. <a href="https://doi.org/10.2202/1544-6115.1638">Sparse Canonical Covariance Analysis for High-throughput Data</a>. Statistical Applications in Genetics and Molecular Biology. 2011;10.
</div>
<div id="ref-abdi2012" class="csl-entry">
130. Abdi H, Williams LJ, Valentin D, Bennani-Dosse M. <a href="https://doi.org/10.1002/wics.198">STATIS and DISTATIS: optimum multitable principal component analysis and three way metric multidimensional scaling</a>. WIREs Computational Statistics. 2012;4:124–67.
</div>
<div id="ref-zhang2012" class="csl-entry">
131. Zhang S, Liu C-C, Li W, Shen H, Laird PW, Zhou XJ. <a href="https://doi.org/10.1093/nar/gks725">Discovery of multi-dimensional modules by integrative analysis of cancer genomic data</a>. Nucleic Acids Research. 2012;40:9379–91.
</div>
<div id="ref-li2012" class="csl-entry">
132. Li W, Zhang S, Liu C-C, Zhou XJ. <a href="https://doi.org/10.1093/bioinformatics/bts476">Identifying multi-layer gene regulatory modules from multi-dimensional genomic data</a>. Bioinformatics. 2012;28:2458–66.
</div>
<div id="ref-virtanen2012a" class="csl-entry">
133. Virtanen S, Klami A, Khan S, Kaski S. <a href="https://proceedings.mlr.press/v22/virtanen12.html">Artificial Intelligence and Statistics</a>. PMLR; 2012. p. 1269–77.
</div>
<div id="ref-zhu2012a" class="csl-entry">
134. Zhu J, Sova P, Xu Q, Dombek KM, Xu EY, Vu H, et al. <a href="https://doi.org/10.1371/journal.pbio.1001301">Stitching together Multiple Data Dimensions Reveals Interacting Metabolomic and Transcriptomic Networks That Modulate Cell Regulation</a>. PLOS Biology. 2012;10:e1001301.
</div>
<div id="ref-abdi2013" class="csl-entry">
135. Abdi H, Williams LJ, Valentin D. <a href="https://doi.org/10.1002/wics.1246">Multiple factor analysis: principal component analysis for multitable and multiblock data sets</a>. WIREs Computational Statistics. 2013;5:149–79.
</div>
<div id="ref-lock2013a" class="csl-entry">
136. Lock EF, Hoadley KA, Marron JS, Nobel AB. <a href="https://doi.org/10.1214/12-AOAS597">JOINT AND INDIVIDUAL VARIATION EXPLAINED (JIVE) FOR INTEGRATED ANALYSIS OF MULTIPLE DATA TYPES</a>. The annals of applied statistics. 2013;7:523–42.
</div>
<div id="ref-schlauch2017" class="csl-entry">
137. Schlauch D, Paulson JN, Young A, Glass K, Quackenbush J. <a href="https://doi.org/10.1093/bioinformatics/btx139">Estimating gene regulatory networks with pandaR</a>. Bioinformatics. 2017;33:2232–4.
</div>
<div id="ref-meng2014a" class="csl-entry">
138. Meng C, Kuster B, Culhane AC, Gholami AM. <a href="https://doi.org/10.1186/1471-2105-15-162">A multivariate approach to the integration of multi-omics datasets</a>. BMC Bioinformatics. 2014;15:162.
</div>
<div id="ref-planell2021a" class="csl-entry">
139. Planell N, Lagani V, Sebastian-Leon P, van der Kloet F, Ewing E, Karathanasis N, et al. <a href="https://doi.org/10.3389/fgene.2021.620453">STATegra: Multi-omics data integration <span></span> a conceptual scheme with a bioinformatics pipeline</a>. Frontiers in Genetics. 2021;12.
</div>
<div id="ref-ray2014" class="csl-entry">
140. Ray P, Zheng L, Lucas J, Carin L. <a href="https://doi.org/10.1093/bioinformatics/btu064">Bayesian joint analysis of heterogeneous genomics data</a>. Bioinformatics. 2014;30:1370–6.
</div>
<div id="ref-bunte2016" class="csl-entry">
141. Bunte K, Leppäaho E, Saarinen I, Kaski S. <a href="https://doi.org/10.1093/bioinformatics/btw207">Sparse group factor analysis for biclustering of multiple data sources</a>. Bioinformatics. 2016;32:2457–63.
</div>
<div id="ref-chen2013" class="csl-entry">
142. Chen M, Gao C, Ren Z, Zhou HH. <a href="http://arxiv.org/abs/1311.6186">Sparse CCA via precision adjusted iterative thresholding</a>. arXiv:13116186 [math, stat]. 2013.
</div>
<div id="ref-leppäaho2017" class="csl-entry">
143. Leppäaho E, Ammad-ud-din M, Kaski S. <a href="http://jmlr.org/papers/v18/16-509.html">GFA: Exploratory analysis of multiple data sources with group factor analysis</a>. Journal of Machine Learning Research. 2017;18:1–5.
</div>
<div id="ref-klami2014" class="csl-entry">
144. Klami A, Bouchard G, Tripathi A. <a href="http://arxiv.org/abs/1312.5921">Group-sparse embeddings in collective matrix factorization</a>. arXiv:13125921 [cs, stat]. 2014.
</div>
<div id="ref-meng2018" class="csl-entry">
145. Meng C, Basunia A, Peters B, Gholami AM, Kuster B, Culhane AC. <a href="https://www.biorxiv.org/content/10.1101/046904v2">MOGSA: integrative single sample gene-set analysis of multiple omics data</a>. 2018.
</div>
<div id="ref-yang2016b" class="csl-entry">
146. Yang Z, Michailidis G. <a href="https://doi.org/10.1093/bioinformatics/btv544">A non-negative matrix factorization method for detecting modules in heterogeneous omics multi-modal data</a>. Bioinformatics. 2016;32:1–8.
</div>
<div id="ref-zhao2015" class="csl-entry">
147. Zhao S, Gao C, Mukherjee S, Engelhardt BE. <a href="http://arxiv.org/abs/1411.2698">Bayesian group latent factor analysis with structured sparsity</a>. arXiv:14112698 [q-bio, stat]. 2015.
</div>
<div id="ref-voillet2016" class="csl-entry">
148. Voillet V, Besse P, Liaubet L, San Cristobal M, González I. <a href="https://doi.org/10.1186/s12859-016-1273-5">Handling missing rows in multi-omics data integration: Multiple imputation in multiple factor analysis framework</a>. BMC Bioinformatics. 2016;17:402.
</div>
<div id="ref-beaton2016" class="csl-entry">
149. Beaton D, Dunlop J, Abdi H, Alzheimer’s Disease Neuroimaging Initiative. <a href="https://doi.org/10.1037/met0000053">Partial least squares correspondence analysis: A framework to simultaneously analyze behavioral and genetic data</a>. Psychological Methods. 2016;21:621–51.
</div>
<div id="ref-singh2019" class="csl-entry">
150. Singh A, Shannon CP, Gautier B, Rohart F, Vacher M, Tebbutt SJ, et al. <a href="https://doi.org/10.1093/bioinformatics/bty1054">DIABLO: An integrative approach for identifying key molecular drivers from multi-omics assays</a>. Bioinformatics. 2019;35:3055–62.
</div>
<div id="ref-yoon2019" class="csl-entry">
151. Yoon G, Carroll RJ, Gaynanova I. <a href="http://arxiv.org/abs/1807.05274">Sparse semiparametric canonical correlation analysis for data of mixed types</a>. arXiv:180705274 [stat]. 2019.
</div>
<div id="ref-gaynanova2017a" class="csl-entry">
152. Gaynanova I, Li G. <a href="http://arxiv.org/abs/1707.06573">Structural learning and integrative decomposition of multi-view data</a>. arXiv:170706573 [stat]. 2017.
</div>
<div id="ref-madrigal2017" class="csl-entry">
153. Madrigal P. <a href="https://doi.org/10.1093/bioinformatics/btw724">fCCAC: Functional canonical correlation analysis to evaluate covariance between nucleic acid sequencing datasets</a>. Bioinformatics. 2017;33:746–8.
</div>
<div id="ref-yoshida2017" class="csl-entry">
154. Yoshida K, Yoshimoto J, Doya K. <a href="https://doi.org/10.1186/s12859-017-1543-x">Sparse kernel canonical correlation analysis for discovery of nonlinear interactions in high-dimensional data</a>. BMC Bioinformatics. 2017;18:108.
</div>
<div id="ref-kawaguchi2017" class="csl-entry">
155. Kawaguchi A, Yamashita F. <a href="https://doi.org/10.1093/biostatistics/kxx011">Supervised multiblock sparse multivariable analysis with application to multimodal brain imaging genetics</a>. Biostatistics. 2017;18:651–65.
</div>
<div id="ref-feng2018" class="csl-entry">
156. Feng Q, Jiang M, Hannig J, Marron JS. <a href="http://arxiv.org/abs/1704.02060">Angle-based joint and individual variation explained</a>. arXiv:170402060 [stat]. 2018.
</div>
<div id="ref-argelaguet2020" class="csl-entry">
157. Argelaguet R, Arnol D, Bredikhin D, Deloro Y, Velten B, Marioni JC, et al. <a href="https://doi.org/10.1186/s13059-020-02015-1">MOFA+: A statistical framework for comprehensive integration of multi-modal single-cell data</a>. Genome Biology. 2020;21:111.
</div>
<div id="ref-brown2018" class="csl-entry">
158. Brown BC, Bray NL, Pachter L. Expression reflects population structure. 2018. <a href="https://doi.org/10.1101/364448">https://doi.org/10.1101/364448</a>.
</div>
<div id="ref-zhang2020" class="csl-entry">
159. Zhang Y, Gaynanova I. <a href="http://arxiv.org/abs/1811.08511">Joint association and classification analysis of multi-view data</a>. arXiv:181108511 [cs, stat]. 2020.
</div>
<div id="ref-tang2021" class="csl-entry">
160. Tang TM, Allen GI. <a href="http://arxiv.org/abs/1810.00832">Integrated principal components analysis</a>. arXiv:181000832 [stat]. 2021.
</div>
<div id="ref-min2019" class="csl-entry">
161. Min EJ, Safo SE, Long Q. <a href="https://doi.org/10.1093/bioinformatics/bty726">Penalized co-inertia analysis with applications to -omics data</a>. Bioinformatics (Oxford, England). 2019;35:1018–25.
</div>
<div id="ref-safo2018" class="csl-entry">
162. Safo SE, Li S, Long Q. <a href="https://doi.org/10.1111/biom.12715">Integrative analysis of transcriptomic and metabolomic data via sparse canonical correlation analysis with incorporation of biological information</a>. Biometrics. 2018;74:300–12.
</div>
<div id="ref-min2017" class="csl-entry">
163. Min W, Liu J, Zhang S. <a href="http://arxiv.org/abs/1710.04792">Sparse weighted canonical correlation analysis</a>. arXiv:171004792 [cs, stat]. 2017.
</div>
<div id="ref-bouhaddani2018" class="csl-entry">
164. Bouhaddani S el, Uh H-W, Jongbloed G, Hayward C, Klarić L, Kiełbasa SM, et al. <a href="https://doi.org/10.1186/s12859-018-2371-3">Integrating omics datasets with the OmicsPLS package</a>. BMC Bioinformatics. 2018;19:371.
</div>
<div id="ref-pimentel2018" class="csl-entry">
165. Pimentel H, Zhiyue H, Huang H. <a href="https://doi.org/10.1007/s40484-017-0127-0">Biclustering by sparse canonical correlation analysis</a>. 2018;6:11.
</div>
<div id="ref-kim2019" class="csl-entry">
166. Kim Y, Bismeijer T, Zwart W, Wessels LFA, Vis DJ. <a href="https://doi.org/10.1038/s41467-019-13027-2">Genomic data integration by WON-PARAFAC identifies interpretable factors for predicting drug-sensitivity in vivo</a>. Nature Communications. 2019;10:5034.
</div>
<div id="ref-lock2020" class="csl-entry">
167. Lock EF, Park JY, Hoadley KA. <a href="http://arxiv.org/abs/2002.02601">Bidimensional linked matrix factorization for pan-omics pan-cancer analysis</a>. arXiv:200202601 [cs, q-bio, stat]. 2020.
</div>
<div id="ref-ronen2019" class="csl-entry">
168. Ronen J, Hayat S, Akalin A. <a href="https://doi.org/10.26508/lsa.201900517">Evaluation of colorectal cancer subtypes and cell lines using deep learning</a>. Life Science Alliance. 2019;2.
</div>
<div id="ref-shi2019" class="csl-entry">
169. Shi WJ, Zhuang Y, Russell PH, Hobbs BD, Parker MM, Castaldi PJ, et al. <a href="https://doi.org/10.1093/bioinformatics/btz226">Unsupervised discovery of phenotype-specific multi-omics networks</a>. Bioinformatics. 2019;35:4336–43.
</div>
<div id="ref-csala2020" class="csl-entry">
170. Csala A, Zwinderman AH, Hof MH. <a href="https://doi.org/10.1186/s12859-019-3286-3">Multiset sparse partial least squares path modeling for high dimensional omics data analysis</a>. BMC Bioinformatics. 2020;21:9.
</div>
<div id="ref-fan2020" class="csl-entry">
171. Fan Z, Zhou Y, Ressom HW. <a href="https://doi.org/10.3390/metabo10040144">MOTA: Network-Based Multi-Omic Data Integration for Biomarker Discovery</a>. Metabolites. 2020;10:144.
</div>
<div id="ref-shu2020" class="csl-entry">
172. Shu H, Wang X, Zhu H. <a href="https://doi.org/10.1080/01621459.2018.1543599">D-CCA: A decomposition-based canonical correlation analysis for high-dimensional datasets</a>. Journal of the American Statistical Association. 2020;115:292–306.
</div>
<div id="ref-hawinkel2020" class="csl-entry">
173. Hawinkel S, Bijnens L, Cao K-AL, Thas O. <a href="https://doi.org/10.1093/nargab/lqaa050">Model-based joint visualization of multiple compositional omics datasets</a>. NAR Genomics and Bioinformatics. 2020;2.
</div>
<div id="ref-gundersen2020" class="csl-entry">
174. Gundersen G, Dumitrascu B, Ash JT, Engelhardt BE. <a href="https://proceedings.mlr.press/v115/gundersen20a.html">Uncertainty in Artificial Intelligence</a>. PMLR; 2020. p. 945–55.
</div>
<div id="ref-velten2020" class="csl-entry">
175. Velten B, Braunger JM, Arnol D, Argelaguet R, Stegle O. <a href="https://doi.org/10.1101/2020.11.03.366674">Identifying temporal and spatial patterns of variation from multi-modal data using MEFISTO</a>. bioRxiv. 2020;2020.11.03.366674.
</div>
<div id="ref-yoon2019a" class="csl-entry">
176. Yoon G, Carroll RJ, Gaynanova I. <a href="http://arxiv.org/abs/1807.05274">Sparse semiparametric canonical correlation analysis for data of mixed types</a>. arXiv:180705274 [stat]. 2019.
</div>
</div>
<div class="footnotes">
<hr />
<ol start="1">
<li id="fn1"><p>A signature is usually a group of features that describe/are representative of a cell line or a process or a stage.<a href="introduction.html#fnref1" class="footnote-back">↩︎</a></p></li>
<li id="fn2"><p>Consulted on 2021/11/10 from <a href="https://github.com/mikelove/awesome-multi-omics" class="uri">https://github.com/mikelove/awesome-multi-omics</a><a href="introduction.html#fnref2" class="footnote-back">↩︎</a></p></li>
</ol>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="abstract.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="hypothesis-and-objectives.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": {
"href": "https://github.com/llrs/thesis/"
},
"facebook": false,
"twitter": {
"href": "https://twitter.com/intent/tweet?text=Nice%20read%20of%20%40Lluis_Revilla%20thesis"
},
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/llrs/thesis/edit/master/01_introduction.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": "https://github.com/llrs/thesis/blob/master/01_introduction.Rmd",
"text": "View"
},
"download": ["_main.pdf", "_main.epub"],
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": false,
"toc_float": true,
"toc_appendix": true,
"toc_depth": 3,
"scroll_highlight": true
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
