<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 1 Introduction | Data integration on inflammatory bowel disease</title>
  <meta name="description" content="Thesis about the relationship between microbiome and transcriptome on the gastrointestinal tract on patients with inflammatory bowel disease and some controls to discover key differences on this relationship." />
  <meta name="generator" content="bookdown 0.24 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 1 Introduction | Data integration on inflammatory bowel disease" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="Thesis about the relationship between microbiome and transcriptome on the gastrointestinal tract on patients with inflammatory bowel disease and some controls to discover key differences on this relationship." />
  <meta name="github-repo" content="llrs/thesis" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 1 Introduction | Data integration on inflammatory bowel disease" />
  <meta name="twitter:site" content="@Lluis_Revilla" />
  <meta name="twitter:description" content="Thesis about the relationship between microbiome and transcriptome on the gastrointestinal tract on patients with inflammatory bowel disease and some controls to discover key differences on this relationship." />
  

<meta name="author" content="Lluís Revilla Sancho" />


<meta name="date" content="2021-10-22" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="index.html"/>
<link rel="next" href="hypothesis-and-objectives.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<script src="libs/accessible-code-block-0.0.1/empty-anchor.js"></script>
<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>




<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><strong><a href="./">Data integration on IBD</a></strong></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Preface</a></li>
<li class="chapter" data-level="1" data-path="introduction.html"><a href="introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a><ul>
<li class="chapter" data-level="1.1" data-path="introduction.html"><a href="introduction.html#IBD"><i class="fa fa-check"></i><b>1.1</b> Inflammatory bowel disease</a><ul>
<li class="chapter" data-level="1.1.1" data-path="introduction.html"><a href="introduction.html#etiology-and-pathogenesis"><i class="fa fa-check"></i><b>1.1.1</b> Etiology and pathogenesis</a></li>
<li class="chapter" data-level="1.1.2" data-path="introduction.html"><a href="introduction.html#crohns-disease-physiology"><i class="fa fa-check"></i><b>1.1.2</b> Crohn’s disease physiology</a></li>
<li class="chapter" data-level="1.1.3" data-path="introduction.html"><a href="introduction.html#ulcerative-colitis-physiology"><i class="fa fa-check"></i><b>1.1.3</b> Ulcerative colitis physiology</a></li>
<li class="chapter" data-level="1.1.4" data-path="introduction.html"><a href="introduction.html#summary"><i class="fa fa-check"></i><b>1.1.4</b> Summary</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="introduction.html"><a href="introduction.html#studies-and-integration-on-ibd"><i class="fa fa-check"></i><b>1.2</b> Studies and integration on IBD</a><ul>
<li class="chapter" data-level="1.2.1" data-path="introduction.html"><a href="introduction.html#transcriptome"><i class="fa fa-check"></i><b>1.2.1</b> Transcriptome</a></li>
<li class="chapter" data-level="1.2.2" data-path="introduction.html"><a href="introduction.html#microbiome-1"><i class="fa fa-check"></i><b>1.2.2</b> Microbiome</a></li>
<li class="chapter" data-level="1.2.3" data-path="introduction.html"><a href="introduction.html#genetics-1"><i class="fa fa-check"></i><b>1.2.3</b> Genetics</a></li>
<li class="chapter" data-level="1.2.4" data-path="introduction.html"><a href="introduction.html#metabolome"><i class="fa fa-check"></i><b>1.2.4</b> Metabolome</a></li>
<li class="chapter" data-level="1.2.5" data-path="introduction.html"><a href="introduction.html#summary-1"><i class="fa fa-check"></i><b>1.2.5</b> Summary</a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="introduction.html"><a href="introduction.html#integration"><i class="fa fa-check"></i><b>1.3</b> Integration</a><ul>
<li class="chapter" data-level="1.3.1" data-path="introduction.html"><a href="introduction.html#classification-of-integration-methods"><i class="fa fa-check"></i><b>1.3.1</b> Classification of integration method’s</a></li>
<li class="chapter" data-level="1.3.2" data-path="introduction.html"><a href="introduction.html#interpretation"><i class="fa fa-check"></i><b>1.3.2</b> Interpretation</a></li>
<li class="chapter" data-level="1.3.3" data-path="introduction.html"><a href="introduction.html#reviews-and-benchmarking"><i class="fa fa-check"></i><b>1.3.3</b> Reviews and benchmarking</a></li>
<li class="chapter" data-level="1.3.4" data-path="introduction.html"><a href="introduction.html#summary-2"><i class="fa fa-check"></i><b>1.3.4</b> Summary</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="2" data-path="hypothesis-and-objectives.html"><a href="hypothesis-and-objectives.html"><i class="fa fa-check"></i><b>2</b> Hypothesis and objectives</a><ul>
<li class="chapter" data-level="2.1" data-path="hypothesis-and-objectives.html"><a href="hypothesis-and-objectives.html#hypothesis"><i class="fa fa-check"></i><b>2.1</b> Hypothesis</a></li>
<li class="chapter" data-level="2.2" data-path="hypothesis-and-objectives.html"><a href="hypothesis-and-objectives.html#objectives"><i class="fa fa-check"></i><b>2.2</b> Objectives</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="materials-and-methods.html"><a href="materials-and-methods.html"><i class="fa fa-check"></i><b>3</b> Materials and methods</a><ul>
<li class="chapter" data-level="3.1" data-path="materials-and-methods.html"><a href="materials-and-methods.html#sample-collection"><i class="fa fa-check"></i><b>3.1</b> Sample collection</a><ul>
<li class="chapter" data-level="3.1.1" data-path="materials-and-methods.html"><a href="materials-and-methods.html#hsct-dataset"><i class="fa fa-check"></i><b>3.1.1</b> HSCT dataset</a></li>
</ul></li>
<li class="chapter" data-level="3.2" data-path="materials-and-methods.html"><a href="materials-and-methods.html#statistics"><i class="fa fa-check"></i><b>3.2</b> Statistics</a><ul>
<li class="chapter" data-level="3.2.1" data-path="materials-and-methods.html"><a href="materials-and-methods.html#functional-enrichment"><i class="fa fa-check"></i><b>3.2.1</b> Functional enrichment</a></li>
<li class="chapter" data-level="3.2.2" data-path="materials-and-methods.html"><a href="materials-and-methods.html#other-methods-used"><i class="fa fa-check"></i><b>3.2.2</b> Other methods used</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="materials-and-methods.html"><a href="materials-and-methods.html#regularized-generalized-canonical-correlation-analysis"><i class="fa fa-check"></i><b>3.3</b> Regularized generalized canonical correlation analysis</a><ul>
<li class="chapter" data-level="3.3.1" data-path="materials-and-methods.html"><a href="materials-and-methods.html#description"><i class="fa fa-check"></i><b>3.3.1</b> Description</a></li>
<li class="chapter" data-level="3.3.2" data-path="materials-and-methods.html"><a href="materials-and-methods.html#relationships-models"><i class="fa fa-check"></i><b>3.3.2</b> Relationships’ models</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="results.html"><a href="results.html"><i class="fa fa-check"></i><b>4</b> Results</a></li>
<li class="chapter" data-level="5" data-path="biological-relevance-of-results.html"><a href="biological-relevance-of-results.html"><i class="fa fa-check"></i><b>5</b> Biological relevance of results</a><ul>
<li class="chapter" data-level="5.1" data-path="biological-relevance-of-results.html"><a href="biological-relevance-of-results.html#comparing-different-dataset"><i class="fa fa-check"></i><b>5.1</b> Comparing different dataset</a></li>
<li class="chapter" data-level="5.2" data-path="biological-relevance-of-results.html"><a href="biological-relevance-of-results.html#shared-selected-variables"><i class="fa fa-check"></i><b>5.2</b> Shared selected variables</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="discussion.html"><a href="discussion.html"><i class="fa fa-check"></i><b>6</b> Discussion</a><ul>
<li class="chapter" data-level="6.1" data-path="discussion.html"><a href="discussion.html#relevance-for-other-research"><i class="fa fa-check"></i><b>6.1</b> Relevance for other research</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="conclusions.html"><a href="conclusions.html"><i class="fa fa-check"></i><b>7</b> Conclusions</a></li>
<li class="chapter" data-level="8" data-path="acknowledgements.html"><a href="acknowledgements.html"><i class="fa fa-check"></i><b>8</b> Acknowledgements</a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="appendix"><span><b>Appendix</b></span></li>
<li class="chapter" data-level="A" data-path="online-resources.html"><a href="online-resources.html"><i class="fa fa-check"></i><b>A</b> Online resources</a></li>
<li class="chapter" data-level="B" data-path="software.html"><a href="software.html"><i class="fa fa-check"></i><b>B</b> Software</a><ul>
<li class="chapter" data-level="B.1" data-path="software.html"><a href="software.html#listed"><i class="fa fa-check"></i><b>B.1</b> Listed</a></li>
<li class="chapter" data-level="B.2" data-path="software.html"><a href="software.html#by-publication"><i class="fa fa-check"></i><b>B.2</b> By publication</a></li>
</ul></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Data integration on inflammatory bowel disease</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div class = "wip">
This tesis is ongoing and still without hard date to finish it. Watch the <a href="https://github.com/llrs/thesis">GitHub repository</a> or wait for a release announcement on Twitter on <a href="https://twitter.com/Lluis_Revilla">my profile</a>.
</div>
<div id="introduction" class="section level1">
<h1><span class="header-section-number">Chapter 1</span> Introduction</h1>
<p>The <a href="https://en.wikipedia.org/wiki/Inflammatory_bowel_disease">inflammatory bowel disease</a> (IBD) involves <a href="https://en.wikipedia.org/wiki/Crohn%27s_disease">Crohn’s disease</a>(CD) and <a href="https://en.wikipedia.org/wiki/Ulcerative_colitis">ulcerative colitis</a>(UC).
It generally affects the terminal ileum and the colon but it can affect any segment of the gastrointestinal tract.
UC is a recurrent, chronic and continuous inflammation of the colon and rectum while the CD is not a continuous inflammation and affects the whole gastrointestinal tract causing transmural inflammation.</p>
<p>IBD etiology is unknown.
However, once it has initiated the most prevalent hypothesis of its chronicity suggests an aberrant immunological response to antigens of the commensal microbiome.</p>
<p>To diagnose the inflammatory bowel disease, factors such as leukocytes levels on blood, a high globular sedimentation speed, increase of protein C reactive and anemia could indicate inflammatory bowel disease.</p>
<p>Treatments provided for the IBD include, noninflammatory drugs, suppressors and biologic.
The therapeutic options can induce remission in some patients, however they often need continuous treatment to avoid recurrence.
Nevertheless, many patients are refractory or intolerant to those therapies and need to undergo surgery.</p>
<p>The disease present unique characteristics that require the usage of integration methods in order to find the specific relationship of the microbiome and the intestine on the disease.</p>
<div id="IBD" class="section level2">
<h2><span class="header-section-number">1.1</span> Inflammatory bowel disease</h2>
<p>Inflammatory bowel disease (IBD) includes the chronic diseases Crohn’s disease (CD) and ulcerative colitis (UC) which are characterized by alternating periods of remission and clinical relapse that mainly affect the gastrointestinal tract.
CD is a progressive relapsing disease that can affect all the gastrointestinal tract but shows mostly on both terminal ilium and colon with a discontinuous inflammation.
The UC is a colonic relapsing disease characterized by a continuous inflammation of the colon.
Both of them have different risk factors, clinical, endoscopic an histological characteristics (see section <a href="introduction.html#crohns-disease-physiology">Crohn’s disease physiology</a> and <a href="introduction.html#ulcerative-colitis-physiology">Ulcerative colitis physiology</a>.</p>
<p>Around 3.5 million individuals have IBD in Europe and North America combined <span class="citation">[<a href="references.html#ref-jairath2020" role="doc-biblioref">1</a>]</span>.
IBD is more commonly found in industrialized and developed regions, suggesting that environmental factors might greatly influence IBD occurrence.
In addition, the incidence of IBD is increasing in areas, such as Asia or Eastern Europe, where the number of cases was relatively low hitherto <span class="citation">[<a href="references.html#ref-burisch2015" role="doc-biblioref">2</a>]</span>.
On some of these countries, having the disease prevents people to <a href="https://civio.es/el-boe-nuestro-de-cada-dia/2019/02/13/las-enfermedades-que-vetan-el-acceso-a-ciertos-empleos-publicos/" title="Link to a Civio.es about disease disqualifying for some public positions">apply to certain positions</a>.</p>
<p>The dysregulation of the inflammatory response observed in IBD requires interplay between host genetic factors and the intestinal microbiome.
Several studies support the concept that IBD arise from an exacerbate immune response against commensal gut microorganisms.
Nonetheless, the disease could result from an imbalanced microbial composition leading to generalized or localized dysbiosis<a href="#fn1" class="footnote-ref" id="fnref1"><sup>1</sup></a>.</p>
<p>The role of the gut microbiome in IBD is an active ongoing field of research.
Several authors are currently studying the alterations reported in IBD of the intestinal microbiome.
However, it is still unclear the cause-effect relation between dysbiosis and IBD.
Partly due to the multiple variables already identified that have been linked to IBD; for instance, age, diet, usage of antibiotic, tobacco, and eventually socioeconomic status (see <span class="citation">[<a href="references.html#ref-human_microbiome_project_consortium_framework_2012" role="doc-biblioref">3</a>]</span>).</p>
<p>The relationship between host and microbiome has been proposed to play a fundamental role to maintain the disease.
For instance, some Proteobacteria which have adherent and invasive properties might exploit host defenses and drive proinflammatory change, altering the intestinal microbiota in favor of dysbiosis <span class="citation">[<a href="references.html#ref-mukhopadhya2012" role="doc-biblioref">4</a>]</span>.</p>
<div id="etiology-and-pathogenesis" class="section level3">
<h3><span class="header-section-number">1.1.1</span> Etiology and pathogenesis</h3>
<p>Several explanations have been proposed as the origin of the disease <span class="citation">[<a href="references.html#ref-zhang2014" role="doc-biblioref">5</a>, <a href="references.html#ref-hugot2004" role="doc-biblioref">6</a>]</span>.
Some of them are based on a relationship between the immune system and the microbiome and different experiments provide partial evidence to this, such as restarting the B-cell repertoire <span class="citation">[<a href="references.html#ref-corralizaDifferencesPeripheralTissue" role="doc-biblioref">7</a>]</span>.
It is also unclear if Crohn’s disease and the ulcerative colitis share the same origin considering their different symptoms.</p>
<p>There is also evidence of some genetic component on the onset of the disease, specially if the disease appears very early (less than 2 years old patients) <span class="citation">[<a href="references.html#ref-mcgovern2015b" role="doc-biblioref">8</a>, <a href="references.html#ref-satsangi2006c" role="doc-biblioref">9</a>]</span>.
Although so far it is not known what triggers the disease at different age, which provides some clues on the mechanism(s) of disease.
There is a classification between very early, early or adult on-set disease <span class="citation">[<a href="references.html#ref-satsangi2006c" role="doc-biblioref">9</a>]</span>.
On very early on-set the genetic component is high (See <a href="introduction.html#clinical-care-UC">Clinical care</a> section).
Genome-wide association studies (GWAS) have linked the IBD disease to <em>NOD2</em> gene, but so far there isn’t any known mechanism how this gene could start the disease <span class="citation">[<a href="references.html#ref-horowitz2021" role="doc-biblioref">10</a>]</span>.
On early and adult disease the genetic component is lower than on very early on set and it is thought that the environmental factors are the main cause of the disease at those ages.</p>
<p>Either case it is not known how it starts or it is sustained and causes those characteristics relapses.
On the following sections I’ll explore the role of several of the possible factors involved on the pahtogenesis, starting with the genetics.</p>
<div id="genetics" class="section level4">
<h4><span class="header-section-number">1.1.1.1</span> Genetics</h4>
<p>The inflammatory bowel disease is not an heritable disease but it has some genetic influence that predisposes people to have it.
On some patients that the disease show up very early, diagnosed before the age of 2 or just early following the Montreal classification (See <a href="introduction.html#clinical-care-UC">Clinical care</a> section), the genetic component is higher.</p>
<p>This has lead to look for genetic factors on IBD both on general population and on the early cases.
Genome wide association studies (GWAS) is one of the most common genetic studies performed, together with methylation studies.
They help discover through linkage desequilibrium genetic variations linked to phenotypes and regulatory transcription changes, respectively.</p>
<p>With GWAS several allels on protein coding loci have been found, rising to around 300 genetic variants <span class="citation">[<a href="references.html#ref-kumar2019" role="doc-biblioref">11</a>]</span>.
Particularly, the <em>NOD2</em> gene is highly relevant for the disease on European patients, as it is a risk alleles for Crohn’s disease loci but show significant (P-value &lt; 0.005) protective effects in ulcerative colitis <span class="citation">[<a href="references.html#ref-jostins2012" role="doc-biblioref">12</a>, <a href="references.html#ref-momozawa2018" role="doc-biblioref">13</a>]</span>.
The mechanism of how this gene protects from ulcerative colitis has not been confirmed yet <span class="citation">[<a href="references.html#ref-horowitz2021" role="doc-biblioref">10</a>]</span>.</p>
<p>Many of the relevant genetic loci related to IBD are not on protein coding fragments of the genome.
Recently expression quantitative trait loci (eQTL) particularly showed <span class="citation">[<a href="references.html#ref-mcgovern2015" role="doc-biblioref">14</a>]</span> that locis are on enhancers or promoters like e.g.
H3K27Ac or promoter e.g.
H3K4me1 marks as found by chromatin immunoprecipitation sequencing (ChIP-Seq).</p>
</div>
<div id="microbiome" class="section level4">
<h4><span class="header-section-number">1.1.1.2</span> Microbiome</h4>
<!--Aida 2.3 p24-->
<p>The human intestinal is a large reservoir of microorganisms (bacteria, fungi, viruses, and unicellular eukaryotes) that coexist within the gastrointestinal tract.
This microbiome community exerts different functions in the human body influencing the nutrients metabolism, immune system maturation and suppression of harmful microorganisms’ growth <span class="citation">[<a href="references.html#ref-khanna2014" role="doc-biblioref">15</a>]</span>.</p>
<p>The role of the gut microbiota is key in the IBD pathogenesis.
IBD has been characterized by a breakdown in the balance between beneficial and harmful bacteria that are present in the human gut compared to healthy individuals <span class="citation">[<a href="references.html#ref-swidsinski2002" role="doc-biblioref">16</a>, <a href="references.html#ref-tamboli2004" role="doc-biblioref">17</a>]</span>.</p>
<!--End-->
<!--Elenea tesis -->
<p>Many studies show that patients with inflammatory bowel disease have less biodiversity.
Biodiversity is measured on <span class="math inline">\(\alpha\)</span> (alfa) and <span class="math inline">\(\beta\)</span> (beta) diversity.
<span class="math inline">\(\alpha\)</span>-diversity is a measure of the species present on a single sample and how much of each taxonomic group is there.
There are some reports of taxonomic changes and increase on <em>Enterobacteriaceae sp</em>, <em>Escherichia coli</em> (specially the invasive strain) at the mucosal layer of IBD patients <span class="citation">[<a href="references.html#ref-ott2004" role="doc-biblioref">18</a>]</span>.
At the same time there is often a reduction on protective species like <em>Bifidobacterium</em>, <em>Lactobacillus</em> and <em>Faecalibacterium.</em> Specially the <em>Faecalibacterium prausnitzii</em> is one microorganism of interest <span class="citation">[<a href="references.html#ref-kostic2014" role="doc-biblioref">19</a>, <a href="references.html#ref-sender2016" role="doc-biblioref">20</a>]</span>.
<span class="math inline">\(\beta\)</span>-diversity compares the diversity between samples, which typically looks like a horse-shoe-like shape when reducing the distances to their first two dimensions via principal component analysis.</p>
<!--Aida 2.3 p24-->
<p>In fact, it has been recently proposed that several unique microbial species can distinguish healthy controls from UC and CD patients <span class="citation">[<a href="references.html#ref-sankarasubramanian2020a" role="doc-biblioref">21</a>, <a href="references.html#ref-lopez-siles2014" role="doc-biblioref">22</a>]</span>.</p>
<div class="figure">
<img src="images/tesis_AM_Figure2.png" title="Microbial composition and epithelial barrier status in healthy versus IBD gut." alt="" />
<p class="caption">The microbial composition in a healthy and inflammed bowel disease gut.</p>
</div>
<!--End-->
<p>One of the proposed mechanism of interaction is through the chemical compounds they generate.
Chemical compounds may interact with the cells and modulate the state of the intestine.
One example of such molecular is butyrate which has been linked to microorganisms presents on healthy intestines and to induce a better healing of intestine cells <span class="citation">[<a href="references.html#ref-ferrer-picón2020" role="doc-biblioref">23</a>]</span>.</p>
<p>As previously mentioned, adherent invasive <em>Escherichia coli</em>, a proteobacteria specie, has been associated with IBD patients.
Adherent invasive strains are mainly found in ileal and colonic samples of CD patients and their presence in UC is less clear.
These adherent invasive cells enter trhough the epithelium of the more permable cells and live on their cytosol.</p>
<!-- End-->
<p>In addition, there has been some observations that mild IBD patients response to treatments that modify their gut microbiota, such as: fecal microbiota transplantation, probiotics and postbiotics.</p>
</div>
<div id="immune-response" class="section level4">
<h4><span class="header-section-number">1.1.1.3</span> Immune response</h4>
<p>As explained previously the immune system might be one of the reasons of the inflammatory bowel disease starts.
IBD is often characterized as an immune-mediated disease, if not outright autoimmune disease.</p>
<p>A loss of tolerance to commensal bacteria has been suggested as the underlying mechanism triggering the inflammation on the intestine.
The immune response involves many different cells lines and regions, which are important to know how they organize for a better understanding of the disease.</p>
<!--Image from Ana tesis p29: -->
<div class="figure">
<img src="images/tesis_AC_FigureX.png" title="The intestinal epithelial barrier" alt="" />
<p class="caption">The intestinal epithelial barrier.</p>
</div>
<p>From the luminal side of intestine, the first layer is the mucosa.
The mucus is organized in two layers: the inner layer, a firm mucus layer; and the outer, loose mucus layer <span class="citation">[<a href="references.html#ref-okumura2017" role="doc-biblioref">24</a>]</span>.
Below the intestinal epithelium there is a single layer of cells organized into crypts and villi (and circular folds on the large intestine) that carries out a diverse array of functions besides digestion performed by specialized cell lineages.
These cells usually come from the basal side of the crypt and mature as they move towards the apical side.</p>
<!--From tesis Aida p20-->
<p>Immune response in the intestinal mucous is mainly excreted by the gut associated lymphoid tissue <span class="citation">[<a href="references.html#ref-faria2012" role="doc-biblioref">25</a>]</span>.
Deregulation of the mucosal immune system has been associated with the pathogenesis of IBD <span class="citation">[<a href="references.html#ref-silva2016a" role="doc-biblioref">26</a>, <a href="references.html#ref-demattos2015" role="doc-biblioref">27</a>]</span>.
Genetically predisposed patients when exposed to certain environmental factors activate immune responses against microbials or self-antigens which in turn, may impair the mucosal barrier of the intestinal mucosa, the first physical barrier on the mucosal surface.</p>
<p>Both the adaptive and the innate immunity related cells are present on the intestine, right below the epithelium.
On IBD due to antigen translocation into the lamina propria, the immune response leads the adaptive cells to generate immune response to harmless components of the intestinal microbiota.
This initial response would induce a local increase in the production of pro-inflammatory cytokines and mediators which would damage the mucosa.
Therefore, the loss of integrity on this barrier enables the intestinal luminal bacteria to access the intestinal epithelium and to interact with the immune system underneath it more directly <span class="citation">[<a href="references.html#ref-hisamatsu2013" role="doc-biblioref">28</a>]</span>.</p>
<p>The intestinal epithelium is the next line of defense against bacterial invasion.
Intestinal epithelial cells play a key role in controlling the integrity of the physical barrier to the intestinal microorganisms <span class="citation">[<a href="references.html#ref-hisamatsu2013" role="doc-biblioref">28</a>]</span> not only physically but also secreting antimicrobial peptides and defensins, both of which are altered in IBD patients <span class="citation">[<a href="references.html#ref-michielan2015" role="doc-biblioref">29</a>]</span>.
The intestinal epithelium also plays a key role on the intake and diffusion of metabolites from the intestinal lumen to the lamina propia.</p>
<p>Damaging or increasing the permeability of the intestinal epithelium results on a response from the immune system.
Detecting signals of any foreign particle can also trigger the immune system.
On the intestine this starts with the identification of these signals by intestinal epithelal cells have pattern recognition receptors.
There are two main pattern recognition receptors: toll-like receptors (TLR), which are present on the surface, and nucleotide-binding oligomerization domain-like receptors (NOD-like receptors), present on the cytoplasm of the cells.
These receptors upon recognition of pathogen associated molecular patterns (PAMPs) start an amplifying signaling producing chemokines and cytokines which activates the transcription and translation of pro-inflammatory mediators to ensure an effective immune response.
Initially the innate response is triggered but the cells also increase the antigen presentation to T cells and thus activate the immune adaptive response.</p>
<p>Other cell types, such as monocytes, macrophages and dendritic cells also present the pattern recognition receptors.
From those, macrophages and dendritic cells are antigen presenting cells too and secrete several cytokines to activate other immune cells.
Usually CD patients express higher amounts of TLR than healthy individuals, which might trigger a stronger immune response.
This response is driven by CD4<sup>+</sup> T cells proliferation in secondary lymphoid tissues to T helpers in the presence of the antigens and cytokines nearby.</p>
<p>T helpers (T<sub>h</sub>) differentiate depending of the cytokines at which they are exposed.
Th type 1 are driven by exposure to IL-12 secreted primarily by dentritic cells.
T<sub>h</sub>2 are dtriven by cytokines secreted by macrophages.
The imbalance between T<sub>h</sub>1 and T<sub>h</sub>2-promoting cytokines determines the intensity and duration of the inflammatory response in experimental colitis <span class="citation">[<a href="references.html#ref-neurath1996" role="doc-biblioref">30</a>]</span>.</p>
<p>T<sub>h</sub>17-promoting cytokines are less well characterized in human.
T<sub>reg</sub> cells differentiate after exposure to cytokines IL-10, IFN-<span class="math inline">\(\gamma\)</span> and TGF-<span class="math inline">\(\beta\)</span>.
Overall the presence of certain cytokines and the response to self-antigens are factors leads to an inflammation and damage that is related to the onset and establishment of IBD</p>
<!--End tesis Aida/ more or less -->
<p>On these kind of diseases autologous hematopoietic stem cell transplantation has shown some benefits on IBD <span class="citation">[<a href="references.html#ref-corraliza" role="doc-biblioref">31</a>]</span>.
The benefit of Hematopoietic Stem Cell Transplant (HSCT) in autoimmunity is thought to originate from the depletion of auto-reactive cells regardless of their specificity.
However, due to its associated risk this therapy is only given when patients are refractory to all available therapeutic options.</p>
</div>
<div id="environmental" class="section level4">
<h4><span class="header-section-number">1.1.1.4</span> Environmental</h4>
<!--p26 Aida sec 2.4 -->
<p>Chronic inflammatory disorders and neoplasms have become the main cause of morbidity and mortality in the countries that follow a strategy of reducing the contact with anything dirty.
This decreases the human exposure to microbes which might affect the proper maturation of the immune system so that it provides a more ineffective immune response or exacerbated response towards “friendly” microbes <span class="citation">[<a href="references.html#ref-strachan1989" role="doc-biblioref">32</a>, <a href="references.html#ref-scudellari2017" role="doc-biblioref">33</a>]</span>.</p>
<p>From other environmental factors related to IBD such as tobacco, diet, certain drugs and stress; tobacco is the most influential environmental factor.
Surprisingly, it has an opposite effect on UC and CD: in CD tobacco is a risk factor that increases the risk of relapse and/or surgical intervention.
In UC, it has been observed that smoking cessation worsens the disease <span class="citation">[<a href="references.html#ref-thomas1998" role="doc-biblioref">34</a>]</span>.</p>
<p>Pharmacological treatments such as oral contraceptives, non steroidal anti-inflammatory drugs are also related to develop or relapse the disease <span class="citation">[<a href="references.html#ref-cornish2008" role="doc-biblioref">35</a>, <a href="references.html#ref-kaufmann1987" role="doc-biblioref">36</a>]</span>.</p>
<p>The psychological welfare of people also plays an important role in the disease progression, stress, anxiety and depression might be important in relapse and deterioration of the disease <span class="citation">[<a href="references.html#ref-bitton2008" role="doc-biblioref">37</a>]</span>.
Other environmental factors have been linked to IBD but without enough evidence to support a causative effect in the development of the disease.</p>
<!--End-->
<!-- Check with Elena Basolas Tesis p18-->
<p>Perhaps the environment factors influence the samples disease etiology towards Crohn’s disease or ulcerative colitis as there are some differences between them.
On CD a granulome, that is a region with big multinucleous cells, can appear on any intestinal layer, while on UC we observe a different damage consisting in many neutrophil in the crypt lumen.</p>
<p>To distinguish between the diseases, in addition to the inflamed location(s), mosaic zones (patches of inflamed and non-inflamed areas) are more characteristic of CD <span class="citation">[<a href="references.html#ref-bassolasmolina2018" role="doc-biblioref">38</a>]</span>.
Another indicator of CD is an increase of calprotectin in fecal samples <span class="citation">[<a href="references.html#ref-sands2015" role="doc-biblioref">39</a>]</span>.</p>
<!-- End -->
</div>
</div>
<div id="crohns-disease-physiology" class="section level3">
<h3><span class="header-section-number">1.1.2</span> Crohn’s disease physiology</h3>
<p>As previously introduced, Crohn’s disease is a chronic inflammatory disorder characterized by alternating periods of remission and clinical relapse it is frequently associated with extraintestinal manifestation and/or concomitant immuno-mediated diseases.
Inflammation on the gastrointestinal tract is transmural and can affect from the mouth to the anus, but mainly it manifests on the ileum and colon <span class="citation">[<a href="references.html#ref-demattos2015a" role="doc-biblioref">40</a>]</span>.</p>
<div id="clinical-presentation" class="section level4">
<h4><span class="header-section-number">1.1.2.1</span> Clinical presentation</h4>
<p>The disease itself manifest an heterogeneous symptoms that can involve, diarrhea, weight loss, abdominal pain, fever, anorexia, malaise.
Other less frequent co-occurring manifestations are arthritis, primary sclerosing cholangitis, skin disorders venous or arterial thromboembolism and/or pulmonary involvement <span class="citation">[<a href="references.html#ref-baumgart2012" role="doc-biblioref">41</a>]</span> .
These symptoms make it hard to correctly diagnose the disease by non-specialists, in addition there isn’t a non invasive easy procedure to diagnose it.
All these can lead to delays on correct diagnosis of the disease.</p>
<!--Tesis Ana p21-->
<p>The detection of parasites or bacteria, such as <em>Clostridium difficile</em>, have been associated with CD although they could be just an infection.
The detection of fecal calprotectin, is generally a good marker of endocopic activity with sensitivity above 70% and specificity above 80% <span class="citation">[<a href="references.html#ref-guardiola2018" role="doc-biblioref">42</a>]</span>.</p>
<p>Usually the best diagnosis method is to perform a colonoscopy, whether there is inflammation on the gastroinestinal tract on discontinuous regions then is Crohn’s disease.
This inflammation could also present ulceration and rectal sparing and histological lesions which also help to diagnose the patients <span class="citation">[<a href="references.html#ref-corralizamárquez2019" role="doc-biblioref">43</a>]</span>.</p>
<!--End-->
<p>To address this difficulty the Montreal classification aims to classify patients according to their age of disease onset, standardized anatomical disease location an disease behaviour (See <a href="introduction.html#clinical-care-UC">below</a>).
This classification assumes that the location of Crohn’s disease remains stable over time after diagnosis but behavioral phenotypes change.
Most patients progress from an inflammatory phenotype to a stricturing or penetrating one <span class="citation">[<a href="references.html#ref-satsangi2006a" role="doc-biblioref">44</a>]</span>.</p>
<!--Add Figure 2 from ACM thesis?-->
<!---->
<p>To some extent, there is a disassociated between patient’s well-being report and the colonoscopy observation.
Often patients report feeling better but their gut is as inflamed as previously <span class="citation">[<a href="references.html#ref-bhattacharya2016a" role="doc-biblioref">45</a>]</span>.
This has lead to several scores and thresholds used on research and treatment of the patients.</p>
</div>
<div id="disease-course" class="section level4">
<h4><span class="header-section-number">1.1.2.2</span> Disease course</h4>
<p>In the early stages of the disease the relapsing and remitting course is more frequent.
Often relapses are accompanied by clinical symptoms, and very few have prolonged clinical remission <span class="citation">[<a href="references.html#ref-peyrin-biroulet2010" role="doc-biblioref">46</a>]</span>.
When there is clinical remission, there can still remain some other lesions and often sublinical inflammation persists.
Frequently the damage caused by the disease evolves to fibrostenotic stricture or penetrating lesions (fistula and abscess).</p>
<p>The damage of the disease might not be apparent to patients and might be only seen several years later than the first detection <span class="citation">[<a href="references.html#ref-bhattacharya2016" role="doc-biblioref">47</a>]</span>.
Mucosal healing is a first step towards the healing of deeper layers of the inflamed bowel wall on the Crohn’s disease.</p>
</div>
</div>
<div id="ulcerative-colitis-physiology" class="section level3">
<h3><span class="header-section-number">1.1.3</span> Ulcerative colitis physiology</h3>
<p>The phenotype of the ulcerative colitis is defined by the extension of the inflammation.
Around a third of the patients with ulcerative colitis suffer proctitis, the inflammation of the final segment of the colon.
The next more common manifestation of the disease is the distal colitis when the rectum to the sigmoid colon are affected.
Left colitis, where the affected area starts on the rectum and extends to the left colon is also common.
Next most common phenotype is the extensive colitis were there is involvement of at least the descending colon.
The last common phenotype is the pancolitis, when the disease affects all the colon.
The extension and severity of the disease correlates with the clinical observation, where the longer the extension of the colitis is, the worse the prognosis is <span class="citation">[<a href="references.html#ref-etchevers2009" role="doc-biblioref">48</a>]</span>.</p>
<div id="clinical-care-UC" class="section level4">
<h4><span class="header-section-number">1.1.3.1</span> Clinical care</h4>
<p>The goal of the clinical care is to recover and reach a healthy life.
As a first step, the symptoms of IBD have to lessen to the point that they’re mostly absent, gone, or barely noticeable, this is known as clinical remission.
However, this is not enough as the mucosa might be still inflammed and thus the reconstitution of the structure and function of the intestinal epithelial barrier not complete.
Other lesions, might aid to the progression to other phenotypes such as fibrostenotic stricture or penetrating lesions increasing the structural bowel damage or primary sclerosing cholangitis <span class="citation">[<a href="references.html#ref-boonstra2012" role="doc-biblioref">49</a>]</span>.</p>
<p>To prevent and avoid further damage several procedure are followed:</p>
<p>When there is colorectal cancer or the damage on the colon has been too big a surgical procedure to remove part or all of the colon must be done.<br />
Patients that undergo a colectomy need to have their bowel reconnected with a procedure called ileoanal anastomosis (also know as J-pouch by the sape it takes) surgery.
This pouch often ends up inflammating starting onn the lining of the pouch created during surgery and extending from there <span class="citation">[<a href="references.html#ref-schieffer2016" role="doc-biblioref">50</a>]</span>.</p>
<p>Dysplasia, an abnormal development of cells within tissues or organs, is often associated with chronic IBD patients, which is considered a precedence before colorectal cancer develops <span class="citation">[<a href="references.html#ref-mark-christensen2018" role="doc-biblioref">51</a>]</span>.</p>
<p>Fistula, an abnormal connection between two body parts, is often found on the penetrating phenotype of IBD patients.</p>
<p>Many scores have been proposed for several purposes, from quality of life to disease severity or patient status.
Among the scores most used are the following:</p>
<ul>
<li><p>Mayo: A score designed to be simple to calculate at the bedside based on stool frequency, bleeding, mucosal apperance at endocopy and disease activity <span class="citation">[<a href="references.html#ref-schroeder1987" role="doc-biblioref">52</a>]</span>.</p></li>
<li><p>SES-CD: simple endoscopic score for Crohn’s disease <span class="citation">[<a href="references.html#ref-daperno2004" role="doc-biblioref">53</a>]</span>.
Score based on size of ulcers, ulcerate surface percentage, affected surface and presence of narrowings on the bowel.</p></li>
<li><p>IBDQ: A 32 questionnaire used to assess the quality of life grouped into four categories: bowel, systemic, social and emotional <span class="citation">[<a href="references.html#ref-irvine1999" role="doc-biblioref">54</a>]</span>.</p></li>
<li><p>Montreal classification allows for early onset of disease to be categorized separately those with age of diagnosis at 16 years or younger, diagnosis at 17–40 years and &gt;40 years, respectively <span class="citation">[<a href="references.html#ref-satsangi2006" role="doc-biblioref">55</a>]</span>.</p></li>
</ul>
<p>Other measured parameters include, weight, effective weight, calprotectin, Proteinc C reactive (PCR), hemoglobine.</p>
</div>
<div id="drugs-and-treaments" class="section level4">
<h4><span class="header-section-number">1.1.3.2</span> Drugs and treaments</h4>
<p>Many drugs can given, although this varies between patients and time according to recurrent meetings doctors have and current recomendations: anti-TNF<span class="math inline">\(\alpha\)</span>, vedolizumab, ustekinumab, risankizumab, tofacitinib, infliximab, adalimumab a 5-asa treatement, corticoesteroids, Azathiprine/Mercaptopurine 6MCP, Methotrexate MTX, Tacrolimus FK, Cyclosporin A CyA. All these drugs and procedures are available at the time of writing, and patients are taking one or more of those drugs as standard of care to have a good quality of life and heal.</p>
<p>To reduce secondary effects of drugs or to find the right dose of the drugs often a reduction of dose is recommended.
Dose reduction often results in losing the response to the drug.
Thus, after an increase of the dose to the previous levels the patients not recover the initial response they had and doctors need to recommend other drugs or combinations of drugs.</p>
<p>If the drugs fail to contain the inflammation and heal the epithelium doctors might recommend to perform a surgery to remove the damaged section.
In some cases HSCT is recommended which have shown to improve the life of the patients <span class="citation">[<a href="references.html#ref-corralizab" role="doc-biblioref">56</a>]</span>.</p>
<p>This is a new procedure given only to the most extreme cases for which several publications have checked that it resets the immunological state of the patient.
Currently it is only performed on the Hospital Clínic (Barcelona).</p>
<p>Last there are several proposal of fecal microbiota transplantation, which are not done on the Hospital Clínic yet.
But are proposed to reset or change the intestinal microbiome of the patients to help them on the disease.
This has been doing experimentally on mice and mouse for some time and some experiments on patients resulted have been performed already.</p>
</div>
</div>
<div id="summary" class="section level3">
<h3><span class="header-section-number">1.1.4</span> Summary</h3>
<p>Inflammatory bowel disease is a complex disease that impacts the health of many people in different ways.</p>
<p>Current clinical care in some cases is enough to have a sustained clinical and endoscopic remission but most often is not enough and relapse is expected.
Several factors, such as becoming refractory to drugs, intermittent discourse of the disease, make the treatment complex.</p>
<p>The lack of knowledge of what are the factors of the ethiology and causes of the disease make those treatments and drugs to be addressed to block further inflammation and damage, but cannot prevent it and often they do not stop it completely.</p>
</div>
</div>
<div id="studies-and-integration-on-ibd" class="section level2">
<h2><span class="header-section-number">1.2</span> Studies and integration on IBD</h2>
<p>Many studies have looked up to the origin of the disease.
As seen, one of the hypothesis behind the maintenance of the inflammation involves the microbiome and the host epithelium.
This has been studied using several data sources, mostly from sequencing data.
The technical methods used to obtain the data can differ between extracted from biopsied samples at colonoscopy or from surgical samples.</p>
<p>To confirm findings usually RT-PCR, immunohistochemistry, histopathology, immunohistochemistry, or fluorescence in situ hybridization is performed.
These techniques allow to visualize where are the cells expressing certain proteins or genes, thus helping with the analysis validation.</p>
<p>Furthermore several studies have been carried out to discover links between microbiome and the inflammation, followed by those looking for some relationship between genetics and the disease and more recently the metabolome.
These studies, known as integration, multi-omic or interaction studies, usually use multiple sequencing assays as the bases of the analysis <span class="citation">[<a href="references.html#ref-beck2021" role="doc-biblioref">57</a>]</span>.</p>
<p>However, confirming the interactions of the variables of each assay is harder as the above mentioned techniques can’t show these relationships.
Very rarely there is an experimental confirmation of the relationships between variables of the different essays because it is complicated to test an interaction and to set up the right conditions for the many variables that are accounted for on the integrations.
Some articles to find the relationships use correlation, like <span class="citation">[<a href="references.html#ref-hasler_uncoupling_2016" role="doc-biblioref">58</a>]</span>, there are others that use a combination of methods from correlations, partial correlations to integrative methods <span class="citation">[<a href="references.html#ref-tang_integrated_2017" role="doc-biblioref">59</a>–<a href="references.html#ref-huWholeExomeSequencing2021" role="doc-biblioref">61</a>]</span> and network integrations.</p>
<p>One of the few methods published that shows an interaction between genes and microorganisms on the inflammatory bowel disease is to expose the <em>ex-vivo</em> sample or cell lines with microbiomes or supernatant of at their culture <span class="citation">[<a href="references.html#ref-mayorgas2021" role="doc-biblioref">62</a>]</span>.</p>
<p>Most of the integrations refer some other source of data to the transcriptomics of the patient.
The transcriptome of the patients has been extensively studied.
There are known marker gens of inflammation and many research focus on identifying prognosis predictors, treatment response prediction based on gene expression.
Recently single-cell technology has enabled to estimate cell populations of the samples with better degree of success than bulk RNA-sequencing, which is helpful to understand the immune response and the population change of the cells.
Single cell technologies are starting to be used for integration.</p>
<div id="transcriptome" class="section level3">
<h3><span class="header-section-number">1.2.1</span> Transcriptome</h3>
<p>Many studies used the transcriptome of colonoscopy samples or from surgery.
The different inflammatory profile of the CD and the UC and the differences between the transcriptome in ileum and colon samples, has been learned that way <span class="citation">[<a href="references.html#ref-planell2013" role="doc-biblioref">63</a>, <a href="references.html#ref-leal2015" role="doc-biblioref">64</a>]</span>.</p>
<p>Notably this is probably the most common data available from IBD patients, probably due to existing commercial solutions to gather this data and the commercial and experience with this kind of protocols.</p>
</div>
<div id="microbiome-1" class="section level3">
<h3><span class="header-section-number">1.2.2</span> Microbiome</h3>
<p>Many of the integration analysis on inflammatory bowel are done between host transcriptome and the microbiome.
These studies use datasets from IBD patients usually stratified by some inflammatory index or location of the disease.
Most of them are based on correlation analysis between the microbiome and RNA-seq <span class="citation">[<a href="references.html#ref-hasler_uncoupling_2016" role="doc-biblioref">58</a>]</span>.
Conclusions of these integrations range from finding differences on the correlation depending on the disease (<span class="citation">[<a href="references.html#ref-hasler_uncoupling_2016" role="doc-biblioref">58</a>]</span>) to finding relationships with inflammatory genes <span class="citation">[<a href="references.html#ref-tang_integrated_2017" role="doc-biblioref">59</a>]</span>.</p>
</div>
<div id="genetics-1" class="section level3">
<h3><span class="header-section-number">1.2.3</span> Genetics</h3>
<p>Genetics is the next most common data source used to integrate data on the inflammatory bowel disease.
Most studies on genetics and IBD are genome-wide association studies <span class="citation">[<a href="references.html#ref-kumar2019" role="doc-biblioref">11</a>, <a href="references.html#ref-huWholeExomeSequencing2021" role="doc-biblioref">61</a>, <a href="references.html#ref-knights2013" role="doc-biblioref">65</a>]</span>.
No clear mechanism has been proved despite clear and significant findings, probably because inflammatory bowel disease is relapses and remit with time.</p>
<p>When using genetic data to integrate it with transcriptomics it is usually to understand how a genetic variant is affecting a gene expression.</p>
</div>
<div id="metabolome" class="section level3">
<h3><span class="header-section-number">1.2.4</span> Metabolome</h3>
<p>More recently an increased interest on metabolome has raised, given that microorganisms interact with the host also via their products and metabolites.
Some studies have integrated the metabolome with the RNAseq and state of the epithelium <span class="citation">[<a href="references.html#ref-ahmed2016" role="doc-biblioref">66</a>, <a href="references.html#ref-gallagher2021" role="doc-biblioref">67</a>]</span>.</p>
<p>There is evidence some of these metabolome products, such as small chemicals regulate the epithelium cells and their interaction with the microbiome and outer agents <span class="citation">[<a href="references.html#ref-ferrer-picón2020b" role="doc-biblioref">68</a>]</span>.</p>
</div>
<div id="summary-1" class="section level3">
<h3><span class="header-section-number">1.2.5</span> Summary</h3>
<p>Multiple methods and multiple studies have been used to learn more about the interaction between different data and understand the inflammatory bowel disease.
The task is challenging and with many different aspects one could use.</p>
<p>There are many studies focused on understanding different aspects of the disease: the relation between location and severity, relapse, remission, treatments, origin.
The integration of data might help to improve the medicine and reveal links in difficult diseases like IBD.</p>
</div>
</div>
<div id="integration" class="section level2">
<h2><span class="header-section-number">1.3</span> Integration</h2>
<p>Data integration is informally widely used with varying definitions, according to the dictionary integration is defined as:</p>
<blockquote>
<p>“the process of combining two or more things into one” — <a href="https://dictionary.cambridge.org/dictionary/english/integration">Cambridge Dictionary</a></p>
</blockquote>
<p>Other words that are used are integrati(-on, -ve), if specific to data from sequencing technologies multi-omics, pluri-omics.
Here integration will be used as it is the more general one and not restricted to omics or sequencing technologies.
Since the beginning of the integration methods there have been many methods proposed.
Some of the early methods were initially used for surveying the agreement of different evaluating systems, others were developed for agricultural sciences <span class="citation">[<a href="references.html#ref-hotelling_relations_1936" role="doc-biblioref">69</a>]</span>.
Some of these methods are specific for one application or data while others are more general.
Lately, the increase of bigger datasets with more variables and often from the same samples has increased the focus of the research community on the methodologies available on several disciplines but mainly on the biological science.
The explosion of data on the biological science has been driven by the new sequencing technologies that allow to measure thousand of variables of many samples at the same time, if done with multiple sequencing tecnologies it is usually referred as multi-omics methods, which usually only integrate omic data.</p>
<p>It has been observed the importance of classification, review and comparison of the tools available, as well as, benchmarking these tools against the same dataset as a way to provide clear recommendations to anyone wishing to use them <span class="citation">[<a href="references.html#ref-wu_selective_2019" role="doc-biblioref">70</a>]</span>.
Part of these efforts uses the methods’ strategies to classify them <span class="citation">[<a href="references.html#ref-cavill_transcriptomic_2016" role="doc-biblioref">71</a>, <a href="references.html#ref-chongComputationalApproachesIntegrative2017" role="doc-biblioref">72</a>]</span>.
Following this view I’ll review the integration methods according to several axis: type of data used, aim of the method, relationships between variables, relationships between samples, relationship between variables and samples, input data, mathematical framework and results of the method.</p>
<div id="classification-of-integration-methods" class="section level3">
<h3><span class="header-section-number">1.3.1</span> Classification of integration method’s</h3>
<p>Here I outline how to classify integration methods in the bioscience field, mainly using omics datasets, with relationships (and references) to concrete methodology and in occasions to articles using them.</p>
<div id="data-type-numeric-vs-categorical" class="section level4">
<h4><span class="header-section-number">1.3.1.1</span> Data type: numeric vs categorical</h4>
<p>The most important distinction in integration methods is what kind of data are combined.
In general data can be divided between categorical and numeric variables, which are usually found in several fields.
Sometimes doctors want to understand the relationship between a phenotype they observe and the underlying mechanism.
Usually this involves looking how the metabolites, the gene expression, the methylation, the number of variants a gene has, and other numeric variables are related to the observed (categorical) phenotype (like pain).</p>
<p>Depending on what does a method aim for it handles both data types or just one, often they are used differently.
The most common way to handle different type of data is converting the categorical values to a mock or dummy variable.
For each categorical factor there is a new variable whose value is 1 if that sample had this factor and 0 otherwise.
For instance, if the categorical variable has three values (A, B, C) it would be converted to A (1, 0, 0) B (0, 1, 0) and C (0, 0, 1).
Often the number of variables created is one less than the number of factors that existed, on this example only A and B would be kept.
This transformation allows to use the categorical values in methods though for numeric variables.</p>
<p>If the method only accepts categorical data but you want to provide numeric values usually those values are categorized.
For example if a variable is (0.123, 0.25, 0.56, 0.78) one could make to categorical values like (“&lt;0.5”, “&lt;0.5”, “&gt;0.5”, “&gt;0.5”).
The number of categories to use and how is the numeric value split depends on case by case.</p>
</div>
<div id="objective" class="section level4">
<h4><span class="header-section-number">1.3.1.2</span> Objective</h4>
<p>The objective of the method might be one of the most important defining properties.
Sometimes the objective of a methods is formally defined as a function to be optimized or a procedure to be followed until certain conditions are meet.</p>
<p>Data integration can be classified according to their objective and the (biological) question they try to answer.
Most of the times one (or all) of the following results are expected from integration methods:</p>
<ul>
<li>An overview of the role of each individual block in a biological system</li>
</ul>
<p>Sometimes the question is which omic method is the best one to describe the disease.
This could prevent to do expensive tests that can be estimated by other cheaper or easier techniques that are enough sensitive and specific for the task.
Some examples of this is the search of markers on blood to identify if there are cancer cells on tissues.</p>
<ul>
<li>A better understanding of the relationships between the ’omics types</li>
</ul>
<p>Several methods are used to understand better how the different data sets interact.
When the relationship between omics is known further focused methods are used to understand the mechanistic relationship between them.
For instance, checking that in a particular case or condition is present on a given relationship, and that this relationships follows our model or not.</p>
<ul>
<li>Finding a molecular signature.</li>
</ul>
<p>A signature is usually a group of features that describe/are representative of a cell line, a process or a stage.
Identifying a subset of the variables from the omics that are related is often a desired goal because it reduces the amount of variables allowing to perform experiments on the bench on just those that might be important.
In other fields, such as machine learning, selecting the important variables is known as <a href="https://en.wikipedia.org/wiki/Feature_selection" title="Wikipedia page of feature selection">feature selection</a>.
There are several methods that are used to do this.</p>
<ul>
<li>A predictive model</li>
</ul>
<p>Predictive models usually require a very good understanding of the current or past relationships, as well as, a good feature selection.
Often to make a prediction the model is first evaluated on a previously known data according to this features.
If a good model of previous data exists it might be used to predict future events.
Sometimes, models are only build to predict events without being able to accurately describe the mechanism, why they work.
This kind of models might improve treatments, diagnosis and prognosis prediction.</p>
<ul>
<li>Impute values</li>
</ul>
<p>Some methods aim to make an accurate guess of which values does have one block given some other information.
Missing values can happen for a variety of reasons from practical ones, like a sample not available, to technical ones, such as laboratory method failing.
However, this is often a intermediate step to other goals.</p>
<p>To complete these goals it is important to have enough statistical power to determine the significance of tests performed (if any) and to understand how complete are the data sources used on the integration <span class="citation">[<a href="references.html#ref-tarazona2021" role="doc-biblioref">73</a>]</span>.
Having more statistical powers helps identifying the relationships one seeks when using this methods.</p>
</div>
<div id="relationship-between-variables-and-samples" class="section level4">
<h4><span class="header-section-number">1.3.1.3</span> Relationship between variables and samples</h4>
<p>Depending on the amount of variables and in which samples they have been measures we can classify two types of integration.
Traditionally for each sample few variables are measured, for instance on a biopsy with RT-PCR only a few genes are measured, however with the new omics techniques (transcriptomics, metabolomics, methylomics, genomics), thousands of variables are measured for the same sample.
This has lead to the following situation:</p>
<ul>
<li>More variables than samples</li>
</ul>
<p>For a single sample of RNA around 50k genome identifiers (genes, long non coding RNAs, iRNA, pseudogenes,…) can be measured.
Which leads to the case where there are many more variables than samples.
Thus high-throughput data analysis typically falls into the category of <span class="math inline">\(p \gg n\)</span> problems ( big p, little n), where the number of genes or proteins, <span class="math inline">\(p\)</span>, is considerably larger than the number of samples, <span class="math inline">\(n\)</span>.
With such high number of variables the identification of the relevant variables is hard because variables will co-variate.
When many variables are tightly correlated, discovering which one is important using just numerical methods is very difficult.
It is even more difficult when looking for causal relationships.</p>
<ul>
<li>More samples than variables</li>
</ul>
<p>This was the usual case when for instance, when from a cohort of patients the temperature is measured along the stage of a disease: two variables for each sample.
If there are more than 2 patients, then the number of samples is greater than the number of variables studied.
This is described in the literature as <span class="math inline">\(n \gg p\)</span> (or big n, little p).
Nowadays this is less frequent on the bioscience world, and does not causes trouble analyzing it because the high number of samples allow to accurately estimate variables dispersion.</p>
<p>Having just the enough amount of samples for the desired statistical power is not enough if the samples of different blocks do not match between them.</p>
</div>
<div id="relationship-between-samples" class="section level4">
<h4><span class="header-section-number">1.3.1.4</span> Relationship between samples</h4>
<p>Depending on the relationship between the samples of the different blocks of data, the questions answerable and the methods that work on them differ.</p>
<p>A sample can have multiple or one data source.
In a study if all the samples have all the data from multiple data sources it is a complete case.
If some samples have data from some data sources but not from others the study is not a with a complete case.</p>
<p>Sometimes because the sample is not enough, or there are some technical or organizational problems we might lose a source of data for a sample (which is known as an incomplete case).
This results in a new source of variation that has to be dealt with, which complicates the conclusion one can draw from the studies of these kind of data.</p>
<p>Even when all the cases of a patient are complete the samples can come from several sites of the same individual or with different combinations of variables, which makes is relevant to understand the relationships between the different samples.</p>
<p>There is no easy classification of this as each experiment might be designed differently.
In general, experiments are designed to be as consistent as possible but in face of adverse events that become a variation of the design the analysis complicates.
Either some data is imputed or some samples are omitted for the analysis, or if the differences are minor these samples are used as being from a different group.
This can happen with samples taken at different timepoints as patients might not come to the visit with the doctor on time.</p>
<div id="time" class="section level5 unnumbered">
<h5>Time</h5>
<p>As mentioned above, time is one of the factors that sometimes cannot be controlled, despite having programmed visits every two weeks some patients might come early or later due to multiple reasons (holidays, other conflicting duties, …).</p>
<p>Sometimes, precisely the objective of the study is to analyze the relationships at different time, or see how the relationships change with time.
To discover causality between two variables the cause must be before the consequence, which highlights the importance of time.
Being aware of the time differences and time scales is crucial in most cases.</p>
<p>On cell lines or other lab experiments conditions can be repeated even if they are at different time points.
However, with patients we can not obtain a sample at the same timepoint.
This makes it harder to study time-related change on patients.</p>
<p>Lastly, time between the collection of a fresh samples and its processing also influences the readings of the samples of the omics technology, specially RNA-seq <span class="citation">[<a href="references.html#ref-massoni-badosa2020" role="doc-biblioref">74</a>, <a href="references.html#ref-zhu2017" role="doc-biblioref">75</a>]</span>.
Some features and genes are more affected by time than others but as they are measured at the same time it might distort the data.
Processing time is also very hard to keep track on practical settings and requires a highly coordinated effort <span class="citation">[<a href="references.html#ref-ferreira2018" role="doc-biblioref">76</a>]</span>.</p>
</div>
</div>
<div id="relationship-between-variables" class="section level4">
<h4><span class="header-section-number">1.3.1.5</span> Relationship between variables</h4>
<p>Once the data is collected, the next step is understand the relationship between the variables present.
As seen, some variables influence other variables which affects the outcome in a complex ways.
With many variables present in a dataset it is important to be aware of known relationships between variables.
Even in a simple dataset, like a RNA-seq dataset, it is important to be aware of the relationships between variables.</p>
<p>Since the lactose <a href="https://en.wikipedia.org/wiki/Operon">operon</a> we know how some genes regulate each other.
We don’t know how other variables are related between them.
For instance, how does the increase in expression of a gene affects the growth of a microorganism?
Usually the relationships between variables are mediated by many factors or interactions.</p>
<p>One of the best example of such interactions is when some variables correlate.
Their correlation can be used to reduce the number of variables being analyzed by ignoring the relationships between them and using the most representative variable (less correlated and with more variation).
This step is usually done by the dimension reduction methods… However, sometimes this is not desired or feasible as the correlation doesn’t explain the direction of the causality of the interaction between the variables (if there is any).</p>
<p>Network approaches relate the variables between them (such as <span class="citation">[<a href="references.html#ref-koh_iomicspass_2019" role="doc-biblioref">77</a>]</span>).
These approaches are fairly new and growing in popularity partly because they can address the direction of the interaction.</p>
<p>In partial correlations some or all of the other variables are considered on how much do they affect the others and this effect is reduced.
This assumes a linear relationship and is computationally expensive when there are thousands of variables.</p>
</div>
<div id="input-data" class="section level4">
<h4><span class="header-section-number">1.3.1.6</span> Input data</h4>
<p>Some of the methods to account for relationships of variables only work when data is from the same patients on each data set used to integrate while other do not.</p>
<ul>
<li>Data from the same samples:</li>
</ul>
<p>These methods do not handle well or at all missing data.
They need complete cases/data of the samples in order to be able to integrate the results.
These methods include Regularized Generalized Canonical Correlation Analysis (RGCCA) <span class="citation">[<a href="references.html#ref-tenenhaus_regularized_2011" role="doc-biblioref">78</a>, <a href="references.html#ref-tenenhaus_variable_2014" role="doc-biblioref">79</a>]</span>, Multiple co-inertia analysis (MCIA) <span class="citation">[<a href="references.html#ref-culhane_cross-platform_2003" role="doc-biblioref">80</a>]</span>, Multi-Study Factor Analysis (MSFA) <span class="citation">[<a href="references.html#ref-vito_multi-study_2019" role="doc-biblioref">81</a>]</span>, Multi-Omics Factor Analysis (MOFA) <span class="citation">[<a href="references.html#ref-argelaguet_multi-omics_2018" role="doc-biblioref">82</a>]</span>, STATegRa <span class="citation">[<a href="references.html#ref-gomez-cabreroSTATegraComprehensiveMultiomics2019" role="doc-biblioref">83</a>]</span>.</p>
<ul>
<li>Data from different samples:</li>
</ul>
<p>These methods do not need data from the same sample.
They draw their conclusions generalizing from the the data available.
Some of them handle missing data, while others do use the data at face value.
These method includes MetaPhlAn2, HUMAnN, LEfSe <span class="citation">[<a href="references.html#ref-franzosaSpecieslevelFunctionalProfiling2018" role="doc-biblioref">84</a>–<a href="references.html#ref-segataMetagenomicBiomarkerDiscovery2011" role="doc-biblioref">86</a>]</span>.</p>
<p>Furthermore, some methods are designed to integrate specific types of datasets, (usually because they make some assumptions that are only met on that kind of data).
For instance, HCG, 16S, RNA-seq and metabolomics do not share the same data distribution, and are different between them.
Also even with the same data depending on the processing of the data they can have very different properties: OTUs (operational taxonomic unit) properties are not the same as ASV (amplicon sequence variants).</p>
</div>
<div id="mathematical-framework" class="section level4">
<h4><span class="header-section-number">1.3.1.7</span> Mathematical framework</h4>
<p>Depending on the input and the objective methods use different mathematical framework to process the data.
Some of them have previously appeared:</p>
<ul>
<li><p>Networks</p>
<p>Networks methods were mentioned because they use and find information about interaction of variables.
Multilayer networks, including the multiplex, Molti-C-DREAM<span class="citation">[<a href="references.html#ref-didier2018" role="doc-biblioref">87</a>]</span>, RWR-MH, RWR-M <span class="citation">[<a href="references.html#ref-valdeolivas2019" role="doc-biblioref">88</a>]</span>.
Network embedding MultiVERSE are some of the methods using networks <span class="citation">[<a href="references.html#ref-pio-lopez2021" role="doc-biblioref">89</a>]</span>.</p>
<p>Bayesian approaches are also quite frequent, these methods use the Bayes theorem to see the relationships between variables.
Some methods that use these approaches are <a href="http://research.mssm.edu/integrative-network-biology/RIMBANET/RIMBANET_overview.html">RIMBANET</a> <span class="citation">[<a href="references.html#ref-zhu2012" role="doc-biblioref">90</a>]</span>, <a href="https://github.com/ttriche/bayesCC">BCC</a> <span class="citation">[<a href="references.html#ref-lock2013" role="doc-biblioref">91</a>]</span> and others like <span class="citation">[<a href="references.html#ref-virtanen2012" role="doc-biblioref">92</a>]</span></p></li>
<li><p>Dimensional Reduction</p>
<p>These methods focus on finding just a few variables and summarizing them using some function that has some properties.
The selection of variables is usually done with L1 or Lasso Regression regularization technique or L2 also known as Ridge Regression.
L1 regularization adds a penalty equal to the absolute value of the magnitude of coefficients which leads to some coefficients can become zero and the variable eliminated from the model.
On the other hand, L2 regularization doesn’t result in elimination of coefficients or sparse models and can only be used when there is multicollinearity.</p>
<p>Momix <span class="citation">[<a href="references.html#ref-cantini_benchmarking_2020" role="doc-biblioref">93</a>]</span>, RGCCA <span class="citation">[<a href="references.html#ref-tenenhaus2017" role="doc-biblioref">94</a>]</span>, mixOmics <span class="citation">[<a href="references.html#ref-rohart2017" role="doc-biblioref">95</a>]</span>, STATegRa <span class="citation">[<a href="references.html#ref-gomez-cabrero2019" role="doc-biblioref">96</a>]</span>, …</p></li>
<li><p>Active module identification</p>
<p>Multiomic objective genetic algorithm (scores based in two metrics, node score and density of interactions score).
MOGAMUN <span class="citation">[<a href="references.html#ref-novoa-del-toro2020" role="doc-biblioref">97</a>]</span></p></li>
</ul>
<p>Usually depending on the mathematical framework used they return these methods return similar output.</p>
</div>
<div id="output-results" class="section level4">
<h4><span class="header-section-number">1.3.1.8</span> Output results</h4>
<p>According to the output the integration methods can be classified in several groups:</p>
<p>For the network methods the following output is usually returned:</p>
<p>Connection between the variables/nodes, a measure of how strong is the connection (or simply if there is a connection or if there isn’t one).</p>
<p>For those from dimensional reduction methods there are three: Shared factor across the data, specific factors for each data or mixed factors.</p>
<ul>
<li>Shared factors:</li>
</ul>
<p>The integration results in a vector of the samples in a lower dimensional space that is shared by all the data used to integrate.
Such methods include iCluster, Multi-Omics Factor Analysis (MOFA) <span class="citation">[<a href="references.html#ref-argelaguet_multi-omics_2018" role="doc-biblioref">82</a>]</span>.</p>
<ul>
<li>Specific factors:</li>
</ul>
<p>The integration results in several vectors of the samples in a lower dimensional space of each data used to integrate.
Such methods include Regularized Generalized Canonical Correlation Analysis (RGCCA) <span class="citation">[<a href="references.html#ref-tenenhaus_regularized_2011" role="doc-biblioref">78</a>, <a href="references.html#ref-tenenhaus_variable_2014" role="doc-biblioref">79</a>]</span>, Multiple co-inertia analysis (MCIA) <span class="citation">[<a href="references.html#ref-culhane_cross-platform_2003" role="doc-biblioref">80</a>]</span>, Multi-Study Factor Analysis (MSFA) <span class="citation">[<a href="references.html#ref-vito_multi-study_2019" role="doc-biblioref">81</a>]</span>.</p>
<ul>
<li>Mixed factors:</li>
</ul>
<p>The integration results in both previous factors, specific of each data and common to all the data.
Such methods include Joint and Individual Variation Explained (JIVE), integrative Non-negative Matrix Factorization (<a href="https://github.com/yangzi4/iNMF">iNMF</a>) <span class="citation">[<a href="references.html#ref-yang2016" role="doc-biblioref">98</a>]</span>.</p>
<p>These methods are used to understand and discuss the results with previous knowledge or independent sources of data.</p>
</div>
</div>
<div id="interpretation" class="section level3">
<h3><span class="header-section-number">1.3.2</span> Interpretation</h3>
<p>How to interpret the results of the methods is highly linked to understanding the method and its output.
On a correlation between two variables, the interpretation of the analysis is clear, if one variable increase, the other one too.
The implications of these observation can be far reaching but the principles to understand them are simple.</p>
<p>However, on more complex methods the interpretation becomes less clear.
The interpretation of a canonical correlation analysis is much harder <span class="citation">[<a href="references.html#ref-sherry2005" role="doc-biblioref">99</a>]</span>.
Also on more complex methods the number of parameters required increases so the time and intellectual effort to understand the relationships between the parameters is also higher.</p>
<p>The interpretation also helps to discuss the results and relate it to other previously know information.</p>
<ul>
<li>Individually:</li>
</ul>
<p>How each variable relates to another, like in the correlation analysis, the relationship between two variables under study.
Or by patient: how do interpret that in these patient variable A and B is X and Y?</p>
<ul>
<li>Globally:</li>
</ul>
<p>In a principal component analysis for instance how do we interpret that some variables have the same loading?
What happens in a more difficult method like canonical correlation analysis?</p>
<p>There have been some articles about how to interpret those methods on real datasets <span class="citation">[<a href="references.html#ref-sherryConductingInterpretingCanonical1981" role="doc-biblioref">100</a>]</span>.
Others, to benchmark and to learn how to interpret propose analyzing a simulated dataset <span class="citation">[<a href="references.html#ref-chung_multi-omics_2019" role="doc-biblioref">101</a>, <a href="references.html#ref-martinez-mira_mosim_2018" role="doc-biblioref">102</a>]</span>.
Which is used to compare the results of the integration with the dataset of interest and to compare different tools.
These datasets are created with some relationships that the tools are expected to find.</p>
<p>There exists several methods to create synthetic datasets like MOSim <span class="citation">[<a href="references.html#ref-haberman2019" role="doc-biblioref">103</a>]</span>, metaSPARSim <span class="citation">[<a href="references.html#ref-patuzzi2019" role="doc-biblioref">104</a>]</span>, CAMISIM <span class="citation">[<a href="references.html#ref-fritz2019" role="doc-biblioref">105</a>]</span>, ballgown <span class="citation">[<a href="references.html#ref-fu2021" role="doc-biblioref">106</a>]</span>, polyester <span class="citation">[<a href="references.html#ref-frazee2021" role="doc-biblioref">107</a>]</span> and even edgeR <span class="citation">[<a href="references.html#ref-mccarthy2012" role="doc-biblioref">108</a>]</span> can be used.
These methods are useful to compare different setup and methods but they can miss some subtle not previously reported relations on real data.</p>
</div>
<div id="reviews-and-benchmarking" class="section level3">
<h3><span class="header-section-number">1.3.3</span> Reviews and benchmarking</h3>
<p>The comparison and review of methods independently from original authors have become a crucial step for selecting the right tool for a research <span class="citation">[<a href="references.html#ref-cantini_benchmarking_2020" role="doc-biblioref">93</a>]</span>.</p>
<p>Some of these reviews are focused on a specific type of data integration: metabolomics <span class="citation">[<a href="references.html#ref-cavill2016" role="doc-biblioref">109</a>]</span>, genomics <span class="citation">[<a href="references.html#ref-mcgovern2015a" role="doc-biblioref">110</a>]</span>, microbiomics… Others focus on the disease and the challenges of each omics and the need of an integrative approach to provide better therapies <span class="citation">[<a href="references.html#ref-de_souza_ibd_2017" role="doc-biblioref">111</a>–<a href="references.html#ref-valles-colomer2016" role="doc-biblioref">113</a>]</span>.
On this regard there are several efforts to integrate data on IBD.</p>
</div>
<div id="summary-2" class="section level3">
<h3><span class="header-section-number">1.3.4</span> Summary</h3>
<p>The field of integration is large and complex, with high interest in the recent days, specially in the psychology and omics field.
As a method they are quite complex and diverse which hasn’t stopped the growing interest on them to help answer complex questions.
The differences between the many methods that exist do not make the interpretation of the results easier</p>
<p>Methods to integrate have many characteristics, depending on the objectives and data that available some methods might not be yet available.
Regardless of the method used, interpretation and reporting is usually a main problem when using any of these methods.</p>
<table>
<thead>
<tr class="header">
<th align="center">Method</th>
<th align="center">Publication</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="center">SCCA</td>
<td align="center"><span class="citation">[<a href="references.html#ref-parkhomenko2009" role="doc-biblioref">114</a>]</span></td>
</tr>
<tr class="even">
<td align="center">PCCA</td>
<td align="center"><span class="citation">[<a href="references.html#ref-waaijenborg2008" role="doc-biblioref">115</a>]</span></td>
</tr>
<tr class="odd">
<td align="center">PMA</td>
<td align="center"><span class="citation">[<a href="references.html#ref-witten2009" role="doc-biblioref">116</a>]</span></td>
</tr>
<tr class="even">
<td align="center">sPLS</td>
<td align="center"><span class="citation">[<a href="references.html#ref-lêcao2009" role="doc-biblioref">117</a>]</span></td>
</tr>
<tr class="odd">
<td align="center">gesca</td>
<td align="center"><span class="citation">[<a href="references.html#ref-hwang2009" role="doc-biblioref">118</a>]</span></td>
</tr>
<tr class="even">
<td align="center">Regularized dual CCA</td>
<td align="center"><span class="citation">[<a href="references.html#ref-soneson2010" role="doc-biblioref">119</a>]</span></td>
</tr>
<tr class="odd">
<td align="center">RGCCA</td>
<td align="center"><span class="citation">[<a href="references.html#ref-tenenhaus_regularized_2011" role="doc-biblioref">78</a>]</span></td>
</tr>
<tr class="even">
<td align="center">SNMNMF</td>
<td align="center"><span class="citation">[<a href="references.html#ref-zhang2011" role="doc-biblioref">120</a>]</span></td>
</tr>
<tr class="odd">
<td align="center">scca</td>
<td align="center"><span class="citation">[<a href="references.html#ref-lee2011" role="doc-biblioref">121</a>]</span></td>
</tr>
<tr class="even">
<td align="center">STATIS</td>
<td align="center"><span class="citation">[<a href="references.html#ref-abdi2012" role="doc-biblioref">122</a>]</span></td>
</tr>
<tr class="odd">
<td align="center">joint NMF</td>
<td align="center"><span class="citation">[<a href="references.html#ref-zhang2012" role="doc-biblioref">123</a>]</span></td>
</tr>
<tr class="even">
<td align="center">sMBPLS</td>
<td align="center"><span class="citation">[<a href="references.html#ref-li2012" role="doc-biblioref">124</a>]</span></td>
</tr>
<tr class="odd">
<td align="center">Bayesian group factor analysis</td>
<td align="center"><span class="citation">[<a href="references.html#ref-virtanen2012a" role="doc-biblioref">125</a>]</span></td>
</tr>
<tr class="even">
<td align="center">RIMBANET</td>
<td align="center"><span class="citation">[<a href="references.html#ref-zhu2012a" role="doc-biblioref">126</a>]</span></td>
</tr>
<tr class="odd">
<td align="center">FactoMineR</td>
<td align="center"><span class="citation">[<a href="references.html#ref-abdi2013" role="doc-biblioref">127</a>]</span></td>
</tr>
<tr class="even">
<td align="center">JIVE</td>
<td align="center"><span class="citation">[<a href="references.html#ref-lock2013a" role="doc-biblioref">128</a>]</span></td>
</tr>
<tr class="odd">
<td align="center">pandaR</td>
<td align="center"><span class="citation">[<a href="references.html#ref-schlauch2017" role="doc-biblioref">129</a>]</span></td>
</tr>
<tr class="even">
<td align="center">omicade4</td>
<td align="center"><span class="citation">[<a href="references.html#ref-meng2014a" role="doc-biblioref">130</a>]</span></td>
</tr>
<tr class="odd">
<td align="center">STATegRa</td>
<td align="center"><span class="citation">[<a href="references.html#ref-planell2021a" role="doc-biblioref">131</a>]</span></td>
</tr>
<tr class="even">
<td align="center">Joint factor model</td>
<td align="center"><span class="citation">[<a href="references.html#ref-ray2014" role="doc-biblioref">132</a>]</span></td>
</tr>
<tr class="odd">
<td align="center">GFAsparse</td>
<td align="center"><span class="citation">[<a href="references.html#ref-bunte2016" role="doc-biblioref">133</a>]</span></td>
</tr>
<tr class="even">
<td align="center">Sparse CCA</td>
<td align="center"><span class="citation">[<a href="references.html#ref-chen2013" role="doc-biblioref">134</a>]</span></td>
</tr>
<tr class="odd">
<td align="center">CCAGFA</td>
<td align="center"><span class="citation">[<a href="references.html#ref-leppäaho2017" role="doc-biblioref">135</a>]</span></td>
</tr>
<tr class="even">
<td align="center">CMF</td>
<td align="center"><span class="citation">[<a href="references.html#ref-klami2014" role="doc-biblioref">136</a>]</span></td>
</tr>
<tr class="odd">
<td align="center">MOGSA</td>
<td align="center"><span class="citation">[<a href="references.html#ref-meng2018" role="doc-biblioref">137</a>]</span></td>
</tr>
<tr class="even">
<td align="center">iNMF</td>
<td align="center"><span class="citation">[<a href="references.html#ref-yang2016b" role="doc-biblioref">138</a>]</span></td>
</tr>
<tr class="odd">
<td align="center">BASS</td>
<td align="center"><span class="citation">[<a href="references.html#ref-zhao2015" role="doc-biblioref">139</a>]</span></td>
</tr>
<tr class="even">
<td align="center">imputeMFA</td>
<td align="center"><span class="citation">[<a href="references.html#ref-voillet2016" role="doc-biblioref">140</a>]</span></td>
</tr>
<tr class="odd">
<td align="center">PLSCA</td>
<td align="center"><span class="citation">[<a href="references.html#ref-beaton2016" role="doc-biblioref">141</a>]</span></td>
</tr>
<tr class="even">
<td align="center">mixOmics</td>
<td align="center"><span class="citation">[<a href="references.html#ref-singh2019" role="doc-biblioref">142</a>]</span></td>
</tr>
<tr class="odd">
<td align="center">mixedCCA</td>
<td align="center"><span class="citation">[<a href="references.html#ref-yoon2019" role="doc-biblioref">143</a>]</span></td>
</tr>
<tr class="even">
<td align="center">SLIDE</td>
<td align="center"><span class="citation">[<a href="references.html#ref-gaynanova2017a" role="doc-biblioref">144</a>]</span></td>
</tr>
<tr class="odd">
<td align="center">fCCAC</td>
<td align="center"><span class="citation">[<a href="references.html#ref-madrigal2017" role="doc-biblioref">145</a>]</span></td>
</tr>
<tr class="even">
<td align="center">TSKCCA</td>
<td align="center"><span class="citation">[<a href="references.html#ref-yoshida2017" role="doc-biblioref">146</a>]</span></td>
</tr>
<tr class="odd">
<td align="center">SMSMA</td>
<td align="center"><span class="citation">[<a href="references.html#ref-kawaguchi2017" role="doc-biblioref">147</a>]</span></td>
</tr>
<tr class="even">
<td align="center">AJIVE</td>
<td align="center"><span class="citation">[<a href="references.html#ref-feng2018" role="doc-biblioref">148</a>]</span></td>
</tr>
<tr class="odd">
<td align="center">MOFA</td>
<td align="center"><span class="citation">[<a href="references.html#ref-argelaguet2020" role="doc-biblioref">149</a>]</span></td>
</tr>
<tr class="even">
<td align="center">PCA+CCA</td>
<td align="center"><span class="citation">[<a href="references.html#ref-brown2018" role="doc-biblioref">150</a>]</span></td>
</tr>
<tr class="odd">
<td align="center">JACA</td>
<td align="center"><span class="citation">[<a href="references.html#ref-zhang2020" role="doc-biblioref">151</a>]</span></td>
</tr>
<tr class="even">
<td align="center">iPCA</td>
<td align="center"><span class="citation">[<a href="references.html#ref-tang2021" role="doc-biblioref">152</a>]</span></td>
</tr>
<tr class="odd">
<td align="center">pCIA</td>
<td align="center"><span class="citation">[<a href="references.html#ref-min2019" role="doc-biblioref">153</a>]</span></td>
</tr>
<tr class="even">
<td align="center">sSCCA</td>
<td align="center"><span class="citation">[<a href="references.html#ref-safo2018" role="doc-biblioref">154</a>]</span></td>
</tr>
<tr class="odd">
<td align="center">SWCCA</td>
<td align="center"><span class="citation">[<a href="references.html#ref-min2017" role="doc-biblioref">155</a>]</span></td>
</tr>
<tr class="even">
<td align="center">OmicsPLS</td>
<td align="center"><span class="citation">[<a href="references.html#ref-bouhaddani2018" role="doc-biblioref">156</a>]</span></td>
</tr>
<tr class="odd">
<td align="center">SCCA-BC</td>
<td align="center"><span class="citation">[<a href="references.html#ref-pimentel2018" role="doc-biblioref">157</a>]</span></td>
</tr>
<tr class="even">
<td align="center">WON-PARAFAC</td>
<td align="center"><span class="citation">[<a href="references.html#ref-kim2019" role="doc-biblioref">158</a>]</span></td>
</tr>
<tr class="odd">
<td align="center">BIDIFAC</td>
<td align="center"><span class="citation">[<a href="references.html#ref-lock2020" role="doc-biblioref">159</a>]</span></td>
</tr>
<tr class="even">
<td align="center">maui</td>
<td align="center"><span class="citation">[<a href="references.html#ref-ronen2019" role="doc-biblioref">160</a>]</span></td>
</tr>
<tr class="odd">
<td align="center">SmCCNet</td>
<td align="center"><span class="citation">[<a href="references.html#ref-shi2019" role="doc-biblioref">161</a>]</span></td>
</tr>
<tr class="even">
<td align="center">msPLS</td>
<td align="center"><span class="citation">[<a href="references.html#ref-csala2020" role="doc-biblioref">162</a>]</span></td>
</tr>
<tr class="odd">
<td align="center">MOTA</td>
<td align="center"><span class="citation">[<a href="references.html#ref-fan2020" role="doc-biblioref">163</a>]</span></td>
</tr>
<tr class="even">
<td align="center">D-CCA</td>
<td align="center"><span class="citation">[<a href="references.html#ref-shu2020" role="doc-biblioref">164</a>]</span></td>
</tr>
<tr class="odd">
<td align="center">COMBI</td>
<td align="center"><span class="citation">[<a href="references.html#ref-hawinkel2020" role="doc-biblioref">165</a>]</span></td>
</tr>
<tr class="even">
<td align="center">DPCCA</td>
<td align="center"><span class="citation">[<a href="references.html#ref-gundersen2020" role="doc-biblioref">166</a>]</span></td>
</tr>
<tr class="odd">
<td align="center">MEFISTO</td>
<td align="center"><span class="citation">[<a href="references.html#ref-velten2020" role="doc-biblioref">167</a>]</span></td>
</tr>
<tr class="even">
<td align="center">MultiPower</td>
<td align="center"><span class="citation">[<a href="references.html#ref-tarazona2020" role="doc-biblioref">168</a>]</span></td>
</tr>
<tr class="odd">
<td align="center">mixedCCA</td>
<td align="center"><span class="citation">[<a href="references.html#ref-yoon2019a" role="doc-biblioref">169</a>]</span></td>
</tr>
</tbody>
</table>
<!--Check https://github.com/mikelove/awesome-multi-omics -->

</div>
</div>
</div>
<div class="footnotes">
<hr />
<ol start="1">
<li id="fn1"><p>A signature is usually a group of features that describe/are representative of a cell line or a process or a stage.<a href="introduction.html#fnref1" class="footnote-back">↩︎</a></p></li>
</ol>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="index.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="hypothesis-and-objectives.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": {
"href": "https://github.com/llrs/thesis/"
},
"facebook": false,
"twitter": {
"href": "https://twitter.com/intent/tweet?text=Nice%20read%20of%20%40Lluis_Revilla%20thesis"
},
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/llrs/thesis/edit/master/01_introduction.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": "https://github.com/llrs/thesis/blob/master/01_introduction.Rmd",
"text": null
},
"download": ["Data integration on inflammatory bowel disease.pdf", "Data integration on inflammatory bowel disease.epub"],
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": false,
"toc_appendix": true,
"scroll_highlight": true
},
"toc_float": true
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
