<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 3 Materials and methods | Data integration on inflammatory bowel disease</title>
  <meta name="description" content="Chapter 3 Materials and methods | Data integration on inflammatory bowel disease" />
  <meta name="generator" content="bookdown 0.25 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 3 Materials and methods | Data integration on inflammatory bowel disease" />
  <meta property="og:type" content="book" />
  
  
  <meta name="github-repo" content="llrs/thesis" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 3 Materials and methods | Data integration on inflammatory bowel disease" />
  <meta name="twitter:site" content="@Lluis_Revilla" />
  
  

<meta name="author" content="Lluís Revilla Sancho" />


<meta name="date" content="2022-03-21" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="hypothesis-and-objectives.html"/>
<link rel="next" href="results.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1, orientation=auto"/>
<meta name="description" content="Thesis searching the relationship between microbiome and transcriptome on the intestine on patients with IBD to discover key genes and bacteria."/>
<meta name="github-repo" content="llrs/thesis"/>
<meta name="twitter:title" content="Chapter 3 Materials and methods | Data integration on inflammatory bowel disease"/>
<meta name="twitter:description" content="Thesis searching the relationship between microbiome and transcriptome on the intestine on patients with IBD to discover key genes and bacteria."/>
<meta name="twitter:url" content="https://thesis.llrs.dev"/>
<meta name="twitter:image" content="https://r4ds.had.co.nz/cover.png"/>
<meta name="twitter:image:alt" content="The cover of the book"/>
<meta name="twitter:card" content="summary_large_image"/>
<meta name="twitter:creator" content="@Lluis_Revilla"/>
<meta name="twitter:site" content="@Lluis_Revilla"/>
<meta property="og:title" content="Chapter 3 Materials and methods | Data integration on inflammatory bowel disease"/>
<meta property="og:description" content="Thesis searching the relationship between microbiome and transcriptome on the intestine on patients with IBD to discover key genes and bacteria."/>
<meta property="og:url" content="https://thesis.llrs.dev"/>
<meta property="og:image" content="https://r4ds.had.co.nz/cover.png"/>
<meta property="og:image:alt" content="The cover of the book"/>
<meta property="og:type" content="book"/>
<meta property="og:locale" content="en_US"/>
<meta property="article:author" content="Llu&lt;U+00ED&gt;s Revilla Sancho"/>
<script src="libs/kePrint-0.0.1/kePrint.js"></script>
<link href="libs/lightable-0.0.1/lightable.css" rel="stylesheet" />



<style type="text/css">
/* Used with Pandoc 2.11+ new --citeproc when CSL is used */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><strong><a href="./">Data integration on IBD</a></strong></li>

<li class="divider"></li>
<li><a href="index.html#preface">Preface<span></span></a></li>
<li><a href="abstracts.html#abstracts">Abstracts<span></span></a>
<ul>
<li><a href="abstracts.html#english">English<span></span></a></li>
<li><a href="abstracts.html#spanish">Spanish<span></span></a></li>
<li><a href="abstracts.html#catalan">Catalan<span></span></a></li>
</ul></li>
<li class="chapter" data-level="1" data-path="introduction.html"><a href="introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction<span></span></a>
<ul>
<li class="chapter" data-level="1.1" data-path="introduction.html"><a href="introduction.html#IBD"><i class="fa fa-check"></i><b>1.1</b> Inflammatory bowel disease<span></span></a>
<ul>
<li class="chapter" data-level="1.1.1" data-path="introduction.html"><a href="introduction.html#etiology-and-pathogenesis"><i class="fa fa-check"></i><b>1.1.1</b> Etiology and pathogenesis<span></span></a></li>
<li class="chapter" data-level="1.1.2" data-path="introduction.html"><a href="introduction.html#CD"><i class="fa fa-check"></i><b>1.1.2</b> CD physiology<span></span></a></li>
<li class="chapter" data-level="1.1.3" data-path="introduction.html"><a href="introduction.html#UC"><i class="fa fa-check"></i><b>1.1.3</b> UC physiology<span></span></a></li>
<li class="chapter" data-level="1.1.4" data-path="introduction.html"><a href="introduction.html#treatment"><i class="fa fa-check"></i><b>1.1.4</b> Treatment<span></span></a></li>
<li class="chapter" data-level="1.1.5" data-path="introduction.html"><a href="introduction.html#summary-ibd"><i class="fa fa-check"></i><b>1.1.5</b> Summary<span></span></a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="introduction.html"><a href="introduction.html#integration-studies-on-ibd"><i class="fa fa-check"></i><b>1.2</b> Integration studies on IBD<span></span></a>
<ul>
<li class="chapter" data-level="1.2.1" data-path="introduction.html"><a href="introduction.html#data-origin"><i class="fa fa-check"></i><b>1.2.1</b> Type of data used for integration analysis<span></span></a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="introduction.html"><a href="introduction.html#integration"><i class="fa fa-check"></i><b>1.3</b> Integration<span></span></a>
<ul>
<li class="chapter" data-level="1.3.1" data-path="introduction.html"><a href="introduction.html#classification-of-integration-method"><i class="fa fa-check"></i><b>1.3.1</b> Classification of integration method<span></span></a></li>
<li class="chapter" data-level="1.3.2" data-path="introduction.html"><a href="introduction.html#interpretation"><i class="fa fa-check"></i><b>1.3.2</b> Interpretation<span></span></a></li>
<li class="chapter" data-level="1.3.3" data-path="introduction.html"><a href="introduction.html#reviews"><i class="fa fa-check"></i><b>1.3.3</b> Reviews<span></span></a></li>
<li class="chapter" data-level="1.3.4" data-path="introduction.html"><a href="introduction.html#methods-integration"><i class="fa fa-check"></i><b>1.3.4</b> Summary<span></span></a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="2" data-path="hypothesis-and-objectives.html"><a href="hypothesis-and-objectives.html"><i class="fa fa-check"></i><b>2</b> Hypothesis and objectives<span></span></a>
<ul>
<li class="chapter" data-level="2.1" data-path="hypothesis-and-objectives.html"><a href="hypothesis-and-objectives.html#hypothesis"><i class="fa fa-check"></i><b>2.1</b> Hypothesis<span></span></a></li>
<li class="chapter" data-level="2.2" data-path="hypothesis-and-objectives.html"><a href="hypothesis-and-objectives.html#objectives"><i class="fa fa-check"></i><b>2.2</b> Objectives<span></span></a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="materials-and-methods.html"><a href="materials-and-methods.html"><i class="fa fa-check"></i><b>3</b> Materials and methods<span></span></a>
<ul>
<li class="chapter" data-level="3.1" data-path="materials-and-methods.html"><a href="materials-and-methods.html#datasets"><i class="fa fa-check"></i><b>3.1</b> Datasets<span></span></a>
<ul>
<li class="chapter" data-level="3.1.1" data-path="materials-and-methods.html"><a href="materials-and-methods.html#methods-puget"><i class="fa fa-check"></i><b>3.1.1</b> Puget’s dataset<span></span></a></li>
<li class="chapter" data-level="3.1.2" data-path="materials-and-methods.html"><a href="materials-and-methods.html#methods-hsct"><i class="fa fa-check"></i><b>3.1.2</b> HSCT dataset<span></span></a></li>
<li class="chapter" data-level="3.1.3" data-path="materials-and-methods.html"><a href="materials-and-methods.html#methods-hasler"><i class="fa fa-check"></i><b>3.1.3</b> Häsler’s dataset<span></span></a></li>
<li class="chapter" data-level="3.1.4" data-path="materials-and-methods.html"><a href="materials-and-methods.html#methods-morgan"><i class="fa fa-check"></i><b>3.1.4</b> Morgan’s dataset<span></span></a></li>
<li class="chapter" data-level="3.1.5" data-path="materials-and-methods.html"><a href="materials-and-methods.html#methods-howell"><i class="fa fa-check"></i><b>3.1.5</b> Howell’s dataset<span></span></a></li>
</ul></li>
<li class="chapter" data-level="3.2" data-path="materials-and-methods.html"><a href="materials-and-methods.html#processing"><i class="fa fa-check"></i><b>3.2</b> Sample processing<span></span></a>
<ul>
<li class="chapter" data-level="3.2.1" data-path="materials-and-methods.html"><a href="materials-and-methods.html#rna-sequencing"><i class="fa fa-check"></i><b>3.2.1</b> RNA sequencing<span></span></a></li>
<li class="chapter" data-level="3.2.2" data-path="materials-and-methods.html"><a href="materials-and-methods.html#microbial-dna-sequencing"><i class="fa fa-check"></i><b>3.2.2</b> Microbial DNA sequencing<span></span></a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="materials-and-methods.html"><a href="materials-and-methods.html#integration-methods"><i class="fa fa-check"></i><b>3.3</b> Integration methods<span></span></a>
<ul>
<li class="chapter" data-level="3.3.1" data-path="materials-and-methods.html"><a href="materials-and-methods.html#mcia"><i class="fa fa-check"></i><b>3.3.1</b> MCIA<span></span></a></li>
<li class="chapter" data-level="3.3.2" data-path="materials-and-methods.html"><a href="materials-and-methods.html#stategra"><i class="fa fa-check"></i><b>3.3.2</b> STATegRa<span></span></a></li>
<li class="chapter" data-level="3.3.3" data-path="materials-and-methods.html"><a href="materials-and-methods.html#rgcca"><i class="fa fa-check"></i><b>3.3.3</b> RGCCA<span></span></a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="materials-and-methods.html"><a href="materials-and-methods.html#regularized-generalized-canonical-correlation-analysis"><i class="fa fa-check"></i><b>3.4</b> Regularized generalized canonical correlation analysis<span></span></a>
<ul>
<li class="chapter" data-level="3.4.1" data-path="materials-and-methods.html"><a href="materials-and-methods.html#description"><i class="fa fa-check"></i><b>3.4.1</b> Description<span></span></a></li>
<li class="chapter" data-level="3.4.2" data-path="materials-and-methods.html"><a href="materials-and-methods.html#output"><i class="fa fa-check"></i><b>3.4.2</b> Output<span></span></a></li>
<li class="chapter" data-level="3.4.3" data-path="materials-and-methods.html"><a href="materials-and-methods.html#models"><i class="fa fa-check"></i><b>3.4.3</b> Models<span></span></a></li>
</ul></li>
<li class="chapter" data-level="3.5" data-path="materials-and-methods.html"><a href="materials-and-methods.html#statistics"><i class="fa fa-check"></i><b>3.5</b> Statistics<span></span></a></li>
<li class="chapter" data-level="3.6" data-path="materials-and-methods.html"><a href="materials-and-methods.html#functional-enrichment-methods"><i class="fa fa-check"></i><b>3.6</b> Functional enrichment methods<span></span></a>
<ul>
<li class="chapter" data-level="3.6.1" data-path="materials-and-methods.html"><a href="materials-and-methods.html#fisher-test"><i class="fa fa-check"></i><b>3.6.1</b> Fisher test<span></span></a></li>
<li class="chapter" data-level="3.6.2" data-path="materials-and-methods.html"><a href="materials-and-methods.html#hypergeometric-test"><i class="fa fa-check"></i><b>3.6.2</b> Hypergeometric test<span></span></a></li>
<li class="chapter" data-level="3.6.3" data-path="materials-and-methods.html"><a href="materials-and-methods.html#proportion-test"><i class="fa fa-check"></i><b>3.6.3</b> Proportion test<span></span></a></li>
<li class="chapter" data-level="3.6.4" data-path="materials-and-methods.html"><a href="materials-and-methods.html#gene-set-enrichment-analysis"><i class="fa fa-check"></i><b>3.6.4</b> Gene Set Enrichment Analysis<span></span></a></li>
<li class="chapter" data-level="3.6.5" data-path="materials-and-methods.html"><a href="materials-and-methods.html#gsva"><i class="fa fa-check"></i><b>3.6.5</b> GSVA<span></span></a></li>
</ul></li>
<li class="chapter" data-level="3.7" data-path="materials-and-methods.html"><a href="materials-and-methods.html#variance"><i class="fa fa-check"></i><b>3.7</b> Variance and diversity methods<span></span></a>
<ul>
<li class="chapter" data-level="3.7.1" data-path="materials-and-methods.html"><a href="materials-and-methods.html#permanova"><i class="fa fa-check"></i><b>3.7.1</b> PERMANOVA<span></span></a></li>
<li class="chapter" data-level="3.7.2" data-path="materials-and-methods.html"><a href="materials-and-methods.html#globaltest"><i class="fa fa-check"></i><b>3.7.2</b> globaltest<span></span></a></li>
<li class="chapter" data-level="3.7.3" data-path="materials-and-methods.html"><a href="materials-and-methods.html#diversity-indices"><i class="fa fa-check"></i><b>3.7.3</b> Diversity indices<span></span></a></li>
</ul></li>
<li class="chapter" data-level="3.8" data-path="materials-and-methods.html"><a href="materials-and-methods.html#other-methods"><i class="fa fa-check"></i><b>3.8</b> Other methods<span></span></a>
<ul>
<li class="chapter" data-level="3.8.1" data-path="materials-and-methods.html"><a href="materials-and-methods.html#wgcna"><i class="fa fa-check"></i><b>3.8.1</b> WGCNA<span></span></a></li>
<li class="chapter" data-level="3.8.2" data-path="materials-and-methods.html"><a href="materials-and-methods.html#baseset"><i class="fa fa-check"></i><b>3.8.2</b> BaseSet<span></span></a></li>
<li class="chapter" data-level="3.8.3" data-path="materials-and-methods.html"><a href="materials-and-methods.html#experdesign"><i class="fa fa-check"></i><b>3.8.3</b> experDesign<span></span></a></li>
<li class="chapter" data-level="3.8.4" data-path="materials-and-methods.html"><a href="materials-and-methods.html#ROC"><i class="fa fa-check"></i><b>3.8.4</b> ROC- AUC<span></span></a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="results.html"><a href="results.html"><i class="fa fa-check"></i><b>4</b> Results<span></span></a>
<ul>
<li class="chapter" data-level="4.1" data-path="results.html"><a href="results.html#packagesmethods"><i class="fa fa-check"></i><b>4.1</b> Packages/methods<span></span></a>
<ul>
<li class="chapter" data-level="4.1.1" data-path="results.html"><a href="results.html#experdesign-1"><i class="fa fa-check"></i><b>4.1.1</b> experDesign<span></span></a></li>
<li class="chapter" data-level="4.1.2" data-path="results.html"><a href="results.html#baseset-1"><i class="fa fa-check"></i><b>4.1.2</b> BaseSet<span></span></a></li>
<li class="chapter" data-level="4.1.3" data-path="results.html"><a href="results.html#intermodel"><i class="fa fa-check"></i><b>4.1.3</b> inteRmodel<span></span></a></li>
</ul></li>
<li class="chapter" data-level="4.2" data-path="results.html"><a href="results.html#analysis"><i class="fa fa-check"></i><b>4.2</b> Analysis<span></span></a>
<ul>
<li class="chapter" data-level="4.2.1" data-path="results.html"><a href="results.html#results-puget"><i class="fa fa-check"></i><b>4.2.1</b> Puget’s dataset<span></span></a></li>
<li class="chapter" data-level="4.2.2" data-path="results.html"><a href="results.html#results-hsct"><i class="fa fa-check"></i><b>4.2.2</b> HSCT dataset<span></span></a></li>
<li class="chapter" data-level="4.2.3" data-path="results.html"><a href="results.html#results-hasler"><i class="fa fa-check"></i><b>4.2.3</b> Häsler’s dataset<span></span></a></li>
<li class="chapter" data-level="4.2.4" data-path="results.html"><a href="results.html#results-morgan"><i class="fa fa-check"></i><b>4.2.4</b> Morgan’s dataset<span></span></a></li>
<li class="chapter" data-level="4.2.5" data-path="results.html"><a href="results.html#results-howell"><i class="fa fa-check"></i><b>4.2.5</b> Howell’s dataset<span></span></a></li>
<li class="chapter" data-level="4.2.6" data-path="results.html"><a href="results.html#between-datasets"><i class="fa fa-check"></i><b>4.2.6</b> Between datasets<span></span></a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="discussion.html"><a href="discussion.html"><i class="fa fa-check"></i><b>5</b> Discussion<span></span></a>
<ul>
<li class="chapter" data-level="5.1" data-path="discussion.html"><a href="discussion.html#preliminary-steps"><i class="fa fa-check"></i><b>5.1</b> Preliminary steps<span></span></a>
<ul>
<li class="chapter" data-level="5.1.1" data-path="discussion.html"><a href="discussion.html#discussion-datasets"><i class="fa fa-check"></i><b>5.1.1</b> The datasets<span></span></a></li>
<li class="chapter" data-level="5.1.2" data-path="discussion.html"><a href="discussion.html#the-methods"><i class="fa fa-check"></i><b>5.1.2</b> The methods<span></span></a></li>
</ul></li>
<li class="chapter" data-level="5.2" data-path="discussion.html"><a href="discussion.html#designing-models"><i class="fa fa-check"></i><b>5.2</b> Designing models<span></span></a></li>
<li class="chapter" data-level="5.3" data-path="discussion.html"><a href="discussion.html#evaluating-models"><i class="fa fa-check"></i><b>5.3</b> Evaluating models<span></span></a></li>
<li class="chapter" data-level="5.4" data-path="discussion.html"><a href="discussion.html#relevance-for-other-research"><i class="fa fa-check"></i><b>5.4</b> Relevance for other research<span></span></a></li>
<li class="chapter" data-level="5.5" data-path="discussion.html"><a href="discussion.html#implications-of-research"><i class="fa fa-check"></i><b>5.5</b> Implications of research<span></span></a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="conclusions.html"><a href="conclusions.html"><i class="fa fa-check"></i><b>6</b> Conclusions<span></span></a></li>
<li class="chapter" data-level="7" data-path="acknowledgments.html"><a href="acknowledgments.html"><i class="fa fa-check"></i><b>7</b> Acknowledgments<span></span></a></li>
<li><a href="references.html#references">References<span></span></a></li>
<li class="appendix"><span><b>Appendix<span></span></b></span></li>
<li class="chapter" data-level="A" data-path="online-resources.html"><a href="online-resources.html"><i class="fa fa-check"></i><b>A</b> Online resources<span></span></a></li>
<li class="chapter" data-level="B" data-path="software.html"><a href="software.html"><i class="fa fa-check"></i><b>B</b> Software<span></span></a>
<ul>
<li class="chapter" data-level="B.1" data-path="software.html"><a href="software.html#star"><i class="fa fa-check"></i><b>B.1</b> STAR<span></span></a></li>
<li class="chapter" data-level="B.2" data-path="software.html"><a href="software.html#rsem"><i class="fa fa-check"></i><b>B.2</b> RSEM<span></span></a></li>
<li class="chapter" data-level="B.3" data-path="software.html"><a href="software.html#listed"><i class="fa fa-check"></i><b>B.3</b> Listed<span></span></a></li>
<li class="chapter" data-level="B.4" data-path="software.html"><a href="software.html#by-projectpublication"><i class="fa fa-check"></i><b>B.4</b> By project/publication<span></span></a></li>
</ul></li>
<li class="chapter" data-level="C" data-path="articles.html"><a href="articles.html"><i class="fa fa-check"></i><b>C</b> Articles<span></span></a>
<ul>
<li class="chapter" data-level="C.1" data-path="articles.html"><a href="articles.html#app-multiomics"><i class="fa fa-check"></i><b>C.1</b> Multi-omic modelling of inflammatory bowel disease with regularized canonical correlation analysis<span></span></a></li>
<li class="chapter" data-level="C.2" data-path="articles.html"><a href="articles.html#app-experDesign"><i class="fa fa-check"></i><b>C.2</b> experDesign: stratifying samples into batches with minimal bias<span></span></a></li>
</ul></li>
<li class="chapter" data-level="D" data-path="other-datasets.html"><a href="other-datasets.html"><i class="fa fa-check"></i><b>D</b> Other datasets<span></span></a>
<ul>
<li class="chapter" data-level="D.1" data-path="other-datasets.html"><a href="other-datasets.html#barcelona"><i class="fa fa-check"></i><b>D.1</b> BARCELONA dataset<span></span></a></li>
<li class="chapter" data-level="D.2" data-path="other-datasets.html"><a href="other-datasets.html#hernandez"><i class="fa fa-check"></i><b>D.2</b> Hernández’ dataset<span></span></a>
<ul>
<li class="chapter" data-level="D.2.1" data-path="other-datasets.html"><a href="other-datasets.html#results-hernandez"><i class="fa fa-check"></i><b>D.2.1</b> Results<span></span></a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="E" data-path="models-output.html"><a href="models-output.html"><i class="fa fa-check"></i><b>E</b> Models output<span></span></a>
<ul>
<li class="chapter" data-level="E.1" data-path="models-output.html"><a href="models-output.html#hsct"><i class="fa fa-check"></i><b>E.1</b> HSCT<span></span></a>
<ul>
<li class="chapter" data-level="E.1.1" data-path="models-output.html"><a href="models-output.html#genes"><i class="fa fa-check"></i><b>E.1.1</b> Genes<span></span></a></li>
<li class="chapter" data-level="E.1.2" data-path="models-output.html"><a href="models-output.html#microbiome-2"><i class="fa fa-check"></i><b>E.1.2</b> Microbiome<span></span></a></li>
</ul></li>
<li class="chapter" data-level="E.2" data-path="models-output.html"><a href="models-output.html#häsler"><i class="fa fa-check"></i><b>E.2</b> Häsler<span></span></a>
<ul>
<li class="chapter" data-level="E.2.1" data-path="models-output.html"><a href="models-output.html#genes-1"><i class="fa fa-check"></i><b>E.2.1</b> Genes<span></span></a></li>
<li class="chapter" data-level="E.2.2" data-path="models-output.html"><a href="models-output.html#microbiome-3"><i class="fa fa-check"></i><b>E.2.2</b> Microbiome<span></span></a></li>
</ul></li>
<li class="chapter" data-level="E.3" data-path="models-output.html"><a href="models-output.html#morgan"><i class="fa fa-check"></i><b>E.3</b> Morgan<span></span></a>
<ul>
<li class="chapter" data-level="E.3.1" data-path="models-output.html"><a href="models-output.html#genes-2"><i class="fa fa-check"></i><b>E.3.1</b> Genes<span></span></a></li>
<li class="chapter" data-level="E.3.2" data-path="models-output.html"><a href="models-output.html#microbiome-4"><i class="fa fa-check"></i><b>E.3.2</b> Microbiome<span></span></a></li>
</ul></li>
<li class="chapter" data-level="E.4" data-path="models-output.html"><a href="models-output.html#howell"><i class="fa fa-check"></i><b>E.4</b> Howell<span></span></a>
<ul>
<li class="chapter" data-level="E.4.1" data-path="models-output.html"><a href="models-output.html#genes-3"><i class="fa fa-check"></i><b>E.4.1</b> Genes<span></span></a></li>
<li class="chapter" data-level="E.4.2" data-path="models-output.html"><a href="models-output.html#microbiome-5"><i class="fa fa-check"></i><b>E.4.2</b> Microbiome<span></span></a></li>
</ul></li>
</ul></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Data integration on inflammatory bowel disease</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div class = "wip">
This tesis is ongoing and still without hard date to finish it. Watch the <a href="https://github.com/llrs/thesis">GitHub repository</a> or wait for a release announcement on Twitter on <a href="https://twitter.com/Lluis_Revilla">my profile</a>.
</div>
<div id="materials-and-methods" class="section level1 hasAnchor" number="3">
<h1><span class="header-section-number">Chapter 3</span> Materials and methods<a href="materials-and-methods.html#materials-and-methods" class="anchor-section" aria-label="Anchor link to header"></a></h1>
<!-- # Some sections of this chapter has been adapted from "Multi-Omic Modelling of IBD with Regularized Canonical Correlation Analysis" [@revilla2021] or other publications of the research group. -->
<p>This chapter contains a brief description of the main characteristics of the different datasets used on this thesis .
The complete processing protocol before obtaining the data of those dataset that were not generated at Hosptial Clínic can be found on their respective reference.</p>
<p>Samples of the different cohorts collected in Hospital Clínic were collected similarly and described only once.
Differences between protocols are noted on the respective dataset’s section.</p>
<p>Here we describe all the methods used to analyze data from the multiple cohorts included in this thesis.
The code used can be found on the links provided in the <a href="software.html#software" title="Link to the software appendix">appendix</a>.</p>
<div id="datasets" class="section level2 hasAnchor" number="3.1">
<h2><span class="header-section-number">3.1</span> Datasets<a href="materials-and-methods.html#datasets" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<div id="methods-puget" class="section level3 hasAnchor" number="3.1.1">
<h3><span class="header-section-number">3.1.1</span> Puget’s dataset<a href="materials-and-methods.html#methods-puget" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>The glioma dataset is the data provided as an example of biological data by the authors of RGCCA from a previous publication <span class="citation">[<a href="#ref-puget2012" role="doc-biblioref">149</a>]</span>.
The data came from diffuse intrinsic pontine glioma patients whose transcriptome was analyzed with Agilent 44K Whole Human Genome Array G4410B and G4112F.
The copy number variation of the samples was processed with the ADM-2 algorithm, and data from comparative genomic hybridization (CGH) analyzed using <a href="https://www.softgenetics.com/mutationSurveyor.php" title="Website of the software">Mutation Surveyor software</a>.
In addition, this dataset contained information on age, localization of the tumor, sex and a numerical grading of the severity of the tumor <span class="citation">[<a href="#ref-puget2012" role="doc-biblioref">149</a>]</span>.</p>
<table>
<caption><span id="tab:Puget">Table 3.1: </span> Characteristics of samples from Puget’s dataset.</caption>
<thead>
<tr class="header">
<th>Characteristic</th>
<th>Puget’s</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Samples</td>
<td>53</td>
</tr>
<tr class="even">
<td>Sex (female/male)</td>
<td>28/25</td>
</tr>
<tr class="odd">
<td>Location (cort/dipg/midl)</td>
<td>20/22/11</td>
</tr>
</tbody>
</table>
</div>
<div id="methods-hsct" class="section level3 hasAnchor" number="3.1.2">
<h3><span class="header-section-number">3.1.2</span> HSCT dataset<a href="materials-and-methods.html#methods-hsct" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<!--# COPIED!!! from article https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0246367#sec005 -->
<p>Samples from the HSCT dataset used in this thesis were from a cohort of patients with severe refractory CD undergoing HSCT.
Patients were treated in the Department of Gastroenterology (Hospital Clínic de Barcelona –Spain–).
The protocol was approved by the Catalan Transplantation Organization and by the Institutional Ethics Committee of the Hospital Clinic de Barcelona (Study Number HCB/2012/7244).
All patients provided written consent following extensive counselling about being included on the study and using their data on publications.</p>
<p>Colonic and ileal biopsies were obtained at several time points during ileo-colonoscopy, at inclusion and every 6 or 12 months after HSCT up to 4 years after the start of the treatment.
Samples were obtained when possible from both uninvolved and involved areas.
In addition, biopsies were taken from the ileum and colon regions of 19 non-IBD controls consisting of individuals with no history of IBD and who presented no significant pathological findings following endoscopic examination for colon cancer surveillance (Hospital Univesitari Mútua de Terrassa –Spain–).
The protocol was approved by the Institutional Ethics Committee of the Hospital Univesitari Mútua de Terrassa (Study Number NA1651).</p>
<p>At least one biopsy was collected and fresh-frozen at -80°C for microbial DNA extraction.
The remaining biopsies were placed in RNAlater RNA Stabilization Reagent (Qiagen, Hilde, Germany) and stored at -80°C until total RNA extraction.
In total 158 samples with both RNA and DNA extraction of the same segment and time were available <a href="materials-and-methods.html#tab:HSCT">3.2</a>:</p>
<table>
<caption><span id="tab:HSCT">Table 3.2: </span> Characteristics of samples from HSCT dataset.</caption>
<thead>
<tr class="header">
<th>Characteristic</th>
<th>HSCT</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Sex (female/male)</td>
<td>22/15</td>
</tr>
<tr class="even">
<td>Age at diagnostic (&lt;17/&lt;40/&gt;40 years)</td>
<td>7/11/0</td>
</tr>
<tr class="odd">
<td>Duration (in years): mean (min-max)</td>
<td>14 (8-28)</td>
</tr>
<tr class="even">
<td>Age: mean (min-max)</td>
<td>44 (23-70)</td>
</tr>
<tr class="odd">
<td>Disease status (non-IBD/CD)</td>
<td>51/107</td>
</tr>
<tr class="even">
<td>Sample location (ileum/colon/unknown)</td>
<td>48/108/2</td>
</tr>
<tr class="odd">
<td>Local simple endoscopic score for CD: mean (min-max)</td>
<td>2.15 (0-12)</td>
</tr>
<tr class="even">
<td>CDAI: mean (min-max)</td>
<td>120 (0-450</td>
</tr>
</tbody>
</table>
</div>
<div id="methods-hasler" class="section level3 hasAnchor" number="3.1.3">
<h3><span class="header-section-number">3.1.3</span> Häsler’s dataset<a href="materials-and-methods.html#methods-hasler" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>An IBD-related dataset was obtained by Prof. Dr. Rosentiel and Prof. Dr. Robert Häsler <span class="citation">[<a href="#ref-hasler_uncoupling_2016" role="doc-biblioref">79</a>]</span>.
Biopsies were obtained endoscopically during routine diagnosis RNA and DNA were extracted using standard procedures.
DNA from the 16S rRNA gene was amplified with primers 319F and 806R.
Both RNA and DNA was then sequenced on HiSeq 2000 and MiSeq respectively.
These biopsies included samples from the terminal ileum and sigma from CD, UC, infectious disease-controls and healthy non-IBD.
The dataset included information about gender, location, age, and the status (inflamed or non-inflamed) of the region from which the biopsy was taken.</p>
<table>
<caption><span id="tab:Hasler">Table 3.3: </span> Characteristics of samples from Häsler’s dataset.</caption>
<thead>
<tr class="header">
<th>Characteristic</th>
<th>Häsler’s</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Disease status (non-IBD/IBD)</td>
<td>33/26</td>
</tr>
<tr class="even">
<td>Sex (female/male)</td>
<td>42/17</td>
</tr>
<tr class="odd">
<td>Sample location (ileum/colon)</td>
<td>30/29</td>
</tr>
</tbody>
</table>
</div>
<div id="methods-morgan" class="section level3 hasAnchor" number="3.1.4">
<h3><span class="header-section-number">3.1.4</span> Morgan’s dataset<a href="materials-and-methods.html#methods-morgan" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>A previously published dataset from a pouchitis study was analyzed <span class="citation">[<a href="#ref-morgan2015" role="doc-biblioref">150</a>]</span>.
In this study patients having undergone proctocolectomy with ileal pouch-anal anastomosis for treatment of UC or familial adenomatous polyposis at least 1 year prior to enrollment were recruited at Mount Sinai Hospital (Toronto, Canada) excluding individuals with a diagnosis of CD.
The dataset has a total of 255 samples from 203 patients, containing data for both host transcriptome and intestinal microbiome.
On some cases several biopsyes were collected from the same patients.
This dataset included anonymous identifiers for patients, whether the sample was from the pre-pouch ileum (PPI) or from the pouch, the sex, the outcome of the procedure and an inflammation score.
The pouch ileum might be inflamed or not.</p>
<table>
<caption><span id="tab:Morgan">Table 3.4: </span> Characteristics of samples from Morgan’s dataset.</caption>
<thead>
<tr class="header">
<th>Characteristic</th>
<th>Morgan’s</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Samples (n)</td>
<td>255</td>
</tr>
<tr class="even">
<td>Sex patients (female/male)</td>
<td>101/102</td>
</tr>
<tr class="odd">
<td>Sample location (Pouch/PPI)</td>
<td>59/196</td>
</tr>
</tbody>
</table>
</div>
<div id="methods-howell" class="section level3 hasAnchor" number="3.1.5">
<h3><span class="header-section-number">3.1.5</span> Howell’s dataset<a href="materials-and-methods.html#methods-howell" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>This dataset included a cohort of 66 treatment-naïve children at diagnosis of their IBD, along with 30 age- and sex-matched non-inflammatory control children, recruited at the Paediatric Gastroenterology unit at Addenbrooke’s Hospital (England) <span class="citation">[<a href="#ref-howell2018" role="doc-biblioref">151</a>]</span>.</p>
<p>Data from 77 samples that had both RNAseq and 16S data was used.
There are 10 non-IBD samples, 11 with CD and 11 with UC.
Data has the following characteristics: disease, age at diagnostic, age at time of study, sex, sample location, and disease activity:</p>
<table>
<caption><span id="tab:howell">Table 3.5: </span> Characteristics of samples from Howell’s dataset.</caption>
<thead>
<tr class="header">
<th>Characteristic</th>
<th>Howell’s</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Disease (non-IBD/CD/UC)</td>
<td>11/10/11</td>
</tr>
<tr class="even">
<td>Age at diagnostic (&lt;17/&lt;40/&gt;40 years)</td>
<td>32/0/0</td>
</tr>
<tr class="odd">
<td>Age: mean (min-max)</td>
<td>12 (6-15)</td>
</tr>
<tr class="even">
<td>Sex (female/male)</td>
<td>10/22</td>
</tr>
<tr class="odd">
<td>Segment (ileum/colon)</td>
<td>31/46</td>
</tr>
<tr class="even">
<td>Clinical history (inflammation/no inflammation)</td>
<td>24/53</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="processing" class="section level2 hasAnchor" number="3.2">
<h2><span class="header-section-number">3.2</span> Sample processing<a href="materials-and-methods.html#processing" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<div id="rna-sequencing" class="section level3 hasAnchor" number="3.2.1">
<h3><span class="header-section-number">3.2.1</span> RNA sequencing<a href="materials-and-methods.html#rna-sequencing" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>Total RNA from mucosal samples (HSCT cohort) was isolated using the RNAeasy kit (Qiagen, Hilde, Germany).
RNA sequencing libraries were prepared for paired-end sequencing using HighSeq-4000 platform.
Samples with good enough quality as recommended by <a href="https://www.bioinformatics.babraham.ac.uk/projects/fastqc/" title="FastQC website">FastQC</a> were processed with <a href="https://github.com/marcelm/cutadapt" title="cutadapt repository">cutadapt</a> (version 1.7.1 <span class="citation">[<a href="#ref-martin2011" role="doc-biblioref">152</a>]</span>) for quality filtering.
Later, the libraries were mapped against the human reference genome using the <a href="https://github.com/alexdobin/STAR" title="STAR repository">STAR aligner</a> (2.5.2a) with Ensembl annotation (<a href="https://www.gencodegenes.org/human/release_26.html" title="GENCODE release 26">release 26 of GENCODE</a>, GRCh38.p10 or superior) <span class="citation">[<a href="#ref-dobin2013" role="doc-biblioref">153</a>]</span>.</p>
<p>Read counts per gene were obtained with <a href="https://github.com/deweylab/RSEM" title="RSEM repository">RSEM</a> (version 1.2.31) <span class="citation">[<a href="#ref-li2011" role="doc-biblioref">154</a>]</span> as previously described <span class="citation">[<a href="#ref-corralizaDifferencesPeripheralTissue" role="doc-biblioref">155</a>]</span>.</p>
</div>
<div id="microbial-dna-sequencing" class="section level3 hasAnchor" number="3.2.2">
<h3><span class="header-section-number">3.2.2</span> Microbial DNA sequencing<a href="materials-and-methods.html#microbial-dna-sequencing" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>Biopsies from the HSCT CD cohort were resuspended in 180 <span class="math inline">\(\mu\)</span>l TET (TrisHCl 0.02M, EDTA 0.002M, Triton 1X) buffer and 20mg/ml lysozyme (Carl Roth, Quimivita, S.A.). Samples were incubated for 1h at 37°C and vortexed with 25 <span class="math inline">\(\mu\)</span>l Proteinase K before incubating at 56°C for 3h.
Buffer B3 (NucleoSpin Tissue Kit–Macherey-Nagel) was added followed by a heat treatment for 10 min at 70°C.
After adding 100% ethanol, samples were centrifuged at 11000 x g for 1 min.
Two washing steps were performed before eluting DNA.
Concentrations and purity were checked using NanoDrop One (Thermo Fisher Scientific).
Samples were immediately used or placed at -20°C for long-term storage until DNA sequencing.</p>
<div id="dna-sequencing" class="section level4 hasAnchor" number="3.2.2.1">
<h4><span class="header-section-number">3.2.2.1</span> DNA sequencing<a href="materials-and-methods.html#dna-sequencing" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>Microbial cells were disrupted by mechanical lysis using FastPrep-24.
Heat treatment and centrifugation were conducted after adding a cooling adaptor.
Supernatants were treated with RNase to eliminate RNA.
Total DNA was purified using gDNA columns as described in detail previously <span class="citation">[<a href="#ref-berry2011" role="doc-biblioref">156</a>]</span>.
Briefly, the V3-V4 regions of 16S rRNA gene were amplified (15x15 cycles) following a previously described two-step protocol <span class="citation">[<a href="#ref-klindworth2013" role="doc-biblioref">157</a>]</span> using forward and reverse primers 341F-ovh/785R-ovh <span class="citation">[<a href="#ref-edgar2013" role="doc-biblioref">158</a>]</span>.
Purification of amplicons was performed by using the AMPure XP system (Beckmann).
Next, sequencing was performed with pooled samples in paired-end modus (PE275) using an MiSeq system (Illumina, Inc.) according to the manufacturer’s instructions and 25% (v/v) PhiX standard library.</p>
<p>Library preparation and sequencing of the HSCT dataset were performed at the Technische Universität München.
Briefly, volumes of 600<span class="math inline">\(\mu\)</span>L DNA stabilization solution (STRATEC biomedical) and 400<span class="math inline">\(\mu\)</span>L Phenol:choloform:isoamyl alcohol (25:24:1, Sigma-Aldrich) were added to the aliquots.</p>
</div>
<div id="microbial-profiling" class="section level4 hasAnchor" number="3.2.2.2">
<h4><span class="header-section-number">3.2.2.2</span> Microbial profiling<a href="materials-and-methods.html#microbial-profiling" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>For the HSCT dataset the processing of raw-reads was performed by using the <a href="https://www.imngs.org/" title="IMNGS website">IMNGS</a> (version 1.0 Build 2007) <span class="citation">[<a href="#ref-lagkouvardos2016" role="doc-biblioref">159</a>]</span> pipeline based on the <a href="https://www.drive5.com/uparse/" title="UPARSE website">UPARSE</a> approach <span class="citation">[<a href="#ref-edgar2013" role="doc-biblioref">158</a>]</span>.
Sequences were demultiplexed, trimmed to the first base with a quality score &lt;3 and then paired.
Sequences with less than 300 and more than 600 nucleotides and paired reads with an expected error &gt;3 were excluded from the analysis.
The 5 nucleotides from each end of the remaining reads were trimmed to avoid GC bias and non-random base composition.
Operational taxonomic units (OTUs) were clustered at 97% sequence similarity.
Taxonomy assignment was performed at 80% confidence level using the RDP classifier and the SILVA ribosomal RNA gene database project.
Later the data was normalized using the same method as for RNA-seq described above.
The microbiome was visually inspected for batch effects in PCA; none were found.
The resulting OTUs table was normalized using edgeR (Version 3.28 or later) <span class="citation">[<a href="#ref-mccarthy2012" role="doc-biblioref">145</a>]</span>.</p>
<p>For all the other datasets <a href="https://bioconductor.org/packages/dada2">dada2</a> <span class="citation">[<a href="#ref-callahan2016" role="doc-biblioref">160</a>]</span> (Version 1.14 or later) was used to analyze microbiome data.
It creates amplicon sequencing variants from the 16S sequencing data, without merging similar sequences at any threshold.
It is an alternative to the use of OTUs which allows to compare results between studies and provides more resolution to identify differences on the fragment of 16S amplified.</p>
<div class="figure"><span style="display:block;" id="fig:workflow-process"></span>
<img src="images/workflow_integration2.png" alt="Workflow of the main analysis process of the thesis. Created with BioRender.com" width="100%" />
<p class="caption">
Figure 3.1: Workflow of the main analysis process of the thesis. Created with BioRender.com
</p>
</div>
</div>
</div>
</div>
<div id="integration-methods" class="section level2 hasAnchor" number="3.3">
<h2><span class="header-section-number">3.3</span> Integration methods<a href="materials-and-methods.html#integration-methods" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<div id="mcia" class="section level3 hasAnchor" number="3.3.1">
<h3><span class="header-section-number">3.3.1</span> MCIA<a href="materials-and-methods.html#mcia" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>Multiple co-inertia analysis, also known as MCIA, is a method to examine covariant gene expression patterns between two blocks <span class="citation">[<a href="#ref-meng2014" role="doc-biblioref">161</a>]</span>.
It is implemented on the package <a href="https://bioconductor.org/packages/omicade4">omicade4</a>.
On its core MCIA maximizes the following formula:</p>
<p><span class="math display">\[
\sum_{k=1}^K w_k {cov}^2 (X_k Q_k u_k, v)
\]</span></p>
<p>where <span class="math inline">\(K\)</span> is the total number of matrices, <span class="math inline">\(X_k\)</span> the transformed matrices and <span class="math inline">\(Q_k\)</span> is a square matrix with <span class="math inline">\(r_{ij}\)</span> in diagonal elements indicating the hyperspace of features metrics, <span class="math inline">\(u_k\)</span> are auxiliary axes, <span class="math inline">\(v\)</span> the reference structure and w the weights of the matrices.
This can be used to obtain a dimension of <span class="math inline">\(P_k^d=u_k^d(u_k^d Q_k {u_k^d}^T)^{-1} u_k^d Q_k\)</span> given that for each dimension the residuals are obtained following <span class="math inline">\(X_k^{d-1} = X_1^d- X_1^d P_1^{d-1}\)</span> where <span class="math inline">\(d\)</span> are the dimensions needed.</p>
<p>MCIA was used as a baseline method to compare the RGCCA integration.</p>
</div>
<div id="stategra" class="section level3 hasAnchor" number="3.3.2">
<h3><span class="header-section-number">3.3.2</span> STATegRa<a href="materials-and-methods.html#stategra" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>We used <a href="https://bioconductor.org/packages/STATegRa"><code>STATegRa</code></a> To explore how much do different blocks of a dataset have in common <span class="citation">[<a href="#ref-planell2021" role="doc-biblioref">162</a>]</span>.
It is a framework for integrating datasets with two data types using parametric and non-parametric methods.
The methods used are omics component analysis based on singular value decomposition (SVD) of the data matrix.
There are three different methods provided to this end: <em>DISCO-SCA</em>, <em>JIVE</em> and <em>O2PLS</em>.</p>
<p>DISCO-SCA uses:</p>
<p><span class="math display">\[
X_k = TP_k ^ T + E_k
\]</span></p>
<p>Where <span class="math inline">\(T\)</span> is the <span class="math inline">\(I \times R\)</span> matrix of components scores that is shared between all blocks and <span class="math inline">\(P_k\)</span> the <span class="math inline">\(J_k \times R\)</span> matrix of components loadings for block <span class="math inline">\(k\)</span>.</p>
<p>Let <span class="math inline">\(X_1,X_2, \dots X_i\)</span> be blocks of data and <span class="math inline">\(X=[X_1,X_2, \dots X_i]\)</span> represent the joint data, then the JIVE decomposition is defined as:</p>
<p><span class="math display">\[X_i=J_i+A_i+\epsilon_i \text{, }i = 1,2, \dots\]</span></p>
<p>where <span class="math inline">\(J=[J_1,J_2, \dots J_i]\)</span> is the <span class="math inline">\(p \times n\)</span> matrix of rank <span class="math inline">\(r&lt;rank(X)\)</span> representing the joint structure, <span class="math inline">\(A_i\)</span> is the <span class="math inline">\(p_i\times n\)</span> matrix of rank <span class="math inline">\(r_i &lt; rank(X_i)\)</span> representing the individual structure of <span class="math inline">\(X_i\)</span> and <span class="math inline">\(\epsilon_i\)</span> are <span class="math inline">\(p_i \times n\)</span> error matrices of independent entries.</p>
<p>Finally, the O2PLS approach uses multiple linear regression to estimate the pure constituent profiles and divides the systematic part into two, one common to both blocks and one not.
The O2PLS model can be written as a factor analysis where some factors are common between both blocks.</p>
<p><span class="math display">\[
\begin{aligned}
&amp; \text{X model : } X = TW ^ T + T_{\text{Y-ortho}} P ^ T_{\text{Y-ortho}} + E \\
&amp; \text{Y model : } Y = UC ^ T + U_{\text{X-ortho}} P ^ T_{\text{X-ortho}} + F \\
&amp; \text{Inner relation : } U = T + H
\end{aligned}
\]</span></p>
<p>Each model is built similarly by adding the subtraction of the projected values of the other component keeping the relationship between them as stated on the third line.</p>
</div>
<div id="rgcca" class="section level3 hasAnchor" number="3.3.3">
<h3><span class="header-section-number">3.3.3</span> RGCCA<a href="materials-and-methods.html#rgcca" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>The main method used on this thesis has been regularized generalized canonical correlation analysis (RGCCA) a method derived from the canonical correlation.
Canonical correlation is a method that uses data about the same unit but from different origins to find how much the different sources agree.
Regularized generalized canonical correlation analysis is implemented on the homonymous package <a href="https://cran.r-project.org/package=RGCCA">RGCCA</a> <span class="citation">[<a href="#ref-tenenhaus2017" role="doc-biblioref">129</a>]</span> which was used here.
The method and implementation will be explained in detail in the next section.</p>
<div class="figure"><span style="display:block;" id="fig:multiomic-datasets"></span>
<img src="images/multiomic_datasets.png" alt="Multiomic relationships on different datasets. Integration methods focus on relationships within datasets. Common relationships between datasets are used as confirmation/validation. Created with BioRender.com" width="100%" />
<p class="caption">
Figure 3.2: Multiomic relationships on different datasets. Integration methods focus on relationships within datasets. Common relationships between datasets are used as confirmation/validation. Created with BioRender.com
</p>
</div>
</div>
</div>
<div id="regularized-generalized-canonical-correlation-analysis" class="section level2 hasAnchor" number="3.4">
<h2><span class="header-section-number">3.4</span> Regularized generalized canonical correlation analysis<a href="materials-and-methods.html#regularized-generalized-canonical-correlation-analysis" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>To understand the regularized canonical correlation analysis I first provide a brief description of principal component analysis.</p>
<p>Principal component analysis defines a new orthogonal coordinate system that optimally describes variance in a single dataset.
It does so by decomposing the numerical matrix into the eigenvalues and eigenvectors with decreasing variance.
Its results include new variables (the eigenvectors) and a vector of loadings or weight indicating the importance of the original variables for these new variables.
Typically, data is normalized and standardized to avoid that a variable with higher variance and different scale dominate the results.</p>
<p>The canonical correlation analysis <span class="citation">[<a href="#ref-jordan1875" role="doc-biblioref">163</a>, <a href="#ref-hotelling1936" role="doc-biblioref">164</a>]</span> is a method to find agreement between two, or more, scorers (as it was first introduced on the literature) or sources.
It extends the PCA to a two sources of data.
It provides a similar output, new variables and weights of the existing variables indicating their importance.
The function it maximizes is:</p>
<p><span class="math display">\[
\rho = \underset{a, b}{\text{argmax}} (\text{cor}(a^T X, b^T Y))
\]</span></p>
<p>On this equation <span class="math inline">\(a\)</span> and <span class="math inline">\(b\)</span> are random variables that given <span class="math inline">\(X\)</span> and <span class="math inline">\(Y\)</span>, the two data sources, maximize the correlation between <span class="math inline">\(a^T X\)</span> and <span class="math inline">\(b^T Y\)</span>, the first canonical variables, the new variables.
<span class="math inline">\(a\)</span> and <span class="math inline">\(b\)</span> are also known as weights or loadings of the variables.</p>
<p>Over several years of progress on the field of canonical correlations <span class="citation">[<a href="#ref-tenenhaus_regularized_2011" role="doc-biblioref">113</a>, <a href="#ref-tenenhaus_variable_2014" role="doc-biblioref">114</a>, <a href="#ref-tenenhaus_component-based_2008" role="doc-biblioref">165</a>–<a href="#ref-gloaguen2020" role="doc-biblioref">169</a>]</span>, regularized generalized canonical correlation analysis (RGCCA) emerged with a generalization from canonical correlations extending the procedure to more than two sources of data <span class="citation">[<a href="#ref-R-RGCCA" role="doc-biblioref"><strong>R-RGCCA?</strong></a>]</span> and being made more flexible generalizing from other proposed methods.</p>
<div id="description" class="section level3 hasAnchor" number="3.4.1">
<h3><span class="header-section-number">3.4.1</span> Description<a href="materials-and-methods.html#description" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>RGCCA works with numeric matrices that can be as big as needed, as it is designed for datasets with more variables than samples ( <span class="math inline">\(p \gg n\)</span>).
However, for each sample it needs to have a complete case with no missing values (at the time of writing this thesis there is an <a href="https://github.com/rgcca-factory/RGCCA/tree/CRAN" title="RGCCA repository on github">in-development version</a>, not released on CRAN<a href="#fn2" class="footnote-ref" id="fnref2"><sup>2</sup></a> yet, that replaces any missing value by a 0) and in which time is not considered as a special variable.
It uses a dimensional reduction approach to relate the different blocks of data between them and produce specific factors for each dataset.
The objective function is:</p>
<p><span class="math display">\[
\underset{a_1,a_2, \dots,a_J}{\text{maximize}} \sum_{j, k = 1}^J c_{jk} g( \text{cov}(X_j a_j, X_k a_k)) \text{ s.t. } (1-\tau_j)var(X_j a_j)+\tau_j \Vert a_j \Vert^2 = 1, j=1, \ldots, J
\]</span></p>
<p>Being <span class="math inline">\(X_j\)</span> the values from sample <span class="math inline">\(j\)</span>, the weights of the variables of said sample are represented by <span class="math inline">\(a\)</span>.
While <span class="math inline">\(g\)</span> is a function that can take the form of <span class="math inline">\(x\)</span>, also known as Horst method, <span class="math inline">\(|x|\)</span> known as centroid method, <span class="math inline">\(x^2\)</span> known as factorial method, or any user-supplied function.
<span class="math inline">\(C\)</span> is a symmetric matrix describing the network between blocks.</p>
<p>The shrinkage parameter is defined as:</p>
<p><span class="math display">\[
\widehat{\lambda}^{\star} = \dfrac{\sum_{i\neq j}\widehat{Var}(r_{ij})}{\sum_{i \neq j}r_{ij}^2}
\]</span></p>
<p>Where the <span class="math inline">\(r_{ij}\)</span> are the correlation coefficients of the matrix between variables <span class="math inline">\(i\)</span> and <span class="math inline">\(j\)</span> .
Where the variance is defined as:</p>
<p><span class="math display">\[
\widehat{Var}(S_{ij}) = \dfrac{n^2}{{(n-1)}²} \widehat{Var}({w}_{ij}) = \dfrac{n}{{(n-1)}^3} \sum_{k=1}^n ( w_{kij} - \overline{w_{ij}})^2
\]</span></p>
<p>And its components are: <span class="math inline">\(w_{kij}=(x_{ki}-\overline{x}_i)(x_{kj}-\overline{x}_j)\)</span> and <span class="math inline">\(\overline{w}_{ij}=\frac{1}{n}\sum_{k=1}^nw_{kij}\)</span> representing <span class="math inline">\(x_{ij}\)</span> the values of a sample <span class="math inline">\(j\)</span> on a variable <span class="math inline">\(i\)</span>.</p>
<p>The authors realized that there is a special problem due to sparsity on biological data which could be handled using first another normalization to improve the stability and success of the canonical correlation methodology.
The method to perform the dimensional reduction using the sparse method consists on maximizing the same equation but with a different constraint: Specifically, RGCCA with all <span class="math inline">\(\tau_j = 1\)</span>, combined with an L1-penalty that gives rise to SGCCA:</p>
<p><span class="math display">\[
\underset{a_1,a_2, \dots,a_J}{\text{maximize}} \sum_{j, k = 1}^J c_{jk}g( \text{cov}(X_j a_j, X_k a_k)) \text{~~s.t.~~} \Vert a_j \Vert_2 = 1 \text{ and } \Vert a_j \Vert_1 \le s_j, j=1,\ldots,J
\]</span></p>
<p>The <span class="math inline">\(s_j\)</span> controls the sparsity estimated of the data, the smaller it is, the higher the sparsity of <span class="math inline">\(X_j\)</span> is.
As <span class="math inline">\(s_j\)</span> is closer to 0, more features are selected as it looks to optimize covariance; while if it is closer to 1, less features are selected and the function resembles the correlation.
The values of <span class="math inline">\(s_j\)</span> were estimated using the Schäfer method <span class="citation">[<a href="#ref-schäfer2005" role="doc-biblioref">170</a>]</span> when the block included 16S data or RNAseq, otherwise 1 was used.</p>
<p>As previously mentioned, there are different <span class="math inline">\(g\)</span> functions that could be used; but the centroid method was chosen to detect both positive and negative relationships.</p>
<p>Categorical data was encoded as binary (dummy) variables for each factor except one to keep degrees of freedom, where 0 indicates not present and 1 indicates present.
One level was omitted to avoid overfitting the data.
Each block, regardless if it had continuous numeric variables or dummy variables was standardized to zero mean and unit variance.
Later, it was divided by the square root of the number of variables of the block for an unbiased estimation.</p>
</div>
<div id="output" class="section level3 hasAnchor" number="3.4.2">
<h3><span class="header-section-number">3.4.2</span> Output<a href="materials-and-methods.html#output" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>RGCCA as other dimensional reduction techniques provides specific weights for each variable on each dimension and a sample score on each dimension, together with quality scores.</p>
<p>To measure the quality of the model, the implementation provides indicators based on the Average Variance Explained (AVE).
RGCCA returns an AVE score for each block, which measures how the variables of the block correlate with the dimension component of the block.
It also provides two AVE scores of the whole model: the inner, which measures how each dimension accounts for the variance, and the outer, which measures how variables correlate with the dimension components.
The closer the inner AVE is to 1, the better the model adjusts to the data.
However, that mathematically fits better to the data does not guarantee that the model provides more insights into the biology.</p>
<p>It can also generate results as other related methods based on the maximization of a function of correlations: SUMCOR (sum of correlations method) <span class="citation">[<a href="#ref-horst1961" role="doc-biblioref">171</a>]</span>, SSQCOR (sum of squared correlations method) <span class="citation">[<a href="#ref-kettenring1971" role="doc-biblioref">172</a>]</span>, SABSCOR (sum of absolute values of the correlations method) <span class="citation">[<a href="#ref-vandegeer1984" role="doc-biblioref">173</a>]</span>.
Others methods are based on the maximization of a function of covariances: SUMCOV (sum of covariances method), SSQCOV (sum of squared covariances method), SABSCOV (sum of absolute values of the covariances method).
The following table summarizes the equivalent parameters needed on RGCCA to work:</p>
<table>
<caption><span id="tab:RGCCA-methods">Table 3.6: </span> Equivalences of RGCCA for multi-block data analysis to other methods</caption>
<thead>
<tr class="header">
<th>Method</th>
<th>Scheme</th>
<th>Normalization</th>
<th>Shrinkage</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>SUMCOR</td>
<td>Horst</td>
<td><span class="math inline">\(Var(X_j a_j) = 1\)</span></td>
<td>0</td>
</tr>
<tr class="even">
<td>SSQCOR</td>
<td>Factorial</td>
<td><span class="math inline">\(Var(X_j a_j) = 1\)</span></td>
<td>0</td>
</tr>
<tr class="odd">
<td>SABSCOR</td>
<td>Centroid</td>
<td><span class="math inline">\(Var(X_j a_j) = 1\)</span></td>
<td>0</td>
</tr>
<tr class="even">
<td>SUMCOV</td>
<td>Horst</td>
<td><span class="math inline">\(\Vert a_j \Vert = 1\)</span></td>
<td>1</td>
</tr>
<tr class="odd">
<td>SSQCOV</td>
<td>Factorial</td>
<td><span class="math inline">\(\Vert a_j \Vert = 1\)</span></td>
<td>1</td>
</tr>
<tr class="even">
<td>SABSCOV</td>
<td>Centroid</td>
<td><span class="math inline">\(\Vert a_j \Vert = 1\)</span></td>
<td>1</td>
</tr>
</tbody>
</table>
<p>There are other methods that can be performed with RGCCA, for example: The classical canonical correlation analysis would be equivalent to data variance equal to 1 and shrinkage of 0.
Partial least squares (PLS) regression, which maximizes covariance, would be equal to data variance of 1 and shrinkage of 1 in RGCCA. Finally, redundancy analysis could be performed where one block’s weight normalized value is 1 and the variance of the other equals to 1.</p>
</div>
<div id="models" class="section level3 hasAnchor" number="3.4.3">
<h3><span class="header-section-number">3.4.3</span> Models<a href="materials-and-methods.html#models" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>There is no formal definition of what constitutes a block of data on multi-omics tools.
Most multi-omics and integration tools assume one block for each type of data, such as an essay a survey or an experiment.
We decided to split the block with the variables about the samples to separate independent variables.
The hypothesis we made was that more blocks with highly related variables but independent from the other blocks would fit better the data and thus help to identify causal or dependent variables.</p>
<p>To model what might be the relationships within datasets current practices include using a pre-selected model of relations between blocks (See figure <a href="materials-and-methods.html#fig:multiomic-datasets">3.2</a>).
However, this model might not be an accurate representation of the relations between blocks and several models might need to be fitted.
To help find the fitting model for the data I created an R package, named <a href="https://llrs.github.io/inteRmodel/">inteRmodel</a>, which helps finding the right model for the dataset via a bootstrapping procedure.</p>
<p>This method was applied to the previously described datasets to find the relationship between microorganisms and the disease.
Following this method; to provide a ground truth, a model with only the relationships between the two experimental obtained data is analyzed, on what it is called the model 0.<br />
The next models analyzed consisted on relationships between the two experimental blocks and a block with all the metadata of the samples.
These models are denoted by 1.Y, where 1 denotes the family 1 and Y is used to label some of the models of this family.</p>
<p>Later instead of a big metadata block, following our theory we split this metadata block on several ones, having a block for time related variables, another one for location and the other about the people on the study.
This allows to design a model with an expected relationships between these blocks and makes more interpretable the relationships.
These models are from the family 2 denoted by the name 2.Y, where 2 denotes the family of the model and Y change for particular models with different relationships between the blocks change.</p>
<p>For each family of models we tested all possible relationships with weights between 0 and 1 by 0.1 intervals to find the best model on each datasets according to the AVE score.</p>
<p>The final models were further validated using a bootstrap approach to measure their accuracy and likelihood on the data available.</p>
</div>
</div>
<div id="statistics" class="section level2 hasAnchor" number="3.5">
<h2><span class="header-section-number">3.5</span> Statistics<a href="materials-and-methods.html#statistics" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>Differential expression analysis was performed with the limma-trend method <span class="citation">[<a href="#ref-ritchie2015" role="doc-biblioref">174</a>, <a href="#ref-law2014" role="doc-biblioref">175</a>]</span> and edgeR <span class="citation">[<a href="#ref-mccarthy2012" role="doc-biblioref">145</a>]</span> (Bioconductor version 3.10 or superior) packages.
Data was normalized using the trimmed mean of M-values and log-2 transformed into counts per millions following the workflow previously described using voom <span class="citation">[<a href="#ref-law2018" role="doc-biblioref">176</a>]</span>.</p>
<p>To correct for multiple testing, the false discovery rate was estimated using the method of Benjamini and Hochberg <span class="citation">[<a href="#ref-yoavbenjamini" role="doc-biblioref">177</a>]</span>.
A gene was considered differentially expressed when it was significant at 5% FDR.
Special attention was given to those genes that showed a fold-change higher than |1.5|.</p>
</div>
<div id="functional-enrichment-methods" class="section level2 hasAnchor" number="3.6">
<h2><span class="header-section-number">3.6</span> Functional enrichment methods<a href="materials-and-methods.html#functional-enrichment-methods" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>Functional enrichment methods are those methods that aim to provide with more information about the variables besides their numerical value measured.
They can be very different in nature but they all use the numeric values of the variables and other information, being it from the same experiment data collection or from external data sources.</p>
<p>Many functional enrichment methods are based on an over representation analysis, where a group of elements is tested for their mesure in other groups.
This can be done with <a href="https://bioconductor.org/packages/clusterProfiler"><code>clusterProfiler</code></a> which tests genes enrichment for functionality based on information on pathway databases <span class="citation">[<a href="#ref-wu2021" role="doc-biblioref">178</a>]</span> that it is used in several publications <span class="citation">[<a href="#ref-richter2021" role="doc-biblioref">179</a>]</span>.
<code>clusterProfiler</code> checks the enrichment of features of a given group on the (background) list provided.</p>
<p><span class="math display">\[
\begin{aligned}
H_0 : P_{subset} \leq  P_{overall} \\
H_1 : P_{subset} &gt; P_{overall}
\end{aligned}
\]</span></p>
<p>The statistical test used is usually the fisher test, the hypergeometric test or the proportion test.
I describe the hypergeometric test and the proportion test below.</p>
<div id="fisher-test" class="section level3 hasAnchor" number="3.6.1">
<h3><span class="header-section-number">3.6.1</span> Fisher test<a href="materials-and-methods.html#fisher-test" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>The fisher test calculates how independently are two categories.
Say if genes in category X are independent of category Y, if we use the contingency table</p>
<table>
<caption><span id="tab:fisher">Table 3.7: </span> Fisher contingency table</caption>
<thead>
<tr class="header">
<th></th>
<th>In category X</th>
<th>Not in category X</th>
<th>Row total</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td><strong>In category Y</strong></td>
<td>a</td>
<td>b</td>
<td>a+b</td>
</tr>
<tr class="even">
<td><strong>Not in category Y</strong></td>
<td>c</td>
<td>d</td>
<td>c+d</td>
</tr>
<tr class="odd">
<td><strong>Column total</strong></td>
<td>a+c</td>
<td>b+d</td>
<td>a+b+c+d = n</td>
</tr>
</tbody>
</table>
<p>Given this contingency table <a href="materials-and-methods.html#tab:fisher">3.7</a>, the probability of X and Y being independent can be calculated with:</p>
<p><span class="math display">\[
\begin{aligned}
P &amp; = \dfrac{ \binom{a+b}{a} \binom{c+d}{c} }{\binom{n}{a+c}} \\
&amp; = \frac{(a+b)!~(c+d)!~(a+c)!~(b+d)!}{a!~~b!~~c!~~d!~~n!}
\end{aligned}
\]</span></p>
</div>
<div id="hypergeometric-test" class="section level3 hasAnchor" number="3.6.2">
<h3><span class="header-section-number">3.6.2</span> Hypergeometric test<a href="materials-and-methods.html#hypergeometric-test" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>The hypergeometric distribution describes the probability of <span class="math inline">\(k\)</span> successes (when the object drawn has a specified feature) in <span class="math inline">\(n\)</span> draws<a href="#fn3" class="footnote-ref" id="fnref3"><sup>3</sup></a>, from a finite population of size <span class="math inline">\(N\)</span> that contains exactly <span class="math inline">\(K\)</span> objects with that feature, wherein each draw is either a success or a failure.</p>
<p>In this context <span class="math inline">\(N\)</span> is the number of genes being used and <span class="math inline">\(n\)</span> the number of genes on a pathway.
It can be used to compare the genes found on a pathway (<span class="math inline">\(k\)</span> genes) compared to the expected <span class="math inline">\(K\)</span> numbers of the distribution using the following equation:</p>
<p><span class="math display">\[
P_X(k) = P(X = k) 
= \frac{\binom{K}{k} \binom{N - K}{n-k}}{\binom{N}{n}}
\]</span></p>
<p>This looks like the fisher test, because hypergeometric test assesses the extremeness of observing <span class="math inline">\(k\)</span> or more of
that overlap (<span class="math inline">\(a\)</span> on ) and thus it is the same as a one-sided Fisher’s exact test.</p>
</div>
<div id="proportion-test" class="section level3 hasAnchor" number="3.6.3">
<h3><span class="header-section-number">3.6.3</span> Proportion test<a href="materials-and-methods.html#proportion-test" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>The overrepresentation of a given group of elements can also be tested with the proportion test, which is sometimes also used on clusterProfiler.
The proportion test uses the <span class="math inline">\(\chi^2\)</span> distribution to test if the observed frequency (<span class="math inline">\(O_i\)</span>) is close to the expected frequency (<span class="math inline">\(E_i\)</span>):</p>
<p><span class="math display">\[
\chi^2 = \sum_{i =1}^n \dfrac{(O_i - E_i)^2}{E_i} \simeq \chi_{n-p}^2
\]</span></p>
<p>As this is usually done on a 2x2 contingency table it is equivalent to the Z-test of proportion.
Sometimes, the expected frequency is so low that a correction must be done to the estimation:</p>
<p><span class="math display">\[
\chi_{\text{Yates}}^2 = \sum_{i =1}^n \dfrac{(|O_i - E_i | - 0.5)^2}{E_i}
\]</span></p>
<p>This increases the <span class="math inline">\(p\)</span>-value as it raises the Chi-square statistic.</p>
</div>
<div id="gene-set-enrichment-analysis" class="section level3 hasAnchor" number="3.6.4">
<h3><span class="header-section-number">3.6.4</span> Gene Set Enrichment Analysis<a href="materials-and-methods.html#gene-set-enrichment-analysis" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>There are other methods that test if some variables show an unexpected importance according to a statistic like fold change or value, such as gene set enrichment analysis (GSEA) <span class="citation">[<a href="#ref-subramanian2005" role="doc-biblioref">180</a>]</span>.
GSEA is a computational method originally developed to determine whether a priori defined set of genes shows statistically significant and concordant differences between two biological states.
This methods check if a group of variables present in an ordered list shows a skewed distribution and it compares it against a random group of similar size.
It has been widely used since its original publication, also on IBD <span class="citation">[<a href="#ref-protiva2016" role="doc-biblioref">181</a>]</span>.</p>
<p>This method calculates the rank of genes <span class="math inline">\(rank(g_j)=r_j\)</span> where each <span class="math inline">\(g\)</span> is a gene, and then it calculates the following functions:</p>
<p><span class="math display">\[
\begin{aligned}
&amp; P_{hit}(S, i) = \sum_{g_j \in S, j \leq i}\dfrac{|r_j|^p}{N_R} \text{, where } N_R = \sum_{g_j \in S}|r_j|^p \\
&amp; P_{miss}(S, i) = \sum_{g_j \not \in S, j \leq i}\dfrac{1}{N - N_H}
\end{aligned}
\]</span></p>
<p>With these values the enrichment score (ES) defined as: <span class="math inline">\(ES=max(|P_{hit}(S, i)-P_{miss}(S, i)\vert)\)</span> is calculated from the walk.
At least 1000 permutations are usually used but a high number of permutations are required for an accurate estimation of the enrichment score.
However, when more than one pathway (<span class="math inline">\(S\)</span>) is evaluated in order to compare between their enrichment scores, they must be normalized by dividing the scores by the mean of all the ES.
When power <span class="math inline">\(p\)</span> is 0 it is equivalent to the standard Kolmogorov–Smirnov statistic, though it is usually set to 1.</p>
<p>For testing GSEA I used <a href="https://bioconductor.org/packages/fgsea"><code>fgsea</code></a> <span class="citation">[<a href="#ref-korotkevich2021" role="doc-biblioref">182</a>]</span> implementation for its speed and integration with other methods used in this thesis.
Gene pathways from the REACTOME database were tested on the weight of different models or on the comparisons performed <span class="citation">[<a href="#ref-fabregat2016" role="doc-biblioref">183</a>]</span>.</p>
</div>
<div id="gsva" class="section level3 hasAnchor" number="3.6.5">
<h3><span class="header-section-number">3.6.5</span> GSVA<a href="materials-and-methods.html#gsva" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>To estimate the expression of the pathways and compare their expression levels between conditions gene set variation analysis as implemented on <a href="https://bioconductor.org/packages/GSVA">GSVA</a> was used <span class="citation">[<a href="#ref-hänzelmann2013" role="doc-biblioref">184</a>]</span>.
It is a method that summarizes the variables’ numerical value changing the space of <span class="math inline">\(\text{variable x sample}\)</span> to <span class="math inline">\(\text{group x sample}\)</span> .
This enables other methods to use this new space instead of the original variables, which provides a successful way to look into data <span class="citation">[<a href="#ref-escudero-hernández2021" role="doc-biblioref">185</a>]</span>.
GSVA was used (again from the REACTOME database) to find the relationships between the pathways and the microbiome at different taxonomic levels.</p>
<p>This is done via an estimation and a comparison with a discrete Poisson kernel: <span class="math inline">\(i\)</span> indicates the gene from a total of <span class="math inline">\(k\)</span> genes, and samples are indicated by <span class="math inline">\(j\)</span> from <span class="math inline">\(n\)</span> number of samples.</p>
<p><span class="math display">\[
z_{ij}=\hat{F_r}(x_{ij}) = \frac{1}{n}\sum_{k=1}^n\sum_{y=0}^{x_{ij}}\dfrac{e^{-(x_{ik}+r)}(x_{ik} + r)^y }{y!}
\]</span></p>
<p><span class="math inline">\(r = 0.5\)</span> is used to set the mode of the Poisson kernel at each <span class="math inline">\(x_{ik}\)</span>, that is, similar to the expression of a gene for a given sample.</p>
<p>Later this is converted to ranks <span class="math inline">\(z_{(i)j}\)</span> for each sample and normalized: <span class="math inline">\(r_{ij}=|\frac{p}{2}-z_{(i)j}|\)</span> to make the distribution of ranks symmetric around zero to later compare with a normal distribution using a Kolmogorov-Smirnov-like random walk statistic:</p>
<p><span class="math display">\[
v_{jk}(l)=\dfrac{\sum_{i=1}^l|r_{ij}|^{\tau}I(g_{(i)}\in \gamma_k)}{\sum_{i=1}^p|r_{ij}|^{\tau}I(g_{(i)} \in \gamma_k)}
- \dfrac{\sum_{i=1}^lI(g_{(i)} \not \in\gamma_k)}{p-|\gamma_k|}
\]</span></p>
<p>Here <span class="math inline">\(\tau\)</span> describes the weight of the tail in the random walk (default is set to 1).
<span class="math inline">\(\gamma_k\)</span> is the k-th gene set and <span class="math inline">\(I(g_{(i)}\in \gamma_k)\)</span> is the indicator function whether the gene ranked i-th belongs to the gene set <span class="math inline">\(\gamma_k\)</span> .
<span class="math inline">\(|\gamma_k|\)</span> indicates the ordination of the gene set, the number of genes of the gene set and <span class="math inline">\(p\)</span> the number of genes in the dataset.</p>
<p>This difference is later converted to enrichment score for each gene set for each sample, similar to GSEA.
This score can be calculated as a difference of hits and misses or the maximum deviation from zero of the random walk (which allows to detect gene sets that have genes with different opposing expression patterns).</p>
</div>
</div>
<div id="variance" class="section level2 hasAnchor" number="3.7">
<h2><span class="header-section-number">3.7</span> Variance and diversity methods<a href="materials-and-methods.html#variance" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<div id="permanova" class="section level3 hasAnchor" number="3.7.1">
<h3><span class="header-section-number">3.7.1</span> PERMANOVA<a href="materials-and-methods.html#permanova" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>The PERMANOVA method <span class="citation">[<a href="#ref-anderson2001" role="doc-biblioref">186</a>, <a href="#ref-warton2012" role="doc-biblioref">187</a>]</span>, provided by the <a href="https://cran.r-project.org/package=vegan">vegan</a> package on the <code>adonis</code> function, was used to test if microbiome data variance is due to other variables when using distances metrics.
It uses the residual sum of squares such as:</p>
<p><span class="math display">\[
SS_W = \frac{1}{n} \sum_{i=1}^{N-1}\sum_{j=i+1}^N d_{ij}^2\epsilon{ij} 
\]</span></p>
<p>When using euclidian distances (<span class="math inline">\(d\)</span>) it is equivalent to MANOVA.
Here <span class="math inline">\(\epsilon_{ij}\)</span> takes the value of 1 if the observation <span class="math inline">\(i\)</span> and the observation <span class="math inline">\(j\)</span> are in the same group, otherwise it takes the value of 0.
This can be later used to test which variance is bigger, inter-groups or intra-groups by using the following formula:</p>
<p><span class="math display">\[
F = \dfrac{SS_A/(\alpha -1)}{SS_W/(N-\alpha)}
\]</span></p>
<p>Where <span class="math inline">\(SS_A\)</span> is the among group sum of squares, representing the intra-group variance.
<span class="math inline">\(N\)</span> is the number of samples and <span class="math inline">\(\alpha\)</span> the number of different groups.</p>
<p>This allows to test if the variables are related to the variance of the data as it can be compared with the <span class="math inline">\(F\)</span> statistic after a high number of permutations.</p>
</div>
<div id="globaltest" class="section level3 hasAnchor" number="3.7.2">
<h3><span class="header-section-number">3.7.2</span> globaltest<a href="materials-and-methods.html#globaltest" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>It is a method for testing complex hypothesis and calculate the influence of each variable on a given outcome <span class="citation">[<a href="#ref-goeman2006" role="doc-biblioref">188</a>]</span>.
I tested which variables, (sex, age, location, time since diagnosis, treatment) are important on the datasets with <a href="https://bioconductor.org/packages/globaltest">globaltest</a> (Version 5.40 or later).
This method provides a general statistic to test hypothesis against a high dimensional dataset.</p>
<p><span class="math display">\[
S = \sum_{i=1}^p x_i^{&#39;} x_i g(t_i^2)
\]</span></p>
<p>The global test performs a test statistic on the transformed t-test, where if <span class="math inline">\(p\)</span>, the number of variables, is large the test is more powerful on average over all possible sparse alternatives of general functions <span class="math inline">\(g\)</span>.</p>
<p>It was performed with variables individually and also with interactions between the different variables.</p>
</div>
<div id="diversity-indices" class="section level3 hasAnchor" number="3.7.3">
<h3><span class="header-section-number">3.7.3</span> Diversity indices<a href="materials-and-methods.html#diversity-indices" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>Microbiome diversity was measured using vegan and <a href="https://bioconductor.org/packages/phyloseq">phyloseq</a> methods <span class="citation">[<a href="#ref-oksanen2020" role="doc-biblioref">189</a>]</span>.
<span class="math inline">\(\alpha\)</span>-diversity is a measure of how much a given microbiome at a taxonomic level is present on a sample.
Several measures exists, on the thesis I used the effective Simpson or effective Shannon diversity index to compare diversity between samples and conditions.
<span class="math inline">\(\beta\)</span>-diversity was calculated using the phyloseq package for exploratory analysis.</p>
<p>The effective Simpson (also known as inverse Simpson) and the effective Shannon are:</p>
<p><span class="math display">\[
\begin{aligned}
&amp; D_{\text{effective Simpson}} = \dfrac{1}{\sum_{1=1}^Sp_i^2} \\
&amp; D_{\text{effective Shannon}} = \dfrac{1}{-\sum_{i =1}^S p_i \log_e{p_i}}
\end{aligned}
\]</span></p>
<p>Where <span class="math inline">\(p_i\)</span> is the proportion of species <span class="math inline">\(i\)</span> and <span class="math inline">\(S\)</span> is the number of species.</p>
</div>
</div>
<div id="other-methods" class="section level2 hasAnchor" number="3.8">
<h2><span class="header-section-number">3.8</span> Other methods<a href="materials-and-methods.html#other-methods" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<div id="wgcna" class="section level3 hasAnchor" number="3.8.1">
<h3><span class="header-section-number">3.8.1</span> WGCNA<a href="materials-and-methods.html#wgcna" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>To look for relationships between the microbiome and the RNAseq we used weighted gene co-expression network analysis.
We used weighted gene co-expression network analysis as implemented on <a href="https://cran.r-project.org/package=WGCNA">WGCNA</a> <span class="citation">[<a href="#ref-langfelder2008" role="doc-biblioref">190</a>]</span> as well as correlations.
The Spearman rank correlation coefficient is:</p>
<p><span class="math display">\[
R_s(X,Y) = \dfrac{\sum_{i=1}^n (R_i - \bar{R}) (S_i - \bar{S} )}{\sqrt{\sum_{i=1}^n (R_i - \bar{R})^2}\sqrt{\sum_{i=1}^n (S_i - \bar{S})^2}}
\]</span></p>
<p>Being <span class="math inline">\((X_1 , Y_1 ),\dots, (X_n , Y_n)\)</span>, assign a rank where <span class="math inline">\((R_1 , S_1 ), \dots , (R_n , S_n )\)</span> for <span class="math inline">\(n\)</span> being all the variables where <span class="math inline">\(\bar{R}=\dfrac{1}{n}\sum_{i=1}^n R_i\)</span> and likewise <span class="math inline">\(\bar{S}=\dfrac{1}{n}\sum_{i=1}^n S_i\)</span>.
The distribution of the Spearman correlation coefficient is symmetric around 0 and can be approximated to a normal distribution as <span class="math inline">\(\sqrt{n-1}R_{s(X,Y)} \sim N(0,1)\)</span> which can be used to calculate the p-value of a given estimation.</p>
</div>
<div id="baseset" class="section level3 hasAnchor" number="3.8.2">
<h3><span class="header-section-number">3.8.2</span> BaseSet<a href="materials-and-methods.html#baseset" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p><a href="https://cran.r-project.org/package=BaseSet">BaseSet</a> was developed and used to find which variables are really involved on the interaction and how likely they were to be together.
It is a package that uses fuzzy set logic to calculate the probability to belong to a group, in this case, those genes and bacteria selected by the model that interact with the other.</p>
<p>Under the standard fuzzy set logic a set <span class="math inline">\(S\)</span> is a group of elements for which each element <span class="math inline">\(e\)</span> has an <span class="math inline">\(\alpha\)</span> membership to that set <span class="citation">[<a href="#ref-filzmoser2004" role="doc-biblioref">191</a>, <a href="#ref-dubois1993" role="doc-biblioref">192</a>]</span>.
<span class="math inline">\(\alpha\)</span> is usually bounded between 0 and 1: <span class="math inline">\(\alpha \in [0, 1]\)</span> .
A given element <span class="math inline">\(e\)</span> can belong to more than one set.
Assimilating the membership function to probability we can calculate the probability of a given element <span class="math inline">\(e\)</span> to belong to a set <span class="math inline">\(S\)</span> and not any other set: <span class="math inline">\(P(e \in S|e \not \in S^c)\)</span>.
Which applied to the data and the case at hand, it is the probability that a given variable is associated with a given outcome and not with any other outcome.</p>
<p>The membership function was derived from the bootstraps used for each model on the thousand iterations of the integrative method applied to give an estimation of how probable is a given gene and bacteria to be selected as relevant for the model.
The bootstraps of the models where used to calculate the probability of a variable to be selected by RGCCA.
This probability was used to calculate (via <code>set_size</code>) the genes and bacteria that are specific of the model that allows to separate the transcriptome by its location and the microbiome by the disease status.</p>
</div>
<div id="experdesign" class="section level3 hasAnchor" number="3.8.3">
<h3><span class="header-section-number">3.8.3</span> experDesign<a href="materials-and-methods.html#experdesign" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p><a href="https://cran.r-project.org/package=experDesign">experDesign</a> was developed <span class="citation">[<a href="#ref-revillasancho2021" role="doc-biblioref">193</a>]</span> to prevent and quantify if a given experiment has batch effect due to the batches used to measure the values or other known variable.
It might help to detect a bad design of the experiment.</p>
<p>On pseudo code the core of the program can be described as:</p>
<pre id="pseudocode-experDesign"><code>for each index:

    for each batch

        pick size(batch) samples

        if samples are in another batch

            pick other samples

    for each batch

        calculate some summary statistics

        compare with the summary statistics of all the samples

keep the index with less differences between the index and all the samples</code></pre>
<p>Summary statistics taken into account are the median, the variance, the range, the number of missing values, and the entropy of the categorical variables.
It can take into account spatial distribution and, given the number of samples that fit on a batch, provide which technical replicates<a href="#fn4" class="footnote-ref" id="fnref4"><sup>4</sup></a> are best to use.</p>
</div>
<div id="ROC" class="section level3 hasAnchor" number="3.8.4">
<h3><span class="header-section-number">3.8.4</span> ROC- AUC<a href="materials-and-methods.html#ROC" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>To estimate if the selected features (genes or microorganism) by the integration methods have some biological meaningful contribution I measured if they can classify features, such as, which gastrointestinal segment is each sample from, or which type of disease does each patient have.</p>
<p>To compare between different models the area under the curve (AUC) of the receiver operating characteristic (ROC) was calculated with the <a href="https://cran.r-project.org/package=pROC">pROC</a> package <span class="citation">[<a href="#ref-robin2011" role="doc-biblioref">194</a>]</span>.
It is based on the following formulas, where <span class="math inline">\(\text{FP}\)</span> is false positive, <span class="math inline">\(N\)</span> is a negative, <span class="math inline">\(P\)</span> is positive, <span class="math inline">\(\text{TP}\)</span> is true positive and <span class="math inline">\(\text{FN}\)</span> is false negative:</p>
<p><span class="math display">\[
\begin{aligned}
FPR = \frac{FP}{N} = \dfrac{FP}{FP+TN} \\
TPR = \frac{TP}{P} = \dfrac{FP}{FP+FN} 
\end{aligned}
\]</span></p>
<p>The ROC curve is that where the true positive rate (TPR) or sensitivity, recall or hit rate is represented against the false positive rate (FPR) on the abscissa.
The area under this curve is a measure of how good such classifications performs overall, being 0.5 as good as a random selection.
The closer it is to 1 the better as it classifies incorrectly less samples and accurately classify more.</p>

</div>
</div>
</div>
<h3>References</h3>
<div id="refs" class="references csl-bib-body">
<div id="ref-hasler_uncoupling_2016" class="csl-entry">
79. Häsler R, Sheibani-Tezerji R, Sinha A, Barann M, Rehman A, Esser D, et al. <a href="https://doi.org/10.1136/gutjnl-2016-311651">Uncoupling of mucosal gene regulation, <span class="nocase">mRNA</span> splicing and adherent microbiota signatures in inflammatory bowel disease</a>. Gut. 2017;66:2087–97.
</div>
<div id="ref-tenenhaus_regularized_2011" class="csl-entry">
113. Tenenhaus A, Tenenhaus M. <a href="https://doi.org/10.1007/s11336-011-9206-8">Regularized <span>Generalized Canonical Correlation Analysis</span></a>. Psychometrika. 2011;76:257–84.
</div>
<div id="ref-tenenhaus_variable_2014" class="csl-entry">
114. Tenenhaus A, Philippe C, Guillemot V, Le Cao K-A, Grill J, Frouin V. <a href="https://doi.org/10.1093/biostatistics/kxu001">Variable selection for generalized canonical correlation analysis</a>. Biostatistics. 2014;15:569–83.
</div>
<div id="ref-tenenhaus2017" class="csl-entry">
129. Tenenhaus M, Tenenhaus A, Groenen PJF. <a href="https://doi.org/10.1007/s11336-017-9573-x">Regularized Generalized Canonical Correlation Analysis: A Framework for Sequential Multiblock Component Methods</a>. Psychometrika. 2017;82:737–77.
</div>
<div id="ref-mccarthy2012" class="csl-entry">
145. McCarthy DJ, Chen Y, Smyth GK. <a href="https://doi.org/10.1093/nar/gks042">Differential expression analysis of multifactor RNA-Seq experiments with respect to biological variation</a>. Nucleic Acids Research. 2012;40:4288–97.
</div>
<div id="ref-puget2012" class="csl-entry">
149. Puget S, Philippe C, Bax DA, Job B, Varlet P, Junier M-P, et al. <a href="https://doi.org/10.1371/journal.pone.0030313">Mesenchymal Transition and PDGFRA Amplification/Mutation Are Key Distinct Oncogenic Events in Pediatric Diffuse Intrinsic Pontine Gliomas</a>. PLOS ONE. 2012;7:e30313.
</div>
<div id="ref-morgan2015" class="csl-entry">
150. Morgan XC, Kabakchiev B, Waldron L, Tyler AD, Tickle TL, Milgrom R, et al. <a href="https://doi.org/10.1186/s13059-015-0637-x">Associations between host gene expression, the mucosal microbiome, and clinical outcome in the pelvic pouch of patients with inflammatory bowel disease</a>. Genome Biology. 2015;16:67.
</div>
<div id="ref-howell2018" class="csl-entry">
151. Howell KJ, Kraiczy J, Nayak KM, Gasparetto M, Ross A, Lee C, et al. <a href="https://doi.org/10.1053/j.gastro.2017.10.007">DNA Methylation and Transcription Patterns in Intestinal Epithelial Cells From Pediatric Patients With Inflammatory Bowel Diseases Differentiate Disease Subtypes and Associate With Outcome</a>. Gastroenterology. 2018;154:585–98.
</div>
<div id="ref-martin2011" class="csl-entry">
152. Martin M. <a href="https://doi.org/10.14806/ej.17.1.200">Cutadapt removes adapter sequences from high-throughput sequencing reads</a>. EMBnetjournal. 2011;17:10–2.
</div>
<div id="ref-dobin2013" class="csl-entry">
153. Dobin A, Davis CA, Schlesinger F, Drenkow J, Zaleski C, Jha S, et al. <a href="https://doi.org/10.1093/bioinformatics/bts635">STAR: ultrafast universal RNA-seq aligner</a>. Bioinformatics (Oxford, England). 2013;29:15–21.
</div>
<div id="ref-li2011" class="csl-entry">
154. Li B, Dewey CN. <a href="https://doi.org/10.1186/1471-2105-12-323">RSEM: Accurate transcript quantification from RNA-seq data with or without a reference genome</a>. BMC Bioinformatics. 2011;12:323.
</div>
<div id="ref-corralizaDifferencesPeripheralTissue" class="csl-entry">
155. Corraliza AM, Ricart E, L’opez-García A, Carme Masamunt M, Veny M, Esteller M, et al. Differences in <span>Peripheral</span> and <span>Tissue Immune Cell Populations Following Haematopoietic Stem Cell Transplantation</span> in <span>Crohn</span>’s <span>Disease Patients</span>. Journal of Crohn’s and Colitis. <a href="https://doi.org/10.1093/ecco-jcc/jjy203">https://doi.org/10.1093/ecco-jcc/jjy203</a>.
</div>
<div id="ref-berry2011" class="csl-entry">
156. Berry D, Ben Mahfoudh K, Wagner M, Loy A. <a href="https://doi.org/10.1128/AEM.05220-11">Barcoded primers used in multiplex amplicon pyrosequencing bias amplification</a>. Applied and Environmental Microbiology. 2011;77:7846–9.
</div>
<div id="ref-klindworth2013" class="csl-entry">
157. Klindworth A, Pruesse E, Schweer T, Peplies J, Quast C, Horn M, et al. <a href="https://doi.org/10.1093/nar/gks808">Evaluation of general 16S ribosomal RNA gene PCR primers for classical and next-generation sequencing-based diversity studies</a>. Nucleic Acids Research. 2013;41:e1–1.
</div>
<div id="ref-edgar2013" class="csl-entry">
158. Edgar RC. <a href="https://doi.org/10.1038/nmeth.2604">UPARSE: highly accurate OTU sequences from microbial amplicon reads</a>. Nature Methods. 2013;10:996–8.
</div>
<div id="ref-lagkouvardos2016" class="csl-entry">
159. Lagkouvardos I, Joseph D, Kapfhammer M, Giritli S, Horn M, Haller D, et al. <a href="https://doi.org/10.1038/srep33721">IMNGS: A comprehensive open resource of processed 16S rRNA microbial profiles for ecology and diversity studies</a>. Scientific Reports. 2016;6.
</div>
<div id="ref-callahan2016" class="csl-entry">
160. Callahan BJ, McMurdie PJ, Rosen MJ, Han AW, Johnson AJA, Holmes SP. <a href="https://doi.org/10.1038/nmeth.3869">DADA2: High-resolution sample inference from Illumina amplicon data</a>. Nature Methods. 2016;13:581–3.
</div>
<div id="ref-meng2014" class="csl-entry">
161. Meng C, Kuster B, Culhane AC, Gholami AM. <a href="https://doi.org/10.1186/1471-2105-15-162">A multivariate approach to the integration of multi-omics datasets</a>. BMC Bioinformatics. 2014;15:162.
</div>
<div id="ref-planell2021" class="csl-entry">
162. Planell N, Lagani V, Sebastian-Leon P, van der Kloet F, Ewing E, Karathanasis N, et al. <a href="https://doi.org/10.3389/fgene.2021.620453">STATegra: Multi-omics data integration <span></span> a conceptual scheme with a bioinformatics pipeline</a>. Frontiers in Genetics. 2021;12.
</div>
<div id="ref-jordan1875" class="csl-entry">
163. Jordan C. <a href="https://doi.org/10.24033/bsmf.90">Essai sur la géométrie à n dimensions</a>. Bulletin de la Société Mathématique de France. 1875;3:103–74.
</div>
<div id="ref-hotelling1936" class="csl-entry">
164. HOTELLING H. <a href="https://doi.org/10.1093/biomet/28.3-4.321">RELATIONS BETWEEN TWO SETS OF VARIATES</a>. Biometrika. 1936;28:321–77.
</div>
<div id="ref-tenenhaus_component-based_2008" class="csl-entry">
165. Tenenhaus M. <a href="https://doi.org/10.1080/14783360802159543">Component-based <span>Structural Equation Modelling</span></a>. Total Quality Management &amp; Business Excellence. 2008;19:871–86.
</div>
<div id="ref-gloaguen2020" class="csl-entry">
169. Gloaguen A, Philippe C, Frouin V, Gennari G, Dehaene-Lambertz G, Le Brusquet L, et al. Multiway generalized canonical correlation analysis. Biostatistics. 2020. <a href="https://doi.org/10.1093/biostatistics/kxaa010">https://doi.org/10.1093/biostatistics/kxaa010</a>.
</div>
<div id="ref-schäfer2005" class="csl-entry">
170. Schäfer J, Strimmer K. <a href="https://doi.org/10.2202/1544-6115.1175">A Shrinkage Approach to Large-Scale Covariance Matrix Estimation and Implications for Functional Genomics</a>. Statistical Applications in Genetics and Molecular Biology. 2005;4.
</div>
<div id="ref-horst1961" class="csl-entry">
171. Horst P. <a href="https://doi.org/10.1007/BF02289710">Relations among m sets of measures</a>. Psychometrika. 1961;26:129–49.
</div>
<div id="ref-kettenring1971" class="csl-entry">
172. KETTENRING JR. <a href="https://doi.org/10.1093/biomet/58.3.433">Canonical analysis of several sets of variables</a>. Biometrika. 1971;58:433–51.
</div>
<div id="ref-vandegeer1984" class="csl-entry">
173. Van de Geer JP. <a href="https://doi.org/10.1007/BF02294207">Linear relations among k sets of variables</a>. Psychometrika. 1984;49:79–94.
</div>
<div id="ref-ritchie2015" class="csl-entry">
174. Ritchie MEM, Phipson B, Wu D, Hu Y, Law CWC, Shi W, et al. <a href="https://doi.org/10.1093/nar/gkv007">Limma powers differential expression analyses for RNA-sequencing and microarray studies</a>. Nucleic acids research. 2015;43:e47.
</div>
<div id="ref-law2014" class="csl-entry">
175. Law CW, Chen Y, Shi W, Smyth GK. <a href="https://doi.org/10.1186/gb-2014-15-2-r29">Voom: Precision weights unlock linear model analysis tools for RNA-seq read counts</a>. Genome Biology. 2014;15:R29.
</div>
<div id="ref-law2018" class="csl-entry">
176. Law CW, Alhamdoosh M, Su S, Dong X, Tian L, Smyth GK, et al. <a href="https://doi.org/10.12688/f1000research.9005.3">RNA-seq analysis is easy as 1-2-3 with limma, Glimma and edgeR</a>. F1000Research. 2018;5.
</div>
<div id="ref-yoavbenjamini" class="csl-entry">
177. Yoav Benjamini, Yosef Hochberg. <a href="http://www.jstor.org/stable/2346101">Controlling the False Discovery Rate: A Practical and Powerful Approach to Multiple Testing</a>. Journal of the Royal Statistical Society Series B (Methodological). 57.
</div>
<div id="ref-wu2021" class="csl-entry">
178. Wu T, Hu E, Xu S, Chen M, Guo P, Dai Z, et al. <a href="https://doi.org/10.1016/j.xinn.2021.100141">clusterProfiler 4.0: A universal enrichment tool for interpreting omics data</a>. The Innovation. 2021;2:100141.
</div>
<div id="ref-richter2021" class="csl-entry">
179. Richter FC, Friedrich M, Pohin M, Alsaleh G, Guschina I, Wideman SK, et al. <a href="https://doi.org/10.1101/2021.10.25.465200">Cell-extrinsic autophagy in mature adipocytes regulates anti-inflammatory response to intestinal tissue injury through lipid mobilization</a>. 2021;2021.10.25.465200.
</div>
<div id="ref-subramanian2005" class="csl-entry">
180. Subramanian A, Tamayo P, Mootha VK, Mukherjee S, Ebert BL, Gillette MA, et al. <a href="https://doi.org/10.1073/pnas.0506580102">Gene set enrichment analysis: A knowledge-based approach for interpreting genome-wide expression profiles</a>. Proceedings of the National Academy of Sciences. 2005;102:15545–50.
</div>
<div id="ref-protiva2016" class="csl-entry">
181. Protiva P, Pendyala S, Nelson C, Augenlicht LH, Lipkin M, Holt PR. <a href="https://doi.org/10.3945/ajcn.114.105304">Calcium and 1,25-dihydroxyvitamin D3 modulate genes of immune and inflammatory pathways in the human colon: A human crossover trial</a>. The American Journal of Clinical Nutrition. 2016;103:1224–31.
</div>
<div id="ref-korotkevich2021" class="csl-entry">
182. Korotkevich G, Sukhov V, Budin N, Shpak B, Artyomov MN, Sergushichev A. Fast gene set enrichment analysis. 2021. https://doi.org/<a href="https://doi.org/10.1101/060012">https://doi.org/10.1101/060012</a>.
</div>
<div id="ref-fabregat2016" class="csl-entry">
183. Fabregat A, Sidiropoulos K, Garapati P, Gillespie M, Hausmann K, Haw R, et al. <a href="https://doi.org/10.1093/nar/gkv1351">The reactome pathway knowledgebase</a>. Nucleic Acids Research. 2016;44.
</div>
<div id="ref-hänzelmann2013" class="csl-entry">
184. Hänzelmann S, Castelo R, Guinney J. <a href="https://doi.org/10.1186/1471-2105-14-7">GSVA: Gene set variation analysis for microarray and RNA-seq data</a>. BMC Bioinformatics. 2013;14:7.
</div>
<div id="ref-escudero-hernández2021" class="csl-entry">
185. Escudero-Hernández C, Beelen Granlund A van, Bruland T, Sandvik AK, Koch S, Østvik AE, et al. <a href="https://doi.org/10.1016/j.jcmgh.2021.04.011">Transcriptomic Profiling of Collagenous Colitis Identifies Hallmarks of Nondestructive Inflammatory Bowel Disease</a>. Cellular and Molecular Gastroenterology and Hepatology. 2021;12:665–87.
</div>
<div id="ref-anderson2001" class="csl-entry">
186. Anderson MJ. <a href="https://doi.org/10.1111/j.1442-9993.2001.01070.pp.x">A new method for non-parametric multivariate analysis of variance</a>. Austral Ecology. 2001;26:32–46.
</div>
<div id="ref-warton2012" class="csl-entry">
187. Warton DI, Wright ST, Wang Y. <a href="https://doi.org/10.1111/j.2041-210X.2011.00127.x">Distance-based multivariate analyses confound location and dispersion effects</a>. Methods in Ecology and Evolution. 2012;3:89–101.
</div>
<div id="ref-goeman2006" class="csl-entry">
188. Goeman JJ, van de Geer SA, van Houwelingen HC. <a href="https://www.jstor.org/stable/3879286">Testing against a high dimensional alternative</a>. Journal of the Royal Statistical Society Series B (Statistical Methodology). 2006;68:477–93.
</div>
<div id="ref-oksanen2020" class="csl-entry">
189. Oksanen J, Blanchet FG, Friendly M, Kindt R, Legendre P, McGlinn D, et al. <a href="https://CRAN.R-project.org/package=vegan">Vegan: Community ecology package</a>. 2020.
</div>
<div id="ref-langfelder2008" class="csl-entry">
190. Langfelder P, Horvath S. <a href="https://doi.org/10.1186/1471-2105-9-559">WGCNA: An r package for weighted correlation network analysis</a>. BMC Bioinformatics. 2008;9:559.
</div>
<div id="ref-filzmoser2004" class="csl-entry">
191. Filzmoser P, Viertl R. <a href="https://doi.org/10.1007/s001840300269">Testing hypotheses with fuzzy data: The fuzzy p -value</a>. Metrika. 2004;59:21–9.
</div>
<div id="ref-dubois1993" class="csl-entry">
192. Dubois D, Prade H. <a href="https://doi.org/10.1109/FUZZY.1993.327367">[Proceedings 1993] second IEEE international conference on fuzzy systems</a>. 1993. p. 1059–1068 vol.2.
</div>
<div id="ref-revillasancho2021" class="csl-entry">
193. Revilla Sancho L, Lozano J-J, Salas A. <a href="https://doi.org/10.21105/joss.03358">experDesign: stratifying samples into batches with minimal bias</a>. Journal of Open Source Software. 2021;6:3358.
</div>
<div id="ref-robin2011" class="csl-entry">
194. Robin X, Turck N, Hainard A, Tiberti N, Lisacek F, Sanchez J-C, et al. <a href="https://doi.org/10.1186/1471-2105-12-77">pROC: An open-source package for r and s+ to analyze and compare ROC curves</a>. BMC Bioinformatics. 2011;12:77.
</div>
</div>
<div class="footnotes">
<hr />
<ol start="2">
<li id="fn2"><p>CRAN is The Comprehensive R Archive Network available at <a href="https://cran.r-project.org/" class="uri">https://cran.r-project.org/</a><a href="materials-and-methods.html#fnref2" class="footnote-back">↩︎</a></p></li>
<li id="fn3"><p>without replacement<a href="materials-and-methods.html#fnref3" class="footnote-back">↩︎</a></p></li>
<li id="fn4"><p>Technical replicates are samples that are measured multiple times to ensure the accuracy of the measurement.
In contrast biological replicates are sample with the same conditions but different individuals or process before the measurement.<a href="materials-and-methods.html#fnref4" class="footnote-back">↩︎</a></p></li>
</ol>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="hypothesis-and-objectives.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="results.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": {
"href": "https://github.com/llrs/thesis/"
},
"facebook": false,
"twitter": {
"href": "https://twitter.com/intent/tweet?text=Nice%20read%20of%20%40Lluis_Revilla%20thesis"
},
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/llrs/thesis/edit/master/03_methods.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": "https://github.com/llrs/thesis/blob/master/03_methods.Rmd",
"text": "View"
},
"download": ["_main.pdf", "_main.epub"],
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": false,
"toc_float": true,
"toc_appendix": true,
"toc_depth": 2,
"scroll_highlight": true
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
