<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>A Online resources | Data integration on inflammatory bowel disease</title>
  <meta name="description" content="A Online resources | Data integration on inflammatory bowel disease" />
  <meta name="generator" content="bookdown 0.25 and GitBook 2.6.7" />

  <meta property="og:title" content="A Online resources | Data integration on inflammatory bowel disease" />
  <meta property="og:type" content="book" />
  
  
  <meta name="github-repo" content="llrs/thesis" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="A Online resources | Data integration on inflammatory bowel disease" />
  <meta name="twitter:site" content="@Lluis_Revilla" />
  
  

<meta name="author" content="Lluís Revilla Sancho" />


<meta name="date" content="2022-04-11" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="references.html"/>
<link rel="next" href="software.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1, orientation=auto"/>
<meta name="description" content="Thesis searching the relationship between microbiome and transcriptome on the intestine on patients with IBD to discover key genes and bacteria."/>
<meta name="github-repo" content="llrs/thesis"/>
<meta name="twitter:title" content="A Online resources | Data integration on inflammatory bowel disease"/>
<meta name="twitter:description" content="Thesis searching the relationship between microbiome and transcriptome on the intestine on patients with IBD to discover key genes and bacteria."/>
<meta name="twitter:url" content="https://thesis.llrs.dev"/>
<meta name="twitter:image:alt" content="The cover of the thesis."/>
<meta name="twitter:card" content="summary_large_image"/>
<meta name="twitter:creator" content="@Lluis_Revilla"/>
<meta name="twitter:site" content="@Lluis_Revilla"/>
<meta property="og:title" content="A Online resources | Data integration on inflammatory bowel disease"/>
<meta property="og:description" content="Thesis searching the relationship between microbiome and transcriptome on the intestine on patients with IBD to discover key genes and bacteria."/>
<meta property="og:url" content="https://thesis.llrs.dev"/>
<meta property="og:image:alt" content="The cover of the thesis."/>
<meta property="og:type" content="book"/>
<meta property="og:locale" content="en_US"/>
<meta property="article:author" content="Llu&lt;U+00ED&gt;s Revilla Sancho"/>
<script src="libs/kePrint-0.0.1/kePrint.js"></script>
<link href="libs/lightable-0.0.1/lightable.css" rel="stylesheet" />



<style type="text/css">
/* Used with Pandoc 2.11+ new --citeproc when CSL is used */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><strong><a href="./">Data integration on IBD</a></strong></li>

<li class="divider"></li>
<li><a href="index.html#preface">Preface<span></span></a></li>
<li><a href="abstracts.html#abstracts">Abstracts<span></span></a>
<ul>
<li><a href="abstracts.html#english">English<span></span></a></li>
<li><a href="abstracts.html#spanish">Spanish<span></span></a></li>
<li><a href="abstracts.html#catalan">Catalan<span></span></a></li>
</ul></li>
<li class="chapter" data-level="1" data-path="introduction.html"><a href="introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction<span></span></a>
<ul>
<li class="chapter" data-level="1.1" data-path="introduction.html"><a href="introduction.html#IBD"><i class="fa fa-check"></i><b>1.1</b> Inflammatory bowel disease<span></span></a>
<ul>
<li class="chapter" data-level="1.1.1" data-path="introduction.html"><a href="introduction.html#etiology-and-pathogenesis"><i class="fa fa-check"></i><b>1.1.1</b> Etiology and pathogenesis<span></span></a></li>
<li class="chapter" data-level="1.1.2" data-path="introduction.html"><a href="introduction.html#CD"><i class="fa fa-check"></i><b>1.1.2</b> CD physiology<span></span></a></li>
<li class="chapter" data-level="1.1.3" data-path="introduction.html"><a href="introduction.html#UC"><i class="fa fa-check"></i><b>1.1.3</b> UC physiology<span></span></a></li>
<li class="chapter" data-level="1.1.4" data-path="introduction.html"><a href="introduction.html#treatment"><i class="fa fa-check"></i><b>1.1.4</b> Treatment<span></span></a></li>
<li class="chapter" data-level="1.1.5" data-path="introduction.html"><a href="introduction.html#summary-ibd"><i class="fa fa-check"></i><b>1.1.5</b> Summary<span></span></a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="introduction.html"><a href="introduction.html#integration-studies-on-ibd"><i class="fa fa-check"></i><b>1.2</b> Integration studies on IBD<span></span></a>
<ul>
<li class="chapter" data-level="1.2.1" data-path="introduction.html"><a href="introduction.html#data-origin"><i class="fa fa-check"></i><b>1.2.1</b> Type of data used for integration analysis<span></span></a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="introduction.html"><a href="introduction.html#integration"><i class="fa fa-check"></i><b>1.3</b> Integration<span></span></a>
<ul>
<li class="chapter" data-level="1.3.1" data-path="introduction.html"><a href="introduction.html#classification-of-integration-methods"><i class="fa fa-check"></i><b>1.3.1</b> Classification of integration methods<span></span></a></li>
<li class="chapter" data-level="1.3.2" data-path="introduction.html"><a href="introduction.html#interpretation"><i class="fa fa-check"></i><b>1.3.2</b> Interpretation<span></span></a></li>
<li class="chapter" data-level="1.3.3" data-path="introduction.html"><a href="introduction.html#reviews"><i class="fa fa-check"></i><b>1.3.3</b> Reviews<span></span></a></li>
<li class="chapter" data-level="1.3.4" data-path="introduction.html"><a href="introduction.html#methods-integration"><i class="fa fa-check"></i><b>1.3.4</b> Summary<span></span></a></li>
</ul></li>
<li class="chapter" data-level="1.4" data-path="introduction.html"><a href="introduction.html#summary"><i class="fa fa-check"></i><b>1.4</b> Summary<span></span></a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="hypothesis-and-objectives.html"><a href="hypothesis-and-objectives.html"><i class="fa fa-check"></i><b>2</b> Hypothesis and objectives<span></span></a>
<ul>
<li class="chapter" data-level="2.1" data-path="hypothesis-and-objectives.html"><a href="hypothesis-and-objectives.html#hypothesis"><i class="fa fa-check"></i><b>2.1</b> Hypothesis<span></span></a></li>
<li class="chapter" data-level="2.2" data-path="hypothesis-and-objectives.html"><a href="hypothesis-and-objectives.html#objectives"><i class="fa fa-check"></i><b>2.2</b> Objectives<span></span></a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="materials-and-methods.html"><a href="materials-and-methods.html"><i class="fa fa-check"></i><b>3</b> Materials and methods<span></span></a>
<ul>
<li class="chapter" data-level="3.1" data-path="materials-and-methods.html"><a href="materials-and-methods.html#datasets"><i class="fa fa-check"></i><b>3.1</b> Datasets<span></span></a>
<ul>
<li class="chapter" data-level="3.1.1" data-path="materials-and-methods.html"><a href="materials-and-methods.html#methods-puget"><i class="fa fa-check"></i><b>3.1.1</b> Puget’s dataset<span></span></a></li>
<li class="chapter" data-level="3.1.2" data-path="materials-and-methods.html"><a href="materials-and-methods.html#methods-hsct"><i class="fa fa-check"></i><b>3.1.2</b> HSCT dataset<span></span></a></li>
<li class="chapter" data-level="3.1.3" data-path="materials-and-methods.html"><a href="materials-and-methods.html#methods-hasler"><i class="fa fa-check"></i><b>3.1.3</b> Häsler’s dataset<span></span></a></li>
<li class="chapter" data-level="3.1.4" data-path="materials-and-methods.html"><a href="materials-and-methods.html#methods-morgan"><i class="fa fa-check"></i><b>3.1.4</b> Morgan’s dataset<span></span></a></li>
<li class="chapter" data-level="3.1.5" data-path="materials-and-methods.html"><a href="materials-and-methods.html#methods-howell"><i class="fa fa-check"></i><b>3.1.5</b> Howell’s dataset<span></span></a></li>
</ul></li>
<li class="chapter" data-level="3.2" data-path="materials-and-methods.html"><a href="materials-and-methods.html#processing"><i class="fa fa-check"></i><b>3.2</b> Sample processing<span></span></a>
<ul>
<li class="chapter" data-level="3.2.1" data-path="materials-and-methods.html"><a href="materials-and-methods.html#rna-sequencing"><i class="fa fa-check"></i><b>3.2.1</b> RNA sequencing<span></span></a></li>
<li class="chapter" data-level="3.2.2" data-path="materials-and-methods.html"><a href="materials-and-methods.html#microbial-dna-sequencing"><i class="fa fa-check"></i><b>3.2.2</b> Microbial DNA sequencing<span></span></a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="materials-and-methods.html"><a href="materials-and-methods.html#integration-methods"><i class="fa fa-check"></i><b>3.3</b> Integration methods<span></span></a>
<ul>
<li class="chapter" data-level="3.3.1" data-path="materials-and-methods.html"><a href="materials-and-methods.html#regularized-generalized-canonical-correlation-analysis"><i class="fa fa-check"></i><b>3.3.1</b> Regularized generalized canonical correlation analysis<span></span></a></li>
<li class="chapter" data-level="3.3.2" data-path="materials-and-methods.html"><a href="materials-and-methods.html#other"><i class="fa fa-check"></i><b>3.3.2</b> Other<span></span></a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="materials-and-methods.html"><a href="materials-and-methods.html#functional-enrichment-methods"><i class="fa fa-check"></i><b>3.4</b> Functional enrichment methods<span></span></a>
<ul>
<li class="chapter" data-level="3.4.1" data-path="materials-and-methods.html"><a href="materials-and-methods.html#ORA"><i class="fa fa-check"></i><b>3.4.1</b> Over representation analysis<span></span></a></li>
<li class="chapter" data-level="3.4.2" data-path="materials-and-methods.html"><a href="materials-and-methods.html#gene-set-enrichment-analysis"><i class="fa fa-check"></i><b>3.4.2</b> Gene Set Enrichment Analysis<span></span></a></li>
</ul></li>
<li class="chapter" data-level="3.5" data-path="materials-and-methods.html"><a href="materials-and-methods.html#variance"><i class="fa fa-check"></i><b>3.5</b> Variance and diversity methods<span></span></a>
<ul>
<li class="chapter" data-level="3.5.1" data-path="materials-and-methods.html"><a href="materials-and-methods.html#permanova"><i class="fa fa-check"></i><b>3.5.1</b> PERMANOVA<span></span></a></li>
<li class="chapter" data-level="3.5.2" data-path="materials-and-methods.html"><a href="materials-and-methods.html#globaltest"><i class="fa fa-check"></i><b>3.5.2</b> globaltest<span></span></a></li>
<li class="chapter" data-level="3.5.3" data-path="materials-and-methods.html"><a href="materials-and-methods.html#diversity-indices"><i class="fa fa-check"></i><b>3.5.3</b> Diversity indices<span></span></a></li>
</ul></li>
<li class="chapter" data-level="3.6" data-path="materials-and-methods.html"><a href="materials-and-methods.html#other-methods"><i class="fa fa-check"></i><b>3.6</b> Other methods<span></span></a>
<ul>
<li class="chapter" data-level="3.6.1" data-path="materials-and-methods.html"><a href="materials-and-methods.html#statistics"><i class="fa fa-check"></i><b>3.6.1</b> Statistics<span></span></a></li>
<li class="chapter" data-level="3.6.2" data-path="materials-and-methods.html"><a href="materials-and-methods.html#wgcna"><i class="fa fa-check"></i><b>3.6.2</b> WGCNA<span></span></a></li>
<li class="chapter" data-level="3.6.3" data-path="materials-and-methods.html"><a href="materials-and-methods.html#baseset"><i class="fa fa-check"></i><b>3.6.3</b> BaseSet<span></span></a></li>
<li class="chapter" data-level="3.6.4" data-path="materials-and-methods.html"><a href="materials-and-methods.html#experdesign"><i class="fa fa-check"></i><b>3.6.4</b> experDesign<span></span></a></li>
<li class="chapter" data-level="3.6.5" data-path="materials-and-methods.html"><a href="materials-and-methods.html#ROC"><i class="fa fa-check"></i><b>3.6.5</b> ROC- AUC<span></span></a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="results.html"><a href="results.html"><i class="fa fa-check"></i><b>4</b> Results<span></span></a>
<ul>
<li class="chapter" data-level="4.1" data-path="results.html"><a href="results.html#packagesmethods"><i class="fa fa-check"></i><b>4.1</b> Packages/methods<span></span></a>
<ul>
<li class="chapter" data-level="4.1.1" data-path="results.html"><a href="results.html#experdesign-1"><i class="fa fa-check"></i><b>4.1.1</b> experDesign<span></span></a></li>
<li class="chapter" data-level="4.1.2" data-path="results.html"><a href="results.html#baseset-1"><i class="fa fa-check"></i><b>4.1.2</b> BaseSet<span></span></a></li>
<li class="chapter" data-level="4.1.3" data-path="results.html"><a href="results.html#intermodel"><i class="fa fa-check"></i><b>4.1.3</b> inteRmodel<span></span></a></li>
</ul></li>
<li class="chapter" data-level="4.2" data-path="results.html"><a href="results.html#analysis"><i class="fa fa-check"></i><b>4.2</b> Analysis<span></span></a>
<ul>
<li class="chapter" data-level="4.2.1" data-path="results.html"><a href="results.html#results-puget"><i class="fa fa-check"></i><b>4.2.1</b> Puget’s dataset<span></span></a></li>
<li class="chapter" data-level="4.2.2" data-path="results.html"><a href="results.html#results-hsct"><i class="fa fa-check"></i><b>4.2.2</b> HSCT dataset<span></span></a></li>
<li class="chapter" data-level="4.2.3" data-path="results.html"><a href="results.html#results-hasler"><i class="fa fa-check"></i><b>4.2.3</b> Häsler’s dataset<span></span></a></li>
<li class="chapter" data-level="4.2.4" data-path="results.html"><a href="results.html#results-morgan"><i class="fa fa-check"></i><b>4.2.4</b> Morgan’s dataset<span></span></a></li>
<li class="chapter" data-level="4.2.5" data-path="results.html"><a href="results.html#results-howell"><i class="fa fa-check"></i><b>4.2.5</b> Howell’s dataset<span></span></a></li>
<li class="chapter" data-level="4.2.6" data-path="results.html"><a href="results.html#between-datasets"><i class="fa fa-check"></i><b>4.2.6</b> Between datasets<span></span></a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="discussion.html"><a href="discussion.html"><i class="fa fa-check"></i><b>5</b> Discussion<span></span></a>
<ul>
<li class="chapter" data-level="5.1" data-path="discussion.html"><a href="discussion.html#preliminary-steps"><i class="fa fa-check"></i><b>5.1</b> Preliminary steps<span></span></a>
<ul>
<li class="chapter" data-level="5.1.1" data-path="discussion.html"><a href="discussion.html#discussion-datasets"><i class="fa fa-check"></i><b>5.1.1</b> The datasets<span></span></a></li>
<li class="chapter" data-level="5.1.2" data-path="discussion.html"><a href="discussion.html#the-methods"><i class="fa fa-check"></i><b>5.1.2</b> The methods<span></span></a></li>
</ul></li>
<li class="chapter" data-level="5.2" data-path="discussion.html"><a href="discussion.html#designing-models"><i class="fa fa-check"></i><b>5.2</b> Designing models<span></span></a></li>
<li class="chapter" data-level="5.3" data-path="discussion.html"><a href="discussion.html#evaluating-models"><i class="fa fa-check"></i><b>5.3</b> Evaluating models<span></span></a></li>
<li class="chapter" data-level="5.4" data-path="discussion.html"><a href="discussion.html#implications"><i class="fa fa-check"></i><b>5.4</b> Implications<span></span></a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="conclusions.html"><a href="conclusions.html"><i class="fa fa-check"></i><b>6</b> Conclusions<span></span></a>
<ul>
<li class="chapter" data-level="6.1" data-path="conclusions.html"><a href="conclusions.html#study-1-multi-omic-modelling-of-inflammatory-bowel-disease-with-regularized-canonical-correlation-analysis"><i class="fa fa-check"></i><b>6.1</b> Study 1: Multi-omic modelling of inflammatory bowel disease with regularized canonical correlation analysis<span></span></a></li>
<li class="chapter" data-level="6.2" data-path="conclusions.html"><a href="conclusions.html#study-2-genes-and-microbiome-relationship-on-inflammatory-bowel-disease"><i class="fa fa-check"></i><b>6.2</b> Study 2: Genes and microbiome relationship on inflammatory bowel disease<span></span></a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="acknowledgments.html"><a href="acknowledgments.html"><i class="fa fa-check"></i><b>7</b> Acknowledgments<span></span></a></li>
<li><a href="references.html#references">References<span></span></a></li>
<li class="appendix"><span><b>Appendix<span></span></b></span></li>
<li class="chapter" data-level="A" data-path="online-resources.html"><a href="online-resources.html"><i class="fa fa-check"></i><b>A</b> Online resources<span></span></a></li>
<li class="chapter" data-level="B" data-path="software.html"><a href="software.html"><i class="fa fa-check"></i><b>B</b> Software<span></span></a>
<ul>
<li class="chapter" data-level="B.1" data-path="software.html"><a href="software.html#star"><i class="fa fa-check"></i><b>B.1</b> STAR<span></span></a></li>
<li class="chapter" data-level="B.2" data-path="software.html"><a href="software.html#rsem"><i class="fa fa-check"></i><b>B.2</b> RSEM<span></span></a></li>
<li class="chapter" data-level="B.3" data-path="software.html"><a href="software.html#listed"><i class="fa fa-check"></i><b>B.3</b> Listed<span></span></a></li>
<li class="chapter" data-level="B.4" data-path="software.html"><a href="software.html#by-projectpublication"><i class="fa fa-check"></i><b>B.4</b> By project/publication<span></span></a></li>
</ul></li>
<li class="chapter" data-level="C" data-path="articles.html"><a href="articles.html"><i class="fa fa-check"></i><b>C</b> Articles<span></span></a>
<ul>
<li class="chapter" data-level="C.1" data-path="articles.html"><a href="articles.html#app-multiomics"><i class="fa fa-check"></i><b>C.1</b> Multi-omic modelling of inflammatory bowel disease with regularized canonical correlation analysis<span></span></a></li>
<li class="chapter" data-level="C.2" data-path="articles.html"><a href="articles.html#app-experDesign"><i class="fa fa-check"></i><b>C.2</b> experDesign: stratifying samples into batches with minimal bias<span></span></a></li>
</ul></li>
<li class="chapter" data-level="D" data-path="other-datasets.html"><a href="other-datasets.html"><i class="fa fa-check"></i><b>D</b> Other datasets<span></span></a>
<ul>
<li class="chapter" data-level="D.1" data-path="other-datasets.html"><a href="other-datasets.html#barcelona"><i class="fa fa-check"></i><b>D.1</b> BARCELONA dataset<span></span></a></li>
<li class="chapter" data-level="D.2" data-path="other-datasets.html"><a href="other-datasets.html#hernandez"><i class="fa fa-check"></i><b>D.2</b> Hernández’ dataset<span></span></a>
<ul>
<li class="chapter" data-level="D.2.1" data-path="other-datasets.html"><a href="other-datasets.html#results-hernandez"><i class="fa fa-check"></i><b>D.2.1</b> Results<span></span></a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="E" data-path="models-output.html"><a href="models-output.html"><i class="fa fa-check"></i><b>E</b> Models output<span></span></a>
<ul>
<li class="chapter" data-level="E.1" data-path="models-output.html"><a href="models-output.html#hsct"><i class="fa fa-check"></i><b>E.1</b> HSCT<span></span></a>
<ul>
<li class="chapter" data-level="E.1.1" data-path="models-output.html"><a href="models-output.html#genes"><i class="fa fa-check"></i><b>E.1.1</b> Genes<span></span></a></li>
<li class="chapter" data-level="E.1.2" data-path="models-output.html"><a href="models-output.html#microbiome-2"><i class="fa fa-check"></i><b>E.1.2</b> Microbiome<span></span></a></li>
</ul></li>
<li class="chapter" data-level="E.2" data-path="models-output.html"><a href="models-output.html#häsler"><i class="fa fa-check"></i><b>E.2</b> Häsler<span></span></a>
<ul>
<li class="chapter" data-level="E.2.1" data-path="models-output.html"><a href="models-output.html#genes-1"><i class="fa fa-check"></i><b>E.2.1</b> Genes<span></span></a></li>
<li class="chapter" data-level="E.2.2" data-path="models-output.html"><a href="models-output.html#microbiome-3"><i class="fa fa-check"></i><b>E.2.2</b> Microbiome<span></span></a></li>
</ul></li>
<li class="chapter" data-level="E.3" data-path="models-output.html"><a href="models-output.html#morgan"><i class="fa fa-check"></i><b>E.3</b> Morgan<span></span></a>
<ul>
<li class="chapter" data-level="E.3.1" data-path="models-output.html"><a href="models-output.html#genes-2"><i class="fa fa-check"></i><b>E.3.1</b> Genes<span></span></a></li>
<li class="chapter" data-level="E.3.2" data-path="models-output.html"><a href="models-output.html#microbiome-4"><i class="fa fa-check"></i><b>E.3.2</b> Microbiome<span></span></a></li>
</ul></li>
<li class="chapter" data-level="E.4" data-path="models-output.html"><a href="models-output.html#howell"><i class="fa fa-check"></i><b>E.4</b> Howell<span></span></a>
<ul>
<li class="chapter" data-level="E.4.1" data-path="models-output.html"><a href="models-output.html#genes-3"><i class="fa fa-check"></i><b>E.4.1</b> Genes<span></span></a></li>
<li class="chapter" data-level="E.4.2" data-path="models-output.html"><a href="models-output.html#microbiome-5"><i class="fa fa-check"></i><b>E.4.2</b> Microbiome<span></span></a></li>
</ul></li>
</ul></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Data integration on inflammatory bowel disease</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div class = "wip">
This tesis is ongoing and still without hard date to finish it. Watch the <a href="https://github.com/llrs/thesis">GitHub repository</a> or wait for a release announcement on Twitter on <a href="https://twitter.com/Lluis_Revilla">my profile</a>.
</div>
<div id="online-resources" class="section level1 hasAnchor" number="8">
<h1><span class="header-section-number">A</span> Online resources<a href="online-resources.html#online-resources" class="anchor-section" aria-label="Anchor link to header"></a></h1>
<p>Some links that we found useful on the thesis and could be useful if you are interested on the multi-omics field.</p>
<ul>
<li><a href="https://github.com/mikelove/awesome-multi-omics">Awesome multi-omics https://github.com/mikelove/awesome-multi-omics</a> : An online repository of references to multi-omics methods. Reproduced here with their references <a href="#fn8" class="footnote-ref" id="fnref8"><sup>8</sup></a>:</li>
</ul>
<table>
<caption><span id="tab:appendix-methods">Table A.1: </span> Integration methods available and their references.</caption>
<thead>
<tr class="header">
<th align="center">Method</th>
<th align="center">Publication</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="center">SCCA</td>
<td align="center"><span class="citation">[<a href="#ref-parkhomenko2009" role="doc-biblioref">243</a>]</span></td>
</tr>
<tr class="even">
<td align="center">PCCA</td>
<td align="center"><span class="citation">[<a href="#ref-waaijenborg2008" role="doc-biblioref">244</a>]</span></td>
</tr>
<tr class="odd">
<td align="center">PMA</td>
<td align="center"><span class="citation">[<a href="#ref-witten2009" role="doc-biblioref">245</a>]</span></td>
</tr>
<tr class="even">
<td align="center">sPLS</td>
<td align="center"><span class="citation">[<a href="#ref-lêcao2009" role="doc-biblioref">246</a>]</span></td>
</tr>
<tr class="odd">
<td align="center">gesca</td>
<td align="center"><span class="citation">[<a href="#ref-hwang2009" role="doc-biblioref">247</a>]</span></td>
</tr>
<tr class="even">
<td align="center">Regularized dual CCA</td>
<td align="center"><span class="citation">[<a href="#ref-soneson2010" role="doc-biblioref">248</a>]</span></td>
</tr>
<tr class="odd">
<td align="center">RGCCA</td>
<td align="center"><span class="citation">[<a href="#ref-tenenhaus_regularized_2011" role="doc-biblioref">114</a>]</span></td>
</tr>
<tr class="even">
<td align="center">SNMNMF</td>
<td align="center"><span class="citation">[<a href="#ref-zhang2011" role="doc-biblioref">249</a>]</span></td>
</tr>
<tr class="odd">
<td align="center">scca</td>
<td align="center"><span class="citation">[<a href="#ref-lee2011" role="doc-biblioref">250</a>]</span></td>
</tr>
<tr class="even">
<td align="center">STATIS</td>
<td align="center"><span class="citation">[<a href="#ref-abdi2012" role="doc-biblioref">251</a>]</span></td>
</tr>
<tr class="odd">
<td align="center">joint NMF</td>
<td align="center"><span class="citation">[<a href="#ref-zhang2012" role="doc-biblioref">252</a>]</span></td>
</tr>
<tr class="even">
<td align="center">sMBPLS</td>
<td align="center"><span class="citation">[<a href="#ref-li2012" role="doc-biblioref">253</a>]</span></td>
</tr>
<tr class="odd">
<td align="center">Bayesian group factor analysis</td>
<td align="center"><span class="citation">[<a href="#ref-virtanen2012" role="doc-biblioref">133</a>]</span></td>
</tr>
<tr class="even">
<td align="center">RIMBANET</td>
<td align="center"><span class="citation">[<a href="#ref-zhu2012" role="doc-biblioref">127</a>]</span></td>
</tr>
<tr class="odd">
<td align="center">FactoMineR</td>
<td align="center"><span class="citation">[<a href="#ref-abdi2013" role="doc-biblioref">254</a>]</span></td>
</tr>
<tr class="even">
<td align="center">JIVE</td>
<td align="center"><span class="citation">[<a href="#ref-lock2013a" role="doc-biblioref">135</a>]</span></td>
</tr>
<tr class="odd">
<td align="center">pandaR</td>
<td align="center"><span class="citation">[<a href="#ref-schlauch2017" role="doc-biblioref">255</a>]</span></td>
</tr>
<tr class="even">
<td align="center">omicade4</td>
<td align="center"><span class="citation">[<a href="#ref-meng2014" role="doc-biblioref">173</a>]</span></td>
</tr>
<tr class="odd">
<td align="center">STATegRa</td>
<td align="center"><span class="citation">[<a href="#ref-planell2021" role="doc-biblioref">174</a>]</span></td>
</tr>
<tr class="even">
<td align="center">Joint factor model</td>
<td align="center"><span class="citation">[<a href="#ref-ray2014" role="doc-biblioref">256</a>]</span></td>
</tr>
<tr class="odd">
<td align="center">GFAsparse</td>
<td align="center"><span class="citation">[<a href="#ref-bunte2016" role="doc-biblioref">257</a>]</span></td>
</tr>
<tr class="even">
<td align="center">Sparse CCA</td>
<td align="center"><span class="citation">[<a href="#ref-chen2013" role="doc-biblioref">258</a>]</span></td>
</tr>
<tr class="odd">
<td align="center">CCAGFA</td>
<td align="center"><span class="citation">[<a href="#ref-leppäaho2017" role="doc-biblioref">259</a>]</span></td>
</tr>
<tr class="even">
<td align="center">CMF</td>
<td align="center"><span class="citation">[<a href="#ref-klami2014" role="doc-biblioref">260</a>]</span></td>
</tr>
<tr class="odd">
<td align="center">MOGSA</td>
<td align="center"><span class="citation">[<a href="#ref-meng2018" role="doc-biblioref">261</a>]</span></td>
</tr>
<tr class="even">
<td align="center">iNMF</td>
<td align="center"><span class="citation">[<a href="#ref-yang2016b" role="doc-biblioref">262</a>]</span></td>
</tr>
<tr class="odd">
<td align="center">BASS</td>
<td align="center"><span class="citation">[<a href="#ref-zhao2015" role="doc-biblioref">263</a>]</span></td>
</tr>
<tr class="even">
<td align="center">imputeMFA</td>
<td align="center"><span class="citation">[<a href="#ref-voillet2016" role="doc-biblioref">264</a>]</span></td>
</tr>
<tr class="odd">
<td align="center">PLSCA</td>
<td align="center"><span class="citation">[<a href="#ref-beaton2016" role="doc-biblioref">265</a>]</span></td>
</tr>
<tr class="even">
<td align="center">mixOmics</td>
<td align="center"><span class="citation">[<a href="#ref-singh2019" role="doc-biblioref">266</a>]</span></td>
</tr>
<tr class="odd">
<td align="center">mixedCCA</td>
<td align="center"><span class="citation">[<a href="#ref-yoon2019" role="doc-biblioref">267</a>]</span></td>
</tr>
<tr class="even">
<td align="center">SLIDE</td>
<td align="center"><span class="citation">[<a href="#ref-gaynanova2017" role="doc-biblioref">268</a>]</span></td>
</tr>
<tr class="odd">
<td align="center">fCCAC</td>
<td align="center"><span class="citation">[<a href="#ref-madrigal2017" role="doc-biblioref">269</a>]</span></td>
</tr>
<tr class="even">
<td align="center">TSKCCA</td>
<td align="center"><span class="citation">[<a href="#ref-yoshida2017" role="doc-biblioref">270</a>]</span></td>
</tr>
<tr class="odd">
<td align="center">SMSMA</td>
<td align="center"><span class="citation">[<a href="#ref-kawaguchi2017" role="doc-biblioref">271</a>]</span></td>
</tr>
<tr class="even">
<td align="center">AJIVE</td>
<td align="center"><span class="citation">[<a href="#ref-feng2018" role="doc-biblioref">272</a>]</span></td>
</tr>
<tr class="odd">
<td align="center">MOFA</td>
<td align="center"><span class="citation">[<a href="#ref-argelaguet2020" role="doc-biblioref">273</a>]</span></td>
</tr>
<tr class="even">
<td align="center">PCA+CCA</td>
<td align="center"><span class="citation">[<a href="#ref-brown2018" role="doc-biblioref">274</a>]</span></td>
</tr>
<tr class="odd">
<td align="center">JACA</td>
<td align="center"><span class="citation">[<a href="#ref-zhang2020" role="doc-biblioref">275</a>]</span></td>
</tr>
<tr class="even">
<td align="center">iPCA</td>
<td align="center"><span class="citation">[<a href="#ref-tang2021" role="doc-biblioref">276</a>]</span></td>
</tr>
<tr class="odd">
<td align="center">pCIA</td>
<td align="center"><span class="citation">[<a href="#ref-min2019" role="doc-biblioref">277</a>]</span></td>
</tr>
<tr class="even">
<td align="center">sSCCA</td>
<td align="center"><span class="citation">[<a href="#ref-safo2018" role="doc-biblioref">278</a>]</span></td>
</tr>
<tr class="odd">
<td align="center">SWCCA</td>
<td align="center"><span class="citation">[<a href="#ref-min2017" role="doc-biblioref">279</a>]</span></td>
</tr>
<tr class="even">
<td align="center">OmicsPLS</td>
<td align="center"><span class="citation">[<a href="#ref-bouhaddani2018" role="doc-biblioref">280</a>]</span></td>
</tr>
<tr class="odd">
<td align="center">SCCA-BC</td>
<td align="center"><span class="citation">[<a href="#ref-pimentel2018" role="doc-biblioref">281</a>]</span></td>
</tr>
<tr class="even">
<td align="center">WON-PARAFAC</td>
<td align="center"><span class="citation">[<a href="#ref-kim2019" role="doc-biblioref">282</a>]</span></td>
</tr>
<tr class="odd">
<td align="center">BIDIFAC</td>
<td align="center"><span class="citation">[<a href="#ref-lock2020" role="doc-biblioref">283</a>]</span></td>
</tr>
<tr class="even">
<td align="center">maui</td>
<td align="center"><span class="citation">[<a href="#ref-ronen2019" role="doc-biblioref">284</a>]</span></td>
</tr>
<tr class="odd">
<td align="center">SmCCNet</td>
<td align="center"><span class="citation">[<a href="#ref-shi2019" role="doc-biblioref">285</a>]</span></td>
</tr>
<tr class="even">
<td align="center">msPLS</td>
<td align="center"><span class="citation">[<a href="#ref-csala2020" role="doc-biblioref">286</a>]</span></td>
</tr>
<tr class="odd">
<td align="center">MOTA</td>
<td align="center"><span class="citation">[<a href="#ref-fan2020" role="doc-biblioref">287</a>]</span></td>
</tr>
<tr class="even">
<td align="center">D-CCA</td>
<td align="center"><span class="citation">[<a href="#ref-shu2020" role="doc-biblioref">288</a>]</span></td>
</tr>
<tr class="odd">
<td align="center">COMBI</td>
<td align="center"><span class="citation">[<a href="#ref-hawinkel2020" role="doc-biblioref">289</a>]</span></td>
</tr>
<tr class="even">
<td align="center">DPCCA</td>
<td align="center"><span class="citation">[<a href="#ref-gundersen2020" role="doc-biblioref">290</a>]</span></td>
</tr>
<tr class="odd">
<td align="center">MEFISTO</td>
<td align="center"><span class="citation">[<a href="#ref-velten2020" role="doc-biblioref">291</a>]</span></td>
</tr>
<tr class="even">
<td align="center">MultiPower</td>
<td align="center"><span class="citation">[<a href="#ref-tarazona2020" role="doc-biblioref">107</a>]</span></td>
</tr>
</tbody>
</table>
<!-- #Check https://github.com/mikelove/awesome-multi-omics -->
<ul>
<li><p><a href="https://github.com/rstudio/bookdown/">Bookdown</a>: The book about how to write this type of books.</p></li>
<li><p><a href="https://bioconducto.org" title="Bioconductor">Bioconductor</a>: The project about bioinformatics on R mostly related to sequencing technologies.</p></li>
<li><p><a href="https://cran.r-project.org" title="CRAN">CRAN</a>: The main archive of R extensions/packages for R.</p></li>
<li><p><a href="https://github.com" title="GitHub">GitHub</a>: Company which allows to freely host remote <a href="https://git-scm.com/" title="git">git</a> repositories of many projects, including some used or developed on this thesis.</p></li>
</ul>
</div>
<h3>References</h3>
<div id="refs" class="references csl-bib-body">
<div id="ref-tarazona2020" class="csl-entry">
107. Tarazona S, Balzano-Nogueira L, Gómez-Cabrero D, Schmidt A, Imhof A, Hankemeier T, et al. <a href="https://doi.org/10.1038/s41467-020-16937-8">Harmonization of quality metrics and power calculation in multi-omic studies</a>. Nature Communications. 2020;11:3092.
</div>
<div id="ref-tenenhaus_regularized_2011" class="csl-entry">
114. Tenenhaus A, Tenenhaus M. <a href="https://doi.org/10.1007/s11336-011-9206-8">Regularized <span>Generalized Canonical Correlation Analysis</span></a>. Psychometrika. 2011;76:257–84.
</div>
<div id="ref-zhu2012" class="csl-entry">
127. Zhu J, Sova P, Xu Q, Dombek KM, Xu EY, Vu H, et al. <a href="https://doi.org/10.1371/journal.pbio.1001301">Stitching together Multiple Data Dimensions Reveals Interacting Metabolomic and Transcriptomic Networks That Modulate Cell Regulation</a>. PLOS Biology. 2012;10:e1001301.
</div>
<div id="ref-virtanen2012" class="csl-entry">
133. Virtanen S, Klami A, Khan S, Kaski S. <a href="https://proceedings.mlr.press/v22/virtanen12.html">Bayesian Group Factor Analysis</a>. PMLR; 2012. p. 1269–77.
</div>
<div id="ref-lock2013a" class="csl-entry">
135. Lock EF, Hoadley KA, Marron JS, Nobel AB. <a href="https://doi.org/10.1214/12-AOAS597">JOINT AND INDIVIDUAL VARIATION EXPLAINED (JIVE) FOR INTEGRATED ANALYSIS OF MULTIPLE DATA TYPES</a>. The annals of applied statistics. 2013;7:523–42.
</div>
<div id="ref-meng2014" class="csl-entry">
173. Meng C, Kuster B, Culhane AC, Gholami AM. <a href="https://doi.org/10.1186/1471-2105-15-162">A multivariate approach to the integration of multi-omics datasets</a>. BMC Bioinformatics. 2014;15:162.
</div>
<div id="ref-planell2021" class="csl-entry">
174. Planell N, Lagani V, Sebastian-Leon P, van der Kloet F, Ewing E, Karathanasis N, et al. <a href="https://doi.org/10.3389/fgene.2021.620453">STATegra: Multi-omics data integration <span></span> a conceptual scheme with a bioinformatics pipeline</a>. Frontiers in Genetics. 2021;12.
</div>
<div id="ref-parkhomenko2009" class="csl-entry">
243. Parkhomenko E, Tritchler D, Beyene J. <a href="https://doi.org/10.2202/1544-6115.1406">Sparse Canonical Correlation Analysis with Application to Genomic Data Integration</a>. Statistical Applications in Genetics and Molecular Biology. 2009;8.
</div>
<div id="ref-waaijenborg2008" class="csl-entry">
244. Waaijenborg S, Hamer PCV de W, Zwinderman AH. <a href="https://doi.org/10.2202/1544-6115.1329">Quantifying the Association between Gene Expressions and DNA-Markers by Penalized Canonical Correlation Analysis</a>. Statistical Applications in Genetics and Molecular Biology. 2008;7.
</div>
<div id="ref-witten2009" class="csl-entry">
245. Witten DM, Tibshirani RJ. <a href="https://doi.org/10.2202/1544-6115.1470">Extensions of sparse canonical correlation analysis with applications to genomic data</a>. Statistical Applications in Genetics and Molecular Biology. 2009;8:127.
</div>
<div id="ref-lêcao2009" class="csl-entry">
246. Lê Cao K-A, Martin PG, Robert-Granié C, Besse P. <a href="https://doi.org/10.1186/1471-2105-10-34">Sparse canonical methods for biological data integration: Application to a cross-platform study</a>. BMC Bioinformatics. 2009;10:34.
</div>
<div id="ref-hwang2009" class="csl-entry">
247. Hwang H. <a href="https://doi.org/10.1007/s11336-009-9119-y">Regularized Generalized Structured Component Analysis</a>. Psychometrika. 2009;74:517–30.
</div>
<div id="ref-soneson2010" class="csl-entry">
248. Soneson C, Lilljebjörn H, Fioretos T, Fontes M. <a href="https://doi.org/10.1186/1471-2105-11-191">Integrative analysis of gene expression and copy number alterations using canonical correlation analysis</a>. BMC Bioinformatics. 2010;11:191.
</div>
<div id="ref-zhang2011" class="csl-entry">
249. Zhang S, Li Q, Liu J, Zhou XJ. <a href="https://doi.org/10.1093/bioinformatics/btr206">A novel computational framework for simultaneous integration of multiple types of genomic data to identify microRNA-gene regulatory modules</a>. Bioinformatics. 2011;27:i401–9.
</div>
<div id="ref-lee2011" class="csl-entry">
250. Lee W, Lee D, Lee Y, Pawitan Y. <a href="https://doi.org/10.2202/1544-6115.1638">Sparse Canonical Covariance Analysis for High-throughput Data</a>. Statistical Applications in Genetics and Molecular Biology. 2011;10.
</div>
<div id="ref-abdi2012" class="csl-entry">
251. Abdi H, Williams LJ, Valentin D, Bennani-Dosse M. <a href="https://doi.org/10.1002/wics.198">STATIS and DISTATIS: optimum multitable principal component analysis and three way metric multidimensional scaling</a>. WIREs Computational Statistics. 2012;4:124–67.
</div>
<div id="ref-zhang2012" class="csl-entry">
252. Zhang S, Liu C-C, Li W, Shen H, Laird PW, Zhou XJ. <a href="https://doi.org/10.1093/nar/gks725">Discovery of multi-dimensional modules by integrative analysis of cancer genomic data</a>. Nucleic Acids Research. 2012;40:9379–91.
</div>
<div id="ref-li2012" class="csl-entry">
253. Li W, Zhang S, Liu C-C, Zhou XJ. <a href="https://doi.org/10.1093/bioinformatics/bts476">Identifying multi-layer gene regulatory modules from multi-dimensional genomic data</a>. Bioinformatics. 2012;28:2458–66.
</div>
<div id="ref-abdi2013" class="csl-entry">
254. Abdi H, Williams LJ, Valentin D. <a href="https://doi.org/10.1002/wics.1246">Multiple factor analysis: principal component analysis for multitable and multiblock data sets</a>. WIREs Computational Statistics. 2013;5:149–79.
</div>
<div id="ref-schlauch2017" class="csl-entry">
255. Schlauch D, Paulson JN, Young A, Glass K, Quackenbush J. <a href="https://doi.org/10.1093/bioinformatics/btx139">Estimating gene regulatory networks with pandaR</a>. Bioinformatics. 2017;33:2232–4.
</div>
<div id="ref-ray2014" class="csl-entry">
256. Ray P, Zheng L, Lucas J, Carin L. <a href="https://doi.org/10.1093/bioinformatics/btu064">Bayesian joint analysis of heterogeneous genomics data</a>. Bioinformatics. 2014;30:1370–6.
</div>
<div id="ref-bunte2016" class="csl-entry">
257. Bunte K, Leppäaho E, Saarinen I, Kaski S. <a href="https://doi.org/10.1093/bioinformatics/btw207">Sparse group factor analysis for biclustering of multiple data sources</a>. Bioinformatics. 2016;32:2457–63.
</div>
<div id="ref-chen2013" class="csl-entry">
258. Chen M, Gao C, Ren Z, Zhou HH. <a href="http://arxiv.org/abs/1311.6186">Sparse CCA via precision adjusted iterative thresholding</a>. arXiv:13116186 [math, stat]. 2013.
</div>
<div id="ref-leppäaho2017" class="csl-entry">
259. Leppäaho E, Ammad-ud-din M, Kaski S. <a href="http://jmlr.org/papers/v18/16-509.html">GFA: Exploratory analysis of multiple data sources with group factor analysis</a>. Journal of Machine Learning Research. 2017;18:1–5.
</div>
<div id="ref-klami2014" class="csl-entry">
260. Klami A, Bouchard G, Tripathi A. <a href="http://arxiv.org/abs/1312.5921">Group-sparse embeddings in collective matrix factorization</a>. arXiv:13125921 [cs, stat]. 2014.
</div>
<div id="ref-meng2018" class="csl-entry">
261. Meng C, Basunia A, Peters B, Gholami AM, Kuster B, Culhane AC. <a href="https://www.biorxiv.org/content/10.1101/046904v2">MOGSA: integrative single sample gene-set analysis of multiple omics data</a>. 2018.
</div>
<div id="ref-yang2016b" class="csl-entry">
262. Yang Z, Michailidis G. <a href="https://doi.org/10.1093/bioinformatics/btv544">A non-negative matrix factorization method for detecting modules in heterogeneous omics multi-modal data</a>. Bioinformatics. 2016;32:1–8.
</div>
<div id="ref-zhao2015" class="csl-entry">
263. Zhao S, Gao C, Mukherjee S, Engelhardt BE. <a href="http://arxiv.org/abs/1411.2698">Bayesian group latent factor analysis with structured sparsity</a>. arXiv:14112698 [q-bio, stat]. 2015.
</div>
<div id="ref-voillet2016" class="csl-entry">
264. Voillet V, Besse P, Liaubet L, San Cristobal M, González I. <a href="https://doi.org/10.1186/s12859-016-1273-5">Handling missing rows in multi-omics data integration: Multiple imputation in multiple factor analysis framework</a>. BMC Bioinformatics. 2016;17:402.
</div>
<div id="ref-beaton2016" class="csl-entry">
265. Beaton D, Dunlop J, Abdi H, Alzheimer’s Disease Neuroimaging Initiative. <a href="https://doi.org/10.1037/met0000053">Partial least squares correspondence analysis: A framework to simultaneously analyze behavioral and genetic data</a>. Psychological Methods. 2016;21:621–51.
</div>
<div id="ref-singh2019" class="csl-entry">
266. Singh A, Shannon CP, Gautier B, Rohart F, Vacher M, Tebbutt SJ, et al. <a href="https://doi.org/10.1093/bioinformatics/bty1054">DIABLO: An integrative approach for identifying key molecular drivers from multi-omics assays</a>. Bioinformatics. 2019;35:3055–62.
</div>
<div id="ref-yoon2019" class="csl-entry">
267. Yoon G, Carroll RJ, Gaynanova I. <a href="http://arxiv.org/abs/1807.05274">Sparse semiparametric canonical correlation analysis for data of mixed types</a>. arXiv:180705274 [stat]. 2019.
</div>
<div id="ref-gaynanova2017" class="csl-entry">
268. Gaynanova I, Li G. <a href="http://arxiv.org/abs/1707.06573">Structural learning and integrative decomposition of multi-view data</a>. arXiv:170706573 [stat]. 2017.
</div>
<div id="ref-madrigal2017" class="csl-entry">
269. Madrigal P. <a href="https://doi.org/10.1093/bioinformatics/btw724">fCCAC: Functional canonical correlation analysis to evaluate covariance between nucleic acid sequencing datasets</a>. Bioinformatics. 2017;33:746–8.
</div>
<div id="ref-yoshida2017" class="csl-entry">
270. Yoshida K, Yoshimoto J, Doya K. <a href="https://doi.org/10.1186/s12859-017-1543-x">Sparse kernel canonical correlation analysis for discovery of nonlinear interactions in high-dimensional data</a>. BMC Bioinformatics. 2017;18:108.
</div>
<div id="ref-kawaguchi2017" class="csl-entry">
271. Kawaguchi A, Yamashita F. <a href="https://doi.org/10.1093/biostatistics/kxx011">Supervised multiblock sparse multivariable analysis with application to multimodal brain imaging genetics</a>. Biostatistics. 2017;18:651–65.
</div>
<div id="ref-feng2018" class="csl-entry">
272. Feng Q, Jiang M, Hannig J, Marron JS. <a href="http://arxiv.org/abs/1704.02060">Angle-based joint and individual variation explained</a>. arXiv:170402060 [stat]. 2018.
</div>
<div id="ref-argelaguet2020" class="csl-entry">
273. Argelaguet R, Arnol D, Bredikhin D, Deloro Y, Velten B, Marioni JC, et al. <a href="https://doi.org/10.1186/s13059-020-02015-1">MOFA+: A statistical framework for comprehensive integration of multi-modal single-cell data</a>. Genome Biology. 2020;21:111.
</div>
<div id="ref-brown2018" class="csl-entry">
274. Brown BC, Bray NL, Pachter L. Expression reflects population structure. 2018. <a href="https://doi.org/10.1101/364448">https://doi.org/10.1101/364448</a>.
</div>
<div id="ref-zhang2020" class="csl-entry">
275. Zhang Y, Gaynanova I. <a href="http://arxiv.org/abs/1811.08511">Joint association and classification analysis of multi-view data</a>. arXiv:181108511 [cs, stat]. 2020.
</div>
<div id="ref-tang2021" class="csl-entry">
276. Tang TM, Allen GI. <a href="http://arxiv.org/abs/1810.00832">Integrated principal components analysis</a>. arXiv:181000832 [stat]. 2021.
</div>
<div id="ref-min2019" class="csl-entry">
277. Min EJ, Safo SE, Long Q. <a href="https://doi.org/10.1093/bioinformatics/bty726">Penalized co-inertia analysis with applications to -omics data</a>. Bioinformatics (Oxford, England). 2019;35:1018–25.
</div>
<div id="ref-safo2018" class="csl-entry">
278. Safo SE, Li S, Long Q. <a href="https://doi.org/10.1111/biom.12715">Integrative analysis of transcriptomic and metabolomic data via sparse canonical correlation analysis with incorporation of biological information</a>. Biometrics. 2018;74:300–12.
</div>
<div id="ref-min2017" class="csl-entry">
279. Min W, Liu J, Zhang S. <a href="http://arxiv.org/abs/1710.04792">Sparse weighted canonical correlation analysis</a>. arXiv:171004792 [cs, stat]. 2017.
</div>
<div id="ref-bouhaddani2018" class="csl-entry">
280. Bouhaddani S el, Uh H-W, Jongbloed G, Hayward C, Klarić L, Kiełbasa SM, et al. <a href="https://doi.org/10.1186/s12859-018-2371-3">Integrating omics datasets with the OmicsPLS package</a>. BMC Bioinformatics. 2018;19:371.
</div>
<div id="ref-pimentel2018" class="csl-entry">
281. Pimentel H, Zhiyue H, Huang H. <a href="https://doi.org/10.1007/s40484-017-0127-0">Biclustering by sparse canonical correlation analysis</a>. 2018;6:11.
</div>
<div id="ref-kim2019" class="csl-entry">
282. Kim Y, Bismeijer T, Zwart W, Wessels LFA, Vis DJ. <a href="https://doi.org/10.1038/s41467-019-13027-2">Genomic data integration by WON-PARAFAC identifies interpretable factors for predicting drug-sensitivity in vivo</a>. Nature Communications. 2019;10:5034.
</div>
<div id="ref-lock2020" class="csl-entry">
283. Lock EF, Park JY, Hoadley KA. <a href="http://arxiv.org/abs/2002.02601">Bidimensional linked matrix factorization for pan-omics pan-cancer analysis</a>. arXiv:200202601 [cs, q-bio, stat]. 2020.
</div>
<div id="ref-ronen2019" class="csl-entry">
284. Ronen J, Hayat S, Akalin A. <a href="https://doi.org/10.26508/lsa.201900517">Evaluation of colorectal cancer subtypes and cell lines using deep learning</a>. Life Science Alliance. 2019;2.
</div>
<div id="ref-shi2019" class="csl-entry">
285. Shi WJ, Zhuang Y, Russell PH, Hobbs BD, Parker MM, Castaldi PJ, et al. <a href="https://doi.org/10.1093/bioinformatics/btz226">Unsupervised discovery of phenotype-specific multi-omics networks</a>. Bioinformatics. 2019;35:4336–43.
</div>
<div id="ref-csala2020" class="csl-entry">
286. Csala A, Zwinderman AH, Hof MH. <a href="https://doi.org/10.1186/s12859-019-3286-3">Multiset sparse partial least squares path modeling for high dimensional omics data analysis</a>. BMC Bioinformatics. 2020;21:9.
</div>
<div id="ref-fan2020" class="csl-entry">
287. Fan Z, Zhou Y, Ressom HW. <a href="https://doi.org/10.3390/metabo10040144">MOTA: Network-Based Multi-Omic Data Integration for Biomarker Discovery</a>. Metabolites. 2020;10:144.
</div>
<div id="ref-shu2020" class="csl-entry">
288. Shu H, Wang X, Zhu H. <a href="https://doi.org/10.1080/01621459.2018.1543599">D-CCA: A decomposition-based canonical correlation analysis for high-dimensional datasets</a>. Journal of the American Statistical Association. 2020;115:292–306.
</div>
<div id="ref-hawinkel2020" class="csl-entry">
289. Hawinkel S, Bijnens L, Cao K-AL, Thas O. <a href="https://doi.org/10.1093/nargab/lqaa050">Model-based joint visualization of multiple compositional omics datasets</a>. NAR Genomics and Bioinformatics. 2020;2.
</div>
<div id="ref-gundersen2020" class="csl-entry">
290. Gundersen G, Dumitrascu B, Ash JT, Engelhardt BE. <a href="https://proceedings.mlr.press/v115/gundersen20a.html">Uncertainty in Artificial Intelligence</a>. PMLR; 2020. p. 945–55.
</div>
<div id="ref-velten2020" class="csl-entry">
291. Velten B, Braunger JM, Arnol D, Argelaguet R, Stegle O. <a href="https://doi.org/10.1101/2020.11.03.366674">Identifying temporal and spatial patterns of variation from multi-modal data using MEFISTO</a>. bioRxiv. 2020;2020.11.03.366674.
</div>
</div>
<div class="footnotes">
<hr />
<ol start="8">
<li id="fn8"><p>Consulted on 2021/11/10 from <a href="https://github.com/mikelove/awesome-multi-omics" class="uri">https://github.com/mikelove/awesome-multi-omics</a><a href="online-resources.html#fnref8" class="footnote-back">↩︎</a></p></li>
</ol>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="references.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="software.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": {
"href": "https://github.com/llrs/thesis/"
},
"facebook": false,
"twitter": {
"href": "https://twitter.com/intent/tweet?text=Nice%20read%20of%20%40Lluis_Revilla%20thesis"
},
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/llrs/thesis/edit/master/98_appendix.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": "https://github.com/llrs/thesis/blob/master/98_appendix.Rmd",
"text": "View"
},
"download": ["IBD_integration_Lluis_Revilla.pdf", "IBD_integration_Lluis_Revilla.epub"],
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": false,
"toc_float": true,
"toc_appendix": true,
"toc_depth": 2,
"scroll_highlight": true
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
