<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>References | Data integration on inflammatory bowel disease</title>
  <meta name="description" content="Thesis about the relationship between microbiome and transcriptome on the gastrointestinal tract on patients with inflammatory bowel disease and some controls to discover key differences on this relationship." />
  <meta name="generator" content="bookdown 0.24 and GitBook 2.6.7" />

  <meta property="og:title" content="References | Data integration on inflammatory bowel disease" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="Thesis about the relationship between microbiome and transcriptome on the gastrointestinal tract on patients with inflammatory bowel disease and some controls to discover key differences on this relationship." />
  <meta name="github-repo" content="llrs/thesis" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="References | Data integration on inflammatory bowel disease" />
  <meta name="twitter:site" content="@Lluis_Revilla" />
  <meta name="twitter:description" content="Thesis about the relationship between microbiome and transcriptome on the gastrointestinal tract on patients with inflammatory bowel disease and some controls to discover key differences on this relationship." />
  

<meta name="author" content="Lluís Revilla Sancho" />


<meta name="date" content="2021-09-10" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="acknowledgements.html"/>
<link rel="next" href="online-resources.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<script src="libs/accessible-code-block-0.0.1/empty-anchor.js"></script>
<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>




<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><strong><a href="./">Data integration on IBD</a></strong></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Preface</a></li>
<li class="chapter" data-level="1" data-path="introduction.html"><a href="introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a><ul>
<li class="chapter" data-level="1.1" data-path="introduction.html"><a href="introduction.html#IBD"><i class="fa fa-check"></i><b>1.1</b> Inflammatory bowel disease</a><ul>
<li class="chapter" data-level="1.1.1" data-path="introduction.html"><a href="introduction.html#etiology-and-pathogenesis"><i class="fa fa-check"></i><b>1.1.1</b> Etiology and pathogenesis</a></li>
<li class="chapter" data-level="1.1.2" data-path="introduction.html"><a href="introduction.html#crohns-disease-physiology"><i class="fa fa-check"></i><b>1.1.2</b> Crohn’s disease physiology</a></li>
<li class="chapter" data-level="1.1.3" data-path="introduction.html"><a href="introduction.html#ulcerative-colitis-physiology"><i class="fa fa-check"></i><b>1.1.3</b> Ulcerative colitis physiology</a></li>
<li class="chapter" data-level="1.1.4" data-path="introduction.html"><a href="introduction.html#clinical-care"><i class="fa fa-check"></i><b>1.1.4</b> Clinical care</a></li>
<li class="chapter" data-level="1.1.5" data-path="introduction.html"><a href="introduction.html#summary"><i class="fa fa-check"></i><b>1.1.5</b> Summary</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="introduction.html"><a href="introduction.html#studies-and-integration-on-ibd"><i class="fa fa-check"></i><b>1.2</b> Studies and integration on IBD</a><ul>
<li class="chapter" data-level="1.2.1" data-path="introduction.html"><a href="introduction.html#transcriptome"><i class="fa fa-check"></i><b>1.2.1</b> Transcriptome</a></li>
<li class="chapter" data-level="1.2.2" data-path="introduction.html"><a href="introduction.html#microbiome-1"><i class="fa fa-check"></i><b>1.2.2</b> Microbiome</a></li>
<li class="chapter" data-level="1.2.3" data-path="introduction.html"><a href="introduction.html#genetics-1"><i class="fa fa-check"></i><b>1.2.3</b> Genetics</a></li>
<li class="chapter" data-level="1.2.4" data-path="introduction.html"><a href="introduction.html#metabolome"><i class="fa fa-check"></i><b>1.2.4</b> Metabolome</a></li>
<li class="chapter" data-level="1.2.5" data-path="introduction.html"><a href="introduction.html#summary-1"><i class="fa fa-check"></i><b>1.2.5</b> Summary</a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="introduction.html"><a href="introduction.html#integration"><i class="fa fa-check"></i><b>1.3</b> Integration</a><ul>
<li class="chapter" data-level="1.3.1" data-path="introduction.html"><a href="introduction.html#classification-of-integration-methods"><i class="fa fa-check"></i><b>1.3.1</b> Classification of integration method’s</a></li>
<li class="chapter" data-level="1.3.2" data-path="introduction.html"><a href="introduction.html#interpretation"><i class="fa fa-check"></i><b>1.3.2</b> Interpretation</a></li>
<li class="chapter" data-level="1.3.3" data-path="introduction.html"><a href="introduction.html#reviews-and-benchmarking"><i class="fa fa-check"></i><b>1.3.3</b> Reviews and benchmarking</a></li>
<li class="chapter" data-level="1.3.4" data-path="introduction.html"><a href="introduction.html#summary-2"><i class="fa fa-check"></i><b>1.3.4</b> Summary</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="2" data-path="hypothesis-and-objectives.html"><a href="hypothesis-and-objectives.html"><i class="fa fa-check"></i><b>2</b> Hypothesis and objectives</a><ul>
<li class="chapter" data-level="2.1" data-path="hypothesis-and-objectives.html"><a href="hypothesis-and-objectives.html#hypothesis"><i class="fa fa-check"></i><b>2.1</b> Hypothesis</a></li>
<li class="chapter" data-level="2.2" data-path="hypothesis-and-objectives.html"><a href="hypothesis-and-objectives.html#objectives"><i class="fa fa-check"></i><b>2.2</b> Objectives</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="materials-and-methods.html"><a href="materials-and-methods.html"><i class="fa fa-check"></i><b>3</b> Materials and methods</a><ul>
<li class="chapter" data-level="3.1" data-path="materials-and-methods.html"><a href="materials-and-methods.html#sample-collection"><i class="fa fa-check"></i><b>3.1</b> Sample collection</a><ul>
<li class="chapter" data-level="3.1.1" data-path="materials-and-methods.html"><a href="materials-and-methods.html#hsct-dataset"><i class="fa fa-check"></i><b>3.1.1</b> HSCT dataset</a></li>
</ul></li>
<li class="chapter" data-level="3.2" data-path="materials-and-methods.html"><a href="materials-and-methods.html#statistics"><i class="fa fa-check"></i><b>3.2</b> Statistics</a><ul>
<li class="chapter" data-level="3.2.1" data-path="materials-and-methods.html"><a href="materials-and-methods.html#functional-enrichment"><i class="fa fa-check"></i><b>3.2.1</b> Functional enrichment</a></li>
<li class="chapter" data-level="3.2.2" data-path="materials-and-methods.html"><a href="materials-and-methods.html#other-methods-used"><i class="fa fa-check"></i><b>3.2.2</b> Other methods used</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="materials-and-methods.html"><a href="materials-and-methods.html#multi-omic-modelling-of-inflammatory-bowel-disease-with-regularized-canonical-correlation-analysis"><i class="fa fa-check"></i><b>3.3</b> Multi-omic modelling of inflammatory bowel disease with regularized canonical correlation analysis</a><ul>
<li class="chapter" data-level="3.3.1" data-path="materials-and-methods.html"><a href="materials-and-methods.html#description"><i class="fa fa-check"></i><b>3.3.1</b> Description</a></li>
<li class="chapter" data-level="3.3.2" data-path="materials-and-methods.html"><a href="materials-and-methods.html#relationships-models"><i class="fa fa-check"></i><b>3.3.2</b> Relationships’ models</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="biological-relevance-of-results.html"><a href="biological-relevance-of-results.html"><i class="fa fa-check"></i><b>4</b> Biological relevance of results</a><ul>
<li class="chapter" data-level="4.1" data-path="biological-relevance-of-results.html"><a href="biological-relevance-of-results.html#comparing-different-dataset"><i class="fa fa-check"></i><b>4.1</b> Comparing different dataset</a></li>
<li class="chapter" data-level="4.2" data-path="biological-relevance-of-results.html"><a href="biological-relevance-of-results.html#shared-selected-variables"><i class="fa fa-check"></i><b>4.2</b> Shared selected variables</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="discussion.html"><a href="discussion.html"><i class="fa fa-check"></i><b>5</b> Discussion</a></li>
<li class="chapter" data-level="6" data-path="conclusions.html"><a href="conclusions.html"><i class="fa fa-check"></i><b>6</b> Conclusions</a></li>
<li class="chapter" data-level="7" data-path="acknowledgements.html"><a href="acknowledgements.html"><i class="fa fa-check"></i><b>7</b> Acknowledgements</a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="appendix"><span><b>Appendix</b></span></li>
<li class="chapter" data-level="A" data-path="online-resources.html"><a href="online-resources.html"><i class="fa fa-check"></i><b>A</b> Online resources</a></li>
<li class="chapter" data-level="B" data-path="software.html"><a href="software.html"><i class="fa fa-check"></i><b>B</b> Software</a><ul>
<li class="chapter" data-level="B.1" data-path="software.html"><a href="software.html#listed"><i class="fa fa-check"></i><b>B.1</b> Listed</a></li>
<li class="chapter" data-level="B.2" data-path="software.html"><a href="software.html#by-publication"><i class="fa fa-check"></i><b>B.2</b> By publication</a></li>
</ul></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Data integration on inflammatory bowel disease</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div class = "wip">
This tesis is ongoing and still without hard date to finish it. Watch the <a href="https://github.com/llrs/thesis">GitHub repository</a> or wait for a release announcement on Twitter on <a href="https://twitter.com/Lluis_Rev">my profile</a>.
</div>
<div id="references" class="section level1 unnumbered">
<h1>References</h1>
<div id="refs">
<div id="ref-bassolasmolina2018">
<p>1. Bassolas Molina H. Resposta T específica contra antígens de la microbiota comensal en la malaltia de Crohn i inhibició de ROR?t com a estratègia terapèutica. PhD thesis. 2018. <a href="http://diposit.ub.edu/dspace/handle/2445/131017">http://diposit.ub.edu/dspace/handle/2445/131017</a>.</p>
</div>
<div id="ref-sands2015">
<p>2. Sands B. Biomarkers of Inflammation in Inflammatory Bowel Disease. Gastroenterology. 2015;149:1275–1285.e2.</p>
</div>
<div id="ref-jairath2020">
<p>3. Jairath V, Feagan B. Global burden of inflammatory bowel disease. The Lancet Gastroenterology &amp; Hepatology. 2020;5:2–3. doi:<a href="https://doi.org/10.1016/S2468-1253(19)30358-9">10.1016/S2468-1253(19)30358-9</a>.</p>
</div>
<div id="ref-burisch2015">
<p>4. Burisch J, Munkholm P. The epidemiology of inflammatory bowel disease. Scandinavian Journal of Gastroenterology. 2015;50:942–51. doi:<a href="https://doi.org/10.3109/00365521.2015.1014407">10.3109/00365521.2015.1014407</a>.</p>
</div>
<div id="ref-human_microbiome_project_consortium_framework_2012">
<p>5. Human Microbiome Project Consortium BA, Nelson KE, Pop M, Creasy HH, Giglio MG, Huttenhower C, et al. A framework for human microbiome research. Nature. 2012;486:215–21.</p>
</div>
<div id="ref-zhang2014">
<p>6. Zhang Y, Li Y. Inflammatory bowel disease: pathogenesis. World Journal of Gastroenterology. 2014;20:91–9.</p>
</div>
<div id="ref-hugot2004">
<p>7. Hugot J. Genetic origin of ibd. Inflammatory Bowel Diseases. 2004;10 suppl_1:S11–5. doi:<a href="https://doi.org/10.1097/00054725-200402001-00003">10.1097/00054725-200402001-00003</a>.</p>
</div>
<div id="ref-corralizaDifferencesPeripheralTissue">
<p>8. Corraliza AM, Ricart E, L’opez-García A, Carme Masamunt M, Veny M, Esteller M, et al. Differences in Peripheral and Tissue Immune Cell Populations Following Haematopoietic Stem Cell Transplantation in Crohn’s Disease Patients. Journal of Crohn’s and Colitis.</p>
</div>
<div id="ref-mcgovern2015b">
<p>9. McGovern D, Kugathasan S, Cho J. Genetics of Inflammatory Bowel Diseases. Gastroenterology. 2015;149:1163–1176.e2.</p>
</div>
<div id="ref-satsangi2006b">
<p>10. Satsangi J, Silverberg M, Vermeire S, Colombel J. The montreal classification of inflammatory bowel disease: Controversies, consensus, and implications. Gut. 2006;55:749–53. doi:<a href="https://doi.org/10.1136/gut.2005.082909">10.1136/gut.2005.082909</a>.</p>
</div>
<div id="ref-horowitz2021">
<p>11. Horowitz J, Warner N, Staples J, Crowley E, Gosalia N, Murchie R, et al. Mutation spectrum of nod2 reveals recessive inheritance as a main driver of early onset crohn’s disease. Scientific Reports. 2021;11:5595. doi:<a href="https://doi.org/10.1038/s41598-021-84938-8">10.1038/s41598-021-84938-8</a>.</p>
</div>
<div id="ref-kumar2019">
<p>12. Kumar M, Garand M, Al Khodor S. Integrating omics for a better understanding of inflammatory bowel disease: A step towards personalized medicine. Journal of Translational Medicine. 2019;17:419. doi:<a href="https://doi.org/10.1186/s12967-019-02174-1">10.1186/s12967-019-02174-1</a>.</p>
</div>
<div id="ref-jostins2012">
<p>13. Jostins L, Ripke S, Weersma R, Duerr R, McGovern D, Hui K, et al. Hostmicrobe interactions have shaped the genetic architecture of inflammatory bowel disease. Nature. 2012;491:119–24. doi:<a href="https://doi.org/10.1038/nature11582">10.1038/nature11582</a>.</p>
</div>
<div id="ref-momozawa2018">
<p>14. Momozawa Y, Dmitrieva J, Théâtre E, Deffontaine V, Rahmouni S, Charloteaux B, et al. IBD risk loci are enriched in multigenic regulatory modules encompassing putative causative genes. Nature Communications. 2018;9. doi:<a href="https://doi.org/10.1038/s41467-018-04365-8">10.1038/s41467-018-04365-8</a>.</p>
</div>
<div id="ref-mcgovern2015">
<p>15. McGovern D, Kugathasan S, Cho J. Genetics of Inflammatory Bowel Diseases. Gastroenterology. 2015;149:1163–1176.e2.</p>
</div>
<div id="ref-khanna2014">
<p>16. Khanna S, Tosh P. A clinician’s primer on the role of the microbiome in human health and disease. Mayo Clinic Proceedings. 2014;89:107–14. doi:<a href="https://doi.org/10.1016/j.mayocp.2013.10.011">10.1016/j.mayocp.2013.10.011</a>.</p>
</div>
<div id="ref-swidsinski2002">
<p>17. Swidsinski A, Ladhoff A, Pernthaler A, Swidsinski S, LoeningBaucke V, Ortner M, et al. Mucosal flora in inflammatory bowel disease. Gastroenterology. 2002;122:44–54. doi:<a href="https://doi.org/10.1053/gast.2002.30294">10.1053/gast.2002.30294</a>.</p>
</div>
<div id="ref-tamboli2004">
<p>18. Tamboli C, Neut C, Desreumaux P, Colombel J. Dysbiosis in inflammatory bowel disease. Gut. 2004;53:1–4. doi:<a href="https://doi.org/10.1136/gut.53.1.1">10.1136/gut.53.1.1</a>.</p>
</div>
<div id="ref-ott2004">
<p>19. Ott S. Reduction in diversity of the colonic mucosa associated bacterial microflora in patients with active inflammatory bowel disease. Gut. 2004;53:685–93. doi:<a href="https://doi.org/10.1136/gut.2003.025403">10.1136/gut.2003.025403</a>.</p>
</div>
<div id="ref-kostic2014">
<p>20. Kostic A, Xavier R, Gevers D. The microbiome in inflammatory bowel disease: Current status and the future ahead. Gastroenterology. 2014;146:1489–99. doi:<a href="https://doi.org/10.1053/j.gastro.2014.02.009">10.1053/j.gastro.2014.02.009</a>.</p>
</div>
<div id="ref-sender2016">
<p>21. Sender R, Fuchs S, Milo R. Revised estimates for the number of human and bacteria cells in the body. PLOS Biology. 2016;14:e1002533. doi:<a href="https://doi.org/10.1371/journal.pbio.1002533">10.1371/journal.pbio.1002533</a>.</p>
</div>
<div id="ref-sankarasubramanian2020a">
<p>22. Sankarasubramanian J, Ahmad R, Avuthu N, Singh A, Guda C. Gut microbiota and metabolic specificity in ulcerative colitis and crohn’s disease. Frontiers in Medicine. 2020;7. doi:<a href="https://doi.org/10.3389/fmed.2020.606298">10.3389/fmed.2020.606298</a>.</p>
</div>
<div id="ref-lopez-siles2014">
<p>23. Lopez-Siles M, Martinez-Medina M, Busquets D, Sabat-Mir M, Duncan S, Flint H, et al. Mucosa-associated Faecalibacterium prausnitzii and Escherichia coli co-abundance can distinguish Irritable Bowel Syndrome and Inflammatory Bowel Disease phenotypes. International journal of medical microbiology: IJMM. 2014;304:464–75.</p>
</div>
<div id="ref-ferrer-picón2020">
<p>24. Ferrer-Picón E, Dotti I, Corraliza A, Mayorgas A, Esteller M, Perales J, et al. Intestinal inflammation modulates the epithelial response to butyrate in patients with inflammatory bowel disease. Inflammatory Bowel Diseases. 2020;26:43–55. doi:<a href="https://doi.org/10.1093/ibd/izz119">10.1093/ibd/izz119</a>.</p>
</div>
<div id="ref-okumura2017">
<p>25. Okumura R, Takeda K. Roles of intestinal epithelial cells in the maintenance of gut homeostasis. Experimental &amp; Molecular Medicine. 2017;49:e338–8. doi:<a href="https://doi.org/10.1038/emm.2017.20">10.1038/emm.2017.20</a>.</p>
</div>
<div id="ref-faria2012">
<p>26. Faria A, Mucida D, McCafferty D, Tsuji N, Verhasselt V. Tolerance and inflammation at the gut mucosa. Clinical &amp; Developmental Immunology. 2012;2012:738475.</p>
</div>
<div id="ref-silva2016a">
<p>27. Silva F, Rodrigues B, Ayrizono M, Leal R. The Immunological Basis of Inflammatory Bowel Disease. Gastroenterology Research and Practice. 2016;2016:2097274.</p>
</div>
<div id="ref-demattos2015">
<p>28. de Mattos B, Garcia M, Nogueira J, Paiatto L, Albuquerque C, Souza C, et al. Inflammatory Bowel Disease: An Overview of Immune Mechanisms and Biological Treatments. Mediators of Inflammation. 2015;2015:493012.</p>
</div>
<div id="ref-hisamatsu2013">
<p>29. Hisamatsu T, Kanai T, Mikami Y, Yoneno K, Matsuoka K, Hibi T. Immune aspects of the pathogenesis of inflammatory bowel disease. Pharmacology &amp; Therapeutics. 2013;137:283–97.</p>
</div>
<div id="ref-michielan2015">
<p>30. Michielan A, D’Incà R. Intestinal Permeability in Inflammatory Bowel Disease: Pathogenesis, Clinical Evaluation, and Therapy of Leaky Gut. Mediators of Inflammation. 2015;2015:628157.</p>
</div>
<div id="ref-neurath1996">
<p>31. Neurath M, Fuss I, Kelsall B, Presky D, Waegell W, Strober W. Experimental granulomatous colitis in mice is abrogated by induction of tgf-beta-mediated oral tolerance. Journal of Experimental Medicine. 1996;183:2605–16. doi:<a href="https://doi.org/10.1084/jem.183.6.2605">10.1084/jem.183.6.2605</a>.</p>
</div>
<div id="ref-corraliza">
<p>32. Corraliza A, Ricart E, López-García A, Carme Masamunt M, Veny M, Esteller M, et al. Differences in peripheral and tissue immune cell populations following haematopoietic stem cell transplantation in crohn’s disease patients. Journal of Crohn’s and Colitis. doi:<a href="https://doi.org/10.1093/ecco-jcc/jjy203">10.1093/ecco-jcc/jjy203</a>.</p>
</div>
<div id="ref-strachan1989">
<p>33. Strachan D. Hay fever, hygiene, and household size. BMJ : British Medical Journal. 1989;299:1259–60. <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1838109/">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1838109/</a>.</p>
</div>
<div id="ref-scudellari2017">
<p>34. Scudellari M. News feature: Cleaning up the hygiene hypothesis. Proceedings of the National Academy of Sciences of the United States of America. 2017;114:1433–6. doi:<a href="https://doi.org/10.1073/pnas.1700688114">10.1073/pnas.1700688114</a>.</p>
</div>
<div id="ref-thomas1998">
<p>35. Thomas G, Rhodes J, Green J. Inflammatory bowel disease and smokingA review. Official journal of the American College of Gastroenterology | ACG. 1998;93:144149. doi:<a href="https://doi.org/10.1111/j.1572-0241.1998.00144.x">10.1111/j.1572-0241.1998.00144.x</a>.</p>
</div>
<div id="ref-cornish2008">
<p>36. Cornish J, Tan E, Simillis C, Clark S, Teare J, Tekkis P. The risk of oral contraceptives in the etiology of inflammatory bowel disease: a meta-analysis. The American Journal of Gastroenterology. 2008;103:2394–400.</p>
</div>
<div id="ref-kaufmann1987">
<p>37. Kaufmann H, Taubin H. Nonsteroidal anti-inflammatory drugs activate quiescent inflammatory bowel disease. Annals of Internal Medicine. 1987;107:513–6.</p>
</div>
<div id="ref-bitton2008">
<p>38. Bitton A, Dobkin P, Edwardes M, Sewitch M, Meddings J, Rawal S, et al. Predicting relapse in Crohn’s disease: a biopsychosocial model. Gut. 2008;57:1386–92.</p>
</div>
<div id="ref-guardiola2018">
<p>39. Guardiola J, Lobatón T, Cerrillo E, Ferreiro-Iglesias R, Gisbert J, Domènech E, et al. Recommendations of the spanish working group on crohn’s disease and ulcerative colitis (geteccu) on the utility of the determination of faecal calprotectin in inflammatory bowel disease. Gastroenterología y Hepatología (English Edition). 2018;41:514–29. doi:<a href="https://doi.org/10.1016/j.gastre.2018.05.018">10.1016/j.gastre.2018.05.018</a>.</p>
</div>
<div id="ref-corralizamárquez2019">
<p>40. Corraliza Márquez A. Immune mechanisms involved in inducing remission in Crohn’s disease patients undergoing hematopoietic stem cell transplant. PhD thesis. 2019. <a href="http://diposit.ub.edu/dspace/handle/2445/143300">http://diposit.ub.edu/dspace/handle/2445/143300</a>.</p>
</div>
<div id="ref-satsangi2006a">
<p>41. Satsangi J, Silverberg M, Vermeire S, Colombel J. The Montreal classification of inflammatory bowel disease: controversies, consensus, and implications. Gut. 2006;55:749–53.</p>
</div>
<div id="ref-peyrin-biroulet2010">
<p>42. Peyrin-Biroulet L, Loftus E, Colombel J, Sandborn W. The natural history of adult crohn’s disease in population-based cohorts. Official journal of the American College of Gastroenterology | ACG. 2010;105:289297. doi:<a href="https://doi.org/10.1038/ajg.2009.579">10.1038/ajg.2009.579</a>.</p>
</div>
<div id="ref-bhattacharya2016">
<p>43. Bhattacharya A, Rao B, Koutroubakis I, Click B, Vargas E, Regueiro M, et al. Silent crohn’s disease predicts increased bowel damage during multiyear follow-up: The consequences of under-reporting active inflammation. Inflammatory Bowel Diseases. 2016;22:2665–71. doi:<a href="https://doi.org/10.1097/MIB.0000000000000935">10.1097/MIB.0000000000000935</a>.</p>
</div>
<div id="ref-etchevers2009">
<p>44. Etchevers M, Aceituno M, García-Bosch O, Ordás I, Sans M, Ricart E, et al. Risk factors and characteristics of extent progression in ulcerative colitis. Inflammatory Bowel Diseases. 2009;15:1320–5. doi:<a href="https://doi.org/10.1002/ibd.20897">10.1002/ibd.20897</a>.</p>
</div>
<div id="ref-schieffer2016">
<p>45. Schieffer K, Williams E, Yochum G, Koltun W. Review article: the pathogenesis of pouchitis. Alimentary Pharmacology &amp; Therapeutics. 2016;44:817–35.</p>
</div>
<div id="ref-mark-christensen2018">
<p>46. Mark-Christensen A, Laurberg S, Haboubi N. Dysplasia in inflammatory bowel disease: Historical review, critical histopathological analysis, and clinical implications. Inflammatory Bowel Diseases. 2018;24:1895–903. doi:<a href="https://doi.org/10.1093/ibd/izy075">10.1093/ibd/izy075</a>.</p>
</div>
<div id="ref-schroeder1987">
<p>47. Schroeder K, Tremaine W, Ilstrup D. Coated oral 5-aminosalicylic acid therapy for mildly to moderately active ulcerative colitis. New England Journal of Medicine. 1987;317:1625–9. doi:<a href="https://doi.org/10.1056/nejm198712243172603">10.1056/nejm198712243172603</a>.</p>
</div>
<div id="ref-daperno2004">
<p>48. Daperno M, D’Haens G, Van Assche G, Baert F, Bulois P, Maunoury V, et al. Development and validation of a new, simplified endoscopic activity score for crohn’s disease: The ses-cd. Gastrointestinal Endoscopy. 2004;60:505–12. doi:<a href="https://doi.org/10.1016/S0016-5107(04)01878-4">10.1016/S0016-5107(04)01878-4</a>.</p>
</div>
<div id="ref-irvine1999">
<p>49. Irvine E. Development and subsequent refinement of the inflammatory bowel disease questionnaire: A quality-of-life instrument for adult patients with inflammatory bowel disease. Journal of Pediatric Gastroenterology &amp; Nutrition. 1999;28 Supplement:S23–7. doi:<a href="https://doi.org/10.1097/00005176-199904001-00003">10.1097/00005176-199904001-00003</a>.</p>
</div>
<div id="ref-satsangi2006">
<p>50. Satsangi J, Silverberg M, Vermeire S, Colombel J. The Montreal classification of inflammatory bowel disease: controversies, consensus, and implications. Gut. 2006;55:749–53.</p>
</div>
<div id="ref-corralizab">
<p>51. Corraliza A, Ricart E, López-García A, Carme Masamunt M, Veny M, Esteller M, et al. Differences in peripheral and tissue immune cell populations following haematopoietic stem cell transplantation in crohn’s disease patients. Journal of Crohn’s and Colitis. doi:<a href="https://doi.org/10.1093/ecco-jcc/jjy203">10.1093/ecco-jcc/jjy203</a>.</p>
</div>
<div id="ref-beck2021">
<p>52. Beck L, Granger C, Masi A, Stewart C. Use of omic technologies in early life gastrointestinal health and disease: From bench to bedside. Expert Review of Proteomics. 2021;18:247–59. doi:<a href="https://doi.org/10.1080/14789450.2021.1922278">10.1080/14789450.2021.1922278</a>.</p>
</div>
<div id="ref-hasler_uncoupling_2016">
<p>53. Häsler R, Sheibani-Tezerji R, Sinha A, Barann M, Rehman A, Esser D, et al. Uncoupling of mucosal gene regulation, mRNA splicing and adherent microbiota signatures in inflammatory bowel disease. Gut. 2017;66:2087–97.</p>
</div>
<div id="ref-tang_integrated_2017">
<p>54. Tang MS, Bowcutt R, Leung JM, Wolff MJ, Gundra UM, Hudesman D, et al. Integrated Analysis of Biopsies from Inflammatory Bowel Disease Patients Identifies SAA1 as a Link Between Mucosal Microbes with TH17 and TH22 Cells. Inflammatory Bowel Diseases. 2017;23:1544–54.</p>
</div>
<div id="ref-hernández-rocha2021">
<p>55. Hernández-Rocha C, Borowski K, Turpin W, Filice M, Nayeri S, Raygoza Garay J, et al. Integrative analysis of colonic biopsies from inflammatory bowel disease patients identifies an interaction between microbial bile-acid inducible gene abundance and human angiopoietin-like 4 gene expression. Journal of Crohn’s and Colitis. 2021. doi:<a href="https://doi.org/10.1093/ecco-jcc/jjab096">10.1093/ecco-jcc/jjab096</a>.</p>
</div>
<div id="ref-huWholeExomeSequencing2021">
<p>56. Hu S, Vila AV, Gacesa R, Collij V, Stevens C, Fu JM, et al. Whole exome sequencing analyses reveal gene–microbiota interactions in the context of ibd. Gut. 2021;70:285–96. doi:<a href="https://doi.org/10.1136/gutjnl-2019-319706">10.1136/gutjnl-2019-319706</a>.</p>
</div>
<div id="ref-mayorgas2021">
<p>57. Mayorgas A, Dotti I, Salas A. Microbial metabolites, postbiotics, and intestinal epithelial function. Molecular Nutrition &amp; Food Research. 2021;65:2000188. doi:<a href="https://doi.org/10.1002/mnfr.202000188">10.1002/mnfr.202000188</a>.</p>
</div>
<div id="ref-planell2013">
<p>58. Planell N, Lozano J, Mora-Buch R, Masamunt M, Jimeno M, Ordás I, et al. Transcriptional analysis of the intestinal mucosa of patients with ulcerative colitis in remission reveals lasting epithelial cell alterations. Gut. 2013;62:967–76. doi:<a href="https://doi.org/10.1136/gutjnl-2012-303333">10.1136/gutjnl-2012-303333</a>.</p>
</div>
<div id="ref-leal2015">
<p>59. Leal R, Planell N, Kajekar R, Lozano J, Ordás I, Dotti I, et al. Identification of inflammatory mediators in patients with crohn’s disease unresponsive to anti-tnf? Therapy. Gut. 2015;64:233–42. doi:<a href="https://doi.org/10.1136/gutjnl-2013-306518">10.1136/gutjnl-2013-306518</a>.</p>
</div>
<div id="ref-knights2013">
<p>60. Knights D, Lassen K, Xavier R. Advances in inflammatory bowel disease pathogenesis: Linking host genetics and the microbiome. Gut. 2013;62. doi:<a href="https://doi.org/10.1136/gutjnl-2012-303954">10.1136/gutjnl-2012-303954</a>.</p>
</div>
<div id="ref-ahmed2016">
<p>61. Ahmed I, Roy B, Khan S, Septer S, Umar S. Microbiome, metabolome and inflammatory bowel disease. Microorganisms. 2016;4:20. doi:<a href="https://doi.org/10.3390/microorganisms4020020">10.3390/microorganisms4020020</a>.</p>
</div>
<div id="ref-gallagher2021">
<p>62. Gallagher K, Catesson A, Griffin J, Holmes E, Williams H. Metabolomic Analysis in Inflammatory Bowel Disease: A Systematic Review. Journal of Crohn’s &amp; Colitis. 2021;15:813–26.</p>
</div>
<div id="ref-ferrer-picón2020b">
<p>63. Ferrer-Picón E, Dotti I, Corraliza A, Mayorgas A, Esteller M, Perales J, et al. Intestinal inflammation modulates the epithelial response to butyrate in patients with inflammatory bowel disease. Inflammatory Bowel Diseases. 2020;26:43–55. doi:<a href="https://doi.org/10.1093/ibd/izz119">10.1093/ibd/izz119</a>.</p>
</div>
<div id="ref-hotelling_relations_1936">
<p>64. HOTELLING H. RELATIONS BETWEEN TWO SETS OF VARIATES. Biometrika. 1936;28:321–77.</p>
</div>
<div id="ref-wu_selective_2019">
<p>65. Wu C, Zhou F, Ren J, Li X, Jiang Y, Ma S. A Selective Review of Multi-Level Omics Data Integration Using Variable Selection. High-Throughput. 2019;8:4.</p>
</div>
<div id="ref-cavill_transcriptomic_2016">
<p>66. Cavill R, Jennen D, Kleinjans J, Bried’e JJ. Transcriptomic and metabolomic data integration. Briefings in Bioinformatics. 2016;17:891–901.</p>
</div>
<div id="ref-chongComputationalApproachesIntegrative2017">
<p>67. Chong J, Xia J. Computational approaches for integrative analysis of the metabolome and microbiome. Metabolites. 2017;7:62. doi:<a href="https://doi.org/10.3390/metabo7040062">10.3390/metabo7040062</a>.</p>
</div>
<div id="ref-tarazona2021">
<p>68. Tarazona S, Arzalluz-Luque A, Conesa A. Undisclosed, unmet and neglected challenges in multi-omics studies. Nature Computational Science. 2021;1–8. doi:<a href="https://doi.org/10.1038/s43588-021-00086-z">10.1038/s43588-021-00086-z</a>.</p>
</div>
<div id="ref-massoni-badosa2020">
<p>69. Massoni-Badosa R, Iacono G, Moutinho C, Kulis M, Palau N, Marchese D, et al. Sampling time-dependent artifacts in single-cell genomics studies. Genome Biology. 2020;21:112. doi:<a href="https://doi.org/10.1186/s13059-020-02032-0">10.1186/s13059-020-02032-0</a>.</p>
</div>
<div id="ref-zhu2017">
<p>70. Zhu Y, Wang L, Yin Y, Yang E. Systematic analysis of gene expression patterns associated with postmortem interval in human tissues. Scientific Reports. 2017;7:5435. doi:<a href="https://doi.org/10.1038/s41598-017-05882-0">10.1038/s41598-017-05882-0</a>.</p>
</div>
<div id="ref-ferreira2018">
<p>71. Ferreira P, Muñoz-Aguirre M, Reverter F, Sá Godinho C, Sousa A, Amadoz A, et al. The effects of death and post-mortem cold ischemia on human tissue transcriptomes. Nature Communications. 2018;9:490. doi:<a href="https://doi.org/10.1038/s41467-017-02772-x">10.1038/s41467-017-02772-x</a>.</p>
</div>
<div id="ref-koh_iomicspass_2019">
<p>72. Koh HWL, Fermin D, Vogel C, Choi KP, Ewing RM, Choi H. iOmicsPASS: Network-based integration of multiomics data for predictive subnetwork discovery. npj Systems Biology and Applications. 2019;5:1–10.</p>
</div>
<div id="ref-tenenhaus_regularized_2011">
<p>73. Tenenhaus A, Tenenhaus M. Regularized Generalized Canonical Correlation Analysis. Psychometrika. 2011;76:257–84.</p>
</div>
<div id="ref-tenenhaus_variable_2014">
<p>74. Tenenhaus A, Philippe C, Guillemot V, Le Cao K-A, Grill J, Frouin V. Variable selection for generalized canonical correlation analysis. Biostatistics. 2014;15:569–83.</p>
</div>
<div id="ref-culhane_cross-platform_2003">
<p>75. Culhane AC, Perri‘ere G, Higgins DG. Cross-platform comparison and visualisation of gene expression data using co-inertia analysis. BMC Bioinformatics. 2003;4:59.</p>
</div>
<div id="ref-vito_multi-study_2019">
<p>76. Vito RD, Bellio R, Trippa L, Parmigiani G. Multi-study factor analysis. Biometrics. 2019;75:337–46.</p>
</div>
<div id="ref-argelaguet_multi-omics_2018">
<p>77. Argelaguet R, Velten B, Arnol D, Dietrich S, Zenz T, Marioni JC, et al. Multi-Omics Factor Analysisa framework for unsupervised integration of multi-omics data sets. Molecular Systems Biology. 2018;14:e8124.</p>
</div>
<div id="ref-gomez-cabreroSTATegraComprehensiveMultiomics2019">
<p>78. Gomez-Cabrero D, Tarazona S, Ferreir’os-Vidal I, Ramirez RN, Company C, Schmidt A, et al. STATegra, a comprehensive multi-omics dataset of B-cell differentiation in mouse. Scientific Data. 2019;6:1–15.</p>
</div>
<div id="ref-franzosaSpecieslevelFunctionalProfiling2018">
<p>79. Franzosa EA, McIver LJ, Rahnavard G, Thompson LR, Schirmer M, Weingart G, et al. Species-level functional profiling of metagenomes and metatranscriptomes. Nature Methods. 2018;15:962. doi:<a href="https://doi.org/10.1038/s41592-018-0176-y">10.1038/s41592-018-0176-y</a>.</p>
</div>
<div id="ref-truongMetaPhlAn2EnhancedMetagenomic2015">
<p>80. Truong DT, Franzosa EA, Tickle TL, Scholz M, Weingart G, Pasolli E, et al. MetaPhlAn2 for enhanced metagenomic taxonomic profiling. Nature Methods. 2015;12:902–3. doi:<a href="https://doi.org/10.1038/nmeth.3589">10.1038/nmeth.3589</a>.</p>
</div>
<div id="ref-segataMetagenomicBiomarkerDiscovery2011">
<p>81. Segata N, Izard J, Waldron L, Gevers D, Miropolsky L, Garrett WS, et al. Metagenomic biomarker discovery and explanation. Genome Biology. 2011;12:R60. doi:<a href="https://doi.org/10.1186/gb-2011-12-6-r60">10.1186/gb-2011-12-6-r60</a>.</p>
</div>
<div id="ref-didier2018">
<p>82. Didier G, Valdeolivas A, Baudot A. Identifying communities from multiplex biological networks by randomized optimization of modularity. F1000Research. 2018;7. doi:<a href="https://doi.org/10.12688/f1000research.15486.2">10.12688/f1000research.15486.2</a>.</p>
</div>
<div id="ref-valdeolivas2019">
<p>83. Valdeolivas A, Tichit L, Navarro C, Perrin S, Odelin G, Levy N, et al. Random walk with restart on multiplex and heterogeneous biological networks. Bioinformatics (Oxford, England). 2019;35:497–505.</p>
</div>
<div id="ref-pio-lopez2021">
<p>84. Pio-Lopez L, Valdeolivas A, Tichit L, Remy É, Baudot A. MultiVERSE: A multiplex and multiplex-heterogeneous network embedding approach. arXiv:200810085 [cs, q-bio]. 2021. <a href="http://arxiv.org/abs/2008.10085">http://arxiv.org/abs/2008.10085</a>.</p>
</div>
<div id="ref-zhu2012">
<p>85. Zhu J, Sova P, Xu Q, Dombek K, Xu E, Vu H, et al. Stitching together multiple data dimensions reveals interacting metabolomic and transcriptomic networks that modulate cell regulation. PLOS Biology. 2012;10:e1001301. doi:<a href="https://doi.org/10.1371/journal.pbio.1001301">10.1371/journal.pbio.1001301</a>.</p>
</div>
<div id="ref-lock2013">
<p>86. Lock E, Dunson D. Bayesian consensus clustering. Bioinformatics. 2013;29:2610–6. doi:<a href="https://doi.org/10.1093/bioinformatics/btt425">10.1093/bioinformatics/btt425</a>.</p>
</div>
<div id="ref-virtanen2012">
<p>87. Virtanen S, Klami A, Khan S, Kaski S. Artificial intelligence and statistics. In. PMLR; 2012. pp. 1269–77. <a href="https://proceedings.mlr.press/v22/virtanen12.html">https://proceedings.mlr.press/v22/virtanen12.html</a>.</p>
</div>
<div id="ref-cantini_benchmarking_2020">
<p>88. Cantini L, Zakeri P, Hernandez C, Naldi A, Thieffry D, Remy E, et al. Benchmarking joint multi-omics dimensionality reduction approaches for cancer study. bioRxiv. 2020;2020.01.14.905760.</p>
</div>
<div id="ref-tenenhaus2017">
<p>89. Tenenhaus M, Tenenhaus A, Groenen P. Regularized generalized canonical correlation analysis: A framework for sequential multiblock component methods. Psychometrika. 2017;82:737–77. doi:<a href="https://doi.org/10.1007/s11336-017-9573-x">10.1007/s11336-017-9573-x</a>.</p>
</div>
<div id="ref-rohart2017">
<p>90. Rohart F, Gautier B, Singh A, Cao K. MixOmics: An r package for ‘omics feature selection and multiple data integration. PLOS Computational Biology. 2017;13:e1005752. doi:<a href="https://doi.org/10.1371/journal.pcbi.1005752">10.1371/journal.pcbi.1005752</a>.</p>
</div>
<div id="ref-gomez-cabrero2019">
<p>91. Gomez-Cabrero D, Tarazona S, Ferreirós-Vidal I, Ramirez R, Company C, Schmidt A, et al. STATegra, a comprehensive multi-omics dataset of b-cell differentiation in mouse. Scientific Data. 2019;6:1–15. doi:<a href="https://doi.org/10.1038/s41597-019-0202-7">10.1038/s41597-019-0202-7</a>.</p>
</div>
<div id="ref-novoa-del-toro2020">
<p>92. Novoa-del-Toro E, Mezura-Montes E, Vignes M, Magdinier F, Tichit L, Baudot A. A multi-objective genetic algorithm to find active modules in multiplex biological networks. bioRxiv. 2020;2020.05.25.114215. doi:<a href="https://doi.org/10.1101/2020.05.25.114215">10.1101/2020.05.25.114215</a>.</p>
</div>
<div id="ref-yang2016">
<p>93. Yang Z, Michailidis G. A non-negative matrix factorization method for detecting modules in heterogeneous omics multi-modal data. Bioinformatics. 2016;32:1–8. doi:<a href="https://doi.org/10.1093/bioinformatics/btv544">10.1093/bioinformatics/btv544</a>.</p>
</div>
<div id="ref-sherry2005">
<p>94. Sherry A, Henson R. Conducting and interpreting canonical correlation analysis in personality research: A user-friendly primer. Journal of Personality Assessment. 2005;84:37–48. doi:<a href="https://doi.org/10.1207/s15327752jpa8401_09">10.1207/s15327752jpa8401_09</a>.</p>
</div>
<div id="ref-sherryConductingInterpretingCanonical1981">
<p>95. Sherry A, Henson RK. Conducting and Interpreting Canonical Correlation Analysis in Personality Research: A User-Friendly Primer. 1981.</p>
</div>
<div id="ref-chung_multi-omics_2019">
<p>96. Chung R-H, Kang C-Y. A multi-omics data simulator for complex disease studies and its application to evaluate multi-omics data analysis methods for disease classification. GigaScience. 2019;8.</p>
</div>
<div id="ref-martinez-mira_mosim_2018">
<p>97. Martínez-Mira C, Conesa A, Tarazona S. MOSim: Multi-Omics Simulation in R. Preprint. Bioinformatics; 2018.</p>
</div>
<div id="ref-haberman2019">
<p>98. Haberman Y, Karns R, Dexheimer P, Schirmer M, Somekh J, Jurickova I, et al. Ulcerative colitis mucosal transcriptomes reveal mitochondriopathy and personalized mechanisms underlying disease severity and treatment response. Nature Communications. 2019;10. doi:<a href="https://doi.org/10.1038/s41467-018-07841-3">10.1038/s41467-018-07841-3</a>.</p>
</div>
<div id="ref-patuzzi2019">
<p>99. Patuzzi I, Baruzzo G, Losasso C, Ricci A, Di Camillo B. MetaSPARSim: A 16S rRNA gene sequencing count data simulator. BMC Bioinformatics. 2019;20:416. doi:<a href="https://doi.org/10.1186/s12859-019-2882-6">10.1186/s12859-019-2882-6</a>.</p>
</div>
<div id="ref-fritz2019">
<p>100. Fritz A, Hofmann P, Majda S, Dahms E, Dröge J, Fiedler J, et al. CAMISIM: Simulating metagenomes and microbial communities. Microbiome. 2019;7:17. doi:<a href="https://doi.org/10.1186/s40168-019-0633-6">10.1186/s40168-019-0633-6</a>.</p>
</div>
<div id="ref-fu2021">
<p>101. Fu J, Frazee A, Collado-Torres L, Jaffe A, Leek J. Ballgown: Flexible, isoform-level differential expression analysis. Bioconductor version: Release (3.13); 2021. doi:<a href="https://doi.org/10.18129/B9.bioc.ballgown">10.18129/B9.bioc.ballgown</a>.</p>
</div>
<div id="ref-frazee2021">
<p>102. Frazee A, Jaffe A, Kirchner R, Leek J. Polyester: Simulate rna-seq reads. Bioconductor version: Release (3.13); 2021. doi:<a href="https://doi.org/10.18129/B9.bioc.polyester">10.18129/B9.bioc.polyester</a>.</p>
</div>
<div id="ref-mccarthy2012">
<p>103. McCarthy D, Chen Y, Smyth G. Differential expression analysis of multifactor rna-seq experiments with respect to biological variation. Nucleic Acids Research. 2012;40:4288–97. doi:<a href="https://doi.org/10.1093/nar/gks042">10.1093/nar/gks042</a>.</p>
</div>
<div id="ref-cavill2016">
<p>104. Cavill R, Jennen D, Kleinjans J, Briedé J. Transcriptomic and metabolomic data integration. Briefings in Bioinformatics. 2016;17:891–901. doi:<a href="https://doi.org/10.1093/bib/bbv090">10.1093/bib/bbv090</a>.</p>
</div>
<div id="ref-mcgovern2015a">
<p>105. McGovern D, Kugathasan S, Cho J. Genetics of Inflammatory Bowel Diseases. Gastroenterology. 2015;149:1163–1176.e2.</p>
</div>
<div id="ref-de_souza_ibd_2017">
<p>106. De Souza HSP, Fiocchi C, Iliopoulos D. The IBD interactome: An integrated view of aetiology, pathogenesis and therapy. 2017;14.</p>
</div>
<div id="ref-tarazona2021a">
<p>107. Tarazona S, Arzalluz-Luque A, Conesa A. Undisclosed, unmet and neglected challenges in multi-omics studies. Nature Computational Science. 2021;1–8. doi:<a href="https://doi.org/10.1038/s43588-021-00086-z">10.1038/s43588-021-00086-z</a>.</p>
</div>
<div id="ref-valles-colomer2016">
<p>108. Valles-Colomer M, Darzi Y, Vieira-Silva S, Falony G, Raes J, Joossens M. Meta-omics in inflammatory bowel disease research: Applications, challenges, and guidelines. Journal of Crohn’s and Colitis. 2016;10:735–46. doi:<a href="https://doi.org/10.1093/ecco-jcc/jjw024">10.1093/ecco-jcc/jjw024</a>.</p>
</div>
<div id="ref-parkhomenko2009">
<p>109. Parkhomenko E, Tritchler D, Beyene J. Sparse canonical correlation analysis with application to genomic data integration. Statistical Applications in Genetics and Molecular Biology. 2009;8. doi:<a href="https://doi.org/10.2202/1544-6115.1406">10.2202/1544-6115.1406</a>.</p>
</div>
<div id="ref-waaijenborg2008">
<p>110. Waaijenborg S, Hamer P, Zwinderman A. Quantifying the association between gene expressions and dna-markers by penalized canonical correlation analysis. Statistical Applications in Genetics and Molecular Biology. 2008;7. doi:<a href="https://doi.org/10.2202/1544-6115.1329">10.2202/1544-6115.1329</a>.</p>
</div>
<div id="ref-witten2009">
<p>111. Witten D, Tibshirani R. Extensions of sparse canonical correlation analysis with applications to genomic data. Statistical Applications in Genetics and Molecular Biology. 2009;8:127. doi:<a href="https://doi.org/10.2202/1544-6115.1470">10.2202/1544-6115.1470</a>.</p>
</div>
<div id="ref-lêcao2009">
<p>112. Lê Cao K, Martin P, Robert-Granié C, Besse P. Sparse canonical methods for biological data integration: Application to a cross-platform study. BMC Bioinformatics. 2009;10:34. doi:<a href="https://doi.org/10.1186/1471-2105-10-34">10.1186/1471-2105-10-34</a>.</p>
</div>
<div id="ref-hwang2009">
<p>113. Hwang H. Regularized generalized structured component analysis. Psychometrika. 2009;74:517–30. doi:<a href="https://doi.org/10.1007/s11336-009-9119-y">10.1007/s11336-009-9119-y</a>.</p>
</div>
<div id="ref-soneson2010">
<p>114. Soneson C, Lilljebjörn H, Fioretos T, Fontes M. Integrative analysis of gene expression and copy number alterations using canonical correlation analysis. BMC Bioinformatics. 2010;11:191. doi:<a href="https://doi.org/10.1186/1471-2105-11-191">10.1186/1471-2105-11-191</a>.</p>
</div>
<div id="ref-zhang2011">
<p>115. Zhang S, Li Q, Liu J, Zhou X. A novel computational framework for simultaneous integration of multiple types of genomic data to identify microRNA-gene regulatory modules. Bioinformatics. 2011;27:i401–9. doi:<a href="https://doi.org/10.1093/bioinformatics/btr206">10.1093/bioinformatics/btr206</a>.</p>
</div>
<div id="ref-lee2011">
<p>116. Lee W, Lee D, Lee Y, Pawitan Y. Sparse canonical covariance analysis for high-throughput data. Statistical Applications in Genetics and Molecular Biology. 2011;10. doi:<a href="https://doi.org/10.2202/1544-6115.1638">10.2202/1544-6115.1638</a>.</p>
</div>
<div id="ref-abdi2012">
<p>117. Abdi H, Williams L, Valentin D, Bennani-Dosse M. STATIS and distatis: Optimum multitable principal component analysis and three way metric multidimensional scaling. WIREs Computational Statistics. 2012;4:124–67. doi:<a href="https://doi.org/10.1002/wics.198">10.1002/wics.198</a>.</p>
</div>
<div id="ref-zhang2012">
<p>118. Zhang S, Liu C, Li W, Shen H, Laird P, Zhou X. Discovery of multi-dimensional modules by integrative analysis of cancer genomic data. Nucleic Acids Research. 2012;40:9379–91. doi:<a href="https://doi.org/10.1093/nar/gks725">10.1093/nar/gks725</a>.</p>
</div>
<div id="ref-li2012">
<p>119. Li W, Zhang S, Liu C, Zhou X. Identifying multi-layer gene regulatory modules from multi-dimensional genomic data. Bioinformatics. 2012;28:2458–66. doi:<a href="https://doi.org/10.1093/bioinformatics/bts476">10.1093/bioinformatics/bts476</a>.</p>
</div>
<div id="ref-virtanen2012a">
<p>120. Virtanen S, Klami A, Khan S, Kaski S. Artificial intelligence and statistics. In. PMLR; 2012. pp. 1269–77. <a href="https://proceedings.mlr.press/v22/virtanen12.html">https://proceedings.mlr.press/v22/virtanen12.html</a>.</p>
</div>
<div id="ref-zhu2012a">
<p>121. Zhu J, Sova P, Xu Q, Dombek K, Xu E, Vu H, et al. Stitching together multiple data dimensions reveals interacting metabolomic and transcriptomic networks that modulate cell regulation. PLOS Biology. 2012;10:e1001301. doi:<a href="https://doi.org/10.1371/journal.pbio.1001301">10.1371/journal.pbio.1001301</a>.</p>
</div>
<div id="ref-abdi2013">
<p>122. Abdi H, Williams L, Valentin D. Multiple factor analysis: Principal component analysis for multitable and multiblock data sets. WIREs Computational Statistics. 2013;5:149–79. doi:<a href="https://doi.org/10.1002/wics.1246">10.1002/wics.1246</a>.</p>
</div>
<div id="ref-lock2013a">
<p>123. Lock E, Hoadley K, Marron J, Nobel A. JOINT and individual variation explained (jive) for integrated analysis of multiple data types. The annals of applied statistics. 2013;7:523–42. doi:<a href="https://doi.org/10.1214/12-AOAS597">10.1214/12-AOAS597</a>.</p>
</div>
<div id="ref-schlauch2017">
<p>124. Schlauch D, Paulson J, Young A, Glass K, Quackenbush J. Estimating gene regulatory networks with pandaR. Bioinformatics. 2017;33:2232–4. doi:<a href="https://doi.org/10.1093/bioinformatics/btx139">10.1093/bioinformatics/btx139</a>.</p>
</div>
<div id="ref-meng2014a">
<p>125. Meng C, Kuster B, Culhane A, Gholami A. A multivariate approach to the integration of multi-omics datasets. BMC Bioinformatics. 2014;15:162. doi:<a href="https://doi.org/10.1186/1471-2105-15-162">10.1186/1471-2105-15-162</a>.</p>
</div>
<div id="ref-planell2021a">
<p>126. Planell N, Lagani V, Sebastian-Leon P, van der Kloet F, Ewing E, Karathanasis N, et al. STATegra: Multi-omics data integration a conceptual scheme with a bioinformatics pipeline. Frontiers in Genetics. 2021;12. doi:<a href="https://doi.org/10.3389/fgene.2021.620453">10.3389/fgene.2021.620453</a>.</p>
</div>
<div id="ref-ray2014">
<p>127. Ray P, Zheng L, Lucas J, Carin L. Bayesian joint analysis of heterogeneous genomics data. Bioinformatics. 2014;30:1370–6. doi:<a href="https://doi.org/10.1093/bioinformatics/btu064">10.1093/bioinformatics/btu064</a>.</p>
</div>
<div id="ref-bunte2016">
<p>128. Bunte K, Leppäaho E, Saarinen I, Kaski S. Sparse group factor analysis for biclustering of multiple data sources. Bioinformatics. 2016;32:2457–63. doi:<a href="https://doi.org/10.1093/bioinformatics/btw207">10.1093/bioinformatics/btw207</a>.</p>
</div>
<div id="ref-chen2013">
<p>129. Chen M, Gao C, Ren Z, Zhou H. Sparse cca via precision adjusted iterative thresholding. arXiv:13116186 [math, stat]. 2013. <a href="http://arxiv.org/abs/1311.6186">http://arxiv.org/abs/1311.6186</a>.</p>
</div>
<div id="ref-leppäaho2017">
<p>130. Leppäaho E, Ammad-ud-din M, Kaski S. GFA: Exploratory analysis of multiple data sources with group factor analysis. Journal of Machine Learning Research. 2017;18:1–5. <a href="http://jmlr.org/papers/v18/16-509.html">http://jmlr.org/papers/v18/16-509.html</a>.</p>
</div>
<div id="ref-klami2014">
<p>131. Klami A, Bouchard G, Tripathi A. Group-sparse embeddings in collective matrix factorization. arXiv:13125921 [cs, stat]. 2014. <a href="http://arxiv.org/abs/1312.5921">http://arxiv.org/abs/1312.5921</a>.</p>
</div>
<div id="ref-meng2018">
<p>132. Meng C, Basunia A, Peters B, Gholami A, Kuster B, Culhane A. MOGSA: Integrative single sample gene-set analysis of multiple omics data. 2018. <a href="https://www.biorxiv.org/content/10.1101/046904v2">https://www.biorxiv.org/content/10.1101/046904v2</a>.</p>
</div>
<div id="ref-yang2016b">
<p>133. Yang Z, Michailidis G. A non-negative matrix factorization method for detecting modules in heterogeneous omics multi-modal data. Bioinformatics. 2016;32:1–8. doi:<a href="https://doi.org/10.1093/bioinformatics/btv544">10.1093/bioinformatics/btv544</a>.</p>
</div>
<div id="ref-zhao2015">
<p>134. Zhao S, Gao C, Mukherjee S, Engelhardt B. Bayesian group latent factor analysis with structured sparsity. arXiv:14112698 [q-bio, stat]. 2015. <a href="http://arxiv.org/abs/1411.2698">http://arxiv.org/abs/1411.2698</a>.</p>
</div>
<div id="ref-voillet2016">
<p>135. Voillet V, Besse P, Liaubet L, San Cristobal M, González I. Handling missing rows in multi-omics data integration: Multiple imputation in multiple factor analysis framework. BMC Bioinformatics. 2016;17:402. doi:<a href="https://doi.org/10.1186/s12859-016-1273-5">10.1186/s12859-016-1273-5</a>.</p>
</div>
<div id="ref-beaton2016">
<p>136. Beaton D, Dunlop J, Abdi H, Alzheimer’s Disease Neuroimaging Initiative. Partial least squares correspondence analysis: A framework to simultaneously analyze behavioral and genetic data. Psychological Methods. 2016;21:621–51.</p>
</div>
<div id="ref-singh2019">
<p>137. Singh A, Shannon C, Gautier B, Rohart F, Vacher M, Tebbutt S, et al. DIABLO: An integrative approach for identifying key molecular drivers from multi-omics assays. Bioinformatics. 2019;35:3055–62. doi:<a href="https://doi.org/10.1093/bioinformatics/bty1054">10.1093/bioinformatics/bty1054</a>.</p>
</div>
<div id="ref-yoon2019">
<p>138. Yoon G, Carroll R, Gaynanova I. Sparse semiparametric canonical correlation analysis for data of mixed types. arXiv:180705274 [stat]. 2019. <a href="http://arxiv.org/abs/1807.05274">http://arxiv.org/abs/1807.05274</a>.</p>
</div>
<div id="ref-gaynanova2017a">
<p>139. Gaynanova I, Li G. Structural learning and integrative decomposition of multi-view data. arXiv:170706573 [stat]. 2017. <a href="http://arxiv.org/abs/1707.06573">http://arxiv.org/abs/1707.06573</a>.</p>
</div>
<div id="ref-madrigal2017">
<p>140. Madrigal P. FCCAC: Functional canonical correlation analysis to evaluate covariance between nucleic acid sequencing datasets. Bioinformatics. 2017;33:746–8. doi:<a href="https://doi.org/10.1093/bioinformatics/btw724">10.1093/bioinformatics/btw724</a>.</p>
</div>
<div id="ref-yoshida2017">
<p>141. Yoshida K, Yoshimoto J, Doya K. Sparse kernel canonical correlation analysis for discovery of nonlinear interactions in high-dimensional data. BMC Bioinformatics. 2017;18:108. doi:<a href="https://doi.org/10.1186/s12859-017-1543-x">10.1186/s12859-017-1543-x</a>.</p>
</div>
<div id="ref-kawaguchi2017">
<p>142. Kawaguchi A, Yamashita F. Supervised multiblock sparse multivariable analysis with application to multimodal brain imaging genetics. Biostatistics. 2017;18:651–65. doi:<a href="https://doi.org/10.1093/biostatistics/kxx011">10.1093/biostatistics/kxx011</a>.</p>
</div>
<div id="ref-feng2018">
<p>143. Feng Q, Jiang M, Hannig J, Marron J. Angle-based joint and individual variation explained. arXiv:170402060 [stat]. 2018. <a href="http://arxiv.org/abs/1704.02060">http://arxiv.org/abs/1704.02060</a>.</p>
</div>
<div id="ref-argelaguet2020">
<p>144. Argelaguet R, Arnol D, Bredikhin D, Deloro Y, Velten B, Marioni J, et al. MOFA+: A statistical framework for comprehensive integration of multi-modal single-cell data. Genome Biology. 2020;21:111. doi:<a href="https://doi.org/10.1186/s13059-020-02015-1">10.1186/s13059-020-02015-1</a>.</p>
</div>
<div id="ref-brown2018">
<p>145. Brown B, Bray N, Pachter L. Expression reflects population structure. 2018. doi:<a href="https://doi.org/10.1101/364448">10.1101/364448</a>.</p>
</div>
<div id="ref-zhang2020">
<p>146. Zhang Y, Gaynanova I. Joint association and classification analysis of multi-view data. arXiv:181108511 [cs, stat]. 2020. <a href="http://arxiv.org/abs/1811.08511">http://arxiv.org/abs/1811.08511</a>.</p>
</div>
<div id="ref-tang2021">
<p>147. Tang T, Allen G. Integrated principal components analysis. arXiv:181000832 [stat]. 2021. <a href="http://arxiv.org/abs/1810.00832">http://arxiv.org/abs/1810.00832</a>.</p>
</div>
<div id="ref-min2019">
<p>148. Min E, Safo S, Long Q. Penalized co-inertia analysis with applications to -omics data. Bioinformatics (Oxford, England). 2019;35:1018–25.</p>
</div>
<div id="ref-safo2018">
<p>149. Safo S, Li S, Long Q. Integrative analysis of transcriptomic and metabolomic data via sparse canonical correlation analysis with incorporation of biological information. Biometrics. 2018;74:300–12. doi:<a href="https://doi.org/10.1111/biom.12715">10.1111/biom.12715</a>.</p>
</div>
<div id="ref-min2017">
<p>150. Min W, Liu J, Zhang S. Sparse weighted canonical correlation analysis. arXiv:171004792 [cs, stat]. 2017. <a href="http://arxiv.org/abs/1710.04792">http://arxiv.org/abs/1710.04792</a>.</p>
</div>
<div id="ref-bouhaddani2018">
<p>151. Bouhaddani S, Uh H, Jongbloed G, Hayward C, Klarić L, Kiełbasa S, et al. Integrating omics datasets with the omicspls package. BMC Bioinformatics. 2018;19:371. doi:<a href="https://doi.org/10.1186/s12859-018-2371-3">10.1186/s12859-018-2371-3</a>.</p>
</div>
<div id="ref-pimentel2018">
<p>152. Pimentel H, Zhiyue H, Huang H. Biclustering by sparse canonical correlation analysis. 2018;6:11. doi:<a href="https://doi.org/https://doi.org/10.1007/s40484-017-0127-0">https://doi.org/10.1007/s40484-017-0127-0</a>.</p>
</div>
<div id="ref-kim2019">
<p>153. Kim Y, Bismeijer T, Zwart W, Wessels L, Vis D. Genomic data integration by won-parafac identifies interpretable factors for predicting drug-sensitivity in vivo. Nature Communications. 2019;10:5034. doi:<a href="https://doi.org/10.1038/s41467-019-13027-2">10.1038/s41467-019-13027-2</a>.</p>
</div>
<div id="ref-lock2020">
<p>154. Lock E, Park J, Hoadley K. Bidimensional linked matrix factorization for pan-omics pan-cancer analysis. arXiv:200202601 [cs, q-bio, stat]. 2020. <a href="http://arxiv.org/abs/2002.02601">http://arxiv.org/abs/2002.02601</a>.</p>
</div>
<div id="ref-ronen2019">
<p>155. Ronen J, Hayat S, Akalin A. Evaluation of colorectal cancer subtypes and cell lines using deep learning. Life Science Alliance. 2019;2. doi:<a href="https://doi.org/10.26508/lsa.201900517">10.26508/lsa.201900517</a>.</p>
</div>
<div id="ref-shi2019">
<p>156. Shi W, Zhuang Y, Russell P, Hobbs B, Parker M, Castaldi P, et al. Unsupervised discovery of phenotype-specific multi-omics networks. Bioinformatics. 2019;35:4336–43. doi:<a href="https://doi.org/10.1093/bioinformatics/btz226">10.1093/bioinformatics/btz226</a>.</p>
</div>
<div id="ref-csala2020">
<p>157. Csala A, Zwinderman A, Hof M. Multiset sparse partial least squares path modeling for high dimensional omics data analysis. BMC Bioinformatics. 2020;21:9. doi:<a href="https://doi.org/10.1186/s12859-019-3286-3">10.1186/s12859-019-3286-3</a>.</p>
</div>
<div id="ref-fan2020">
<p>158. Fan Z, Zhou Y, Ressom H. MOTA: Network-based multi-omic data integration for biomarker discovery. Metabolites. 2020;10:144. doi:<a href="https://doi.org/10.3390/metabo10040144">10.3390/metabo10040144</a>.</p>
</div>
<div id="ref-shu2020">
<p>159. Shu H, Wang X, Zhu H. D-cca: A decomposition-based canonical correlation analysis for high-dimensional datasets. Journal of the American Statistical Association. 2020;115:292–306. doi:<a href="https://doi.org/10.1080/01621459.2018.1543599">10.1080/01621459.2018.1543599</a>.</p>
</div>
<div id="ref-hawinkel2020">
<p>160. Hawinkel S, Bijnens L, Cao K, Thas O. Model-based joint visualization of multiple compositional omics datasets. NAR Genomics and Bioinformatics. 2020;2. doi:<a href="https://doi.org/10.1093/nargab/lqaa050">10.1093/nargab/lqaa050</a>.</p>
</div>
<div id="ref-gundersen2020">
<p>161. Gundersen G, Dumitrascu B, Ash J, Engelhardt B. Uncertainty in artificial intelligence. In. PMLR; 2020. pp. 945–55. <a href="https://proceedings.mlr.press/v115/gundersen20a.html">https://proceedings.mlr.press/v115/gundersen20a.html</a>.</p>
</div>
<div id="ref-velten2020">
<p>162. Velten B, Braunger J, Arnol D, Argelaguet R, Stegle O. Identifying temporal and spatial patterns of variation from multi-modal data using mefisto. bioRxiv. 2020;2020.11.03.366674. doi:<a href="https://doi.org/10.1101/2020.11.03.366674">10.1101/2020.11.03.366674</a>.</p>
</div>
<div id="ref-tarazona2020">
<p>163. Tarazona S, Balzano-Nogueira L, Gómez-Cabrero D, Schmidt A, Imhof A, Hankemeier T, et al. Harmonization of quality metrics and power calculation in multi-omic studies. Nature Communications. 2020;11:3092. doi:<a href="https://doi.org/10.1038/s41467-020-16937-8">10.1038/s41467-020-16937-8</a>.</p>
</div>
<div id="ref-yoon2019a">
<p>164. Yoon G, Carroll R, Gaynanova I. Sparse semiparametric canonical correlation analysis for data of mixed types. arXiv:180705274 [stat]. 2019. <a href="http://arxiv.org/abs/1807.05274">http://arxiv.org/abs/1807.05274</a>.</p>
</div>
<div id="ref-martin2011">
<p>165. Martin M. Cutadapt removes adapter sequences from high-throughput sequencing reads. EMBnetjournal. 2011;17:10–2. doi:<a href="https://doi.org/10.14806/ej.17.1.200">10.14806/ej.17.1.200</a>.</p>
</div>
<div id="ref-dobin2013">
<p>166. Dobin A, Davis C, Schlesinger F, Drenkow J, Zaleski C, Jha S, et al. STAR: ultrafast universal RNA-seq aligner. Bioinformatics (Oxford, England). 2013;29:15–21.</p>
</div>
<div id="ref-li2011">
<p>167. Li B, Dewey C. RSEM: Accurate transcript quantification from rna-seq data with or without a reference genome. BMC Bioinformatics. 2011;12:323. doi:<a href="https://doi.org/10.1186/1471-2105-12-323">10.1186/1471-2105-12-323</a>.</p>
</div>
<div id="ref-berry2011">
<p>168. Berry D, Ben Mahfoudh K, Wagner M, Loy A. Barcoded primers used in multiplex amplicon pyrosequencing bias amplification. Applied and Environmental Microbiology. 2011;77:7846–9.</p>
</div>
<div id="ref-klindworth2013">
<p>169. Klindworth A, Pruesse E, Schweer T, Peplies J, Quast C, Horn M, et al. Evaluation of general 16S ribosomal rna gene pcr primers for classical and next-generation sequencing-based diversity studies. Nucleic Acids Research. 2013;41:e1–1. doi:<a href="https://doi.org/10.1093/nar/gks808">10.1093/nar/gks808</a>.</p>
</div>
<div id="ref-edgar2013a">
<p>170. Edgar R. UPARSE: Highly accurate otu sequences from microbial amplicon reads. Nature Methods. 2013;10:996–8. doi:<a href="https://doi.org/10.1038/nmeth.2604">10.1038/nmeth.2604</a>.</p>
</div>
<div id="ref-lagkouvardos2016">
<p>171. Lagkouvardos I, Joseph D, Kapfhammer M, Giritli S, Horn M, Haller D, et al. IMNGS: A comprehensive open resource of processed 16S rRNA microbial profiles for ecology and diversity studies. Scientific Reports. 2016;6. doi:<a href="https://doi.org/10.1038/srep33721">10.1038/srep33721</a>.</p>
</div>
<div id="ref-callahan2016">
<p>172. Callahan B, McMurdie P, Rosen M, Han A, Johnson A, Holmes S. DADA2: High-resolution sample inference from illumina amplicon data. Nature Methods. 2016;13:581–3. doi:<a href="https://doi.org/10.1038/nmeth.3869">10.1038/nmeth.3869</a>.</p>
</div>
<div id="ref-bolyen2019">
<p>173. Bolyen E, Rideout J, Dillon M, Bokulich N, Abnet C, Al-Ghalith G, et al. Reproducible, interactive, scalable and extensible microbiome data science using qiime 2. Nature Biotechnology. 2019;37:852–7. doi:<a href="https://doi.org/10.1038/s41587-019-0209-9">10.1038/s41587-019-0209-9</a>.</p>
</div>
<div id="ref-korotkevich2021">
<p>174. Korotkevich G, Sukhov V, Budin N, Shpak B, Artyomov M, Sergushichev A. Fast gene set enrichment analysis. 2021. doi:<a href="https://doi.org/https://doi.org/10.1101/060012">https://doi.org/10.1101/060012</a>.</p>
</div>
<div id="ref-hänzelmann2013">
<p>175. Hänzelmann S, Castelo R, Guinney J. GSVA: Gene set variation analysis for microarray and rna-seq data. BMC Bioinformatics. 2013;14:7. doi:<a href="https://doi.org/10.1186/1471-2105-14-7">10.1186/1471-2105-14-7</a>.</p>
</div>
<div id="ref-oksanen2020">
<p>176. Oksanen J, Blanchet F, Friendly M, Kindt R, Legendre P, McGlinn D, et al. Vegan: Community ecology package. 2020. <a href="https://CRAN.R-project.org/package=vegan">https://CRAN.R-project.org/package=vegan</a>.</p>
</div>
<div id="ref-wu2021">
<p>177. Wu T, Hu E, Xu S, Chen M, Guo P, Dai Z, et al. ClusterProfiler 4.0: A universal enrichment tool for interpreting omics data. The Innovation. 2021;2:100141. doi:<a href="https://doi.org/10.1016/j.xinn.2021.100141">10.1016/j.xinn.2021.100141</a>.</p>
</div>
<div id="ref-langfelder2008">
<p>178. Langfelder P, Horvath S. WGCNA: An r package for weighted correlation network analysis. BMC Bioinformatics. 2008;9:559. doi:<a href="https://doi.org/10.1186/1471-2105-9-559">10.1186/1471-2105-9-559</a>.</p>
</div>
<div id="ref-meng2014">
<p>179. Meng C, Kuster B, Culhane A, Gholami A. A multivariate approach to the integration of multi-omics datasets. BMC Bioinformatics. 2014;15:162. doi:<a href="https://doi.org/10.1186/1471-2105-15-162">10.1186/1471-2105-15-162</a>.</p>
</div>
<div id="ref-goeman2006">
<p>180. Goeman J, van de Geer S, van Houwelingen H. Testing against a high dimensional alternative. Journal of the Royal Statistical Society Series B (Statistical Methodology). 2006;68:477–93. <a href="https://www.jstor.org/stable/3879286">https://www.jstor.org/stable/3879286</a>.</p>
</div>
<div id="ref-planell2021">
<p>181. Planell N, Lagani V, Sebastian-Leon P, van der Kloet F, Ewing E, Karathanasis N, et al. STATegra: Multi-omics data integration a conceptual scheme with a bioinformatics pipeline. Frontiers in Genetics. 2021;12. doi:<a href="https://doi.org/10.3389/fgene.2021.620453">10.3389/fgene.2021.620453</a>.</p>
</div>
<div id="ref-tenenhaus2017a">
<p>182. Tenenhaus M, Tenenhaus A, Groenen P. Regularized generalized canonical correlation analysis: A framework for sequential multiblock component methods. Psychometrika. 2017;82:737–77. doi:<a href="https://doi.org/10.1007/s11336-017-9573-x">10.1007/s11336-017-9573-x</a>.</p>
</div>
<div id="ref-tenenhaus_component-based_2008">
<p>183. Tenenhaus M. Component-based Structural Equation Modelling. Total Quality Management &amp; Business Excellence. 2008;19:871–86.</p>
</div>
<div id="ref-esposito_vinzi_bridge_2010">
<p>184. Tenenhaus M, Hanafi M. A Bridge Between PLS Path Modeling and Multi-Block Data Analysis. In: Esposito Vinzi V, Chin WW, Henseler J, Wang H, editors. Handbook of Partial Least Squares. Berlin, Heidelberg: Springer Berlin Heidelberg; 2010. pp. 99–123.</p>
</div>
<div id="ref-tenenhaus_regularized_2014">
<p>185. Tenenhaus A, Tenenhaus M. Regularized generalized canonical correlation analysis for multiblock or multigroup data analysis. European Journal of Operational Research. 2014;238:391–403.</p>
</div>
<div id="ref-tenenhaus_kernel_2015">
<p>186. Tenenhaus A, Philippe C, Frouin V. Kernel Generalized Canonical Correlation Analysis. Computational Statistics &amp; Data Analysis. 2015;90:114–31.</p>
</div>
<div id="ref-mengMultivariateApproachIntegration2014">
<p>187. Meng C, Kuster B, Culhane AC, Gholami AM. A multivariate approach to the integration of multi-omics datasets. BMC Bioinformatics. 2014;15:162. doi:<a href="https://doi.org/10.1186/1471-2105-15-162">10.1186/1471-2105-15-162</a>.</p>
</div>
<div id="ref-morgan2015">
<p>188. Morgan X, Kabakchiev B, Waldron L, Tyler A, Tickle T, Milgrom R, et al. Associations between host gene expression, the mucosal microbiome, and clinical outcome in the pelvic pouch of patients with inflammatory bowel disease. Genome Biology. 2015;16:67. doi:<a href="https://doi.org/10.1186/s13059-015-0637-x">10.1186/s13059-015-0637-x</a>.</p>
</div>
</div>
</div>



            </section>

          </div>
        </div>
      </div>
<a href="acknowledgements.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="online-resources.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": {
"href": "https://github.com/llrs/thesis/"
},
"facebook": false,
"twitter": {
"href": "https://twitter.com/intent/tweet?text=Nice%20read%20of%20%40Lluis_Rev%20thesis"
},
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/llrs/thesis/edit/master/98_appendix.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": "https://github.com/llrs/thesis/blob/master/98_appendix.Rmd",
"text": null
},
"download": ["Data integration on inflammatory bowel disease.pdf", "Data integration on inflammatory bowel disease.epub"],
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": false,
"toc_appendix": true,
"scroll_highlight": true
},
"toc_float": true
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
