<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>References | Data integration in inflammatory bowel disease</title>
  <meta name="description" content="References | Data integration in inflammatory bowel disease" />
  <meta name="generator" content="bookdown 0.26 and GitBook 2.6.7" />

  <meta property="og:title" content="References | Data integration in inflammatory bowel disease" />
  <meta property="og:type" content="book" />
  
  
  <meta name="github-repo" content="llrs/thesis" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="References | Data integration in inflammatory bowel disease" />
  <meta name="twitter:site" content="@Lluis_Revilla" />
  
  

<meta name="author" content="Lluís Revilla Sancho" />


<meta name="date" content="2022-05-11" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="acknowledgments.html"/>
<link rel="next" href="online-resources.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1, orientation=auto"/>
<meta name="description" content="Thesis searching the relationship between microbiome and transcriptome on the intestine on patients with IBD to discover key genes and bacteria."/>
<meta name="github-repo" content="llrs/thesis"/>
<meta name="twitter:title" content="References | Data integration in inflammatory bowel disease"/>
<meta name="twitter:description" content="Thesis searching the relationship between microbiome and transcriptome on the intestine on patients with IBD to discover key genes and bacteria."/>
<meta name="twitter:url" content="https://thesis.llrs.dev"/>
<meta name="twitter:image:alt" content="The cover of the thesis."/>
<meta name="twitter:card" content="summary_large_image"/>
<meta name="twitter:creator" content="@Lluis_Revilla"/>
<meta name="twitter:site" content="@Lluis_Revilla"/>
<meta property="og:title" content="References | Data integration in inflammatory bowel disease"/>
<meta property="og:description" content="Thesis searching the relationship between microbiome and transcriptome on the intestine on patients with IBD to discover key genes and bacteria."/>
<meta property="og:url" content="https://thesis.llrs.dev"/>
<meta property="og:image:alt" content="The cover of the thesis."/>
<meta property="og:type" content="book"/>
<meta property="og:locale" content="en_US"/>
<meta property="article:author" content="Llu&lt;U+00ED&gt;s Revilla Sancho"/>
<script src="libs/kePrint-0.0.1/kePrint.js"></script>
<link href="libs/lightable-0.0.1/lightable.css" rel="stylesheet" />
<link rel="apple-touch-icon" sizes="180x180" href="images/apple-touch-icon.png">
<link rel="icon" type="image/png" sizes="32x32" href="images/favicon-32x32.png">
<link rel="icon" type="image/png" sizes="16x16" href="images/favicon-16x16.png">
<link rel="manifest" href="images/site.webmanifest">
<link rel="mask-icon" href="images/safari-pinned-tab.svg" color="#5bbad5">
<link rel="shortcut icon" href="images/favicon.ico">
<meta name="msapplication-TileColor" content="#da532c">
<meta name="msapplication-config" content="images/browserconfig.xml">
<meta name="theme-color" content="#ffffff">
<!-- Global site tag (gtag.js) - Google Analytics -->
  <script async src="https://www.googletagmanager.com/gtag/js?id=G-EN2MTH2FCR"></script>
    <script>
    window.dataLayer = window.dataLayer || [];
    function gtag(){dataLayer.push(arguments);}
    gtag('js', new Date());

    gtag('config', 'G-EN2MTH2FCR');
    </script>



<style type="text/css">
/* Used with Pandoc 2.11+ new --citeproc when CSL is used */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<link rel="stylesheet" href="style/style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><strong><a href="./">Data integration in IBD</a></strong></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Preface</a></li>
<li class="chapter" data-level="" data-path="abstracts.html"><a href="abstracts.html"><i class="fa fa-check"></i>Abstracts</a>
<ul>
<li class="chapter" data-level="" data-path="abstracts.html"><a href="abstracts.html#english"><i class="fa fa-check"></i>English</a></li>
<li class="chapter" data-level="" data-path="abstracts.html"><a href="abstracts.html#spanish"><i class="fa fa-check"></i>Spanish</a></li>
<li class="chapter" data-level="" data-path="abstracts.html"><a href="abstracts.html#catalan"><i class="fa fa-check"></i>Catalan</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="glossary.html"><a href="glossary.html"><i class="fa fa-check"></i>Glossary</a></li>
<li class="chapter" data-level="1" data-path="introduction.html"><a href="introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a>
<ul>
<li class="chapter" data-level="1.1" data-path="introduction.html"><a href="introduction.html#IBD"><i class="fa fa-check"></i><b>1.1</b> Inflammatory bowel disease</a>
<ul>
<li class="chapter" data-level="1.1.1" data-path="introduction.html"><a href="introduction.html#etiology-and-pathogenesis"><i class="fa fa-check"></i><b>1.1.1</b> Etiology and pathogenesis</a></li>
<li class="chapter" data-level="1.1.2" data-path="introduction.html"><a href="introduction.html#CD"><i class="fa fa-check"></i><b>1.1.2</b> <span>CD</span> physiology</a></li>
<li class="chapter" data-level="1.1.3" data-path="introduction.html"><a href="introduction.html#UC"><i class="fa fa-check"></i><b>1.1.3</b> <span>UC</span> physiology</a></li>
<li class="chapter" data-level="1.1.4" data-path="introduction.html"><a href="introduction.html#treatment"><i class="fa fa-check"></i><b>1.1.4</b> Treatment</a></li>
<li class="chapter" data-level="1.1.5" data-path="introduction.html"><a href="introduction.html#summary-ibd"><i class="fa fa-check"></i><b>1.1.5</b> Summary</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="introduction.html"><a href="introduction.html#integration-studies-in"><i class="fa fa-check"></i><b>1.2</b> Integration studies in <span>IBD</span></a>
<ul>
<li class="chapter" data-level="1.2.1" data-path="introduction.html"><a href="introduction.html#data-origin"><i class="fa fa-check"></i><b>1.2.1</b> Type of data used for integration analysis</a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="introduction.html"><a href="introduction.html#integration"><i class="fa fa-check"></i><b>1.3</b> Integration</a>
<ul>
<li class="chapter" data-level="1.3.1" data-path="introduction.html"><a href="introduction.html#classification-of-integration-methods"><i class="fa fa-check"></i><b>1.3.1</b> Classification of integration methods</a></li>
<li class="chapter" data-level="1.3.2" data-path="introduction.html"><a href="introduction.html#interpretation"><i class="fa fa-check"></i><b>1.3.2</b> Interpretation</a></li>
<li class="chapter" data-level="1.3.3" data-path="introduction.html"><a href="introduction.html#reviews"><i class="fa fa-check"></i><b>1.3.3</b> Reviews</a></li>
<li class="chapter" data-level="1.3.4" data-path="introduction.html"><a href="introduction.html#methods-integration"><i class="fa fa-check"></i><b>1.3.4</b> Summary</a></li>
</ul></li>
<li class="chapter" data-level="1.4" data-path="introduction.html"><a href="introduction.html#summary"><i class="fa fa-check"></i><b>1.4</b> Summary</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="hypothesis-and-objectives.html"><a href="hypothesis-and-objectives.html"><i class="fa fa-check"></i><b>2</b> Hypothesis and objectives</a>
<ul>
<li class="chapter" data-level="2.1" data-path="hypothesis-and-objectives.html"><a href="hypothesis-and-objectives.html#hypothesis"><i class="fa fa-check"></i><b>2.1</b> Hypothesis</a></li>
<li class="chapter" data-level="2.2" data-path="hypothesis-and-objectives.html"><a href="hypothesis-and-objectives.html#objectives"><i class="fa fa-check"></i><b>2.2</b> Objectives</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="materials-and-methods.html"><a href="materials-and-methods.html"><i class="fa fa-check"></i><b>3</b> Materials and methods</a>
<ul>
<li class="chapter" data-level="3.1" data-path="materials-and-methods.html"><a href="materials-and-methods.html#datasets"><i class="fa fa-check"></i><b>3.1</b> Datasets</a>
<ul>
<li class="chapter" data-level="3.1.1" data-path="materials-and-methods.html"><a href="materials-and-methods.html#methods-puget"><i class="fa fa-check"></i><b>3.1.1</b> Puget’s dataset</a></li>
<li class="chapter" data-level="3.1.2" data-path="materials-and-methods.html"><a href="materials-and-methods.html#methods-hsct"><i class="fa fa-check"></i><b>3.1.2</b> HSCT dataset</a></li>
<li class="chapter" data-level="3.1.3" data-path="materials-and-methods.html"><a href="materials-and-methods.html#methods-hasler"><i class="fa fa-check"></i><b>3.1.3</b> Häsler’s dataset</a></li>
<li class="chapter" data-level="3.1.4" data-path="materials-and-methods.html"><a href="materials-and-methods.html#methods-morgan"><i class="fa fa-check"></i><b>3.1.4</b> Morgan’s dataset</a></li>
<li class="chapter" data-level="3.1.5" data-path="materials-and-methods.html"><a href="materials-and-methods.html#methods-howell"><i class="fa fa-check"></i><b>3.1.5</b> Howell’s dataset</a></li>
</ul></li>
<li class="chapter" data-level="3.2" data-path="materials-and-methods.html"><a href="materials-and-methods.html#processing"><i class="fa fa-check"></i><b>3.2</b> Sample processing</a>
<ul>
<li class="chapter" data-level="3.2.1" data-path="materials-and-methods.html"><a href="materials-and-methods.html#rna-sequencing"><i class="fa fa-check"></i><b>3.2.1</b> RNA sequencing</a></li>
<li class="chapter" data-level="3.2.2" data-path="materials-and-methods.html"><a href="materials-and-methods.html#microbial-dna-sequencing"><i class="fa fa-check"></i><b>3.2.2</b> Microbial DNA sequencing</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="materials-and-methods.html"><a href="materials-and-methods.html#integration-methods"><i class="fa fa-check"></i><b>3.3</b> Integration methods</a>
<ul>
<li class="chapter" data-level="3.3.1" data-path="materials-and-methods.html"><a href="materials-and-methods.html#regularized-generalized-canonical-correlation-analysis"><i class="fa fa-check"></i><b>3.3.1</b> Regularized generalized canonical correlation analysis</a></li>
<li class="chapter" data-level="3.3.2" data-path="materials-and-methods.html"><a href="materials-and-methods.html#other"><i class="fa fa-check"></i><b>3.3.2</b> Other</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="materials-and-methods.html"><a href="materials-and-methods.html#functional-enrichment-methods"><i class="fa fa-check"></i><b>3.4</b> Functional enrichment methods</a>
<ul>
<li class="chapter" data-level="3.4.1" data-path="materials-and-methods.html"><a href="materials-and-methods.html#ORA"><i class="fa fa-check"></i><b>3.4.1</b> Over representation analysis</a></li>
<li class="chapter" data-level="3.4.2" data-path="materials-and-methods.html"><a href="materials-and-methods.html#gene-set-enrichment-analysis"><i class="fa fa-check"></i><b>3.4.2</b> Gene Set Enrichment Analysis</a></li>
</ul></li>
<li class="chapter" data-level="3.5" data-path="materials-and-methods.html"><a href="materials-and-methods.html#variance"><i class="fa fa-check"></i><b>3.5</b> Variance and diversity methods</a>
<ul>
<li class="chapter" data-level="3.5.1" data-path="materials-and-methods.html"><a href="materials-and-methods.html#permanova"><i class="fa fa-check"></i><b>3.5.1</b> PERMANOVA</a></li>
<li class="chapter" data-level="3.5.2" data-path="materials-and-methods.html"><a href="materials-and-methods.html#globaltest"><i class="fa fa-check"></i><b>3.5.2</b> globaltest</a></li>
<li class="chapter" data-level="3.5.3" data-path="materials-and-methods.html"><a href="materials-and-methods.html#diversity-indices"><i class="fa fa-check"></i><b>3.5.3</b> Diversity indices</a></li>
</ul></li>
<li class="chapter" data-level="3.6" data-path="materials-and-methods.html"><a href="materials-and-methods.html#other-methods"><i class="fa fa-check"></i><b>3.6</b> Other methods</a>
<ul>
<li class="chapter" data-level="3.6.1" data-path="materials-and-methods.html"><a href="materials-and-methods.html#statistics"><i class="fa fa-check"></i><b>3.6.1</b> Statistics</a></li>
<li class="chapter" data-level="3.6.2" data-path="materials-and-methods.html"><a href="materials-and-methods.html#wgcna"><i class="fa fa-check"></i><b>3.6.2</b> WGCNA</a></li>
<li class="chapter" data-level="3.6.3" data-path="materials-and-methods.html"><a href="materials-and-methods.html#baseset"><i class="fa fa-check"></i><b>3.6.3</b> BaseSet</a></li>
<li class="chapter" data-level="3.6.4" data-path="materials-and-methods.html"><a href="materials-and-methods.html#experdesign"><i class="fa fa-check"></i><b>3.6.4</b> experDesign</a></li>
<li class="chapter" data-level="3.6.5" data-path="materials-and-methods.html"><a href="materials-and-methods.html#ROC"><i class="fa fa-check"></i><b>3.6.5</b> ROC- AUC</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="results.html"><a href="results.html"><i class="fa fa-check"></i><b>4</b> Results</a>
<ul>
<li class="chapter" data-level="4.1" data-path="results.html"><a href="results.html#packagesmethods"><i class="fa fa-check"></i><b>4.1</b> Packages/methods</a>
<ul>
<li class="chapter" data-level="4.1.1" data-path="results.html"><a href="results.html#experdesign-1"><i class="fa fa-check"></i><b>4.1.1</b> experDesign</a></li>
<li class="chapter" data-level="4.1.2" data-path="results.html"><a href="results.html#baseset-1"><i class="fa fa-check"></i><b>4.1.2</b> BaseSet</a></li>
<li class="chapter" data-level="4.1.3" data-path="results.html"><a href="results.html#intermodel"><i class="fa fa-check"></i><b>4.1.3</b> inteRmodel</a></li>
</ul></li>
<li class="chapter" data-level="4.2" data-path="results.html"><a href="results.html#analysis"><i class="fa fa-check"></i><b>4.2</b> Analysis</a>
<ul>
<li class="chapter" data-level="4.2.1" data-path="results.html"><a href="results.html#results-puget"><i class="fa fa-check"></i><b>4.2.1</b> Puget’s dataset</a></li>
<li class="chapter" data-level="4.2.2" data-path="results.html"><a href="results.html#results-hsct"><i class="fa fa-check"></i><b>4.2.2</b> HSCT dataset</a></li>
<li class="chapter" data-level="4.2.3" data-path="results.html"><a href="results.html#results-hasler"><i class="fa fa-check"></i><b>4.2.3</b> Häsler’s dataset</a></li>
<li class="chapter" data-level="4.2.4" data-path="results.html"><a href="results.html#results-morgan"><i class="fa fa-check"></i><b>4.2.4</b> Morgan’s dataset</a></li>
<li class="chapter" data-level="4.2.5" data-path="results.html"><a href="results.html#results-howell"><i class="fa fa-check"></i><b>4.2.5</b> Howell’s dataset</a></li>
<li class="chapter" data-level="4.2.6" data-path="results.html"><a href="results.html#between-datasets"><i class="fa fa-check"></i><b>4.2.6</b> Between datasets</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="discussion.html"><a href="discussion.html"><i class="fa fa-check"></i><b>5</b> Discussion</a>
<ul>
<li class="chapter" data-level="5.1" data-path="discussion.html"><a href="discussion.html#preliminary-steps"><i class="fa fa-check"></i><b>5.1</b> Preliminary steps</a>
<ul>
<li class="chapter" data-level="5.1.1" data-path="discussion.html"><a href="discussion.html#discussion-datasets"><i class="fa fa-check"></i><b>5.1.1</b> The datasets</a></li>
<li class="chapter" data-level="5.1.2" data-path="discussion.html"><a href="discussion.html#the-methods"><i class="fa fa-check"></i><b>5.1.2</b> The methods</a></li>
</ul></li>
<li class="chapter" data-level="5.2" data-path="discussion.html"><a href="discussion.html#designing-models"><i class="fa fa-check"></i><b>5.2</b> Designing models</a></li>
<li class="chapter" data-level="5.3" data-path="discussion.html"><a href="discussion.html#evaluating-models"><i class="fa fa-check"></i><b>5.3</b> Evaluating models</a></li>
<li class="chapter" data-level="5.4" data-path="discussion.html"><a href="discussion.html#implications"><i class="fa fa-check"></i><b>5.4</b> Implications</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="conclusions.html"><a href="conclusions.html"><i class="fa fa-check"></i><b>6</b> Conclusions</a>
<ul>
<li class="chapter" data-level="6.1" data-path="conclusions.html"><a href="conclusions.html#study-1-multi-omic-modelling-of-inflammatory-bowel-disease-with-regularized-canonical-correlation-analysis"><i class="fa fa-check"></i><b>6.1</b> Study 1: Multi-omic modelling of inflammatory bowel disease with regularized canonical correlation analysis</a></li>
<li class="chapter" data-level="6.2" data-path="conclusions.html"><a href="conclusions.html#study-2-the-relationship-of-genes-and-the-microbiome-with-inflammatory-bowel-disease"><i class="fa fa-check"></i><b>6.2</b> Study 2: The relationship of genes and the microbiome with inflammatory bowel disease</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="acknowledgments.html"><a href="acknowledgments.html"><i class="fa fa-check"></i><b>7</b> Acknowledgments</a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="appendix"><span><b>Appendix</b></span></li>
<li class="chapter" data-level="A" data-path="online-resources.html"><a href="online-resources.html"><i class="fa fa-check"></i><b>A</b> Online resources</a></li>
<li class="chapter" data-level="B" data-path="software.html"><a href="software.html"><i class="fa fa-check"></i><b>B</b> Software</a>
<ul>
<li class="chapter" data-level="B.1" data-path="software.html"><a href="software.html#star"><i class="fa fa-check"></i><b>B.1</b> STAR</a></li>
<li class="chapter" data-level="B.2" data-path="software.html"><a href="software.html#rsem"><i class="fa fa-check"></i><b>B.2</b> RSEM</a></li>
<li class="chapter" data-level="B.3" data-path="software.html"><a href="software.html#listed"><i class="fa fa-check"></i><b>B.3</b> Listed</a></li>
<li class="chapter" data-level="B.4" data-path="software.html"><a href="software.html#by-projectpublication"><i class="fa fa-check"></i><b>B.4</b> By project/publication</a></li>
</ul></li>
<li class="chapter" data-level="C" data-path="articles.html"><a href="articles.html"><i class="fa fa-check"></i><b>C</b> Articles</a>
<ul>
<li class="chapter" data-level="C.1" data-path="articles.html"><a href="articles.html#app-multiomics"><i class="fa fa-check"></i><b>C.1</b> Multi-omic modelling of inflammatory bowel disease with regularized canonical correlation analysis</a></li>
<li class="chapter" data-level="C.2" data-path="articles.html"><a href="articles.html#app-experDesign"><i class="fa fa-check"></i><b>C.2</b> experDesign: stratifying samples into batches with minimal bias</a></li>
<li class="chapter" data-level="C.3" data-path="articles.html"><a href="articles.html#app-upa"><i class="fa fa-check"></i><b>C.3</b> Randomized Controlled Trial Substudy of Cell-specific Mechanisms of Janus Kinase 1 Inhibition With Upadacitinib in the Crohn’s Disease Intestinal Mucosa: Analysis From the CELEST Study</a></li>
</ul></li>
<li class="chapter" data-level="D" data-path="other-datasets.html"><a href="other-datasets.html"><i class="fa fa-check"></i><b>D</b> Other datasets</a>
<ul>
<li class="chapter" data-level="D.1" data-path="other-datasets.html"><a href="other-datasets.html#barcelona"><i class="fa fa-check"></i><b>D.1</b> BARCELONA dataset</a></li>
<li class="chapter" data-level="D.2" data-path="other-datasets.html"><a href="other-datasets.html#hernandez"><i class="fa fa-check"></i><b>D.2</b> Hernández’s dataset</a>
<ul>
<li class="chapter" data-level="D.2.1" data-path="other-datasets.html"><a href="other-datasets.html#results-hernandez"><i class="fa fa-check"></i><b>D.2.1</b> Results</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="E" data-path="models-output.html"><a href="models-output.html"><i class="fa fa-check"></i><b>E</b> Models output</a>
<ul>
<li class="chapter" data-level="E.1" data-path="models-output.html"><a href="models-output.html#hsct"><i class="fa fa-check"></i><b>E.1</b> HSCT</a>
<ul>
<li class="chapter" data-level="E.1.1" data-path="models-output.html"><a href="models-output.html#genes"><i class="fa fa-check"></i><b>E.1.1</b> Genes</a></li>
<li class="chapter" data-level="E.1.2" data-path="models-output.html"><a href="models-output.html#microbiome-2"><i class="fa fa-check"></i><b>E.1.2</b> Microbiome</a></li>
</ul></li>
<li class="chapter" data-level="E.2" data-path="models-output.html"><a href="models-output.html#häsler"><i class="fa fa-check"></i><b>E.2</b> Häsler</a>
<ul>
<li class="chapter" data-level="E.2.1" data-path="models-output.html"><a href="models-output.html#genes-1"><i class="fa fa-check"></i><b>E.2.1</b> Genes</a></li>
<li class="chapter" data-level="E.2.2" data-path="models-output.html"><a href="models-output.html#microbiome-3"><i class="fa fa-check"></i><b>E.2.2</b> Microbiome</a></li>
</ul></li>
<li class="chapter" data-level="E.3" data-path="models-output.html"><a href="models-output.html#morgan"><i class="fa fa-check"></i><b>E.3</b> Morgan</a>
<ul>
<li class="chapter" data-level="E.3.1" data-path="models-output.html"><a href="models-output.html#genes-2"><i class="fa fa-check"></i><b>E.3.1</b> Genes</a></li>
<li class="chapter" data-level="E.3.2" data-path="models-output.html"><a href="models-output.html#microbiome-4"><i class="fa fa-check"></i><b>E.3.2</b> Microbiome</a></li>
</ul></li>
<li class="chapter" data-level="E.4" data-path="models-output.html"><a href="models-output.html#howell"><i class="fa fa-check"></i><b>E.4</b> Howell</a>
<ul>
<li class="chapter" data-level="E.4.1" data-path="models-output.html"><a href="models-output.html#genes-3"><i class="fa fa-check"></i><b>E.4.1</b> Genes</a></li>
<li class="chapter" data-level="E.4.2" data-path="models-output.html"><a href="models-output.html#microbiome-5"><i class="fa fa-check"></i><b>E.4.2</b> Microbiome</a></li>
</ul></li>
</ul></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Data integration in inflammatory bowel disease</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div class = "print">
  If you want to download or save this thesis you can use <a href="https://thesis.llrs.dev/IBD_integration_Lluis_Revilla.pdf">the pdf</a> link or the <a href="https://thesis.llrs.dev/IBD_integration_Lluis_Revilla.epub">epub format</a> too.
  The thesis has many links to make it easier to reach external resources, if printed they will display as blue text.
  </div>
<div id="references" class="section level1 unnumbered hasAnchor">
<h1>References<a href="references.html#references" class="anchor-section" aria-label="Anchor link to header"></a></h1>
<div id="refs" class="references csl-bib-body">
<div class="csl-entry">
1. Raine T, Verstockt B, Kopylov U, Karmiris K, Goldberg R, Atreya R, et al. <a href="https://doi.org/10.1093/ecco-jcc/jjab112">ECCO topical review: Refractory inflammatory bowel disease</a>. Journal of Crohn’s and Colitis. 2021;15:1605–20.
</div>
<div class="csl-entry">
2. Jairath V, Feagan BG. <a href="https://doi.org/10.1016/S2468-1253(19)30358-9">Global burden of inflammatory bowel disease</a>. The Lancet Gastroenterology &amp; Hepatology. 2020;5:2–3.
</div>
<div class="csl-entry">
3. Burisch J, Munkholm P. <a href="https://doi.org/10.3109/00365521.2015.1014407">The epidemiology of inflammatory bowel disease</a>. Scandinavian Journal of Gastroenterology. 2015;50:942–51.
</div>
<div class="csl-entry">
4. Human Microbiome Project Consortium BA, Nelson KE, Pop M, Creasy HH, Giglio MG, Huttenhower C, et al. <a href="https://doi.org/10.1038/nature11209">A framework for human microbiome research.</a> Nature. 2012;486:21521.
</div>
<div class="csl-entry">
5. Shaw KA, Bertha M, Hofmekler T, Chopra P, Vatanen T, Srivatsa A, et al. <a href="https://doi.org/10.1186/s13073-016-0331-y">Dysbiosis, inflammation, and response to treatment: A longitudinal study of pediatric subjects with newly diagnosed inflammatory bowel disease</a>. Genome Medicine. 2016;8:75.
</div>
<div class="csl-entry">
6. Mukhopadhya I, Hansen R, El-Omar EM, Hold GL. <a href="https://doi.org/10.1038/nrgastro.2012.14">IBD<span></span>what role do Proteobacteria play?</a> Nature Reviews Gastroenterology &amp; Hepatology. 2012;9:219–30.
</div>
<div class="csl-entry">
7. Zhang Y-Z, Li Y-Y. <a href="https://doi.org/10.3748/wjg.v20.i1.91">Inflammatory bowel disease: pathogenesis</a>. World Journal of Gastroenterology. 2014;20:91–9.
</div>
<div class="csl-entry">
8. Hugot J-P. <a href="https://doi.org/10.1097/00054725-200402001-00003">Genetic origin of IBD</a>. Inflammatory Bowel Diseases. 2004;10:S11–5.
</div>
<div class="csl-entry">
9. Silva FAR, Rodrigues BL, Ayrizono M de LS, Leal RF. <a href="https://doi.org/10.1155/2016/2097274">The Immunological Basis of Inflammatory Bowel Disease</a>. Gastroenterology Research and Practice. 2016;2016:2097274.
</div>
<div class="csl-entry">
10. de Mattos BRR, Garcia MPG, Nogueira JB, Paiatto LN, Albuquerque CG, Souza CL, et al. <a href="https://doi.org/10.1155/2015/493012">Inflammatory Bowel Disease: An Overview of Immune Mechanisms and Biological Treatments</a>. Mediators of Inflammation. 2015;2015:493012.
</div>
<div class="csl-entry">
11. McGovern DPB, Kugathasan S, Cho JH. <a href="https://doi.org/10.1053/j.gastro.2015.08.001">Genetics of Inflammatory Bowel Diseases</a>. Gastroenterology. 2015;149:1163–1176.e2.
</div>
<div class="csl-entry">
12. Satsangi J, Silverberg MS, Vermeire S, Colombel J-F. <a href="https://doi.org/10.1136/gut.2005.082909">The Montreal classification of inflammatory bowel disease: controversies, consensus, and implications</a>. Gut. 2006;55:749–53.
</div>
<div class="csl-entry">
13. Horowitz JE, Warner N, Staples J, Crowley E, Gosalia N, Murchie R, et al. <a href="https://doi.org/10.1038/s41598-021-84938-8">Mutation spectrum of NOD2 reveals recessive inheritance as a main driver of early onset crohn<span>’</span>s disease</a>. Scientific Reports. 2021;11:5595.
</div>
<div class="csl-entry">
14. Kumar M, Garand M, Al Khodor S. <a href="https://doi.org/10.1186/s12967-019-02174-1">Integrating omics for a better understanding of inflammatory bowel disease: A step towards personalized medicine</a>. Journal of Translational Medicine. 2019;17:419.
</div>
<div class="csl-entry">
15. Jostins L, Ripke S, Weersma RK, Duerr RH, McGovern DP, Hui KY, et al. <a href="https://doi.org/10.1038/nature11582">Host<span></span>microbe interactions have shaped the genetic architecture of inflammatory bowel disease</a>. Nature. 2012;491:119–24.
</div>
<div class="csl-entry">
16. Momozawa Y, Dmitrieva J, Théâtre E, Deffontaine V, Rahmouni S, Charloteaux B, et al. <a href="https://doi.org/10.1038/s41467-018-04365-8">IBD risk loci are enriched in multigenic regulatory modules encompassing putative causative genes</a>. Nature Communications. 2018;9.
</div>
<div class="csl-entry">
17. Khanna S, Tosh PK. <a href="https://doi.org/10.1016/j.mayocp.2013.10.011">A Clinician’s Primer on the Role of the Microbiome in Human Health and Disease</a>. Mayo Clinic Proceedings. 2014;89:107–14.
</div>
<div class="csl-entry">
18. Swidsinski A, Ladhoff A, Pernthaler A, Swidsinski S, LoeningBaucke V, Ortner M, et al. <a href="https://doi.org/10.1053/gast.2002.30294">Mucosal flora in inflammatory bowel disease</a>. Gastroenterology. 2002;122:44–54.
</div>
<div class="csl-entry">
19. Tamboli CP, Neut C, Desreumaux P, Colombel JF. <a href="https://doi.org/10.1136/gut.53.1.1">Dysbiosis in inflammatory bowel disease</a>. Gut. 2004;53:1–4.
</div>
<div class="csl-entry">
20. Ott SJ. <a href="https://doi.org/10.1136/gut.2003.025403">Reduction in diversity of the colonic mucosa associated bacterial microflora in patients with active inflammatory bowel disease</a>. Gut. 2004;53:685–93.
</div>
<div class="csl-entry">
21. Kostic AD, Xavier RJ, Gevers D. <a href="https://doi.org/10.1053/j.gastro.2014.02.009">The microbiome in inflammatory bowel disease: Current status and the future ahead</a>. Gastroenterology. 2014;146:1489–99.
</div>
<div class="csl-entry">
22. Sender R, Fuchs S, Milo R. <a href="https://doi.org/10.1371/journal.pbio.1002533">Revised Estimates for the Number of Human and Bacteria Cells in the Body</a>. PLOS Biology. 2016;14:e1002533.
</div>
<div class="csl-entry">
23. Sankarasubramanian J, Ahmad R, Avuthu N, Singh AB, Guda C. <a href="https://doi.org/10.3389/fmed.2020.606298">Gut microbiota and metabolic specificity in ulcerative colitis and crohn’s disease</a>. Frontiers in Medicine. 2020;7.
</div>
<div class="csl-entry">
24. Lopez-Siles M, Martinez-Medina M, Busquets D, Sabat-Mir M, Duncan SH, Flint HJ, et al. <a href="https://doi.org/10.1016/j.ijmm.2014.02.009">Mucosa-associated Faecalibacterium prausnitzii and Escherichia coli co-abundance can distinguish Irritable Bowel Syndrome and Inflammatory Bowel Disease phenotypes</a>. International journal of medical microbiology: IJMM. 2014;304:464–75.
</div>
<div class="csl-entry">
25. Ferrer-Picón E, Dotti I, Corraliza AM, Mayorgas A, Esteller M, Perales JC, et al. <a href="https://doi.org/10.1093/ibd/izz119">Intestinal Inflammation Modulates the Epithelial Response to Butyrate in Patients With Inflammatory Bowel Disease</a>. Inflammatory Bowel Diseases. 2020;26:43–55.
</div>
<div class="csl-entry">
26. Darfeuille-Michaud A, Neut C, Barnich N, Lederman E, Di Martino P, Desreumaux P, et al. <a href="https://doi.org/10.1016/S0016-5085(98)70019-8">Presence of adherent Escherichia coli strains in ileal mucosa of patients with Crohn’s disease</a>. Gastroenterology. 1998;115:1405–13.
</div>
<div class="csl-entry">
27. Tsilingiri K, Barbosa T, Penna G, Caprioli F, Sonzogni A, Viale G, et al. <a href="https://doi.org/10.1136/gutjnl-2011-300971">Probiotic and postbiotic activity in health and disease: comparison on a novel polarised ex-vivo organ culture model</a>. Gut. 2012;61:1007–15.
</div>
<div class="csl-entry">
28. Vanderpool C, Yan F, Polk BD. <a href="https://doi.org/10.1002/ibd.20525">Mechanisms of probiotic action: Implications for therapeutic applications in inflammatory bowel diseases</a>. Inflammatory Bowel Diseases. 2008;14:1585–96.
</div>
<div class="csl-entry">
29. Isaacs K, Herfarth H. <a href="https://doi.org/10.1002/ibd.20465">Role of probiotic therapy in IBD</a>. Inflammatory Bowel Diseases. 2008;14:1597–605.
</div>
<div class="csl-entry">
30. Plaza-Diaz J, Ruiz-Ojeda FJ, Gil-Campos M, Gil A. <a href="https://doi.org/10.1093/advances/nmy063">Mechanisms of action of probiotics</a>. Advances in Nutrition. 2019;10:S49–66.
</div>
<div class="csl-entry">
31. Morelli L, Capurso L. <a href="https://doi.org/10.1097/MCG.0b013e318269fdd5">FAO/WHO guidelines on probiotics: 10 years later</a>. Journal of Clinical Gastroenterology. 2012;46:S1.
</div>
<div class="csl-entry">
32. Okumura R, Takeda K. <a href="https://doi.org/10.1038/emm.2017.20">Roles of intestinal epithelial cells in the maintenance of gut homeostasis</a>. Experimental &amp; Molecular Medicine. 2017;49:e338–8.
</div>
<div class="csl-entry">
33. Faria AMC, Mucida D, McCafferty D-M, Tsuji NM, Verhasselt V. <a href="https://doi.org/10.1155/2012/738475">Tolerance and inflammation at the gut mucosa</a>. Clinical &amp; Developmental Immunology. 2012;2012:738475.
</div>
<div class="csl-entry">
34. Hisamatsu T, Kanai T, Mikami Y, Yoneno K, Matsuoka K, Hibi T. <a href="https://doi.org/10.1016/j.pharmthera.2012.10.008">Immune aspects of the pathogenesis of inflammatory bowel disease</a>. Pharmacology &amp; Therapeutics. 2013;137:283–97.
</div>
<div class="csl-entry">
35. Michielan A, D’Incà R. <a href="https://doi.org/10.1155/2015/628157">Intestinal Permeability in Inflammatory Bowel Disease: Pathogenesis, Clinical Evaluation, and Therapy of Leaky Gut</a>. Mediators of Inflammation. 2015;2015:628157.
</div>
<div class="csl-entry">
36. Mayorgas A. <a href="http://www.tdx.cat/handle/10803/672414">Human Primary Organoid-Derived Epithelial Monolayers as a Novel Strategy for the Study of Adherent Invasive Escherichia coli pathogenicity and the effects of Postbiotics on Intestinal Epithelial Function</a>. PhD thesis. 2021.
</div>
<div class="csl-entry">
37. Neurath MF, Fuss I, Kelsall BL, Presky DH, Waegell W, Strober W. <a href="https://doi.org/10.1084/jem.183.6.2605">Experimental granulomatous colitis in mice is abrogated by induction of TGF-beta-mediated oral tolerance.</a> Journal of Experimental Medicine. 1996;183:2605–16.
</div>
<div class="csl-entry">
38. Corraliza AM, Ricart E, López-García A, Carme Masamunt M, Veny M, Esteller M, et al. Differences in Peripheral and Tissue Immune Cell Populations Following Haematopoietic Stem Cell Transplantation in Crohn<span>’</span>s Disease Patients. Journal of Crohn’s and Colitis. <a href="https://doi.org/10.1093/ecco-jcc/jjy203">https://doi.org/10.1093/ecco-jcc/jjy203</a>.
</div>
<div class="csl-entry">
39. Strachan DP. <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1838109/">Hay fever, hygiene, and household size.</a> BMJ : British Medical Journal. 1989;299:1259–60.
</div>
<div class="csl-entry">
40. Scudellari M. <a href="https://doi.org/10.1073/pnas.1700688114">News feature: Cleaning up the hygiene hypothesis</a>. Proceedings of the National Academy of Sciences of the United States of America. 2017;114:1433–6.
</div>
<div class="csl-entry">
41. Thomas GAO, Rhodes J, Green JT. <a href="https://doi.org/10.1111/j.1572-0241.1998.00144.x">Inflammatory bowel disease and smoking<span></span>a review</a>. Official journal of the American College of Gastroenterology | ACG. 1998;93:144149.
</div>
<div class="csl-entry">
42. Cornish JA, Tan E, Simillis C, Clark SK, Teare J, Tekkis PP. <a href="https://doi.org/10.1111/j.1572-0241.2008.02064.x">The risk of oral contraceptives in the etiology of inflammatory bowel disease: a meta-analysis</a>. The American Journal of Gastroenterology. 2008;103:2394–400.
</div>
<div class="csl-entry">
43. Kaufmann HJ, Taubin HL. <a href="https://doi.org/10.7326/0003-4819-107-4-513">Nonsteroidal anti-inflammatory drugs activate quiescent inflammatory bowel disease</a>. Annals of Internal Medicine. 1987;107:513–6.
</div>
<div class="csl-entry">
44. Bitton A, Dobkin PL, Edwardes MD, Sewitch MJ, Meddings JB, Rawal S, et al. <a href="https://doi.org/10.1136/gut.2007.134817">Predicting relapse in Crohn’s disease: a biopsychosocial model</a>. Gut. 2008;57:1386–92.
</div>
<div class="csl-entry">
45. Baumgart DC, Sandborn WJ. <a href="https://doi.org/10.1016/S0140-6736(12)60026-9">Crohn’s disease</a>. The Lancet. 2012;380:1590–605.
</div>
<div class="csl-entry">
46. Khanna S, Shin A, Kelly CP. <a href="https://doi.org/10.1016/j.cgh.2016.10.024">Management of clostridium difficile infection in inflammatory bowel disease: Expert review from the clinical practice updates committee of the AGA institute</a>. Clinical Gastroenterology and Hepatology. 2017;15:166–74.
</div>
<div class="csl-entry">
47. Guardiola J, Lobatón T, Cerrillo E, Ferreiro-Iglesias R, Gisbert JP, Domènech E, et al. <a href="https://doi.org/10.1016/j.gastre.2018.05.018">Recommendations of the Spanish Working Group on Crohn’s Disease and Ulcerative Colitis (GETECCU) on the utility of the determination of faecal calprotectin in inflammatory bowel disease</a>. Gastroenterología y Hepatología (English Edition). 2018;41:514–29.
</div>
<div class="csl-entry">
48. Sands BE. <a href="https://doi.org/10.1053/j.gastro.2015.07.003">Biomarkers of Inflammation in Inflammatory Bowel Disease</a>. Gastroenterology. 2015;149:1275–1285.e2.
</div>
<div class="csl-entry">
49. Corraliza Márquez A. <a href="http://diposit.ub.edu/dspace/handle/2445/143300">Immune mechanisms involved in inducing remission in Crohn<span>’</span>s disease patients undergoing hematopoietic stem cell transplant</a>. PhD thesis. 2019.
</div>
<div class="csl-entry">
50. Bassolas Molina H. <a href="http://diposit.ub.edu/dspace/handle/2445/131017">Resposta T específica contra antígens de la microbiota comensal en la malaltia de Crohn i inhibició de ROR?t com a estratègia terapèutica</a>. PhD thesis. 2018.
</div>
<div class="csl-entry">
51. Daperno M, D’Haens G, Van Assche G, Baert F, Bulois P, Maunoury V, et al. <a href="https://doi.org/10.1016/S0016-5107(04)01878-4">Development and validation of a new, simplified endoscopic activity score for crohn’s disease: The SES-CD</a>. Gastrointestinal Endoscopy. 2004;60:505–12.
</div>
<div class="csl-entry">
52. Best WR, Becktel JM, Singleton JW, Kern F. <a href="https://doi.org/10.1016/S0016-5085(76)80163-1">Development of a Crohn’s Disease Activity Index: National Cooperative Crohn’s Disease Study</a>. Gastroenterology. 1976;70:439–44.
</div>
<div class="csl-entry">
53. Bhattacharya A, Rao BB, Koutroubakis IE, Click B, Vargas EJ, Regueiro M, et al. <a href="https://doi.org/10.1097/MIB.0000000000000935">Silent crohn’s disease predicts increased bowel damage during multiyear follow-up: The consequences of under-reporting active inflammation</a>. Inflammatory Bowel Diseases. 2016;22:2665–71.
</div>
<div class="csl-entry">
54. Peyrin-Biroulet L, Loftus EVJ, Colombel J-F, Sandborn WJ. <a href="https://doi.org/10.1038/ajg.2009.579">The natural history of adult crohn’s disease in population-based cohorts</a>. Official journal of the American College of Gastroenterology | ACG. 2010;105:289297.
</div>
<div class="csl-entry">
55. Etchevers MJ, Aceituno M, García-Bosch O, Ordás I, Sans M, Ricart E, et al. <a href="https://doi.org/10.1002/ibd.20897">Risk factors and characteristics of extent progression in ulcerative colitis</a>. Inflammatory Bowel Diseases. 2009;15:1320–5.
</div>
<div class="csl-entry">
56. Boonstra K, van Erpecum KJ, van Nieuwkerk KMJ, Drenth JPH, Poen AC, Witteman BJM, et al. <a href="https://doi.org/10.1002/ibd.22938">Primary sclerosing cholangitis is associated with a distinct phenotype of inflammatory bowel disease</a>. Inflammatory Bowel Diseases. 2012;18:2270–6.
</div>
<div class="csl-entry">
57. Mark-Christensen A, Laurberg S, Haboubi N. <a href="https://doi.org/10.1093/ibd/izy075">Dysplasia in Inflammatory Bowel Disease: Historical Review, Critical Histopathological Analysis, and Clinical Implications</a>. Inflammatory Bowel Diseases. 2018;24:1895–903.
</div>
<div class="csl-entry">
58. Schieffer KM, Williams ED, Yochum GS, Koltun WA. <a href="https://doi.org/10.1111/apt.13780">Review article: the pathogenesis of pouchitis</a>. Alimentary Pharmacology &amp; Therapeutics. 2016;44:817–35.
</div>
<div class="csl-entry">
59. Schroeder KW, Tremaine WJ, Ilstrup DM. <a href="https://doi.org/10.1056/nejm198712243172603">Coated Oral 5-Aminosalicylic Acid Therapy for Mildly to Moderately Active Ulcerative Colitis</a>. New England Journal of Medicine. 1987;317:1625–9.
</div>
<div class="csl-entry">
60. Irvine EJ. <a href="https://doi.org/10.1097/00005176-199904001-00003">Development and Subsequent Refinement of the Inflammatory Bowel Disease Questionnaire: A Quality-of-Life Instrument for Adult Patients with Inflammatory Bowel Disease</a>. Journal of Pediatric Gastroenterology &amp; Nutrition. 1999;28 Supplement:S23–7.
</div>
<div class="csl-entry">
61. Travis SPL, Schnell D, Krzeski P, Abreu MT, Altman DG, Colombel J-F, et al. <a href="https://doi.org/10.1136/gutjnl-2011-300486">Developing an instrument to assess the endoscopic severity of ulcerative colitis: The ulcerative colitis endoscopic index of severity (UCEIS)</a>. Gut. 2012;61:535–42.
</div>
<div class="csl-entry">
62. Travis SPL, Stange EF, Lémann M, Öresland T, Chowers Y, Forbes A, et al. <a href="https://doi.org/10.1136/gut.2005.081950b">European evidence based consensus on the diagnosis and management of Crohn<span>’</span>s disease: current management</a>. Gut. 2006;55 suppl 1:i16–35.
</div>
<div class="csl-entry">
63. Akobeng AK, Zhang D, Gordon M, MacDonald JK. Oral 5<span>-</span>aminosalicylic acid for maintenance of medically<span>-</span>induced remission in Crohn’s disease. Cochrane Database of Systematic Reviews. 2016. <a href="https://doi.org/10.1002/14651858.CD003715.pub3">https://doi.org/10.1002/14651858.CD003715.pub3</a>.
</div>
<div class="csl-entry">
64. Feller M, Huwiler K, Schoepfer A, Shang A, Furrer H, Egger M. <a href="https://doi.org/10.1086/649923">Long-term antibiotic treatment for crohn’s disease: Systematic review and meta-analysis of placebo-controlled trials</a>. Clinical Infectious Diseases. 2010;50:473–80.
</div>
<div class="csl-entry">
65. Prantera C, Scribano ML. <a href="https://doi.org/10.1097/MOG.0b013e32832b20bf">Antibiotics and probiotics in inflammatory bowel disease: Why, when, and how</a>. Current Opinion in Gastroenterology. 2009;25:329333.
</div>
<div class="csl-entry">
66. Rezaie A, Kuenzig ME, Benchimol EI, Griffiths AM, Otley AR, Steinhart AH, et al. Budesonide for induction of remission in Crohn’s disease. Cochrane Database of Systematic Reviews. 2015. <a href="https://doi.org/10.1002/14651858.CD000296.pub4">https://doi.org/10.1002/14651858.CD000296.pub4</a>.
</div>
<div class="csl-entry">
67. Lichtenstein GR, Hanauer SB, Sandborn WJ, Gastroenterology TPPC of the AC of. <a href="https://journals.lww.com/ajg/Abstract/2009/02000/Management_of_Crohn_s_Disease_in_Adults.32.aspx">Management of crohn’s disease in adults</a>. Official journal of the American College of Gastroenterology | ACG. 2009;104:465483.
</div>
<div class="csl-entry">
68. Ouellette AJ, Bevins CL. <a href="https://doi.org/10.1097/00054725-200102000-00007">Paneth cell defensins and innate immunity of the small bowel</a>. Inflammatory Bowel Diseases. 2001;7:43–50.
</div>
<div class="csl-entry">
69. Warner B, Johnston E, Arenas-Hernandez M, Marinaki A, Irving P, Sanderson J. <a href="https://doi.org/10.1136/flgastro-2016-100738">A practical guide to thiopurine prescribing and monitoring in IBD</a>. Frontline Gastroenterology. 2018;9:10–5.
</div>
<div class="csl-entry">
70. Chande N, Patton PH, Tsoulis DJ, Thomas BS, MacDonald JK. Azathioprine or 6<span>-</span>mercaptopurine for maintenance of remission in Crohn’s disease. Cochrane Database of Systematic Reviews. 2015. <a href="https://doi.org/10.1002/14651858.CD000067.pub3">https://doi.org/10.1002/14651858.CD000067.pub3</a>.
</div>
<div class="csl-entry">
71. Gisbert JP, Linares PM, Mcnicholl AG, Maté J, Gomollón F. <a href="https://doi.org/10.1111/j.1365-2036.2009.04023.x">Meta-analysis: the efficacy of azathioprine and mercaptopurine in ulcerative colitis</a>. Alimentary Pharmacology &amp; Therapeutics. 2009;30:126–37.
</div>
<div class="csl-entry">
72. Peyrin-Biroulet L, Lémann M. <a href="https://doi.org/10.1111/j.1365-2036.2011.04599.x">Review article: remission rates achievable by current therapies for inflammatory bowel disease</a>. Alimentary Pharmacology &amp; Therapeutics. 2011;33:870–9.
</div>
<div class="csl-entry">
73. Billioud V, Sandborn WJ, Peyrin-Biroulet L. <a href="https://doi.org/10.1038/ajg.2011.60">Loss of response and need for adalimumab dose intensification in crohn’s disease: A systematic review</a>. Official journal of the American College of Gastroenterology | ACG. 2011;106:674684.
</div>
<div class="csl-entry">
74. Hwang JM, Varma MG. <a href="https://doi.org/10.3748/wjg.14.2678">Surgery for inflammatory bowel disease</a>. World Journal of Gastroenterology : WJG. 2008;14:2678–90.
</div>
<div class="csl-entry">
75. Gardiner KR, Dasari BVM. <a href="https://doi.org/10.1016/j.suc.2007.03.011">Operative Management of Small Bowel Crohn’s Disease</a>. Surgical Clinics of North America. 2007;87:587–610.
</div>
<div class="csl-entry">
76. Lewis RT, Maron DJ. <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2976865/">Efficacy and complications of surgery for crohn’s disease</a>. Gastroenterology &amp; Hepatology. 2010;6:587–96.
</div>
<div class="csl-entry">
77. Corraliza AM, Ricart E, López-García A, Carme Masamunt M, Veny M, Esteller M, et al. Differences in Peripheral and Tissue Immune Cell Populations Following Haematopoietic Stem Cell Transplantation in Crohn<span>’</span>s Disease Patients. Journal of Crohn’s and Colitis. <a href="https://doi.org/10.1093/ecco-jcc/jjy203">https://doi.org/10.1093/ecco-jcc/jjy203</a>.
</div>
<div class="csl-entry">
78. Weingarden AR, Vaughn BP. <a href="https://doi.org/10.1080/19490976.2017.1290757">Intestinal microbiota, fecal microbiota transplantation, and inflammatory bowel disease</a>. Gut Microbes. 2017;8:238–52.
</div>
<div class="csl-entry">
79. Beck LC, Granger CL, Masi AC, Stewart CJ. <a href="https://doi.org/10.1080/14789450.2021.1922278">Use of omic technologies in early life gastrointestinal health and disease: From bench to bedside</a>. Expert Review of Proteomics. 2021;18:247–59.
</div>
<div class="csl-entry">
80. Häsler R, Sheibani-Tezerji R, Sinha A, Barann M, Rehman A, Esser D, et al. <a href="https://doi.org/10.1136/gutjnl-2016-311651">Uncoupling of mucosal gene regulation, <span class="nocase">mRNA</span> splicing and adherent microbiota signatures in inflammatory bowel disease</a>. Gut. 2017;66:2087–97.
</div>
<div class="csl-entry">
81. Tang MS, Bowcutt R, Leung JM, Wolff MJ, Gundra UM, Hudesman D, et al. <a href="https://doi.org/10.1097/MIB.0000000000001208">Integrated Analysis of Biopsies from Inflammatory Bowel Disease Patients Identifies SAA1 as a Link Between Mucosal Microbes with TH17 and TH22 Cells</a>. Inflammatory Bowel Diseases. 2017;23:1544–54.
</div>
<div class="csl-entry">
82. Hernández-Rocha C, Borowski K, Turpin W, Filice M, Nayeri S, Raygoza Garay JA, et al. Integrative analysis of colonic biopsies from inflammatory bowel disease patients identifies an interaction between microbial bile-acid inducible gene abundance and human angiopoietin-like 4 gene expression. Journal of Crohn’s and Colitis. 2021. <a href="https://doi.org/10.1093/ecco-jcc/jjab096">https://doi.org/10.1093/ecco-jcc/jjab096</a>.
</div>
<div class="csl-entry">
83. Hu S, Vila AV, Gacesa R, Collij V, Stevens C, Fu JM, et al. <a href="https://doi.org/10.1136/gutjnl-2019-319706">Whole exome sequencing analyses reveal gene–microbiota interactions in the context of IBD</a>. Gut. 2021;70:285–96.
</div>
<div class="csl-entry">
84. Mayorgas A, Dotti I, Salas A. <a href="https://doi.org/10.1002/mnfr.202000188">Microbial Metabolites, Postbiotics, and Intestinal Epithelial Function</a>. Molecular Nutrition &amp; Food Research. 2021;65:2000188.
</div>
<div class="csl-entry">
85. Planell N, Lozano JJ, Mora-Buch R, Masamunt MC, Jimeno M, Ordás I, et al. <a href="https://doi.org/10.1136/gutjnl-2012-303333">Transcriptional analysis of the intestinal mucosa of patients with ulcerative colitis in remission reveals lasting epithelial cell alterations</a>. Gut. 2013;62:967–76.
</div>
<div class="csl-entry">
86. Leal RF, Planell N, Kajekar R, Lozano JJ, Ordás I, Dotti I, et al. <a href="https://doi.org/10.1136/gutjnl-2013-306518">Identification of inflammatory mediators in patients with Crohn’s disease unresponsive to anti-TNF? therapy</a>. Gut. 2015;64:233–42.
</div>
<div class="csl-entry">
87. Massimino L, Lamparelli LA, Houshyar Y, D’Alessio S, Peyrin-Biroulet L, Vetrano S, et al. <a href="https://doi.org/10.1038/s43588-021-00114-y">The Inflammatory Bowel Disease Transcriptome and Metatranscriptome Meta-Analysis (IBD TaMMA) framework</a>. Nature Computational Science. 2021;1:511–5.
</div>
<div class="csl-entry">
88. Knights D, Lassen KG, Xavier RJ. <a href="https://doi.org/10.1136/gutjnl-2012-303954">Advances in inflammatory bowel disease pathogenesis: Linking host genetics and the microbiome</a>. Gut. 2013;62.
</div>
<div class="csl-entry">
89. Repnik K, Potočnik U. <a href="https://doi.org/10.1007/s13353-015-0334-1">eQTL analysis links inflammatory bowel disease associated 1q21 locus to ECM1 gene</a>. Journal of Applied Genetics. 2016;57:363–72.
</div>
<div class="csl-entry">
90. Hu S, Uniken Venema WT, Westra H-J, Vich Vila A, Barbieri R, Voskuil MD, et al. <a href="https://doi.org/10.1038/s41467-021-21458-z">Inflammation status modulates the effect of host genetic variation on intestinal gene expression in inflammatory bowel disease</a>. Nature Communications. 2021;12:1122.
</div>
<div class="csl-entry">
91. Jung S, Liu W, Baek J, Moon JW, Ye BD, Lee H-S, et al. <a href="https://www.frontiersin.org/article/10.3389/fgene.2020.00486">Expression quantitative trait loci (eQTL) mapping in korean patients with crohn<span>’</span>s disease and identification of potential causal genes through integration with disease associations</a>. Frontiers in Genetics. 2020;11.
</div>
<div class="csl-entry">
92. Dai Y, Pei G, Zhao Z, Jia P. <a href="https://www.frontiersin.org/article/10.3389/fgene.2019.00318">A convergent study of genetic variants associated with crohn<span>’</span>s disease: Evidence from GWAS, gene expression, methylation, eQTL and TWAS</a>. Frontiers in Genetics. 2019;10.
</div>
<div class="csl-entry">
93. Ahmed I, Roy BC, Khan SA, Septer S, Umar S. <a href="https://doi.org/10.3390/microorganisms4020020">Microbiome, metabolome and inflammatory bowel disease</a>. Microorganisms. 2016;4:20.
</div>
<div class="csl-entry">
94. Gallagher K, Catesson A, Griffin JL, Holmes E, Williams HRT. <a href="https://doi.org/10.1093/ecco-jcc/jjaa227">Metabolomic Analysis in Inflammatory Bowel Disease: A Systematic Review</a>. Journal of Crohn’s &amp; Colitis. 2021;15:813–26.
</div>
<div class="csl-entry">
95. Krassowski M, Das V, Sahu SK, Misra BB. <a href="https://doi.org/10.3389/fgene.2020.610798">State of the field in multi-omics research: From computational needs to data mining and sharing</a>. Frontiers in Genetics. 2020;11:1598.
</div>
<div class="csl-entry">
96. Yannakoudakis H, Cummins R. <a href="https://doi.org/10.3115/v1/W15-0625">Evaluating the performance of automated text scoring systems</a>. Denver, Colorado: Association for Computational Linguistics; 2015. p. 213223.
</div>
<div class="csl-entry">
97. HOTELLING H. <a href="https://doi.org/10.1093/biomet/28.3-4.321">RELATIONS BETWEEN TWO SETS OF VARIATES</a>. Biometrika. 1936;28:321–77.
</div>
<div class="csl-entry">
98. Biancolillo A. <a href="http://models.life.ku.dk/sites/default/files/Thesis_Alessandra_Biancolillo.pdf">Method development in the area of multi-block analysis focused on food analysis</a>. PhD thesis.
</div>
<div class="csl-entry">
99. Wu C, Zhou F, Ren J, Li X, Jiang Y, Ma S. <a href="https://doi.org/10.3390/ht8010004">A <span>Selective Review</span> of <span>Multi</span>-<span>Level Omics Data Integration Using Variable Selection</span></a>. High-Throughput. 2019;8:4.
</div>
<div class="csl-entry">
100. Cavill R, Jennen D, Kleinjans J, Briedé JJ. <a href="https://doi.org/10.1093/bib/bbv090">Transcriptomic and metabolomic data integration</a>. Briefings in Bioinformatics. 2016;17:891–901.
</div>
<div class="csl-entry">
101. Chong J, Xia J. <a href="https://doi.org/10.3390/metabo7040062">Computational approaches for integrative analysis of the metabolome and microbiome</a>. Metabolites. 2017;7:62.
</div>
<div class="csl-entry">
102. Huang S, Chaudhary K, Garmire LX. <a href="https://doi.org/10.3389/fgene.2017.00084">More is better: Recent progress in multi-omics data integration methods</a>. Frontiers in Genetics. 2017;8.
</div>
<div class="csl-entry">
103. Rohart F, Mason EA, Matigian N, Mosbergen R, Korn O, Chen T, et al. <a href="https://doi.org/10.7717/peerj.1845">A molecular classification of human mesenchymal stromal cells</a>. PeerJ. 2016;4:e1845.
</div>
<div class="csl-entry">
104. Ibrahim EC, Guillemot V, Comte M, Tenenhaus A, Zendjidjian XY, Cancel A, et al. <a href="https://doi.org/10.1038/s41537-017-0027-3">Modeling a linkage between blood transcriptional expression and activity in brain regions to infer the phenotype of schizophrenia patients</a>. npj Schizophrenia. 2017;3:25.
</div>
<div class="csl-entry">
105. Wheeler HE, Aquino-Michaels K, Gamazon ER, Trubetskoy VV, Dolan ME, Huang RS, et al. <a href="https://doi.org/10.1002/gepi.21808">Poly-Omic Prediction of Complex Traits: OmicKriging</a>. Genetic Epidemiology. 2014;38:402–15.
</div>
<div class="csl-entry">
106. Yin L, Chau CKL, Sham P-C, So H-C. <a href="https://doi.org/10.1016/j.ajhg.2019.10.012">Integrating Clinical Data and Imputed Transcriptome from GWAS to Uncover Complex Disease Subtypes: Applications in Psychiatry and Cardiology</a>. The American Journal of Human Genetics. 2019;105:1193–212.
</div>
<div class="csl-entry">
107. Tarazona S, Arzalluz-Luque A, Conesa A. <a href="https://doi.org/10.1038/s43588-021-00086-z">Undisclosed, unmet and neglected challenges in multi-omics studies</a>. Nature Computational Science. 2021;1–8.
</div>
<div class="csl-entry">
108. Tarazona S, Balzano-Nogueira L, Gómez-Cabrero D, Schmidt A, Imhof A, Hankemeier T, et al. <a href="https://doi.org/10.1038/s41467-020-16937-8">Harmonization of quality metrics and power calculation in multi-omic studies</a>. Nature Communications. 2020;11:3092.
</div>
<div class="csl-entry">
109. Massoni-Badosa R, Iacono G, Moutinho C, Kulis M, Palau N, Marchese D, et al. <a href="https://doi.org/10.1186/s13059-020-02032-0">Sampling time-dependent artifacts in single-cell genomics studies</a>. Genome Biology. 2020;21:112.
</div>
<div class="csl-entry">
110. Zhu Y, Wang L, Yin Y, Yang E. <a href="https://doi.org/10.1038/s41598-017-05882-0">Systematic analysis of gene expression patterns associated with postmortem interval in human tissues</a>. Scientific Reports. 2017;7:5435.
</div>
<div class="csl-entry">
111. Ferreira PG, Muñoz-Aguirre M, Reverter F, Sá Godinho CP, Sousa A, Amadoz A, et al. <a href="https://doi.org/10.1038/s41467-017-02772-x">The effects of death and post-mortem cold ischemia on human tissue transcriptomes</a>. Nature Communications. 2018;9:490.
</div>
<div class="csl-entry">
112. Jacob F, Monod J. <a href="https://doi.org/10.1016/S0022-2836(61)80072-7">Genetic regulatory mechanisms in the synthesis of proteins</a>. Journal of Molecular Biology. 1961;3:318–56.
</div>
<div class="csl-entry">
113. Koh HWL, Fermin D, Vogel C, Choi KP, Ewing RM, Choi H. <a href="https://doi.org/10.1038/s41540-019-0099-y"><span class="nocase">iOmicsPASS</span>: Network-based integration of multiomics data for predictive subnetwork discovery</a>. npj Systems Biology and Applications. 2019;5:1–0.
</div>
<div class="csl-entry">
114. Yule GU. <a href="https://www.jstor.org/stable/92723">On the theory of correlation for any number of variables, treated by a new system of notation</a>. Proceedings of the Royal Society of London Series A, Containing Papers of a Mathematical and Physical Character. 1907;79:182–93.
</div>
<div class="csl-entry">
115. Tenenhaus A, Tenenhaus M. <a href="https://doi.org/10.1007/s11336-011-9206-8">Regularized <span>Generalized Canonical Correlation Analysis</span></a>. Psychometrika. 2011;76:257–84.
</div>
<div class="csl-entry">
116. Tenenhaus A, Philippe C, Guillemot V, Le Cao K-A, Grill J, Frouin V. <a href="https://doi.org/10.1093/biostatistics/kxu001">Variable selection for generalized canonical correlation analysis</a>. Biostatistics. 2014;15:569–83.
</div>
<div class="csl-entry">
117. Culhane AC, Perri‘ere G, Higgins DG. <a href="https://doi.org/10.1186/1471-2105-4-59">Cross-platform comparison and visualisation of gene expression data using co-inertia analysis</a>. BMC Bioinformatics. 2003;4:59.
</div>
<div class="csl-entry">
118. Vito RD, Bellio R, Trippa L, Parmigiani G. <a href="https://doi.org/10.1111/biom.12974">Multi-study factor analysis</a>. Biometrics. 2019;75:337–46.
</div>
<div class="csl-entry">
119. Argelaguet R, Velten B, Arnol D, Dietrich S, Zenz T, Marioni JC, et al. <a href="https://doi.org/10.15252/msb.20178124">Multi-<span>Omics Factor Analysis</span>a framework for unsupervised integration of multi-omics data sets</a>. Molecular Systems Biology. 2018;14:e8124.
</div>
<div class="csl-entry">
120. Gomez-Cabrero D, Tarazona S, Ferreirós-Vidal I, Ramirez RN, Company C, Schmidt A, et al. <a href="https://doi.org/10.1038/s41597-019-0202-7">STATegra, a comprehensive multi-omics dataset of B-cell differentiation in mouse</a>. Scientific Data. 2019;6:1–5.
</div>
<div class="csl-entry">
121. Franzosa EA, McIver LJ, Rahnavard G, Thompson LR, Schirmer M, Weingart G, et al. <a href="https://doi.org/10.1038/s41592-018-0176-y">Species-level functional profiling of metagenomes and metatranscriptomes</a>. Nature Methods. 2018;15:962.
</div>
<div class="csl-entry">
122. Truong DT, Franzosa EA, Tickle TL, Scholz M, Weingart G, Pasolli E, et al. <a href="https://doi.org/10.1038/nmeth.3589">MetaPhlAn2 for enhanced metagenomic taxonomic profiling</a>. Nature Methods. 2015;12:902–3.
</div>
<div class="csl-entry">
123. Segata N, Izard J, Waldron L, Gevers D, Miropolsky L, Garrett WS, et al. <a href="https://doi.org/10.1186/gb-2011-12-6-r60">Metagenomic biomarker discovery and explanation</a>. Genome Biology. 2011;12:R60.
</div>
<div class="csl-entry">
124. Didier G, Valdeolivas A, Baudot A. <a href="https://doi.org/10.12688/f1000research.15486.2">Identifying communities from multiplex biological networks by randomized optimization of modularity</a>. F1000Research. 2018;7.
</div>
<div class="csl-entry">
125. Valdeolivas A, Tichit L, Navarro C, Perrin S, Odelin G, Levy N, et al. <a href="https://doi.org/10.1093/bioinformatics/bty637">Random walk with restart on multiplex and heterogeneous biological networks</a>. Bioinformatics (Oxford, England). 2019;35:497–505.
</div>
<div class="csl-entry">
126. Pio-Lopez L, Valdeolivas A, Tichit L, Remy É, Baudot A. <a href="http://arxiv.org/abs/2008.10085">MultiVERSE: A multiplex and multiplex-heterogeneous network embedding approach</a>. arXiv:200810085 [cs, q-bio]. 2021.
</div>
<div class="csl-entry">
127. Bayes T, Price null. <a href="https://doi.org/10.1098/rstl.1763.0053">An essay towards solving a problem in the doctrine of chances. By the late rev. Mr. Bayes, f. R. s.</a> Philosophical Transactions of the Royal Society of London. 1763;53:370–418.
</div>
<div class="csl-entry">
128. Zhu J, Sova P, Xu Q, Dombek KM, Xu EY, Vu H, et al. <a href="https://doi.org/10.1371/journal.pbio.1001301">Stitching together Multiple Data Dimensions Reveals Interacting Metabolomic and Transcriptomic Networks That Modulate Cell Regulation</a>. PLOS Biology. 2012;10:e1001301.
</div>
<div class="csl-entry">
129. Lock EF, Dunson DB. <a href="https://doi.org/10.1093/bioinformatics/btt425">Bayesian consensus clustering</a>. Bioinformatics. 2013;29:2610–6.
</div>
<div class="csl-entry">
130. Cantini L, Zakeri P, Hernandez C, Naldi A, Thieffry D, Remy E, et al. <a href="https://doi.org/10.1038/s41467-020-20430-7">Benchmarking joint multi-omics dimensionality reduction approaches for the study of cancer</a>. Nature Communications. 2021;12:124.
</div>
<div class="csl-entry">
131. Tenenhaus M, Tenenhaus A, Groenen PJF. <a href="https://doi.org/10.1007/s11336-017-9573-x">Regularized Generalized Canonical Correlation Analysis: A Framework for Sequential Multiblock Component Methods</a>. Psychometrika. 2017;82:737–77.
</div>
<div class="csl-entry">
132. Rohart F, Gautier B, Singh A, Cao K-AL. <a href="https://doi.org/10.1371/journal.pcbi.1005752">mixOmics: An R package for <span>‘</span>omics feature selection and multiple data integration</a>. PLOS Computational Biology. 2017;13:e1005752.
</div>
<div class="csl-entry">
133. Virtanen S, Klami A, Khan S, Kaski S. <a href="https://proceedings.mlr.press/v22/virtanen12.html">Bayesian Group Factor Analysis</a>. PMLR; 2012. p. 1269–77.
</div>
<div class="csl-entry">
134. Novoa-del-Toro E-M, Mezura-Montes E, Vignes M, Magdinier F, Tichit L, Baudot A. <a href="https://doi.org/10.1101/2020.05.25.114215">A Multi-Objective Genetic Algorithm to Find Active Modules in Multiplex Biological Networks</a>. bioRxiv. 2020;2020.05.25.114215.
</div>
<div class="csl-entry">
135. Lock EF, Hoadley KA, Marron JS, Nobel AB. <a href="https://doi.org/10.1214/12-AOAS597">JOINT AND INDIVIDUAL VARIATION EXPLAINED (JIVE) FOR INTEGRATED ANALYSIS OF MULTIPLE DATA TYPES</a>. The annals of applied statistics. 2013;7:523–42.
</div>
<div class="csl-entry">
136. Yang Z, Michailidis G. <a href="https://doi.org/10.1093/bioinformatics/btv544">A non-negative matrix factorization method for detecting modules in heterogeneous omics multi-modal data</a>. Bioinformatics. 2016;32:1–8.
</div>
<div class="csl-entry">
137. Sherry A, Henson RK. <a href="https://doi.org/10.1207/s15327752jpa8401_09">Conducting and Interpreting Canonical Correlation Analysis in Personality Research: A User-Friendly Primer</a>. Journal of Personality Assessment. 2005;84:37–48.
</div>
<div class="csl-entry">
138. Sherry A, Henson RK. Conducting and <span>Interpreting Canonical Correlation Analysis</span> in <span>Personality Research</span>: <span>A User</span>-<span>Friendly Primer</span>. 1981.
</div>
<div class="csl-entry">
139. Chung R-H, Kang C-Y. <a href="https://doi.org/10.1093/gigascience/giz045">A multi-omics data simulator for complex disease studies and its application to evaluate multi-omics data analysis methods for disease classification</a>. GigaScience. 2019;8.
</div>
<div class="csl-entry">
140. Martínez-Mira C, Conesa A, Tarazona S. <a href="https://doi.org/10.1101/421834">MOSim: Multi-Omics Simulation in R</a>. 2018.
</div>
<div class="csl-entry">
141. Patuzzi I, Baruzzo G, Losasso C, Ricci A, Di Camillo B. <a href="https://doi.org/10.1186/s12859-019-2882-6">metaSPARSim: A 16S rRNA gene sequencing count data simulator</a>. BMC Bioinformatics. 2019;20:416.
</div>
<div class="csl-entry">
142. Fritz A, Hofmann P, Majda S, Dahms E, Dröge J, Fiedler J, et al. <a href="https://doi.org/10.1186/s40168-019-0633-6">CAMISIM: Simulating metagenomes and microbial communities</a>. Microbiome. 2019;7:17.
</div>
<div class="csl-entry">
143. Fu J, Frazee AC, Collado-Torres L, Jaffe AE, Leek JT. <a href="https://doi.org/10.18129/B9.bioc.ballgown">Ballgown: Flexible, isoform-level differential expression analysis</a>. Bioconductor version: Release (3.13); 2021.
</div>
<div class="csl-entry">
144. Frazee AC, Jaffe AE, Kirchner R, Leek JT. <a href="https://doi.org/10.18129/B9.bioc.polyester">Polyester: Simulate RNA-seq reads</a>. Bioconductor version: Release (3.13); 2021.
</div>
<div class="csl-entry">
145. McCarthy DJ, Chen Y, Smyth GK. <a href="https://doi.org/10.1093/nar/gks042">Differential expression analysis of multifactor RNA-Seq experiments with respect to biological variation</a>. Nucleic Acids Research. 2012;40:4288–97.
</div>
<div class="csl-entry">
146. De Souza HSP, Fiocchi C, Iliopoulos D. <a href="https://doi.org/10.1038/nrgastro.2017.110">The <span>IBD</span> interactome: <span>An</span> integrated view of aetiology, pathogenesis and therapy</a>. 2017;14.
</div>
<div class="csl-entry">
147. Valles-Colomer M, Darzi Y, Vieira-Silva S, Falony G, Raes J, Joossens M. <a href="https://doi.org/10.1093/ecco-jcc/jjw024">Meta-omics in inflammatory bowel disease research: Applications, challenges, and guidelines</a>. Journal of Crohn’s and Colitis. 2016;10:735–46.
</div>
<div class="csl-entry">
148. Sudhakar P, Alsoud D, Wellens J, Verstockt S, Arnauts K, Verstockt B, et al. <a href="https://doi.org/10.1093/ecco-jcc/jjac027">Tailoring multi-omics to inflammatory bowel diseases: All for one and one for all</a>. Journal of Crohn’s and Colitis. 2022;jjac027.
</div>
<div class="csl-entry">
149. Puget S, Philippe C, Bax DA, Job B, Varlet P, Junier M-P, et al. <a href="https://doi.org/10.1371/journal.pone.0030313">Mesenchymal Transition and PDGFRA Amplification/Mutation Are Key Distinct Oncogenic Events in Pediatric Diffuse Intrinsic Pontine Gliomas</a>. PLOS ONE. 2012;7:e30313.
</div>
<div class="csl-entry">
150. Morgan XC, Kabakchiev B, Waldron L, Tyler AD, Tickle TL, Milgrom R, et al. <a href="https://doi.org/10.1186/s13059-015-0637-x">Associations between host gene expression, the mucosal microbiome, and clinical outcome in the pelvic pouch of patients with inflammatory bowel disease</a>. Genome Biology. 2015;16:67.
</div>
<div class="csl-entry">
151. Howell KJ, Kraiczy J, Nayak KM, Gasparetto M, Ross A, Lee C, et al. <a href="https://doi.org/10.1053/j.gastro.2017.10.007">DNA Methylation and Transcription Patterns in Intestinal Epithelial Cells From Pediatric Patients With Inflammatory Bowel Diseases Differentiate Disease Subtypes and Associate With Outcome</a>. Gastroenterology. 2018;154:585–98.
</div>
<div class="csl-entry">
152. Martin M. <a href="https://doi.org/10.14806/ej.17.1.200">Cutadapt removes adapter sequences from high-throughput sequencing reads</a>. EMBnetjournal. 2011;17:10–2.
</div>
<div class="csl-entry">
153. Dobin A, Davis CA, Schlesinger F, Drenkow J, Zaleski C, Jha S, et al. <a href="https://doi.org/10.1093/bioinformatics/bts635">STAR: ultrafast universal RNA-seq aligner</a>. Bioinformatics (Oxford, England). 2013;29:15–21.
</div>
<div class="csl-entry">
154. Li B, Dewey CN. <a href="https://doi.org/10.1186/1471-2105-12-323">RSEM: Accurate transcript quantification from RNA-seq data with or without a reference genome</a>. BMC Bioinformatics. 2011;12:323.
</div>
<div class="csl-entry">
155. Corraliza AM, Ricart E, L’opez-García A, Carme Masamunt M, Veny M, Esteller M, et al. Differences in <span>Peripheral</span> and <span>Tissue Immune Cell Populations Following Haematopoietic Stem Cell Transplantation</span> in <span>Crohn</span>’s <span>Disease Patients</span>. Journal of Crohn’s and Colitis. <a href="https://doi.org/10.1093/ecco-jcc/jjy203">https://doi.org/10.1093/ecco-jcc/jjy203</a>.
</div>
<div class="csl-entry">
156. Berry D, Ben Mahfoudh K, Wagner M, Loy A. <a href="https://doi.org/10.1128/AEM.05220-11">Barcoded primers used in multiplex amplicon pyrosequencing bias amplification</a>. Applied and Environmental Microbiology. 2011;77:7846–9.
</div>
<div class="csl-entry">
157. Klindworth A, Pruesse E, Schweer T, Peplies J, Quast C, Horn M, et al. <a href="https://doi.org/10.1093/nar/gks808">Evaluation of general 16S ribosomal RNA gene PCR primers for classical and next-generation sequencing-based diversity studies</a>. Nucleic Acids Research. 2013;41:e1–1.
</div>
<div class="csl-entry">
158. Edgar RC. <a href="https://doi.org/10.1038/nmeth.2604">UPARSE: highly accurate OTU sequences from microbial amplicon reads</a>. Nature Methods. 2013;10:996–8.
</div>
<div class="csl-entry">
159. Lagkouvardos I, Joseph D, Kapfhammer M, Giritli S, Horn M, Haller D, et al. <a href="https://doi.org/10.1038/srep33721">IMNGS: A comprehensive open resource of processed 16S rRNA microbial profiles for ecology and diversity studies</a>. Scientific Reports. 2016;6.
</div>
<div class="csl-entry">
160. Callahan BJ, McMurdie PJ, Rosen MJ, Han AW, Johnson AJA, Holmes SP. <a href="https://doi.org/10.1038/nmeth.3869">DADA2: High-resolution sample inference from Illumina amplicon data</a>. Nature Methods. 2016;13:581–3.
</div>
<div class="csl-entry">
161. Quast C, Pruesse E, Yilmaz P, Gerken J, Schweer T, Yarza P, et al. <a href="https://doi.org/10.1093/nar/gks1219">The SILVA ribosomal RNA gene database project: Improved data processing and web-based tools</a>. Nucleic Acids Research. 2013;41 Database issue:D590–6.
</div>
<div class="csl-entry">
162. Jordan C. <a href="https://doi.org/10.24033/bsmf.90">Essai sur la géométrie à n dimensions</a>. Bulletin de la Société Mathématique de France. 1875;3:103–74.
</div>
<div class="csl-entry">
163. Tenenhaus M. <a href="https://doi.org/10.1080/14783360802159543">Component-based <span>Structural Equation Modelling</span></a>. Total Quality Management &amp; Business Excellence. 2008;19:871–86.
</div>
<div class="csl-entry">
164. Tenenhaus M, Hanafi M. <a href="https://doi.org/10.1007/978-3-540-32827-8_5">A <span>Bridge Between PLS Path Modeling</span> and <span>Multi</span>-<span>Block Data Analysis</span></a>. In: Esposito Vinzi V, Chin WW, Henseler J, Wang H, editors. Handbook of <span>Partial Least Squares</span>. Berlin, Heidelberg: <span>Springer Berlin Heidelberg</span>; 2010. p. 99–123.
</div>
<div class="csl-entry">
165. Tenenhaus A, Tenenhaus M. <a href="https://doi.org/10.1016/j.ejor.2014.01.008">Regularized generalized canonical correlation analysis for multiblock or multigroup data analysis</a>. European Journal of Operational Research. 2014;238:391–403.
</div>
<div class="csl-entry">
166. Tenenhaus A, Philippe C, Frouin V. <a href="https://doi.org/10.1016/j.csda.2015.04.004">Kernel <span>Generalized Canonical Correlation Analysis</span></a>. Computational Statistics &amp; Data Analysis. 2015;90:114–31.
</div>
<div class="csl-entry">
167. Gloaguen A, Philippe C, Frouin V, Gennari G, Dehaene-Lambertz G, Le Brusquet L, et al. Multiway generalized canonical correlation analysis. Biostatistics. 2020. <a href="https://doi.org/10.1093/biostatistics/kxaa010">https://doi.org/10.1093/biostatistics/kxaa010</a>.
</div>
<div class="csl-entry">
168. Schäfer J, Strimmer K. <a href="https://doi.org/10.2202/1544-6115.1175">A Shrinkage Approach to Large-Scale Covariance Matrix Estimation and Implications for Functional Genomics</a>. Statistical Applications in Genetics and Molecular Biology. 2005;4.
</div>
<div class="csl-entry">
169. Horst P. <a href="https://doi.org/10.1007/BF02289710">Relations among m sets of measures</a>. Psychometrika. 1961;26:129–49.
</div>
<div class="csl-entry">
170. KETTENRING JR. <a href="https://doi.org/10.1093/biomet/58.3.433">Canonical analysis of several sets of variables</a>. Biometrika. 1971;58:433–51.
</div>
<div class="csl-entry">
171. Van de Geer JP. <a href="https://doi.org/10.1007/BF02294207">Linear relations among k sets of variables</a>. Psychometrika. 1984;49:79–94.
</div>
<div class="csl-entry">
172. Meng C, Kuster B, Culhane AC, Gholami AM. <a href="https://doi.org/10.1186/1471-2105-15-162">A multivariate approach to the integration of multi-omics datasets</a>. BMC Bioinformatics. 2014;15:162.
</div>
<div class="csl-entry">
173. Planell N, Lagani V, Sebastian-Leon P, van der Kloet F, Ewing E, Karathanasis N, et al. <a href="https://doi.org/10.3389/fgene.2021.620453">STATegra: Multi-omics data integration <span></span> a conceptual scheme with a bioinformatics pipeline</a>. Frontiers in Genetics. 2021;12.
</div>
<div class="csl-entry">
174. Wu T, Hu E, Xu S, Chen M, Guo P, Dai Z, et al. <a href="https://doi.org/10.1016/j.xinn.2021.100141">clusterProfiler 4.0: A universal enrichment tool for interpreting omics data</a>. The Innovation. 2021;2:100141.
</div>
<div class="csl-entry">
175. Richter FC, Friedrich M, Pohin M, Alsaleh G, Guschina I, Wideman SK, et al. <a href="https://doi.org/10.1101/2021.10.25.465200">Cell-extrinsic autophagy in mature adipocytes regulates anti-inflammatory response to intestinal tissue injury through lipid mobilization</a>. 2021;2021.10.25.465200.
</div>
<div class="csl-entry">
176. Subramanian A, Tamayo P, Mootha VK, Mukherjee S, Ebert BL, Gillette MA, et al. <a href="https://doi.org/10.1073/pnas.0506580102">Gene set enrichment analysis: A knowledge-based approach for interpreting genome-wide expression profiles</a>. Proceedings of the National Academy of Sciences. 2005;102:15545–50.
</div>
<div class="csl-entry">
177. Protiva P, Pendyala S, Nelson C, Augenlicht LH, Lipkin M, Holt PR. <a href="https://doi.org/10.3945/ajcn.114.105304">Calcium and 1,25-dihydroxyvitamin D3 modulate genes of immune and inflammatory pathways in the human colon: A human crossover trial</a>. The American Journal of Clinical Nutrition. 2016;103:1224–31.
</div>
<div class="csl-entry">
178. Korotkevich G, Sukhov V, Budin N, Shpak B, Artyomov MN, Sergushichev A. Fast gene set enrichment analysis. 2021. https://doi.org/<a href="https://doi.org/10.1101/060012">https://doi.org/10.1101/060012</a>.
</div>
<div class="csl-entry">
179. Fabregat A, Sidiropoulos K, Garapati P, Gillespie M, Hausmann K, Haw R, et al. <a href="https://doi.org/10.1093/nar/gkv1351">The reactome pathway knowledgebase</a>. Nucleic Acids Research. 2016;44.
</div>
<div class="csl-entry">
180. Hänzelmann S, Castelo R, Guinney J. <a href="https://doi.org/10.1186/1471-2105-14-7">GSVA: Gene set variation analysis for microarray and RNA-seq data</a>. BMC Bioinformatics. 2013;14:7.
</div>
<div class="csl-entry">
181. Escudero-Hernández C, Beelen Granlund A van, Bruland T, Sandvik AK, Koch S, Østvik AE, et al. <a href="https://doi.org/10.1016/j.jcmgh.2021.04.011">Transcriptomic Profiling of Collagenous Colitis Identifies Hallmarks of Nondestructive Inflammatory Bowel Disease</a>. Cellular and Molecular Gastroenterology and Hepatology. 2021;12:665–87.
</div>
<div class="csl-entry">
182. Anderson MJ. <a href="https://doi.org/10.1111/j.1442-9993.2001.01070.pp.x">A new method for non-parametric multivariate analysis of variance</a>. Austral Ecology. 2001;26:32–46.
</div>
<div class="csl-entry">
183. Warton DI, Wright ST, Wang Y. <a href="https://doi.org/10.1111/j.2041-210X.2011.00127.x">Distance-based multivariate analyses confound location and dispersion effects</a>. Methods in Ecology and Evolution. 2012;3:89–101.
</div>
<div class="csl-entry">
184. Goeman JJ, van de Geer SA, van Houwelingen HC. <a href="https://www.jstor.org/stable/3879286">Testing against a high dimensional alternative</a>. Journal of the Royal Statistical Society Series B (Statistical Methodology). 2006;68:477–93.
</div>
<div class="csl-entry">
185. Oksanen J, Blanchet FG, Friendly M, Kindt R, Legendre P, McGlinn D, et al. <a href="https://CRAN.R-project.org/package=vegan">Vegan: Community ecology package</a>. 2020.
</div>
<div class="csl-entry">
186. Ritchie MEM, Phipson B, Wu D, Hu Y, Law CWC, Shi W, et al. <a href="https://doi.org/10.1093/nar/gkv007">Limma powers differential expression analyses for RNA-sequencing and microarray studies</a>. Nucleic acids research. 2015;43:e47.
</div>
<div class="csl-entry">
187. Law CW, Chen Y, Shi W, Smyth GK. <a href="https://doi.org/10.1186/gb-2014-15-2-r29">Voom: Precision weights unlock linear model analysis tools for RNA-seq read counts</a>. Genome Biology. 2014;15:R29.
</div>
<div class="csl-entry">
188. Law CW, Alhamdoosh M, Su S, Dong X, Tian L, Smyth GK, et al. <a href="https://doi.org/10.12688/f1000research.9005.3">RNA-seq analysis is easy as 1-2-3 with limma, Glimma and edgeR</a>. F1000Research. 2018;5.
</div>
<div class="csl-entry">
189. Yoav Benjamini, Yosef Hochberg. <a href="http://www.jstor.org/stable/2346101">Controlling the False Discovery Rate: A Practical and Powerful Approach to Multiple Testing</a>. Journal of the Royal Statistical Society Series B (Methodological). 57.
</div>
<div class="csl-entry">
190. Langfelder P, Horvath S. <a href="https://doi.org/10.1186/1471-2105-9-559">WGCNA: An r package for weighted correlation network analysis</a>. BMC Bioinformatics. 2008;9:559.
</div>
<div class="csl-entry">
191. Filzmoser P, Viertl R. <a href="https://doi.org/10.1007/s001840300269">Testing hypotheses with fuzzy data: The fuzzy p -value</a>. Metrika. 2004;59:21–9.
</div>
<div class="csl-entry">
192. Dubois D, Prade H. <a href="https://doi.org/10.1109/FUZZY.1993.327367">[Proceedings 1993] second IEEE international conference on fuzzy systems</a>. 1993. p. 1059–1068 vol.2.
</div>
<div class="csl-entry">
193. Revilla Sancho L, Lozano J-J, Salas A. <a href="https://doi.org/10.21105/joss.03358">experDesign: stratifying samples into batches with minimal bias</a>. Journal of Open Source Software. 2021;6:3358.
</div>
<div class="csl-entry">
194. Robin X, Turck N, Hainard A, Tiberti N, Lisacek F, Sanchez J-C, et al. <a href="https://doi.org/10.1186/1471-2105-12-77">pROC: An open-source package for r and s+ to analyze and compare ROC curves</a>. BMC Bioinformatics. 2011;12:77.
</div>
<div class="csl-entry">
195. Wickham H. <a href="https://adv-r.hadley.nz/index.html">Advanced r</a>. Second Edition. Boca Raton: Chapman &amp; Hall; 2019.
</div>
<div class="csl-entry">
196. Greenland S, Brumback B. <a href="https://doi.org/10.1093/ije/31.5.1030">An overview of relations among causal modelling methods</a>. International Journal of Epidemiology. 2002;31:1030–7.
</div>
<div class="csl-entry">
197. Revilla L, Mayorgas A, Corraliza AM, Masamunt MC, Metwaly A, Haller D, et al. <a href="https://doi.org/10.1371/journal.pone.0246367">Multi-omic modelling of inflammatory bowel disease with regularized canonical correlation analysis</a>. PLOS ONE. 2021;16:e0246367.
</div>
<div class="csl-entry">
198. Callahan BJ, McMurdie PJ, Holmes SP. <a href="https://doi.org/10.1038/ismej.2017.119">Exact sequence variants should replace operational taxonomic units in marker-gene data analysis</a>. The ISME Journal. 2017;11:2639–43.
</div>
<div class="csl-entry">
199. Nearing JT, Douglas GM, Hayes MG, MacDonald J, Desai DK, Allward N, et al. <a href="https://doi.org/10.1038/s41467-022-28034-z">Microbiome differential abundance methods produce different results across 38 datasets</a>. Nature Communications. 2022;13:342.
</div>
<div class="csl-entry">
200. Louca S, Doebeli M, Parfrey LW. <a href="https://doi.org/10.1186/s40168-018-0420-9">Correcting for 16S rRNA gene copy numbers in microbiome surveys remains an unsolved problem</a>. Microbiome. 2018;6:41.
</div>
<div class="csl-entry">
201. Nadalian B, Yadegar A, Houri H, Olfatifar M, Shahrokh S, Asadzadeh Aghdaei H, et al. <a href="https://doi.org/10.1111/jgh.15260">Prevalence of the pathobiont adherent-invasive Escherichia coli and inflammatory bowel disease: a systematic review and meta-analysis</a>. Journal of Gastroenterology and Hepatology. 2021;36:852–63.
</div>
<div class="csl-entry">
202. Chalise P, Raghavan R, Fridley BL. <a href="https://doi.org/10.1016/j.cmpb.2016.02.011">InterSIM: Simulation tool for multiple integrative <span>‘</span>omic datasets<span>’</span></a>. Computer methods and programs in biomedicine. 2016;128:69–74.
</div>
<div class="csl-entry">
203. Chung R-H, Kang C-Y. <a href="https://doi.org/10.1093/gigascience/giz045">A multi-omics data simulator for complex disease studies and its application to evaluate multi-omics data analysis methods for disease classification</a>. GigaScience. 2019;8.
</div>
<div class="csl-entry">
204. Ugidos M, Tarazona S, Prats-Montalbán JM, Ferrer A, Conesa A. MultiBaC: A strategy to remove batch effects between different omic data types: Statistical Methods in Medical Research. 2020. <a href="https://doi.org/10.1177/0962280220907365">https://doi.org/10.1177/0962280220907365</a>.
</div>
<div class="csl-entry">
205. Bersanelli M, Mosca E, Remondini D, Giampieri E, Sala C, Castellani G, et al. <a href="https://doi.org/10.1186/s12859-015-0857-9">Methods for the integration of multi-omics data: Mathematical aspects</a>. BMC Bioinformatics. 2016;17:S15.
</div>
<div class="csl-entry">
206. Holmes E, Li JV, Marchesi JR, Nicholson JK. <a href="https://doi.org/10.1016/j.cmet.2012.10.007">Gut Microbiota Composition and Activity in Relation to Host Metabolic Phenotype and Disease Risk</a>. Cell Metabolism. 2012;16:559–64.
</div>
<div class="csl-entry">
207. Stappenbeck TS, Hooper LV, Gordon JI. <a href="https://doi.org/10.1073/pnas.202604299">Developmental regulation of intestinal angiogenesis by indigenous microbes via Paneth cells</a>. Proceedings of the National Academy of Sciences of the United States of America. 2002;99:15451–5.
</div>
<div class="csl-entry">
208. Brand EC, Klaassen MAY, Gacesa R, Vich Vila A, Ghosh H, Zoete MR de, et al. <a href="https://doi.org/10.1053/j.gastro.2021.01.030">Healthy Cotwins Share Gut Microbiome Signatures With Their Inflammatory Bowel Disease Twins and Unrelated Patients</a>. Gastroenterology. 2021;160:1970–85.
</div>
<div class="csl-entry">
209. Susin A, Wang Y, Lê Cao K-A, Calle ML. <a href="https://doi.org/10.1093/nargab/lqaa029">Variable selection in microbiome compositional data analysis</a>. NAR Genomics and Bioinformatics. 2020;2:lqaa029.
</div>
<div class="csl-entry">
210. Sartor RB. <a href="https://doi.org/10.1038/ncpgasthep0528">Mechanisms of disease: pathogenesis of Crohn’s disease and ulcerative colitis</a>. Nature Clinical Practice Gastroenterology &amp; Hepatology. 2006;3:390–407.
</div>
<div class="csl-entry">
211. Vandeputte D, Kathagen G, D’hoe K, Vieira-Silva S, Valles-Colomer M, Sabino J, et al. <a href="https://doi.org/10.1038/nature24460">Quantitative microbiome profiling links gut community variation to microbial load</a>. Nature. 2017;551:507–11.
</div>
<div class="csl-entry">
212. Vila-Casadesús M, Gironella M, Lozano JJ. <a href="https://doi.org/10.1371/journal.pone.0151127">MiRComb: An R Package to Analyse miRNA-mRNA Interactions. Examples across Five Digestive Cancers</a>. PloS One. 2016;11:e0151127.
</div>
<div class="csl-entry">
213. Goldberg D. <a href="https://doi.org/10.1145/103162.103163">What every computer scientist should know about floating-point arithmetic</a>. ACM Computing Surveys. 1991;23:548.
</div>
<div class="csl-entry">
214. Yan L, Ma C, Wang D, Hu Q, Qin M, Conroy JM, et al. <a href="https://doi.org/10.1186/1471-2164-13-689">OSAT: A tool for sample-to-batch allocations in genomics experiments</a>. BMC Genomics. 2012;13:689.
</div>
<div class="csl-entry">
215. Papenberg M, Klau GW. Using anticlustering to partition data sets into equivalent parts. Psychological Methods. 2020. <a href="https://doi.org/10.1037/met0000301">https://doi.org/10.1037/met0000301</a>.
</div>
<div class="csl-entry">
216. Sinke L, Cats D, Heijmans BT. Omixer: Multivariate and reproducible sample randomization to proactively counter batch effects in omics studies. Bioinformatics. 2021. <a href="https://doi.org/10.1093/bioinformatics/btab159">https://doi.org/10.1093/bioinformatics/btab159</a>.
</div>
<div class="csl-entry">
217. Haberman Y, Tickle TL, Dexheimer PJ, Kim M-O, Tang D, Karns R, et al. <a href="https://doi.org/10.1172/JCI75436">Pediatric Crohn disease patients exhibit specific ileal transcriptome and microbiome signature</a>. The Journal of Clinical Investigation. 2014;124:3617–33.
</div>
<div class="csl-entry">
218. Community TTW. <a href="https://doi.org/10.5281/ZENODO.3233853">The turing way: A handbook for reproducible, ethical and collaborative research</a>. The Turing Way Community; 2021.
</div>
<div class="csl-entry">
219. Meng C, Kuster B, Culhane AC, Gholami AM. <a href="https://doi.org/10.1186/1471-2105-15-162">A multivariate approach to the integration of multi-omics datasets</a>. BMC Bioinformatics. 2014;15:162.
</div>
<div class="csl-entry">
220. Pearl J. <a href="https://escholarship.org/uc/item/3tv1b3bg">Transportability across studies: A formal approach</a>. 2011.
</div>
<div class="csl-entry">
221. Simpson EH. <a href="https://doi.org/10.1111/j.2517-6161.1951.tb00088.x">The Interpretation of Interaction in Contingency Tables</a>. Journal of the Royal Statistical Society: Series B (Methodological). 1951;13:238–41.
</div>
<div class="csl-entry">
222. Stillman DJ. Interactions of a Eukaryotic RNA Polymerase III Transcription Factor with Promoters. Biology; 1984.
</div>
<div class="csl-entry">
223. Lee RC, Feinbaum RL, Ambros V. <a href="https://doi.org/10.1016/0092-8674(93)90529-Y">The C. elegans heterochronic gene lin-4 encodes small RNAs with antisense complementarity to lin-14</a>. Cell. 1993;75:843–54.
</div>
<div class="csl-entry">
224. Hamilton AJ, Baulcombe DC. <a href="https://doi.org/10.1126/science.286.5441.950">A species of small antisense RNA in posttranscriptional gene silencing in plants</a>. Science. 1999;286:950–2.
</div>
<div class="csl-entry">
225. Thornburg ZR, Bianchi DM, Brier TA, Gilbert BR, Earnest TM, Melo MCR, et al. <a href="https://doi.org/10.1016/j.cell.2021.12.025">Fundamental behaviors emerge from simulations of a living minimal cell</a>. Cell. 2022;185:345–360.e28.
</div>
<div class="csl-entry">
226. Criss ZK, Bhasin N, Di Rienzi SC, Rajan A, Deans-Fielder K, Swaminathan G, et al. <a href="https://doi.org/10.1152/physiolgenomics.00061.2021">Drivers of transcriptional variance in human intestinal epithelial organoids</a>. Physiological Genomics. 2021;53:486–508.
</div>
<div class="csl-entry">
227. Arumugam M, Raes J, Pelletier E, Le Paslier D, Yamada T, Mende DR, et al. <a href="https://doi.org/10.1038/nature09944">Enterotypes of the human gut microbiome</a>. Nature. 2011;473:174–80.
</div>
<div class="csl-entry">
228. Koren O, Knights D, Gonzalez A, Waldron L, Segata N, Knight R, et al. <a href="https://doi.org/10.1371/journal.pcbi.1002863">A Guide to Enterotypes across the Human Body: Meta-Analysis of Microbial Community Structures in Human Microbiome Datasets</a>. PLOS Computational Biology. 2013;9:e1002863.
</div>
<div class="csl-entry">
229. Cheng M, Ning K. <a href="https://doi.org/10.1016/j.gpb.2018.02.004">Stereotypes About Enterotype: the Old and New Ideas</a>. Genomics, Proteomics &amp; Bioinformatics. 2019;17:4–12.
</div>
<div class="csl-entry">
230. Hasan N, Yang H. <a href="https://doi.org/10.7717/peerj.7502">Factors affecting the composition of the gut microbiota, and its modulation</a>. PeerJ. 2019;7:e7502.
</div>
<div class="csl-entry">
231. Bringiotti R, Ierardi E, Lovero R, Losurdo G, Leo AD, Principi M. <a href="https://doi.org/10.4291/wjgp.v5.i4.550">Intestinal microbiota: The explosive mixture at the origin of inflammatory bowel disease?</a> World Journal of Gastrointestinal Pathophysiology. 2014;5:550–9.
</div>
<div class="csl-entry">
232. Lloyd-Price J, Arze C, Ananthakrishnan AN, Schirmer M, Avila-Pacheco J, Poon TW, et al. <a href="https://doi.org/10.1038/s41586-019-1237-9">Multi-omics of the gut microbial ecosystem in inflammatory bowel diseases</a>. Nature. 2019;569:655.
</div>
<div class="csl-entry">
233. Douglas GM, Hansen R, Jones CMA, Dunn KA, Comeau AM, Bielawski JP, et al. <a href="https://doi.org/10.1186/s40168-018-0398-3">Multi-omics differentially classify disease state and treatment outcome in pediatric crohn<span>’</span>s disease</a>. Microbiome. 2018;6:13.
</div>
<div class="csl-entry">
234. Halfvarson J, Brislawn CJ, Lamendella R, Vázquez-Baeza Y, Walters WA, Bramer LM, et al. <a href="https://doi.org/10.1038/nmicrobiol.2017.4">Dynamics of the human gut microbiome in inflammatory bowel disease</a>. Nature Microbiology. 2017;2:1–7.
</div>
<div class="csl-entry">
235. Edgar R. <a href="https://doi.org/10.7717/peerj.5030">Taxonomy annotation and guide tree errors in 16S rRNA databases</a>. PeerJ. 2018;6:e5030.
</div>
<div class="csl-entry">
236. Ungar B, Yavzori M, Fudim E, Picard O, Kopylov U, Eliakim R, et al. Host transcriptome signatures in human faecal-washes predict histological remission in patients with IBD. Gut. 2022. <a href="https://doi.org/10.1136/gutjnl-2021-325516">https://doi.org/10.1136/gutjnl-2021-325516</a>.
</div>
<div class="csl-entry">
237. Zhang X, Li L, Butcher J, Stintzi A, Figeys D. <a href="https://doi.org/10.1186/s40168-019-0767-6">Advancing functional and translational microbiome research using meta-omics approaches</a>. Microbiome. 2019;7:154.
</div>
<div class="csl-entry">
238. Jiang D, Armour CR, Hu C, Mei M, Tian C, Sharpton TJ, et al. <a href="https://doi.org/10.3389/fgene.2019.00995">Microbiome multi-omics network analysis: Statistical considerations, limitations, and opportunities</a>. Frontiers in Genetics. 2019;10.
</div>
<div class="csl-entry">
239. Haynes WA, Tomczak A, Khatri P. <a href="https://doi.org/10.1038/s41598-018-19333-x">Gene annotation bias impedes biomedical research</a>. Scientific Reports. 2018;8:1362.
</div>
<div class="csl-entry">
240. Collij V, Klaassen MAY, Weersma RK, Vila AV. <a href="https://doi.org/10.1007/s00439-020-02218-3">Gut microbiota in inflammatory bowel diseases: moving from basic science to clinical applications</a>. Human Genetics. 2021;140:703–8.
</div>
<div class="csl-entry">
241. Parkhomenko E, Tritchler D, Beyene J. <a href="https://doi.org/10.2202/1544-6115.1406">Sparse Canonical Correlation Analysis with Application to Genomic Data Integration</a>. Statistical Applications in Genetics and Molecular Biology. 2009;8.
</div>
<div class="csl-entry">
242. Waaijenborg S, Hamer PCV de W, Zwinderman AH. <a href="https://doi.org/10.2202/1544-6115.1329">Quantifying the Association between Gene Expressions and DNA-Markers by Penalized Canonical Correlation Analysis</a>. Statistical Applications in Genetics and Molecular Biology. 2008;7.
</div>
<div class="csl-entry">
243. Witten DM, Tibshirani RJ. <a href="https://doi.org/10.2202/1544-6115.1470">Extensions of sparse canonical correlation analysis with applications to genomic data</a>. Statistical Applications in Genetics and Molecular Biology. 2009;8:127.
</div>
<div class="csl-entry">
244. Lê Cao K-A, Martin PG, Robert-Granié C, Besse P. <a href="https://doi.org/10.1186/1471-2105-10-34">Sparse canonical methods for biological data integration: Application to a cross-platform study</a>. BMC Bioinformatics. 2009;10:34.
</div>
<div class="csl-entry">
245. Hwang H. <a href="https://doi.org/10.1007/s11336-009-9119-y">Regularized Generalized Structured Component Analysis</a>. Psychometrika. 2009;74:517–30.
</div>
<div class="csl-entry">
246. Soneson C, Lilljebjörn H, Fioretos T, Fontes M. <a href="https://doi.org/10.1186/1471-2105-11-191">Integrative analysis of gene expression and copy number alterations using canonical correlation analysis</a>. BMC Bioinformatics. 2010;11:191.
</div>
<div class="csl-entry">
247. Zhang S, Li Q, Liu J, Zhou XJ. <a href="https://doi.org/10.1093/bioinformatics/btr206">A novel computational framework for simultaneous integration of multiple types of genomic data to identify microRNA-gene regulatory modules</a>. Bioinformatics. 2011;27:i401–9.
</div>
<div class="csl-entry">
248. Lee W, Lee D, Lee Y, Pawitan Y. <a href="https://doi.org/10.2202/1544-6115.1638">Sparse Canonical Covariance Analysis for High-throughput Data</a>. Statistical Applications in Genetics and Molecular Biology. 2011;10.
</div>
<div class="csl-entry">
249. Abdi H, Williams LJ, Valentin D, Bennani-Dosse M. <a href="https://doi.org/10.1002/wics.198">STATIS and DISTATIS: optimum multitable principal component analysis and three way metric multidimensional scaling</a>. WIREs Computational Statistics. 2012;4:124–67.
</div>
<div class="csl-entry">
250. Zhang S, Liu C-C, Li W, Shen H, Laird PW, Zhou XJ. <a href="https://doi.org/10.1093/nar/gks725">Discovery of multi-dimensional modules by integrative analysis of cancer genomic data</a>. Nucleic Acids Research. 2012;40:9379–91.
</div>
<div class="csl-entry">
251. Li W, Zhang S, Liu C-C, Zhou XJ. <a href="https://doi.org/10.1093/bioinformatics/bts476">Identifying multi-layer gene regulatory modules from multi-dimensional genomic data</a>. Bioinformatics. 2012;28:2458–66.
</div>
<div class="csl-entry">
252. Abdi H, Williams LJ, Valentin D. <a href="https://doi.org/10.1002/wics.1246">Multiple factor analysis: principal component analysis for multitable and multiblock data sets</a>. WIREs Computational Statistics. 2013;5:149–79.
</div>
<div class="csl-entry">
253. Schlauch D, Paulson JN, Young A, Glass K, Quackenbush J. <a href="https://doi.org/10.1093/bioinformatics/btx139">Estimating gene regulatory networks with pandaR</a>. Bioinformatics. 2017;33:2232–4.
</div>
<div class="csl-entry">
254. Ray P, Zheng L, Lucas J, Carin L. <a href="https://doi.org/10.1093/bioinformatics/btu064">Bayesian joint analysis of heterogeneous genomics data</a>. Bioinformatics. 2014;30:1370–6.
</div>
<div class="csl-entry">
255. Bunte K, Leppäaho E, Saarinen I, Kaski S. <a href="https://doi.org/10.1093/bioinformatics/btw207">Sparse group factor analysis for biclustering of multiple data sources</a>. Bioinformatics. 2016;32:2457–63.
</div>
<div class="csl-entry">
256. Chen M, Gao C, Ren Z, Zhou HH. <a href="http://arxiv.org/abs/1311.6186">Sparse CCA via precision adjusted iterative thresholding</a>. arXiv:13116186 [math, stat]. 2013.
</div>
<div class="csl-entry">
257. Leppäaho E, Ammad-ud-din M, Kaski S. <a href="http://jmlr.org/papers/v18/16-509.html">GFA: Exploratory analysis of multiple data sources with group factor analysis</a>. Journal of Machine Learning Research. 2017;18:1–5.
</div>
<div class="csl-entry">
258. Klami A, Bouchard G, Tripathi A. <a href="http://arxiv.org/abs/1312.5921">Group-sparse embeddings in collective matrix factorization</a>. arXiv:13125921 [cs, stat]. 2014.
</div>
<div class="csl-entry">
259. Meng C, Basunia A, Peters B, Gholami AM, Kuster B, Culhane AC. <a href="https://www.biorxiv.org/content/10.1101/046904v2">MOGSA: integrative single sample gene-set analysis of multiple omics data</a>. 2018.
</div>
<div class="csl-entry">
260. Zhao S, Gao C, Mukherjee S, Engelhardt BE. <a href="http://arxiv.org/abs/1411.2698">Bayesian group latent factor analysis with structured sparsity</a>. arXiv:14112698 [q-bio, stat]. 2015.
</div>
<div class="csl-entry">
261. Voillet V, Besse P, Liaubet L, San Cristobal M, González I. <a href="https://doi.org/10.1186/s12859-016-1273-5">Handling missing rows in multi-omics data integration: Multiple imputation in multiple factor analysis framework</a>. BMC Bioinformatics. 2016;17:402.
</div>
<div class="csl-entry">
262. Beaton D, Dunlop J, Abdi H, Alzheimer’s Disease Neuroimaging Initiative. <a href="https://doi.org/10.1037/met0000053">Partial least squares correspondence analysis: A framework to simultaneously analyze behavioral and genetic data</a>. Psychological Methods. 2016;21:621–51.
</div>
<div class="csl-entry">
263. Singh A, Shannon CP, Gautier B, Rohart F, Vacher M, Tebbutt SJ, et al. <a href="https://doi.org/10.1093/bioinformatics/bty1054">DIABLO: An integrative approach for identifying key molecular drivers from multi-omics assays</a>. Bioinformatics. 2019;35:3055–62.
</div>
<div class="csl-entry">
264. Yoon G, Carroll RJ, Gaynanova I. <a href="http://arxiv.org/abs/1807.05274">Sparse semiparametric canonical correlation analysis for data of mixed types</a>. arXiv:180705274 [stat]. 2019.
</div>
<div class="csl-entry">
265. Gaynanova I, Li G. <a href="http://arxiv.org/abs/1707.06573">Structural learning and integrative decomposition of multi-view data</a>. arXiv:170706573 [stat]. 2017.
</div>
<div class="csl-entry">
266. Madrigal P. <a href="https://doi.org/10.1093/bioinformatics/btw724">fCCAC: Functional canonical correlation analysis to evaluate covariance between nucleic acid sequencing datasets</a>. Bioinformatics. 2017;33:746–8.
</div>
<div class="csl-entry">
267. Yoshida K, Yoshimoto J, Doya K. <a href="https://doi.org/10.1186/s12859-017-1543-x">Sparse kernel canonical correlation analysis for discovery of nonlinear interactions in high-dimensional data</a>. BMC Bioinformatics. 2017;18:108.
</div>
<div class="csl-entry">
268. Kawaguchi A, Yamashita F. <a href="https://doi.org/10.1093/biostatistics/kxx011">Supervised multiblock sparse multivariable analysis with application to multimodal brain imaging genetics</a>. Biostatistics. 2017;18:651–65.
</div>
<div class="csl-entry">
269. Feng Q, Jiang M, Hannig J, Marron JS. <a href="http://arxiv.org/abs/1704.02060">Angle-based joint and individual variation explained</a>. arXiv:170402060 [stat]. 2018.
</div>
<div class="csl-entry">
270. Argelaguet R, Arnol D, Bredikhin D, Deloro Y, Velten B, Marioni JC, et al. <a href="https://doi.org/10.1186/s13059-020-02015-1">MOFA+: A statistical framework for comprehensive integration of multi-modal single-cell data</a>. Genome Biology. 2020;21:111.
</div>
<div class="csl-entry">
271. Brown BC, Bray NL, Pachter L. Expression reflects population structure. 2018. <a href="https://doi.org/10.1101/364448">https://doi.org/10.1101/364448</a>.
</div>
<div class="csl-entry">
272. Zhang Y, Gaynanova I. <a href="http://arxiv.org/abs/1811.08511">Joint association and classification analysis of multi-view data</a>. arXiv:181108511 [cs, stat]. 2020.
</div>
<div class="csl-entry">
273. Tang TM, Allen GI. <a href="http://arxiv.org/abs/1810.00832">Integrated principal components analysis</a>. arXiv:181000832 [stat]. 2021.
</div>
<div class="csl-entry">
274. Min EJ, Safo SE, Long Q. <a href="https://doi.org/10.1093/bioinformatics/bty726">Penalized co-inertia analysis with applications to -omics data</a>. Bioinformatics (Oxford, England). 2019;35:1018–25.
</div>
<div class="csl-entry">
275. Safo SE, Li S, Long Q. <a href="https://doi.org/10.1111/biom.12715">Integrative analysis of transcriptomic and metabolomic data via sparse canonical correlation analysis with incorporation of biological information</a>. Biometrics. 2018;74:300–12.
</div>
<div class="csl-entry">
276. Min W, Liu J, Zhang S. <a href="http://arxiv.org/abs/1710.04792">Sparse weighted canonical correlation analysis</a>. arXiv:171004792 [cs, stat]. 2017.
</div>
<div class="csl-entry">
277. Bouhaddani S el, Uh H-W, Jongbloed G, Hayward C, Klarić L, Kiełbasa SM, et al. <a href="https://doi.org/10.1186/s12859-018-2371-3">Integrating omics datasets with the OmicsPLS package</a>. BMC Bioinformatics. 2018;19:371.
</div>
<div class="csl-entry">
278. Pimentel H, Zhiyue H, Huang H. <a href="https://doi.org/10.1007/s40484-017-0127-0">Biclustering by sparse canonical correlation analysis</a>. 2018;6:11.
</div>
<div class="csl-entry">
279. Kim Y, Bismeijer T, Zwart W, Wessels LFA, Vis DJ. <a href="https://doi.org/10.1038/s41467-019-13027-2">Genomic data integration by WON-PARAFAC identifies interpretable factors for predicting drug-sensitivity in vivo</a>. Nature Communications. 2019;10:5034.
</div>
<div class="csl-entry">
280. Lock EF, Park JY, Hoadley KA. <a href="http://arxiv.org/abs/2002.02601">Bidimensional linked matrix factorization for pan-omics pan-cancer analysis</a>. arXiv:200202601 [cs, q-bio, stat]. 2020.
</div>
<div class="csl-entry">
281. Ronen J, Hayat S, Akalin A. <a href="https://doi.org/10.26508/lsa.201900517">Evaluation of colorectal cancer subtypes and cell lines using deep learning</a>. Life Science Alliance. 2019;2.
</div>
<div class="csl-entry">
282. Shi WJ, Zhuang Y, Russell PH, Hobbs BD, Parker MM, Castaldi PJ, et al. <a href="https://doi.org/10.1093/bioinformatics/btz226">Unsupervised discovery of phenotype-specific multi-omics networks</a>. Bioinformatics. 2019;35:4336–43.
</div>
<div class="csl-entry">
283. Csala A, Zwinderman AH, Hof MH. <a href="https://doi.org/10.1186/s12859-019-3286-3">Multiset sparse partial least squares path modeling for high dimensional omics data analysis</a>. BMC Bioinformatics. 2020;21:9.
</div>
<div class="csl-entry">
284. Fan Z, Zhou Y, Ressom HW. <a href="https://doi.org/10.3390/metabo10040144">MOTA: Network-Based Multi-Omic Data Integration for Biomarker Discovery</a>. Metabolites. 2020;10:144.
</div>
<div class="csl-entry">
285. Shu H, Wang X, Zhu H. <a href="https://doi.org/10.1080/01621459.2018.1543599">D-CCA: A decomposition-based canonical correlation analysis for high-dimensional datasets</a>. Journal of the American Statistical Association. 2020;115:292–306.
</div>
<div class="csl-entry">
286. Hawinkel S, Bijnens L, Cao K-AL, Thas O. <a href="https://doi.org/10.1093/nargab/lqaa050">Model-based joint visualization of multiple compositional omics datasets</a>. NAR Genomics and Bioinformatics. 2020;2.
</div>
<div class="csl-entry">
287. Gundersen G, Dumitrascu B, Ash JT, Engelhardt BE. <a href="https://proceedings.mlr.press/v115/gundersen20a.html">Uncertainty in Artificial Intelligence</a>. PMLR; 2020. p. 945–55.
</div>
<div class="csl-entry">
288. Velten B, Braunger JM, Arnol D, Argelaguet R, Stegle O. <a href="https://doi.org/10.1101/2020.11.03.366674">Identifying temporal and spatial patterns of variation from multi-modal data using MEFISTO</a>. bioRxiv. 2020;2020.11.03.366674.
</div>
</div>

</div>



            </section>

          </div>
        </div>
      </div>
<a href="acknowledgments.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="online-resources.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": {
"href": "https://github.com/llrs/thesis/"
},
"facebook": false,
"twitter": {
"href": "https://twitter.com/intent/tweet?text=Nice%20read%20of%20%40Lluis_Revilla%20thesis"
},
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/llrs/thesis/edit/master/08_references.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": "https://github.com/llrs/thesis/blob/master/08_references.Rmd",
"text": "View"
},
"download": ["IBD_integration_Lluis_Revilla.pdf", "IBD_integration_Lluis_Revilla.epub"],
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": false,
"toc_float": true,
"toc_appendix": true,
"toc_depth": 2,
"scroll_highlight": true
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
