<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>References | Data integration on inflammatory bowel disease</title>
  <meta name="description" content="Thesis about the relationship between microbiome and transcriptome on the gastrointestinal tract on patients with inflammatory bowel disease and some controls to discover key differences on this relationship." />
  <meta name="generator" content="bookdown 0.21 and GitBook 2.6.7" />

  <meta property="og:title" content="References | Data integration on inflammatory bowel disease" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="Thesis about the relationship between microbiome and transcriptome on the gastrointestinal tract on patients with inflammatory bowel disease and some controls to discover key differences on this relationship." />
  <meta name="github-repo" content="llrs/thesis" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="References | Data integration on inflammatory bowel disease" />
  <meta name="twitter:site" content="@Lluis_Rev" />
  <meta name="twitter:description" content="Thesis about the relationship between microbiome and transcriptome on the gastrointestinal tract on patients with inflammatory bowel disease and some controls to discover key differences on this relationship." />
  

<meta name="author" content="Lluís Revilla Sancho" />


<meta name="date" content="2020-10-16" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="biological-relevance-of-results.html"/>
<link rel="next" href="online-resources.html"/>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />












</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><strong><a href="./">Data integration on IBD</a></strong></li>
<li><a href="https://www.ub.edu", target = "_blank"><img src:"ub_logo.png" alt="UB logo"/></a></li>
<li><a href="https://idibaps.org", target = "_blank"><img src:"idibaps_logo.png" alt="IDIBAPS logo"/></a></li>
<li><a href="https://www.ciberisciii.es", target = "_blank"><img src:'logo_ciber.png' alt="CIBER logo"/></a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Preface</a></li>
<li class="chapter" data-level="1" data-path="introduction.html"><a href="introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a><ul>
<li class="chapter" data-level="1.1" data-path="integration.html"><a href="integration.html"><i class="fa fa-check"></i><b>1.1</b> Integration</a><ul>
<li class="chapter" data-level="1.1.1" data-path="integration.html"><a href="integration.html#classification"><i class="fa fa-check"></i><b>1.1.1</b> Classification</a></li>
<li class="chapter" data-level="1.1.2" data-path="integration.html"><a href="integration.html#benchmarking"><i class="fa fa-check"></i><b>1.1.2</b> Benchmarking</a></li>
<li class="chapter" data-level="1.1.3" data-path="integration.html"><a href="integration.html#conclusions"><i class="fa fa-check"></i><b>1.1.3</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="IBD.html"><a href="IBD.html"><i class="fa fa-check"></i><b>1.2</b> Inflammatory bowel disease</a><ul>
<li class="chapter" data-level="1.2.1" data-path="IBD.html"><a href="IBD.html#disease-onset"><i class="fa fa-check"></i><b>1.2.1</b> Disease onset</a></li>
<li class="chapter" data-level="1.2.2" data-path="IBD.html"><a href="IBD.html#disease-discourse"><i class="fa fa-check"></i><b>1.2.2</b> Disease discourse</a></li>
<li class="chapter" data-level="1.2.3" data-path="IBD.html"><a href="IBD.html#integration-on-ibd"><i class="fa fa-check"></i><b>1.2.3</b> Integration on IBD</a></li>
<li class="chapter" data-level="1.2.4" data-path="IBD.html"><a href="IBD.html#summary"><i class="fa fa-check"></i><b>1.2.4</b> Summary</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="2" data-path="rgcca.html"><a href="rgcca.html"><i class="fa fa-check"></i><b>2</b> RGCCA</a><ul>
<li class="chapter" data-level="2.1" data-path="this-requires-a-model-of-relationships-.html"><a href="this-requires-a-model-of-relationships-.html"><i class="fa fa-check"></i><b>2.1</b> This requires a model of relationships.</a></li>
<li class="chapter" data-level="2.2" data-path="designing-models.html"><a href="designing-models.html"><i class="fa fa-check"></i><b>2.2</b> Designing models</a></li>
<li class="chapter" data-level="2.3" data-path="evaluating-models.html"><a href="evaluating-models.html"><i class="fa fa-check"></i><b>2.3</b> Evaluating models</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="biological-relevance-of-results.html"><a href="biological-relevance-of-results.html"><i class="fa fa-check"></i><b>3</b> Biological relevance of results</a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="appendix"><span><b>Appendix</b></span></li>
<li class="chapter" data-level="A" data-path="online-resources.html"><a href="online-resources.html"><i class="fa fa-check"></i><b>A</b> Online resources</a></li>
<li class="chapter" data-level="B" data-path="software.html"><a href="software.html"><i class="fa fa-check"></i><b>B</b> Software</a><ul>
<li class="chapter" data-level="B.1" data-path="listed.html"><a href="listed.html"><i class="fa fa-check"></i><b>B.1</b> Listed</a></li>
<li class="chapter" data-level="B.2" data-path="by-publication.html"><a href="by-publication.html"><i class="fa fa-check"></i><b>B.2</b> By publication</a></li>
</ul></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Data integration on inflammatory bowel disease</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="references" class="section level1 unnumbered">
<h1>References</h1>
<div id="refs">
<div id="ref-wu_selective_2019">
<p>1. Wu C, Zhou F, Ren J, Li X, Jiang Y, Ma S. A Selective Review of Multi-Level Omics Data Integration Using Variable Selection. High-Throughput. 2019;8:4.</p>
</div>
<div id="ref-cavill_transcriptomic_2016">
<p>2. Cavill R, Jennen D, Kleinjans J, Bried’e JJ. Transcriptomic and metabolomic data integration. Briefings in Bioinformatics. 2016;17:891–901.</p>
</div>
<div id="ref-koh_iomicspass_2019">
<p>3. Koh HWL, Fermin D, Vogel C, Choi KP, Ewing RM, Choi H. iOmicsPASS: Network-based integration of multiomics data for predictive subnetwork discovery. npj Systems Biology and Applications. 2019;5:1–10.</p>
</div>
<div id="ref-tenenhaus_regularized_2011">
<p>4. Tenenhaus A, Tenenhaus M. Regularized Generalized Canonical Correlation Analysis. Psychometrika. 2011;76:257–84.</p>
</div>
<div id="ref-tenenhaus_variable_2014">
<p>5. Tenenhaus A, Philippe C, Guillemot V, Le Cao K-A, Grill J, Frouin V. Variable selection for generalized canonical correlation analysis. Biostatistics. 2014;15:569–83.</p>
</div>
<div id="ref-culhane_cross-platform_2003">
<p>6. Culhane AC, Perri‘ere G, Higgins DG. Cross-platform comparison and visualisation of gene expression data using co-inertia analysis. BMC Bioinformatics. 2003;4:59.</p>
</div>
<div id="ref-vito_multi-study_2019">
<p>7. Vito RD, Bellio R, Trippa L, Parmigiani G. Multi-study factor analysis. Biometrics. 2019;75:337–46.</p>
</div>
<div id="ref-argelaguet_multi-omics_2018">
<p>8. Argelaguet R, Velten B, Arnol D, Dietrich S, Zenz T, Marioni JC, et al. Multi-Omics Factor Analysisa framework for unsupervised integration of multi-omics data sets. Molecular Systems Biology. 2018;14:e8124.</p>
</div>
<div id="ref-gomez-cabreroSTATegraComprehensiveMultiomics2019">
<p>9. Gomez-Cabrero D, Tarazona S, Ferreir’os-Vidal I, Ramirez RN, Company C, Schmidt A, et al. STATegra, a comprehensive multi-omics dataset of B-cell differentiation in mouse. Scientific Data. 2019;6:1–15.</p>
</div>
<div id="ref-sherryConductingInterpretingCanonical1981">
<p>10. Sherry A, Henson RK. Conducting and Interpreting Canonical Correlation Analysis in Personality Research: A User-Friendly Primer. 1981.</p>
</div>
<div id="ref-chung_multi-omics_2019">
<p>11. Chung R-H, Kang C-Y. A multi-omics data simulator for complex disease studies and its application to evaluate multi-omics data analysis methods for disease classification. GigaScience. 2019;8.</p>
</div>
<div id="ref-martinez-mira_mosim_2018">
<p>12. Martínez-Mira C, Conesa A, Tarazona S. MOSim: Multi-Omics Simulation in R. Preprint. Bioinformatics; 2018.</p>
</div>
<div id="ref-cantini_benchmarking_2020">
<p>13. Cantini L, Zakeri P, Hernandez C, Naldi A, Thieffry D, Remy E, et al. Benchmarking joint multi-omics dimensionality reduction approaches for cancer study. bioRxiv. 2020;2020.01.14.905760.</p>
</div>
<div id="ref-human_microbiome_project_consortium_framework_2012">
<p>14. Human Microbiome Project Consortium BA, Nelson KE, Pop M, Creasy HH, Giglio MG, Huttenhower C, et al. A framework for human microbiome research. Nature. 2012;486:215–21.</p>
</div>
<div id="ref-hasler_uncoupling_2016">
<p>15. Häsler R, Sheibani-Tezerji R, Sinha A, Barann M, Rehman A, Esser D, et al. Uncoupling of mucosal gene regulation, mRNA splicing and adherent microbiota signatures in inflammatory bowel disease. Gut. 2016;gutjnl–2016–311651.</p>
</div>
<div id="ref-tang_integrated_2017">
<p>16. Tang MS, Bowcutt R, Leung JM, Wolff MJ, Gundra UM, Hudesman D, et al. Integrated Analysis of Biopsies from Inflammatory Bowel Disease Patients Identifies SAA1 as a Link Between Mucosal Microbes with TH17 and TH22 Cells. Inflammatory Bowel Diseases. 2017;23:1544–54.</p>
</div>
<div id="ref-de_souza_ibd_2017">
<p>17. De Souza HS, Fiocchi C, Iliopoulos D. The IBD interactome: An integrated view of aetiology, pathogenesis and therapy. 2017;14.</p>
</div>
<div id="ref-tenenhaus_component-based_2008">
<p>18. Tenenhaus M. Component-based Structural Equation Modelling. Total Quality Management &amp; Business Excellence. 2008;19:871–86.</p>
</div>
<div id="ref-esposito_vinzi_bridge_2010">
<p>19. Tenenhaus M, Hanafi M. A Bridge Between PLS Path Modeling and Multi-Block Data Analysis. In: Esposito Vinzi V, Chin WW, Henseler J, Wang H, editors. Handbook of Partial Least Squares. Berlin, Heidelberg: Springer Berlin Heidelberg; 2010. pp. 99–123.</p>
</div>
<div id="ref-tenenhaus_regularized_2014">
<p>20. Tenenhaus A, Tenenhaus M. Regularized generalized canonical correlation analysis for multiblock or multigroup data analysis. European Journal of Operational Research. 2014;238:391–403.</p>
</div>
<div id="ref-tenenhaus_kernel_2015">
<p>21. Tenenhaus A, Philippe C, Frouin V. Kernel Generalized Canonical Correlation Analysis. Computational Statistics &amp; Data Analysis. 2015;90:114–31.</p>
</div>
</div>
</div>



            </section>

          </div>
        </div>
      </div>
<a href="biological-relevance-of-results.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="online-resources.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": {
"href": "https://github.com/llrs/thesis/"
},
"facebook": false,
"twitter": {
"href": "https://twitter.com/intent/tweet?text=Nice%20read%20of%20%40Lluis_Rev%20thesis"
},
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/llrs/thesis/edit/master/98_appendix.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": "https://github.com/llrs/thesis/blob/master/98_appendix.Rmd",
"text": null
},
"download": ["Data integration on inflammatory bowel disease.pdf", "Data integration on inflammatory bowel disease.epub"],
"toc": {
"collapse": false,
"toc_appendix": true,
"scroll_highlight": true
},
"toc_float": true
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
