<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>2.1 Relationships' models | Data integration on inflammatory bowel disease</title>
  <meta name="description" content="Thesis about the relationship between microbiome and transcriptome on the gastrointestinal tract on patients with inflammatory bowel disease and some controls to discover key differences on this relationship." />
  <meta name="generator" content="bookdown 0.21 and GitBook 2.6.7" />

  <meta property="og:title" content="2.1 Relationships' models | Data integration on inflammatory bowel disease" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="Thesis about the relationship between microbiome and transcriptome on the gastrointestinal tract on patients with inflammatory bowel disease and some controls to discover key differences on this relationship." />
  <meta name="github-repo" content="llrs/thesis" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="2.1 Relationships' models | Data integration on inflammatory bowel disease" />
  <meta name="twitter:site" content="@Lluis_Revilla" />
  <meta name="twitter:description" content="Thesis about the relationship between microbiome and transcriptome on the gastrointestinal tract on patients with inflammatory bowel disease and some controls to discover key differences on this relationship." />
  

<meta name="author" content="LluÃ­s Revilla Sancho" />


<meta name="date" content="2021-02-24" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="multi-omic-modelling-of-inflammatory-bowel-disease-with-regularized-canonical-correlation-analysis.html"/>
<link rel="next" href="conclusions.html"/>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />












<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><strong><a href="./">Data integration on IBD</a></strong></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Preface</a></li>
<li class="chapter" data-level="1" data-path="introduction.html"><a href="introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a><ul>
<li class="chapter" data-level="1.1" data-path="integration.html"><a href="integration.html"><i class="fa fa-check"></i><b>1.1</b> Integration</a><ul>
<li class="chapter" data-level="1.1.1" data-path="integration.html"><a href="integration.html#classification-of-integration-methods"><i class="fa fa-check"></i><b>1.1.1</b> Classification of integration method's</a></li>
<li class="chapter" data-level="1.1.2" data-path="integration.html"><a href="integration.html#reviews-and-benchmarking"><i class="fa fa-check"></i><b>1.1.2</b> Reviews and benchmarking</a></li>
<li class="chapter" data-level="1.1.3" data-path="integration.html"><a href="integration.html#summary"><i class="fa fa-check"></i><b>1.1.3</b> Summary</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="IBD.html"><a href="IBD.html"><i class="fa fa-check"></i><b>1.2</b> Inflammatory bowel disease</a><ul>
<li class="chapter" data-level="1.2.1" data-path="IBD.html"><a href="IBD.html#disease-etiology"><i class="fa fa-check"></i><b>1.2.1</b> Disease etiology</a></li>
<li class="chapter" data-level="1.2.2" data-path="IBD.html"><a href="IBD.html#crohns-disease"><i class="fa fa-check"></i><b>1.2.2</b> Crohn's disease</a></li>
<li class="chapter" data-level="1.2.3" data-path="IBD.html"><a href="IBD.html#ulcerative-colitis"><i class="fa fa-check"></i><b>1.2.3</b> Ulcerative colitis</a></li>
<li class="chapter" data-level="1.2.4" data-path="IBD.html"><a href="IBD.html#clinical-care"><i class="fa fa-check"></i><b>1.2.4</b> Clinical care</a></li>
<li class="chapter" data-level="1.2.5" data-path="IBD.html"><a href="IBD.html#summary-1"><i class="fa fa-check"></i><b>1.2.5</b> Summary</a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="integration-on-ibd.html"><a href="integration-on-ibd.html"><i class="fa fa-check"></i><b>1.3</b> Integration on IBD</a><ul>
<li class="chapter" data-level="1.3.1" data-path="integration-on-ibd.html"><a href="integration-on-ibd.html#genetics"><i class="fa fa-check"></i><b>1.3.1</b> Genetics</a></li>
<li class="chapter" data-level="1.3.2" data-path="integration-on-ibd.html"><a href="integration-on-ibd.html#microbiome"><i class="fa fa-check"></i><b>1.3.2</b> Microbiome</a></li>
<li class="chapter" data-level="1.3.3" data-path="integration-on-ibd.html"><a href="integration-on-ibd.html#multiple-data"><i class="fa fa-check"></i><b>1.3.3</b> Multiple data</a></li>
<li class="chapter" data-level="1.3.4" data-path="integration-on-ibd.html"><a href="integration-on-ibd.html#summary-2"><i class="fa fa-check"></i><b>1.3.4</b> Summary</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="2" data-path="multi-omic-modelling-of-inflammatory-bowel-disease-with-regularized-canonical-correlation-analysis.html"><a href="multi-omic-modelling-of-inflammatory-bowel-disease-with-regularized-canonical-correlation-analysis.html"><i class="fa fa-check"></i><b>2</b> Multi-omic modelling of inflammatory bowel disease with regularized canonical correlation analysis</a><ul>
<li class="chapter" data-level="2.1" data-path="relationships-models.html"><a href="relationships-models.html"><i class="fa fa-check"></i><b>2.1</b> Relationships' models</a><ul>
<li class="chapter" data-level="2.1.1" data-path="relationships-models.html"><a href="relationships-models.html#designing-models"><i class="fa fa-check"></i><b>2.1.1</b> Designing models</a></li>
<li class="chapter" data-level="2.1.2" data-path="relationships-models.html"><a href="relationships-models.html#evaluating-models"><i class="fa fa-check"></i><b>2.1.2</b> Evaluating models</a></li>
</ul></li>
<li class="chapter" data-level="2.2" data-path="conclusions.html"><a href="conclusions.html"><i class="fa fa-check"></i><b>2.2</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="biological-relevance-of-results.html"><a href="biological-relevance-of-results.html"><i class="fa fa-check"></i><b>3</b> Biological relevance of results</a><ul>
<li class="chapter" data-level="3.1" data-path="comparing-different-dataset.html"><a href="comparing-different-dataset.html"><i class="fa fa-check"></i><b>3.1</b> Comparing different dataset</a></li>
<li class="chapter" data-level="3.2" data-path="shared-selected-variables.html"><a href="shared-selected-variables.html"><i class="fa fa-check"></i><b>3.2</b> Shared selected variables</a></li>
<li class="chapter" data-level="3.3" data-path="conclusions-1.html"><a href="conclusions-1.html"><i class="fa fa-check"></i><b>3.3</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="appendix"><span><b>Appendix</b></span></li>
<li class="chapter" data-level="A" data-path="online-resources.html"><a href="online-resources.html"><i class="fa fa-check"></i><b>A</b> Online resources</a></li>
<li class="chapter" data-level="B" data-path="software.html"><a href="software.html"><i class="fa fa-check"></i><b>B</b> Software</a><ul>
<li class="chapter" data-level="B.1" data-path="listed.html"><a href="listed.html"><i class="fa fa-check"></i><b>B.1</b> Listed</a></li>
<li class="chapter" data-level="B.2" data-path="by-publication.html"><a href="by-publication.html"><i class="fa fa-check"></i><b>B.2</b> By publication</a></li>
</ul></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Data integration on inflammatory bowel disease</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div class = "wip">
This tesis is ongoing and still without hard date to finish it. Watch the <a href="https://github.com/llrs/thesis">GitHub repository</a> or wait for a release announcement on Twitter on <a href="https://twitter.com/Lluis_Rev">my profile</a>.
</div>
<div id="relationships-models" class="section level2">
<h2><span class="header-section-number">2.1</span> Relationships' models</h2>
<p>Most multi-omics and integration tools assume one block for each type of data. However, RGCCA uses linear relationships between blocks of data. But doesn't say what constitutes a block of data. Usually a block of data is just a source of data, such as an essay a survey or an experiment. We decided to split the block with data about the samples to separate independent variables from the same block. The hypothesis we made was that with independent blocks the combined linear model would fit better the data.</p>
<p>These news blocks can also help make more interpretable the relationships, because instead of a big metadata block we have a block for time related variables, another one for location and so on. This allows to design a model with an expected relationships between these dimensions.</p>
<p>However, one limitation of the generalized canonical correlation analysis an in general on the dimension reduction methods is that the design matrix must be symmetric. Which implies that we can't infer causal relationship using these methods.</p>
<div id="designing-models" class="section level3">
<h3><span class="header-section-number">2.1.1</span> Designing models</h3>
<p>What defines a block: Blocks of variables are treated independently but variables within a block are used in a linear combination.</p>
<p>Create models with random links between blocks.</p>
<p>Create models considering the knowledge of the system or disease.</p>
</div>
<div id="evaluating-models" class="section level3">
<h3><span class="header-section-number">2.1.2</span> Evaluating models</h3>
<p>To evaluate a model RGCCA provides the average variance explained (AVE), inner and outer. Inner AVE is for how well do all the canonical dimensions correlate with the design of the experiment, so it a measure of how good the model is. While outer AVE is a measure of how well do the variables of each block correlate with the canonical dimension, so it measures the agreement between the variables and the canonical dimensions.</p>
<p>To evaluate a design a bootstraping method can be used to know how well the design does apply to a variety of data. Another option is to use an external cohort to validate the same model, or using an external model to see if you reach the same model, which is what we did.</p>
<p>Next it must be evaluated compared to other methods to see if it brings something new or not. Of the multiple methods available we used <a href="https://bioconductor.org/packages/omicade4" title="Omicade4">MCIA</a> <span class="citation">[<a href="#ref-mengMultivariateApproachIntegration2014">34</a>]</span>. Which was compared by looking at the area under the curve for classifying the samples according to their location.</p>
<p>Besides a way to compare methods, these models do need to be evaluated by the insights they provide on the biological system they are being applied to, in our case the Crohn's disease. In this article we didn't look in depth to the biological relevance of the microorganisms an genes found.</p>
</div>
</div>
<h3>References</h3>
<div id="refs" class="references">
<div id="ref-mengMultivariateApproachIntegration2014">
<p>34. Meng C, Kuster B, Culhane AC, Gholami AM. A multivariate approach to the integration of multi-omics datasets. BMC Bioinformatics. 2014;15:162. doi:<a href="https://doi.org/10.1186/1471-2105-15-162">10.1186/1471-2105-15-162</a>.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="multi-omic-modelling-of-inflammatory-bowel-disease-with-regularized-canonical-correlation-analysis.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="conclusions.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": {
"href": "https://github.com/llrs/thesis/"
},
"facebook": false,
"twitter": {
"href": "https://twitter.com/intent/tweet?text=Nice%20read%20of%20%40Lluis_Rev%20thesis"
},
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/llrs/thesis/edit/master/02_article1.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": "https://github.com/llrs/thesis/blob/master/02_article1.Rmd",
"text": null
},
"download": ["Data integration on inflammatory bowel disease.pdf", "Data integration on inflammatory bowel disease.epub"],
"toc": {
"collapse": false,
"toc_appendix": true,
"scroll_highlight": true
},
"toc_float": true
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
