<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>B Software | Data integration in inflammatory bowel disease</title>
  <meta name="description" content="B Software | Data integration in inflammatory bowel disease" />
  <meta name="generator" content="bookdown 0.26 and GitBook 2.6.7" />

  <meta property="og:title" content="B Software | Data integration in inflammatory bowel disease" />
  <meta property="og:type" content="book" />
  
  
  <meta name="github-repo" content="llrs/thesis" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="B Software | Data integration in inflammatory bowel disease" />
  <meta name="twitter:site" content="@Lluis_Revilla" />
  
  

<meta name="author" content="Lluís Revilla Sancho" />


<meta name="date" content="2022-05-25" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="online-resources.html"/>
<link rel="next" href="articles.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>
<script src="libs/kePrint-0.0.1/kePrint.js"></script>
<link href="libs/lightable-0.0.1/lightable.css" rel="stylesheet" />
<link rel="apple-touch-icon" sizes="180x180" href="images/apple-touch-icon.png">
<link rel="icon" type="image/png" sizes="32x32" href="images/favicon-32x32.png">
<link rel="icon" type="image/png" sizes="16x16" href="images/favicon-16x16.png">
<link rel="manifest" href="images/site.webmanifest">
<link rel="mask-icon" href="images/safari-pinned-tab.svg" color="#5bbad5">
<link rel="shortcut icon" href="images/favicon.ico">
<meta name="msapplication-TileColor" content="#da532c">
<meta name="msapplication-config" content="images/browserconfig.xml">
<meta name="theme-color" content="#ffffff">
<meta name="viewport" content="width=device-width, initial-scale=1, orientation=auto"/>
<meta name="description" content="Thesis searching the relationship between microbiome and transcriptome in the intestine on patients with IBD."/>
<meta name="github-repo" content="llrs/thesis"/>
<meta name="twitter:title" content="B Software | Data integration in inflammatory bowel disease"/>
<meta name="twitter:description" content="Thesis searching the relationship between microbiome and transcriptome in the intestine on patients with IBD."/>
<meta name="twitter:url" content="https://thesis.llrs.dev"/>
<meta name="twitter:image" content="https://thesis.llrs.dev/images/cover.png"/>
<meta name="twitter:image:alt" content="The cover of the thesis showing the title, and author. Below a drawing of a colon with microbiome on the ascending colon, numbers in a the transversal segment and some epithelium with different smoothness, thickness and colors (red, orange and brown). At the bottom doctoral thesis."/>
<meta name="twitter:card" content="summary"/>
<meta name="twitter:creator" content="@Lluis_Revilla"/>
<meta name="twitter:site" content="@Lluis_Revilla"/>
<meta property="og:title" content="B Software | Data integration in inflammatory bowel disease"/>
<meta property="og:description" content="Thesis searching the relationship between microbiome and transcriptome in the intestine on patients with IBD."/>
<meta property="og:url" content="https://thesis.llrs.dev"/>
<meta property="og:image" content="https://thesis.llrs.dev/images/cover.png"/>
<meta property="og:image:alt" content="The cover of the thesis showing the title, and author. Below a drawing of a colon with microbiome on the ascending colon, numbers in a the transversal segment and some epithelium with different smoothness, thickness and colors (red, orange and brown). At the bottom doctoral thesis."/>
<meta property="og:type" content="book"/>
<meta property="og:locale" content="en_US"/>
<meta property="article:author" content="Llu&lt;U+00ED&gt;s Revilla Sancho"/>
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=G-EN2MTH2FCR"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());
  gtag('config', 'G-EN2MTH2FCR');
</script>



<style type="text/css">
/* Used with Pandoc 2.11+ new --citeproc when CSL is used */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<link rel="stylesheet" href="style/style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><strong><a href="./">Data integration in IBD</a></strong></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Preface</a></li>
<li class="chapter" data-level="" data-path="abstracts.html"><a href="abstracts.html"><i class="fa fa-check"></i>Abstracts</a>
<ul>
<li class="chapter" data-level="" data-path="abstracts.html"><a href="abstracts.html#english"><i class="fa fa-check"></i>English</a></li>
<li class="chapter" data-level="" data-path="abstracts.html"><a href="abstracts.html#spanish"><i class="fa fa-check"></i>Spanish</a></li>
<li class="chapter" data-level="" data-path="abstracts.html"><a href="abstracts.html#catalan"><i class="fa fa-check"></i>Catalan</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="glossary.html"><a href="glossary.html"><i class="fa fa-check"></i>Glossary</a></li>
<li class="chapter" data-level="1" data-path="introduction.html"><a href="introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a>
<ul>
<li class="chapter" data-level="1.1" data-path="introduction.html"><a href="introduction.html#IBD"><i class="fa fa-check"></i><b>1.1</b> Inflammatory bowel disease</a>
<ul>
<li class="chapter" data-level="1.1.1" data-path="introduction.html"><a href="introduction.html#etiology-and-pathogenesis"><i class="fa fa-check"></i><b>1.1.1</b> Etiology and pathogenesis</a></li>
<li class="chapter" data-level="1.1.2" data-path="introduction.html"><a href="introduction.html#CD"><i class="fa fa-check"></i><b>1.1.2</b> <span>CD</span> physiology</a></li>
<li class="chapter" data-level="1.1.3" data-path="introduction.html"><a href="introduction.html#UC"><i class="fa fa-check"></i><b>1.1.3</b> <span>UC</span> physiology</a></li>
<li class="chapter" data-level="1.1.4" data-path="introduction.html"><a href="introduction.html#treatment"><i class="fa fa-check"></i><b>1.1.4</b> Treatment</a></li>
<li class="chapter" data-level="1.1.5" data-path="introduction.html"><a href="introduction.html#summary-ibd"><i class="fa fa-check"></i><b>1.1.5</b> Summary</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="introduction.html"><a href="introduction.html#integration-studies-in"><i class="fa fa-check"></i><b>1.2</b> Integration studies in <span>IBD</span></a>
<ul>
<li class="chapter" data-level="1.2.1" data-path="introduction.html"><a href="introduction.html#data-origin"><i class="fa fa-check"></i><b>1.2.1</b> Type of data used for integration analysis</a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="introduction.html"><a href="introduction.html#integration"><i class="fa fa-check"></i><b>1.3</b> Integration</a>
<ul>
<li class="chapter" data-level="1.3.1" data-path="introduction.html"><a href="introduction.html#classification-of-integration-methods"><i class="fa fa-check"></i><b>1.3.1</b> Classification of integration methods</a></li>
<li class="chapter" data-level="1.3.2" data-path="introduction.html"><a href="introduction.html#interpretation"><i class="fa fa-check"></i><b>1.3.2</b> Interpretation</a></li>
<li class="chapter" data-level="1.3.3" data-path="introduction.html"><a href="introduction.html#reviews"><i class="fa fa-check"></i><b>1.3.3</b> Reviews</a></li>
<li class="chapter" data-level="1.3.4" data-path="introduction.html"><a href="introduction.html#methods-integration"><i class="fa fa-check"></i><b>1.3.4</b> Summary</a></li>
</ul></li>
<li class="chapter" data-level="1.4" data-path="introduction.html"><a href="introduction.html#summary"><i class="fa fa-check"></i><b>1.4</b> Summary</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="hypothesis-and-objectives.html"><a href="hypothesis-and-objectives.html"><i class="fa fa-check"></i><b>2</b> Hypothesis and objectives</a>
<ul>
<li class="chapter" data-level="2.1" data-path="hypothesis-and-objectives.html"><a href="hypothesis-and-objectives.html#hypothesis"><i class="fa fa-check"></i><b>2.1</b> Hypothesis</a></li>
<li class="chapter" data-level="2.2" data-path="hypothesis-and-objectives.html"><a href="hypothesis-and-objectives.html#objectives"><i class="fa fa-check"></i><b>2.2</b> Objectives</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="materials-and-methods.html"><a href="materials-and-methods.html"><i class="fa fa-check"></i><b>3</b> Materials and methods</a>
<ul>
<li class="chapter" data-level="3.1" data-path="materials-and-methods.html"><a href="materials-and-methods.html#datasets"><i class="fa fa-check"></i><b>3.1</b> Datasets</a>
<ul>
<li class="chapter" data-level="3.1.1" data-path="materials-and-methods.html"><a href="materials-and-methods.html#methods-puget"><i class="fa fa-check"></i><b>3.1.1</b> Puget’s dataset</a></li>
<li class="chapter" data-level="3.1.2" data-path="materials-and-methods.html"><a href="materials-and-methods.html#methods-hsct"><i class="fa fa-check"></i><b>3.1.2</b> HSCT dataset</a></li>
<li class="chapter" data-level="3.1.3" data-path="materials-and-methods.html"><a href="materials-and-methods.html#methods-hasler"><i class="fa fa-check"></i><b>3.1.3</b> Häsler’s dataset</a></li>
<li class="chapter" data-level="3.1.4" data-path="materials-and-methods.html"><a href="materials-and-methods.html#methods-morgan"><i class="fa fa-check"></i><b>3.1.4</b> Morgan’s dataset</a></li>
<li class="chapter" data-level="3.1.5" data-path="materials-and-methods.html"><a href="materials-and-methods.html#methods-howell"><i class="fa fa-check"></i><b>3.1.5</b> Howell’s dataset</a></li>
</ul></li>
<li class="chapter" data-level="3.2" data-path="materials-and-methods.html"><a href="materials-and-methods.html#processing"><i class="fa fa-check"></i><b>3.2</b> Sample processing</a>
<ul>
<li class="chapter" data-level="3.2.1" data-path="materials-and-methods.html"><a href="materials-and-methods.html#rna-sequencing"><i class="fa fa-check"></i><b>3.2.1</b> RNA sequencing</a></li>
<li class="chapter" data-level="3.2.2" data-path="materials-and-methods.html"><a href="materials-and-methods.html#microbial-dna-sequencing"><i class="fa fa-check"></i><b>3.2.2</b> Microbial DNA sequencing</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="materials-and-methods.html"><a href="materials-and-methods.html#integration-methods"><i class="fa fa-check"></i><b>3.3</b> Integration methods</a>
<ul>
<li class="chapter" data-level="3.3.1" data-path="materials-and-methods.html"><a href="materials-and-methods.html#regularized-generalized-canonical-correlation-analysis"><i class="fa fa-check"></i><b>3.3.1</b> Regularized generalized canonical correlation analysis</a></li>
<li class="chapter" data-level="3.3.2" data-path="materials-and-methods.html"><a href="materials-and-methods.html#other"><i class="fa fa-check"></i><b>3.3.2</b> Other</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="materials-and-methods.html"><a href="materials-and-methods.html#functional-enrichment-methods"><i class="fa fa-check"></i><b>3.4</b> Functional enrichment methods</a>
<ul>
<li class="chapter" data-level="3.4.1" data-path="materials-and-methods.html"><a href="materials-and-methods.html#ORA"><i class="fa fa-check"></i><b>3.4.1</b> Over representation analysis</a></li>
<li class="chapter" data-level="3.4.2" data-path="materials-and-methods.html"><a href="materials-and-methods.html#gene-set-enrichment-analysis"><i class="fa fa-check"></i><b>3.4.2</b> Gene Set Enrichment Analysis</a></li>
</ul></li>
<li class="chapter" data-level="3.5" data-path="materials-and-methods.html"><a href="materials-and-methods.html#variance"><i class="fa fa-check"></i><b>3.5</b> Variance and diversity methods</a>
<ul>
<li class="chapter" data-level="3.5.1" data-path="materials-and-methods.html"><a href="materials-and-methods.html#permanova"><i class="fa fa-check"></i><b>3.5.1</b> PERMANOVA</a></li>
<li class="chapter" data-level="3.5.2" data-path="materials-and-methods.html"><a href="materials-and-methods.html#globaltest"><i class="fa fa-check"></i><b>3.5.2</b> globaltest</a></li>
<li class="chapter" data-level="3.5.3" data-path="materials-and-methods.html"><a href="materials-and-methods.html#diversity-indices"><i class="fa fa-check"></i><b>3.5.3</b> Diversity indices</a></li>
</ul></li>
<li class="chapter" data-level="3.6" data-path="materials-and-methods.html"><a href="materials-and-methods.html#other-methods"><i class="fa fa-check"></i><b>3.6</b> Other methods</a>
<ul>
<li class="chapter" data-level="3.6.1" data-path="materials-and-methods.html"><a href="materials-and-methods.html#statistics"><i class="fa fa-check"></i><b>3.6.1</b> Statistics</a></li>
<li class="chapter" data-level="3.6.2" data-path="materials-and-methods.html"><a href="materials-and-methods.html#wgcna"><i class="fa fa-check"></i><b>3.6.2</b> WGCNA</a></li>
<li class="chapter" data-level="3.6.3" data-path="materials-and-methods.html"><a href="materials-and-methods.html#baseset"><i class="fa fa-check"></i><b>3.6.3</b> BaseSet</a></li>
<li class="chapter" data-level="3.6.4" data-path="materials-and-methods.html"><a href="materials-and-methods.html#experdesign"><i class="fa fa-check"></i><b>3.6.4</b> experDesign</a></li>
<li class="chapter" data-level="3.6.5" data-path="materials-and-methods.html"><a href="materials-and-methods.html#ROC"><i class="fa fa-check"></i><b>3.6.5</b> ROC- AUC</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="results.html"><a href="results.html"><i class="fa fa-check"></i><b>4</b> Results</a>
<ul>
<li class="chapter" data-level="4.1" data-path="results.html"><a href="results.html#packagesmethods"><i class="fa fa-check"></i><b>4.1</b> Packages/methods</a>
<ul>
<li class="chapter" data-level="4.1.1" data-path="results.html"><a href="results.html#experdesign-1"><i class="fa fa-check"></i><b>4.1.1</b> experDesign</a></li>
<li class="chapter" data-level="4.1.2" data-path="results.html"><a href="results.html#baseset-1"><i class="fa fa-check"></i><b>4.1.2</b> BaseSet</a></li>
<li class="chapter" data-level="4.1.3" data-path="results.html"><a href="results.html#intermodel"><i class="fa fa-check"></i><b>4.1.3</b> inteRmodel</a></li>
</ul></li>
<li class="chapter" data-level="4.2" data-path="results.html"><a href="results.html#analysis"><i class="fa fa-check"></i><b>4.2</b> Analysis</a>
<ul>
<li class="chapter" data-level="4.2.1" data-path="results.html"><a href="results.html#results-puget"><i class="fa fa-check"></i><b>4.2.1</b> Puget’s dataset</a></li>
<li class="chapter" data-level="4.2.2" data-path="results.html"><a href="results.html#results-hsct"><i class="fa fa-check"></i><b>4.2.2</b> HSCT dataset</a></li>
<li class="chapter" data-level="4.2.3" data-path="results.html"><a href="results.html#results-hasler"><i class="fa fa-check"></i><b>4.2.3</b> Häsler’s dataset</a></li>
<li class="chapter" data-level="4.2.4" data-path="results.html"><a href="results.html#results-morgan"><i class="fa fa-check"></i><b>4.2.4</b> Morgan’s dataset</a></li>
<li class="chapter" data-level="4.2.5" data-path="results.html"><a href="results.html#results-howell"><i class="fa fa-check"></i><b>4.2.5</b> Howell’s dataset</a></li>
<li class="chapter" data-level="4.2.6" data-path="results.html"><a href="results.html#between-datasets"><i class="fa fa-check"></i><b>4.2.6</b> Between datasets</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="discussion.html"><a href="discussion.html"><i class="fa fa-check"></i><b>5</b> Discussion</a>
<ul>
<li class="chapter" data-level="5.1" data-path="discussion.html"><a href="discussion.html#preliminary-steps"><i class="fa fa-check"></i><b>5.1</b> Preliminary steps</a>
<ul>
<li class="chapter" data-level="5.1.1" data-path="discussion.html"><a href="discussion.html#discussion-datasets"><i class="fa fa-check"></i><b>5.1.1</b> The datasets</a></li>
<li class="chapter" data-level="5.1.2" data-path="discussion.html"><a href="discussion.html#the-methods"><i class="fa fa-check"></i><b>5.1.2</b> The methods</a></li>
</ul></li>
<li class="chapter" data-level="5.2" data-path="discussion.html"><a href="discussion.html#designing-models"><i class="fa fa-check"></i><b>5.2</b> Designing models</a></li>
<li class="chapter" data-level="5.3" data-path="discussion.html"><a href="discussion.html#evaluating-models"><i class="fa fa-check"></i><b>5.3</b> Evaluating models</a></li>
<li class="chapter" data-level="5.4" data-path="discussion.html"><a href="discussion.html#implications"><i class="fa fa-check"></i><b>5.4</b> Implications</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="conclusions.html"><a href="conclusions.html"><i class="fa fa-check"></i><b>6</b> Conclusions</a>
<ul>
<li class="chapter" data-level="6.1" data-path="conclusions.html"><a href="conclusions.html#study-1-multi-omic-modelling-of-inflammatory-bowel-disease-with-regularized-canonical-correlation-analysis"><i class="fa fa-check"></i><b>6.1</b> Study 1: Multi-omic modelling of inflammatory bowel disease with regularized canonical correlation analysis</a></li>
<li class="chapter" data-level="6.2" data-path="conclusions.html"><a href="conclusions.html#study-2-the-relationship-of-genes-and-the-microbiome-with-inflammatory-bowel-disease"><i class="fa fa-check"></i><b>6.2</b> Study 2: The relationship of genes and the microbiome with inflammatory bowel disease</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="acknowledgments.html"><a href="acknowledgments.html"><i class="fa fa-check"></i><b>7</b> Acknowledgments</a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="appendix"><span><b>Appendix</b></span></li>
<li class="chapter" data-level="A" data-path="online-resources.html"><a href="online-resources.html"><i class="fa fa-check"></i><b>A</b> Online resources</a></li>
<li class="chapter" data-level="B" data-path="software.html"><a href="software.html"><i class="fa fa-check"></i><b>B</b> Software</a>
<ul>
<li class="chapter" data-level="B.1" data-path="software.html"><a href="software.html#star"><i class="fa fa-check"></i><b>B.1</b> STAR</a></li>
<li class="chapter" data-level="B.2" data-path="software.html"><a href="software.html#rsem"><i class="fa fa-check"></i><b>B.2</b> RSEM</a></li>
<li class="chapter" data-level="B.3" data-path="software.html"><a href="software.html#listed"><i class="fa fa-check"></i><b>B.3</b> Listed</a></li>
<li class="chapter" data-level="B.4" data-path="software.html"><a href="software.html#by-projectpublication"><i class="fa fa-check"></i><b>B.4</b> By project/publication</a></li>
</ul></li>
<li class="chapter" data-level="C" data-path="articles.html"><a href="articles.html"><i class="fa fa-check"></i><b>C</b> Articles</a>
<ul>
<li class="chapter" data-level="C.1" data-path="articles.html"><a href="articles.html#app-multiomics"><i class="fa fa-check"></i><b>C.1</b> Multi-omic modelling of inflammatory bowel disease with regularized canonical correlation analysis</a></li>
<li class="chapter" data-level="C.2" data-path="articles.html"><a href="articles.html#app-experDesign"><i class="fa fa-check"></i><b>C.2</b> experDesign: stratifying samples into batches with minimal bias</a></li>
<li class="chapter" data-level="C.3" data-path="articles.html"><a href="articles.html#app-upa"><i class="fa fa-check"></i><b>C.3</b> Randomized Controlled Trial Substudy of Cell-specific Mechanisms of Janus Kinase 1 Inhibition With Upadacitinib in the Crohn’s Disease Intestinal Mucosa: Analysis From the CELEST Study</a></li>
</ul></li>
<li class="chapter" data-level="D" data-path="other-datasets.html"><a href="other-datasets.html"><i class="fa fa-check"></i><b>D</b> Other datasets</a>
<ul>
<li class="chapter" data-level="D.1" data-path="other-datasets.html"><a href="other-datasets.html#barcelona"><i class="fa fa-check"></i><b>D.1</b> BARCELONA dataset</a></li>
<li class="chapter" data-level="D.2" data-path="other-datasets.html"><a href="other-datasets.html#hernandez"><i class="fa fa-check"></i><b>D.2</b> Hernández’s dataset</a>
<ul>
<li class="chapter" data-level="D.2.1" data-path="other-datasets.html"><a href="other-datasets.html#results-hernandez"><i class="fa fa-check"></i><b>D.2.1</b> Results</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="E" data-path="models-output.html"><a href="models-output.html"><i class="fa fa-check"></i><b>E</b> Models output</a>
<ul>
<li class="chapter" data-level="E.1" data-path="models-output.html"><a href="models-output.html#hsct"><i class="fa fa-check"></i><b>E.1</b> HSCT</a>
<ul>
<li class="chapter" data-level="E.1.1" data-path="models-output.html"><a href="models-output.html#genes"><i class="fa fa-check"></i><b>E.1.1</b> Genes</a></li>
<li class="chapter" data-level="E.1.2" data-path="models-output.html"><a href="models-output.html#microbiome-2"><i class="fa fa-check"></i><b>E.1.2</b> Microbiome</a></li>
</ul></li>
<li class="chapter" data-level="E.2" data-path="models-output.html"><a href="models-output.html#häsler"><i class="fa fa-check"></i><b>E.2</b> Häsler</a>
<ul>
<li class="chapter" data-level="E.2.1" data-path="models-output.html"><a href="models-output.html#genes-1"><i class="fa fa-check"></i><b>E.2.1</b> Genes</a></li>
<li class="chapter" data-level="E.2.2" data-path="models-output.html"><a href="models-output.html#microbiome-3"><i class="fa fa-check"></i><b>E.2.2</b> Microbiome</a></li>
</ul></li>
<li class="chapter" data-level="E.3" data-path="models-output.html"><a href="models-output.html#morgan"><i class="fa fa-check"></i><b>E.3</b> Morgan</a>
<ul>
<li class="chapter" data-level="E.3.1" data-path="models-output.html"><a href="models-output.html#genes-2"><i class="fa fa-check"></i><b>E.3.1</b> Genes</a></li>
<li class="chapter" data-level="E.3.2" data-path="models-output.html"><a href="models-output.html#microbiome-4"><i class="fa fa-check"></i><b>E.3.2</b> Microbiome</a></li>
</ul></li>
<li class="chapter" data-level="E.4" data-path="models-output.html"><a href="models-output.html#howell"><i class="fa fa-check"></i><b>E.4</b> Howell</a>
<ul>
<li class="chapter" data-level="E.4.1" data-path="models-output.html"><a href="models-output.html#genes-3"><i class="fa fa-check"></i><b>E.4.1</b> Genes</a></li>
<li class="chapter" data-level="E.4.2" data-path="models-output.html"><a href="models-output.html#microbiome-5"><i class="fa fa-check"></i><b>E.4.2</b> Microbiome</a></li>
</ul></li>
</ul></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Data integration in inflammatory bowel disease</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div class = "print">
  If you want to download or save this thesis you can use <a href="https://thesis.llrs.dev/IBD_integration_Lluis_Revilla.pdf">the pdf</a> link or the <a href="https://thesis.llrs.dev/IBD_integration_Lluis_Revilla.epub">epub format</a> too.
  The thesis has many links to make it easier to reach external resources, if printed they will display as blue text.
  </div>
<div id="software" class="section level1 hasAnchor" number="9">
<h1><span class="header-section-number">B</span> Software<a href="software.html#software" class="anchor-section" aria-label="Anchor link to header"></a></h1>
<p>In the years since this thesis was first embarked upon, several software pieces have been developed, including some packages.
We listed them here for easier retrieval.
They are provided below in two ways, the first with a brief explanation and the second ordered by the relevant software used for each analysis.</p>
<div id="star" class="section level2 hasAnchor" number="9.1">
<h2><span class="header-section-number">B.1</span> STAR<a href="software.html#star" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>The parameters and options used with STAR are as follows:</p>
<pre><code>STAR \
    --outSAMtype BAM SortedByCoordinate \
    --outFilterIntronMotifs RemoveNoncanonical \
    --outSAMattributes All \
    --outReadsUnmapped Fastx \
    --outSAMstrandField intronMotif \
    --outFilterScoreMinOverLread 0.5 \
    --outFilterMatchNminOverLread 0.5 \
    --outFilterType BySJout \
    --alignSJoverhangMin 8 \
    --alignSJDBoverhangMin 1 \
    --outFilterMismatchNmax 999 \
    --outFilterMismatchNoverLmax 0.04 \
    --genomeDir &quot;$genome/STAR&quot; \
    --limitBAMsortRAM 10000000000 \
    --runMode alignReads \
    --genomeLoad NoSharedMemory \
    --quantMode TranscriptomeSAM \
    --outFileNamePrefix $output \
    --runThreadN &quot;$threads&quot; \
    --readFilesCommand zcat \
    --readFilesIn &quot;$file1&quot; &quot;$file2&quot;</code></pre>
<p>The <code>$genome</code> is the path to the location on the computer where the genome is located, <code>$output</code> is the prefix of the output file, <code>$threads</code> is the number of threads used and <code>$file1</code> and <code>$file2</code> are the paired fastq files.</p>
</div>
<div id="rsem" class="section level2 hasAnchor" number="9.2">
<h2><span class="header-section-number">B.2</span> RSEM<a href="software.html#rsem" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>The code used for RSEM where <code>$threads</code> is the number of threads used, <code>$rseminp</code> is the input file in BAM format, <code>$genome</code> is the path to the location on the computer where the genome is located, and <code>$rsem</code> is the output file.</p>
<pre><code>rsem-calculate-expression \
    --quiet \
    --paired-end \
    -p &quot;$threads&quot;  \
    --estimate-rspd \
    --append-names \
    --no-bam-output \
    --bam &quot;$rseminp&quot; &quot;$genome/RSEM/RSEM&quot; &quot;$rsem&quot;
    
    </code></pre>
</div>
<div id="listed" class="section level2 hasAnchor" number="9.3">
<h2><span class="header-section-number">B.3</span> Listed<a href="software.html#listed" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>An improved/tested version of <a href="https://github.com/llrs/RGCCA"><code>RGCCA</code></a>, includes some modifications concerning internal functions to simplify the maintenance, as well as, additional tests and better documentation.
Moreover, it has been modified such that it is possible to use a vector of models whereby the model of the first dimension is not the same as the model on the second dimension (Mathematically speaking, we cannot attest to its coherence, but from a biological standpoint we believe such a version of RGCCA might prove very useful).</p>
<p>We coded the package <a href="https:github.com/llrs/inteRmodel">inteRmodel</a> to make the bootstrapping and model selection for <code>RGCCA</code> easier and more readily accessible.</p>
<p>A package to assist in batch design in order to avoid batch effects - see <a href="https://cran.r-project.org/package=experDesign">experDesign</a> and its corresponding website on <a href="https://github.com/llrs/experDesign">GitHub</a>.</p>
<p>Explore the effects of hyperparameters on <a href="glossary.html#acronyms_RGCCA">RGCCA</a> on the provided dataset of <a href="https://github.com/llrs/gliomaData">gliomaData</a> (originally provided <a href="http://biodev.cea.fr/sgcca/">here</a>) available at the <a href="https://github.com/llrs/sgcca_hyperparameters">sgcca_hyperparameters</a> repository.</p>
<p>We utilized a pouchitis cohort published in this <a href="https://dx.doi.org/10.1186/s13059-015-0637-x">article</a><span class="citation">[<a href="#ref-morgan2015" role="doc-biblioref">150</a>]</span>, which was used to compare the effectiveness of our method with other’s dataset.
The code used can be found at <a href="https://github.com/llrs/pouchitis">this repository</a>.</p>
<p>Some functions used to explore the TRIM dataset were incorporated into the <a href="https://github.com/llrs/integration-helper">integration</a> package.This includes functions for correlation, network analysis, enrichment, and normalization of metadata, among other components…</p>
<p>We developed a package to analyze both sets and fuzzy sets; see <a href="https://github.com/rOpenSci/BaseSet">BaseSet</a>, which is based on what we learned from a previous iteration of the <a href="https://github.com/llrs/GSEAdv" title="GSEAdv package">GSEAdv package</a>.
This package was intended to be used with those probabilities that arise from bootstrapping the models.
However, due to the prolonged calculation times required, ultimately it was not used.</p>
<p>To analyze the <a href="glossary.html#acronyms_BARCELONA">BARCELONA</a> cohort (also designated antiTNF) a <a href="https://github.com/llrs/Barcelona" title="Barcelona repository">different repository</a> was created in order to analyze the data using the previously developed packages.</p>
</div>
<div id="by-projectpublication" class="section level2 hasAnchor" number="9.4">
<h2><span class="header-section-number">B.4</span> By project/publication<a href="software.html#by-projectpublication" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>All of the code underlying our analysis of the publications is available (in its messed state and complicated history) as well as a brief description of the code used:</p>
<p><a href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0246367">Multi-omic modelling of inflammatory bowel disease with regularized canonical correlation analysis</a>:</p>
<ul>
<li><a href="https://github.com/llrs/TRIM/" title="HSCT dataset exploration code on github">TRIM</a>: Data cleaning with the sample, dataset, explore several methods…</li>
<li><a href="https://github.com/llrs/sgcca_hyperparameters/" title="RGCCA exploration code on github">Puget’s</a>: Explore the effects of the hyperparameters on <code>RGCCA</code> with the provided dataset.</li>
<li><a href="https://github.com/llrs/inteRmodel/" title="Integration package on github">inteRmodel</a>: Package for easily reproducing the methodology developed with TRIM.</li>
<li><a href="https://github.com/llrs/pouchitis/" title="Pouchitis code on github">Morgan’s</a>: Work with the pouchitis cohort used in this article.</li>
<li><a href="https://github.com/llrs/Uncoupling/" title="Uncoupling code on github">Häsler’s</a>: Work with the <a href="glossary.html#acronyms_UC">UC</a>/<a href="glossary.html#acronyms_CD">CD</a> dataset used in this article.</li>
<li><a href="https://github.com/llrs/integration-helper/" title="Integration package on github">integration</a>: Package that incorporates functions we wrote or used for the different aspects of exploring the TRIM dataset are published here.</li>
</ul>
<p>BaseSet:</p>
<ul>
<li><a href="https://cran.r-project.org/package=BaseSet" title="BaseSet landing page on CRAN">BaseSet</a>: Fuzzy logic implementation, available at <a href="https://github.com/ropensci/BaseSet/" title="BaseSet git repository">rOpenSci too</a> see also its corresponding <a href="https://docs.ropensci.org/BaseSet/" title="BaseSet">documentation website</a> .</li>
</ul>
<p>experDesign:</p>
<ul>
<li><a href="https://cran.r-project.org/package=experDesign" title="experDesign landing page on CRAN">experDesign</a>: Can assist in the design of batch experiments; also with a <a href="https://experDesign.llrs.dev/" title="experDesign">documentation website too</a>.</li>
</ul>
<p>BARCELONA:</p>
<ul>
<li><a href="https://github.com/llrs/Barcelona/">BARCELONA</a>: Code for analyzing the <a href="glossary.html#acronyms_BARCELONA">BARCELONA</a>’s dataset</li>
</ul>
<p>Validation:</p>
<ul>
<li><a href="https://github.com/llrs/howell_2018/">Howell’s</a>: Code to work with Howell’s 2018 dataset.</li>
<li><a href="https://github.com/llrs/cristian_mark/">Cristian’s</a>: Code to work with Cristian’s 2020 dataset.</li>
</ul>
</div>
</div>
<h3>References</h3>
<div id="refs" class="references csl-bib-body">
<div id="ref-morgan2015" class="csl-entry">
150. Morgan XC, Kabakchiev B, Waldron L, Tyler AD, Tickle TL, Milgrom R, et al. <a href="https://doi.org/10.1186/s13059-015-0637-x">Associations between host gene expression, the mucosal microbiome, and clinical outcome in the pelvic pouch of patients with inflammatory bowel disease</a>. Genome Biology. 2015;16:67.
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="online-resources.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="articles.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": {
"href": "https://github.com/llrs/thesis/"
},
"facebook": false,
"twitter": {
"href": "https://twitter.com/intent/tweet?text=Nice%20read%20of%20%40Lluis_Revilla%20thesis"
},
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/llrs/thesis/edit/master/98_appendix.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": "https://github.com/llrs/thesis/blob/master/98_appendix.Rmd",
"text": "View"
},
"download": ["IBD_integration_Lluis_Revilla.pdf", "IBD_integration_Lluis_Revilla.epub"],
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": false,
"toc_float": true,
"toc_appendix": true,
"toc_depth": 2,
"scroll_highlight": true
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
